{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# This python notebook annotate the bioconda recipes using the ids from bio.tools in the internal local path to the \n",
    "# in bioconda recipes. Two different input should be provided: \n",
    "# @param bioconda_recipes_path the local path to the bioconda recipes, this can be download using the git clone command\n",
    "# @param mapping_file is the local path to the mapping file generated with the notebook biotools-biocontainers-venn.ipynb\n",
    "\n",
    "\n",
    "import glob2\n",
    "import os\n",
    "import sys\n",
    "\n",
    "from pandas import DataFrame, read_csv\n",
    "\n",
    "from ruamel.yaml import YAML\n",
    "import ruamel.yaml\n",
    "\n",
    "from yaml.constructor import ConstructorError\n",
    "from yaml.scanner import ScannerError"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def init_yaml_parser():\n",
    "    yaml = YAML(typ='jinja2')\n",
    "    yaml.allow_duplicate_keys = True\n",
    "    yaml.indent(mapping=2, sequence=4, offset=2)\n",
    "    yaml.preserve_quotes = True\n",
    "    yaml.width = 255\n",
    "    yaml.boolean_representation = ['False', 'True']\n",
    "    return yaml"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_bioconda_yalm(bioconda_path, yaml_parser):\n",
    "    yaml_files = dict()\n",
    "    matches = glob2.glob(bioconda_path + '/**/*.yaml', with_matches=True)\n",
    "    if len(matches) > 0:\n",
    "        for file_name in matches:\n",
    "            if (os.path.isfile(file_name[0])):\n",
    "                # print(file_name[0])\n",
    "                path_list = file_name[0].replace(bioconda_path, \"\").split(os.sep)\n",
    "                software_name = path_list[1]\n",
    "                try:\n",
    "                    with open(file_name[0]) as fp:\n",
    "                        yalm_file = yaml_parser.load(fp)\n",
    "                    yaml_files[software_name] = (file_name[0], yalm_file)\n",
    "                    #print(yaml_files[software_name])\n",
    "                except (ScannerError, ConstructorError, TypeError, AttributeError) as error:\n",
    "                    print(\"ERROR | \" + software_name + \"\\t\" + file_name[0])\n",
    "    return yaml_files         "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_annotation_mapping(mapping_file):\n",
    "    mapping = read_csv(mapping_file)\n",
    "    return mapping    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def annotate_yaml(yaml_file, biotool, yaml_parser):\n",
    "    path_file = yaml_file[0]\n",
    "    print(yaml_file)\n",
    "    yaml_file_content = yaml_file[1]\n",
    "    if yaml_file_content.get('extra') is None:\n",
    "        yaml_file_content['extra'] = {}\n",
    "    else:\n",
    "        print(\"EXTRA has been found: \" + biotool) \n",
    "    yaml_file_content['extra']['identifiers'] = {}   \n",
    "    tmp = ruamel.yaml.comments.CommentedSeq(['http://bio.tools/' + biotool])\n",
    "    tmp.fa.set_block_style()\n",
    "    yaml_file_content['extra']['identifiers']['biotools'] = tmp\n",
    "\n",
    "    with open(path_file, 'w') as fp:\n",
    "        yaml_parser.dump(yaml_file_content, fp)\n",
    "\n",
    "    \n",
    "\n",
    "def annotate_yamls(yaml_dictonary, mappings, yaml):\n",
    "    for (idx, row) in mappings.iterrows():\n",
    "        if yaml_dictonary.get(row.bioconda) is not None:\n",
    "            print(yaml_dictonary.get(row.bioconda))\n",
    "            annotate_yaml(yaml_dictonary[row.bioconda], row.biotools, yaml)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/weeder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'weeder'), ('version', '2.0')])), ('source', CommentedMap([('fn', 'weeder2.0.tar.gz'), ('url', 'http://depot.galaxyproject.org/software/weeder/weeder_2.0_src_all.tar.gz'), ('md5', '3401675b24ca928b7cc54ce9ebf18b6a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'python'])])), ('test', CommentedMap([('commands', ['weeder2'])])), ('about', CommentedMap([('home', 'http://159.149.160.51/modtools/'), ('license', 'GPL3'), ('summary', 'Motif (transcription factor binding sites) discovery in sequences from coregulated genes of a single species. This is a new Weeder release rewritten to be faster and optimized for large ChIP-Seq data.')])), ('extra', CommentedMap([('notes', 'Includes a simple wrapper script to be able to run weeder from any directory. It passes arguments transparently to the weeder2 executable.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/weeder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'weeder'), ('version', '2.0')])), ('source', CommentedMap([('fn', 'weeder2.0.tar.gz'), ('url', 'http://depot.galaxyproject.org/software/weeder/weeder_2.0_src_all.tar.gz'), ('md5', '3401675b24ca928b7cc54ce9ebf18b6a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'python'])])), ('test', CommentedMap([('commands', ['weeder2'])])), ('about', CommentedMap([('home', 'http://159.149.160.51/modtools/'), ('license', 'GPL3'), ('summary', 'Motif (transcription factor binding sites) discovery in sequences from coregulated genes of a single species. This is a new Weeder release rewritten to be faster and optimized for large ChIP-Seq data.')])), ('extra', CommentedMap([('notes', 'Includes a simple wrapper script to be able to run weeder from any directory. It passes arguments transparently to the weeder2 executable.')]))]))\n",
      "EXTRA has been found: weeder\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motifrg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motifrg'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'motifRG_1.18.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-motifrg/bioconductor-motifrg_1.18.0_src_all.tar.gz'), ('md5', '2f1dab8afccdc3077109dcb514945f3b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-seqlogo', 'bioconductor-bsgenome', 'bioconductor-xvector', 'bioconductor-bsgenome.hsapiens.ucsc.hg19']), ('run', ['r-base', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-seqlogo', 'bioconductor-bsgenome', 'bioconductor-xvector', 'bioconductor-bsgenome.hsapiens.ucsc.hg19'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'motifRG\\')\"'])])), ('about', CommentedMap([('home', 'https://www.bioconductor.org/packages/release/bioc/html/motifRG.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for discriminative motif discovery using regression methods')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motifrg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motifrg'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'motifRG_1.18.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-motifrg/bioconductor-motifrg_1.18.0_src_all.tar.gz'), ('md5', '2f1dab8afccdc3077109dcb514945f3b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-seqlogo', 'bioconductor-bsgenome', 'bioconductor-xvector', 'bioconductor-bsgenome.hsapiens.ucsc.hg19']), ('run', ['r-base', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-seqlogo', 'bioconductor-bsgenome', 'bioconductor-xvector', 'bioconductor-bsgenome.hsapiens.ucsc.hg19'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'motifRG\\')\"'])])), ('about', CommentedMap([('home', 'https://www.bioconductor.org/packages/release/bioc/html/motifRG.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for discriminative motif discovery using regression methods')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/muscle/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '1b7c9661f275a82d3cf708f923736bf8'), ('url', 'http://www.drive5.com/muscle/muscle_src_3.8.1551.tar.gz'), ('patches', ['osx-makefile.patch'])])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['muscle -version'])])), ('about', CommentedMap([('home', 'http://www.drive5.com/muscle/'), ('license', 'http://www.drive5.com/muscle/manual/license.html'), ('summary', 'MUSCLE: multiple sequence alignment with high accuracy and high throughput')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/muscle/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '1b7c9661f275a82d3cf708f923736bf8'), ('url', 'http://www.drive5.com/muscle/muscle_src_3.8.1551.tar.gz'), ('patches', ['osx-makefile.patch'])])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['muscle -version'])])), ('about', CommentedMap([('home', 'http://www.drive5.com/muscle/'), ('license', 'http://www.drive5.com/muscle/manual/license.html'), ('summary', 'MUSCLE: multiple sequence alignment with high accuracy and high throughput')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genonets/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genonets'), ('version', '1.1.6')])), ('source', CommentedMap([('fn', 'genonets-1.1.6.tar.gz'), ('url', 'https://pypi.python.org/packages/c5/fa/7c26ef0b264b1217184dcbc20b5eed3abea107484a20c756197f97ffadf6/genonets-1.1.6.tar.gz'), ('md5', 'a3ad65614c18f8ec436325e9a08e3515')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'python-igraph >=0.6', 'numpy >=1.8.2']), ('run', ['python', 'python-igraph >=0.6', 'numpy >=1.8.2'])])), ('test', CommentedMap([('imports', ['genonets'])])), ('about', CommentedMap([('home', 'https://github.com/fkhalid/genonets'), ('license', 'MIT License'), ('summary', 'Framework for creating and analyzing genotype networks from data.'), ('license_family', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genonets/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genonets'), ('version', '1.1.6')])), ('source', CommentedMap([('fn', 'genonets-1.1.6.tar.gz'), ('url', 'https://pypi.python.org/packages/c5/fa/7c26ef0b264b1217184dcbc20b5eed3abea107484a20c756197f97ffadf6/genonets-1.1.6.tar.gz'), ('md5', 'a3ad65614c18f8ec436325e9a08e3515')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'python-igraph >=0.6', 'numpy >=1.8.2']), ('run', ['python', 'python-igraph >=0.6', 'numpy >=1.8.2'])])), ('test', CommentedMap([('imports', ['genonets'])])), ('about', CommentedMap([('home', 'https://github.com/fkhalid/genonets'), ('license', 'MIT License'), ('summary', 'Framework for creating and analyzing genotype networks from data.'), ('license_family', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dsk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dsk'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'dsk-v2.2.0-Source.tar.gz'), ('url', 'https://github.com/GATB/dsk/releases/download/v2.2.0/dsk-v2.2.0-Source.tar.gz'), ('md5', 'dbdfe583b45e78f965e6f78cf2c9628c'), ('patches', ['install_paths.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'llvm', 'zlib', 'cppunit']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['dsk -h'])])), ('about', CommentedMap([('home', 'https://github.com/GATB/dsk/'), ('license', 'AGPL-3.0'), ('license_file', 'LICENSE'), ('summary', 'DSK is a k-mer counter for reads or genomes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dsk/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dsk'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'dsk-v2.2.0-Source.tar.gz'), ('url', 'https://github.com/GATB/dsk/releases/download/v2.2.0/dsk-v2.2.0-Source.tar.gz'), ('md5', 'dbdfe583b45e78f965e6f78cf2c9628c'), ('patches', ['install_paths.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'llvm', 'zlib', 'cppunit']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['dsk -h'])])), ('about', CommentedMap([('home', 'https://github.com/GATB/dsk/'), ('license', 'AGPL-3.0'), ('license_file', 'LICENSE'), ('summary', 'DSK is a k-mer counter for reads or genomes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bionet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bionet'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'BioNet_1.34.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BioNet_1.34.0.tar.gz'), ('md5', '3fd2c44a09b7c3914d553ca75b05c153')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base', 'r-igraph >=1.0.1']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base', 'r-igraph >=1.0.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BioNet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BioNet.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package provides functions for the integrated analysis of protein-protein interaction networks and the detection of functional modules. Different datasets can be integrated into the network by assigning p-values of statistical tests to the nodes of the network. E.g. p-values obtained from the differential expression of the genes from an Affymetrix array are assigned to the nodes of the network. By fitting a beta-uniform mixture model and calculating scores from the p-values, overall scores of network regions can be calculated and an integer linear programming algorithm identifies the maximum scoring subnetwork.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bionet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bionet'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'BioNet_1.34.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BioNet_1.34.0.tar.gz'), ('md5', '3fd2c44a09b7c3914d553ca75b05c153')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base', 'r-igraph >=1.0.1']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base', 'r-igraph >=1.0.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BioNet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BioNet.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package provides functions for the integrated analysis of protein-protein interaction networks and the detection of functional modules. Different datasets can be integrated into the network by assigning p-values of statistical tests to the nodes of the network. E.g. p-values obtained from the differential expression of the genes from an Affymetrix array are assigned to the nodes of the network. By fitting a beta-uniform mixture model and calculating scores from the p-values, overall scores of network regions can be calculated and an integer linear programming algorithm identifies the maximum scoring subnetwork.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylkit'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'methylKit_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/methylKit_1.0.0.tar.gz'), ('md5', '3612ce6bf0b748d8869a4a16c57265dc'), ('patches', ['rpath.patch'])])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-fastseg', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.18.1', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-rhtslib', 'bioconductor-zlibbioc', 'r-base', 'r-data.table >=1.9.6', 'r-emdbook', 'r-gtools', 'r-mclust', 'r-r.utils', 'r-rcpp', 'r-kernsmooth', 'gcc', 'llvm']), ('run', ['bioconductor-fastseg', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.18.1', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-rhtslib', 'bioconductor-zlibbioc', 'r-base', 'r-data.table >=1.9.6', 'r-emdbook', 'r-gtools', 'r-mclust', 'r-r.utils', 'r-rcpp', 'r-kernsmooth', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'methylKit\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/methylKit.html'), ('license', 'Artistic-2.0'), ('summary', 'methylKit is an R package for DNA methylation analysis and annotation from high-throughput bisulfite sequencing. The package is designed to deal with sequencing data from RRBS and its variants, but also target-capture methods and whole genome bisulfite sequencing. It also has functions to analyze base-pair resolution 5hmC data from experimental protocols such as oxBS-Seq and TAB-Seq. Perl is needed to read SAM files only.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylkit'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'methylKit_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/methylKit_1.0.0.tar.gz'), ('md5', '3612ce6bf0b748d8869a4a16c57265dc'), ('patches', ['rpath.patch'])])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-fastseg', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.18.1', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-rhtslib', 'bioconductor-zlibbioc', 'r-base', 'r-data.table >=1.9.6', 'r-emdbook', 'r-gtools', 'r-mclust', 'r-r.utils', 'r-rcpp', 'r-kernsmooth', 'gcc', 'llvm']), ('run', ['bioconductor-fastseg', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.18.1', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-rhtslib', 'bioconductor-zlibbioc', 'r-base', 'r-data.table >=1.9.6', 'r-emdbook', 'r-gtools', 'r-mclust', 'r-r.utils', 'r-rcpp', 'r-kernsmooth', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'methylKit\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/methylKit.html'), ('license', 'Artistic-2.0'), ('summary', 'methylKit is an R package for DNA methylation analysis and annotation from high-throughput bisulfite sequencing. The package is designed to deal with sequencing data from RRBS and its variants, but also target-capture methods and whole genome bisulfite sequencing. It also has functions to analyze base-pair resolution 5hmC data from experimental protocols such as oxBS-Seq and TAB-Seq. Perl is needed to read SAM files only.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-titancna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-titancna'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'TitanCNA_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TitanCNA_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/TitanCNA_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/TitanCNA/TitanCNA_1.14.0_src_all.tar.gz']), ('md5', '989f84cb8d0d687859c4175cea634fdc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomeinfodb >=1.4.0', 'bioconductor-genomicranges >=1.20.5', 'bioconductor-iranges >=2.2.4', 'bioconductor-rsamtools >=1.20.4', 'r-base', 'r-foreach >=1.4.2']), ('run', ['bioconductor-genomeinfodb >=1.4.0', 'bioconductor-genomicranges >=1.20.5', 'bioconductor-iranges >=2.2.4', 'bioconductor-rsamtools >=1.20.4', 'r-base', 'r-foreach >=1.4.2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TitanCNA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TitanCNA.html'), ('license', 'GPL-3'), ('summary', 'Hidden Markov model to segment and predict regions of subclonal copy number alterations (CNA) and loss of heterozygosity (LOH), and estimate cellular prevalenece of clonal clusters in tumour whole genome sequencing data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-titancna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-titancna'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'TitanCNA_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TitanCNA_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/TitanCNA_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/TitanCNA/TitanCNA_1.14.0_src_all.tar.gz']), ('md5', '989f84cb8d0d687859c4175cea634fdc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomeinfodb >=1.4.0', 'bioconductor-genomicranges >=1.20.5', 'bioconductor-iranges >=2.2.4', 'bioconductor-rsamtools >=1.20.4', 'r-base', 'r-foreach >=1.4.2']), ('run', ['bioconductor-genomeinfodb >=1.4.0', 'bioconductor-genomicranges >=1.20.5', 'bioconductor-iranges >=2.2.4', 'bioconductor-rsamtools >=1.20.4', 'r-base', 'r-foreach >=1.4.2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TitanCNA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TitanCNA.html'), ('license', 'GPL-3'), ('summary', 'Hidden Markov model to segment and predict regions of subclonal copy number alterations (CNA) and loss of heterozygosity (LOH), and estimate cellular prevalenece of clonal clusters in tumour whole genome sequencing data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-roc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-roc'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'ROC_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ROC_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/ROC_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/ROC/ROC_1.52.0_src_all.tar.gz']), ('md5', '15d20f25f96473afc11ffe9ee9a73b93')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ROC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ROC.html'), ('license', 'Artistic-2.0'), ('summary', 'utilities for ROC, with uarray focus')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-roc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-roc'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'ROC_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ROC_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/ROC_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/ROC/ROC_1.52.0_src_all.tar.gz']), ('md5', '15d20f25f96473afc11ffe9ee9a73b93')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ROC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ROC.html'), ('license', 'Artistic-2.0'), ('summary', 'utilities for ROC, with uarray focus')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'igv'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'IGV_<{ version }}.zip'), ('url', 'http://data.broadinstitute.org/igv/projects/downloads/IGV_<{ version }}.zip'), ('sha256', '82a261e41f46d2490f2a490f8706575bd3bc07a4ccf9b38dcf6c6989fad64dae')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', None)])), ('about', CommentedMap([('home', 'http://www.broadinstitute.org/software/igv/home'), ('license', 'MIT'), ('summary', 'Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'igv'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'IGV_<{ version }}.zip'), ('url', 'http://data.broadinstitute.org/igv/projects/downloads/IGV_<{ version }}.zip'), ('sha256', '82a261e41f46d2490f2a490f8706575bd3bc07a4ccf9b38dcf6c6989fad64dae')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', None)])), ('about', CommentedMap([('home', 'http://www.broadinstitute.org/software/igv/home'), ('license', 'MIT'), ('summary', 'Integrative Genomics Viewer. Fast, efficient, scalable visualization tool for genomics data and annotations')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/corset/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/Oshlack/Corset/archive/version-1.06.tar.gz'), ('md5', 'decc3bc6b499d6d7ac448ff3683e07aa')])), ('requirements', CommentedMap([('build', ['gcc', 'samtools 0.1.19']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['corset 2>&1 | grep Usage'])])), ('about', CommentedMap([('home', 'https://github.com/Oshlack/Corset'), ('license', 'GPLv3'), ('summary', 'Software for clustering de novo assembled transcripts and counting overlapping reads.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/corset/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/Oshlack/Corset/archive/version-1.06.tar.gz'), ('md5', 'decc3bc6b499d6d7ac448ff3683e07aa')])), ('requirements', CommentedMap([('build', ['gcc', 'samtools 0.1.19']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['corset 2>&1 | grep Usage'])])), ('about', CommentedMap([('home', 'https://github.com/Oshlack/Corset'), ('license', 'GPLv3'), ('summary', 'Software for clustering de novo assembled transcripts and counting overlapping reads.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-suprahex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-suprahex'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'supraHex_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/supraHex_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/supraHex_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/supraHex/supraHex_1.14.0_src_all.tar.gz']), ('md5', '751bb08f19253c574cc51f9c46ec5eee')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-ape', 'r-base', 'r-hexbin', 'r-mass']), ('run', ['r-ape', 'r-base', 'r-hexbin', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'supraHex\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/supraHex.html'), ('license', 'GPL-2'), ('summary', 'A supra-hexagonal map is a giant hexagon on a 2-dimensional grid seamlessly consisting of smaller hexagons. It is supposed to train, analyse and visualise a high-dimensional omics input data. The supraHex is able to carry out gene clustering/meta-clustering and sample correlation, plus intuitive visualisations to facilitate exploratory analysis. More importantly, it allows for overlaying additional data onto the trained map to explore relations between input and additional data. So with supraHex, it is also possible to carry out multilayer omics data comparisons. Newly added utilities are advanced heatmap visualisation and tree-based analysis of sample relationships. Uniquely to this package, users can ultrafastly understand any tabular omics data, both scientifically and artistically, especially in a sample-specific fashion but without loss of information on large genes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-suprahex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-suprahex'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'supraHex_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/supraHex_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/supraHex_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/supraHex/supraHex_1.14.0_src_all.tar.gz']), ('md5', '751bb08f19253c574cc51f9c46ec5eee')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-ape', 'r-base', 'r-hexbin', 'r-mass']), ('run', ['r-ape', 'r-base', 'r-hexbin', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'supraHex\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/supraHex.html'), ('license', 'GPL-2'), ('summary', 'A supra-hexagonal map is a giant hexagon on a 2-dimensional grid seamlessly consisting of smaller hexagons. It is supposed to train, analyse and visualise a high-dimensional omics input data. The supraHex is able to carry out gene clustering/meta-clustering and sample correlation, plus intuitive visualisations to facilitate exploratory analysis. More importantly, it allows for overlaying additional data onto the trained map to explore relations between input and additional data. So with supraHex, it is also possible to carry out multilayer omics data comparisons. Newly added utilities are advanced heatmap visualisation and tree-based analysis of sample relationships. Uniquely to this package, users can ultrafastly understand any tabular omics data, both scientifically and artistically, especially in a sample-specific fashion but without loss of information on large genes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flash/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'flash'), ('version', '1.2.11')])), ('about', CommentedMap([('home', 'https://ccb.jhu.edu/software/FLASH/'), ('license', 'GPLv3+'), ('summary', 'Merge mates from fragments that are shorter than twice the read length')])), ('source', CommentedMap([('fn', 'FLASH-1.2.11.tar.gz'), ('md5', 'e4d355023a766afaaab2d62f912b605c'), ('url', 'http://downloads.sourceforge.net/project/flashpage/FLASH-1.2.11.tar.gz')])), ('test', CommentedMap([('commands', ['flash -v > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flash/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'flash'), ('version', '1.2.11')])), ('about', CommentedMap([('home', 'https://ccb.jhu.edu/software/FLASH/'), ('license', 'GPLv3+'), ('summary', 'Merge mates from fragments that are shorter than twice the read length')])), ('source', CommentedMap([('fn', 'FLASH-1.2.11.tar.gz'), ('md5', 'e4d355023a766afaaab2d62f912b605c'), ('url', 'http://downloads.sourceforge.net/project/flashpage/FLASH-1.2.11.tar.gz')])), ('test', CommentedMap([('commands', ['flash -v > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/blast/2.5.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'blast'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'ncbi-blast-<{ version }}+-x64-macosx.tar.gz'), ('url', 'http://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/<{ version }}/ncbi-blast-<{ version }}+-x64-macosx.tar.gz'), ('md5', '9a2ac418e3031ed266787ea06585d468'), ('patches', ['boost_106400.patch'])])), ('build', CommentedMap([('number', 2), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['boost <{CONDA_BOOST}}*', 'gcc']), ('run', ['boost <{CONDA_BOOST}}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['blastn -help', 'blastp -help'])])), ('about', CommentedMap([('home', 'http://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs'), ('license', 'Public Domain'), ('summary', 'BLAST+ is a new suite of BLAST tools that utilizes the NCBI C++ Toolkit.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/blast/2.5.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'blast'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'ncbi-blast-<{ version }}+-x64-macosx.tar.gz'), ('url', 'http://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/<{ version }}/ncbi-blast-<{ version }}+-x64-macosx.tar.gz'), ('md5', '9a2ac418e3031ed266787ea06585d468'), ('patches', ['boost_106400.patch'])])), ('build', CommentedMap([('number', 2), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['boost <{CONDA_BOOST}}*', 'gcc']), ('run', ['boost <{CONDA_BOOST}}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['blastn -help', 'blastp -help'])])), ('about', CommentedMap([('home', 'http://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs'), ('license', 'Public Domain'), ('summary', 'BLAST+ is a new suite of BLAST tools that utilizes the NCBI C++ Toolkit.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rontotools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rontotools'), ('version', '2.4.0')])), ('source', CommentedMap([('fn', 'ROntoTools_2.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ROntoTools_2.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/ROntoTools_2.4.0.tar.gz', 'https://depot.galaxyproject.org/software/ROntoTools/ROntoTools_2.4.0_src_all.tar.gz']), ('md5', '00ef46ebced2cb6b09f4bf7f0aa5d178')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-rgraphviz', 'r-base', 'r-boot']), ('run', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-rgraphviz', 'r-base', 'r-boot'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ROntoTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ROntoTools.html'), ('license', 'CC BY-NC-ND 4.0 + file LICENSE'), ('summary', 'Suite of tools for functional analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rontotools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rontotools'), ('version', '2.4.0')])), ('source', CommentedMap([('fn', 'ROntoTools_2.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ROntoTools_2.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/ROntoTools_2.4.0.tar.gz', 'https://depot.galaxyproject.org/software/ROntoTools/ROntoTools_2.4.0_src_all.tar.gz']), ('md5', '00ef46ebced2cb6b09f4bf7f0aa5d178')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-rgraphviz', 'r-base', 'r-boot']), ('run', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-rgraphviz', 'r-base', 'r-boot'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ROntoTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ROntoTools.html'), ('license', 'CC BY-NC-ND 4.0 + file LICENSE'), ('summary', 'Suite of tools for functional analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-interactivedisplaybase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-interactivedisplaybase'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'interactiveDisplayBase_1.12.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/interactiveDisplayBase_1.12.0.tar.gz'), ('md5', 'ef3fc69145648ea333677e4f8c3d0a0d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'r-base', 'r-shiny']), ('run', ['bioconductor-biocgenerics', 'r-base', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'interactiveDisplayBase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/interactiveDisplayBase.html'), ('license', 'Artistic-2.0'), ('summary', 'The interactiveDisplayBase package contains the the basic methods needed to generate interactive Shiny based display methods for Bioconductor objects.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-interactivedisplaybase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-interactivedisplaybase'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'interactiveDisplayBase_1.12.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/interactiveDisplayBase_1.12.0.tar.gz'), ('md5', 'ef3fc69145648ea333677e4f8c3d0a0d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'r-base', 'r-shiny']), ('run', ['bioconductor-biocgenerics', 'r-base', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'interactiveDisplayBase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/interactiveDisplayBase.html'), ('license', 'Artistic-2.0'), ('summary', 'The interactiveDisplayBase package contains the the basic methods needed to generate interactive Shiny based display methods for Bioconductor objects.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-aroma.light/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-aroma.light'), ('version', '3.4.0')])), ('source', CommentedMap([('fn', 'aroma.light_3.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/aroma.light_3.4.0.tar.gz'), ('md5', '3ef387057a6734dc03e7a8f6aced4803')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-matrixstats >=0.50.2', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.20.0', 'r-r.utils >=2.4.0']), ('run', ['r-base', 'r-matrixstats >=0.50.2', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.20.0', 'r-r.utils >=2.4.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'aroma.light\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/aroma.light.html'), ('license', 'GPL (>= 2)'), ('summary', 'Methods for microarray analysis that take basic data types such as matrices and lists of vectors.  These methods can be used standalone, be utilized in other packages, or be wrapped up in higher-level classes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-aroma.light/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-aroma.light'), ('version', '3.4.0')])), ('source', CommentedMap([('fn', 'aroma.light_3.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/aroma.light_3.4.0.tar.gz'), ('md5', '3ef387057a6734dc03e7a8f6aced4803')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-matrixstats >=0.50.2', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.20.0', 'r-r.utils >=2.4.0']), ('run', ['r-base', 'r-matrixstats >=0.50.2', 'r-r.methodss3 >=1.7.1', 'r-r.oo >=1.20.0', 'r-r.utils >=2.4.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'aroma.light\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/aroma.light.html'), ('license', 'GPL (>= 2)'), ('summary', 'Methods for microarray analysis that take basic data types such as matrices and lists of vectors.  These methods can be used standalone, be utilized in other packages, or be wrapped up in higher-level classes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/somatic-sniper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'somatic-sniper'), ('version', '1.0.5.0')])), ('source', CommentedMap([('fn', 'v1.0.5.0.tar.gz'), ('url', 'https://github.com/genome/somatic-sniper/archive/v1.0.5.0.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'zlib']), ('run', ['zlib'])])), ('test', CommentedMap([('commands', ['bam-somaticsniper -v'])])), ('about', CommentedMap([('home', 'https://github.com/genome/somatic-sniper/'), ('license', 'MIT'), ('summary', 'A tool to call somatic single nucleotide variants.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/somatic-sniper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'somatic-sniper'), ('version', '1.0.5.0')])), ('source', CommentedMap([('fn', 'v1.0.5.0.tar.gz'), ('url', 'https://github.com/genome/somatic-sniper/archive/v1.0.5.0.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'zlib']), ('run', ['zlib'])])), ('test', CommentedMap([('commands', ['bam-somaticsniper -v'])])), ('about', CommentedMap([('home', 'https://github.com/genome/somatic-sniper/'), ('license', 'MIT'), ('summary', 'A tool to call somatic single nucleotide variants.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-xcms/1.48.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-xcms'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'xcms_1.48.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/xcms_1.48.0.tar.gz'), ('md5', '83adb6afdb8f6d819bd6d37c7bf583f7')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'xcms\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/xcms.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data. Imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files. Preprocesses data for high-throughput, untargeted analyte profiling.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-xcms/1.48.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-xcms'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'xcms_1.48.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/xcms_1.48.0.tar.gz'), ('md5', '83adb6afdb8f6d819bd6d37c7bf583f7')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-mzr >=1.1.6', 'bioconductor-protgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'xcms\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/xcms.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'Framework for processing and visualization of chromatographically separated and single-spectra mass spectral data. Imports from AIA/ANDI NetCDF, mzXML, mzData and mzML files. Preprocesses data for high-throughput, untargeted analyte profiling.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-aspli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-aspli'), ('version', '1.2.3')])), ('source', CommentedMap([('fn', 'ASpli_1.2.3.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASpli_1.2.3.tar.gz', 'https://depot.galaxyproject.org/software/ASpli/ASpli_1.2.3_src_all.tar.gz']), ('md5', '8f896b652087ca75b109763ffa670454')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-deseq2', 'bioconductor-dexseq', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-deseq2', 'bioconductor-dexseq', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASpli\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASpli.html'), ('license', 'GPL'), ('summary', 'Integrative pipeline for the analysis of alternative splicing using RNA-seq.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-aspli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-aspli'), ('version', '1.2.3')])), ('source', CommentedMap([('fn', 'ASpli_1.2.3.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASpli_1.2.3.tar.gz', 'https://depot.galaxyproject.org/software/ASpli/ASpli_1.2.3_src_all.tar.gz']), ('md5', '8f896b652087ca75b109763ffa670454')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-deseq2', 'bioconductor-dexseq', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-deseq2', 'bioconductor-dexseq', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASpli\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASpli.html'), ('license', 'GPL'), ('summary', 'Integrative pipeline for the analysis of alternative splicing using RNA-seq.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pviz'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'Pviz_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Pviz_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/Pviz_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/Pviz/Pviz_1.10.0_src_all.tar.gz']), ('md5', '55a484071ac1974537ab7b70163b8896')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-genomicranges', 'bioconductor-gviz >=1.7.10', 'bioconductor-iranges', 'r-base', 'r-data.table']), ('run', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-genomicranges', 'bioconductor-gviz >=1.7.10', 'bioconductor-iranges', 'r-base', 'r-data.table'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Pviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Pviz.html'), ('license', 'Artistic-2.0'), ('summary', 'Pviz adapts the Gviz package for protein sequences and data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pviz'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'Pviz_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Pviz_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/Pviz_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/Pviz/Pviz_1.10.0_src_all.tar.gz']), ('md5', '55a484071ac1974537ab7b70163b8896')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-genomicranges', 'bioconductor-gviz >=1.7.10', 'bioconductor-iranges', 'r-base', 'r-data.table']), ('run', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-genomicranges', 'bioconductor-gviz >=1.7.10', 'bioconductor-iranges', 'r-base', 'r-data.table'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Pviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Pviz.html'), ('license', 'Artistic-2.0'), ('summary', 'Pviz adapts the Gviz package for protein sequences and data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-lsd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-lsd'), ('version', '3.0')])), ('source', CommentedMap([('fn', 'LSD_3.0.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/LSD_3.0.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/LSD/LSD_3.0.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'LSD\\')\"', '\"%R%\" -e \"library(\\'LSD\\')\"'])])), ('about', CommentedMap([('license', 'Unlimited'), ('summary', <ruamel.yaml.comments.TaggedScalar object at 0x120fd3390>)]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-lsd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-lsd'), ('version', '3.0')])), ('source', CommentedMap([('fn', 'LSD_3.0.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/LSD_3.0.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/LSD/LSD_3.0.tar.gz'])])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'LSD\\')\"', '\"%R%\" -e \"library(\\'LSD\\')\"'])])), ('about', CommentedMap([('license', 'Unlimited'), ('summary', <ruamel.yaml.comments.TaggedScalar object at 0x120fd3390>)]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/antarna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'antarna'), ('version', '2.0.1.2')])), ('source', CommentedMap([('fn', 'antarna-2.0.1.2.tar.gz'), ('url', 'https://github.com/RobertKleinkauf/antarna/archive/v2.0.1.2.tar.gz'), ('sha256', '537d8e7cf94adf0d58ea3e381e21d9ff2fedc03a745abd4fbf4e48250cf2b4f3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'uuid', 'networkx', 'scipy']), ('run', ['python', 'numpy', 'uuid', 'networkx', 'scipy', 'viennarna'])])), ('test', CommentedMap([('commands', ['antarna.py -tGC 0.5 MFE -Cstr \"(((((...)))))\"', 'antarna.py -tGC 0.5 DP --accuracy \"((((((....)))))).((((((....)))))) UB 0.5\" --accuracy \"((((((((((((((.....)))))))))))))) UB 0.5\"', 'antarna.py -tGC 0.5 DP -Cstr \"(((((...)))))................\" --diff-accuracy \"................(((((...))))) UB 0.0 B 1.0\" --diff-accuracy \"........(((((...)))))........ UB 1.0 B 0.0\"'])])), ('about', CommentedMap([('home', 'https://github.com/RobertKleinkauf/antarna'), ('license', 'The MIT License (MIT)'), ('license_file', 'LICENSE'), ('summary', 'antaRNA is a python based implementation of ant-colony optimization of the RNA inverse folding problem.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/antarna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'antarna'), ('version', '2.0.1.2')])), ('source', CommentedMap([('fn', 'antarna-2.0.1.2.tar.gz'), ('url', 'https://github.com/RobertKleinkauf/antarna/archive/v2.0.1.2.tar.gz'), ('sha256', '537d8e7cf94adf0d58ea3e381e21d9ff2fedc03a745abd4fbf4e48250cf2b4f3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'uuid', 'networkx', 'scipy']), ('run', ['python', 'numpy', 'uuid', 'networkx', 'scipy', 'viennarna'])])), ('test', CommentedMap([('commands', ['antarna.py -tGC 0.5 MFE -Cstr \"(((((...)))))\"', 'antarna.py -tGC 0.5 DP --accuracy \"((((((....)))))).((((((....)))))) UB 0.5\" --accuracy \"((((((((((((((.....)))))))))))))) UB 0.5\"', 'antarna.py -tGC 0.5 DP -Cstr \"(((((...)))))................\" --diff-accuracy \"................(((((...))))) UB 0.0 B 1.0\" --diff-accuracy \"........(((((...)))))........ UB 1.0 B 0.0\"'])])), ('about', CommentedMap([('home', 'https://github.com/RobertKleinkauf/antarna'), ('license', 'The MIT License (MIT)'), ('license_file', 'LICENSE'), ('summary', 'antaRNA is a python based implementation of ant-colony optimization of the RNA inverse folding problem.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-asafe/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-asafe'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'ASAFE_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASAFE_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/ASAFE_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/ASAFE/ASAFE_1.2.0_src_all.tar.gz']), ('md5', '7d710be786263956d727cee14f71d7f6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASAFE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASAFE.html'), ('license', 'Artistic-2.0'), ('summary', \"Given admixed individuals' bi-allelic SNP genotypes and ancestry pairs (where each ancestry can take one of three values) for multiple SNPs, perform an EM algorithm to deal with the fact that SNP genotypes are unphased with respect to ancestry pairs, in order to estimate ancestry-specific allele frequencies for all SNPs.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-asafe/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-asafe'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'ASAFE_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASAFE_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/ASAFE_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/ASAFE/ASAFE_1.2.0_src_all.tar.gz']), ('md5', '7d710be786263956d727cee14f71d7f6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASAFE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASAFE.html'), ('license', 'Artistic-2.0'), ('summary', \"Given admixed individuals' bi-allelic SNP genotypes and ancestry pairs (where each ancestry can take one of three values) for multiple SNPs, perform an EM algorithm to deal with the fact that SNP genotypes are unphased with respect to ancestry pairs, in order to estimate ancestry-specific allele frequencies for all SNPs.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowworkspace/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowworkspace'), ('version', '3.24.4')])), ('source', CommentedMap([('fn', 'flowWorkspace_3.24.4.tar.gz'), ('url', 'https://bioconductor.org/packages/release/bioc/src/contrib/flowWorkspace_3.24.4.tar.gz'), ('md5', '76ca821e29806d133f651e00bc06b5c0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore >=1.39.9', 'bioconductor-ncdfflow >=2.19.5', 'r-base', 'r-rcpp', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-graph', 'r-bh >=1.62.0.1', 'r-lattice', 'bioconductor-rbgl', 'r-xml', 'r-gridextra', 'bioconductor-rgraphviz', 'r-data.table', 'r-dplyr', 'r-latticeextra', 'r-rcolorbrewer', 'r-stringr', 'r-scales', 'autoconf 2.69 pl5.*', 'automake 1.15 pl5.*']), ('run', ['libgcc', 'bioconductor-flowcore >=1.39.9', 'r-base', 'r-rcpp', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-graph', 'r-lattice', 'bioconductor-rbgl', 'r-xml', 'r-gridextra', 'bioconductor-rgraphviz', 'r-data.table', 'r-dplyr', 'r-latticeextra', 'r-rcolorbrewer', 'r-stringr', 'r-scales', 'bioconductor-ncdfflow >=2.19.5', 'bioconductor-flowviz'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowWorkspace\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowWorkspace.html'), ('license', 'Artistic-2.0'), ('summary', 'This package is designed to facilitate comparison of automated gating methods against manual gating done in flowJo. This package allows you to import basic flowJo workspaces into BioConductor and replicate the gating from flowJo using the flowCore functionality. Gating hierarchies, groups of samples, compensation, and transformation are performed so that the output matches the flowJo analysis')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowworkspace/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowworkspace'), ('version', '3.24.4')])), ('source', CommentedMap([('fn', 'flowWorkspace_3.24.4.tar.gz'), ('url', 'https://bioconductor.org/packages/release/bioc/src/contrib/flowWorkspace_3.24.4.tar.gz'), ('md5', '76ca821e29806d133f651e00bc06b5c0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore >=1.39.9', 'bioconductor-ncdfflow >=2.19.5', 'r-base', 'r-rcpp', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-graph', 'r-bh >=1.62.0.1', 'r-lattice', 'bioconductor-rbgl', 'r-xml', 'r-gridextra', 'bioconductor-rgraphviz', 'r-data.table', 'r-dplyr', 'r-latticeextra', 'r-rcolorbrewer', 'r-stringr', 'r-scales', 'autoconf 2.69 pl5.*', 'automake 1.15 pl5.*']), ('run', ['libgcc', 'bioconductor-flowcore >=1.39.9', 'r-base', 'r-rcpp', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-graph', 'r-lattice', 'bioconductor-rbgl', 'r-xml', 'r-gridextra', 'bioconductor-rgraphviz', 'r-data.table', 'r-dplyr', 'r-latticeextra', 'r-rcolorbrewer', 'r-stringr', 'r-scales', 'bioconductor-ncdfflow >=2.19.5', 'bioconductor-flowviz'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowWorkspace\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowWorkspace.html'), ('license', 'Artistic-2.0'), ('summary', 'This package is designed to facilitate comparison of automated gating methods against manual gating done in flowJo. This package allows you to import basic flowJo workspaces into BioConductor and replicate the gating from flowJo using the flowCore functionality. Gating hierarchies, groups of samples, compensation, and transformation are performed so that the output matches the flowJo analysis')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/raxml/8.2.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'raxml'), ('version', '8.2.9')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'raxml_v8.2.9.tar.gz'), ('url', 'https://github.com/stamatak/standard-RAxML/archive/v8.2.9.tar.gz'), ('md5', 'ba77f7df7ce5e2a7967decd9e4278a72')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['raxmlHPC -h > /dev/null', 'raxmlHPC-PTHREADS -h > /dev/null', 'raxmlHPC-SSE3 -h > /dev/null', 'raxmlHPC-PTHREADS-SSE3 -h > /dev/null'])])), ('about', CommentedMap([('home', 'http://sco.h-its.org/exelixis/web/software/raxml/index.html'), ('license', 'GPL'), ('summary', 'Phylogenetics - Randomized Axelerated Maximum Likelihood.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/raxml/8.2.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'raxml'), ('version', '8.2.9')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'raxml_v8.2.9.tar.gz'), ('url', 'https://github.com/stamatak/standard-RAxML/archive/v8.2.9.tar.gz'), ('md5', 'ba77f7df7ce5e2a7967decd9e4278a72')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['raxmlHPC -h > /dev/null', 'raxmlHPC-PTHREADS -h > /dev/null', 'raxmlHPC-SSE3 -h > /dev/null', 'raxmlHPC-PTHREADS-SSE3 -h > /dev/null'])])), ('about', CommentedMap([('home', 'http://sco.h-its.org/exelixis/web/software/raxml/index.html'), ('license', 'GPL'), ('summary', 'Phylogenetics - Randomized Axelerated Maximum Likelihood.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dirichletmultinomial/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dirichletmultinomial'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'DirichletMultinomial_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/DirichletMultinomial_1.16.0.tar.gz'), ('md5', '59f90d37be2dfd1763a6b8a384196ded')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/']), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'blas', 'bioconductor-biocgenerics', 'bioconductor-iranges', 'bioconductor-s4vectors', 'gsl <{CONDA_GSL}}*']), ('run', ['libgcc', 'bioconductor-biocgenerics', 'bioconductor-iranges', 'bioconductor-s4vectors', 'blas', 'gsl <{CONDA_GSL}}*'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DirichletMultinomial\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/DirichletMultinomial.html'), ('license', 'LGPL-3'), ('summary', 'Dirichlet-multinomial mixture models can be used to describe variability in microbial metagenomic data. This package is an interface to code originally made available by Holmes, Harris, and Quince, 2012, PLoS ONE 7(2): 1-15, as discussed further in the man page for this package, ?DirichletMultinomial.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dirichletmultinomial/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dirichletmultinomial'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'DirichletMultinomial_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/DirichletMultinomial_1.16.0.tar.gz'), ('md5', '59f90d37be2dfd1763a6b8a384196ded')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/']), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'blas', 'bioconductor-biocgenerics', 'bioconductor-iranges', 'bioconductor-s4vectors', 'gsl <{CONDA_GSL}}*']), ('run', ['libgcc', 'bioconductor-biocgenerics', 'bioconductor-iranges', 'bioconductor-s4vectors', 'blas', 'gsl <{CONDA_GSL}}*'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DirichletMultinomial\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/DirichletMultinomial.html'), ('license', 'LGPL-3'), ('summary', 'Dirichlet-multinomial mixture models can be used to describe variability in microbial metagenomic data. This package is an interface to code originally made available by Holmes, Harris, and Quince, 2012, PLoS ONE 7(2): 1-15, as discussed further in the man page for this package, ?DirichletMultinomial.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/htslib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'htslib'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'htslib-<{ version }}.tar.bz2'), ('url', 'https://github.com/samtools/htslib/releases/download/<{ version }}/htslib-<{ version }}.tar.bz2'), ('md5', 'e7c269d519a0c01de680bbc72fc76a57')])), ('requirements', CommentedMap([('build', ['curl', 'bzip2', 'xz']), ('run', ['curl', 'bzip2', 'xz'])])), ('about', CommentedMap([('home', 'https://github.com/samtools/htslib'), ('license', 'MIT'), ('summary', 'C library for high-throughput sequencing data formats.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/htslib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'htslib'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'htslib-<{ version }}.tar.bz2'), ('url', 'https://github.com/samtools/htslib/releases/download/<{ version }}/htslib-<{ version }}.tar.bz2'), ('md5', 'e7c269d519a0c01de680bbc72fc76a57')])), ('requirements', CommentedMap([('build', ['curl', 'bzip2', 'xz']), ('run', ['curl', 'bzip2', 'xz'])])), ('about', CommentedMap([('home', 'https://github.com/samtools/htslib'), ('license', 'MIT'), ('summary', 'C library for high-throughput sequencing data formats.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mirsynergy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mirsynergy'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'Mirsynergy_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Mirsynergy_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/Mirsynergy_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/Mirsynergy/Mirsynergy_1.12.0_src_all.tar.gz']), ('md5', '14f46bc5e653d284295e7baa11857791')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-igraph', 'r-matrix', 'r-rcolorbrewer', 'r-reshape', 'r-scales']), ('run', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-igraph', 'r-matrix', 'r-rcolorbrewer', 'r-reshape', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Mirsynergy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Mirsynergy.html'), ('license', 'GPL-2'), ('summary', 'Detect synergistic miRNA regulatory modules by overlapping neighbourhood expansion.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mirsynergy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mirsynergy'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'Mirsynergy_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Mirsynergy_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/Mirsynergy_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/Mirsynergy/Mirsynergy_1.12.0_src_all.tar.gz']), ('md5', '14f46bc5e653d284295e7baa11857791')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-igraph', 'r-matrix', 'r-rcolorbrewer', 'r-reshape', 'r-scales']), ('run', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-igraph', 'r-matrix', 'r-rcolorbrewer', 'r-reshape', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Mirsynergy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Mirsynergy.html'), ('license', 'GPL-2'), ('summary', 'Detect synergistic miRNA regulatory modules by overlapping neighbourhood expansion.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-a4core/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-a4core'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'a4Core_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/a4Core_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/a4Core_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/a4Core/a4Core_1.24.0_src_all.tar.gz']), ('md5', 'd7f79c350ae0a6175f2ecc9a337ca61f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-glmnet']), ('run', ['bioconductor-biobase', 'r-base', 'r-glmnet'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'a4Core\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/a4Core.html'), ('license', 'GPL-3'), ('summary', 'Automated Affymetrix Array Analysis Core Package')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-a4core/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-a4core'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'a4Core_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/a4Core_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/a4Core_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/a4Core/a4Core_1.24.0_src_all.tar.gz']), ('md5', 'd7f79c350ae0a6175f2ecc9a337ca61f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-glmnet']), ('run', ['bioconductor-biobase', 'r-base', 'r-glmnet'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'a4Core\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/a4Core.html'), ('license', 'GPL-3'), ('summary', 'Automated Affymetrix Array Analysis Core Package')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-makecdfenv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-makecdfenv'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'makecdfenv_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/makecdfenv_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/makecdfenv_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/makecdfenv/makecdfenv_1.52.0_src_all.tar.gz']), ('md5', '6471ca53c57eb2c9d3580e7306337790')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-affyio', 'bioconductor-biobase', 'bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-affyio', 'bioconductor-biobase', 'bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'makecdfenv\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/makecdfenv.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package has two functions. One reads a Affymetrix chip description file (CDF) and creates a hash table environment containing the location/probe set membership mapping. The other creates a package that automatically loads that environment.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-makecdfenv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-makecdfenv'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'makecdfenv_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/makecdfenv_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/makecdfenv_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/makecdfenv/makecdfenv_1.52.0_src_all.tar.gz']), ('md5', '6471ca53c57eb2c9d3580e7306337790')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-affyio', 'bioconductor-biobase', 'bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-affyio', 'bioconductor-biobase', 'bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'makecdfenv\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/makecdfenv.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package has two functions. One reads a Affymetrix chip description file (CDF) and creates a hash table environment containing the location/probe set membership mapping. The other creates a package that automatically loads that environment.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-slqpcr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-slqpcr'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'SLqPCR_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SLqPCR_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/SLqPCR_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/SLqPCR/SLqPCR_1.42.0_src_all.tar.gz']), ('md5', 'db88b99f4c365a7f8381611d89e5fa8c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SLqPCR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SLqPCR.html'), ('license', 'GPL (>= 2)'), ('summary', 'Functions for analysis of real-time quantitative PCR data at SIRS-Lab GmbH')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-slqpcr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-slqpcr'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'SLqPCR_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SLqPCR_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/SLqPCR_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/SLqPCR/SLqPCR_1.42.0_src_all.tar.gz']), ('md5', 'db88b99f4c365a7f8381611d89e5fa8c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SLqPCR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SLqPCR.html'), ('license', 'GPL (>= 2)'), ('summary', 'Functions for analysis of real-time quantitative PCR data at SIRS-Lab GmbH')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mantelcorr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mantelcorr'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'MantelCorr_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MantelCorr_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/MantelCorr_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/MantelCorr/MantelCorr_1.46.0_src_all.tar.gz']), ('md5', '4d422d6b713bbb38bdee222c0c66a307')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MantelCorr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MantelCorr.html'), ('license', 'GPL (>= 2)'), ('summary', 'Computes Mantel cluster correlations from a (p x n) numeric data matrix (e.g. microarray gene-expression data).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mantelcorr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mantelcorr'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'MantelCorr_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MantelCorr_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/MantelCorr_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/MantelCorr/MantelCorr_1.46.0_src_all.tar.gz']), ('md5', '4d422d6b713bbb38bdee222c0c66a307')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MantelCorr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MantelCorr.html'), ('license', 'GPL (>= 2)'), ('summary', 'Computes Mantel cluster correlations from a (p x n) numeric data matrix (e.g. microarray gene-expression data).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bbmap/37.17/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/projects/bbmap'), ('license', 'UC-LBL license (see package)'), ('summary', 'BBMap is a short read aligner, as well as various other bioinformatic tools')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('url', 'http://downloads.sourceforge.net/project/<{ name|lower }}/<{ name }}_<{ version }}.tar.gz')])), ('requirements', CommentedMap([('build', ['openjdk >=7.0']), ('run', ['openjdk >=7.0'])])), ('test', CommentedMap([('commands', ['bbmap.sh -V 2>&1 | grep \"Executing\" > /dev/null'])])), ('extra', CommentedMap([('notes', 'BBMap is a series of Java programs, but they come with a number of custom\\nwrapper shell scripts. Each of these is symlinked to the conda bin directory\\nduring install.\\n')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bbmap/37.17/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/projects/bbmap'), ('license', 'UC-LBL license (see package)'), ('summary', 'BBMap is a short read aligner, as well as various other bioinformatic tools')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('url', 'http://downloads.sourceforge.net/project/<{ name|lower }}/<{ name }}_<{ version }}.tar.gz')])), ('requirements', CommentedMap([('build', ['openjdk >=7.0']), ('run', ['openjdk >=7.0'])])), ('test', CommentedMap([('commands', ['bbmap.sh -V 2>&1 | grep \"Executing\" > /dev/null'])])), ('extra', CommentedMap([('notes', 'BBMap is a series of Java programs, but they come with a number of custom\\nwrapper shell scripts. Each of these is symlinked to the conda bin directory\\nduring install.\\n')]))]))\n",
      "EXTRA has been found: bbmap\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biovizbase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biovizbase'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'biovizBase_1.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/biovizBase_1.20.0.tar.gz'), ('md5', '6d646b803e5aa620b4758e5f488fb6e2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.11', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.5.14', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.21.19', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges >=1.99.28', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-dichromat', 'r-hmisc', 'r-rcolorbrewer', 'r-scales']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.11', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.5.14', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.21.19', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges >=1.99.28', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-dichromat', 'r-hmisc', 'r-rcolorbrewer', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biovizBase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/biovizBase.html'), ('license', 'Artistic-2.0'), ('summary', 'The biovizBase package is designed to provide a set of utilities, color schemes and conventions for genomic data. It serves as the base for various high-level packages for biological data visualization. This saves development effort and encourages consistency.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biovizbase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biovizbase'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'biovizBase_1.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/biovizBase_1.20.0.tar.gz'), ('md5', '6d646b803e5aa620b4758e5f488fb6e2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.11', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.5.14', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.21.19', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges >=1.99.28', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-dichromat', 'r-hmisc', 'r-rcolorbrewer', 'r-scales']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.11', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.5.14', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.21.19', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges >=1.99.28', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-dichromat', 'r-hmisc', 'r-rcolorbrewer', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biovizBase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/biovizBase.html'), ('license', 'Artistic-2.0'), ('summary', 'The biovizBase package is designed to provide a set of utilities, color schemes and conventions for genomic data. It serves as the base for various high-level packages for biological data visualization. This saves development effort and encourages consistency.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ncigraph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ncigraph'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'NCIgraph_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/NCIgraph_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/NCIgraph_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/NCIgraph/NCIgraph_1.24.0_src_all.tar.gz']), ('md5', '494714008fc79bf0e2e945742277c07f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-rbgl', 'bioconductor-rcytoscape', 'r-base', 'r-r.methodss3']), ('run', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-rbgl', 'bioconductor-rcytoscape', 'r-base', 'r-r.methodss3'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NCIgraph\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/NCIgraph.html'), ('license', 'GPL-3'), ('summary', 'Provides various methods to load the pathways from the NCI Pathways Database in R graph objects and to re-format them.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ncigraph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ncigraph'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'NCIgraph_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/NCIgraph_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/NCIgraph_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/NCIgraph/NCIgraph_1.24.0_src_all.tar.gz']), ('md5', '494714008fc79bf0e2e945742277c07f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-rbgl', 'bioconductor-rcytoscape', 'r-base', 'r-r.methodss3']), ('run', ['bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-rbgl', 'bioconductor-rcytoscape', 'r-base', 'r-r.methodss3'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NCIgraph\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/NCIgraph.html'), ('license', 'GPL-3'), ('summary', 'Provides various methods to load the pathways from the NCI Pathways Database in R graph objects and to re-format them.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/besst/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'besst'), ('version', '2.2.3')])), ('source', CommentedMap([('fn', 'BESST-2.2.3.tar.gz'), ('url', 'https://pypi.python.org/packages/0f/d8/e5a4408a6ed452567a160c22222b1a94c08ee498208ca79dec0293bc4ff6/BESST-2.2.3.tar.gz'), ('md5', 'b94e93751a14d9350506dca6203980ed')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pysam >=0.7', 'networkx >=1.9', 'mathstats >=0.2.3', 'scipy >=0.9']), ('run', ['python', 'pysam >=0.7', 'networkx >=1.9', 'mathstats >=0.2.3', 'scipy >=0.9'])])), ('test', CommentedMap([('imports', ['BESST', 'BESST.diploid']), ('commands', ['runBESST -h > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/ksahlin/BESST'), ('license', 'GPLv3'), ('summary', 'Scaffolder for genomic assemblies.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/besst/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'besst'), ('version', '2.2.3')])), ('source', CommentedMap([('fn', 'BESST-2.2.3.tar.gz'), ('url', 'https://pypi.python.org/packages/0f/d8/e5a4408a6ed452567a160c22222b1a94c08ee498208ca79dec0293bc4ff6/BESST-2.2.3.tar.gz'), ('md5', 'b94e93751a14d9350506dca6203980ed')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pysam >=0.7', 'networkx >=1.9', 'mathstats >=0.2.3', 'scipy >=0.9']), ('run', ['python', 'pysam >=0.7', 'networkx >=1.9', 'mathstats >=0.2.3', 'scipy >=0.9'])])), ('test', CommentedMap([('imports', ['BESST', 'BESST.diploid']), ('commands', ['runBESST -h > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/ksahlin/BESST'), ('license', 'GPLv3'), ('summary', 'Scaffolder for genomic assemblies.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyfasta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyfasta'), ('version', '0.5.2')])), ('source', CommentedMap([('fn', 'pyfasta-0.5.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pyfasta/pyfasta-0.5.2.tar.gz'), ('md5', 'bf61ab997dca329675c3eb2ee7cdfcf2')])), ('build', CommentedMap([('skip', False), ('entry_points', ['pyfasta = pyfasta:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('imports', ['pyfasta']), ('commands', ['pyfasta extract --help | grep Usage > /dev/null']), ('requires', ['nose'])])), ('about', CommentedMap([('home', 'http://github.com/brentp/pyfasta/'), ('license', 'MIT'), ('summary', 'fast, memory-efficient, pythonic (and command-line) access to fasta sequence files')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyfasta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyfasta'), ('version', '0.5.2')])), ('source', CommentedMap([('fn', 'pyfasta-0.5.2.tar.gz'), ('url', 'https://pypi.python.org/packages/source/p/pyfasta/pyfasta-0.5.2.tar.gz'), ('md5', 'bf61ab997dca329675c3eb2ee7cdfcf2')])), ('build', CommentedMap([('skip', False), ('entry_points', ['pyfasta = pyfasta:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'numpy'])])), ('test', CommentedMap([('imports', ['pyfasta']), ('commands', ['pyfasta extract --help | grep Usage > /dev/null']), ('requires', ['nose'])])), ('about', CommentedMap([('home', 'http://github.com/brentp/pyfasta/'), ('license', 'MIT'), ('summary', 'fast, memory-efficient, pythonic (and command-line) access to fasta sequence files')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mergeomics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mergeomics'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'Mergeomics_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Mergeomics_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/Mergeomics_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/Mergeomics/Mergeomics_1.4.0_src_all.tar.gz']), ('md5', '0a40c90a30eb100c29686cc569b0847d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Mergeomics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Mergeomics.html'), ('license', 'GPL (>= 2)'), ('summary', 'The Mergeomics pipeline serves as a flexible framework for integrating multidimensional omics-disease associations, functional genomics,  canonical pathways and gene-gene interaction networks to generate mechanistic hypotheses. It includes two main parts, 1) Marker set enrichment analysis (MSEA); 2) Weighted Key Driver Analysis (wKDA).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mergeomics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mergeomics'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'Mergeomics_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Mergeomics_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/Mergeomics_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/Mergeomics/Mergeomics_1.4.0_src_all.tar.gz']), ('md5', '0a40c90a30eb100c29686cc569b0847d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Mergeomics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Mergeomics.html'), ('license', 'GPL (>= 2)'), ('summary', 'The Mergeomics pipeline serves as a flexible framework for integrating multidimensional omics-disease associations, functional genomics,  canonical pathways and gene-gene interaction networks to generate mechanistic hypotheses. It includes two main parts, 1) Marker set enrichment analysis (MSEA); 2) Weighted Key Driver Analysis (wKDA).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igblast/1.4.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'igblast'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'ncbi-igblast-1.4.0-universal-macosx.tar.gz'), ('url', 'ftp://ftp.ncbi.nih.gov/blast/executables/igblast/release/1.4.0/ncbi-igblast-1.4.0-universal-macosx.tar.gz')])), ('about', CommentedMap([('home', 'http://www.ncbi.nlm.nih.gov/projects/igblast/'), ('license', 'Public Domain'), ('summary', 'A tool for analyzing immunoglobulin (IG) and T cell receptor (TR) sequences.')])), ('build', CommentedMap([('number', 6)])), ('requirements', CommentedMap([('run', ['perl-threaded >=5.8.8'])])), ('test', CommentedMap([('commands', ['igblastn -h'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igblast/1.4.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'igblast'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'ncbi-igblast-1.4.0-universal-macosx.tar.gz'), ('url', 'ftp://ftp.ncbi.nih.gov/blast/executables/igblast/release/1.4.0/ncbi-igblast-1.4.0-universal-macosx.tar.gz')])), ('about', CommentedMap([('home', 'http://www.ncbi.nlm.nih.gov/projects/igblast/'), ('license', 'Public Domain'), ('summary', 'A tool for analyzing immunoglobulin (IG) and T cell receptor (TR) sequences.')])), ('build', CommentedMap([('number', 6)])), ('requirements', CommentedMap([('run', ['perl-threaded >=5.8.8'])])), ('test', CommentedMap([('commands', ['igblastn -h'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocinstaller/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocinstaller'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'BiocInstaller_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BiocInstaller_1.24.0.tar.gz'), ('sha256', '74d89e27db12d216cd22d99e12372f3e1ffbad5eb71ab6f446ab7190cbcb7943')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiocInstaller\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BiocInstaller.html'), ('license', 'Artistic-2.0'), ('summary', 'This package is used to install and update Bioconductor, CRAN, and (some) github packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocinstaller/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocinstaller'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'BiocInstaller_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BiocInstaller_1.24.0.tar.gz'), ('sha256', '74d89e27db12d216cd22d99e12372f3e1ffbad5eb71ab6f446ab7190cbcb7943')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiocInstaller\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BiocInstaller.html'), ('license', 'Artistic-2.0'), ('summary', 'This package is used to install and update Bioconductor, CRAN, and (some) github packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mason/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'mason'), ('version', '2.0.7')])), ('source', CommentedMap([('fn', 'mason2-2.0.7-Linux-x86_64.tar.xz'), ('url', 'http://packages.seqan.de/mason2/mason2-2.0.7-Linux-x86_64.tar.xz'), ('md5', 'af8ae28f177ad95840a9f4ea62901f78')])), ('test', CommentedMap([('commands', ['mason_frag_sequencing --help 2>&1 > /dev/null', 'mason_genome --help 2>&1 > /dev/null', 'mason_materializer --help 2>&1 > /dev/null', 'mason_methylation --help 2>&1 > /dev/null', 'mason_simulator --help 2>&1 > /dev/null', 'mason_splicing --help 2>&1 > /dev/null', 'mason_variator --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/seqan/tree/master/apps/mason2/README'), ('license', 'https://github.com/seqan/seqan/tree/master/apps/mason2/LICENSE'), ('summary', 'Mason is a collection of tools for the simulation of biological sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mason/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'mason'), ('version', '2.0.7')])), ('source', CommentedMap([('fn', 'mason2-2.0.7-Linux-x86_64.tar.xz'), ('url', 'http://packages.seqan.de/mason2/mason2-2.0.7-Linux-x86_64.tar.xz'), ('md5', 'af8ae28f177ad95840a9f4ea62901f78')])), ('test', CommentedMap([('commands', ['mason_frag_sequencing --help 2>&1 > /dev/null', 'mason_genome --help 2>&1 > /dev/null', 'mason_materializer --help 2>&1 > /dev/null', 'mason_methylation --help 2>&1 > /dev/null', 'mason_simulator --help 2>&1 > /dev/null', 'mason_splicing --help 2>&1 > /dev/null', 'mason_variator --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/seqan/tree/master/apps/mason2/README'), ('license', 'https://github.com/seqan/seqan/tree/master/apps/mason2/LICENSE'), ('summary', 'Mason is a collection of tools for the simulation of biological sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/control-freec/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'control-freec'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/BoevaLab/FREEC/archive/v<{ version }}.tar.gz'), ('fn', 'control-freec-<{ version }}.tar.gz'), ('md5', 'fb9e4f3c78343689c38fecae14bc13fc'), ('patches', ['makeGraph.R.patch', 'assess_significance.R.patch', 'makeGraph_Chromosome.R.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc', 'perl', 'r-base', 'bioconductor-rtracklayer'])])), ('test', CommentedMap([('commands', ['freec', 'freec2bed.pl', 'freec2circos.pl', 'which get_fasta_lengths.pl', 'which makeGraph.R', 'which makeGraph_Chromosome.R', 'vcf2snpFreec.pl', 'which assess_significance.R'])])), ('about', CommentedMap([('home', 'https://github.com/BoevaLab/FREEC'), ('license', 'GPL (>=2)'), ('summary', 'Copy number and genotype annotation from whole genome and whole exome \\nsequencing data.\\n')])), ('extra', CommentedMap([('notes', 'The tool will be available as `freec` in the command line.\\nSee the homepage for example config files. Auxiliary scripts\\nlike e.g. freec2bed.pl and freec2circos.pl (see homepage) are available in the \\ncommand line as well.\\n')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/control-freec/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'control-freec'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/BoevaLab/FREEC/archive/v<{ version }}.tar.gz'), ('fn', 'control-freec-<{ version }}.tar.gz'), ('md5', 'fb9e4f3c78343689c38fecae14bc13fc'), ('patches', ['makeGraph.R.patch', 'assess_significance.R.patch', 'makeGraph_Chromosome.R.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc', 'perl', 'r-base', 'bioconductor-rtracklayer'])])), ('test', CommentedMap([('commands', ['freec', 'freec2bed.pl', 'freec2circos.pl', 'which get_fasta_lengths.pl', 'which makeGraph.R', 'which makeGraph_Chromosome.R', 'vcf2snpFreec.pl', 'which assess_significance.R'])])), ('about', CommentedMap([('home', 'https://github.com/BoevaLab/FREEC'), ('license', 'GPL (>=2)'), ('summary', 'Copy number and genotype annotation from whole genome and whole exome \\nsequencing data.\\n')])), ('extra', CommentedMap([('notes', 'The tool will be available as `freec` in the command line.\\nSee the homepage for example config files. Auxiliary scripts\\nlike e.g. freec2bed.pl and freec2circos.pl (see homepage) are available in the \\ncommand line as well.\\n')]))]))\n",
      "EXTRA has been found: freec\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtreemix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtreemix'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'Rtreemix_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Rtreemix_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/Rtreemix_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/Rtreemix/Rtreemix_1.38.0_src_all.tar.gz']), ('md5', 'c1093bf1a6e197a241eed28f980f4bdf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph', 'r-base', 'r-hmisc']), ('run', ['bioconductor-biobase', 'bioconductor-graph', 'r-base', 'r-hmisc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rtreemix\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Rtreemix.html'), ('license', 'LGPL'), ('summary', 'Rtreemix is a package that offers an environment for estimating the mutagenetic trees mixture models from cross-sectional data and using them for various predictions. It includes functions for fitting the trees mixture models, likelihood computations, model comparisons, waiting time estimations, stability analysis, etc.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtreemix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtreemix'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'Rtreemix_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Rtreemix_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/Rtreemix_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/Rtreemix/Rtreemix_1.38.0_src_all.tar.gz']), ('md5', 'c1093bf1a6e197a241eed28f980f4bdf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph', 'r-base', 'r-hmisc']), ('run', ['bioconductor-biobase', 'bioconductor-graph', 'r-base', 'r-hmisc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rtreemix\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Rtreemix.html'), ('license', 'LGPL'), ('summary', 'Rtreemix is a package that offers an environment for estimating the mutagenetic trees mixture models from cross-sectional data and using them for various predictions. It includes functions for fitting the trees mixture models, likelihood computations, model comparisons, waiting time estimations, stability analysis, etc.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ngsutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ngsutils'), ('version', '0.5.9')])), ('source', CommentedMap([('url', 'https://github.com/ngsutils/ngsutils/archive/ngsutils-0.5.9.tar.gz'), ('md5', '6212d6de740c42aa2c6b5f6785465860')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'pysam', 'cython', 'samtools', 'eta', 'coverage', 'swalign']), ('run', ['python', 'pysam', 'cython', 'samtools', 'eta', 'coverage', 'swalign'])])), ('test', CommentedMap([('commands', ['python -m unittest discover ngsutils'])])), ('about', CommentedMap([('home', 'http://ngsutils.org'), ('license', 'BSD'), ('license_file', 'LICENSE'), ('summary', 'Tools for next-generation sequencing analysis http://ngsutils.org')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ngsutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ngsutils'), ('version', '0.5.9')])), ('source', CommentedMap([('url', 'https://github.com/ngsutils/ngsutils/archive/ngsutils-0.5.9.tar.gz'), ('md5', '6212d6de740c42aa2c6b5f6785465860')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'pysam', 'cython', 'samtools', 'eta', 'coverage', 'swalign']), ('run', ['python', 'pysam', 'cython', 'samtools', 'eta', 'coverage', 'swalign'])])), ('test', CommentedMap([('commands', ['python -m unittest discover ngsutils'])])), ('about', CommentedMap([('home', 'http://ngsutils.org'), ('license', 'BSD'), ('license_file', 'LICENSE'), ('summary', 'Tools for next-generation sequencing analysis http://ngsutils.org')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cufflinks/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://cole-trapnell-lab.github.io/cufflinks/'), ('license', 'Boost Software License'), ('summary', 'Transcriptome assembly and differential expression analysis for RNA-Seq.')])), ('package', CommentedMap([('name', 'cufflinks'), ('version', '2.2.1')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['(cuffmerge --help 2>&1) | grep Options > /dev/null'])])), ('source', CommentedMap([('fn', 'cufflinks-2.2.1.Linux_x86_64.tar.gz'), ('url', 'http://cole-trapnell-lab.github.io/cufflinks/assets/downloads/cufflinks-2.2.1.Linux_x86_64.tar.gz'), ('patches', ['cuffmerge.python3.patch'])])), ('build', CommentedMap([('number', 1), ('skip', False)]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cufflinks/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://cole-trapnell-lab.github.io/cufflinks/'), ('license', 'Boost Software License'), ('summary', 'Transcriptome assembly and differential expression analysis for RNA-Seq.')])), ('package', CommentedMap([('name', 'cufflinks'), ('version', '2.2.1')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['(cuffmerge --help 2>&1) | grep Options > /dev/null'])])), ('source', CommentedMap([('fn', 'cufflinks-2.2.1.Linux_x86_64.tar.gz'), ('url', 'http://cole-trapnell-lab.github.io/cufflinks/assets/downloads/cufflinks-2.2.1.Linux_x86_64.tar.gz'), ('patches', ['cuffmerge.python3.patch'])])), ('build', CommentedMap([('number', 1), ('skip', False)]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-deseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-deseq'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'DESeq_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/DESeq_1.24.0.tar.gz'), ('md5', 'e5b94046a8d1a62b20d81cfe720982cb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.21.7', 'bioconductor-biocgenerics >=0.7.5', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'r-base', 'r-locfit', 'r-rcolorbrewer', 'r-mass']), ('run', ['bioconductor-biobase >=2.21.7', 'bioconductor-biocgenerics >=0.7.5', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'r-base', 'r-locfit', 'r-rcolorbrewer', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DESeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/DESeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-deseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-deseq'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'DESeq_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/DESeq_1.24.0.tar.gz'), ('md5', 'e5b94046a8d1a62b20d81cfe720982cb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.21.7', 'bioconductor-biocgenerics >=0.7.5', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'r-base', 'r-locfit', 'r-rcolorbrewer', 'r-mass']), ('run', ['bioconductor-biobase >=2.21.7', 'bioconductor-biocgenerics >=0.7.5', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'r-base', 'r-locfit', 'r-rcolorbrewer', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DESeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/DESeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'Estimate variance-mean dependence in count data from high-throughput sequencing assays and test for differential expression based on a model using the negative binomial distribution')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affyio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affyio'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'affyio_1.42.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/affyio_1.42.0.tar.gz'), ('md5', 'aa52c3eacb3401eaf7bdaefea62e6c06')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affyio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/affyio.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Routines for parsing Affymetrix data files based upon file format information. Primary focus is on accessing the CEL and CDF file formats.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affyio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affyio'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'affyio_1.42.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/affyio_1.42.0.tar.gz'), ('md5', 'aa52c3eacb3401eaf7bdaefea62e6c06')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affyio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/affyio.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Routines for parsing Affymetrix data files based upon file format information. Primary focus is on accessing the CEL and CDF file formats.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genometools-genometools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genometools-genometools'), ('version', '1.5.9')])), ('source', CommentedMap([('fn', 'genometools-1.5.9.tar.gz'), ('url', 'https://github.com/genometools/genometools/archive/v1.5.9.tar.gz'), ('sha1', '733530dfe875cd058da0c3e1b6530b9e085517f4')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'pango']), ('run', ['gettext', 'pango', 'xorg-libxrender', 'xorg-libsm', 'xorg-libxext', 'libgcc'])])), ('test', CommentedMap([('commands', ['gt -version > /dev/null']), ('imports', ['gt'])])), ('about', CommentedMap([('home', 'http://genometools.org/'), ('summary', 'GenomeTools genome analysis system.'), ('license', 'BSD'), ('license_file', 'LICENSE')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genometools-genometools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genometools-genometools'), ('version', '1.5.9')])), ('source', CommentedMap([('fn', 'genometools-1.5.9.tar.gz'), ('url', 'https://github.com/genometools/genometools/archive/v1.5.9.tar.gz'), ('sha1', '733530dfe875cd058da0c3e1b6530b9e085517f4')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'pango']), ('run', ['gettext', 'pango', 'xorg-libxrender', 'xorg-libsm', 'xorg-libxext', 'libgcc'])])), ('test', CommentedMap([('commands', ['gt -version > /dev/null']), ('imports', ['gt'])])), ('about', CommentedMap([('home', 'http://genometools.org/'), ('summary', 'GenomeTools genome analysis system.'), ('license', 'BSD'), ('license_file', 'LICENSE')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-arrayquality/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-arrayquality'), ('version', '1.54.0')])), ('source', CommentedMap([('fn', 'arrayQuality_1.54.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/arrayQuality_1.54.0.tar.gz', 'https://bioarchive.galaxyproject.org/arrayQuality_1.54.0.tar.gz', 'https://depot.galaxyproject.org/software/arrayQuality/arrayQuality_1.54.0_src_all.tar.gz']), ('md5', '56ea80ed13e2924289b84d317d56f6e3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-gridbase', 'r-hexbin', 'r-rcolorbrewer']), ('run', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-gridbase', 'r-hexbin', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'arrayQuality\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/arrayQuality.html'), ('license', 'LGPL'), ('summary', 'Functions for performing print-run and array level quality assessment.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-arrayquality/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-arrayquality'), ('version', '1.54.0')])), ('source', CommentedMap([('fn', 'arrayQuality_1.54.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/arrayQuality_1.54.0.tar.gz', 'https://bioarchive.galaxyproject.org/arrayQuality_1.54.0.tar.gz', 'https://depot.galaxyproject.org/software/arrayQuality/arrayQuality_1.54.0_src_all.tar.gz']), ('md5', '56ea80ed13e2924289b84d317d56f6e3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-gridbase', 'r-hexbin', 'r-rcolorbrewer']), ('run', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-gridbase', 'r-hexbin', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'arrayQuality\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/arrayQuality.html'), ('license', 'LGPL'), ('summary', 'Functions for performing print-run and array level quality assessment.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mummer/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://mummer.sourceforge.net/'), ('license', 'The Artistic License'), ('summary', 'MUMmer is a system for rapidly aligning entire genomes')])), ('build', CommentedMap([('number', 4), ('skip', False)])), ('package', CommentedMap([('name', 'mummer'), ('version', '3.23')])), ('source', CommentedMap([('fn', 'MUMmer3.23.tar.gz'), ('sha256', '1efad4f7d8cee0d8eaebb320a2d63745bb3a160bb513a15ef7af46f330af662f'), ('url', 'https://downloads.sourceforge.net/project/mummer/mummer/3.23/MUMmer3.23.tar.gz')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', ['mummer -h &> /dev/null', 'mummerplot -h &> /dev/null', 'nucmer -h &> /dev/null', 'promer -h &> /dev/null', 'show-aligns -h &> /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mummer/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://mummer.sourceforge.net/'), ('license', 'The Artistic License'), ('summary', 'MUMmer is a system for rapidly aligning entire genomes')])), ('build', CommentedMap([('number', 4), ('skip', False)])), ('package', CommentedMap([('name', 'mummer'), ('version', '3.23')])), ('source', CommentedMap([('fn', 'MUMmer3.23.tar.gz'), ('sha256', '1efad4f7d8cee0d8eaebb320a2d63745bb3a160bb513a15ef7af46f330af662f'), ('url', 'https://downloads.sourceforge.net/project/mummer/mummer/3.23/MUMmer3.23.tar.gz')])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', ['mummer -h &> /dev/null', 'mummerplot -h &> /dev/null', 'nucmer -h &> /dev/null', 'promer -h &> /dev/null', 'show-aligns -h &> /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/validate-fasta-database/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'validate-fasta-database'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/caleb-easterly/validate_fasta_database/releases/download/v<{ version }}/validate_fasta_database-<{ version }}.jar'), ('md5', '04fa47141882ee32f5cdaec93e5a890b')])), ('requirements', CommentedMap([('build', ['openjdk >=7']), ('run', ['openjdk >=7', 'python'])])), ('test', CommentedMap([('commands', ['echo -e \">abc|001\\\\nMMCTARRK\" > test.fasta && validate-fasta-database test.fasta good.fasta bad.fasta false true \"\" false 0'])])), ('about', CommentedMap([('home', 'https://github.com/caleb-easterly/validate_fasta_database'), ('license', 'GPL-3.0'), ('summary', 'Code for Galaxy tool for quality control on FASTA database')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/validate-fasta-database/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'validate-fasta-database'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/caleb-easterly/validate_fasta_database/releases/download/v<{ version }}/validate_fasta_database-<{ version }}.jar'), ('md5', '04fa47141882ee32f5cdaec93e5a890b')])), ('requirements', CommentedMap([('build', ['openjdk >=7']), ('run', ['openjdk >=7', 'python'])])), ('test', CommentedMap([('commands', ['echo -e \">abc|001\\\\nMMCTARRK\" > test.fasta && validate-fasta-database test.fasta good.fasta bad.fasta false true \"\" false 0'])])), ('about', CommentedMap([('home', 'https://github.com/caleb-easterly/validate_fasta_database'), ('license', 'GPL-3.0'), ('summary', 'Code for Galaxy tool for quality control on FASTA database')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-micropan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-micropan'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'micropan_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/micropan_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/micropan/micropan_<{ version }}.tar.gz']), ('sha256', 'efb4491b04f781a99fb9180689b7788618800618c3433a8af897ee926f556a67')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph', 'r-microseq']), ('run', ['r-base', 'r-igraph', 'r-microseq'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'micropan\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=micropan'), ('license', 'GPL-2'), ('summary', 'A collection of functions for computations and visualizations of microbial pan-genomes.'), ('license_family', 'GPL2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-micropan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-micropan'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'micropan_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/micropan_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/micropan/micropan_<{ version }}.tar.gz']), ('sha256', 'efb4491b04f781a99fb9180689b7788618800618c3433a8af897ee926f556a67')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph', 'r-microseq']), ('run', ['r-base', 'r-igraph', 'r-microseq'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'micropan\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=micropan'), ('license', 'GPL-2'), ('summary', 'A collection of functions for computations and visualizations of microbial pan-genomes.'), ('license_family', 'GPL2')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rain/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rain'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'rain_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rain_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/rain_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/rain/rain_1.10.0_src_all.tar.gz']), ('md5', '7eb25126c0851e7feb3e576d1e910d91')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-multtest', 'r-base', 'r-gmp']), ('run', ['bioconductor-multtest', 'r-base', 'r-gmp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rain\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rain.html'), ('license', 'GPL-2'), ('summary', 'This package uses non-parametric methods to detect rhythms in time series. It deals with outliers, missing values and is optimized for time series comprising 10-100 measurements. As it does not assume expect any distinct waveform it is optimal or detecting oscillating behavior (e.g. circadian or cell cycle) in e.g. genome- or proteome-wide biological measurements such as: micro arrays, proteome mass spectrometry, or metabolome measurements.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rain/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rain'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'rain_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rain_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/rain_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/rain/rain_1.10.0_src_all.tar.gz']), ('md5', '7eb25126c0851e7feb3e576d1e910d91')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-multtest', 'r-base', 'r-gmp']), ('run', ['bioconductor-multtest', 'r-base', 'r-gmp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rain\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rain.html'), ('license', 'GPL-2'), ('summary', 'This package uses non-parametric methods to detect rhythms in time series. It deals with outliers, missing values and is optimized for time series comprising 10-100 measurements. As it does not assume expect any distinct waveform it is optimal or detecting oscillating behavior (e.g. circadian or cell cycle) in e.g. genome- or proteome-wide biological measurements such as: micro arrays, proteome mass spectrometry, or metabolome measurements.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbsurv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbsurv'), ('version', '2.34.0')])), ('source', CommentedMap([('fn', 'rbsurv_2.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rbsurv_2.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/rbsurv_2.34.0.tar.gz', 'https://depot.galaxyproject.org/software/rbsurv/rbsurv_2.34.0_src_all.tar.gz']), ('md5', '3f7ac25ebdeb23271b55987a8a8dd8b8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-survival']), ('run', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rbsurv\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rbsurv.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package selects genes associated with survival.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbsurv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbsurv'), ('version', '2.34.0')])), ('source', CommentedMap([('fn', 'rbsurv_2.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rbsurv_2.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/rbsurv_2.34.0.tar.gz', 'https://depot.galaxyproject.org/software/rbsurv/rbsurv_2.34.0_src_all.tar.gz']), ('md5', '3f7ac25ebdeb23271b55987a8a8dd8b8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-survival']), ('run', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rbsurv\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rbsurv.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package selects genes associated with survival.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rcas/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rcas'), ('version', '1.1.1')])), ('source', CommentedMap([('fn', 'v1.1.1.tar.gz'), ('url', 'https://github.com/BIMSBbioinfo/RCAS/archive/v1.1.1.tar.gz'), ('md5', 'c97766ff5ad5fa58da34a3451fdb4354')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biomart', 'bioconductor-topgo', 'bioconductor-genomicfeatures', 'bioconductor-genomation >=1.5.5', 'bioconductor-rtracklayer', 'bioconductor-annotationdbi', 'bioconductor-genomicranges', 'bioconductor-bsgenome.hsapiens.ucsc.hg19', 'bioconductor-org.hs.eg.db', 'bioconductor-genomeinfodb', 'bioconductor-biostrings', 'bioconductor-motifrg', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-rmarkdown >=0.9.5', 'r-plotly >=4.5.2', 'r-dt >=0.2', 'r-data.table', 'pandoc >=1.12.3', 'r-knitr >=1.12.3', 'r-plotrix']), ('run', ['r-base', 'bioconductor-biomart', 'bioconductor-topgo', 'bioconductor-genomicfeatures', 'bioconductor-genomation >=1.5.5', 'bioconductor-rtracklayer', 'bioconductor-annotationdbi', 'bioconductor-genomicranges', 'bioconductor-bsgenome.hsapiens.ucsc.hg19', 'bioconductor-org.hs.eg.db', 'bioconductor-genomeinfodb', 'bioconductor-biostrings', 'bioconductor-motifrg', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-rmarkdown >=0.9.5', 'r-plotly >=4.5.2', 'r-dt >=0.2', 'r-data.table', 'pandoc >=1.12.3', 'r-knitr >=1.12.3', 'r-plotrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RCAS\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/BIMSBbioinfo/RCAS/tree/rcas_R/rpackage/RCAS'), ('license', 'Artistic-2.0'), ('summary', 'RNA Centric Annotation System (RCAS) is an automated system that provides dynamic annotations for custom input files that contain transcriptomic targe regions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rcas/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rcas'), ('version', '1.1.1')])), ('source', CommentedMap([('fn', 'v1.1.1.tar.gz'), ('url', 'https://github.com/BIMSBbioinfo/RCAS/archive/v1.1.1.tar.gz'), ('md5', 'c97766ff5ad5fa58da34a3451fdb4354')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biomart', 'bioconductor-topgo', 'bioconductor-genomicfeatures', 'bioconductor-genomation >=1.5.5', 'bioconductor-rtracklayer', 'bioconductor-annotationdbi', 'bioconductor-genomicranges', 'bioconductor-bsgenome.hsapiens.ucsc.hg19', 'bioconductor-org.hs.eg.db', 'bioconductor-genomeinfodb', 'bioconductor-biostrings', 'bioconductor-motifrg', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-rmarkdown >=0.9.5', 'r-plotly >=4.5.2', 'r-dt >=0.2', 'r-data.table', 'pandoc >=1.12.3', 'r-knitr >=1.12.3', 'r-plotrix']), ('run', ['r-base', 'bioconductor-biomart', 'bioconductor-topgo', 'bioconductor-genomicfeatures', 'bioconductor-genomation >=1.5.5', 'bioconductor-rtracklayer', 'bioconductor-annotationdbi', 'bioconductor-genomicranges', 'bioconductor-bsgenome.hsapiens.ucsc.hg19', 'bioconductor-org.hs.eg.db', 'bioconductor-genomeinfodb', 'bioconductor-biostrings', 'bioconductor-motifrg', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-rmarkdown >=0.9.5', 'r-plotly >=4.5.2', 'r-dt >=0.2', 'r-data.table', 'pandoc >=1.12.3', 'r-knitr >=1.12.3', 'r-plotrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RCAS\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/BIMSBbioinfo/RCAS/tree/rcas_R/rpackage/RCAS'), ('license', 'Artistic-2.0'), ('summary', 'RNA Centric Annotation System (RCAS) is an automated system that provides dynamic annotations for custom input files that contain transcriptomic targe regions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghcall/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghcall'), ('version', '2.34.1')])), ('source', CommentedMap([('fn', 'CGHcall_2.34.1.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CGHcall_2.34.1.tar.gz'), ('md5', '4e2a7e89d577b984ce8e760d645397e1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-cghbase >=1.15.1', 'bioconductor-dnacopy >=1.6.0', 'bioconductor-impute >=1.8.0', 'r-base', 'r-snowfall']), ('run', ['bioconductor-biobase', 'bioconductor-cghbase >=1.15.1', 'bioconductor-dnacopy >=1.6.0', 'bioconductor-impute >=1.8.0', 'r-base', 'r-snowfall'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CGHcall\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CGHcall.html'), ('license', 'GPL (http://www.gnu.org/copyleft/gpl.html)'), ('summary', 'Calls aberrations for array CGH data using a six state mixture model as well as several biological concepts that are ignored by existing algorithms. Visualization of profiles is also provided.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghcall/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghcall'), ('version', '2.34.1')])), ('source', CommentedMap([('fn', 'CGHcall_2.34.1.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CGHcall_2.34.1.tar.gz'), ('md5', '4e2a7e89d577b984ce8e760d645397e1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-cghbase >=1.15.1', 'bioconductor-dnacopy >=1.6.0', 'bioconductor-impute >=1.8.0', 'r-base', 'r-snowfall']), ('run', ['bioconductor-biobase', 'bioconductor-cghbase >=1.15.1', 'bioconductor-dnacopy >=1.6.0', 'bioconductor-impute >=1.8.0', 'r-base', 'r-snowfall'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CGHcall\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CGHcall.html'), ('license', 'GPL (http://www.gnu.org/copyleft/gpl.html)'), ('summary', 'Calls aberrations for array CGH data using a six state mixture model as well as several biological concepts that are ignored by existing algorithms. Visualization of profiles is also provided.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mbamethyl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mbamethyl'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MBAmethyl_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MBAmethyl_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MBAmethyl_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MBAmethyl/MBAmethyl_1.10.0_src_all.tar.gz']), ('md5', 'b960d48ffe66f080d8daa5b4e9baa318')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MBAmethyl\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MBAmethyl.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides a function for reconstructing DNA methylation values from raw measurements. It iteratively implements the group fused lars to smooth related-by-location methylation values and the constrained least squares to remove probe affinity effect across multiple sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mbamethyl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mbamethyl'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MBAmethyl_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MBAmethyl_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MBAmethyl_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MBAmethyl/MBAmethyl_1.10.0_src_all.tar.gz']), ('md5', 'b960d48ffe66f080d8daa5b4e9baa318')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MBAmethyl\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MBAmethyl.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides a function for reconstructing DNA methylation values from raw measurements. It iteratively implements the group fused lars to smooth related-by-location methylation values and the constrained least squares to remove probe affinity effect across multiple sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fasttree/2.1.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fasttree'), ('version', '2.1.8')])), ('source', CommentedMap([('fn', 'FastTree.c'), ('url', 'http://meta.microbesonline.org/fasttree/FastTree-2.1.8.c'), ('md5', 'c7e85689a26cabba241378d4f633c2fa')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('test', CommentedMap([('commands', ['FastTree | cat'])])), ('about', CommentedMap([('home', 'http://www.microbesonline.org/fasttree/'), ('license', 'GPL v2'), ('summary', 'FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fasttree/2.1.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fasttree'), ('version', '2.1.8')])), ('source', CommentedMap([('fn', 'FastTree.c'), ('url', 'http://meta.microbesonline.org/fasttree/FastTree-2.1.8.c'), ('md5', 'c7e85689a26cabba241378d4f633c2fa')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('test', CommentedMap([('commands', ['FastTree | cat'])])), ('about', CommentedMap([('home', 'http://www.microbesonline.org/fasttree/'), ('license', 'GPL v2'), ('summary', 'FastTree infers approximately-maximum-likelihood phylogenetic trees from alignments of nucleotide or protein sequences')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-santa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-santa'), ('version', '2.14.0')])), ('source', CommentedMap([('fn', 'SANTA_2.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SANTA_2.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/SANTA_2.14.0.tar.gz', 'https://depot.galaxyproject.org/software/SANTA/SANTA_2.14.0_src_all.tar.gz']), ('md5', '913f31736c9f499d9075b762a6c8f07f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph', 'r-matrix', 'r-snow']), ('run', ['r-base', 'r-igraph', 'r-matrix', 'r-snow'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SANTA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SANTA.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package provides methods for measuring the strength of association between a network and a phenotype. It does this by measuring clustering of the phenotype across the network (Knet). Vertices can also be individually ranked by their strength of association with high-weight vertices (Knode).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-santa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-santa'), ('version', '2.14.0')])), ('source', CommentedMap([('fn', 'SANTA_2.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SANTA_2.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/SANTA_2.14.0.tar.gz', 'https://depot.galaxyproject.org/software/SANTA/SANTA_2.14.0_src_all.tar.gz']), ('md5', '913f31736c9f499d9075b762a6c8f07f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph', 'r-matrix', 'r-snow']), ('run', ['r-base', 'r-igraph', 'r-matrix', 'r-snow'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SANTA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SANTA.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package provides methods for measuring the strength of association between a network and a phenotype. It does this by measuring clustering of the phenotype across the network (Knet). Vertices can also be individually ranked by their strength of association with high-weight vertices (Knode).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rhtslib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rhtslib'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Rhtslib_1.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Rhtslib_1.6.0.tar.gz'), ('md5', 'a4749fbe0ac9bcbc544decde28dac713'), ('patches', ['rpath.patch'])])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-zlibbioc', 'autoconf 2.69 pl5.20.3.1_4', 'gcc', 'llvm']), ('run', ['r-base', 'bioconductor-zlibbioc', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rhtslib\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Rhtslib.html'), ('license', 'LGPL (>= 2)'), ('summary', 'This package provides version 1.1 of the \"HTSlib\" C library for high-throughput sequence analysis. The package is primarily useful to developers of other R packages who wish to make use of HTSlib. Motivation and instructions for use of this package are in the vignette, vignette(package=\"Rhtslib\", \"Rhtslib\").')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rhtslib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rhtslib'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Rhtslib_1.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Rhtslib_1.6.0.tar.gz'), ('md5', 'a4749fbe0ac9bcbc544decde28dac713'), ('patches', ['rpath.patch'])])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-zlibbioc', 'autoconf 2.69 pl5.20.3.1_4', 'gcc', 'llvm']), ('run', ['r-base', 'bioconductor-zlibbioc', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rhtslib\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Rhtslib.html'), ('license', 'LGPL (>= 2)'), ('summary', 'This package provides version 1.1 of the \"HTSlib\" C library for high-throughput sequence analysis. The package is primarily useful to developers of other R packages who wish to make use of HTSlib. Motivation and instructions for use of this package are in the vignette, vignette(package=\"Rhtslib\", \"Rhtslib\").')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ccat/3.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://cmb.gis.a-star.edu.sg/ChIPSeq/paperCCAT.htm'), ('license', None), ('summary', 'CCAT is a software package for the analysis of ChIP-seq data with negative control.')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'ccat'), ('version', 3.0)])), ('source', CommentedMap([('fn', 'CCAT3.0.tar.gz'), ('sha256', 'd8944465c8327bec22eb41059ba73f780fcdcedf4470507bfdae5a54c59a9b07'), ('url', 'http://cmb.gis.a-star.edu.sg/ChIPSeq/tools/CCAT3.0.tar.gz')])), ('test', CommentedMap([('commands', ['CCAT --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ccat/3.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://cmb.gis.a-star.edu.sg/ChIPSeq/paperCCAT.htm'), ('license', None), ('summary', 'CCAT is a software package for the analysis of ChIP-seq data with negative control.')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'ccat'), ('version', 3.0)])), ('source', CommentedMap([('fn', 'CCAT3.0.tar.gz'), ('sha256', 'd8944465c8327bec22eb41059ba73f780fcdcedf4470507bfdae5a54c59a9b07'), ('url', 'http://cmb.gis.a-star.edu.sg/ChIPSeq/tools/CCAT3.0.tar.gz')])), ('test', CommentedMap([('commands', ['CCAT --help'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kaiju/1.4.5/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kaiju'), ('version', '1.4.5')])), ('source', CommentedMap([('fn', 'c13b7575c212a70c0ca4a2d2baefa544f0103363.tar.gz'), ('url', 'https://github.com/bioinformatics-centre/kaiju/archive/c13b7575c212a70c0ca4a2d2baefa544f0103363.tar.gz'), ('md5', 'dea77fa5856cb9261f523536eb917e68')])), ('build', CommentedMap([('number', 0), ('no_link', ['bin/makeDB.sh'])])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc', 'perl', 'gnu-wget >=1.16'])])), ('test', CommentedMap([('commands', [\"kaiju 2>&1 | head -n 1 | grep -q '^Error:'; echo $?\"])])), ('about', CommentedMap([('home', 'http://kaiju.binf.ku.dk/'), ('license', 'GNU GPL v3'), ('summary', 'Fast and sensitive taxonomic classification for metagenomics'), ('license_file', 'LICENSE')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kaiju/1.4.5/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kaiju'), ('version', '1.4.5')])), ('source', CommentedMap([('fn', 'c13b7575c212a70c0ca4a2d2baefa544f0103363.tar.gz'), ('url', 'https://github.com/bioinformatics-centre/kaiju/archive/c13b7575c212a70c0ca4a2d2baefa544f0103363.tar.gz'), ('md5', 'dea77fa5856cb9261f523536eb917e68')])), ('build', CommentedMap([('number', 0), ('no_link', ['bin/makeDB.sh'])])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc', 'perl', 'gnu-wget >=1.16'])])), ('test', CommentedMap([('commands', [\"kaiju 2>&1 | head -n 1 | grep -q '^Error:'; echo $?\"])])), ('about', CommentedMap([('home', 'http://kaiju.binf.ku.dk/'), ('license', 'GNU GPL v3'), ('summary', 'Fast and sensitive taxonomic classification for metagenomics'), ('license_file', 'LICENSE')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cancermutationanalysis/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cancermutationanalysis'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'CancerMutationAnalysis_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CancerMutationAnalysis_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/CancerMutationAnalysis_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/CancerMutationAnalysis/CancerMutationAnalysis_1.18.0_src_all.tar.gz']), ('md5', '607e9ad684f2a0b260107836197caa1e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CancerMutationAnalysis\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CancerMutationAnalysis.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'This package implements gene and gene-set level analysis methods for somatic mutation studies of cancer.  The gene-level methods distinguish between driver genes (which play an active role in tumorigenesis) and passenger genes (which are mutated in tumor samples, but have no role in tumorigenesis) and incorporate a two-stage study design.  The gene-set methods implement a patient-oriented approach, which calculates gene-set scores for each sample, then combines them across samples; a gene-oriented approach which uses the Wilcoxon test is also provided for comparison.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cancermutationanalysis/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cancermutationanalysis'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'CancerMutationAnalysis_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CancerMutationAnalysis_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/CancerMutationAnalysis_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/CancerMutationAnalysis/CancerMutationAnalysis_1.18.0_src_all.tar.gz']), ('md5', '607e9ad684f2a0b260107836197caa1e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CancerMutationAnalysis\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CancerMutationAnalysis.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'This package implements gene and gene-set level analysis methods for somatic mutation studies of cancer.  The gene-level methods distinguish between driver genes (which play an active role in tumorigenesis) and passenger genes (which are mutated in tumor samples, but have no role in tumorigenesis) and incorporate a two-stage study design.  The gene-set methods implement a patient-oriented approach, which calculates gene-set scores for each sample, then combines them across samples; a gene-oriented approach which uses the Wilcoxon test is also provided for comparison.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qdnaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qdnaseq'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'QDNAseq_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/QDNAseq_1.10.0.tar.gz'), ('md5', 'd46bbb4166bdeeb8be196a3047e28963')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.18.0', 'bioconductor-cghbase >=1.18.0', 'bioconductor-cghcall >=2.18.0', 'bioconductor-dnacopy >=1.32.0', 'bioconductor-genomicranges >=1.20', 'bioconductor-iranges >=2.2', 'bioconductor-rsamtools >=1.20', 'r-base', 'r-matrixstats >=0.50.2', 'r-r.utils >=2.3.0']), ('run', ['bioconductor-biobase >=2.18.0', 'bioconductor-cghbase >=1.18.0', 'bioconductor-cghcall >=2.18.0', 'bioconductor-dnacopy >=1.32.0', 'bioconductor-genomicranges >=1.20', 'bioconductor-iranges >=2.2', 'bioconductor-rsamtools >=1.20', 'r-base', 'r-matrixstats >=0.50.2', 'r-r.utils >=2.3.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'QDNAseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/QDNAseq.html'), ('license', 'GPL'), ('summary', 'Quantitative DNA sequencing for chromosomal aberrations. The genome is divided into non-overlapping fixed-sized bins, number of sequence reads in each counted, adjusted with a simultaneous two-dimensional loess correction for sequence mappability and GC content, and filtered to remove spurious regions in the genome. Downstream steps of segmentation and calling are also implemented via packages DNAcopy and CGHcall, respectively.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qdnaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qdnaseq'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'QDNAseq_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/QDNAseq_1.10.0.tar.gz'), ('md5', 'd46bbb4166bdeeb8be196a3047e28963')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.18.0', 'bioconductor-cghbase >=1.18.0', 'bioconductor-cghcall >=2.18.0', 'bioconductor-dnacopy >=1.32.0', 'bioconductor-genomicranges >=1.20', 'bioconductor-iranges >=2.2', 'bioconductor-rsamtools >=1.20', 'r-base', 'r-matrixstats >=0.50.2', 'r-r.utils >=2.3.0']), ('run', ['bioconductor-biobase >=2.18.0', 'bioconductor-cghbase >=1.18.0', 'bioconductor-cghcall >=2.18.0', 'bioconductor-dnacopy >=1.32.0', 'bioconductor-genomicranges >=1.20', 'bioconductor-iranges >=2.2', 'bioconductor-rsamtools >=1.20', 'r-base', 'r-matrixstats >=0.50.2', 'r-r.utils >=2.3.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'QDNAseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/QDNAseq.html'), ('license', 'GPL'), ('summary', 'Quantitative DNA sequencing for chromosomal aberrations. The genome is divided into non-overlapping fixed-sized bins, number of sequence reads in each counted, adjusted with a simultaneous two-dimensional loess correction for sequence mappability and GC content, and filtered to remove spurious regions in the genome. Downstream steps of segmentation and calling are also implemented via packages DNAcopy and CGHcall, respectively.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ballgown/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ballgown'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'ballgown_2.2.0.tar.gz'), ('url', 'http://bioconductor.org/packages/3.2/bioc/src/contrib/ballgown_2.2.0.tar.gz'), ('md5', '717570395edef570a4606858bafc6edb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-sva', 'r-base', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-sva', 'r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ballgown\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ballgown.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for statistical analysis of assembled transcriptomes, including flexible differential expression analysis, visualization of transcript structures, and matching of assembled transcripts to annotation.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ballgown/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ballgown'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'ballgown_2.2.0.tar.gz'), ('url', 'http://bioconductor.org/packages/3.2/bioc/src/contrib/ballgown_2.2.0.tar.gz'), ('md5', '717570395edef570a4606858bafc6edb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-sva', 'r-base', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-sva', 'r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ballgown\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ballgown.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for statistical analysis of assembled transcriptomes, including flexible differential expression analysis, visualization of transcript structures, and matching of assembled transcripts to annotation.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/searchgui/3.2.8/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('md5', '3c1c82325e19d30dbe8bb7233e9d8c0e')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/searchgui/3.2.8/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/compomics/searchgui'), ('license', 'Apache License, Version 2.0'), ('summary', 'SearchGUI is a user-friendly open-source graphical user interface for configuring and running proteomics identification search engines, currently supporting X!Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda and OMSSA.\\n')])), ('package', CommentedMap([('name', 'searchgui'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('url', 'http://genesis.ugent.be/maven2/eu/isas/searchgui/<{ name }}/<{ version }}/<{ name }}-<{ version }}-mac_and_linux.tar.gz'), ('md5', '3c1c82325e19d30dbe8bb7233e9d8c0e')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=6', 'mono >=4.0.0', 'python'])])), ('test', CommentedMap([('commands', ['searchgui eu.isas.searchgui.cmd.FastaCLI', 'searchgui eu.isas.searchgui.cmd.FastaCLI -Xms512m -Xmx1g'])])), ('extra', CommentedMap([('notes', 'SearchGUI is Java program that comes with a custom wrapper shell script.\\nThis shell wrapper is called \"opsin\" and is on $PATH by default. By default\\n\"-Xms512m -Xmx1g\" is set in the wrapper. If you want to overwrite it you can\\nspecify these values directly after your binaries. If you have _JAVA_OPTIONS\\nset globally this will take precedence.\\nFor example run it with \"searchgui -Xms512m -Xmx1g\"\\n')]))]))\n",
      "EXTRA has been found: searchgui\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trinity/2.1.1/date.2011_11_26/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/trinityrnaseq/trinityrnaseq'), ('license', 'https://raw.githubusercontent.com/trinityrnaseq/trinityrnaseq/master/LICENSE.txt'), ('summary', 'Trinity assembles transcript sequences from Illumina RNA-Seq data')])), ('build', CommentedMap([('number', 4), ('string', 'ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('package', CommentedMap([('name', 'trinity'), ('version', 'date.2011_11_26')])), ('source', CommentedMap([('fn', 'trinityrnaseq_r2011-11-26.tgz'), ('md5', '3a4c4691908d37042ca63ca469c75fd3'), ('url', 'http://sourceforge.net/projects/trinityrnaseq/files/trinityrnaseq_r2011-11-26.tgz'), ('patches', ['RunButterfly.cc.patch', 'Trinity.pl.patch', 'samtools_makefile.patch'])])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc', 'java-jdk >=6', 'bzip2', 'zlib', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['perl-threaded', 'libgcc', 'java-jdk >=6', 'bzip2', 'zlib', 'ncurses <{CONDA_NCURSES}}*'])])), ('extra', CommentedMap([('notes', 'Trinity version date.2011_11_26 relies on a custom wrapper script, symlinked to bin/: Trinity-runner.sh. This script changes the current working directory to that of the Trinity.pl perl script before calling Trinity, so relative paths work as expected (particularly for Trinity plugins). All arguments are passed to Trinity.pl (via $@)')])), ('test', CommentedMap([('commands', ['$PREFIX/bin/Trinity-test &> /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trinity/2.1.1/date.2011_11_26/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/trinityrnaseq/trinityrnaseq'), ('license', 'https://raw.githubusercontent.com/trinityrnaseq/trinityrnaseq/master/LICENSE.txt'), ('summary', 'Trinity assembles transcript sequences from Illumina RNA-Seq data')])), ('build', CommentedMap([('number', 4), ('string', 'ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('package', CommentedMap([('name', 'trinity'), ('version', 'date.2011_11_26')])), ('source', CommentedMap([('fn', 'trinityrnaseq_r2011-11-26.tgz'), ('md5', '3a4c4691908d37042ca63ca469c75fd3'), ('url', 'http://sourceforge.net/projects/trinityrnaseq/files/trinityrnaseq_r2011-11-26.tgz'), ('patches', ['RunButterfly.cc.patch', 'Trinity.pl.patch', 'samtools_makefile.patch'])])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc', 'java-jdk >=6', 'bzip2', 'zlib', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['perl-threaded', 'libgcc', 'java-jdk >=6', 'bzip2', 'zlib', 'ncurses <{CONDA_NCURSES}}*'])])), ('extra', CommentedMap([('notes', 'Trinity version date.2011_11_26 relies on a custom wrapper script, symlinked to bin/: Trinity-runner.sh. This script changes the current working directory to that of the Trinity.pl perl script before calling Trinity, so relative paths work as expected (particularly for Trinity plugins). All arguments are passed to Trinity.pl (via $@)')])), ('test', CommentedMap([('commands', ['$PREFIX/bin/Trinity-test &> /dev/null'])]))]))\n",
      "EXTRA has been found: trinity\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationhub/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationhub'), ('version', '2.6.0')])), ('source', CommentedMap([('fn', 'AnnotationHub_2.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/AnnotationHub_2.6.0.tar.gz'), ('md5', 'b5dc791eaa7a3a7f1b4ee03ffb1c93f1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-biocinstaller', 'bioconductor-interactivedisplaybase', 'bioconductor-s4vectors', 'r-base', 'r-httr', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-biocinstaller', 'bioconductor-interactivedisplaybase', 'bioconductor-s4vectors', 'r-base', 'r-httr', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationHub\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/AnnotationHub.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides a client for the Bioconductor AnnotationHub web resource. The AnnotationHub web resource provides a central location where genomic files (e.g., VCF, bed, wig) and other resources from standard locations (e.g., UCSC, Ensembl) can be discovered. The resource includes metadata about each resource, e.g., a textual description, tags, and date of modification. The client creates and manages a local cache of files retrieved by the user, helping with quick and reproducible access.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationhub/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationhub'), ('version', '2.6.0')])), ('source', CommentedMap([('fn', 'AnnotationHub_2.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/AnnotationHub_2.6.0.tar.gz'), ('md5', 'b5dc791eaa7a3a7f1b4ee03ffb1c93f1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-biocinstaller', 'bioconductor-interactivedisplaybase', 'bioconductor-s4vectors', 'r-base', 'r-httr', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-biocinstaller', 'bioconductor-interactivedisplaybase', 'bioconductor-s4vectors', 'r-base', 'r-httr', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationHub\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/AnnotationHub.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides a client for the Bioconductor AnnotationHub web resource. The AnnotationHub web resource provides a central location where genomic files (e.g., VCF, bed, wig) and other resources from standard locations (e.g., UCSC, Ensembl) can be discovered. The resource includes metadata about each resource, e.g., a textual description, tags, and date of modification. The client creates and manages a local cache of files retrieved by the user, helping with quick and reproducible access.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-mutationalpatterns/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-mutationalpatterns'), ('version', '0.2b')])), ('source', CommentedMap([('fn', 'MutationalPatterns-v0.2-beta.tar.gz'), ('url', 'https://github.com/CuppenResearch/MutationalPatterns/archive/v0.2-beta.tar.gz')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-genomeinfodb', 'bioconductor-iranges', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-ggplot2', 'r-gridextra', 'r-plyr', 'r-pracma', 'r-nmf', 'r-reshape2']), ('run', ['r-base', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-genomeinfodb', 'bioconductor-iranges', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-ggplot2', 'r-gridextra', 'r-plyr', 'r-pracma', 'r-nmf', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MutationalPatterns\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/CuppenResearch/MutationalPatterns'), ('license', 'MIT'), ('summary', 'R package for extracting and visualizing mutational patterns in base substitution catalogues')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-mutationalpatterns/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-mutationalpatterns'), ('version', '0.2b')])), ('source', CommentedMap([('fn', 'MutationalPatterns-v0.2-beta.tar.gz'), ('url', 'https://github.com/CuppenResearch/MutationalPatterns/archive/v0.2-beta.tar.gz')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-genomeinfodb', 'bioconductor-iranges', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-ggplot2', 'r-gridextra', 'r-plyr', 'r-pracma', 'r-nmf', 'r-reshape2']), ('run', ['r-base', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-genomeinfodb', 'bioconductor-iranges', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation', 'r-ggplot2', 'r-gridextra', 'r-plyr', 'r-pracma', 'r-nmf', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MutationalPatterns\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/CuppenResearch/MutationalPatterns'), ('license', 'MIT'), ('summary', 'R package for extracting and visualizing mutational patterns in base substitution catalogues')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-r4rna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-r4rna'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'R4RNA_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/R4RNA_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/R4RNA_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/R4RNA/R4RNA_1.4.0_src_all.tar.gz']), ('md5', '1569e97b37150559d3ac7c82addf415a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.38.0', 'r-base']), ('run', ['bioconductor-biostrings >=2.38.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'R4RNA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/R4RNA.html'), ('license', 'GPL-3'), ('summary', 'A package for RNA basepair analysis, including the visualization of basepairs as arc diagrams for easy comparison and annotation of sequence and structure.  Arc diagrams can additionally be projected onto multiple sequence alignments to assess basepair conservation and covariation, with numerical methods for computing statistics for each.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-r4rna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-r4rna'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'R4RNA_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/R4RNA_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/R4RNA_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/R4RNA/R4RNA_1.4.0_src_all.tar.gz']), ('md5', '1569e97b37150559d3ac7c82addf415a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.38.0', 'r-base']), ('run', ['bioconductor-biostrings >=2.38.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'R4RNA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/R4RNA.html'), ('license', 'GPL-3'), ('summary', 'A package for RNA basepair analysis, including the visualization of basepairs as arc diagrams for easy comparison and annotation of sequence and structure.  Arc diagrams can additionally be projected onto multiple sequence alignments to assess basepair conservation and covariation, with numerical methods for computing statistics for each.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/phyml/3.2.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name | lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name | lower }}-<{ version }}.tar.gz'), ('url', 'https://github.com/stephaneguindon/<{ name | lower }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['pin_version.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'autoconf', 'automake', 'pkg-config', 'openmpi 2.*', 'beagle-lib', 'libtool']), ('run', ['libgcc', 'openmpi 2.*', 'beagle-lib', 'libtool'])])), ('test', CommentedMap([('requirements', ['conda']), ('commands', ['if command -v conda; then conda inspect linkages -p $PREFIX $PKG_NAME; fi', 'if command -v conda; then conda inspect objects -p $PREFIX $PKG_NAME; fi', 'phyml --help | grep \"PhyML <{ version }}\"', 'mpirun -mca orte_rsh_agent true --allow-run-as-root phyml-mpi --help | grep \"PhyML <{ version }}\"', 'phyml-beagle --help | grep \"PhyML <{ version }}\"'])])), ('about', CommentedMap([('home', 'http://www.atgc-montpellier.fr/phyml/'), ('dev_url', 'https://github.com/stephaneguindon/phyml/'), ('license', 'GPLv2'), ('license_family', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Phylogenetic estimation using (Maximum) Likelihood'), ('description', 'PhyML is a software that estimates maximum likelihood phylogenies from\\nalignments of nucleotide or amino acid sequences. The main strength of\\nPhyML lies in the large number of substitution models coupled to various\\noptions to search the space of phylogenetic tree topologies, going from\\nvery fast and efficient methods to slower but generally more accurate\\napproaches. PhyML was designed to process moderate to large data sets. In\\ntheory, alignments with up to 4,000 sequences 2,000,000 character-long\\ncan be processed. PhyML can process data sets made of multiple genes and\\nfit sophisticated substitution models with heterogeneous components\\nacross partition elements.\\n')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/phyml/3.2.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name | lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name | lower }}-<{ version }}.tar.gz'), ('url', 'https://github.com/stephaneguindon/<{ name | lower }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['pin_version.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'autoconf', 'automake', 'pkg-config', 'openmpi 2.*', 'beagle-lib', 'libtool']), ('run', ['libgcc', 'openmpi 2.*', 'beagle-lib', 'libtool'])])), ('test', CommentedMap([('requirements', ['conda']), ('commands', ['if command -v conda; then conda inspect linkages -p $PREFIX $PKG_NAME; fi', 'if command -v conda; then conda inspect objects -p $PREFIX $PKG_NAME; fi', 'phyml --help | grep \"PhyML <{ version }}\"', 'mpirun -mca orte_rsh_agent true --allow-run-as-root phyml-mpi --help | grep \"PhyML <{ version }}\"', 'phyml-beagle --help | grep \"PhyML <{ version }}\"'])])), ('about', CommentedMap([('home', 'http://www.atgc-montpellier.fr/phyml/'), ('dev_url', 'https://github.com/stephaneguindon/phyml/'), ('license', 'GPLv2'), ('license_family', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Phylogenetic estimation using (Maximum) Likelihood'), ('description', 'PhyML is a software that estimates maximum likelihood phylogenies from\\nalignments of nucleotide or amino acid sequences. The main strength of\\nPhyML lies in the large number of substitution models coupled to various\\noptions to search the space of phylogenetic tree topologies, going from\\nvery fast and efficient methods to slower but generally more accurate\\napproaches. PhyML was designed to process moderate to large data sets. In\\ntheory, alignments with up to 4,000 sequences 2,000,000 character-long\\ncan be processed. PhyML can process data sets made of multiple genes and\\nfit sophisticated substitution models with heterogeneous components\\nacross partition elements.\\n')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/velvet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'velvet'), ('version', '1.2.10')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'velvet_1.2.10.tgz'), ('url', 'https://www.ebi.ac.uk/~zerbino/velvet/velvet_1.2.10.tgz'), ('sha256', '884dd488c2d12f1f89cdc530a266af5d3106965f21ab9149e8cb5c633c977640')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['velvetg | grep velvetg > /dev/null', 'velveth | grep velveth > /dev/null'])])), ('about', CommentedMap([('home', 'https://www.ebi.ac.uk/~zerbino/velvet/'), ('license', 'GPL'), ('summary', 'Sequence Assembler for short reads')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/velvet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'velvet'), ('version', '1.2.10')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'velvet_1.2.10.tgz'), ('url', 'https://www.ebi.ac.uk/~zerbino/velvet/velvet_1.2.10.tgz'), ('sha256', '884dd488c2d12f1f89cdc530a266af5d3106965f21ab9149e8cb5c633c977640')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['velvetg | grep velvetg > /dev/null', 'velveth | grep velveth > /dev/null'])])), ('about', CommentedMap([('home', 'https://www.ebi.ac.uk/~zerbino/velvet/'), ('license', 'GPL'), ('summary', 'Sequence Assembler for short reads')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lastz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lastz'), ('version', '1.0.2')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'lastz-1.02.00.tar.gz'), ('url', 'http://www.bx.psu.edu/miller_lab/dist/lastz-1.02.00.tar.gz'), ('md5', 'b5b9a9b4e088aad720ff5ad6eaa7cac4'), ('patches', ['makefile.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['lastz --version  2>&1 | grep 1.02.00 > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.bx.psu.edu/~rsharris/lastz/'), ('license', 'MIT'), ('summary', 'LASTZ is a program for aligning DNA sequences, a pairwise aligne')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/lastz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'lastz'), ('version', '1.0.2')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'lastz-1.02.00.tar.gz'), ('url', 'http://www.bx.psu.edu/miller_lab/dist/lastz-1.02.00.tar.gz'), ('md5', 'b5b9a9b4e088aad720ff5ad6eaa7cac4'), ('patches', ['makefile.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['lastz --version  2>&1 | grep 1.02.00 > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.bx.psu.edu/~rsharris/lastz/'), ('license', 'MIT'), ('summary', 'LASTZ is a program for aligning DNA sequences, a pairwise aligne')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-marray/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-marray'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'marray_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/marray_1.50.0.tar.gz'), ('md5', 'c068e960357aa8d9d84b9f517dd1ef50')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base']), ('run', ['bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'marray\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/marray.html'), ('license', 'LGPL'), ('summary', 'Class definitions for two-color spotted microarray data. Fuctions for data input, diagnostic plots, normalization and quality checking.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-marray/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-marray'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'marray_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/marray_1.50.0.tar.gz'), ('md5', 'c068e960357aa8d9d84b9f517dd1ef50')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base']), ('run', ['bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'marray\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/marray.html'), ('license', 'LGPL'), ('summary', 'Class definitions for two-color spotted microarray data. Fuctions for data input, diagnostic plots, normalization and quality checking.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/discosnp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'discosnp'), ('version', '2.2.10')])), ('source', CommentedMap([('fn', 'DiscoSNP.-v2.2.10-Source.tar.gz'), ('url', 'https://github.com/GATB/DiscoSnp/releases/download/v2.2.10/DiscoSNP.-v2.2.10-Source.tar.gz'), ('md5', 'a5590184534e387a21cd35630dc09529')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['run_discoSnp++.sh -h'])])), ('about', CommentedMap([('home', 'https://gatb.inria.fr/software/discosnp/'), ('license', 'GNU Affero General Public License v3.0'), ('summary', 'Designed for discovering all kinds of SNPs')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/discosnp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'discosnp'), ('version', '2.2.10')])), ('source', CommentedMap([('fn', 'DiscoSNP.-v2.2.10-Source.tar.gz'), ('url', 'https://github.com/GATB/DiscoSnp/releases/download/v2.2.10/DiscoSNP.-v2.2.10-Source.tar.gz'), ('md5', 'a5590184534e387a21cd35630dc09529')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['run_discoSnp++.sh -h'])])), ('about', CommentedMap([('home', 'https://gatb.inria.fr/software/discosnp/'), ('license', 'GNU Affero General Public License v3.0'), ('summary', 'Designed for discovering all kinds of SNPs')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sradb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sradb'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'SRAdb_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/SRAdb_1.32.0.tar.gz'), ('md5', '37c10ff958a239ef1b778ca15d166fe0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-geoquery', 'bioconductor-graph', 'r-base', 'r-rcurl', 'r-rsqlite']), ('run', ['bioconductor-geoquery', 'bioconductor-graph', 'r-base', 'r-rcurl', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SRAdb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/SRAdb.html'), ('license', 'Artistic-2.0'), ('summary', 'The Sequence Read Archive (SRA) is the largest public repository of sequencing data from the next generation of sequencing platforms including Roche 454 GS System, Illumina Genome Analyzer, Applied Biosystems SOLiD System, Helicos Heliscope, and others. However, finding data of interest can be challenging using current tools. SRAdb is an attempt to make access to the metadata associated with submission, study, sample, experiment and run much more feasible. This is accomplished by parsing all the NCBI SRA metadata into a SQLite database that can be stored and queried locally. Fulltext search in the package make querying metadata very flexible and powerful.  fastq and sra files can be downloaded for doing alignment locally. Beside ftp protocol, the SRAdb has funcitons supporting fastp protocol (ascp from Aspera Connect) for faster downloading large data files over long distance. The SQLite database is updated regularly as new data is added to SRA and can be downloaded at will for the most up-to-date metadata.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sradb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sradb'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'SRAdb_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/SRAdb_1.32.0.tar.gz'), ('md5', '37c10ff958a239ef1b778ca15d166fe0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-geoquery', 'bioconductor-graph', 'r-base', 'r-rcurl', 'r-rsqlite']), ('run', ['bioconductor-geoquery', 'bioconductor-graph', 'r-base', 'r-rcurl', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SRAdb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/SRAdb.html'), ('license', 'Artistic-2.0'), ('summary', 'The Sequence Read Archive (SRA) is the largest public repository of sequencing data from the next generation of sequencing platforms including Roche 454 GS System, Illumina Genome Analyzer, Applied Biosystems SOLiD System, Helicos Heliscope, and others. However, finding data of interest can be challenging using current tools. SRAdb is an attempt to make access to the metadata associated with submission, study, sample, experiment and run much more feasible. This is accomplished by parsing all the NCBI SRA metadata into a SQLite database that can be stored and queried locally. Fulltext search in the package make querying metadata very flexible and powerful.  fastq and sra files can be downloaded for doing alignment locally. Beside ftp protocol, the SRAdb has funcitons supporting fastp protocol (ascp from Aspera Connect) for faster downloading large data files over long distance. The SQLite database is updated regularly as new data is added to SRA and can be downloaded at will for the most up-to-date metadata.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mira/4.9.6/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/p/mira-assembler/wiki/Home/'), ('license', 'GNU General Public License v2 or later (GPLv2+)'), ('license_file', 'LICENCE'), ('summary', 'MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads)')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'mira'), ('version', '4.9.6')])), ('source', CommentedMap([('fn', 'mira_4.9.6.tar.bz2'), ('url', 'https://depot.galaxyproject.org/software/mira/mira_4.9.6_darwin_x64.tar.bz2'), ('sha256', '3d0bb15adad0e1e27899d21223cb552968d34d35ad9b1184e85832fc0b3d22ce')])), ('test', CommentedMap([('commands', ['mira --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mira/4.9.6/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://sourceforge.net/p/mira-assembler/wiki/Home/'), ('license', 'GNU General Public License v2 or later (GPLv2+)'), ('license_file', 'LICENCE'), ('summary', 'MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads)')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'mira'), ('version', '4.9.6')])), ('source', CommentedMap([('fn', 'mira_4.9.6.tar.bz2'), ('url', 'https://depot.galaxyproject.org/software/mira/mira_4.9.6_darwin_x64.tar.bz2'), ('sha256', '3d0bb15adad0e1e27899d21223cb552968d34d35ad9b1184e85832fc0b3d22ce')])), ('test', CommentedMap([('commands', ['mira --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-geneplotter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-geneplotter'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'geneplotter_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/geneplotter_1.50.0.tar.gz'), ('md5', '869b681a8d568ac5e1fe9fc576c8cb65')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'geneplotter\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/geneplotter.html'), ('license', 'Artistic-2.0'), ('summary', 'Functions for plotting genomic data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-geneplotter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-geneplotter'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'geneplotter_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/geneplotter_1.50.0.tar.gz'), ('md5', '869b681a8d568ac5e1fe9fc576c8cb65')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-rcolorbrewer', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'geneplotter\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/geneplotter.html'), ('license', 'Artistic-2.0'), ('summary', 'Functions for plotting genomic data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cutadapt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cutadapt'), ('version', '1.14')])), ('about', CommentedMap([('home', 'https://cutadapt.readthedocs.io/'), ('license', 'MIT License'), ('summary', 'trim adapters from high-throughput sequencing reads')])), ('source', CommentedMap([('fn', 'cutadapt-1.14.tar.gz'), ('url', 'https://pypi.python.org/packages/16/e3/06b45eea35359833e7c6fac824b604f1551c2fc7ba0f2bd318d8dd883eb9/cutadapt-1.14.tar.gz'), ('md5', '57403c3794aa940699df3032a5e9b208')])), ('build', CommentedMap([('number', 0), ('script', '$PYTHON setup.py install')])), ('requirements', CommentedMap([('build', ['python', 'xopen >=0.1.1']), ('run', ['python', 'xopen >=0.1.1'])])), ('test', CommentedMap([('imports', ['cutadapt', 'cutadapt.scripts']), ('commands', ['cutadapt --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cutadapt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cutadapt'), ('version', '1.14')])), ('about', CommentedMap([('home', 'https://cutadapt.readthedocs.io/'), ('license', 'MIT License'), ('summary', 'trim adapters from high-throughput sequencing reads')])), ('source', CommentedMap([('fn', 'cutadapt-1.14.tar.gz'), ('url', 'https://pypi.python.org/packages/16/e3/06b45eea35359833e7c6fac824b604f1551c2fc7ba0f2bd318d8dd883eb9/cutadapt-1.14.tar.gz'), ('md5', '57403c3794aa940699df3032a5e9b208')])), ('build', CommentedMap([('number', 0), ('script', '$PYTHON setup.py install')])), ('requirements', CommentedMap([('build', ['python', 'xopen >=0.1.1']), ('run', ['python', 'xopen >=0.1.1'])])), ('test', CommentedMap([('imports', ['cutadapt', 'cutadapt.scripts']), ('commands', ['cutadapt --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metagenomeseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metagenomeseq'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'metagenomeSeq_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/metagenomeSeq_1.16.0.tar.gz'), ('md5', '537eed40d187418841a19e89a70a802b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma', 'r-base', 'r-foreach', 'r-glmnet', 'r-gplots', 'r-matrixstats', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase', 'bioconductor-limma', 'r-base', 'r-foreach', 'r-glmnet', 'r-gplots', 'r-matrixstats', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metagenomeSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/metagenomeSeq.html'), ('license', 'Artistic-2.0'), ('summary', 'metagenomeSeq is designed to determine features (be it Operational Taxanomic Unit (OTU), species, etc.) that are differentially abundant between two or more groups of multiple samples. metagenomeSeq is designed to address the effects of both normalization and under-sampling of microbial communities on disease association detection and the testing of feature correlations.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metagenomeseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metagenomeseq'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'metagenomeSeq_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/metagenomeSeq_1.16.0.tar.gz'), ('md5', '537eed40d187418841a19e89a70a802b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma', 'r-base', 'r-foreach', 'r-glmnet', 'r-gplots', 'r-matrixstats', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase', 'bioconductor-limma', 'r-base', 'r-foreach', 'r-glmnet', 'r-gplots', 'r-matrixstats', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metagenomeSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/metagenomeSeq.html'), ('license', 'Artistic-2.0'), ('summary', 'metagenomeSeq is designed to determine features (be it Operational Taxanomic Unit (OTU), species, etc.) that are differentially abundant between two or more groups of multiple samples. metagenomeSeq is designed to address the effects of both normalization and under-sampling of microbial communities on disease association detection and the testing of feature correlations.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/busco/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 3)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://gitlab.com/ezlab/busco/repository/archive.tar.bz2?ref=<{ version }}'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'wget', 'blast', 'hmmer', 'augustus >=3.2.3', 'r-ggplot2 >=2.2.1'])])), ('test', CommentedMap([('commands', ['run_BUSCO.py -h', 'run_busco -h', 'generate_plot.py -h', 'generate_plot -h', 'run_busco | grep \"Please add it in the config file\"'])])), ('about', CommentedMap([('home', 'http://busco.ezlab.org/'), ('license', 'GPL'), ('summary', 'BUSCO provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs selected from OrthoDB.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/busco/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 3)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://gitlab.com/ezlab/busco/repository/archive.tar.bz2?ref=<{ version }}'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'wget', 'blast', 'hmmer', 'augustus >=3.2.3', 'r-ggplot2 >=2.2.1'])])), ('test', CommentedMap([('commands', ['run_BUSCO.py -h', 'run_busco -h', 'generate_plot.py -h', 'generate_plot -h', 'run_busco | grep \"Please add it in the config file\"'])])), ('about', CommentedMap([('home', 'http://busco.ezlab.org/'), ('license', 'GPL'), ('summary', 'BUSCO provides measures for quantitative assessment of genome assembly, gene set, and transcriptome completeness based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs selected from OrthoDB.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sortmerna/2.1b/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sortmerna'), ('version', '2.1b')])), ('source', CommentedMap([('fn', '2.1b.tar.gz'), ('url', 'https://github.com/biocore/sortmerna/archive/2.1b.tar.gz'), ('sha256', 'b3d122776c323813971b35991cda21a2c2f3ce817daba68a4c4e09d4367c0abe')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['sortmerna --version', \"merge-paired-reads.sh  --help | grep 'merged-reads.fastq'\", \"unmerge-paired-reads.sh  --help | grep 'merged-reads-file.fastq'\"])])), ('about', CommentedMap([('home', 'http://bioinfo.lifl.fr/RNA/sortmerna'), ('license', 'LGPL'), ('summary', 'SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sortmerna/2.1b/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'sortmerna'), ('version', '2.1b')])), ('source', CommentedMap([('fn', '2.1b.tar.gz'), ('url', 'https://github.com/biocore/sortmerna/archive/2.1b.tar.gz'), ('sha256', 'b3d122776c323813971b35991cda21a2c2f3ce817daba68a4c4e09d4367c0abe')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['sortmerna --version', \"merge-paired-reads.sh  --help | grep 'merged-reads.fastq'\", \"unmerge-paired-reads.sh  --help | grep 'merged-reads-file.fastq'\"])])), ('about', CommentedMap([('home', 'http://bioinfo.lifl.fr/RNA/sortmerna'), ('license', 'LGPL'), ('summary', 'SortMeRNA is a biological sequence analysis tool for filtering, mapping and OTU-picking NGS reads.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-streamer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-streamer'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'Streamer_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Streamer_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/Streamer_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/Streamer/Streamer_1.22.0_src_all.tar.gz']), ('md5', 'c1c584dd90ea170f2d19db19df0b4de1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Streamer\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Streamer.html'), ('license', 'Artistic-2.0'), ('summary', \"Large data files can be difficult to work with in R, where data generally resides in memory. This package encourages a style of programming where data is 'streamed' from disk into R via a `producer' and through a series of `consumers' that, typically reduce the original data to a manageable size. The package provides useful Producer and Consumer stream components for operations such as data input, sampling, indexing, and transformation; see package?Streamer for details.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-streamer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-streamer'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'Streamer_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Streamer_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/Streamer_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/Streamer/Streamer_1.22.0_src_all.tar.gz']), ('md5', 'c1c584dd90ea170f2d19db19df0b4de1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-graph', 'bioconductor-rbgl', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Streamer\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Streamer.html'), ('license', 'Artistic-2.0'), ('summary', \"Large data files can be difficult to work with in R, where data generally resides in memory. This package encourages a style of programming where data is 'streamed' from disk into R via a `producer' and through a series of `consumers' that, typically reduce the original data to a manageable size. The package provides useful Producer and Consumer stream components for operations such as data input, sampling, indexing, and transformation; see package?Streamer for details.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-globalseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-globalseq'), ('version', '1.4.2')])), ('source', CommentedMap([('fn', 'globalSeq_1.4.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/globalSeq_1.4.2.tar.gz', 'https://depot.galaxyproject.org/software/globalSeq/globalSeq_1.4.2_src_all.tar.gz']), ('md5', 'df77fed2999dca5a318409dd15141493')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'globalSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/globalSeq.html'), ('license', 'GPL-3'), ('summary', 'The method may be conceptualised as a test of overall significance in regression analysis, where the response variable is overdispersed and the number of explanatory variables exceeds the sample size.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-globalseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-globalseq'), ('version', '1.4.2')])), ('source', CommentedMap([('fn', 'globalSeq_1.4.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/globalSeq_1.4.2.tar.gz', 'https://depot.galaxyproject.org/software/globalSeq/globalSeq_1.4.2_src_all.tar.gz']), ('md5', 'df77fed2999dca5a318409dd15141493')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'globalSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/globalSeq.html'), ('license', 'GPL-3'), ('summary', 'The method may be conceptualised as a test of overall significance in regression analysis, where the response variable is overdispersed and the number of explanatory variables exceeds the sample size.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-xvector/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-xvector'), ('version', '0.14.1')])), ('source', CommentedMap([('fn', 'XVector_0.14.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/XVector_0.14.1.tar.gz'), ('md5', '27445c0311b9eab3cf1cf032bc55f538')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.19.2', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.9.29', 'bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.19.2', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.9.29', 'bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'XVector\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/XVector.html'), ('license', 'Artistic-2.0'), ('summary', 'Memory efficient S4 classes for storing sequences \"externally\" (behind an R external pointer, or on disk).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-xvector/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-xvector'), ('version', '0.14.1')])), ('source', CommentedMap([('fn', 'XVector_0.14.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/XVector_0.14.1.tar.gz'), ('md5', '27445c0311b9eab3cf1cf032bc55f538')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.19.2', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.9.29', 'bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.19.2', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.9.29', 'bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'XVector\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/XVector.html'), ('license', 'Artistic-2.0'), ('summary', 'Memory efficient S4 classes for storing sequences \"externally\" (behind an R external pointer, or on disk).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/skewer/0.1.126/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'skewer'), ('version', '0.1.126')])), ('source', CommentedMap([('fn', '0.1.126.zip'), ('url', 'https://github.com/relipmoc/skewer/archive/0.1.126.zip')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['skewer 2>&1 | grep \"skewer (0.1.126)\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/relipmoc/skewer.git'), ('license', 'MIT'), ('summary', 'A fast and accurate adapter trimmer for next-generation sequencing paired-end reads.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/skewer/0.1.126/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'skewer'), ('version', '0.1.126')])), ('source', CommentedMap([('fn', '0.1.126.zip'), ('url', 'https://github.com/relipmoc/skewer/archive/0.1.126.zip')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['skewer 2>&1 | grep \"skewer (0.1.126)\" > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/relipmoc/skewer.git'), ('license', 'MIT'), ('summary', 'A fast and accurate adapter trimmer for next-generation sequencing paired-end reads.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/merlin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'merlin'), ('version', '1.1.2')])), ('source', CommentedMap([('fn', 'merlin-1.1.2.tar.gz'), ('url', 'http://csg.sph.umich.edu/abecasis/merlin/download/merlin-1.1.2.tar.gz'), ('md5', '01ce17395aff3fba8d49447aeb2128fa'), ('sha1', 'd5227276eae862506c7ed4475255c9be182585df'), ('sha256', 'aa3c087b1c717209389705bf2ace4c24884575014a561dff250c0be93fe1e411')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['merlin | grep MERLIN'])])), ('about', CommentedMap([('home', 'http://csg.sph.umich.edu/abecasis/merlin'), ('license', 'OpenSource'), ('license_file', 'LICENSE.twister'), ('summary', 'MERLIN uses sparse trees to represent gene flow in pedigrees and is a fast pedigree analysis package')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/merlin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'merlin'), ('version', '1.1.2')])), ('source', CommentedMap([('fn', 'merlin-1.1.2.tar.gz'), ('url', 'http://csg.sph.umich.edu/abecasis/merlin/download/merlin-1.1.2.tar.gz'), ('md5', '01ce17395aff3fba8d49447aeb2128fa'), ('sha1', 'd5227276eae862506c7ed4475255c9be182585df'), ('sha256', 'aa3c087b1c717209389705bf2ace4c24884575014a561dff250c0be93fe1e411')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['merlin | grep MERLIN'])])), ('about', CommentedMap([('home', 'http://csg.sph.umich.edu/abecasis/merlin'), ('license', 'OpenSource'), ('license_file', 'LICENSE.twister'), ('summary', 'MERLIN uses sparse trees to represent gene flow in pedigrees and is a fast pedigree analysis package')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/edena/meta.yaml', CommentedMap([('source', CommentedMap([('fn', 'EdenaV<{ version }}.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'http://www.genomic.ch/edena/EdenaV<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['edena -h'])])), ('about', CommentedMap([('home', 'http://www.genomic.ch/edena.php'), ('license', 'GPL3'), ('summary', 'de novo short reads assembler')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/edena/meta.yaml', CommentedMap([('source', CommentedMap([('fn', 'EdenaV<{ version }}.tar.gz'), ('md5', '<{ md5 }}'), ('url', 'http://www.genomic.ch/edena/EdenaV<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['edena -h'])])), ('about', CommentedMap([('home', 'http://www.genomic.ch/edena.php'), ('license', 'GPL3'), ('summary', 'de novo short reads assembler')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rots/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rots'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'ROTS_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ROTS_1.0.0.tar.gz'), ('md5', '4891e471a606cc72998bd0d87da0066d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-biobase', 'r-base', 'r-rcpp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ROTS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ROTS.html'), ('license', 'GPL (>= 2)'), ('summary', 'Calculates the Reproducibility-Optimized Test Statistic (ROTS) for differential testing in omics data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rots/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rots'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'ROTS_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ROTS_1.0.0.tar.gz'), ('md5', '4891e471a606cc72998bd0d87da0066d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-biobase', 'r-base', 'r-rcpp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ROTS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ROTS.html'), ('license', 'GPL (>= 2)'), ('summary', 'Calculates the Reproducibility-Optimized Test Statistic (ROTS) for differential testing in omics data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pathview/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pathview'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'pathview_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/pathview_<{ version }}.tar.gz'), ('sha256', '21db53ffa15bea36c1c6699eee60eae9537f078b6d68806e9defe062652ea9fd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-org.hs.eg.db', 'bioconductor-rgraphviz', 'r-base', 'r-png', 'r-xml']), ('run', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-org.hs.eg.db', 'bioconductor-rgraphviz', 'r-base', 'r-png', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pathview\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/pathview.html'), ('license', 'GPL (>=3.0)'), ('summary', 'Pathview is a tool set for pathway based data integration and visualization. It maps and renders a wide variety of biological data on relevant pathway graphs. All users need is to supply their data and specify the target pathway. Pathview automatically downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, Pathview also seamlessly integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pathview/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pathview'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'pathview_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/pathview_<{ version }}.tar.gz'), ('sha256', '21db53ffa15bea36c1c6699eee60eae9537f078b6d68806e9defe062652ea9fd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-org.hs.eg.db', 'bioconductor-rgraphviz', 'r-base', 'r-png', 'r-xml']), ('run', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-kegggraph', 'bioconductor-keggrest', 'bioconductor-org.hs.eg.db', 'bioconductor-rgraphviz', 'r-base', 'r-png', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pathview\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/pathview.html'), ('license', 'GPL (>=3.0)'), ('summary', 'Pathview is a tool set for pathway based data integration and visualization. It maps and renders a wide variety of biological data on relevant pathway graphs. All users need is to supply their data and specify the target pathway. Pathview automatically downloads the pathway graph data, parses the data file, maps user data to the pathway, and render pathway graph with the mapped data. In addition, Pathview also seamlessly integrates with pathway and gene set (enrichment) analysis tools for large-scale and fully automated analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-amountain/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-amountain'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'AMOUNTAIN_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AMOUNTAIN_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/AMOUNTAIN_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/AMOUNTAIN/AMOUNTAIN_1.2.0_src_all.tar.gz']), ('md5', 'd6994f3f83620937b4a6110d505403ed')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AMOUNTAIN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AMOUNTAIN.html'), ('license', 'GPL (>= 2)'), ('summary', 'A pure data-driven gene network, weighted gene co-expression network (WGCN) could be constructed only from expression profile. Different layers in such networks may represent different time points, multiple conditions or various species. AMOUNTAIN aims to search active modules in multi-layer WGCN using a continuous optimization approach.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-amountain/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-amountain'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'AMOUNTAIN_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AMOUNTAIN_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/AMOUNTAIN_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/AMOUNTAIN/AMOUNTAIN_1.2.0_src_all.tar.gz']), ('md5', 'd6994f3f83620937b4a6110d505403ed')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AMOUNTAIN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AMOUNTAIN.html'), ('license', 'GPL (>= 2)'), ('summary', 'A pure data-driven gene network, weighted gene co-expression network (WGCN) could be constructed only from expression profile. Different layers in such networks may represent different time points, multiple conditions or various species. AMOUNTAIN aims to search active modules in multi-layer WGCN using a continuous optimization approach.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rlmm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rlmm'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'RLMM_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RLMM_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/RLMM_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/RLMM/RLMM_1.38.0_src_all.tar.gz']), ('md5', 'af5d1ecdc28a9d4d1563a298d6673fa3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass']), ('run', ['r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RLMM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RLMM.html'), ('license', 'LGPL (>= 2)'), ('summary', 'A classification algorithm, based on a multi-chip, multi-SNP approach for Affymetrix SNP arrays. Using a large training sample where the genotype labels are known, this aglorithm will obtain more accurate classification results on new data. RLMM is based on a robust, linear model and uses the Mahalanobis distance for classification. The chip-to-chip non-biological variation is removed through normalization. This model-based algorithm captures the similarities across genotype groups and probes, as well as thousands other SNPs for accurate classification. NOTE: 100K-Xba only at for now.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rlmm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rlmm'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'RLMM_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RLMM_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/RLMM_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/RLMM/RLMM_1.38.0_src_all.tar.gz']), ('md5', 'af5d1ecdc28a9d4d1563a298d6673fa3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass']), ('run', ['r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RLMM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RLMM.html'), ('license', 'LGPL (>= 2)'), ('summary', 'A classification algorithm, based on a multi-chip, multi-SNP approach for Affymetrix SNP arrays. Using a large training sample where the genotype labels are known, this aglorithm will obtain more accurate classification results on new data. RLMM is based on a robust, linear model and uses the Mahalanobis distance for classification. The chip-to-chip non-biological variation is removed through normalization. This model-based algorithm captures the similarities across genotype groups and probes, as well as thousands other SNPs for accurate classification. NOTE: 100K-Xba only at for now.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mapsplice/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mapsplice'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'MapSplice-v2.2.0.zip'), ('url', 'http://protocols.netlab.uky.edu/~zeng/MapSplice-v2.2.0.zip'), ('md5', '171975d48d2ddcc111a314d926f3527a'), ('patches', ['mapsplice.patch', 'samtools.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'py<{CONDA_PY}}_ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['python', 'zlib', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['python', 'zlib', 'ncurses <{CONDA_NCURSES}}*'])])), ('test', CommentedMap([('commands', ['mapsplice.py -v 2>&1 |grep MapSplice > /dev/null'])])), ('about', CommentedMap([('license', 'Custom'), ('summary', 'MapSplice is a software for mapping RNA-seq data to reference genome for splice junction discovery that depends only on reference genome, and not on any further annotations.'), ('license_file', 'Copyright.txt')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mapsplice/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mapsplice'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'MapSplice-v2.2.0.zip'), ('url', 'http://protocols.netlab.uky.edu/~zeng/MapSplice-v2.2.0.zip'), ('md5', '171975d48d2ddcc111a314d926f3527a'), ('patches', ['mapsplice.patch', 'samtools.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'py<{CONDA_PY}}_ncurses<{CONDA_NCURSES}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['python', 'zlib', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['python', 'zlib', 'ncurses <{CONDA_NCURSES}}*'])])), ('test', CommentedMap([('commands', ['mapsplice.py -v 2>&1 |grep MapSplice > /dev/null'])])), ('about', CommentedMap([('license', 'Custom'), ('summary', 'MapSplice is a software for mapping RNA-seq data to reference genome for splice junction discovery that depends only on reference genome, and not on any further annotations.'), ('license_file', 'Copyright.txt')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowcl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowcl'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'flowCL_1.14.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/flowCL_1.14.0.tar.gz'), ('md5', '5d523ab593a8f48d52db0ac70249c946')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-rgraphviz', 'r-base', 'r-sparql']), ('run', ['bioconductor-graph', 'bioconductor-rgraphviz', 'r-base', 'r-sparql'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowCL\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowCL.html'), ('license', 'Artistic-2.0'), ('summary', 'Semantic labelling of flow cytometric cell populations.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowcl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowcl'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'flowCL_1.14.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/flowCL_1.14.0.tar.gz'), ('md5', '5d523ab593a8f48d52db0ac70249c946')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-rgraphviz', 'r-base', 'r-sparql']), ('run', ['bioconductor-graph', 'bioconductor-rgraphviz', 'r-base', 'r-sparql'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowCL\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowCL.html'), ('license', 'Artistic-2.0'), ('summary', 'Semantic labelling of flow cytometric cell populations.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/whatshap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'whatshap'), ('version', '0.14.1')])), ('about', CommentedMap([('home', 'https://whatshap.readthedocs.io/'), ('license', 'MIT License'), ('summary', 'phase genomic variants using DNA sequencing reads (haplotype assembly)')])), ('source', CommentedMap([('fn', 'whatshap-0.14.1.tar.gz'), ('url', 'https://pypi.python.org/packages/ec/d4/d6c19b716926895ee3def919ab23194212172ed5cf89ccb91c2949fb258d/whatshap-0.14.1.tar.gz'), ('md5', 'edaaeada862844285de1fee9d76da93d')])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'setuptools', 'pysam >0.11.0', 'pyvcf', 'pyfaidx', 'xopen']), ('run', ['libgcc', 'python', 'pysam >0.11.0', 'pyvcf', 'pyfaidx', 'xopen'])])), ('build', CommentedMap([('skip', True), ('script', '$PYTHON setup.py install --single-version-externally-managed --record=record.txt')])), ('test', CommentedMap([('imports', ['whatshap']), ('commands', ['whatshap --help > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/whatshap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'whatshap'), ('version', '0.14.1')])), ('about', CommentedMap([('home', 'https://whatshap.readthedocs.io/'), ('license', 'MIT License'), ('summary', 'phase genomic variants using DNA sequencing reads (haplotype assembly)')])), ('source', CommentedMap([('fn', 'whatshap-0.14.1.tar.gz'), ('url', 'https://pypi.python.org/packages/ec/d4/d6c19b716926895ee3def919ab23194212172ed5cf89ccb91c2949fb258d/whatshap-0.14.1.tar.gz'), ('md5', 'edaaeada862844285de1fee9d76da93d')])), ('requirements', CommentedMap([('build', ['gcc', 'python', 'setuptools', 'pysam >0.11.0', 'pyvcf', 'pyfaidx', 'xopen']), ('run', ['libgcc', 'python', 'pysam >0.11.0', 'pyvcf', 'pyfaidx', 'xopen'])])), ('build', CommentedMap([('skip', True), ('script', '$PYTHON setup.py install --single-version-externally-managed --record=record.txt')])), ('test', CommentedMap([('imports', ['whatshap']), ('commands', ['whatshap --help > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-oligoclasses/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-oligoclasses'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'oligoClasses_1.38.0.tar.gz'), ('url', 'https://bioconductor.org/packages/3.5/bioc/src/contrib/oligoClasses_1.38.0.tar.gz'), ('md5', 'dc3ec1a000bbe9dd1ba5e6fc07e7d6e0')])), ('build', CommentedMap([('script', 'R CMD INSTALL --build .'), ('skip', True)])), ('requirements', CommentedMap([('build', ['r-base', 'r-ff', 'r-foreach', 'r-rcpp', 'r-digest', 'r-rsqlite', 'r-rcurl', 'bioconductor-affyio', 'bioconductor-biostrings', 'bioconductor-summarizedexperiment', 'bioconductor-biocinstaller']), ('run', ['r-base', 'r-ff', 'r-foreach', 'r-rcpp', 'r-digest', 'r-rsqlite', 'r-rcurl', 'bioconductor-affyio', 'bioconductor-biostrings', 'bioconductor-summarizedexperiment', 'bioconductor-biocinstaller'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'oligoClasses\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/3.5/bioc/html/oligoClasses.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package contains class definitions, validity checks, and initialization methods for classes used by the oligo and crlmm packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-oligoclasses/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-oligoclasses'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'oligoClasses_1.38.0.tar.gz'), ('url', 'https://bioconductor.org/packages/3.5/bioc/src/contrib/oligoClasses_1.38.0.tar.gz'), ('md5', 'dc3ec1a000bbe9dd1ba5e6fc07e7d6e0')])), ('build', CommentedMap([('script', 'R CMD INSTALL --build .'), ('skip', True)])), ('requirements', CommentedMap([('build', ['r-base', 'r-ff', 'r-foreach', 'r-rcpp', 'r-digest', 'r-rsqlite', 'r-rcurl', 'bioconductor-affyio', 'bioconductor-biostrings', 'bioconductor-summarizedexperiment', 'bioconductor-biocinstaller']), ('run', ['r-base', 'r-ff', 'r-foreach', 'r-rcpp', 'r-digest', 'r-rsqlite', 'r-rcurl', 'bioconductor-affyio', 'bioconductor-biostrings', 'bioconductor-summarizedexperiment', 'bioconductor-biocinstaller'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'oligoClasses\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/3.5/bioc/html/oligoClasses.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package contains class definitions, validity checks, and initialization methods for classes used by the oligo and crlmm packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/deeptools/2.4.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'deeptools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'deepTools-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/2e/21/a6836deaee7a48025182bf8588acb58716b7b216342479b6bf225f0b147b/deepTools-<{ version }}.tar.gz'), ('md5', '9ab5327fa198ed317fc20d2bea71196b')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.9.0', 'scipy >=0.17.0', 'matplotlib >=1.4.0', 'pysam >=0.8.2', 'py2bit >=0.2.0', 'gcc']), ('run', ['python', 'pybigwig >=0.2.3', 'numpy >=1.9.0', 'scipy >=0.17.0', 'matplotlib >=1.4.0', 'pysam >=0.8.2', 'py2bit >=0.2.0'])])), ('test', CommentedMap([('imports', ['deeptools']), ('commands', ['bamCompare --version'])])), ('about', CommentedMap([('home', 'https://github.com/fidelram/deepTools'), ('license', 'GPL3'), ('summary', 'A set of user-friendly tools for normalization and visualzation of deep-sequencing data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/deeptools/2.4.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'deeptools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'deepTools-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/2e/21/a6836deaee7a48025182bf8588acb58716b7b216342479b6bf225f0b147b/deepTools-<{ version }}.tar.gz'), ('md5', '9ab5327fa198ed317fc20d2bea71196b')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy >=1.9.0', 'scipy >=0.17.0', 'matplotlib >=1.4.0', 'pysam >=0.8.2', 'py2bit >=0.2.0', 'gcc']), ('run', ['python', 'pybigwig >=0.2.3', 'numpy >=1.9.0', 'scipy >=0.17.0', 'matplotlib >=1.4.0', 'pysam >=0.8.2', 'py2bit >=0.2.0'])])), ('test', CommentedMap([('imports', ['deeptools']), ('commands', ['bamCompare --version'])])), ('about', CommentedMap([('home', 'https://github.com/fidelram/deepTools'), ('license', 'GPL3'), ('summary', 'A set of user-friendly tools for normalization and visualzation of deep-sequencing data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stringtie/1.3.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/'), ('license', 'Artistic License 2.0'), ('summary', 'Transcriptome assembly and quantification for RNA-seq')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'stringtie'), ('version', '1.3.0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('source', CommentedMap([('fn', 'stringtie-1.3.0.tar.gz'), ('sha256', '3a21c662904cdab0c749d5c3f5ef988e84fa036857ce2ca35782b3c19469c652'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/stringtie-1.3.0.tar.gz')])), ('test', CommentedMap([('commands', ['stringtie 2>&1 | grep Assemble'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stringtie/1.3.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/'), ('license', 'Artistic License 2.0'), ('summary', 'Transcriptome assembly and quantification for RNA-seq')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'stringtie'), ('version', '1.3.0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('source', CommentedMap([('fn', 'stringtie-1.3.0.tar.gz'), ('sha256', '3a21c662904cdab0c749d5c3f5ef988e84fa036857ce2ca35782b3c19469c652'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/stringtie-1.3.0.tar.gz')])), ('test', CommentedMap([('commands', ['stringtie 2>&1 | grep Assemble'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-degseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-degseq'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'DEGseq_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/DEGseq_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/DEGseq_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/DEGseq/DEGseq_1.30.0_src_all.tar.gz']), ('md5', 'f5dbf8f5bbe4de681f37a451f75fd9f9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-qvalue', 'r-base', 'r-samr']), ('run', ['bioconductor-qvalue', 'r-base', 'r-samr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DEGseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/DEGseq.html'), ('license', 'LGPL (>=2)'), ('summary', 'DEGseq is an R package to identify differentially expressed genes from RNA-Seq data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-degseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-degseq'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'DEGseq_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/DEGseq_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/DEGseq_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/DEGseq/DEGseq_1.30.0_src_all.tar.gz']), ('md5', 'f5dbf8f5bbe4de681f37a451f75fd9f9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-qvalue', 'r-base', 'r-samr']), ('run', ['bioconductor-qvalue', 'r-base', 'r-samr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DEGseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/DEGseq.html'), ('license', 'LGPL (>=2)'), ('summary', 'DEGseq is an R package to identify differentially expressed genes from RNA-Seq data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prokka/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'prokka'), ('version', '1.12')])), ('source', CommentedMap([('url', 'https://github.com/tseemann/prokka/archive/v1.12.tar.gz'), ('md5', '658c4c203ddded3623e68a36f94cabec')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['perl-bioperl ==1.6.924', 'blast >=2.6']), ('run', ['aragorn >=1.2', 'barrnap >=0.7', 'blast >=2.6', 'hmmer >=3.1b2', 'infernal >=1.1.2', 'minced >=0.2', 'parallel >=20130422', 'perl-bioperl ==1.6.924', 'perl-xml-simple', 'prodigal >=2.6', 'tbl2asn >=25.3', 'libidn'])])), ('test', CommentedMap([('commands', ['prokka --listdb', 'prokka --version', 'blastp -help', 'barrnap --help 2>&1 | grep Options', 'prodigal -h'])])), ('about', CommentedMap([('home', 'http://www.vicbioinformatics.com/software.prokka.shtml'), ('license', 'GPLv2'), ('summary', 'A tool for the rapid annotation of prokaryotic genomes')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prokka/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'prokka'), ('version', '1.12')])), ('source', CommentedMap([('url', 'https://github.com/tseemann/prokka/archive/v1.12.tar.gz'), ('md5', '658c4c203ddded3623e68a36f94cabec')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['perl-bioperl ==1.6.924', 'blast >=2.6']), ('run', ['aragorn >=1.2', 'barrnap >=0.7', 'blast >=2.6', 'hmmer >=3.1b2', 'infernal >=1.1.2', 'minced >=0.2', 'parallel >=20130422', 'perl-bioperl ==1.6.924', 'perl-xml-simple', 'prodigal >=2.6', 'tbl2asn >=25.3', 'libidn'])])), ('test', CommentedMap([('commands', ['prokka --listdb', 'prokka --version', 'blastp -help', 'barrnap --help 2>&1 | grep Options', 'prodigal -h'])])), ('about', CommentedMap([('home', 'http://www.vicbioinformatics.com/software.prokka.shtml'), ('license', 'GPLv2'), ('summary', 'A tool for the rapid annotation of prokaryotic genomes')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-edger/3.12.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-edger'), ('version', '3.12.1')])), ('source', CommentedMap([('fn', 'c9928062b0d179009dcc7173dc78b90abae7e3d0.zip'), ('url', 'https://github.com/Bioconductor-mirror/edgeR/archive/c9928062b0d179009dcc7173dc78b90abae7e3d0.zip'), ('md5', 'a76554192ad964763f2c1040e04b7347')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base']), ('run', ['bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'edgeR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/edgeR.html'), ('license', 'GPL (>=2)'), ('summary', 'Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-edger/3.12.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-edger'), ('version', '3.12.1')])), ('source', CommentedMap([('fn', 'c9928062b0d179009dcc7173dc78b90abae7e3d0.zip'), ('url', 'https://github.com/Bioconductor-mirror/edgeR/archive/c9928062b0d179009dcc7173dc78b90abae7e3d0.zip'), ('md5', 'a76554192ad964763f2c1040e04b7347')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base']), ('run', ['bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'edgeR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/edgeR.html'), ('license', 'GPL (>=2)'), ('summary', 'Differential expression analysis of RNA-seq expression profiles with biological replication. Implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi-likelihood tests. As well as RNA-seq, it be applied to differential signal analysis of other types of genomic data that produce counts, including ChIP-seq, SAGE and CAGE.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/beagle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'beagle'), ('version', '4.1_21Jan17.6cc.jar')])), ('source', CommentedMap([('fn', 'beagle.21Jan17.6cc.jar'), ('url', 'https://faculty.washington.edu/browning/beagle/beagle.21Jan17.6cc.jar'), ('md5', 'd23fbcef8707b9added46937149c3e76')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', [\"beagle | grep -q 'Browning'\"])])), ('about', CommentedMap([('home', 'http://faculty.washington.edu/browning/beagle/beagle.html'), ('license', 'GPLv3'), ('summary', 'Beagle is a software package that performs genotype calling, genotype phasing, imputation of ungenotyped markers, and identity-by-descent segment detection.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/beagle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'beagle'), ('version', '4.1_21Jan17.6cc.jar')])), ('source', CommentedMap([('fn', 'beagle.21Jan17.6cc.jar'), ('url', 'https://faculty.washington.edu/browning/beagle/beagle.21Jan17.6cc.jar'), ('md5', 'd23fbcef8707b9added46937149c3e76')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', [\"beagle | grep -q 'Browning'\"])])), ('about', CommentedMap([('home', 'http://faculty.washington.edu/browning/beagle/beagle.html'), ('license', 'GPLv3'), ('summary', 'Beagle is a software package that performs genotype calling, genotype phasing, imputation of ungenotyped markers, and identity-by-descent segment detection.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metahdep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metahdep'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'metahdep_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/metahdep_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/metahdep_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/metahdep/metahdep_1.34.0_src_all.tar.gz']), ('md5', '769b7067870b6c965bc8b49aa60bbaea')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metahdep\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/metahdep.html'), ('license', 'GPL-3'), ('summary', 'Tools for meta-analysis in the presence of hierarchical (and/or sampling) dependence, including with gene expression studies')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metahdep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metahdep'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'metahdep_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/metahdep_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/metahdep_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/metahdep/metahdep_1.34.0_src_all.tar.gz']), ('md5', '769b7067870b6c965bc8b49aa60bbaea')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metahdep\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/metahdep.html'), ('license', 'GPL-3'), ('summary', 'Tools for meta-analysis in the presence of hierarchical (and/or sampling) dependence, including with gene expression studies')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dada2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dada2'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'dada2-<{ version }}.tar.gz'), ('url', 'https://github.com/benjjneb/dada2/archive/v<{ version }}.tar.gz'), ('sha256', '5a590edd581a37a6b6b6c5f560b3456c0b81d7c319eaf10a76dcb8030277b5f7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.32.1', 'bioconductor-shortread >=1.24.0', 'r-base', 'r-data.table >=1.9.4', 'r-ggplot2 >=1.0', 'r-rcpp >=0.11.2', 'r-rcppparallel >=4.3.0', 'r-reshape2 >=1.4.1', 'gcc', 'llvm']), ('run', ['bioconductor-biostrings >=2.32.1', 'bioconductor-shortread >=1.24.0', 'r-base', 'r-data.table >=1.9.4', 'r-ggplot2 >=1.0', 'r-rcpp >=0.11.2', 'r-rcppparallel >=4.3.0', 'r-reshape2 >=1.4.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'dada2\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/dada2.html'), ('license', 'LGPL-3'), ('summary', 'The dada2 package provides \"OTU picking\" functionality, but instead of picking OTUs the DADA2 algorithm exactly infers samples sequences. The dada2 pipeline starts from demultiplexed fastq files, and outputs inferred sample sequences and associated abundances after removing substitution and chimeric errors. Taxonomic classification is also available via a native implementation of the RDP classifier method.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dada2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dada2'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'dada2-<{ version }}.tar.gz'), ('url', 'https://github.com/benjjneb/dada2/archive/v<{ version }}.tar.gz'), ('sha256', '5a590edd581a37a6b6b6c5f560b3456c0b81d7c319eaf10a76dcb8030277b5f7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.32.1', 'bioconductor-shortread >=1.24.0', 'r-base', 'r-data.table >=1.9.4', 'r-ggplot2 >=1.0', 'r-rcpp >=0.11.2', 'r-rcppparallel >=4.3.0', 'r-reshape2 >=1.4.1', 'gcc', 'llvm']), ('run', ['bioconductor-biostrings >=2.32.1', 'bioconductor-shortread >=1.24.0', 'r-base', 'r-data.table >=1.9.4', 'r-ggplot2 >=1.0', 'r-rcpp >=0.11.2', 'r-rcppparallel >=4.3.0', 'r-reshape2 >=1.4.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'dada2\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/dada2.html'), ('license', 'LGPL-3'), ('summary', 'The dada2 package provides \"OTU picking\" functionality, but instead of picking OTUs the DADA2 algorithm exactly infers samples sequences. The dada2 pipeline starts from demultiplexed fastq files, and outputs inferred sample sequences and associated abundances after removing substitution and chimeric errors. Taxonomic classification is also available via a native implementation of the RDP classifier method.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shapeit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'shapeit'), ('version', '2.r837')])), ('source', CommentedMap([('fn', 'shapeit.v2.r837.GLIBCv2.12.Linux.static.tgz'), ('url', 'https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.v2.r837.GLIBCv2.12.Linux.static.tgz'), ('md5', 'd778d2e6963a5fdbf4edb1f2dafa9f63')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'boost']), ('run', ['libgcc', 'boost'])])), ('about', CommentedMap([('home', 'https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html'), ('license', 'Free for Academic Use'), ('summary', 'SHAPEIT is a fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data.')])), ('test', CommentedMap([('commands', ['shapeit --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shapeit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'shapeit'), ('version', '2.r837')])), ('source', CommentedMap([('fn', 'shapeit.v2.r837.GLIBCv2.12.Linux.static.tgz'), ('url', 'https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.v2.r837.GLIBCv2.12.Linux.static.tgz'), ('md5', 'd778d2e6963a5fdbf4edb1f2dafa9f63')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'boost']), ('run', ['libgcc', 'boost'])])), ('about', CommentedMap([('home', 'https://mathgen.stats.ox.ac.uk/genetics_software/shapeit/shapeit.html'), ('license', 'Free for Academic Use'), ('summary', 'SHAPEIT is a fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data.')])), ('test', CommentedMap([('commands', ['shapeit --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-squadd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-squadd'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'SQUADD_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SQUADD_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/SQUADD_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/SQUADD/SQUADD_1.26.0_src_all.tar.gz']), ('md5', 'e031d825157b32d711e31564181dfaba')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SQUADD\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SQUADD.html'), ('license', 'GPL (>=2)'), ('summary', 'This package SQUADD is a SQUAD add-on. It permits to generate SQUAD simulation matrix, prediction Heat-Map and Correlation Circle from PCA analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-squadd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-squadd'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'SQUADD_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SQUADD_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/SQUADD_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/SQUADD/SQUADD_1.26.0_src_all.tar.gz']), ('md5', 'e031d825157b32d711e31564181dfaba')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SQUADD\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SQUADD.html'), ('license', 'GPL (>=2)'), ('summary', 'This package SQUADD is a SQUAD add-on. It permits to generate SQUAD simulation matrix, prediction Heat-Map and Correlation Circle from PCA analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nucler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nucler'), ('version', '2.8.0')])), ('source', CommentedMap([('fn', 'nucleR_2.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nucleR_2.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/nucleR_2.8.0.tar.gz', 'https://depot.galaxyproject.org/software/nucleR/nucleR_2.8.0_src_all.tar.gz']), ('md5', 'c23125444464c780fce47cf09391a6bd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-s4vectors >=0.9.39', 'bioconductor-shortread', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-s4vectors >=0.9.39', 'bioconductor-shortread', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nucleR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nucleR.html'), ('license', 'LGPL (>= 3)'), ('summary', 'Nucleosome positioning for Tiling Arrays and NGS experiments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nucler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nucler'), ('version', '2.8.0')])), ('source', CommentedMap([('fn', 'nucleR_2.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nucleR_2.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/nucleR_2.8.0.tar.gz', 'https://depot.galaxyproject.org/software/nucleR/nucleR_2.8.0_src_all.tar.gz']), ('md5', 'c23125444464c780fce47cf09391a6bd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-s4vectors >=0.9.39', 'bioconductor-shortread', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-s4vectors >=0.9.39', 'bioconductor-shortread', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nucleR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nucleR.html'), ('license', 'LGPL (>= 3)'), ('summary', 'Nucleosome positioning for Tiling Arrays and NGS experiments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ascat/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/Crick-CancerGenomics/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', 'True [win]'), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base >=2.13.0', 'r-rcolorbrewer', 'r-r.devices', 'r-r.utils']), ('run', ['r-base >=2.13.0', 'r-rcolorbrewer', 'r-r.devices', 'r-r.utils'])])), ('test', CommentedMap([('commands', ['$R -e \"library(ASCAT)\"'])])), ('about', CommentedMap([('home', 'https://www.crick.ac.uk/research/a-z-researchers/researchers-v-y/peter-van-loo/software/'), ('license', 'GPL v3'), ('summary', 'ASCAT is a method to derive copy number profiles of tumour cells,\\naccounting for normal cell admixture and tumour aneuploidy (Figure 1).\\nASCAT infers tumour purity (the fraction of tumour cells) and ploidy (the\\namount of DNA per tumour cell, expressed as multiples of haploid genomes)\\nfrom SNP array or massively parallel sequencing data, and calculates\\nwhole-genome allele-specific copy number profiles (the number of copies of\\nboth parental alleles for all SNP loci across the genome).\\n')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ascat/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/Crick-CancerGenomics/<{ name }}/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0), ('skip', 'True [win]'), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base >=2.13.0', 'r-rcolorbrewer', 'r-r.devices', 'r-r.utils']), ('run', ['r-base >=2.13.0', 'r-rcolorbrewer', 'r-r.devices', 'r-r.utils'])])), ('test', CommentedMap([('commands', ['$R -e \"library(ASCAT)\"'])])), ('about', CommentedMap([('home', 'https://www.crick.ac.uk/research/a-z-researchers/researchers-v-y/peter-van-loo/software/'), ('license', 'GPL v3'), ('summary', 'ASCAT is a method to derive copy number profiles of tumour cells,\\naccounting for normal cell admixture and tumour aneuploidy (Figure 1).\\nASCAT infers tumour purity (the fraction of tumour cells) and ploidy (the\\namount of DNA per tumour cell, expressed as multiples of haploid genomes)\\nfrom SNP array or massively parallel sequencing data, and calculates\\nwhole-genome allele-specific copy number profiles (the number of copies of\\nboth parental alleles for all SNP loci across the genome).\\n')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gmap/2017.05.08/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gmap'), ('version', '2017.05.08')])), ('source', CommentedMap([('fn', 'gmap-gsnap-2017-05-08.tar.gz'), ('url', 'http://research-pub.gene.com/gmap/src/gmap-gsnap-2017-05-08.tar.gz'), ('sha1', '592797008a0251175e95ce0653bdfa764ee99617')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'zlib', 'bzip2']), ('run', ['libgcc', 'perl', 'zlib', 'bzip2'])])), ('about', CommentedMap([('home', 'http://research-pub.gene.com/gmap/'), ('license', 'Non-commercial'), ('summary', 'Genomic mapping and alignment program for mRNA and EST sequences')])), ('build', CommentedMap([('number', 0), ('binary_has_prefix_files', ['bin/atoiindex', 'bin/cmetindex', 'bin/get-genome', 'bin/gmap.sse42', 'bin/gmap.nosimd', 'bin/gmapl.sse42', 'bin/gmapl.nosimd', 'bin/gsnap.sse42', 'bin/gsnap.nosimd', 'bin/gsnapl.sse42', 'bin/gsnapl.nosimd', 'bin/sam_sort', 'bin/snpindex', 'bin/uniqscan', 'bin/uniqscanl']), ('has_prefix_files', ['bin/gff3_introns', 'bin/gff3_splicesites', 'bin/gmap_build', 'bin/gmap_uncompress', 'bin/gtf_introns', 'bin/gtf_splicesites'])])), ('test', CommentedMap([('commands', ['gmap --version', 'gsnap --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gmap/2017.05.08/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gmap'), ('version', '2017.05.08')])), ('source', CommentedMap([('fn', 'gmap-gsnap-2017-05-08.tar.gz'), ('url', 'http://research-pub.gene.com/gmap/src/gmap-gsnap-2017-05-08.tar.gz'), ('sha1', '592797008a0251175e95ce0653bdfa764ee99617')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'perl', 'zlib', 'bzip2']), ('run', ['libgcc', 'perl', 'zlib', 'bzip2'])])), ('about', CommentedMap([('home', 'http://research-pub.gene.com/gmap/'), ('license', 'Non-commercial'), ('summary', 'Genomic mapping and alignment program for mRNA and EST sequences')])), ('build', CommentedMap([('number', 0), ('binary_has_prefix_files', ['bin/atoiindex', 'bin/cmetindex', 'bin/get-genome', 'bin/gmap.sse42', 'bin/gmap.nosimd', 'bin/gmapl.sse42', 'bin/gmapl.nosimd', 'bin/gsnap.sse42', 'bin/gsnap.nosimd', 'bin/gsnapl.sse42', 'bin/gsnapl.nosimd', 'bin/sam_sort', 'bin/snpindex', 'bin/uniqscan', 'bin/uniqscanl']), ('has_prefix_files', ['bin/gff3_introns', 'bin/gff3_splicesites', 'bin/gmap_build', 'bin/gmap_uncompress', 'bin/gtf_introns', 'bin/gtf_splicesites'])])), ('test', CommentedMap([('commands', ['gmap --version', 'gsnap --version'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pysamstats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pysamstats'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'v1.0.1.tar.gz'), ('url', 'https://github.com/alimanfoo/pysamstats/archive/v1.0.1.tar.gz'), ('md5', 'db8a5ad0db0027f55fd954a5510a797a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'cython', 'pysam >=0.11', 'zlib', 'gcc', 'numpy', 'pytables', 'nose']), ('run', ['python', 'pysam >=0.11', 'zlib', 'libgcc', 'numpy', 'pytables'])])), ('test', CommentedMap([('imports', ['pysamstats']), ('commands', ['pysamstats --help'])])), ('about', CommentedMap([('home', 'https://github.com/alimanfoo/pysamstats'), ('license', 'MIT'), ('summary', 'A Python utility for calculating statistics against genome position based on sequence alignments from a SAM, BAM or CRAM file.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pysamstats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pysamstats'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'v1.0.1.tar.gz'), ('url', 'https://github.com/alimanfoo/pysamstats/archive/v1.0.1.tar.gz'), ('md5', 'db8a5ad0db0027f55fd954a5510a797a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'cython', 'pysam >=0.11', 'zlib', 'gcc', 'numpy', 'pytables', 'nose']), ('run', ['python', 'pysam >=0.11', 'zlib', 'libgcc', 'numpy', 'pytables'])])), ('test', CommentedMap([('imports', ['pysamstats']), ('commands', ['pysamstats --help'])])), ('about', CommentedMap([('home', 'https://github.com/alimanfoo/pysamstats'), ('license', 'MIT'), ('summary', 'A Python utility for calculating statistics against genome position based on sequence alignments from a SAM, BAM or CRAM file.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqtools'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'seqTools_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/seqTools_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/seqTools_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/seqTools/seqTools_1.10.0_src_all.tar.gz']), ('md5', 'ac3d1b9efc95565f58068ba5825e2ed4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'seqTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/seqTools.html'), ('license', 'Artistic-2.0'), ('summary', 'Analyze read length, phred scores and alphabet frequency and DNA k-mers on uncompressed and compressed fastq files.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqtools'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'seqTools_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/seqTools_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/seqTools_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/seqTools/seqTools_1.10.0_src_all.tar.gz']), ('md5', 'ac3d1b9efc95565f58068ba5825e2ed4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'seqTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/seqTools.html'), ('license', 'Artistic-2.0'), ('summary', 'Analyze read length, phred scores and alphabet frequency and DNA k-mers on uncompressed and compressed fastq files.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-mlgt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-mlgt'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'mlgt_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/mlgt_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/mlgt/mlgt_<{ version }}.tar.gz']), ('sha256', 'a93d6a2aafec41946773484a1bc5087cd56b967e4f8de3e68c69e4b7abc16ddb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-seqinr']), ('run', ['r-base', 'r-seqinr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mlgt\\')\"'])])), ('about', CommentedMap([('home', 'http://personalpages.manchester.ac.uk/staff/David.Gerrard/'), ('license', 'GPL (>= 2)'), ('summary', 'Processing and analysis of high throughput (Roche 454) sequences generated from multiple loci and multiple biological samples. Sequences are assigned to their locus and sample of origin, aligned and trimmed. Where possible, genotypes are called and variants mapped to known alleles.'), ('license_family', 'GPL3')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-mlgt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-mlgt'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'mlgt_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/mlgt_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/mlgt/mlgt_<{ version }}.tar.gz']), ('sha256', 'a93d6a2aafec41946773484a1bc5087cd56b967e4f8de3e68c69e4b7abc16ddb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-seqinr']), ('run', ['r-base', 'r-seqinr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mlgt\\')\"'])])), ('about', CommentedMap([('home', 'http://personalpages.manchester.ac.uk/staff/David.Gerrard/'), ('license', 'GPL (>= 2)'), ('summary', 'Processing and analysis of high throughput (Roche 454) sequences generated from multiple loci and multiple biological samples. Sequences are assigned to their locus and sample of origin, aligned and trimmed. Where possible, genotypes are called and variants mapped to known alleles.'), ('license_family', 'GPL3')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/arem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'arem'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'AREM-1.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/bc/0d/6db36d917d821d150f37c68839d8911e52b78c867b27372da5be41d4cc4f/AREM-1.0.1.tar.gz'), ('md5', 'fab4d78f1fd12c691d4bd62140b927ea')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['AREM', 'AREM.IO'])])), ('about', CommentedMap([('home', 'http://cbcl.ics.uci.edu/AREM'), ('license', 'Artistic License'), ('summary', 'Aligning Reads by Expectation-Maximization.\\\\nBased on MACS (Model Based Analysis for ChIP-Seq data)'), ('license_family', 'OTHER')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/arem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'arem'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'AREM-1.0.1.tar.gz'), ('url', 'https://pypi.python.org/packages/bc/0d/6db36d917d821d150f37c68839d8911e52b78c867b27372da5be41d4cc4f/AREM-1.0.1.tar.gz'), ('md5', 'fab4d78f1fd12c691d4bd62140b927ea')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['AREM', 'AREM.IO'])])), ('about', CommentedMap([('home', 'http://cbcl.ics.uci.edu/AREM'), ('license', 'Artistic License'), ('summary', 'Aligning Reads by Expectation-Maximization.\\\\nBased on MACS (Model Based Analysis for ChIP-Seq data)'), ('license_family', 'OTHER')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rdp-readseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/rdpstaff/ReadSeq/archive/<{ version }}.tar.gz'), ('md5', 'fb969e436c9d7a2d5e87bc7363c30bbf')])), ('about', CommentedMap([('home', 'https://github.com/rdpstaff/ReadSeq'), ('license', 'GPL'), ('summary', 'Java based common sequence file format reader and sequence file manipulation.')])), ('requirements', CommentedMap([('build', ['openjdk', 'ant']), ('run', ['openjdk'])])), ('test', CommentedMap([('commands', [\"ReadSeq 2>&1 | grep 'ReadSeqMain'\"])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rdp-readseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/rdpstaff/ReadSeq/archive/<{ version }}.tar.gz'), ('md5', 'fb969e436c9d7a2d5e87bc7363c30bbf')])), ('about', CommentedMap([('home', 'https://github.com/rdpstaff/ReadSeq'), ('license', 'GPL'), ('summary', 'Java based common sequence file format reader and sequence file manipulation.')])), ('requirements', CommentedMap([('build', ['openjdk', 'ant']), ('run', ['openjdk'])])), ('test', CommentedMap([('commands', [\"ReadSeq 2>&1 | grep 'ReadSeqMain'\"])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-hilbertvis/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-hilbertvis'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'HilbertVis_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/HilbertVis_1.32.0.tar.gz'), ('md5', '59c578593bf698fecfbd2fad4db832ba')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-lattice']), ('run', ['r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'HilbertVis\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/HilbertVis.html'), ('license', 'GPL (>= 3)'), ('summary', 'Functions to visualize long vectors of integer data by means of Hilbert curves')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-hilbertvis/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-hilbertvis'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'HilbertVis_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/HilbertVis_1.32.0.tar.gz'), ('md5', '59c578593bf698fecfbd2fad4db832ba')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-lattice']), ('run', ['r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'HilbertVis\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/HilbertVis.html'), ('license', 'GPL (>= 3)'), ('summary', 'Functions to visualize long vectors of integer data by means of Hilbert curves')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eagle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eagle'), ('version', '0.9.0')])), ('source', CommentedMap([('fn', 'eagle-0.9.tar.gz'), ('url', 'https://bitbucket.org/christopherschroeder/eagle/get/v0.9.tar.gz'), ('md5', '5f1abfc540f234551760043a53b26268')])), ('build', CommentedMap([('skip', True), ('entry_points', ['eagle=run:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'flask', 'numpy', 'scipy', 'h5py', 'pyliftover']), ('run', ['python', 'flask', 'numpy', 'scipy', 'h5py', 'pyliftover'])])), ('test', CommentedMap([('commands', ['eagle --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/christopherschroeder/eagle'), ('license', 'MIT License'), ('summary', 'Eagle is a webtool for genome variants and snp analysis'), ('license_family', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eagle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eagle'), ('version', '0.9.0')])), ('source', CommentedMap([('fn', 'eagle-0.9.tar.gz'), ('url', 'https://bitbucket.org/christopherschroeder/eagle/get/v0.9.tar.gz'), ('md5', '5f1abfc540f234551760043a53b26268')])), ('build', CommentedMap([('skip', True), ('entry_points', ['eagle=run:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'flask', 'numpy', 'scipy', 'h5py', 'pyliftover']), ('run', ['python', 'flask', 'numpy', 'scipy', 'h5py', 'pyliftover'])])), ('test', CommentedMap([('commands', ['eagle --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/christopherschroeder/eagle'), ('license', 'MIT License'), ('summary', 'Eagle is a webtool for genome variants and snp analysis'), ('license_family', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'metaseq'), ('version', '0.5.6')])), ('source', CommentedMap([('fn', 'metaseq-0.5.6.tar.gz'), ('url', 'https://pypi.python.org/packages/source/m/metaseq/metaseq-0.5.6.tar.gz'), ('md5', 'f29575db23019a5cd1003bfccb599847')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scipy >=0.10.1', 'biopython', 'scikit-learn', 'bx-python >=0.7.1', 'cython', 'numpy >=1.8.0', 'matplotlib >=1.3.1', 'pandas >=0.13.1', 'pysam >=0.7', 'gffutils >=0.8.2', 'pyyaml >=3.10', 'pybedtools >=0.6.6', 'fisher']), ('run', ['python', 'scipy >=0.10.1', 'biopython', 'scikit-learn', 'bx-python >=0.7.1', 'cython', 'numpy >=1.8.0', 'matplotlib >=1.3.1', 'pandas >=0.13.1', 'pysam >=0.7', 'gffutils >=0.8.2', 'pyyaml >=3.10', 'pybedtools >=0.6.6', 'fisher'])])), ('test', CommentedMap([('imports', ['metaseq'])])), ('about', CommentedMap([('home', 'http://github.com/daler/metaseq'), ('license', 'MIT License'), ('summary', 'Integrative analysis of high-thoughput sequencing data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'metaseq'), ('version', '0.5.6')])), ('source', CommentedMap([('fn', 'metaseq-0.5.6.tar.gz'), ('url', 'https://pypi.python.org/packages/source/m/metaseq/metaseq-0.5.6.tar.gz'), ('md5', 'f29575db23019a5cd1003bfccb599847')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scipy >=0.10.1', 'biopython', 'scikit-learn', 'bx-python >=0.7.1', 'cython', 'numpy >=1.8.0', 'matplotlib >=1.3.1', 'pandas >=0.13.1', 'pysam >=0.7', 'gffutils >=0.8.2', 'pyyaml >=3.10', 'pybedtools >=0.6.6', 'fisher']), ('run', ['python', 'scipy >=0.10.1', 'biopython', 'scikit-learn', 'bx-python >=0.7.1', 'cython', 'numpy >=1.8.0', 'matplotlib >=1.3.1', 'pandas >=0.13.1', 'pysam >=0.7', 'gffutils >=0.8.2', 'pyyaml >=3.10', 'pybedtools >=0.6.6', 'fisher'])])), ('test', CommentedMap([('imports', ['metaseq'])])), ('about', CommentedMap([('home', 'http://github.com/daler/metaseq'), ('license', 'MIT License'), ('summary', 'Integrative analysis of high-thoughput sequencing data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trimal/1.4.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'trimal'), ('version', '1.4.1')])), ('source', CommentedMap([('fn', 'v1.4.1.tar.gz'), ('url', 'https://github.com/scapella/trimal/archive/v1.4.1.tar.gz'), ('md5', 'cb193397635da45be350e9ac96dafe45')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['readal --version | grep readAl > /dev/null', 'statal --version | grep statAl > /dev/null', 'trimal --version | grep trimAl > /dev/null'])])), ('about', CommentedMap([('home', 'http://trimal.cgenomics.org'), ('summary', 'A tool for the automated removal of spurious sequences or poorly aligned regions from a multiple sequence alignment'), ('license', 'GNU General Public License v3 or later (GPLv3+)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trimal/1.4.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'trimal'), ('version', '1.4.1')])), ('source', CommentedMap([('fn', 'v1.4.1.tar.gz'), ('url', 'https://github.com/scapella/trimal/archive/v1.4.1.tar.gz'), ('md5', 'cb193397635da45be350e9ac96dafe45')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['readal --version | grep readAl > /dev/null', 'statal --version | grep statAl > /dev/null', 'trimal --version | grep trimAl > /dev/null'])])), ('about', CommentedMap([('home', 'http://trimal.cgenomics.org'), ('summary', 'A tool for the automated removal of spurious sequences or poorly aligned regions from a multiple sequence alignment'), ('license', 'GNU General Public License v3 or later (GPLv3+)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cexor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cexor'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'CexoR_1.14.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/CexoR_1.14.0.tar.gz'), ('sha256', 'a45fed681bcbe9f6845b6ccd92b312fda8ec221f771861fa6127adb7e6546373')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomation', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-idr', 'r-rcolorbrewer']), ('run', ['bioconductor-genomation', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-idr', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CexoR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CexoR.html'), ('license', 'Artistic-2.0 | GPL-2 + file LICENSE'), ('summary', \"Strand specific peak-pair calling in ChIP-exo replicates. The cumulative Skellam distribution function (package 'skellam') is used to detect significant normalised count differences of opposed sign at each DNA strand (peak-pairs). Irreproducible discovery rate for overlapping peak-pairs across biological replicates is estimated using the package 'idr'.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cexor/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cexor'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'CexoR_1.14.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/CexoR_1.14.0.tar.gz'), ('sha256', 'a45fed681bcbe9f6845b6ccd92b312fda8ec221f771861fa6127adb7e6546373')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomation', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-idr', 'r-rcolorbrewer']), ('run', ['bioconductor-genomation', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-idr', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CexoR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CexoR.html'), ('license', 'Artistic-2.0 | GPL-2 + file LICENSE'), ('summary', \"Strand specific peak-pair calling in ChIP-exo replicates. The cumulative Skellam distribution function (package 'skellam') is used to detect significant normalised count differences of opposed sign at each DNA strand (peak-pairs). Irreproducible discovery rate for overlapping peak-pairs across biological replicates is estimated using the package 'idr'.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/monovar/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://bitbucket.org/dlaehnemann/monovar/get/<{ version }}.tar.gz'), ('md5', 'a5b49a53aa660ce084b1ef0f3d5bbb32')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scipy', 'pysam']), ('run', ['python', 'numpy', 'scipy', 'pysam'])])), ('test', CommentedMap([('commands', ['monovar 2>&1 | grep \" Reference genome file not provided. \"'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/hamimzafar/monovar'), ('license', 'MIT'), ('summary', 'single cell joint genotyper')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/monovar/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://bitbucket.org/dlaehnemann/monovar/get/<{ version }}.tar.gz'), ('md5', 'a5b49a53aa660ce084b1ef0f3d5bbb32')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scipy', 'pysam']), ('run', ['python', 'numpy', 'scipy', 'pysam'])])), ('test', CommentedMap([('commands', ['monovar 2>&1 | grep \" Reference genome file not provided. \"'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/hamimzafar/monovar'), ('license', 'MIT'), ('summary', 'single cell joint genotyper')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bowtie2/2.2.5/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml'), ('license', 'GPLv3'), ('summary', 'Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.   http://bowtie-bio.sourceforge.net/bowtie2/index.shtml ')])), ('package', CommentedMap([('name', 'bowtie2'), ('version', '2.2.5')])), ('source', CommentedMap([('fn', 'bowtie2-2.2.5-macos-x86_64.zip'), ('url', 'http://downloads.sourceforge.net/project/bowtie-bio/bowtie2/2.2.5/bowtie2-2.2.5-macos-x86_64.zip'), ('sha256', '0f142477738accc73276f3e47877d54a64b6f7ac7911d0609b662adcd15374a0')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'python']), ('run', ['python', 'perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', ['bowtie2 --version', 'bowtie2-build --help', 'bowtie2-inspect-l --help', 'bowtie2-align-s --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bowtie2/2.2.5/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml'), ('license', 'GPLv3'), ('summary', 'Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes.   http://bowtie-bio.sourceforge.net/bowtie2/index.shtml ')])), ('package', CommentedMap([('name', 'bowtie2'), ('version', '2.2.5')])), ('source', CommentedMap([('fn', 'bowtie2-2.2.5-macos-x86_64.zip'), ('url', 'http://downloads.sourceforge.net/project/bowtie-bio/bowtie2/2.2.5/bowtie2-2.2.5-macos-x86_64.zip'), ('sha256', '0f142477738accc73276f3e47877d54a64b6f7ac7911d0609b662adcd15374a0')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'python']), ('run', ['python', 'perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', ['bowtie2 --version', 'bowtie2-build --help', 'bowtie2-inspect-l --help', 'bowtie2-align-s --help'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mbttest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mbttest'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'MBttest_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MBttest_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/MBttest_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/MBttest/MBttest_1.4.0_src_all.tar.gz']), ('md5', '984d689b8f577d0920dca8985ee4d800')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gplots', 'r-gtools']), ('run', ['r-base', 'r-gplots', 'r-gtools'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MBttest\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MBttest.html'), ('license', 'GPL-3'), ('summary', 'MBttest method was developed from beta t-test method of Baggerly et al(2003). Compared to baySeq (Hard castle and Kelly 2010), DESeq (Anders and Huber 2010) and exact test (Robinson and Smyth 2007, 2008) and the GLM of McCarthy et al(2012), MBttest is of high work efficiency,that is, it has high power, high conservativeness of FDR estimation and high stability. MBttest is suit- able to transcriptomic data, tag data, SAGE data (count data) from small samples or a few replicate libraries. It can be used to identify genes, mRNA isoforms or tags differentially expressed between two conditions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mbttest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mbttest'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'MBttest_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MBttest_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/MBttest_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/MBttest/MBttest_1.4.0_src_all.tar.gz']), ('md5', '984d689b8f577d0920dca8985ee4d800')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gplots', 'r-gtools']), ('run', ['r-base', 'r-gplots', 'r-gtools'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MBttest\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MBttest.html'), ('license', 'GPL-3'), ('summary', 'MBttest method was developed from beta t-test method of Baggerly et al(2003). Compared to baySeq (Hard castle and Kelly 2010), DESeq (Anders and Huber 2010) and exact test (Robinson and Smyth 2007, 2008) and the GLM of McCarthy et al(2012), MBttest is of high work efficiency,that is, it has high power, high conservativeness of FDR estimation and high stability. MBttest is suit- able to transcriptomic data, tag data, SAGE data (count data) from small samples or a few replicate libraries. It can be used to identify genes, mRNA isoforms or tags differentially expressed between two conditions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-summarizedexperiment/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-summarizedexperiment'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'SummarizedExperiment_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/SummarizedExperiment_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/SummarizedExperiment_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/SummarizedExperiment/SummarizedExperiment_1.4.0_src_all.tar.gz']), ('md5', '35159a339981efa22e96db639dadc850')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.15', 'bioconductor-iranges >=2.7.2', 'bioconductor-s4vectors >=0.11.7', 'r-base', 'r-matrix']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.15', 'bioconductor-iranges >=2.7.2', 'bioconductor-s4vectors >=0.11.7', 'r-base', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SummarizedExperiment\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SummarizedExperiment.html'), ('license', 'Artistic-2.0'), ('summary', 'The SummarizedExperiment container contains one or more assays, each represented by a matrix-like object of numeric or other mode. The rows typically represent genomic ranges of interest and the columns represent samples.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-summarizedexperiment/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-summarizedexperiment'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'SummarizedExperiment_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/SummarizedExperiment_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/SummarizedExperiment_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/SummarizedExperiment/SummarizedExperiment_1.4.0_src_all.tar.gz']), ('md5', '35159a339981efa22e96db639dadc850')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.15', 'bioconductor-iranges >=2.7.2', 'bioconductor-s4vectors >=0.11.7', 'r-base', 'r-matrix']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.15', 'bioconductor-iranges >=2.7.2', 'bioconductor-s4vectors >=0.11.7', 'r-base', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SummarizedExperiment\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SummarizedExperiment.html'), ('license', 'Artistic-2.0'), ('summary', 'The SummarizedExperiment container contains one or more assays, each represented by a matrix-like object of numeric or other mode. The rows typically represent genomic ranges of interest and the columns represent samples.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-targetscore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-targetscore'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'TargetScore_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TargetScore_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/TargetScore_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/TargetScore/TargetScore_1.14.0_src_all.tar.gz']), ('md5', '63f01d25aec571ebaaa4cce97b0ff42c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-matrix', 'r-pracma']), ('run', ['r-base', 'r-matrix', 'r-pracma'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TargetScore\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TargetScore.html'), ('license', 'GPL-2'), ('summary', 'Infer the posterior distributions of microRNA targets by probabilistically modelling the likelihood microRNA-overexpression fold-changes and sequence-based scores. Variaitonal Bayesian Gaussian mixture model (VB-GMM) is applied to log fold-changes and sequence scores to obtain the posteriors of latent variable being the miRNA targets. The final targetScore is computed as the sigmoid-transformed fold-change weighted by the averaged posteriors of target components over all of the features.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-targetscore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-targetscore'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'TargetScore_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TargetScore_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/TargetScore_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/TargetScore/TargetScore_1.14.0_src_all.tar.gz']), ('md5', '63f01d25aec571ebaaa4cce97b0ff42c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-matrix', 'r-pracma']), ('run', ['r-base', 'r-matrix', 'r-pracma'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TargetScore\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TargetScore.html'), ('license', 'GPL-2'), ('summary', 'Infer the posterior distributions of microRNA targets by probabilistically modelling the likelihood microRNA-overexpression fold-changes and sequence-based scores. Variaitonal Bayesian Gaussian mixture model (VB-GMM) is applied to log fold-changes and sequence scores to obtain the posteriors of latent variable being the miRNA targets. The final targetScore is computed as the sigmoid-transformed fold-change weighted by the averaged posteriors of target components over all of the features.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/chromhmm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'chromhmm'), ('version', 1.12)])), ('source', CommentedMap([('fn', 'ChromHMM.zip'), ('url', 'http://compbio.mit.edu/ChromHMM/ChromHMM.zip'), ('md5', '0689d01a9a80a57cd3cb3fc49efa443d')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk', 'unzip'])])), ('test', CommentedMap([('commands', [\"ChromHMM.sh -Xmx512M BinarizeBam | grep -q 'usage BinarizeBam'\"])])), ('about', CommentedMap([('home', 'http://compbio.mit.edu/ChromHMM/'), ('license', 'GPLv3'), ('summary', 'ChromHMM is software for learning and characterizing chromatin states. ChromHMM can integrate multiple chromatin datasets such as ChIP-seq data of various histone modifications to discover de novo the major re-occuring combinatorial and spatial patterns of marks.')])), ('extra', CommentedMap([('notes', 'ChromHMM comes with about 36MB of example data which is not included in the recipe. This recipe installs a script, \"download_chromhmm_data.sh\", which downloads the data in the proper location, and which can be run after ChromHMM has been installed.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/chromhmm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'chromhmm'), ('version', 1.12)])), ('source', CommentedMap([('fn', 'ChromHMM.zip'), ('url', 'http://compbio.mit.edu/ChromHMM/ChromHMM.zip'), ('md5', '0689d01a9a80a57cd3cb3fc49efa443d')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk', 'unzip'])])), ('test', CommentedMap([('commands', [\"ChromHMM.sh -Xmx512M BinarizeBam | grep -q 'usage BinarizeBam'\"])])), ('about', CommentedMap([('home', 'http://compbio.mit.edu/ChromHMM/'), ('license', 'GPLv3'), ('summary', 'ChromHMM is software for learning and characterizing chromatin states. ChromHMM can integrate multiple chromatin datasets such as ChIP-seq data of various histone modifications to discover de novo the major re-occuring combinatorial and spatial patterns of marks.')])), ('extra', CommentedMap([('notes', 'ChromHMM comes with about 36MB of example data which is not included in the recipe. This recipe installs a script, \"download_chromhmm_data.sh\", which downloads the data in the proper location, and which can be run after ChromHMM has been installed.')]))]))\n",
      "EXTRA has been found: chromhmm\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-clusterprofiler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-clusterprofiler'), ('version', '3.0.5')])), ('source', CommentedMap([('fn', 'clusterProfiler_3.0.5.tar.gz'), ('url', 'https://www.bioconductor.org/packages/3.3/bioc/src/contrib/clusterProfiler_3.0.5.tar.gz'), ('md5', '501b5ff766c9458faffcd3e483d527c6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-dose', 'bioconductor-go.db', 'bioconductor-gosemsim', 'bioconductor-keggrest', 'bioconductor-gseabase', 'bioconductor-qvalue', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-magrittr', 'r-plyr', 'r-tidyr']), ('run', ['bioconductor-annotationdbi', 'bioconductor-dose', 'bioconductor-go.db', 'bioconductor-gosemsim', 'bioconductor-keggrest', 'bioconductor-gseabase', 'bioconductor-qvalue', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-magrittr', 'r-plyr', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'clusterProfiler\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/clusterProfiler.html'), ('license', 'Artistic-2.0'), ('summary', 'This package implements methods to analyze and visualize functional profiles (GO and KEGG) of gene and gene clusters.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-clusterprofiler/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-clusterprofiler'), ('version', '3.0.5')])), ('source', CommentedMap([('fn', 'clusterProfiler_3.0.5.tar.gz'), ('url', 'https://www.bioconductor.org/packages/3.3/bioc/src/contrib/clusterProfiler_3.0.5.tar.gz'), ('md5', '501b5ff766c9458faffcd3e483d527c6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-dose', 'bioconductor-go.db', 'bioconductor-gosemsim', 'bioconductor-keggrest', 'bioconductor-gseabase', 'bioconductor-qvalue', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-magrittr', 'r-plyr', 'r-tidyr']), ('run', ['bioconductor-annotationdbi', 'bioconductor-dose', 'bioconductor-go.db', 'bioconductor-gosemsim', 'bioconductor-keggrest', 'bioconductor-gseabase', 'bioconductor-qvalue', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-magrittr', 'r-plyr', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'clusterProfiler\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/clusterProfiler.html'), ('license', 'Artistic-2.0'), ('summary', 'This package implements methods to analyze and visualize functional profiles (GO and KEGG) of gene and gene clusters.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-psicquic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-psicquic'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'PSICQUIC_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PSICQUIC_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/PSICQUIC_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/PSICQUIC/PSICQUIC_1.14.0_src_all.tar.gz']), ('md5', '1db512605bb1da83e7ce5371a99db009')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-iranges', 'r-base', 'r-httr', 'r-plyr', 'r-rcurl']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-iranges', 'r-base', 'r-httr', 'r-plyr', 'r-rcurl'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PSICQUIC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PSICQUIC.html'), ('license', 'Apache License 2.0'), ('summary', 'PSICQUIC is a project within the HUPO Proteomics Standard Initiative (HUPO-PSI).  It standardises programmatic access to molecular interaction databases.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-psicquic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-psicquic'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'PSICQUIC_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PSICQUIC_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/PSICQUIC_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/PSICQUIC/PSICQUIC_1.14.0_src_all.tar.gz']), ('md5', '1db512605bb1da83e7ce5371a99db009')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-iranges', 'r-base', 'r-httr', 'r-plyr', 'r-rcurl']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-iranges', 'r-base', 'r-httr', 'r-plyr', 'r-rcurl'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PSICQUIC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PSICQUIC.html'), ('license', 'Apache License 2.0'), ('summary', 'PSICQUIC is a project within the HUPO Proteomics Standard Initiative (HUPO-PSI).  It standardises programmatic access to molecular interaction databases.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pickgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pickgene'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'pickgene_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pickgene_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/pickgene_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/pickgene/pickgene_1.48.0_src_all.tar.gz']), ('md5', 'c1dafca6dc33faac3f80efedd37dd6f8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass']), ('run', ['r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pickgene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pickgene.html'), ('license', 'GPL (>= 2)'), ('summary', 'Functions to Analyze Microarray (Gene Expression) Data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pickgene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pickgene'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'pickgene_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pickgene_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/pickgene_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/pickgene/pickgene_1.48.0_src_all.tar.gz']), ('md5', 'c1dafca6dc33faac3f80efedd37dd6f8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass']), ('run', ['r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pickgene\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pickgene.html'), ('license', 'GPL (>= 2)'), ('summary', 'Functions to Analyze Microarray (Gene Expression) Data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-randpack/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-randpack'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'randPack_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/randPack_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/randPack_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/randPack/randPack_1.22.0_src_all.tar.gz']), ('md5', '0914bfeeeb98eec4683e397668d10d4c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'randPack\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/randPack.html'), ('license', 'Artistic 2.0'), ('summary', 'A suite of classes and functions for randomizing patients in clinical trials.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-randpack/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-randpack'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'randPack_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/randPack_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/randPack_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/randPack/randPack_1.22.0_src_all.tar.gz']), ('md5', '0914bfeeeb98eec4683e397668d10d4c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'randPack\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/randPack.html'), ('license', 'Artistic 2.0'), ('summary', 'A suite of classes and functions for randomizing patients in clinical trials.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mzid/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mzid'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'mzID_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mzID_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/mzID_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/mzID/mzID_1.14.0_src_all.tar.gz']), ('md5', '7069a877ea94c63ecfea78f8556a6fd2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-protgenerics', 'r-base', 'r-doparallel', 'r-foreach', 'r-iterators', 'r-plyr', 'r-xml']), ('run', ['bioconductor-protgenerics', 'r-base', 'r-doparallel', 'r-foreach', 'r-iterators', 'r-plyr', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mzID\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mzID.html'), ('license', 'GPL (>= 2)'), ('summary', \"A parser for mzIdentML files implemented using the XML package. The parser tries to be general and able to handle all types of mzIdentML files with the drawback of having less 'pretty' output than a vendor specific parser. Please contact the maintainer with any problems and supply an mzIdentML file so the problems can be fixed quickly.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mzid/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mzid'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'mzID_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mzID_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/mzID_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/mzID/mzID_1.14.0_src_all.tar.gz']), ('md5', '7069a877ea94c63ecfea78f8556a6fd2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-protgenerics', 'r-base', 'r-doparallel', 'r-foreach', 'r-iterators', 'r-plyr', 'r-xml']), ('run', ['bioconductor-protgenerics', 'r-base', 'r-doparallel', 'r-foreach', 'r-iterators', 'r-plyr', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mzID\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mzID.html'), ('license', 'GPL (>= 2)'), ('summary', \"A parser for mzIdentML files implemented using the XML package. The parser tries to be general and able to handle all types of mzIdentML files with the drawback of having less 'pretty' output than a vendor specific parser. Please contact the maintainer with any problems and supply an mzIdentML file so the problems can be fixed quickly.\")]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cager/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cager'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'CAGEr_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CAGEr_1.16.0.tar.gz'), ('md5', '7228ea30d29211190ecf8ba9c854ee61')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-beanplot', 'r-data.table', 'r-som', 'r-vgam']), ('run', ['bioconductor-bsgenome', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-beanplot', 'r-data.table', 'r-som', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CAGEr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CAGEr.html'), ('license', 'GPL-3'), ('summary', 'Preprocessing of CAGE sequencing data, identification and normalization of transcription start sites and downstream analysis of transcription start sites clusters (promoters).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cager/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cager'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'CAGEr_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CAGEr_1.16.0.tar.gz'), ('md5', '7228ea30d29211190ecf8ba9c854ee61')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-beanplot', 'r-data.table', 'r-som', 'r-vgam']), ('run', ['bioconductor-bsgenome', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-beanplot', 'r-data.table', 'r-som', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CAGEr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CAGEr.html'), ('license', 'GPL-3'), ('summary', 'Preprocessing of CAGE sequencing data, identification and normalization of transcription start sites and downstream analysis of transcription start sites clusters (promoters).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqcluster/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/lpantano/seqclsuter'), ('license', 'MIT'), ('summary', 'small RNA analysis from NGS data')])), ('package', CommentedMap([('name', 'seqcluster'), ('version', '1.2.4a')])), ('source', CommentedMap([('git_url', 'https://github.com/lpantano/seqcluster'), ('git_rev', '9f9f886e6c45b31121a75d1f47a4443dfaa0e96a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'gcc']), ('run', ['python', 'pyyaml', 'pysam >=0.8.1', 'pandas', 'pybedtools', 'progressbar', 'matplotlib'])])), ('test', CommentedMap([('commands', ['seqcluster cluster -h']), ('imports', ['seqcluster'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqcluster/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/lpantano/seqclsuter'), ('license', 'MIT'), ('summary', 'small RNA analysis from NGS data')])), ('package', CommentedMap([('name', 'seqcluster'), ('version', '1.2.4a')])), ('source', CommentedMap([('git_url', 'https://github.com/lpantano/seqcluster'), ('git_rev', '9f9f886e6c45b31121a75d1f47a4443dfaa0e96a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'gcc']), ('run', ['python', 'pyyaml', 'pysam >=0.8.1', 'pandas', 'pybedtools', 'progressbar', 'matplotlib'])])), ('test', CommentedMap([('commands', ['seqcluster cluster -h']), ('imports', ['seqcluster'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rsvsim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rsvsim'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'RSVSim_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RSVSim_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/RSVSim_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/RSVSim/RSVSim_1.16.0_src_all.tar.gz']), ('md5', 'a74e5ea3c69cc4d16df35fb61e6cce86')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-shortread', 'r-base']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-shortread', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RSVSim\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RSVSim.html'), ('license', 'LGPL-3'), ('summary', 'RSVSim is a package for the simulation of deletions, insertions, inversion, tandem-duplications and translocations of various sizes in any genome available as FASTA-file or BSgenome data package. SV breakpoints can be placed uniformly accross the whole genome, with a bias towards repeat regions and regions of high homology (for hg19) or at user-supplied coordinates.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rsvsim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rsvsim'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'RSVSim_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RSVSim_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/RSVSim_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/RSVSim/RSVSim_1.16.0_src_all.tar.gz']), ('md5', 'a74e5ea3c69cc4d16df35fb61e6cce86')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-shortread', 'r-base']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-shortread', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RSVSim\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RSVSim.html'), ('license', 'LGPL-3'), ('summary', 'RSVSim is a package for the simulation of deletions, insertions, inversion, tandem-duplications and translocations of various sizes in any genome available as FASTA-file or BSgenome data package. SV breakpoints can be placed uniformly accross the whole genome, with a bias towards repeat regions and regions of high homology (for hg19) or at user-supplied coordinates.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowai/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowai'), ('version', '1.2.9')])), ('source', CommentedMap([('fn', 'flowAI_1.2.9.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/flowAI_1.2.9.tar.gz'), ('md5', '1dd66dd6dde1a351bda75a00d6f69a85')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore', 'r-base', 'r-changepoint', 'r-ggplot2', 'r-knitr', 'r-plyr', 'r-rcolorbrewer', 'r-reshape2', 'r-scales']), ('run', ['bioconductor-flowcore', 'r-base', 'r-changepoint', 'r-ggplot2', 'r-knitr', 'r-plyr', 'r-rcolorbrewer', 'r-reshape2', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowAI\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowAI.html'), ('license', 'GPL'), ('summary', 'The package is able to perform an automatic or interactive quality control on FCS data acquired using flow cytometry instruments. By evaluating three different properties: 1) flow rate, 2) signal acquisition, 3) dynamic range, the quality control enables the detection and removal of anomalies.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowai/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowai'), ('version', '1.2.9')])), ('source', CommentedMap([('fn', 'flowAI_1.2.9.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/flowAI_1.2.9.tar.gz'), ('md5', '1dd66dd6dde1a351bda75a00d6f69a85')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore', 'r-base', 'r-changepoint', 'r-ggplot2', 'r-knitr', 'r-plyr', 'r-rcolorbrewer', 'r-reshape2', 'r-scales']), ('run', ['bioconductor-flowcore', 'r-base', 'r-changepoint', 'r-ggplot2', 'r-knitr', 'r-plyr', 'r-rcolorbrewer', 'r-reshape2', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowAI\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowAI.html'), ('license', 'GPL'), ('summary', 'The package is able to perform an automatic or interactive quality control on FCS data acquired using flow cytometry instruments. By evaluating three different properties: 1) flow rate, 2) signal acquisition, 3) dynamic range, the quality control enables the detection and removal of anomalies.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bgmix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bgmix'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'BGmix_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BGmix_1.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/BGmix_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/BGmix/BGmix_1.36.0_src_all.tar.gz']), ('md5', 'bdcc2f4d74fcc889d2a7da4392ee230f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-kernsmooth']), ('run', ['r-base', 'r-kernsmooth'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BGmix\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BGmix.html'), ('license', 'GPL-2'), ('summary', 'Fully Bayesian mixture models for differential gene expression')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bgmix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bgmix'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'BGmix_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BGmix_1.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/BGmix_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/BGmix/BGmix_1.36.0_src_all.tar.gz']), ('md5', 'bdcc2f4d74fcc889d2a7da4392ee230f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-kernsmooth']), ('run', ['r-base', 'r-kernsmooth'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BGmix\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BGmix.html'), ('license', 'GPL-2'), ('summary', 'Fully Bayesian mixture models for differential gene expression')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylumi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylumi'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'methylumi_2.22.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/methylumi_2.22.0.tar.gz'), ('md5', '3d169d5e6fbd51d829c691353428478d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-fdb.infiniummethylation.hg19 >=2.2.0', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-minfi', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-ggplot2', 'r-matrixstats', 'r-reshape2', 'r-scales']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-fdb.infiniummethylation.hg19 >=2.2.0', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-minfi', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-ggplot2', 'r-matrixstats', 'r-reshape2', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'methylumi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/methylumi.html'), ('license', 'GPL-2'), ('summary', 'This package provides classes for holding and manipulating Illumina methylation data.  Based on eSet, it can contain MIAME information, sample information, feature information, and multiple matrices of data.  An \"intelligent\" import function, methylumiR can read the Illumina text files and create a MethyLumiSet. methylumIDAT can directly read raw IDAT files from HumanMethylation27 and HumanMethylation450 microarrays.  Normalization, background correction, and quality control features for GoldenGate, Infinium, and Infinium HD arrays are also included.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylumi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylumi'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'methylumi_2.22.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/methylumi_2.22.0.tar.gz'), ('md5', '3d169d5e6fbd51d829c691353428478d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-fdb.infiniummethylation.hg19 >=2.2.0', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-minfi', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-ggplot2', 'r-matrixstats', 'r-reshape2', 'r-scales']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-fdb.infiniummethylation.hg19 >=2.2.0', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-minfi', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-ggplot2', 'r-matrixstats', 'r-reshape2', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'methylumi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/methylumi.html'), ('license', 'GPL-2'), ('summary', 'This package provides classes for holding and manipulating Illumina methylation data.  Based on eSet, it can contain MIAME information, sample information, feature information, and multiple matrices of data.  An \"intelligent\" import function, methylumiR can read the Illumina text files and create a MethyLumiSet. methylumIDAT can directly read raw IDAT files from HumanMethylation27 and HumanMethylation450 microarrays.  Normalization, background correction, and quality control features for GoldenGate, Infinium, and Infinium HD arrays are also included.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-osat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-osat'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'OSAT_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OSAT_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/OSAT_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/OSAT/OSAT_1.24.0_src_all.tar.gz']), ('md5', 'f45375c9d6f49b7049c1c9285b64a3bd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OSAT\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OSAT.html'), ('license', 'Artistic-2.0'), ('summary', 'A sizable genomics study such as microarray often involves the use of multiple batches (groups) of experiment due to practical complication. To minimize batch effects, a careful experiment design should ensure the even distribution of biological groups and confounding factors across batches. OSAT (Optimal Sample Assignment Tool) is developed to facilitate the allocation of collected samples to different batches. With minimum steps, it produces setup that optimizes the even distribution of samples in groups of biological interest into different batches, reducing the confounding or correlation between batches and the biological variables of interest. It can also optimize the even distribution of confounding factors across batches. Our tool can handle challenging instances where incomplete and unbalanced sample collections are involved as well as ideal balanced RCBD. OSAT provides a number of predefined layout for some of the most commonly used genomics platform. Related paper can be find at http://www.biomedcentral.com/1471-2164/13/689 .')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-osat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-osat'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'OSAT_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OSAT_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/OSAT_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/OSAT/OSAT_1.24.0_src_all.tar.gz']), ('md5', 'f45375c9d6f49b7049c1c9285b64a3bd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OSAT\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OSAT.html'), ('license', 'Artistic-2.0'), ('summary', 'A sizable genomics study such as microarray often involves the use of multiple batches (groups) of experiment due to practical complication. To minimize batch effects, a careful experiment design should ensure the even distribution of biological groups and confounding factors across batches. OSAT (Optimal Sample Assignment Tool) is developed to facilitate the allocation of collected samples to different batches. With minimum steps, it produces setup that optimizes the even distribution of samples in groups of biological interest into different batches, reducing the confounding or correlation between batches and the biological variables of interest. It can also optimize the even distribution of confounding factors across batches. Our tool can handle challenging instances where incomplete and unbalanced sample collections are involved as well as ideal balanced RCBD. OSAT provides a number of predefined layout for some of the most commonly used genomics platform. Related paper can be find at http://www.biomedcentral.com/1471-2164/13/689 .')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qpcrnorm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qpcrnorm'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'qpcrNorm_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/qpcrNorm_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/qpcrNorm_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/qpcrNorm/qpcrNorm_1.34.0_src_all.tar.gz']), ('md5', '83dec7f7f085cd88683b982104254259')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-limma', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'qpcrNorm\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/qpcrNorm.html'), ('license', 'LGPL (>= 2)'), ('summary', 'The package contains functions to perform normalization of high-throughput qPCR data. Basic functions for processing raw Ct data plus functions to generate diagnostic plots are also available.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qpcrnorm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qpcrnorm'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'qpcrNorm_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/qpcrNorm_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/qpcrNorm_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/qpcrNorm/qpcrNorm_1.34.0_src_all.tar.gz']), ('md5', '83dec7f7f085cd88683b982104254259')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-limma', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'qpcrNorm\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/qpcrNorm.html'), ('license', 'LGPL (>= 2)'), ('summary', 'The package contains functions to perform normalization of high-throughput qPCR data. Basic functions for processing raw Ct data plus functions to generate diagnostic plots are also available.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bgafun/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bgafun'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'bgafun_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/bgafun_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/bgafun_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/bgafun/bgafun_1.38.0_src_all.tar.gz']), ('md5', 'b8553334229959eed8569fd90827cc8d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-made4', 'r-ade4', 'r-base', 'r-seqinr']), ('run', ['bioconductor-made4', 'r-ade4', 'r-base', 'r-seqinr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'bgafun\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/bgafun.html'), ('license', 'Artistic-2.0'), ('summary', 'A method to identify specifity determining residues in protein families using Between Group Analysis')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bgafun/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bgafun'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'bgafun_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/bgafun_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/bgafun_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/bgafun/bgafun_1.38.0_src_all.tar.gz']), ('md5', 'b8553334229959eed8569fd90827cc8d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-made4', 'r-ade4', 'r-base', 'r-seqinr']), ('run', ['bioconductor-made4', 'r-ade4', 'r-base', 'r-seqinr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'bgafun\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/bgafun.html'), ('license', 'Artistic-2.0'), ('summary', 'A method to identify specifity determining residues in protein families using Between Group Analysis')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/strelka/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'strelka'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'strelka-<{ version }}.tar.bz2'), ('url', 'https://github.com/Illumina/strelka/releases/download/v<{ version }}/strelka-<{ version }}.centos5_x86_64.tar.bz2'), ('md5', '72c57c3c90dd33a0d91c32298bb6320d')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', ['python'])])), ('test', CommentedMap([('commands', ['configureStrelkaGermlineWorkflow.py -h', 'configureStrelkaSomaticWorkflow.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/Illumina/strelka'), ('license', 'GPLv3'), ('summary', 'Strelka calls somatic and germline small variants from mapped sequencing reads')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/strelka/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'strelka'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'strelka-<{ version }}.tar.bz2'), ('url', 'https://github.com/Illumina/strelka/releases/download/v<{ version }}/strelka-<{ version }}.centos5_x86_64.tar.bz2'), ('md5', '72c57c3c90dd33a0d91c32298bb6320d')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', ['python'])])), ('test', CommentedMap([('commands', ['configureStrelkaGermlineWorkflow.py -h', 'configureStrelkaSomaticWorkflow.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/Illumina/strelka'), ('license', 'GPLv3'), ('summary', 'Strelka calls somatic and germline small variants from mapped sequencing reads')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghregions/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghregions'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'CGHregions_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CGHregions_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/CGHregions_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/CGHregions/CGHregions_1.34.0_src_all.tar.gz']), ('md5', '4b2caa536a8bf1f923620f8144dd85e2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-cghbase', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-cghbase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CGHregions\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CGHregions.html'), ('license', 'GPL (http://www.gnu.org/copyleft/gpl.html)'), ('summary', 'Dimension Reduction for Array CGH Data with Minimal Information Loss')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghregions/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghregions'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'CGHregions_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CGHregions_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/CGHregions_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/CGHregions/CGHregions_1.34.0_src_all.tar.gz']), ('md5', '4b2caa536a8bf1f923620f8144dd85e2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-cghbase', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-cghbase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CGHregions\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CGHregions.html'), ('license', 'GPL (http://www.gnu.org/copyleft/gpl.html)'), ('summary', 'Dimension Reduction for Array CGH Data with Minimal Information Loss')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gimmemotifs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gimmemotifs'), ('version', '0.10.0')])), ('source', CommentedMap([('url', 'https://github.com/simonvh/gimmemotifs/releases/download/0.10.0/gimmemotifs-0.10.0.tar.gz'), ('fn', 'gimmemotifs-0.10.0.tar.gz'), ('md5', '6e0ab0269db321819457eb5845cf6c6b')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scipy', 'matplotlib >=2.0', 'pyyaml', 'pybedtools', 'pymc', 'scikit-learn >=0.18', 'lightning', 'moods', 'statsmodels', 'jinja2', 'seaborn', 'perl >=5.22', 'meme', 'gcc', 'llvm', 'htslib <1.4', 'ete3']), ('run', ['libgcc', 'python', 'numpy', 'scipy', 'matplotlib >=2.0', 'pyyaml', 'bedtools', 'pybedtools', 'pymc', 'scikit-learn >=0.18', 'lightning', 'moods', 'ucsc-genepredtobed', 'statsmodels', 'r-robustrankaggreg', 'jinja2', 'seaborn', 'perl >=5.22', 'meme', 'xgboost', 'htslib <1.4', 'pillow', 'ete3'])])), ('test', CommentedMap([('imports', ['gimmemotifs']), ('commands', ['gimme --help'])])), ('about', CommentedMap([('home', 'https://github.com/simonvh/gimmemotifs'), ('license', 'MIT'), ('summary', 'Motif prediction pipeline and various motif-related tools')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gimmemotifs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gimmemotifs'), ('version', '0.10.0')])), ('source', CommentedMap([('url', 'https://github.com/simonvh/gimmemotifs/releases/download/0.10.0/gimmemotifs-0.10.0.tar.gz'), ('fn', 'gimmemotifs-0.10.0.tar.gz'), ('md5', '6e0ab0269db321819457eb5845cf6c6b')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy', 'scipy', 'matplotlib >=2.0', 'pyyaml', 'pybedtools', 'pymc', 'scikit-learn >=0.18', 'lightning', 'moods', 'statsmodels', 'jinja2', 'seaborn', 'perl >=5.22', 'meme', 'gcc', 'llvm', 'htslib <1.4', 'ete3']), ('run', ['libgcc', 'python', 'numpy', 'scipy', 'matplotlib >=2.0', 'pyyaml', 'bedtools', 'pybedtools', 'pymc', 'scikit-learn >=0.18', 'lightning', 'moods', 'ucsc-genepredtobed', 'statsmodels', 'r-robustrankaggreg', 'jinja2', 'seaborn', 'perl >=5.22', 'meme', 'xgboost', 'htslib <1.4', 'pillow', 'ete3'])])), ('test', CommentedMap([('imports', ['gimmemotifs']), ('commands', ['gimme --help'])])), ('about', CommentedMap([('home', 'https://github.com/simonvh/gimmemotifs'), ('license', 'MIT'), ('summary', 'Motif prediction pipeline and various motif-related tools')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/andi/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/EvolBioInf/andi/releases/download/v<{ version }}/andi-<{ version }}.tar.gz'), ('md5', 'af3430c1d817a7a7f6045b97c0518121')])), ('requirements', CommentedMap([('build', ['gcc', 'libdivsufsort', 'gsl']), ('run', ['libgcc', 'libdivsufsort', 'gsl'])])), ('test', CommentedMap([('commands', ['andi -h'])])), ('about', CommentedMap([('home', 'https://github.com/evolbioinf/andi/'), ('license', 'GPL-3.0'), ('summary', 'Efficient Estimation of Evolutionary Distances')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/andi/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/EvolBioInf/andi/releases/download/v<{ version }}/andi-<{ version }}.tar.gz'), ('md5', 'af3430c1d817a7a7f6045b97c0518121')])), ('requirements', CommentedMap([('build', ['gcc', 'libdivsufsort', 'gsl']), ('run', ['libgcc', 'libdivsufsort', 'gsl'])])), ('test', CommentedMap([('commands', ['andi -h'])])), ('about', CommentedMap([('home', 'https://github.com/evolbioinf/andi/'), ('license', 'GPL-3.0'), ('summary', 'Efficient Estimation of Evolutionary Distances')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/newt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'newt'), ('version', '0.52.18')])), ('source', CommentedMap([('fn', 'newt-0.52.18.tar.gz'), ('url', 'http://fedorahosted.org/releases/n/e/newt/newt-0.52.18.tar.gz'), ('md5', '685721bee1a318570704b19dcf31d268')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['popt', 'slang', 'python']), ('run', ['popt', 'slang', 'python'])])), ('test', CommentedMap([('commands', ['whiptail --help']), ('imports', ['snack'])])), ('about', CommentedMap([('home', 'https://fedorahosted.org/newt/'), ('license', 'GPL')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/newt/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'newt'), ('version', '0.52.18')])), ('source', CommentedMap([('fn', 'newt-0.52.18.tar.gz'), ('url', 'http://fedorahosted.org/releases/n/e/newt/newt-0.52.18.tar.gz'), ('md5', '685721bee1a318570704b19dcf31d268')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('build', ['popt', 'slang', 'python']), ('run', ['popt', 'slang', 'python'])])), ('test', CommentedMap([('commands', ['whiptail --help']), ('imports', ['snack'])])), ('about', CommentedMap([('home', 'https://fedorahosted.org/newt/'), ('license', 'GPL')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-complexheatmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-complexheatmap'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'ComplexHeatmap_1.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ComplexHeatmap_1.6.0.tar.gz'), ('md5', '326b74a2700c659cef63bb88884bd61f')])), ('build', CommentedMap([('skip', False), ('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-circlize >=0.3.1', 'r-colorspace', 'r-dendextend >=1.0.1', 'r-getoptlong', 'r-globaloptions >=0.0.6', 'r-rcolorbrewer']), ('run', ['r-base', 'r-circlize >=0.3.1', 'r-colorspace', 'r-dendextend >=1.0.1', 'r-getoptlong', 'r-globaloptions >=0.0.6', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ComplexHeatmap\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html'), ('license', 'GPL (>= 2)'), ('summary', 'Complex heatmaps are efficient to visualize associations between different sources of data sets and reveal potential structures. Here the ComplexHeatmap package provides a highly flexible way to arrange multiple heatmaps and supports self-defined annotation graphics.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-complexheatmap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-complexheatmap'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'ComplexHeatmap_1.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ComplexHeatmap_1.6.0.tar.gz'), ('md5', '326b74a2700c659cef63bb88884bd61f')])), ('build', CommentedMap([('skip', False), ('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-circlize >=0.3.1', 'r-colorspace', 'r-dendextend >=1.0.1', 'r-getoptlong', 'r-globaloptions >=0.0.6', 'r-rcolorbrewer']), ('run', ['r-base', 'r-circlize >=0.3.1', 'r-colorspace', 'r-dendextend >=1.0.1', 'r-getoptlong', 'r-globaloptions >=0.0.6', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ComplexHeatmap\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html'), ('license', 'GPL (>= 2)'), ('summary', 'Complex heatmaps are efficient to visualize associations between different sources of data sets and reveal potential structures. Here the ComplexHeatmap package provides a highly flexible way to arrange multiple heatmaps and supports self-defined annotation graphics.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mreps/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mreps'), ('version', '0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'mreps-2.6.tar'), ('url', 'http://mreps.univ-mlv.fr/mreps-2.6.tar'), ('md5', '4cc5444676501adeddf09f0aaf68dde8')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('files', ['DmelChr4.fa']), ('commands', ['mreps -res 3 -exp 3.0 -from 10000 -to 12000 DmelChr4.fa'])])), ('about', CommentedMap([('home', 'http://mreps.univ-mlv.fr/'), ('license', 'GNU GENERAL PUBLIC LICENSE'), ('license_file', 'LICENSE.md'), ('summary', 'mreps is a flexible and efficient software for identifying serial repeats (usually called tandem repeats) in DNA sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mreps/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mreps'), ('version', '0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'mreps-2.6.tar'), ('url', 'http://mreps.univ-mlv.fr/mreps-2.6.tar'), ('md5', '4cc5444676501adeddf09f0aaf68dde8')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('files', ['DmelChr4.fa']), ('commands', ['mreps -res 3 -exp 3.0 -from 10000 -to 12000 DmelChr4.fa'])])), ('about', CommentedMap([('home', 'http://mreps.univ-mlv.fr/'), ('license', 'GNU GENERAL PUBLIC LICENSE'), ('license_file', 'LICENSE.md'), ('summary', 'mreps is a flexible and efficient software for identifying serial repeats (usually called tandem repeats) in DNA sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-riboprofiling/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-riboprofiling'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'RiboProfiling_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RiboProfiling_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/RiboProfiling_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/RiboProfiling/RiboProfiling_1.6.0_src_all.tar.gz']), ('md5', 'cb557c96cdcd6a681da0cffe90ed616a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-ggbio', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-data.table', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-sqldf']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-ggbio', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-data.table', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-sqldf'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RiboProfiling\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RiboProfiling.html'), ('license', 'GPL-3'), ('summary', \"Starting with a BAM file, this package provides the necessary functions for quality assessment, read start position recalibration, the counting of reads on CDS, 3'UTR, and 5'UTR, plotting of count data: pairs, log fold-change, codon frequency and coverage assessment, principal component analysis on codon coverage.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-riboprofiling/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-riboprofiling'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'RiboProfiling_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RiboProfiling_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/RiboProfiling_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/RiboProfiling/RiboProfiling_1.6.0_src_all.tar.gz']), ('md5', 'cb557c96cdcd6a681da0cffe90ed616a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-ggbio', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-data.table', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-sqldf']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-ggbio', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-data.table', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-sqldf'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RiboProfiling\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RiboProfiling.html'), ('license', 'GPL-3'), ('summary', \"Starting with a BAM file, this package provides the necessary functions for quality assessment, read start position recalibration, the counting of reads on CDS, 3'UTR, and 5'UTR, plotting of count data: pairs, log fold-change, codon frequency and coverage assessment, principal component analysis on codon coverage.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-splicegear/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-splicegear'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'splicegear_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/splicegear_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/splicegear_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/splicegear/splicegear_1.48.0_src_all.tar.gz']), ('md5', '8ba5ae791768e9328c6f3b55f806b159')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-biobase >=2.5.5', 'r-base', 'r-xml']), ('run', ['bioconductor-annotate', 'bioconductor-biobase >=2.5.5', 'r-base', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'splicegear\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/splicegear.html'), ('license', 'LGPL'), ('summary', 'A set of tools to work with alternative splicing')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-splicegear/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-splicegear'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'splicegear_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/splicegear_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/splicegear_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/splicegear/splicegear_1.48.0_src_all.tar.gz']), ('md5', '8ba5ae791768e9328c6f3b55f806b159')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-biobase >=2.5.5', 'r-base', 'r-xml']), ('run', ['bioconductor-annotate', 'bioconductor-biobase >=2.5.5', 'r-base', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'splicegear\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/splicegear.html'), ('license', 'LGPL'), ('summary', 'A set of tools to work with alternative splicing')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stride/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}_v<{ version }}.tar.gz'), ('url', 'https://github.com/ythuang0522/StriDe/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'google-sparsehash', 'autoconf', 'automake', 'm4', 'perl', 'zlib']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['stride 2>&1 | grep \"stride\"'])])), ('about', CommentedMap([('home', 'https://github.com/ythuang0522/StriDe'), ('license', 'GPL3'), ('license_file', 'LICENSE'), ('summary', 'The StriDe Assembler integrates string and de Bruijn graph by decomposing reads within error-prone regions, while extending paire-end read into long reads for assembly through repetitive regions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stride/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}_v<{ version }}.tar.gz'), ('url', 'https://github.com/ythuang0522/StriDe/archive/v<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'google-sparsehash', 'autoconf', 'automake', 'm4', 'perl', 'zlib']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['stride 2>&1 | grep \"stride\"'])])), ('about', CommentedMap([('home', 'https://github.com/ythuang0522/StriDe'), ('license', 'GPL3'), ('license_file', 'LICENSE'), ('summary', 'The StriDe Assembler integrates string and de Bruijn graph by decomposing reads within error-prone regions, while extending paire-end read into long reads for assembly through repetitive regions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/phylip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'phylip'), ('version', 3.696)])), ('source', CommentedMap([('fn', 'phylip-3.696.tar.gz'), ('url', 'http://evolution.gs.washington.edu/phylip/download/phylip-3.696.tar.gz'), ('md5', 'dbe5abc26f6089ead3dba41c2db526ee'), ('patches', ['makefile.patch', 'font-paths.patch', 'maxtrees.patch'])])), ('build', CommentedMap([('number', 3), ('detect_binary_files_with_prefix', True), ('script_env', ['BB_ARCH_FLAGS', 'BB_OPT_FLAGS', 'BB_MAKE_JOBS'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['openjdk >=6', 'libgcc', 'python'])])), ('test', CommentedMap([('commands', ['mkdir __tmp4phyliptests__; pushd __tmp4phyliptests__; phylip test; pushd; rm -rf __tmp4phyliptests__'])])), ('about', CommentedMap([('home', 'http://evolution.genetics.washington.edu/phylip/'), ('license', 'BSD'), ('summary', 'Package of programs for inferring phylogenies')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/phylip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'phylip'), ('version', 3.696)])), ('source', CommentedMap([('fn', 'phylip-3.696.tar.gz'), ('url', 'http://evolution.gs.washington.edu/phylip/download/phylip-3.696.tar.gz'), ('md5', 'dbe5abc26f6089ead3dba41c2db526ee'), ('patches', ['makefile.patch', 'font-paths.patch', 'maxtrees.patch'])])), ('build', CommentedMap([('number', 3), ('detect_binary_files_with_prefix', True), ('script_env', ['BB_ARCH_FLAGS', 'BB_OPT_FLAGS', 'BB_MAKE_JOBS'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['openjdk >=6', 'libgcc', 'python'])])), ('test', CommentedMap([('commands', ['mkdir __tmp4phyliptests__; pushd __tmp4phyliptests__; phylip test; pushd; rm -rf __tmp4phyliptests__'])])), ('about', CommentedMap([('home', 'http://evolution.genetics.washington.edu/phylip/'), ('license', 'BSD'), ('summary', 'Package of programs for inferring phylogenies')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rdgidb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rdgidb'), ('version', '1.2.1')])), ('source', CommentedMap([('fn', 'rDGIdb_1.2.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rDGIdb_1.2.1.tar.gz', 'https://bioarchive.galaxyproject.org/rDGIdb_1.2.1.tar.gz', 'https://depot.galaxyproject.org/software/rDGIdb/rDGIdb_1.2.1_src_all.tar.gz']), ('md5', 'f5b16f45076b3bcacd4fb1fc2659ba04')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-httr', 'r-jsonlite']), ('run', ['r-base', 'r-httr', 'r-jsonlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rDGIdb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rDGIdb.html'), ('license', 'MIT + file LICENSE'), ('summary', 'The rDGIdb package provides a wrapper for the Drug Gene Interaction Database (DGIdb). For simplicity, the wrapper query function and output resembles the user interface and results format provided on the DGIdb website (http://dgidb.genome.wustl.edu/).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rdgidb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rdgidb'), ('version', '1.2.1')])), ('source', CommentedMap([('fn', 'rDGIdb_1.2.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rDGIdb_1.2.1.tar.gz', 'https://bioarchive.galaxyproject.org/rDGIdb_1.2.1.tar.gz', 'https://depot.galaxyproject.org/software/rDGIdb/rDGIdb_1.2.1_src_all.tar.gz']), ('md5', 'f5b16f45076b3bcacd4fb1fc2659ba04')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-httr', 'r-jsonlite']), ('run', ['r-base', 'r-httr', 'r-jsonlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rDGIdb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rDGIdb.html'), ('license', 'MIT + file LICENSE'), ('summary', 'The rDGIdb package provides a wrapper for the Drug Gene Interaction Database (DGIdb). For simplicity, the wrapper query function and output resembles the user interface and results format provided on the DGIdb website (http://dgidb.genome.wustl.edu/).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgraphviz/2.13.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgraphviz'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Rgraphviz_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Rgraphviz_<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base']), ('run', ['bioconductor-graph', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rgraphviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Rgraphviz.html'), ('license', 'EPL'), ('summary', 'Interfaces R with the AT and T graphviz library for plotting R graph objects from the graph package.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgraphviz/2.13.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgraphviz'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Rgraphviz_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Rgraphviz_<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base']), ('run', ['bioconductor-graph', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rgraphviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Rgraphviz.html'), ('license', 'EPL'), ('summary', 'Interfaces R with the AT and T graphviz library for plotting R graph objects from the graph package.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-adsplit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-adsplit'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'adSplit_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/adSplit_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/adSplit/adSplit_1.46.0_src_all.tar.gz']), ('md5', '7638432c734c1fe458acbb9b29384f57')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase >=1.5.12', 'bioconductor-go.db >=1.8.1', 'bioconductor-kegg.db >=1.8.1', 'bioconductor-multtest >=1.6.0', 'r-base', 'r-cluster >=1.9.1']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase >=1.5.12', 'bioconductor-go.db >=1.8.1', 'bioconductor-kegg.db >=1.8.1', 'bioconductor-multtest >=1.6.0', 'r-base', 'r-cluster >=1.9.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'adSplit\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/adSplit.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package implements clustering of microarray gene expression profiles according to functional annotations. For each term genes are annotated to, splits into two subclasses are computed and a significance of the supporting gene set is determined.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-adsplit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-adsplit'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'adSplit_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/adSplit_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/adSplit/adSplit_1.46.0_src_all.tar.gz']), ('md5', '7638432c734c1fe458acbb9b29384f57')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase >=1.5.12', 'bioconductor-go.db >=1.8.1', 'bioconductor-kegg.db >=1.8.1', 'bioconductor-multtest >=1.6.0', 'r-base', 'r-cluster >=1.9.1']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase >=1.5.12', 'bioconductor-go.db >=1.8.1', 'bioconductor-kegg.db >=1.8.1', 'bioconductor-multtest >=1.6.0', 'r-base', 'r-cluster >=1.9.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'adSplit\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/adSplit.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package implements clustering of microarray gene expression profiles according to functional annotations. For each term genes are annotated to, splits into two subclasses are computed and a significance of the supporting gene set is determined.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mvcclass/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mvcclass'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'MVCClass_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MVCClass_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/MVCClass_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/MVCClass/MVCClass_1.50.0_src_all.tar.gz']), ('md5', 'fde9aa791e789b3eab70420f3081886f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MVCClass\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MVCClass.html'), ('license', 'LGPL'), ('summary', 'Creates classes used in model-view-controller (MVC) design')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mvcclass/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mvcclass'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'MVCClass_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MVCClass_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/MVCClass_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/MVCClass/MVCClass_1.50.0_src_all.tar.gz']), ('md5', 'fde9aa791e789b3eab70420f3081886f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MVCClass\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MVCClass.html'), ('license', 'LGPL'), ('summary', 'Creates classes used in model-view-controller (MVC) design')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-monocle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-<{ name|lower }}'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/<{ name }}_<{ version }}.tar.gz'), ('md5', '5d9bca1e5c32ae86a9a970e95867e230')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-hsmmsinglecell >=0.101.5', 'bioconductor-limma', 'r-base', 'r-combinat', 'r-ddrtree >=0.1.4', 'r-densityclust', 'r-dplyr', 'r-fastica', 'r-ggplot2 >=1.0.0', 'r-igraph >=1.0.1', 'r-irlba >=2.0.0', 'r-matrixstats', 'r-pheatmap', 'r-plyr', 'r-proxy', 'r-qlcmatrix', 'r-reshape2', 'r-rtsne', 'r-slam', 'r-stringr', 'r-vgam >=1.0-1']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-hsmmsinglecell >=0.101.5', 'bioconductor-limma', 'r-base', 'r-combinat', 'r-ddrtree >=0.1.4', 'r-densityclust', 'r-dplyr', 'r-fastica', 'r-ggplot2 >=1.0.0', 'r-igraph >=1.0.1', 'r-irlba >=2.0.0', 'r-matrixstats', 'r-pheatmap', 'r-plyr', 'r-proxy', 'r-qlcmatrix', 'r-reshape2', 'r-rtsne', 'r-slam', 'r-stringr', 'r-vgam >=1.0-1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'monocle\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/monocle.html'), ('license', 'Artistic-2.0'), ('summary', 'Monocle performs differential expression and time-series analysis for single-cell expression experiments. It orders individual cells according to progress through a biological process, without knowing ahead of time which genes define progress through that process. Monocle also performs differential expression analysis, clustering, visualization, and other useful tasks on single cell expression data.  It is designed to work with RNA-Seq and qPCR data, but could be used with other types as well.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-monocle/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-<{ name|lower }}'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', '<{ name }}_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/<{ name }}_<{ version }}.tar.gz'), ('md5', '5d9bca1e5c32ae86a9a970e95867e230')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-hsmmsinglecell >=0.101.5', 'bioconductor-limma', 'r-base', 'r-combinat', 'r-ddrtree >=0.1.4', 'r-densityclust', 'r-dplyr', 'r-fastica', 'r-ggplot2 >=1.0.0', 'r-igraph >=1.0.1', 'r-irlba >=2.0.0', 'r-matrixstats', 'r-pheatmap', 'r-plyr', 'r-proxy', 'r-qlcmatrix', 'r-reshape2', 'r-rtsne', 'r-slam', 'r-stringr', 'r-vgam >=1.0-1']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-hsmmsinglecell >=0.101.5', 'bioconductor-limma', 'r-base', 'r-combinat', 'r-ddrtree >=0.1.4', 'r-densityclust', 'r-dplyr', 'r-fastica', 'r-ggplot2 >=1.0.0', 'r-igraph >=1.0.1', 'r-irlba >=2.0.0', 'r-matrixstats', 'r-pheatmap', 'r-plyr', 'r-proxy', 'r-qlcmatrix', 'r-reshape2', 'r-rtsne', 'r-slam', 'r-stringr', 'r-vgam >=1.0-1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'monocle\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/monocle.html'), ('license', 'Artistic-2.0'), ('summary', 'Monocle performs differential expression and time-series analysis for single-cell expression experiments. It orders individual cells according to progress through a biological process, without knowing ahead of time which genes define progress through that process. Monocle also performs differential expression analysis, clustering, visualization, and other useful tasks on single cell expression data.  It is designed to work with RNA-Seq and qPCR data, but could be used with other types as well.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/quast/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/<{ name|lower }}/<{ name|lower }}-<{ version }}.tar.gz'), ('md5', '<{ hash }}')])), ('requirements', CommentedMap([('build', ['gcc', 'boost <{CONDA_BOOST}}*', 'perl', 'python', 'matplotlib', 'openjdk', 'joblib', 'simplejson']), ('run', ['boost <{CONDA_BOOST}}*', 'perl', 'python', 'libgcc', 'matplotlib', 'openjdk', 'joblib', 'simplejson'])])), ('test', CommentedMap([('commands', ['quast --version', 'quast --test', 'metaquast --version', 'metaquast --test'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/<{ name|lower }}/'), ('license', 'GPL2'), ('license_file', 'LICENSE.txt'), ('summary', 'Quality Assessment Tool for Genome Assemblies')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/quast/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/<{ name|lower }}/<{ name|lower }}-<{ version }}.tar.gz'), ('md5', '<{ hash }}')])), ('requirements', CommentedMap([('build', ['gcc', 'boost <{CONDA_BOOST}}*', 'perl', 'python', 'matplotlib', 'openjdk', 'joblib', 'simplejson']), ('run', ['boost <{CONDA_BOOST}}*', 'perl', 'python', 'libgcc', 'matplotlib', 'openjdk', 'joblib', 'simplejson'])])), ('test', CommentedMap([('commands', ['quast --version', 'quast --test', 'metaquast --version', 'metaquast --test'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/<{ name|lower }}/'), ('license', 'GPL2'), ('license_file', 'LICENSE.txt'), ('summary', 'Quality Assessment Tool for Genome Assemblies')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qualimap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qualimap'), ('version', '2.2.2a')])), ('source', CommentedMap([('fn', 'qualimap-build-11-12-16.tar.gz'), ('url', 'https://bitbucket.org/kokonech/qualimap/downloads/qualimap-build-11-12-16.tar.gz'), ('md5', '2bacc61c0ed0412f09c6f02cae25bb7c')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['qualimap -h'])])), ('about', CommentedMap([('home', 'http://qualimap.bioinfo.cipf.es/'), ('license', 'GPLv2'), ('summary', 'Quality control of alignment sequencing data and its derivatives like feature counts')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qualimap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qualimap'), ('version', '2.2.2a')])), ('source', CommentedMap([('fn', 'qualimap-build-11-12-16.tar.gz'), ('url', 'https://bitbucket.org/kokonech/qualimap/downloads/qualimap-build-11-12-16.tar.gz'), ('md5', '2bacc61c0ed0412f09c6f02cae25bb7c')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['qualimap -h'])])), ('about', CommentedMap([('home', 'http://qualimap.bioinfo.cipf.es/'), ('license', 'GPLv2'), ('summary', 'Quality control of alignment sequencing data and its derivatives like feature counts')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/clustalw/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.clustal.org/clustal2/'), ('license', 'GNU Lesser GPL'), ('summary', 'ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins.')])), ('package', CommentedMap([('name', 'clustalw'), ('version', 2.1)])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'clustalw-2.1.tar.gz'), ('url', 'http://www.clustal.org/download/current/clustalw-2.1.tar.gz'), ('md5', '144df8440a0ae083d5167616c8ceeb41')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['clustalw2 H 2>&1 | grep \"CLUSTAL 2.1\"', 'clustalw H 2>&1 | grep \"CLUSTAL 2.1\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/clustalw/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.clustal.org/clustal2/'), ('license', 'GNU Lesser GPL'), ('summary', 'ClustalW2 is a general purpose multiple sequence alignment program for DNA or proteins.')])), ('package', CommentedMap([('name', 'clustalw'), ('version', 2.1)])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'clustalw-2.1.tar.gz'), ('url', 'http://www.clustal.org/download/current/clustalw-2.1.tar.gz'), ('md5', '144df8440a0ae083d5167616c8ceeb41')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['clustalw2 H 2>&1 | grep \"CLUSTAL 2.1\"', 'clustalw H 2>&1 | grep \"CLUSTAL 2.1\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affxparser/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affxparser'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'affxparser_1.48.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/affxparser_1.48.0.tar.gz'), ('md5', '20ae3f61e3ea25c3baeabf949b1f1165')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affxparser\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/affxparser.html'), ('license', 'LGPL (>= 2)'), ('summary', \"Package for parsing Affymetrix files (CDF, CEL, CHP, BPMAP, BAR).  It provides methods for fast and memory efficient parsing of Affymetrix files using the Affymetrix' Fusion SDK.  Both ASCII- and binary-based files are supported.  Currently, there are methods for reading chip definition file (CDF) and a cell intensity file (CEL).  These files can be read either in full or in part.  For example, probe signals from a few probesets can be extracted very quickly from a set of CEL files into a convenient list structure.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affxparser/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affxparser'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'affxparser_1.48.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/affxparser_1.48.0.tar.gz'), ('md5', '20ae3f61e3ea25c3baeabf949b1f1165')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affxparser\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/affxparser.html'), ('license', 'LGPL (>= 2)'), ('summary', \"Package for parsing Affymetrix files (CDF, CEL, CHP, BPMAP, BAR).  It provides methods for fast and memory efficient parsing of Affymetrix files using the Affymetrix' Fusion SDK.  Both ASCII- and binary-based files are supported.  Currently, there are methods for reading chip definition file (CDF) and a cell intensity file (CEL).  These files can be read either in full or in part.  For example, probe signals from a few probesets can be extracted very quickly from a set of CEL files into a convenient list structure.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtopper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtopper'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'RTopper_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RTopper_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/RTopper_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/RTopper/RTopper_1.22.0_src_all.tar.gz']), ('md5', 'e5900c1e1f7e33a2a42c5abe3f6bce79')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RTopper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RTopper.html'), ('license', 'GPL (>= 3)'), ('summary', 'the RTopper package is designed to perform and integrate gene set enrichment results across multiple genomic platforms.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtopper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtopper'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'RTopper_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RTopper_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/RTopper_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/RTopper/RTopper_1.22.0_src_all.tar.gz']), ('md5', 'e5900c1e1f7e33a2a42c5abe3f6bce79')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RTopper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RTopper.html'), ('license', 'GPL (>= 3)'), ('summary', 'the RTopper package is designed to perform and integrate gene set enrichment results across multiple genomic platforms.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-macorrplot/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-macorrplot'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'maCorrPlot_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/maCorrPlot_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/maCorrPlot_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/maCorrPlot/maCorrPlot_1.46.0_src_all.tar.gz']), ('md5', '96c19ed0a305c4da96b1619e4db20fda')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-lattice']), ('run', ['r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'maCorrPlot\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/maCorrPlot.html'), ('license', 'GPL (>= 2)'), ('summary', 'Graphically displays correlation in microarray data that is due to insufficient normalization')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-macorrplot/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-macorrplot'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'maCorrPlot_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/maCorrPlot_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/maCorrPlot_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/maCorrPlot/maCorrPlot_1.46.0_src_all.tar.gz']), ('md5', '96c19ed0a305c4da96b1619e4db20fda')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-lattice']), ('run', ['r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'maCorrPlot\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/maCorrPlot.html'), ('license', 'GPL (>= 2)'), ('summary', 'Graphically displays correlation in microarray data that is due to insufficient normalization')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rcytoscape/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rcytoscape'), ('version', '1.21.1')])), ('source', CommentedMap([('fn', 'RCytoscape_1.21.1.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/RCytoscape_1.21.1.tar.gz'), ('md5', '06aa503a70ada9bae79b38cc5de1eba2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-graph >=1.31.0', 'r-base', 'r-xmlrpc >=0.2.4']), ('run', ['bioconductor-biocgenerics', 'bioconductor-graph >=1.31.0', 'r-base', 'r-xmlrpc >=0.2.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RCytoscape\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RCytoscape.html'), ('license', 'GPL-2'), ('summary', 'Interactvive viewing and exploration of graphs, connecting R to Cytoscape.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rcytoscape/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rcytoscape'), ('version', '1.21.1')])), ('source', CommentedMap([('fn', 'RCytoscape_1.21.1.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/RCytoscape_1.21.1.tar.gz'), ('md5', '06aa503a70ada9bae79b38cc5de1eba2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-graph >=1.31.0', 'r-base', 'r-xmlrpc >=0.2.4']), ('run', ['bioconductor-biocgenerics', 'bioconductor-graph >=1.31.0', 'r-base', 'r-xmlrpc >=0.2.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RCytoscape\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RCytoscape.html'), ('license', 'GPL-2'), ('summary', 'Interactvive viewing and exploration of graphs, connecting R to Cytoscape.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-ipc-system-simple/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-ipc-system-simple'), ('version', '1.25')])), ('source', CommentedMap([('fn', 'IPC-System-Simple-1.25.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PJ/PJF/IPC-System-Simple-1.25.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['IPC::System::Simple'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/IPC::System::Simple'), ('license', 'Perl'), ('summary', 'Run commands simply, with detailed diagnostics')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-ipc-system-simple/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-ipc-system-simple'), ('version', '1.25')])), ('source', CommentedMap([('fn', 'IPC-System-Simple-1.25.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PJ/PJF/IPC-System-Simple-1.25.tar.gz')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('imports', ['IPC::System::Simple'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/IPC::System::Simple'), ('license', 'Perl'), ('summary', 'Run commands simply, with detailed diagnostics')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tximport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tximport'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'tximport_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/tximport_1.4.0.tar.gz'), ('md5', 'a2a9235064ea1fa89af71d260b50b790')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'tximport\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/tximport.html'), ('license', 'GPL (>=2)'), ('summary', 'Imports transcript-level abundance, estimated counts and transcript lengths, and summarizes into matrices for use with downstream gene-level analysis packages. Average transcript length, weighted by sample-specific transcript abundance estimates, is provided as a matrix which can be used as an offset for different expression of gene-level counts.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tximport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tximport'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'tximport_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/tximport_1.4.0.tar.gz'), ('md5', 'a2a9235064ea1fa89af71d260b50b790')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'tximport\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/tximport.html'), ('license', 'GPL (>=2)'), ('summary', 'Imports transcript-level abundance, estimated counts and transcript lengths, and summarizes into matrices for use with downstream gene-level analysis packages. Average transcript length, weighted by sample-specific transcript abundance estimates, is provided as a matrix which can be used as an offset for different expression of gene-level counts.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-scran/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-scran'), ('version', '1.4.5')])), ('source', CommentedMap([('fn', 'scran_1.4.5.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/scran_1.4.5.tar.gz', 'https://depot.galaxyproject.org/software/scran/scran_1.4.5_src_all.tar.gz']), ('sha256', '91eac3907d4cf75c532d0fbe9a3f3f88652523d67b9d852fe391be37bfa93977')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-scater', 'r-base >=3.4', 'r-dt', 'r-dynamictreecut', 'r-fnn', 'r-ggplot2', 'r-igraph', 'r-mass', 'r-matrix', 'r-shiny', 'r-statmod', 'r-viridis', 'r-zoo']), ('run', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-scater', 'r-base >=3.4', 'r-dt', 'r-dynamictreecut', 'r-fnn', 'r-ggplot2', 'r-igraph', 'r-mass', 'r-matrix', 'r-shiny', 'r-statmod', 'r-viridis', 'r-zoo'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'scran\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/scran.html'), ('license', 'GPL-3'), ('summary', 'Implements a variety of low-level analyses of single-cell RNA-seq data. Methods are provided for normalization of cell-specific biases, assignment of cell cycle phase, and detection of highly variable and significantly correlated genes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-scran/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-scran'), ('version', '1.4.5')])), ('source', CommentedMap([('fn', 'scran_1.4.5.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/scran_1.4.5.tar.gz', 'https://depot.galaxyproject.org/software/scran/scran_1.4.5_src_all.tar.gz']), ('sha256', '91eac3907d4cf75c532d0fbe9a3f3f88652523d67b9d852fe391be37bfa93977')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-scater', 'r-base >=3.4', 'r-dt', 'r-dynamictreecut', 'r-fnn', 'r-ggplot2', 'r-igraph', 'r-mass', 'r-matrix', 'r-shiny', 'r-statmod', 'r-viridis', 'r-zoo']), ('run', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-scater', 'r-base >=3.4', 'r-dt', 'r-dynamictreecut', 'r-fnn', 'r-ggplot2', 'r-igraph', 'r-mass', 'r-matrix', 'r-shiny', 'r-statmod', 'r-viridis', 'r-zoo'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'scran\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/scran.html'), ('license', 'GPL-3'), ('summary', 'Implements a variety of low-level analyses of single-cell RNA-seq data. Methods are provided for normalization of cell-specific biases, assignment of cell cycle phase, and detection of highly variable and significantly correlated genes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fraggenescan/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/fraggenescan/<{ name }}<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['hmm_lib.c.patch', 'util_lib.c.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['perl', 'libgcc'])])), ('test', CommentedMap([('commands', ['FragGeneScan &> /dev/null || [[ \"$?\" == 1 ]]', 'run_FragGeneScan.pl &> /dev/null || [[ \"$?\" == 1 ]]'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/fraggenescan/'), ('license', 'BSD'), ('summary', 'FragGeneScan is an application for finding (fragmented) genes in short reads.'), ('description', 'FragGeneScan is an application for finding (fragmented) genes in short\\nreads. It can also be applied to predict prokaryotic genes in incomplete\\nassemblies or complete genomes.\\n')])), ('extra', CommentedMap([('maintainers', ['keuv-grvl']), ('doi', '10.1093/nar/gkq747')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fraggenescan/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/fraggenescan/<{ name }}<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['hmm_lib.c.patch', 'util_lib.c.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['perl', 'libgcc'])])), ('test', CommentedMap([('commands', ['FragGeneScan &> /dev/null || [[ \"$?\" == 1 ]]', 'run_FragGeneScan.pl &> /dev/null || [[ \"$?\" == 1 ]]'])])), ('about', CommentedMap([('home', 'https://sourceforge.net/projects/fraggenescan/'), ('license', 'BSD'), ('summary', 'FragGeneScan is an application for finding (fragmented) genes in short reads.'), ('description', 'FragGeneScan is an application for finding (fragmented) genes in short\\nreads. It can also be applied to predict prokaryotic genes in incomplete\\nassemblies or complete genomes.\\n')])), ('extra', CommentedMap([('maintainers', ['keuv-grvl']), ('doi', '10.1093/nar/gkq747')]))]))\n",
      "EXTRA has been found: fraggenescan\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chipcomp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chipcomp'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'ChIPComp_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ChIPComp_1.0.0.tar.gz'), ('md5', '8f7972f826e866923a2691a3a634b01d')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-genomeinfodb', 'bioconductor-s4vectors', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-biocgenerics', 'bioconductor-rsamtools', 'bioconductor-limma', 'bioconductor-bsgenome.hsapiens.ucsc.hg19']), ('run', ['r-base', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-genomeinfodb', 'bioconductor-s4vectors', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-biocgenerics', 'bioconductor-rsamtools', 'bioconductor-limma', 'bioconductor-bsgenome.hsapiens.ucsc.hg19'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ChIPComp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ChIPComp.html'), ('license', 'GPL'), ('summary', 'ChIPComp detects differentially bound sharp binding sites across multiple conditions considering matching control.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chipcomp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chipcomp'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'ChIPComp_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ChIPComp_1.0.0.tar.gz'), ('md5', '8f7972f826e866923a2691a3a634b01d')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-genomeinfodb', 'bioconductor-s4vectors', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-biocgenerics', 'bioconductor-rsamtools', 'bioconductor-limma', 'bioconductor-bsgenome.hsapiens.ucsc.hg19']), ('run', ['r-base', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-genomeinfodb', 'bioconductor-s4vectors', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-biocgenerics', 'bioconductor-rsamtools', 'bioconductor-limma', 'bioconductor-bsgenome.hsapiens.ucsc.hg19'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ChIPComp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ChIPComp.html'), ('license', 'GPL'), ('summary', 'ChIPComp detects differentially bound sharp binding sites across multiple conditions considering matching control.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/biomaj/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'biomaj'), ('version', '3.0.19')])), ('source', CommentedMap([('fn', 'biomaj-3.0.19.tar.gz'), ('url', 'https://pypi.python.org/packages/ae/27/1ca70bd9b7c33dc66068d970158a0af1f79ad8823c5bb04cccb21a4eec3b/biomaj-3.0.19.tar.gz'), ('md5', '834631961cab1d87b1a454e08489ef09'), ('patches', ['deps.patch'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'nose', 'pymongo', 'pycurl', 'bcrypt', 'ldap3', 'mock', 'drmaa', 'future', 'tabulate', 'elasticsearch']), ('run', ['python', 'pymongo', 'pycurl', 'bcrypt', 'ldap3', 'drmaa', 'future', 'tabulate', 'elasticsearch'])])), ('test', CommentedMap([('imports', ['biomaj', 'biomaj.download', 'biomaj.process']), ('commands', ['biomaj-cli.py --version'])])), ('about', CommentedMap([('home', 'http://biomaj.genouest.org'), ('license', 'GNU Affero General Public License v3 or later (AGPLv3+)'), ('license_family', 'AGPL'), ('summary', 'Automates the update cycle and the supervision of the locally mirrored databank repository')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/biomaj/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'biomaj'), ('version', '3.0.19')])), ('source', CommentedMap([('fn', 'biomaj-3.0.19.tar.gz'), ('url', 'https://pypi.python.org/packages/ae/27/1ca70bd9b7c33dc66068d970158a0af1f79ad8823c5bb04cccb21a4eec3b/biomaj-3.0.19.tar.gz'), ('md5', '834631961cab1d87b1a454e08489ef09'), ('patches', ['deps.patch'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'nose', 'pymongo', 'pycurl', 'bcrypt', 'ldap3', 'mock', 'drmaa', 'future', 'tabulate', 'elasticsearch']), ('run', ['python', 'pymongo', 'pycurl', 'bcrypt', 'ldap3', 'drmaa', 'future', 'tabulate', 'elasticsearch'])])), ('test', CommentedMap([('imports', ['biomaj', 'biomaj.download', 'biomaj.process']), ('commands', ['biomaj-cli.py --version'])])), ('about', CommentedMap([('home', 'http://biomaj.genouest.org'), ('license', 'GNU Affero General Public License v3 or later (AGPLv3+)'), ('license_family', 'AGPL'), ('summary', 'Automates the update cycle and the supervision of the locally mirrored databank repository')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgalaxy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgalaxy'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'RGalaxy_1.18.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/RGalaxy_1.18.0.tar.gz'), ('md5', '51d1b7553447676c5a48d48133ba6902')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-digest', 'r-optparse', 'r-roxygen2', 'r-xml']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-digest', 'r-optparse', 'r-roxygen2', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RGalaxy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RGalaxy.html'), ('license', 'Artistic-2.0'), ('summary', 'Given an R function and its manual page, make the documented function available in Galaxy.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgalaxy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgalaxy'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'RGalaxy_1.18.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/RGalaxy_1.18.0.tar.gz'), ('md5', '51d1b7553447676c5a48d48133ba6902')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-digest', 'r-optparse', 'r-roxygen2', 'r-xml']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-digest', 'r-optparse', 'r-roxygen2', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RGalaxy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RGalaxy.html'), ('license', 'Artistic-2.0'), ('summary', 'Given an R function and its manual page, make the documented function available in Galaxy.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-starr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-starr'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'Starr_1.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Starr_1.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/Starr_1.32.0.tar.gz', 'https://depot.galaxyproject.org/software/Starr/Starr_1.32.0_src_all.tar.gz']), ('md5', 'f1593771aa0eaa26868fe88e4af5944d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affxparser', 'bioconductor-affy', 'bioconductor-ringo', 'bioconductor-zlibbioc', 'r-base', 'r-mass', 'r-pspline']), ('run', ['bioconductor-affxparser', 'bioconductor-affy', 'bioconductor-ringo', 'bioconductor-zlibbioc', 'r-base', 'r-mass', 'r-pspline'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Starr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Starr.html'), ('license', 'Artistic-2.0'), ('summary', 'Starr facilitates the analysis of ChIP-chip data, in particular that of Affymetrix tiling arrays. The package provides functions for data import, quality assessment, data visualization and exploration. Furthermore, it includes high-level analysis features like association of ChIP signals with annotated features, correlation analysis of ChIP signals and other genomic data (e.g. gene expression), peak-finding with the CMARRT algorithm and comparative display of multiple clusters of ChIP-profiles. It uses the basic Bioconductor classes ExpressionSet and probeAnno for maximum compatibility with other software on Bioconductor. All functions from Starr can be used to investigate preprocessed data from the Ringo package, and vice versa. An important novel tool is the the automated generation of correct, up-to-date microarray probe annotation (bpmap) files, which relies on an efficient mapping of short sequences (e.g. the probe sequences on a microarray) to an arbitrary genome.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-starr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-starr'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'Starr_1.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Starr_1.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/Starr_1.32.0.tar.gz', 'https://depot.galaxyproject.org/software/Starr/Starr_1.32.0_src_all.tar.gz']), ('md5', 'f1593771aa0eaa26868fe88e4af5944d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affxparser', 'bioconductor-affy', 'bioconductor-ringo', 'bioconductor-zlibbioc', 'r-base', 'r-mass', 'r-pspline']), ('run', ['bioconductor-affxparser', 'bioconductor-affy', 'bioconductor-ringo', 'bioconductor-zlibbioc', 'r-base', 'r-mass', 'r-pspline'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Starr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Starr.html'), ('license', 'Artistic-2.0'), ('summary', 'Starr facilitates the analysis of ChIP-chip data, in particular that of Affymetrix tiling arrays. The package provides functions for data import, quality assessment, data visualization and exploration. Furthermore, it includes high-level analysis features like association of ChIP signals with annotated features, correlation analysis of ChIP signals and other genomic data (e.g. gene expression), peak-finding with the CMARRT algorithm and comparative display of multiple clusters of ChIP-profiles. It uses the basic Bioconductor classes ExpressionSet and probeAnno for maximum compatibility with other software on Bioconductor. All functions from Starr can be used to investigate preprocessed data from the Ringo package, and vice versa. An important novel tool is the the automated generation of correct, up-to-date microarray probe annotation (bpmap) files, which relies on an efficient mapping of short sequences (e.g. the probe sequences on a microarray) to an arbitrary genome.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylseekr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylseekr'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'MethylSeekR_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MethylSeekR_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/MethylSeekR_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/MethylSeekR/MethylSeekR_1.16.0_src_all.tar.gz']), ('md5', 'ba38b395a899e88619f414301225447d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome >=1.26.1', 'bioconductor-geneplotter >=1.34.0', 'bioconductor-genomicranges >=1.10.5', 'bioconductor-iranges >=1.16.3', 'bioconductor-rtracklayer >=1.16.3', 'r-base', 'r-mhsmm >=0.4.4']), ('run', ['bioconductor-bsgenome >=1.26.1', 'bioconductor-geneplotter >=1.34.0', 'bioconductor-genomicranges >=1.10.5', 'bioconductor-iranges >=1.16.3', 'bioconductor-rtracklayer >=1.16.3', 'r-base', 'r-mhsmm >=0.4.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MethylSeekR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MethylSeekR.html'), ('license', 'GPL (>=2)'), ('summary', 'This is a package for the discovery of regulatory regions from Bis-seq data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylseekr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylseekr'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'MethylSeekR_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MethylSeekR_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/MethylSeekR_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/MethylSeekR/MethylSeekR_1.16.0_src_all.tar.gz']), ('md5', 'ba38b395a899e88619f414301225447d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome >=1.26.1', 'bioconductor-geneplotter >=1.34.0', 'bioconductor-genomicranges >=1.10.5', 'bioconductor-iranges >=1.16.3', 'bioconductor-rtracklayer >=1.16.3', 'r-base', 'r-mhsmm >=0.4.4']), ('run', ['bioconductor-bsgenome >=1.26.1', 'bioconductor-geneplotter >=1.34.0', 'bioconductor-genomicranges >=1.10.5', 'bioconductor-iranges >=1.16.3', 'bioconductor-rtracklayer >=1.16.3', 'r-base', 'r-mhsmm >=0.4.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MethylSeekR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MethylSeekR.html'), ('license', 'GPL (>=2)'), ('summary', 'This is a package for the discovery of regulatory regions from Bis-seq data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-scater/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-scater'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'scater_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/scater_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/scater_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/scater/scater_1.4.0_src_all.tar.gz']), ('md5', '8033f360d74414444205ec87898675e6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-rhdf5', 'bioconductor-tximport', 'r-base', 'r-data.table', 'r-dplyr', 'r-ggbeeswarm', 'r-ggplot2', 'r-matrix', 'r-matrixstats', 'r-plyr', 'r-reshape2', 'r-rjson', 'r-shiny', 'r-shinydashboard', 'r-viridis']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-rhdf5', 'bioconductor-tximport', 'r-base', 'r-data.table', 'r-dplyr', 'r-ggbeeswarm', 'r-ggplot2', 'r-matrix', 'r-matrixstats', 'r-plyr', 'r-reshape2', 'r-rjson', 'r-shiny', 'r-shinydashboard', 'r-viridis'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'scater\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/scater.html'), ('license', 'GPL (>= 2)'), ('summary', 'A collection of tools for doing various analyses of single-cell RNA-seq gene expression data, with a focus on quality control.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-scater/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-scater'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'scater_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/scater_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/scater_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/scater/scater_1.4.0_src_all.tar.gz']), ('md5', '8033f360d74414444205ec87898675e6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-rhdf5', 'bioconductor-tximport', 'r-base', 'r-data.table', 'r-dplyr', 'r-ggbeeswarm', 'r-ggplot2', 'r-matrix', 'r-matrixstats', 'r-plyr', 'r-reshape2', 'r-rjson', 'r-shiny', 'r-shinydashboard', 'r-viridis']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-edger', 'bioconductor-limma', 'bioconductor-rhdf5', 'bioconductor-tximport', 'r-base', 'r-data.table', 'r-dplyr', 'r-ggbeeswarm', 'r-ggplot2', 'r-matrix', 'r-matrixstats', 'r-plyr', 'r-reshape2', 'r-rjson', 'r-shiny', 'r-shinydashboard', 'r-viridis'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'scater\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/scater.html'), ('license', 'GPL (>= 2)'), ('summary', 'A collection of tools for doing various analyses of single-cell RNA-seq gene expression data, with a focus on quality control.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-filter-simple/0.91/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Filter::Simple\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Filter::Simple'), ('license', 'perl_5'), ('summary', 'Simplified source filtering')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-filter-simple/0.91/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Filter::Simple\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Filter::Simple'), ('license', 'perl_5'), ('summary', 'Simplified source filtering')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-string-util/1.26/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-string-util'), ('version', '1.26')])), ('source', CommentedMap([('fn', 'String-Util-1.26.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MI/MIKO/String-Util-1.26.tar.gz'), ('md5', 'd0f13f8688bedef7174634bb57f7fbe4')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build', 'perl-number-misc', 'perl-test-toolbox']), ('run', ['perl', 'perl-number-misc', 'perl-test-toolbox'])])), ('test', CommentedMap([('imports', ['String::Util'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/String::Util'), ('license', 'perl_5'), ('summary', 'String::Util -- String processing utilities')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-string-util/1.26/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-string-util'), ('version', '1.26')])), ('source', CommentedMap([('fn', 'String-Util-1.26.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/M/MI/MIKO/String-Util-1.26.tar.gz'), ('md5', 'd0f13f8688bedef7174634bb57f7fbe4')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-module-build', 'perl-number-misc', 'perl-test-toolbox']), ('run', ['perl', 'perl-number-misc', 'perl-test-toolbox'])])), ('test', CommentedMap([('imports', ['String::Util'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/String::Util'), ('license', 'perl_5'), ('summary', 'String::Util -- String processing utilities')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-multiscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-multiscan'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'multiscan_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/multiscan_1.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/multiscan_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/multiscan/multiscan_1.36.0_src_all.tar.gz']), ('md5', '27ed1fe589004fef769d3179b5231f0b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'multiscan\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/multiscan.html'), ('license', 'GPL (>= 2)'), ('summary', 'Estimates gene expressions from several laser scans of the same microarray')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-multiscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-multiscan'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'multiscan_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/multiscan_1.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/multiscan_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/multiscan/multiscan_1.36.0_src_all.tar.gz']), ('md5', '27ed1fe589004fef769d3179b5231f0b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'multiscan\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/multiscan.html'), ('license', 'GPL (>= 2)'), ('summary', 'Estimates gene expressions from several laser scans of the same microarray')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-a4preproc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-a4preproc'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'a4Preproc_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/a4Preproc_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/a4Preproc_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/a4Preproc/a4Preproc_1.24.0_src_all.tar.gz']), ('md5', '8de73ad20a28155f34a4636c3f7c6539')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'r-base']), ('run', ['bioconductor-annotationdbi', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'a4Preproc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/a4Preproc.html'), ('license', 'GPL-3'), ('summary', 'Automated Affymetrix Array Analysis Preprocessing Package')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-a4preproc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-a4preproc'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'a4Preproc_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/a4Preproc_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/a4Preproc_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/a4Preproc/a4Preproc_1.24.0_src_all.tar.gz']), ('md5', '8de73ad20a28155f34a4636c3f7c6539')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'r-base']), ('run', ['bioconductor-annotationdbi', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'a4Preproc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/a4Preproc.html'), ('license', 'GPL-3'), ('summary', 'Automated Affymetrix Array Analysis Preprocessing Package')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/moods/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'moods'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/jhkorhonen/MOODS/releases/download/v<{ version }}/MOODS-python-<{ version }}.tar.gz'), ('fn', 'moods-<{ version }}.tar.gz'), ('md5', '40113c829ec6c699bd59978f313add1f')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'gcc']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('imports', ['MOODS.scan'])])), ('about', CommentedMap([('home', 'https://github.com/jhkorhonen/MOODS'), ('license', 'GPL-3.0'), ('summary', 'MOODS, Motif Occurrence Detection Suite')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/moods/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'moods'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/jhkorhonen/MOODS/releases/download/v<{ version }}/MOODS-python-<{ version }}.tar.gz'), ('fn', 'moods-<{ version }}.tar.gz'), ('md5', '40113c829ec6c699bd59978f313add1f')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'gcc']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('imports', ['MOODS.scan'])])), ('about', CommentedMap([('home', 'https://github.com/jhkorhonen/MOODS'), ('license', 'GPL-3.0'), ('summary', 'MOODS, Motif Occurrence Detection Suite')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metams/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metams'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'metaMS_1.8.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/metaMS_1.8.0.tar.gz'), ('md5', 'a1188b28fc2f0df985fe44bc69c2fb2b')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-camera', 'bioconductor-xcms >=1.35', 'r-base', 'r-robustbase']), ('run', ['bioconductor-biocgenerics', 'bioconductor-camera', 'bioconductor-xcms >=1.35', 'r-base', 'r-robustbase'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metaMS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/metaMS.html'), ('license', 'GPL (>= 2)'), ('summary', 'MS-based metabolomics data processing and compound annotation pipeline.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metams/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metams'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'metaMS_1.8.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/metaMS_1.8.0.tar.gz'), ('md5', 'a1188b28fc2f0df985fe44bc69c2fb2b')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-camera', 'bioconductor-xcms >=1.35', 'r-base', 'r-robustbase']), ('run', ['bioconductor-biocgenerics', 'bioconductor-camera', 'bioconductor-xcms >=1.35', 'r-base', 'r-robustbase'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metaMS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/metaMS.html'), ('license', 'GPL (>= 2)'), ('summary', 'MS-based metabolomics data processing and compound annotation pipeline.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/percolator/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'percolator'), ('version', '3.1')])), ('source', CommentedMap([('fn', 'rel-3-01.tar.gz'), ('url', 'https://github.com/percolator/percolator/archive/rel-3-01.tar.gz'), ('md5', 'fcb37bc5fcd0149cc775c3e72ff3dc24')])), ('build', CommentedMap([('number', 2), ('string', 'boost_1.62<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['cmake ==2.8.10.2', 'gcc', 'zlib', 'boost ==1.62', 'tokyocabinet', 'sqlite', 'bzip2', 'xerces-c ==3.1.2', 'xsd']), ('run', ['libgcc', 'zlib', 'tokyocabinet', 'sqlite', 'bzip2', 'xerces-c ==3.1.2', 'xsd'])])), ('test', CommentedMap([('commands', ['percolator --help 2>&1 > /dev/null | grep \"Percolator version 3.01\"', 'msgf2pin --help'])])), ('about', CommentedMap([('home', 'https://github.com/percolator/percolator'), ('license', 'Apache 2.0'), ('summary', 'Semi-supervised learning for peptide identification from shotgun proteomics datasets')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/percolator/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'percolator'), ('version', '3.1')])), ('source', CommentedMap([('fn', 'rel-3-01.tar.gz'), ('url', 'https://github.com/percolator/percolator/archive/rel-3-01.tar.gz'), ('md5', 'fcb37bc5fcd0149cc775c3e72ff3dc24')])), ('build', CommentedMap([('number', 2), ('string', 'boost_1.62<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['cmake ==2.8.10.2', 'gcc', 'zlib', 'boost ==1.62', 'tokyocabinet', 'sqlite', 'bzip2', 'xerces-c ==3.1.2', 'xsd']), ('run', ['libgcc', 'zlib', 'tokyocabinet', 'sqlite', 'bzip2', 'xerces-c ==3.1.2', 'xsd'])])), ('test', CommentedMap([('commands', ['percolator --help 2>&1 > /dev/null | grep \"Percolator version 3.01\"', 'msgf2pin --help'])])), ('about', CommentedMap([('home', 'https://github.com/percolator/percolator'), ('license', 'Apache 2.0'), ('summary', 'Semi-supervised learning for peptide identification from shotgun proteomics datasets')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-anota/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-anota'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'anota_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/anota_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/anota_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/anota/anota_1.24.0_src_all.tar.gz']), ('md5', '76b4cfb1482f0b3f84ec08389bce1599')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-multtest', 'bioconductor-qvalue', 'r-base']), ('run', ['bioconductor-multtest', 'bioconductor-qvalue', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'anota\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/anota.html'), ('license', 'GPL-3'), ('summary', 'Genome wide studies of translational control is emerging as a tool to study verious biological conditions. The output from such analysis is both the mRNA level (e.g. cytosolic mRNA level) and the levl of mRNA actively involved in translation (the actively translating mRNA level) for each mRNA. The standard analysis of such data strives towards identifying differential translational between two or more sample classes - i.e. differences in actively translated mRNA levels that are independent of underlying differences in cytosolic mRNA levels. This package allows for such analysis using partial variances and the random variance model. As 10s of thousands of mRNAs are analyzed in parallell the library performs a number of tests to assure that the data set is suitable for such analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-anota/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-anota'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'anota_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/anota_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/anota_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/anota/anota_1.24.0_src_all.tar.gz']), ('md5', '76b4cfb1482f0b3f84ec08389bce1599')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-multtest', 'bioconductor-qvalue', 'r-base']), ('run', ['bioconductor-multtest', 'bioconductor-qvalue', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'anota\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/anota.html'), ('license', 'GPL-3'), ('summary', 'Genome wide studies of translational control is emerging as a tool to study verious biological conditions. The output from such analysis is both the mRNA level (e.g. cytosolic mRNA level) and the levl of mRNA actively involved in translation (the actively translating mRNA level) for each mRNA. The standard analysis of such data strives towards identifying differential translational between two or more sample classes - i.e. differences in actively translated mRNA levels that are independent of underlying differences in cytosolic mRNA levels. This package allows for such analysis using partial variances and the random variance model. As 10s of thousands of mRNAs are analyzed in parallell the library performs a number of tests to assure that the data set is suitable for such analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gemini/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/arq5x/gemini/archive/v<{ version }}.tar.gz'), ('md5', 'd703e6f4b3b29e389903b94cac9b20a3')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'bcolz', 'bottle', 'bx-python', 'cyordereddict', 'cython', 'cyvcf2 >0.6.5', 'geneimpacts', 'inheritance', 'ipyparallel', 'ipython-cluster-helper', 'jinja2', 'networkx', 'numexpr', 'numpy', 'openpyxl', 'pandas', 'pybedtools', 'pysam', 'pyyaml', 'scipy', 'unidecode', 'snappy', 'python-snappy']), ('run', ['python', 'bcolz', 'bottle', 'bx-python', 'cyordereddict', 'cython', 'cyvcf2 >0.6.5', 'geneimpacts', 'inheritance', 'ipyparallel', 'ipython-cluster-helper', 'jinja2', 'networkx', 'numexpr', 'numpy', 'sqlalchemy', 'openpyxl', 'pandas', 'pybedtools', 'pysam', 'pyyaml', 'scipy', 'unidecode', 'grabix', 'snappy', 'python-snappy'])])), ('test', CommentedMap([('commands', ['gemini -h']), ('imports', ['gemini'])])), ('about', CommentedMap([('home', 'https://github.com/arq5x/gemini'), ('license', 'MIT License'), ('summary', 'a lightweight db framework for disease and population genetics.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gemini/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/arq5x/gemini/archive/v<{ version }}.tar.gz'), ('md5', 'd703e6f4b3b29e389903b94cac9b20a3')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'bcolz', 'bottle', 'bx-python', 'cyordereddict', 'cython', 'cyvcf2 >0.6.5', 'geneimpacts', 'inheritance', 'ipyparallel', 'ipython-cluster-helper', 'jinja2', 'networkx', 'numexpr', 'numpy', 'openpyxl', 'pandas', 'pybedtools', 'pysam', 'pyyaml', 'scipy', 'unidecode', 'snappy', 'python-snappy']), ('run', ['python', 'bcolz', 'bottle', 'bx-python', 'cyordereddict', 'cython', 'cyvcf2 >0.6.5', 'geneimpacts', 'inheritance', 'ipyparallel', 'ipython-cluster-helper', 'jinja2', 'networkx', 'numexpr', 'numpy', 'sqlalchemy', 'openpyxl', 'pandas', 'pybedtools', 'pysam', 'pyyaml', 'scipy', 'unidecode', 'grabix', 'snappy', 'python-snappy'])])), ('test', CommentedMap([('commands', ['gemini -h']), ('imports', ['gemini'])])), ('about', CommentedMap([('home', 'https://github.com/arq5x/gemini'), ('license', 'MIT License'), ('summary', 'a lightweight db framework for disease and population genetics.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pear/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pear'), ('version', '0.9.6')])), ('about', CommentedMap([('home', 'http://sco.h-its.org/exelixis/web/software/pear/'), ('license', 'Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported'), ('summary', 'paired-end read merger')])), ('source', CommentedMap([('fn', 'pear-0.9.6-src.tar.gz'), ('md5', '5ed68d50c1620cd55c58681e38771799'), ('url', 'http://sco.h-its.org/exelixis/web/software/pear/files/pear-0.9.6-src.tar.gz')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'pkgconfig', 'zlib', 'bzip2']), ('run', ['libgcc', 'zlib', 'bzip2'])])), ('test', CommentedMap([('commands', [\"pear 2>&1 | grep 'PEAR v0.9.6'\", \"pearRM 2>&1 | grep 'PEAR v0.9.6'\"])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pear/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pear'), ('version', '0.9.6')])), ('about', CommentedMap([('home', 'http://sco.h-its.org/exelixis/web/software/pear/'), ('license', 'Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported'), ('summary', 'paired-end read merger')])), ('source', CommentedMap([('fn', 'pear-0.9.6-src.tar.gz'), ('md5', '5ed68d50c1620cd55c58681e38771799'), ('url', 'http://sco.h-its.org/exelixis/web/software/pear/files/pear-0.9.6-src.tar.gz')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'pkgconfig', 'zlib', 'bzip2']), ('run', ['libgcc', 'zlib', 'bzip2'])])), ('test', CommentedMap([('commands', [\"pear 2>&1 | grep 'PEAR v0.9.6'\", \"pearRM 2>&1 | grep 'PEAR v0.9.6'\"])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnalien/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/eggzilla/RNAlien/archive/<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('requirements', CommentedMap([('build', ['stack >=1.3.2', 'gmp', 'zlib', 'perl']), ('run', ['viennarna ==2.3.3', 'rnaz ==2.1', 'locarna ==1.9.1', 'infernal ==1.1.2', 'rnacode ==0.3', 'gmp', 'perl', 'libgcc', 'zlib', 'openssl'])])), ('test', CommentedMap([('commands', ['RNAlien --help'])])), ('about', CommentedMap([('home', 'http://rna.tbi.univie.ac.at/rnalien/tool'), ('license', 'GPL-3'), ('summary', 'A tool for unsupervised construction of RNA family models')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnalien/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/eggzilla/RNAlien/archive/<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('requirements', CommentedMap([('build', ['stack >=1.3.2', 'gmp', 'zlib', 'perl']), ('run', ['viennarna ==2.3.3', 'rnaz ==2.1', 'locarna ==1.9.1', 'infernal ==1.1.2', 'rnacode ==0.3', 'gmp', 'perl', 'libgcc', 'zlib', 'openssl'])])), ('test', CommentedMap([('commands', ['RNAlien --help'])])), ('about', CommentedMap([('home', 'http://rna.tbi.univie.ac.at/rnalien/tool'), ('license', 'GPL-3'), ('summary', 'A tool for unsupervised construction of RNA family models')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-medips/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-medips'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'MEDIPS_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/MEDIPS_1.24.0.tar.gz'), ('md5', '29d4cdec874a39a6b9c255fc5e634d49')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-preprocesscore', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-gtools']), ('run', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-preprocesscore', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-gtools'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MEDIPS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/MEDIPS.html'), ('license', 'GPL (>=2)'), ('summary', 'MEDIPS was developed for analyzing data derived from methylated DNA immunoprecipitation (MeDIP) experiments followed by sequencing (MeDIP-seq). However, MEDIPS provides functionalities for the analysis of any kind of quantitative sequencing data (e.g. ChIP-seq, MBD-seq, CMS-seq and others) including calculation of differential coverage between groups of samples and saturation and correlation analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-medips/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-medips'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'MEDIPS_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/MEDIPS_1.24.0.tar.gz'), ('md5', '29d4cdec874a39a6b9c255fc5e634d49')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-preprocesscore', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-gtools']), ('run', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-preprocesscore', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-gtools'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MEDIPS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/MEDIPS.html'), ('license', 'GPL (>=2)'), ('summary', 'MEDIPS was developed for analyzing data derived from methylated DNA immunoprecipitation (MeDIP) experiments followed by sequencing (MeDIP-seq). However, MEDIPS provides functionalities for the analysis of any kind of quantitative sequencing data (e.g. ChIP-seq, MBD-seq, CMS-seq and others) including calculation of differential coverage between groups of samples and saturation and correlation analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-plgem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-plgem'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'plgem_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/plgem_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/plgem_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/plgem/plgem_1.48.0_src_all.tar.gz']), ('md5', 'c501ba945aba8b65309f38785925cab0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-mass']), ('run', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'plgem\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/plgem.html'), ('license', 'GPL-2'), ('summary', 'The Power Law Global Error Model (PLGEM) has been shown to faithfully model the variance-versus-mean dependence that exists in a variety of genome-wide datasets, including microarray and proteomics data. The use of PLGEM has been shown to improve the detection of differentially expressed genes or proteins in these datasets.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-plgem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-plgem'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'plgem_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/plgem_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/plgem_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/plgem/plgem_1.48.0_src_all.tar.gz']), ('md5', 'c501ba945aba8b65309f38785925cab0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-mass']), ('run', ['bioconductor-biobase >=2.5.5', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'plgem\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/plgem.html'), ('license', 'GPL-2'), ('summary', 'The Power Law Global Error Model (PLGEM) has been shown to faithfully model the variance-versus-mean dependence that exists in a variety of genome-wide datasets, including microarray and proteomics data. The use of PLGEM has been shown to improve the detection of differentially expressed genes or proteins in these datasets.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ogsa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ogsa'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'OGSA_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OGSA_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/OGSA_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/OGSA/OGSA_1.6.0_src_all.tar.gz']), ('md5', '146f36f67ca15d7958556fd3ff67621e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma >=3.18.13', 'r-base', 'r-gplots >=2.8.0']), ('run', ['bioconductor-biobase', 'bioconductor-limma >=3.18.13', 'r-base', 'r-gplots >=2.8.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OGSA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OGSA.html'), ('license', 'GPL (== 2)'), ('summary', 'OGSA provides a global estimate of pathway deregulation in cancer subtypes by integrating the estimates of significance for individual pathway members that have been identified by outlier analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ogsa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ogsa'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'OGSA_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OGSA_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/OGSA_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/OGSA/OGSA_1.6.0_src_all.tar.gz']), ('md5', '146f36f67ca15d7958556fd3ff67621e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma >=3.18.13', 'r-base', 'r-gplots >=2.8.0']), ('run', ['bioconductor-biobase', 'bioconductor-limma >=3.18.13', 'r-base', 'r-gplots >=2.8.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OGSA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OGSA.html'), ('license', 'GPL (== 2)'), ('summary', 'OGSA provides a global estimate of pathway deregulation in cancer subtypes by integrating the estimates of significance for individual pathway members that have been identified by outlier analysis.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgadem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgadem'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'rGADEM_2.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/rGADEM_2.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/rGADEM_2.22.0.tar.gz', 'https://depot.galaxyproject.org/software/rGADEM/rGADEM_2.22.0_src_all.tar.gz']), ('md5', '68855b9cecf4cb029027b6df0a0b6b2d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-iranges', 'bioconductor-seqlogo', 'r-base']), ('run', ['bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-iranges', 'bioconductor-seqlogo', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rGADEM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rGADEM.html'), ('license', 'Artistic-2.0'), ('summary', 'rGADEM is an efficient de novo motif discovery tool for large-scale genomic sequence data. It is an open-source R package, which is based on the GADEM software.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgadem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgadem'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'rGADEM_2.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/rGADEM_2.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/rGADEM_2.22.0.tar.gz', 'https://depot.galaxyproject.org/software/rGADEM/rGADEM_2.22.0_src_all.tar.gz']), ('md5', '68855b9cecf4cb029027b6df0a0b6b2d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-iranges', 'bioconductor-seqlogo', 'r-base']), ('run', ['bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-iranges', 'bioconductor-seqlogo', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rGADEM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rGADEM.html'), ('license', 'Artistic-2.0'), ('summary', 'rGADEM is an efficient de novo motif discovery tool for large-scale genomic sequence data. It is an open-source R package, which is based on the GADEM software.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-graph-readwrite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-graph-readwrite'), ('version', '2.09')])), ('source', CommentedMap([('fn', 'Graph-ReadWrite-2.09.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NE/NEILB/Graph-ReadWrite-2.09.tar.gz'), ('md5', '5cd9191eadd2fe8fe8bb431575434f67')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-graph', 'perl-xml-parser', 'perl-xml-writer', 'perl-parse-yapp']), ('run', ['perl-threaded', 'perl-graph', 'perl-xml-parser', 'perl-xml-writer', 'perl-parse-yapp'])])), ('test', CommentedMap([('imports', ['Graph::Writer::Dot'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Graph::Writer::Dot'), ('license', 'perl_5'), ('summary', 'Write out directed graph in Dot format')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-graph-readwrite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-graph-readwrite'), ('version', '2.09')])), ('source', CommentedMap([('fn', 'Graph-ReadWrite-2.09.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/N/NE/NEILB/Graph-ReadWrite-2.09.tar.gz'), ('md5', '5cd9191eadd2fe8fe8bb431575434f67')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-graph', 'perl-xml-parser', 'perl-xml-writer', 'perl-parse-yapp']), ('run', ['perl-threaded', 'perl-graph', 'perl-xml-parser', 'perl-xml-writer', 'perl-parse-yapp'])])), ('test', CommentedMap([('imports', ['Graph::Writer::Dot'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Graph::Writer::Dot'), ('license', 'perl_5'), ('summary', 'Write out directed graph in Dot format')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ray/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Ray-<{ version }}.tar.bz2'), ('url', 'https://downloads.sourceforge.net/project/denovoassembler/Ray-<{ version }}.tar.bz2'), ('md5', '82f693c4db60af4328263c9279701009')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'openmpi >=2.0.2', 'cmake']), ('run', ['libgcc', 'openmpi >=2.0.2'])])), ('test', CommentedMap([('commands', ['which Ray'])])), ('about', CommentedMap([('home', 'http://denovoassembler.sourceforge.net/index.html'), ('license', 'GPL3'), ('summary', 'Parallel genome assemblies for parallel DNA sequencing')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ray/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Ray-<{ version }}.tar.bz2'), ('url', 'https://downloads.sourceforge.net/project/denovoassembler/Ray-<{ version }}.tar.bz2'), ('md5', '82f693c4db60af4328263c9279701009')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'openmpi >=2.0.2', 'cmake']), ('run', ['libgcc', 'openmpi >=2.0.2'])])), ('test', CommentedMap([('commands', ['which Ray'])])), ('about', CommentedMap([('home', 'http://denovoassembler.sourceforge.net/index.html'), ('license', 'GPL3'), ('summary', 'Parallel genome assemblies for parallel DNA sequencing')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-synergyfinder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-synergyfinder'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'synergyfinder_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/synergyfinder_1.0.0.tar.gz'), ('md5', '2b282463638871a0074fe6cb829d112a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-drc >=2.5-12', 'r-ggplot2 >=2.1.0', 'r-gplots >=3.0.0', 'r-gridbase >=0.4-7', 'r-kriging >=1.1', 'r-nleqslv >=3.0', 'r-reshape2 >=1.4.1', 'r-nloptr']), ('run', ['r-base', 'r-drc >=2.5-12', 'r-ggplot2 >=2.1.0', 'r-gplots >=3.0.0', 'r-gridbase >=0.4-7', 'r-kriging >=1.1', 'r-nleqslv >=3.0', 'r-reshape2 >=1.4.1', 'r-nloptr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'synergyfinder\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/synergyfinder.html'), ('license', 'Artistic License 2.0'), ('summary', 'Efficient implementations for all the popular synergy scoring models for drug combinations, including HSA, Loewe, Bliss and ZIP and visualization of the synergy scores as either a two-dimensional or a three-dimensional interaction surface over the dose matrix.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-synergyfinder/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-synergyfinder'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'synergyfinder_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/synergyfinder_1.0.0.tar.gz'), ('md5', '2b282463638871a0074fe6cb829d112a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-drc >=2.5-12', 'r-ggplot2 >=2.1.0', 'r-gplots >=3.0.0', 'r-gridbase >=0.4-7', 'r-kriging >=1.1', 'r-nleqslv >=3.0', 'r-reshape2 >=1.4.1', 'r-nloptr']), ('run', ['r-base', 'r-drc >=2.5-12', 'r-ggplot2 >=2.1.0', 'r-gplots >=3.0.0', 'r-gridbase >=0.4-7', 'r-kriging >=1.1', 'r-nleqslv >=3.0', 'r-reshape2 >=1.4.1', 'r-nloptr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'synergyfinder\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/synergyfinder.html'), ('license', 'Artistic License 2.0'), ('summary', 'Efficient implementations for all the popular synergy scoring models for drug combinations, including HSA, Loewe, Bliss and ZIP and visualization of the synergy scores as either a two-dimensional or a three-dimensional interaction surface over the dose matrix.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-microrna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-microrna'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'microRNA_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/microRNA_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/microRNA_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/microRNA/microRNA_1.34.0_src_all.tar.gz']), ('md5', '3760e1ec8b1830541621a9f98acb2d9d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.11.32', 'r-base']), ('run', ['bioconductor-biostrings >=2.11.32', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'microRNA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/microRNA.html'), ('license', 'Artistic-2.0'), ('summary', 'Different data resources for microRNAs and some functions for manipulating them.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-microrna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-microrna'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'microRNA_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/microRNA_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/microRNA_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/microRNA/microRNA_1.34.0_src_all.tar.gz']), ('md5', '3760e1ec8b1830541621a9f98acb2d9d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.11.32', 'r-base']), ('run', ['bioconductor-biostrings >=2.11.32', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'microRNA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/microRNA.html'), ('license', 'Artistic-2.0'), ('summary', 'Different data resources for microRNAs and some functions for manipulating them.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtrm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtrm'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'rTRM_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rTRM_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/rTRM_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/rTRM/rTRM_1.14.0_src_all.tar.gz']), ('md5', '3c7c12c8294ca635073deff5caeb8712')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'r-base', 'r-dbi', 'r-igraph >=1.0', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi', 'r-base', 'r-dbi', 'r-igraph >=1.0', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rTRM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rTRM.html'), ('license', 'GPL-3'), ('summary', 'rTRM identifies transcriptional regulatory modules (TRMs) from protein-protein interaction networks.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtrm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtrm'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'rTRM_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rTRM_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/rTRM_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/rTRM/rTRM_1.14.0_src_all.tar.gz']), ('md5', '3c7c12c8294ca635073deff5caeb8712')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'r-base', 'r-dbi', 'r-igraph >=1.0', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi', 'r-base', 'r-dbi', 'r-igraph >=1.0', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rTRM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rTRM.html'), ('license', 'GPL-3'), ('summary', 'rTRM identifies transcriptional regulatory modules (TRMs) from protein-protein interaction networks.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-variantannotation/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-variantannotation'), ('version', '1.20.3')])), ('source', CommentedMap([('fn', 'VariantAnnotation_1.20.3.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/VariantAnnotation_1.20.3.tar.gz'), ('md5', 'a16f2e9d24eb4c19607c2e5fcd1d5ea3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.27.9', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings >=2.33.5', 'bioconductor-bsgenome >=1.37.6', 'bioconductor-genomeinfodb >=1.7.1', 'bioconductor-genomicfeatures >=1.19.17', 'bioconductor-genomicranges >=1.19.47', 'bioconductor-iranges >=2.3.25', 'bioconductor-rsamtools >=1.23.10', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-summarizedexperiment >=0.3.1', 'bioconductor-xvector >=0.5.6', 'bioconductor-zlibbioc', 'r-base', 'r-dbi']), ('run', ['bioconductor-annotationdbi >=1.27.9', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings >=2.33.5', 'bioconductor-bsgenome >=1.37.6', 'bioconductor-genomeinfodb >=1.7.1', 'bioconductor-genomicfeatures >=1.19.17', 'bioconductor-genomicranges >=1.19.47', 'bioconductor-iranges >=2.3.25', 'bioconductor-rsamtools >=1.23.10', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-summarizedexperiment >=0.3.1', 'bioconductor-xvector >=0.5.6', 'bioconductor-zlibbioc', 'r-base', 'r-dbi'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'VariantAnnotation\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html'), ('license', 'Artistic-2.0'), ('summary', 'Annotate variants, compute amino acid coding changes, predict coding outcomes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-variantannotation/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-variantannotation'), ('version', '1.20.3')])), ('source', CommentedMap([('fn', 'VariantAnnotation_1.20.3.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/VariantAnnotation_1.20.3.tar.gz'), ('md5', 'a16f2e9d24eb4c19607c2e5fcd1d5ea3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.27.9', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings >=2.33.5', 'bioconductor-bsgenome >=1.37.6', 'bioconductor-genomeinfodb >=1.7.1', 'bioconductor-genomicfeatures >=1.19.17', 'bioconductor-genomicranges >=1.19.47', 'bioconductor-iranges >=2.3.25', 'bioconductor-rsamtools >=1.23.10', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-summarizedexperiment >=0.3.1', 'bioconductor-xvector >=0.5.6', 'bioconductor-zlibbioc', 'r-base', 'r-dbi']), ('run', ['bioconductor-annotationdbi >=1.27.9', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings >=2.33.5', 'bioconductor-bsgenome >=1.37.6', 'bioconductor-genomeinfodb >=1.7.1', 'bioconductor-genomicfeatures >=1.19.17', 'bioconductor-genomicranges >=1.19.47', 'bioconductor-iranges >=2.3.25', 'bioconductor-rsamtools >=1.23.10', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-summarizedexperiment >=0.3.1', 'bioconductor-xvector >=0.5.6', 'bioconductor-zlibbioc', 'r-base', 'r-dbi'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'VariantAnnotation\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html'), ('license', 'Artistic-2.0'), ('summary', 'Annotate variants, compute amino acid coding changes, predict coding outcomes.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-velvetoptimiser/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-velvetoptimiser'), ('version', '2.2.5')])), ('source', CommentedMap([('fn', '2.2.5.tar.gz'), ('url', 'https://github.com/tseemann/VelvetOptimiser/archive/2.2.5.tar.gz'), ('md5', '66966ce03f61321a2f81a5cfb45a3be9'), ('patches', ['perl_path.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded', 'perl-bioperl >=1.4', 'velvet >=0.7.51'])])), ('test', CommentedMap([('commands', ['VelvetOptimiser.pl --version > /dev/null 2>&1'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/VelvetOptimiser'), ('license', 'GPLv2'), ('summary', \"Automatically optimise three of Velvet's assembly parameters.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-velvetoptimiser/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-velvetoptimiser'), ('version', '2.2.5')])), ('source', CommentedMap([('fn', '2.2.5.tar.gz'), ('url', 'https://github.com/tseemann/VelvetOptimiser/archive/2.2.5.tar.gz'), ('md5', '66966ce03f61321a2f81a5cfb45a3be9'), ('patches', ['perl_path.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded', 'perl-bioperl >=1.4', 'velvet >=0.7.51'])])), ('test', CommentedMap([('commands', ['VelvetOptimiser.pl --version > /dev/null 2>&1'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/VelvetOptimiser'), ('license', 'GPLv2'), ('summary', \"Automatically optimise three of Velvet's assembly parameters.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-isomirs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-isomirs'), ('version', '1.0.3')])), ('source', CommentedMap([('fn', 'isomiRs_1.0.3.tar.gz'), ('url', 'http://www.imsbio.co.jp/RGM-files/R_BC/download/isomiRs_1.0.3.tar.gz'), ('md5', '36669b2e85ce69a0f168afe55e11c928')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.7.5', 'bioconductor-deseq2', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-discriminer', 'r-dplyr', 'r-ggally', 'r-ggplot2', 'r-gplots', 'r-plyr', 'r-rcolorbrewer']), ('run', ['bioconductor-biocgenerics >=0.7.5', 'bioconductor-deseq2', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-discriminer', 'r-dplyr', 'r-ggally', 'r-ggplot2', 'r-gplots', 'r-plyr', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'isomiRs\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/isomiRs.html'), ('license', 'MIT + file LICENSE'), ('summary', 'Characterization of miRNAs and isomiRs, clustering and differential expression.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-isomirs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-isomirs'), ('version', '1.0.3')])), ('source', CommentedMap([('fn', 'isomiRs_1.0.3.tar.gz'), ('url', 'http://www.imsbio.co.jp/RGM-files/R_BC/download/isomiRs_1.0.3.tar.gz'), ('md5', '36669b2e85ce69a0f168afe55e11c928')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.7.5', 'bioconductor-deseq2', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-discriminer', 'r-dplyr', 'r-ggally', 'r-ggplot2', 'r-gplots', 'r-plyr', 'r-rcolorbrewer']), ('run', ['bioconductor-biocgenerics >=0.7.5', 'bioconductor-deseq2', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-discriminer', 'r-dplyr', 'r-ggally', 'r-ggplot2', 'r-gplots', 'r-plyr', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'isomiRs\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/isomiRs.html'), ('license', 'MIT + file LICENSE'), ('summary', 'Characterization of miRNAs and isomiRs, clustering and differential expression.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-oppar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-oppar'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'oppar_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/oppar_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/oppar_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/oppar/oppar_1.4.0_src_all.tar.gz']), ('md5', 'de85939cfe9df6f69cf4c72792ed4445')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-gseabase', 'bioconductor-gsva', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-gseabase', 'bioconductor-gsva', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'oppar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/oppar.html'), ('license', 'GPL-2'), ('summary', 'The R implementation of mCOPA package published by Wang et al. (2012). Oppar provides methods for Cancer Outlier profile Analysis. Although initially developed to detect outlier genes in cancer studies, methods presented in oppar can be used for outlier profile analysis in general. In addition, tools are provided for gene set enrichment and pathway analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-oppar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-oppar'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'oppar_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/oppar_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/oppar_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/oppar/oppar_1.4.0_src_all.tar.gz']), ('md5', 'de85939cfe9df6f69cf4c72792ed4445')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-gseabase', 'bioconductor-gsva', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-gseabase', 'bioconductor-gsva', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'oppar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/oppar.html'), ('license', 'GPL-2'), ('summary', 'The R implementation of mCOPA package published by Wang et al. (2012). Oppar provides methods for Cancer Outlier profile Analysis. Although initially developed to detect outlier genes in cancer studies, methods presented in oppar can be used for outlier profile analysis in general. In addition, tools are provided for gene set enrichment and pathway analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-acme/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-acme'), ('version', '2.32.0')])), ('source', CommentedMap([('fn', 'ACME_2.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ACME_2.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/ACME_2.32.0.tar.gz', 'https://depot.galaxyproject.org/software/ACME/ACME_2.32.0_src_all.tar.gz']), ('md5', 'f99ea6b94399fd7a10f55ac7e7ec04fa')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'r-base']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ACME\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ACME.html'), ('license', 'GPL (>= 2)'), ('summary', 'ACME (Algorithms for Calculating Microarray Enrichment) is a set of tools for analysing tiling array ChIP/chip, DNAse hypersensitivity, or other experiments that result in regions of the genome showing \"enrichment\".  It does not rely on a specific array technology (although the array should be a \"tiling\" array), is very general (can be applied in experiments resulting in regions of enrichment), and is very insensitive to array noise or normalization methods.  It is also very fast and can be applied on whole-genome tiling array experiments quite easily with enough memory.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-acme/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-acme'), ('version', '2.32.0')])), ('source', CommentedMap([('fn', 'ACME_2.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ACME_2.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/ACME_2.32.0.tar.gz', 'https://depot.galaxyproject.org/software/ACME/ACME_2.32.0_src_all.tar.gz']), ('md5', 'f99ea6b94399fd7a10f55ac7e7ec04fa')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'r-base']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ACME\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ACME.html'), ('license', 'GPL (>= 2)'), ('summary', 'ACME (Algorithms for Calculating Microarray Enrichment) is a set of tools for analysing tiling array ChIP/chip, DNAse hypersensitivity, or other experiments that result in regions of the genome showing \"enrichment\".  It does not rely on a specific array technology (although the array should be a \"tiling\" array), is very general (can be applied in experiments resulting in regions of enrichment), and is very insensitive to array noise or normalization methods.  It is also very fast and can be applied on whole-genome tiling array experiments quite easily with enough memory.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-snpstats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-snpstats'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'snpStats_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/snpStats_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/snpStats_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/snpStats/snpStats_1.26.0_src_all.tar.gz']), ('md5', '14404c3e01849fa7836f704be8a085cd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-matrix', 'r-survival']), ('run', ['bioconductor-biocgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-matrix', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'snpStats\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/snpStats.html'), ('license', 'GPL-3'), ('summary', 'Classes and statistical methods for large SNP association studies. This extends the earlier snpMatrix package, allowing for uncertainty in genotypes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-snpstats/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-snpstats'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'snpStats_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/snpStats_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/snpStats_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/snpStats/snpStats_1.26.0_src_all.tar.gz']), ('md5', '14404c3e01849fa7836f704be8a085cd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-matrix', 'r-survival']), ('run', ['bioconductor-biocgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-matrix', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'snpStats\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/snpStats.html'), ('license', 'GPL-3'), ('summary', 'Classes and statistical methods for large SNP association studies. This extends the earlier snpMatrix package, allowing for uncertainty in genotypes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gatk/3.5/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://www.broadinstitute.org/gatk/'), ('license', 'https://www.broadinstitute.org/gatk/about/#licensing'), ('summary', 'The full Genome Analysis Toolkit (GATK) framework, license restricted.')])), ('package', CommentedMap([('name', 'gatk'), ('version', '3.5')])), ('build', CommentedMap([('number', 5), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk >=8,<9', 'bzip2'])])), ('extra', CommentedMap([('notes', 'Due to license restrictions, this recipe cannot distribute and install GATK directly. To fully install GATK, you must download a licensed copy of GATK from the Broad Institute (https://www.broadinstitute.org/gatk/download/), install this package, and call \"gatk-register /path/to/GenomeAnalysisTK[-$PKG_VERSION.tar.bz2|.jar]\", which will copy GATK into your conda environment.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gatk/3.5/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://www.broadinstitute.org/gatk/'), ('license', 'https://www.broadinstitute.org/gatk/about/#licensing'), ('summary', 'The full Genome Analysis Toolkit (GATK) framework, license restricted.')])), ('package', CommentedMap([('name', 'gatk'), ('version', '3.5')])), ('build', CommentedMap([('number', 5), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk >=8,<9', 'bzip2'])])), ('extra', CommentedMap([('notes', 'Due to license restrictions, this recipe cannot distribute and install GATK directly. To fully install GATK, you must download a licensed copy of GATK from the Broad Institute (https://www.broadinstitute.org/gatk/download/), install this package, and call \"gatk-register /path/to/GenomeAnalysisTK[-$PKG_VERSION.tar.bz2|.jar]\", which will copy GATK into your conda environment.')]))]))\n",
      "EXTRA has been found: gatk\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/treemix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'treemix'), ('version', 1.12)])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}__gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'treemix-1.12.tar.gz'), ('url', 'https://bitbucket.org/nygcresearch/treemix/downloads/treemix-1.12.tar.gz'), ('md5', '183f605751a0fa445e249db840907104')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'gsl <{CONDA_GSL}}*', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'zlib', 'gsl <{CONDA_GSL}}*'])])), ('about', CommentedMap([('home', 'http://pritchardlab.stanford.edu/software.html'), ('license', 'GPLv3'), ('summary', 'TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a set of populations')])), ('test', CommentedMap([('commands', ['treemix --help | grep 1.12'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/treemix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'treemix'), ('version', 1.12)])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}__gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'treemix-1.12.tar.gz'), ('url', 'https://bitbucket.org/nygcresearch/treemix/downloads/treemix-1.12.tar.gz'), ('md5', '183f605751a0fa445e249db840907104')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'gsl <{CONDA_GSL}}*', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'zlib', 'gsl <{CONDA_GSL}}*'])])), ('about', CommentedMap([('home', 'http://pritchardlab.stanford.edu/software.html'), ('license', 'GPLv3'), ('summary', 'TreeMix is a method for inferring the patterns of population splits and mixtures in the history of a set of populations')])), ('test', CommentedMap([('commands', ['treemix --help | grep 1.12'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/astalavista/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'astalavista'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'astalavista-<{ version }}.tgz'), ('url', 'http://sammeth.net/data/public/astalavista-<{ version }}.tgz'), ('sha256', '46c61e00ddf3427f2646046060708ffd05ecc4610509580fdfda53765f6a2eb2')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['astalavista --help 2>&1 | grep \"List of available tools\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://sammeth.net/confluence/display/ASTA/Home'), ('license', 'BSD'), ('summary', 'AStalavista is a computer program to extract alternative splicing (AS) events from a given genomic annotation of exon-intron gene coordinates. By comparing all given transcripts, AStalavista detects the variations in their splicing structure and identify all AS events (like exon skipping, alternate donor, etc) by assigning to each of them an AS code.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/astalavista/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'astalavista'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'astalavista-<{ version }}.tgz'), ('url', 'http://sammeth.net/data/public/astalavista-<{ version }}.tgz'), ('sha256', '46c61e00ddf3427f2646046060708ffd05ecc4610509580fdfda53765f6a2eb2')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['astalavista --help 2>&1 | grep \"List of available tools\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://sammeth.net/confluence/display/ASTA/Home'), ('license', 'BSD'), ('summary', 'AStalavista is a computer program to extract alternative splicing (AS) events from a given genomic annotation of exon-intron gene coordinates. By comparing all given transcripts, AStalavista detects the variations in their splicing structure and identify all AS events (like exon skipping, alternate donor, etc) by assigning to each of them an AS code.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bayespeak/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bayespeak'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'BayesPeak_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BayesPeak_1.24.0.tar.gz'), ('md5', '821ed8cbee83a5b8dfd7530306d9e3ee')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-iranges', 'r-base >=2.14', 'gcc', 'llvm']), ('run', ['bioconductor-iranges', 'r-base >=2.14', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BayesPeak\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BayesPeak.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package is an implementation of the BayesPeak algorithm for peak-calling in ChIP-seq data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bayespeak/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bayespeak'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'BayesPeak_1.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BayesPeak_1.24.0.tar.gz'), ('md5', '821ed8cbee83a5b8dfd7530306d9e3ee')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-iranges', 'r-base >=2.14', 'gcc', 'llvm']), ('run', ['bioconductor-iranges', 'r-base >=2.14', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BayesPeak\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BayesPeak.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package is an implementation of the BayesPeak algorithm for peak-calling in ChIP-seq data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sscore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sscore'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'sscore_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sscore_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/sscore_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/sscore/sscore_1.48.0_src_all.tar.gz']), ('md5', 'af130ad415483c52c333ca179d6b1b60')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-affyio', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-affyio', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sscore\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sscore.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package contains an implementation of the S-Score algorithm as described by Zhang et al (2002).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sscore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sscore'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'sscore_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sscore_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/sscore_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/sscore/sscore_1.48.0_src_all.tar.gz']), ('md5', 'af130ad415483c52c333ca179d6b1b60')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-affyio', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-affyio', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sscore\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sscore.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package contains an implementation of the S-Score algorithm as described by Zhang et al (2002).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-spia/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-spia'), ('version', '2.28.0')])), ('source', CommentedMap([('fn', 'SPIA_2.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SPIA_2.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/SPIA_2.28.0.tar.gz', 'https://depot.galaxyproject.org/software/SPIA/SPIA_2.28.0_src_all.tar.gz']), ('md5', '5aff2977b9b84cf4fa21de1d8325c8d0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-kegggraph', 'r-base']), ('run', ['bioconductor-kegggraph', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SPIA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SPIA.html'), ('license', 'file LICENSE'), ('summary', 'This package implements the Signaling Pathway Impact Analysis (SPIA) which uses the information form a list of differentially expressed genes and their log fold changes together with signaling pathways topology, in order to identify the pathways most relevant to the condition under the study.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-spia/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-spia'), ('version', '2.28.0')])), ('source', CommentedMap([('fn', 'SPIA_2.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SPIA_2.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/SPIA_2.28.0.tar.gz', 'https://depot.galaxyproject.org/software/SPIA/SPIA_2.28.0_src_all.tar.gz']), ('md5', '5aff2977b9b84cf4fa21de1d8325c8d0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-kegggraph', 'r-base']), ('run', ['bioconductor-kegggraph', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SPIA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SPIA.html'), ('license', 'file LICENSE'), ('summary', 'This package implements the Signaling Pathway Impact Analysis (SPIA) which uses the information form a list of differentially expressed genes and their log fold changes together with signaling pathways topology, in order to identify the pathways most relevant to the condition under the study.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bacon/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bacon'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'bacon_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/bacon_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/bacon_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/bacon/bacon_1.4.0_src_all.tar.gz']), ('md5', 'f43327ba3b4e0a8871e047ad6fc9ae88')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'r-base', 'r-ellipse', 'r-ggplot2']), ('run', ['bioconductor-biocparallel', 'r-base', 'r-ellipse', 'r-ggplot2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'bacon\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/bacon.html'), ('license', 'GPL (>= 2)'), ('summary', 'Bacon can be used to remove inflation and bias often observed in epigenome- and transcriptome-wide association studies. To this end bacon constructs an empirical null distribution using a Gibbs Sampling algorithm by fitting a three-component normal mixture on z-scores.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bacon/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bacon'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'bacon_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/bacon_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/bacon_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/bacon/bacon_1.4.0_src_all.tar.gz']), ('md5', 'f43327ba3b4e0a8871e047ad6fc9ae88')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'r-base', 'r-ellipse', 'r-ggplot2']), ('run', ['bioconductor-biocparallel', 'r-base', 'r-ellipse', 'r-ggplot2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'bacon\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/bacon.html'), ('license', 'GPL (>= 2)'), ('summary', 'Bacon can be used to remove inflation and bias often observed in epigenome- and transcriptome-wide association studies. To this end bacon constructs an empirical null distribution using a Gibbs Sampling algorithm by fitting a three-component normal mixture on z-scores.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/inforna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'inforna'), ('version', '2.1.2')])), ('build', CommentedMap([('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/BackofenLab/INFO-RNA/archive/1e9bb8236c13f5861f07dc19625c394aaeae4c79.tar.gz'), ('md5', '219ddf683b78c1c314d896c1b367847e')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'viennarna']), ('run', ['libgcc', 'viennarna'])])), ('test', CommentedMap([('commands', ['INFO-RNA -h', 'INFO-RNA \"(...)\"'])])), ('about', CommentedMap([('home', 'https://github.com/BackofenLab/INFO-RNA'), ('license', 'MIT-like'), ('summary', 'A server for the design of RNA sequences that fold into a given pseudo-knot free RNA secondary structure.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/inforna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'inforna'), ('version', '2.1.2')])), ('build', CommentedMap([('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/BackofenLab/INFO-RNA/archive/1e9bb8236c13f5861f07dc19625c394aaeae4c79.tar.gz'), ('md5', '219ddf683b78c1c314d896c1b367847e')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'viennarna']), ('run', ['libgcc', 'viennarna'])])), ('test', CommentedMap([('commands', ['INFO-RNA -h', 'INFO-RNA \"(...)\"'])])), ('about', CommentedMap([('home', 'https://github.com/BackofenLab/INFO-RNA'), ('license', 'MIT-like'), ('summary', 'A server for the design of RNA sequences that fold into a given pseudo-knot free RNA secondary structure.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ropls/1.2.14/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ropls'), ('version', '1.2.14')])), ('source', CommentedMap([('fn', 'ropls_1.2.14.tar.gz'), ('url', 'https://bioconductor.riken.jp/packages/3.2/bioc/src/contrib/ropls_1.2.14.tar.gz'), ('md5', '0d0cfa9b7458da283ce343eff9211a81')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ropls\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ropls.html'), ('license', 'CeCILL'), ('summary', 'Latent variable modeling with Principal Component Analysis (PCA) and Partial Least Squares (PLS) are powerful methods for visualization, regression, classification, and feature selection of omics data where the number of variables exceeds the number of samples and with multicollinearity among variables. Orthogonal Partial Least Squares (OPLS) enables to separately model the variation correlated (predictive) to the factor of interest and the uncorrelated (orthogonal) variation. While performing similarly to PLS, OPLS facilitates interpretation. Successful applications of these chemometrics techniques include spectroscopic data such as Raman spectroscopy, nuclear magnetic resonance (NMR), mass spectrometry (MS) in metabolomics and proteomics, but also transcriptomics data. In addition to scores, loadings and weights plots, the package provides metrics and graphics to determine the optimal number of components (e.g. with the R2 and Q2 coefficients), check the validity of the model by permutation testing, detect outliers, and perform feature selection (e.g. with Variable Importance in Projection or regression coefficients). The package can be accessed via a user interface on the Workflow4Metabolomics.org online resource for computational metabolomics (built upon the Galaxy environment).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ropls/1.2.14/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ropls'), ('version', '1.2.14')])), ('source', CommentedMap([('fn', 'ropls_1.2.14.tar.gz'), ('url', 'https://bioconductor.riken.jp/packages/3.2/bioc/src/contrib/ropls_1.2.14.tar.gz'), ('md5', '0d0cfa9b7458da283ce343eff9211a81')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ropls\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ropls.html'), ('license', 'CeCILL'), ('summary', 'Latent variable modeling with Principal Component Analysis (PCA) and Partial Least Squares (PLS) are powerful methods for visualization, regression, classification, and feature selection of omics data where the number of variables exceeds the number of samples and with multicollinearity among variables. Orthogonal Partial Least Squares (OPLS) enables to separately model the variation correlated (predictive) to the factor of interest and the uncorrelated (orthogonal) variation. While performing similarly to PLS, OPLS facilitates interpretation. Successful applications of these chemometrics techniques include spectroscopic data such as Raman spectroscopy, nuclear magnetic resonance (NMR), mass spectrometry (MS) in metabolomics and proteomics, but also transcriptomics data. In addition to scores, loadings and weights plots, the package provides metrics and graphics to determine the optimal number of components (e.g. with the R2 and Q2 coefficients), check the validity of the model by permutation testing, detect outliers, and perform feature selection (e.g. with Variable Importance in Projection or regression coefficients). The package can be accessed via a user interface on the Workflow4Metabolomics.org online resource for computational metabolomics (built upon the Galaxy environment).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtcgatoolbox/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtcgatoolbox'), ('version', '2.6.0')])), ('source', CommentedMap([('fn', 'RTCGAToolbox_2.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RTCGAToolbox_2.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/RTCGAToolbox_2.6.0.tar.gz', 'https://depot.galaxyproject.org/software/RTCGAToolbox/RTCGAToolbox_2.6.0_src_all.tar.gz']), ('md5', '7ec10941e9be27dfb553305e96f4007c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma >=3.18', 'r-base', 'r-data.table >=1.9.4', 'r-rcircos', 'r-rcurl', 'r-rjsonio', 'r-survival', 'r-xml']), ('run', ['bioconductor-limma >=3.18', 'r-base', 'r-data.table >=1.9.4', 'r-rcircos', 'r-rcurl', 'r-rjsonio', 'r-survival', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RTCGAToolbox\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RTCGAToolbox.html'), ('license', 'GPL (>= 2)'), ('summary', 'Managing data from large scale projects such as The Cancer Genome Atlas (TCGA) for further analysis is an important and time consuming step for research projects. Several efforts, such as Firehose project, make TCGA pre-processed data publicly available via web services and data portals but it requires managing, downloading and preparing the data for following steps. We developed an open source and extensible R based data client for Firehose pre-processed data and demonstrated its use with sample case studies. Results showed that RTCGAToolbox could improve data management for researchers who are interested with TCGA data. In addition, it can be integrated with other analysis pipelines for following data analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rtcgatoolbox/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rtcgatoolbox'), ('version', '2.6.0')])), ('source', CommentedMap([('fn', 'RTCGAToolbox_2.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RTCGAToolbox_2.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/RTCGAToolbox_2.6.0.tar.gz', 'https://depot.galaxyproject.org/software/RTCGAToolbox/RTCGAToolbox_2.6.0_src_all.tar.gz']), ('md5', '7ec10941e9be27dfb553305e96f4007c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma >=3.18', 'r-base', 'r-data.table >=1.9.4', 'r-rcircos', 'r-rcurl', 'r-rjsonio', 'r-survival', 'r-xml']), ('run', ['bioconductor-limma >=3.18', 'r-base', 'r-data.table >=1.9.4', 'r-rcircos', 'r-rcurl', 'r-rjsonio', 'r-survival', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RTCGAToolbox\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RTCGAToolbox.html'), ('license', 'GPL (>= 2)'), ('summary', 'Managing data from large scale projects such as The Cancer Genome Atlas (TCGA) for further analysis is an important and time consuming step for research projects. Several efforts, such as Firehose project, make TCGA pre-processed data publicly available via web services and data portals but it requires managing, downloading and preparing the data for following steps. We developed an open source and extensible R based data client for Firehose pre-processed data and demonstrated its use with sample case studies. Results showed that RTCGAToolbox could improve data management for researchers who are interested with TCGA data. In addition, it can be integrated with other analysis pipelines for following data analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcftools/1.3.1/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/bcftools'), ('license', 'MIT'), ('summary', 'BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.  Most commands accept VCF, bgzipped VCF and BCF with filetype detected automatically even when streaming from a pipe. Indexed VCF and BCF will work in all situations. Un-indexed VCF and BCF and streams will work in most, but not all situations.')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'bcftools'), ('version', '1.3.1')])), ('requirements', CommentedMap([('build', ['zlib ==1.2.8']), ('run', ['zlib ==1.2.8'])])), ('source', CommentedMap([('fn', 'bcftools-1.3.1.tar.bz2'), ('sha256', '12c37a4054cbf1980223e2b3a80a7fdb3fd850324a4ba6832e38fdba91f1b924'), ('url', 'https://github.com/samtools/bcftools/releases/download/1.3.1/bcftools-1.3.1.tar.bz2')])), ('test', CommentedMap([('commands', ['bcftools -h > /dev/null', 'bcftools --version', 'bcftools plugin -lv'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bcftools/1.3.1/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/bcftools'), ('license', 'MIT'), ('summary', 'BCFtools is a set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.  Most commands accept VCF, bgzipped VCF and BCF with filetype detected automatically even when streaming from a pipe. Indexed VCF and BCF will work in all situations. Un-indexed VCF and BCF and streams will work in most, but not all situations.')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'bcftools'), ('version', '1.3.1')])), ('requirements', CommentedMap([('build', ['zlib ==1.2.8']), ('run', ['zlib ==1.2.8'])])), ('source', CommentedMap([('fn', 'bcftools-1.3.1.tar.bz2'), ('sha256', '12c37a4054cbf1980223e2b3a80a7fdb3fd850324a4ba6832e38fdba91f1b924'), ('url', 'https://github.com/samtools/bcftools/releases/download/1.3.1/bcftools-1.3.1.tar.bz2')])), ('test', CommentedMap([('commands', ['bcftools -h > /dev/null', 'bcftools --version', 'bcftools plugin -lv'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/varscan/2.4.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://dkoboldt.github.io/varscan/'), ('license', 'The Non-Profit Open Software License version 3.0 (NPOSL-3.0)'), ('summary', 'variant detection in massively parallel sequencing data')])), ('package', CommentedMap([('name', 'varscan'), ('version', '2.4.2')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'VarScan-2.4.2.jar'), ('url', 'https://github.com/dkoboldt/varscan/raw/master/VarScan.v2.4.2.jar'), ('md5', '4b810741505a8145a7f8f9f6791bbacf')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['varscan mpileup2cns --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/varscan/2.4.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://dkoboldt.github.io/varscan/'), ('license', 'The Non-Profit Open Software License version 3.0 (NPOSL-3.0)'), ('summary', 'variant detection in massively parallel sequencing data')])), ('package', CommentedMap([('name', 'varscan'), ('version', '2.4.2')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'VarScan-2.4.2.jar'), ('url', 'https://github.com/dkoboldt/varscan/raw/master/VarScan.v2.4.2.jar'), ('md5', '4b810741505a8145a7f8f9f6791bbacf')])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['varscan mpileup2cns --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-heatplus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-heatplus'), ('version', '2.20.0')])), ('source', CommentedMap([('fn', 'Heatplus_2.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Heatplus_2.20.0.tar.gz'), ('md5', '7607b82c9960e3991cb850d484eb800a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Heatplus\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Heatplus.html'), ('license', 'GPL (>= 2)'), ('summary', 'Display a rectangular heatmap (intensity plot) of a data matrix. By default, both samples (columns) and features (row) of the matrix are sorted according to a hierarchical clustering, and the corresponding dendrogram is plotted. Optionally, panels with additional information about samples and features can be added to the plot.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-heatplus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-heatplus'), ('version', '2.20.0')])), ('source', CommentedMap([('fn', 'Heatplus_2.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Heatplus_2.20.0.tar.gz'), ('md5', '7607b82c9960e3991cb850d484eb800a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Heatplus\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Heatplus.html'), ('license', 'GPL (>= 2)'), ('summary', 'Display a rectangular heatmap (intensity plot) of a data matrix. By default, both samples (columns) and features (row) of the matrix are sorted according to a hierarchical clustering, and the corresponding dendrogram is plotted. Optionally, panels with additional information about samples and features can be added to the plot.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-roleswitch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-roleswitch'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'Roleswitch_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Roleswitch_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/Roleswitch_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/Roleswitch/Roleswitch_1.14.0_src_all.tar.gz']), ('md5', 'fd8b45d19324e41f4390d8d7803d4bb4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-microrna', 'r-base', 'r-dbi', 'r-plotrix', 'r-pracma', 'r-reshape']), ('run', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-microrna', 'r-base', 'r-dbi', 'r-plotrix', 'r-pracma', 'r-reshape'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Roleswitch\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Roleswitch.html'), ('license', 'GPL-2'), ('summary', 'Infer Probabilities of MiRNA-mRNA Interaction Signature (ProMISe) using paired expression data from a single sample. Roleswitch operates in two phases by inferring the probability of mRNA (miRNA) being the targets (\"targets\") of miRNA (mRNA), taking into account the expression of all of the mRNAs (miRNAs) due to their potential competition for the same miRNA (mRNA). Due to dynamic miRNA repression in the cell, Roleswitch assumes that the total transcribed mRNA levels are higher than the observed (equilibrium) mRNA levels and iteratively updates the total transcription of each mRNA targets based on the above inference. NB: in the paper, we used ProMISe as both the model name and inferred score name.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-roleswitch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-roleswitch'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'Roleswitch_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Roleswitch_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/Roleswitch_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/Roleswitch/Roleswitch_1.14.0_src_all.tar.gz']), ('md5', 'fd8b45d19324e41f4390d8d7803d4bb4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-microrna', 'r-base', 'r-dbi', 'r-plotrix', 'r-pracma', 'r-reshape']), ('run', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-microrna', 'r-base', 'r-dbi', 'r-plotrix', 'r-pracma', 'r-reshape'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Roleswitch\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Roleswitch.html'), ('license', 'GPL-2'), ('summary', 'Infer Probabilities of MiRNA-mRNA Interaction Signature (ProMISe) using paired expression data from a single sample. Roleswitch operates in two phases by inferring the probability of mRNA (miRNA) being the targets (\"targets\") of miRNA (mRNA), taking into account the expression of all of the mRNAs (miRNAs) due to their potential competition for the same miRNA (mRNA). Due to dynamic miRNA repression in the cell, Roleswitch assumes that the total transcribed mRNA levels are higher than the observed (equilibrium) mRNA levels and iteratively updates the total transcription of each mRNA targets based on the above inference. NB: in the paper, we used ProMISe as both the model name and inferred score name.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-repitools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-repitools'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'Repitools_1.20.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-repitools/bioconductor-repitools_1.20.0_src_all.tar.gz'), ('md5', '3f27f4e075425ed0b3a9bb47bcaf8aaf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.8.0', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger >=3.4.0', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.20.0', 'bioconductor-ringo', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-aroma.affymetrix', 'r-gplots', 'r-gsmoothr', 'r-rsolnp', 'r-cluster']), ('run', ['bioconductor-biocgenerics >=0.8.0', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger >=3.4.0', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.20.0', 'bioconductor-ringo', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-aroma.affymetrix', 'r-gplots', 'r-gsmoothr', 'r-rsolnp', 'r-cluster'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Repitools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Repitools.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Tools for the analysis of enrichment-based epigenomic data.  Features include summarization and visualization of epigenomic data across promoters according to gene expression context, finding regions of differential methylation/binding, BayMeth for quantifying methylation etc.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-repitools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-repitools'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'Repitools_1.20.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-repitools/bioconductor-repitools_1.20.0_src_all.tar.gz'), ('md5', '3f27f4e075425ed0b3a9bb47bcaf8aaf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.8.0', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger >=3.4.0', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.20.0', 'bioconductor-ringo', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-aroma.affymetrix', 'r-gplots', 'r-gsmoothr', 'r-rsolnp', 'r-cluster']), ('run', ['bioconductor-biocgenerics >=0.8.0', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-dnacopy', 'bioconductor-edger >=3.4.0', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.20.0', 'bioconductor-ringo', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-aroma.affymetrix', 'r-gplots', 'r-gsmoothr', 'r-rsolnp', 'r-cluster'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Repitools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Repitools.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Tools for the analysis of enrichment-based epigenomic data.  Features include summarization and visualization of epigenomic data across promoters according to gene expression context, finding regions of differential methylation/binding, BayMeth for quantifying methylation etc.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kraken/0.10.5beta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kraken'), ('version', '0.10.5beta')])), ('source', CommentedMap([('fn', 'kraken-0.10.5-beta.tar.gz'), ('url', 'https://github.com/DerrickWood/kraken/archive/v0.10.5-beta.tar.gz'), ('md5', '0231a7bfc067f564ad28fa91e9f71606')])), ('build', CommentedMap([('skip', True), ('has_prefix_files', ['libexec/kraken', 'libexec/kraken-build', 'libexec/kraken-filter', 'libexec/kraken-mpa-report', 'libexec/kraken-report', 'libexec/kraken-translate'])])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['kraken --version 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/kraken/'), ('license', 'GPLv3'), ('summary', 'Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kraken/0.10.5beta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kraken'), ('version', '0.10.5beta')])), ('source', CommentedMap([('fn', 'kraken-0.10.5-beta.tar.gz'), ('url', 'https://github.com/DerrickWood/kraken/archive/v0.10.5-beta.tar.gz'), ('md5', '0231a7bfc067f564ad28fa91e9f71606')])), ('build', CommentedMap([('skip', True), ('has_prefix_files', ['libexec/kraken', 'libexec/kraken-build', 'libexec/kraken-filter', 'libexec/kraken-mpa-report', 'libexec/kraken-report', 'libexec/kraken-translate'])])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['kraken --version 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/kraken/'), ('license', 'GPLv3'), ('summary', 'Kraken is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bayseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bayseq'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'baySeq_2.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/baySeq_2.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/baySeq_2.10.0.tar.gz', 'https://depot.galaxyproject.org/software/baySeq/baySeq_2.10.0_src_all.tar.gz']), ('md5', 'cff209dfec913f634109386cb2e3a74c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-edger', 'bioconductor-genomicranges', 'r-abind', 'r-base']), ('run', ['bioconductor-edger', 'bioconductor-genomicranges', 'r-abind', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'baySeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/baySeq.html'), ('license', 'GPL-3'), ('summary', \"This package identifies differential expression in high-throughput 'count' data, such as that derived from next-generation sequencing machines, calculating estimated posterior likelihoods of differential expression (or more complex hypotheses) via empirical Bayesian methods.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bayseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bayseq'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'baySeq_2.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/baySeq_2.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/baySeq_2.10.0.tar.gz', 'https://depot.galaxyproject.org/software/baySeq/baySeq_2.10.0_src_all.tar.gz']), ('md5', 'cff209dfec913f634109386cb2e3a74c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-edger', 'bioconductor-genomicranges', 'r-abind', 'r-base']), ('run', ['bioconductor-edger', 'bioconductor-genomicranges', 'r-abind', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'baySeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/baySeq.html'), ('license', 'GPL-3'), ('summary', \"This package identifies differential expression in high-throughput 'count' data, such as that derived from next-generation sequencing machines, calculating estimated posterior likelihoods of differential expression (or more complex hypotheses) via empirical Bayesian methods.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-hpar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-hpar'), ('version', '1.15.0')])), ('source', CommentedMap([('fn', 'hpar_1.15.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/hpar_1.15.0.tar.gz'), ('md5', 'c27602ea84d160c0b71b65c68dea6bf0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'hpar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/hpar.html'), ('license', 'Artistic-2.0'), ('summary', 'A simple interface to and data from the Human Protein Atlas project.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-hpar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-hpar'), ('version', '1.15.0')])), ('source', CommentedMap([('fn', 'hpar_1.15.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/hpar_1.15.0.tar.gz'), ('md5', 'c27602ea84d160c0b71b65c68dea6bf0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'hpar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/hpar.html'), ('license', 'Artistic-2.0'), ('summary', 'A simple interface to and data from the Human Protein Atlas project.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-readqpcr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-readqpcr'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'ReadqPCR_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ReadqPCR_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/ReadqPCR_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/ReadqPCR/ReadqPCR_1.22.0_src_all.tar.gz']), ('md5', '356a63511d979779b0ec27dd1c393b10')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ReadqPCR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ReadqPCR.html'), ('license', 'LGPL-3'), ('summary', 'The package provides functions to read raw RT-qPCR data of different platforms.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-readqpcr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-readqpcr'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'ReadqPCR_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ReadqPCR_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/ReadqPCR_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/ReadqPCR/ReadqPCR_1.22.0_src_all.tar.gz']), ('md5', '356a63511d979779b0ec27dd1c393b10')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ReadqPCR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ReadqPCR.html'), ('license', 'LGPL-3'), ('summary', 'The package provides functions to read raw RT-qPCR data of different platforms.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-cnogpro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-cnogpro'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'CNOGpro_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/CNOGpro_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/CNOGpro/CNOGpro_<{ version }}.tar.gz']), ('sha256', '889aba7c5201bda7edc369798e5fb9043ffb73c54b310394dee1d18c1dac3abb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-seqinr']), ('run', ['r-base', 'r-seqinr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CNOGpro\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=CNOGpro'), ('license', 'GPL-2'), ('summary', 'Methods for assigning copy number states and breakpoints in resequencing experiments of prokaryotic organisms.'), ('license_family', 'GPL2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-cnogpro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-cnogpro'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'CNOGpro_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/CNOGpro_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/CNOGpro/CNOGpro_<{ version }}.tar.gz']), ('sha256', '889aba7c5201bda7edc369798e5fb9043ffb73c54b310394dee1d18c1dac3abb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-seqinr']), ('run', ['r-base', 'r-seqinr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CNOGpro\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=CNOGpro'), ('license', 'GPL-2'), ('summary', 'Methods for assigning copy number states and breakpoints in resequencing experiments of prokaryotic organisms.'), ('license_family', 'GPL2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rrho/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rrho'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'RRHO_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RRHO_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/RRHO_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/RRHO/RRHO_1.16.0_src_all.tar.gz']), ('md5', '5975b6636247b09cc94e3770f215a064')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-venndiagram']), ('run', ['r-base', 'r-venndiagram'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RRHO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RRHO.html'), ('license', 'GPL-2'), ('summary', 'The package is aimed at inference on the amount of agreement in two sorted lists using the Rank-Rank Hypergeometric Overlap test.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rrho/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rrho'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'RRHO_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RRHO_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/RRHO_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/RRHO/RRHO_1.16.0_src_all.tar.gz']), ('md5', '5975b6636247b09cc94e3770f215a064')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-venndiagram']), ('run', ['r-base', 'r-venndiagram'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RRHO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RRHO.html'), ('license', 'GPL-2'), ('summary', 'The package is aimed at inference on the amount of agreement in two sorted lists using the Rank-Rank Hypergeometric Overlap test.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pepnovo/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://proteomics.ucsd.edu/Software/PepNovo/'), ('license_file', 'LICENSE'), ('summary', 'PepNovo serves as a high throughput de novo peptide sequencing tool for tandem mass spectrometry data')])), ('package', CommentedMap([('name', 'pepnovo'), ('version', 20101117)])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'PepNovo.20120423.zip'), ('url', 'http://proteomics.ucsd.edu/Software/PepNovo/PepNovo.20120423.zip'), ('md5', 'af2a863487cb53bf0e92f00d395c4f0d')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['pepnovo -h 2>&1 | grep \"PepNovo+\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pepnovo/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://proteomics.ucsd.edu/Software/PepNovo/'), ('license_file', 'LICENSE'), ('summary', 'PepNovo serves as a high throughput de novo peptide sequencing tool for tandem mass spectrometry data')])), ('package', CommentedMap([('name', 'pepnovo'), ('version', 20101117)])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'PepNovo.20120423.zip'), ('url', 'http://proteomics.ucsd.edu/Software/PepNovo/PepNovo.20120423.zip'), ('md5', 'af2a863487cb53bf0e92f00d395c4f0d')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['pepnovo -h 2>&1 | grep \"PepNovo+\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/jbrowse/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'jbrowse'), ('version', '1.12.3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'jbrowse-1.12.3.zip'), ('sha256', 'd97aa89e1862efb8f7be12762528f1e32b0f93d252ee197d753eb724c5730465'), ('url', 'http://jbrowse.org/wordpress/wp-content/plugins/download-monitor/download.php?id=109')])), ('requirements', CommentedMap([('build', None), ('run', ['perl', 'perl-json', 'perl-clone', 'perl-io-string', 'perl-bioperl', 'perl-test-differences', 'perl-test-deep', 'perl-test-most', 'perl-test-exception', 'perl-test-nowarnings', 'perl-test-simple', 'perl-test-warn', 'perl-exporter-tiny', 'perl-list-moreutils', 'perl-class-data-inheritable', 'perl-devel-stacktrace', 'perl-exception-class', 'perl-sub-uplevel', 'perl-algorithm-diff', 'perl-capture-tiny', 'perl-text-diff', 'perl-data-stag', 'perl-uri', 'perl-scalar-list-utils', 'perl-perlio-gzip', 'perl-bio-gff3', 'perl-devel-size', 'perl-digest-crc32', 'perl-file-next', 'perl-hash-merge', 'perl-heap-simple', 'perl-text-tabs-wrap', 'perl-db-file'])])), ('test', CommentedMap([('commands', ['command -v add-json.pl'])])), ('about', CommentedMap([('home', 'http://jbrowse.org/'), ('license', 'LGPL'), ('summary', 'The JBrowse Genome Browser')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/jbrowse/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'jbrowse'), ('version', '1.12.3')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'jbrowse-1.12.3.zip'), ('sha256', 'd97aa89e1862efb8f7be12762528f1e32b0f93d252ee197d753eb724c5730465'), ('url', 'http://jbrowse.org/wordpress/wp-content/plugins/download-monitor/download.php?id=109')])), ('requirements', CommentedMap([('build', None), ('run', ['perl', 'perl-json', 'perl-clone', 'perl-io-string', 'perl-bioperl', 'perl-test-differences', 'perl-test-deep', 'perl-test-most', 'perl-test-exception', 'perl-test-nowarnings', 'perl-test-simple', 'perl-test-warn', 'perl-exporter-tiny', 'perl-list-moreutils', 'perl-class-data-inheritable', 'perl-devel-stacktrace', 'perl-exception-class', 'perl-sub-uplevel', 'perl-algorithm-diff', 'perl-capture-tiny', 'perl-text-diff', 'perl-data-stag', 'perl-uri', 'perl-scalar-list-utils', 'perl-perlio-gzip', 'perl-bio-gff3', 'perl-devel-size', 'perl-digest-crc32', 'perl-file-next', 'perl-hash-merge', 'perl-heap-simple', 'perl-text-tabs-wrap', 'perl-db-file'])])), ('test', CommentedMap([('commands', ['command -v add-json.pl'])])), ('about', CommentedMap([('home', 'http://jbrowse.org/'), ('license', 'LGPL'), ('summary', 'The JBrowse Genome Browser')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sickle/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'sickle'), ('version', '0.5')])), ('source', CommentedMap([('fn', 'Sickle-0.5.tar.gz'), ('url', 'https://pypi.python.org/packages/source/S/Sickle/Sickle-0.5.tar.gz'), ('md5', '9c93e7f3e0db98ab1a18107dfa096082')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=1.1.0', 'lxml >=3.2.3']), ('run', ['python', 'requests >=1.1.0', 'lxml >=3.2.3'])])), ('test', CommentedMap([('imports', ['sickle']), ('requires', ['mock >=1.0.1', 'nose >=1.0'])])), ('about', CommentedMap([('home', 'http://github.com/mloesch/sickle'), ('license', 'BSD License'), ('summary', 'A lightweight OAI client library for Python')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sickle/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'sickle'), ('version', '0.5')])), ('source', CommentedMap([('fn', 'Sickle-0.5.tar.gz'), ('url', 'https://pypi.python.org/packages/source/S/Sickle/Sickle-0.5.tar.gz'), ('md5', '9c93e7f3e0db98ab1a18107dfa096082')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=1.1.0', 'lxml >=3.2.3']), ('run', ['python', 'requests >=1.1.0', 'lxml >=3.2.3'])])), ('test', CommentedMap([('imports', ['sickle']), ('requires', ['mock >=1.0.1', 'nose >=1.0'])])), ('about', CommentedMap([('home', 'http://github.com/mloesch/sickle'), ('license', 'BSD License'), ('summary', 'A lightweight OAI client library for Python')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnaprobr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnaprobr'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'RNAprobR_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RNAprobR_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/RNAprobR_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/RNAprobR/RNAprobR_1.8.0_src_all.tar.gz']), ('md5', '2c13899031d3b95778f1de8ca0963ef5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.10.0', 'bioconductor-biostrings >=2.32.1', 'bioconductor-genomicalignments >=1.5.12', 'bioconductor-genomicfeatures >=1.16.3', 'bioconductor-genomicranges >=1.16.4', 'bioconductor-rsamtools >=1.16.1', 'bioconductor-rtracklayer >=1.24.2', 'r-base', 'r-plyr >=1.8.1']), ('run', ['bioconductor-biocgenerics >=0.10.0', 'bioconductor-biostrings >=2.32.1', 'bioconductor-genomicalignments >=1.5.12', 'bioconductor-genomicfeatures >=1.16.3', 'bioconductor-genomicranges >=1.16.4', 'bioconductor-rsamtools >=1.16.1', 'bioconductor-rtracklayer >=1.24.2', 'r-base', 'r-plyr >=1.8.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RNAprobR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RNAprobR.html'), ('license', 'GPL (>=2)'), ('summary', 'This package facilitates analysis of Next Generation Sequencing data for which positional information with a single nucleotide resolution is a key. It allows for applying different types of relevant normalizations, data visualization and export in a table or UCSC compatible bedgraph file.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnaprobr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnaprobr'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'RNAprobR_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RNAprobR_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/RNAprobR_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/RNAprobR/RNAprobR_1.8.0_src_all.tar.gz']), ('md5', '2c13899031d3b95778f1de8ca0963ef5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.10.0', 'bioconductor-biostrings >=2.32.1', 'bioconductor-genomicalignments >=1.5.12', 'bioconductor-genomicfeatures >=1.16.3', 'bioconductor-genomicranges >=1.16.4', 'bioconductor-rsamtools >=1.16.1', 'bioconductor-rtracklayer >=1.24.2', 'r-base', 'r-plyr >=1.8.1']), ('run', ['bioconductor-biocgenerics >=0.10.0', 'bioconductor-biostrings >=2.32.1', 'bioconductor-genomicalignments >=1.5.12', 'bioconductor-genomicfeatures >=1.16.3', 'bioconductor-genomicranges >=1.16.4', 'bioconductor-rsamtools >=1.16.1', 'bioconductor-rtracklayer >=1.24.2', 'r-base', 'r-plyr >=1.8.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RNAprobR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RNAprobR.html'), ('license', 'GPL (>=2)'), ('summary', 'This package facilitates analysis of Next Generation Sequencing data for which positional information with a single nucleotide resolution is a key. It allows for applying different types of relevant normalizations, data visualization and export in a table or UCSC compatible bedgraph file.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/segemehl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'segemehl'), ('version', '0.2.0')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-leipzig.de/Software/segemehl/'), ('summary', 'Short read mapping with gaps')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'segemehl_0_2_0.tar.gz'), ('url', 'http://www.bioinf.uni-leipzig.de/Software/segemehl/segemehl_0_2_0.tar.gz'), ('sha256', '2575139d2f2ba7bd534afcbe485c0570c177d1faaeb267c4cd873e65b8e72afb')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc', 'zlib', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['libgcc', 'zlib', 'ncurses <{CONDA_NCURSES}}*'])])), ('test', CommentedMap([('commands', ['segemehl.x --help 2>&1 | grep -A1 VERSION', 'lack.x --help 2>&1 | grep -A1 VERSION', 'testrealign.x --help 2>&1 | grep -A1 VERSION'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/segemehl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'segemehl'), ('version', '0.2.0')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-leipzig.de/Software/segemehl/'), ('summary', 'Short read mapping with gaps')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'segemehl_0_2_0.tar.gz'), ('url', 'http://www.bioinf.uni-leipzig.de/Software/segemehl/segemehl_0_2_0.tar.gz'), ('sha256', '2575139d2f2ba7bd534afcbe485c0570c177d1faaeb267c4cd873e65b8e72afb')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc', 'zlib', 'ncurses <{CONDA_NCURSES}}*']), ('run', ['libgcc', 'zlib', 'ncurses <{CONDA_NCURSES}}*'])])), ('test', CommentedMap([('commands', ['segemehl.x --help 2>&1 | grep -A1 VERSION', 'lack.x --help 2>&1 | grep -A1 VERSION', 'testrealign.x --help 2>&1 | grep -A1 VERSION'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sigpathway/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sigpathway'), ('version', '1.44.1')])), ('source', CommentedMap([('fn', 'sigPathway_1.44.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sigPathway_1.44.1.tar.gz', 'https://depot.galaxyproject.org/software/sigPathway/sigPathway_1.44.1_src_all.tar.gz']), ('md5', '0a34f8a57894f81ef21c17bac663e15b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sigPathway\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sigPathway.html'), ('license', 'GPL-2'), ('summary', 'Conducts pathway analysis by calculating the NT_k and NE_k statistics as described in Tian et al. (2005)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sigpathway/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sigpathway'), ('version', '1.44.1')])), ('source', CommentedMap([('fn', 'sigPathway_1.44.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sigPathway_1.44.1.tar.gz', 'https://depot.galaxyproject.org/software/sigPathway/sigPathway_1.44.1_src_all.tar.gz']), ('md5', '0a34f8a57894f81ef21c17bac663e15b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sigPathway\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sigPathway.html'), ('license', 'GPL-2'), ('summary', 'Conducts pathway analysis by calculating the NT_k and NE_k statistics as described in Tian et al. (2005)')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-quantsmooth/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-quantsmooth'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'quantsmooth_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/quantsmooth_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/quantsmooth_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/quantsmooth/quantsmooth_1.42.0_src_all.tar.gz']), ('md5', '09f500a2a2f6d55d66a40a73f66be4d0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-quantreg']), ('run', ['r-base', 'r-quantreg'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'quantsmooth\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/quantsmooth.html'), ('license', 'GPL-2'), ('summary', 'Implements quantile smoothing as introduced in: Quantile smoothing of array CGH data; Eilers PH, de Menezes RX; Bioinformatics. 2005 Apr 1;21(7):1146-53.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-quantsmooth/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-quantsmooth'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'quantsmooth_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/quantsmooth_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/quantsmooth_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/quantsmooth/quantsmooth_1.42.0_src_all.tar.gz']), ('md5', '09f500a2a2f6d55d66a40a73f66be4d0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-quantreg']), ('run', ['r-base', 'r-quantreg'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'quantsmooth\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/quantsmooth.html'), ('license', 'GPL-2'), ('summary', 'Implements quantile smoothing as introduced in: Quantile smoothing of array CGH data; Eilers PH, de Menezes RX; Bioinformatics. 2005 Apr 1;21(7):1146-53.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mdqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mdqc'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'mdqc_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mdqc_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/mdqc_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/mdqc/mdqc_1.38.0_src_all.tar.gz']), ('md5', 'ab47f4c330b4825b4b73cfb77fa665d5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-cluster', 'r-mass']), ('run', ['r-base', 'r-cluster', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mdqc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mdqc.html'), ('license', 'LGPL (>= 2)'), ('summary', \"MDQC is a multivariate quality assessment method for microarrays based on quality control (QC) reports. The Mahalanobis distance of an array's quality attributes is used to measure the similarity of the quality of that array against the quality of the other arrays. Then, arrays with unusually high distances can be flagged as potentially low-quality.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mdqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mdqc'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'mdqc_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mdqc_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/mdqc_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/mdqc/mdqc_1.38.0_src_all.tar.gz']), ('md5', 'ab47f4c330b4825b4b73cfb77fa665d5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-cluster', 'r-mass']), ('run', ['r-base', 'r-cluster', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mdqc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mdqc.html'), ('license', 'LGPL (>= 2)'), ('summary', \"MDQC is a multivariate quality assessment method for microarrays based on quality control (QC) reports. The Mahalanobis distance of an array's quality attributes is used to measure the similarity of the quality of that array against the quality of the other arrays. Then, arrays with unusually high distances can be flagged as potentially low-quality.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biosigner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biosigner'), ('version', '1.1.10')])), ('source', CommentedMap([('fn', 'biosigner_1.1.10.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/biosigner_1.1.10.tar.gz'), ('md5', 'c66f001a78bf92af46568ed81cfdf243')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-ropls', 'bioconductor-biobase', 'r-base', 'r-e1071', 'r-randomforest']), ('run', ['bioconductor-ropls', 'bioconductor-biobase', 'r-base', 'r-e1071', 'r-randomforest'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biosigner\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/biosigner.html'), ('license', 'CeCILL'), ('summary', \"Feature selection is critical in omics data analysis to extract restricted and meaningful molecular signatures from complex and high-dimension data, and to build robust classifiers. This package implements a new method to assess the relevance of the variables for the prediction performances of the classifier. The approach can be run in parallel with the PLS-DA, Random Forest, and SVM binary classifiers. The signatures and the corresponding 'restricted' models are returned, enabling future predictions on new datasets. A Galaxy implementation of the package is available within the Workflow4metabolomics.org online infrastructure for computational metabolomics.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biosigner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biosigner'), ('version', '1.1.10')])), ('source', CommentedMap([('fn', 'biosigner_1.1.10.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/biosigner_1.1.10.tar.gz'), ('md5', 'c66f001a78bf92af46568ed81cfdf243')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-ropls', 'bioconductor-biobase', 'r-base', 'r-e1071', 'r-randomforest']), ('run', ['bioconductor-ropls', 'bioconductor-biobase', 'r-base', 'r-e1071', 'r-randomforest'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biosigner\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/biosigner.html'), ('license', 'CeCILL'), ('summary', \"Feature selection is critical in omics data analysis to extract restricted and meaningful molecular signatures from complex and high-dimension data, and to build robust classifiers. This package implements a new method to assess the relevance of the variables for the prediction performances of the classifier. The approach can be run in parallel with the PLS-DA, Random Forest, and SVM binary classifiers. The signatures and the corresponding 'restricted' models are returned, enabling future predictions on new datasets. A Galaxy implementation of the package is available within the Workflow4metabolomics.org online infrastructure for computational metabolomics.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snippy/3.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'snippy'), ('version', '3.0')])), ('source', CommentedMap([('fn', 'snippy_v3.0.tar.gz'), ('md5', '081ddf6ffad192136de2fc9d6635199d'), ('url', 'https://github.com/tseemann/snippy/archive/v3.0.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-bioperl', 'perl-app-cpanminus', 'perl-module-build', 'bwa', 'samtools', 'parallel', 'freebayes', 'vcflib', 'vcftools', 'snpeff']), ('run', ['perl-threaded', 'perl-bioperl', 'bwa', 'samtools', 'parallel', 'freebayes', 'vcflib', 'vcftools', 'snpeff', 'htslib'])])), ('test', CommentedMap([('commands', ['snippy --version'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/snippy'), ('license', 'GPL2'), ('summary', 'Rapid bacterial SNP calling and core genome alignments')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snippy/3.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'snippy'), ('version', '3.0')])), ('source', CommentedMap([('fn', 'snippy_v3.0.tar.gz'), ('md5', '081ddf6ffad192136de2fc9d6635199d'), ('url', 'https://github.com/tseemann/snippy/archive/v3.0.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-bioperl', 'perl-app-cpanminus', 'perl-module-build', 'bwa', 'samtools', 'parallel', 'freebayes', 'vcflib', 'vcftools', 'snpeff']), ('run', ['perl-threaded', 'perl-bioperl', 'bwa', 'samtools', 'parallel', 'freebayes', 'vcflib', 'vcftools', 'snpeff', 'htslib'])])), ('test', CommentedMap([('commands', ['snippy --version'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/snippy'), ('license', 'GPL2'), ('summary', 'Rapid bacterial SNP calling and core genome alignments')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomicfeatures/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomicfeatures'), ('version', '1.26.4')])), ('source', CommentedMap([('fn', 'GenomicFeatures_1.26.4.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/GenomicFeatures_1.26.4.tar.gz'), ('md5', '275a00ad50b7a5ec5ca5389b136491ca')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.33.15', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-biomart >=2.17.1', 'bioconductor-biostrings >=2.23.3', 'bioconductor-genomeinfodb >=1.5.16', 'bioconductor-genomicranges >=1.21.32', 'bioconductor-iranges >=2.3.21', 'bioconductor-rtracklayer >=1.29.24', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base', 'r-dbi', 'r-rcurl', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.33.15', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-biomart >=2.17.1', 'bioconductor-biostrings >=2.23.3', 'bioconductor-genomeinfodb >=1.5.16', 'bioconductor-genomicranges >=1.21.32', 'bioconductor-iranges >=2.3.21', 'bioconductor-rtracklayer >=1.29.24', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base', 'r-dbi', 'r-rcurl', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenomicFeatures\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/GenomicFeatures.html'), ('license', 'Artistic-2.0'), ('summary', 'A set of tools and methods for making and manipulating transcript centric annotations. With these tools the user can easily download the genomic locations of the transcripts, exons and cds of a given organism, from either the UCSC Genome Browser or a BioMart database (more sources will be supported in the future). This information is then stored in a local database that keeps track of the relationship between transcripts, exons, cds and genes. Flexible methods are provided for extracting the desired features in a convenient format.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomicfeatures/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomicfeatures'), ('version', '1.26.4')])), ('source', CommentedMap([('fn', 'GenomicFeatures_1.26.4.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/GenomicFeatures_1.26.4.tar.gz'), ('md5', '275a00ad50b7a5ec5ca5389b136491ca')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.33.15', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-biomart >=2.17.1', 'bioconductor-biostrings >=2.23.3', 'bioconductor-genomeinfodb >=1.5.16', 'bioconductor-genomicranges >=1.21.32', 'bioconductor-iranges >=2.3.21', 'bioconductor-rtracklayer >=1.29.24', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base', 'r-dbi', 'r-rcurl', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.33.15', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-biomart >=2.17.1', 'bioconductor-biostrings >=2.23.3', 'bioconductor-genomeinfodb >=1.5.16', 'bioconductor-genomicranges >=1.21.32', 'bioconductor-iranges >=2.3.21', 'bioconductor-rtracklayer >=1.29.24', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base', 'r-dbi', 'r-rcurl', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenomicFeatures\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/GenomicFeatures.html'), ('license', 'Artistic-2.0'), ('summary', 'A set of tools and methods for making and manipulating transcript centric annotations. With these tools the user can easily download the genomic locations of the transcripts, exons and cds of a given organism, from either the UCSC Genome Browser or a BioMart database (more sources will be supported in the future). This information is then stored in a local database that keeps track of the relationship between transcripts, exons, cds and genes. Flexible methods are provided for extracting the desired features in a convenient format.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-maanova/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-maanova'), ('version', '1.46.1')])), ('source', CommentedMap([('fn', 'maanova_1.46.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/maanova_1.46.1.tar.gz', 'https://depot.galaxyproject.org/software/maanova/maanova_1.46.1_src_all.tar.gz']), ('md5', '1edc4dab038dfeb276a518c37d04783b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'maanova\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/maanova.html'), ('license', 'GPL (>= 2)'), ('summary', 'Analysis of N-dye Micro Array experiment using mixed model effect. Containing analysis of variance, permutation and bootstrap, cluster and consensus tree.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-maanova/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-maanova'), ('version', '1.46.1')])), ('source', CommentedMap([('fn', 'maanova_1.46.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/maanova_1.46.1.tar.gz', 'https://depot.galaxyproject.org/software/maanova/maanova_1.46.1_src_all.tar.gz']), ('md5', '1edc4dab038dfeb276a518c37d04783b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'maanova\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/maanova.html'), ('license', 'GPL (>= 2)'), ('summary', 'Analysis of N-dye Micro Array experiment using mixed model effect. Containing analysis of variance, permutation and bootstrap, cluster and consensus tree.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-viper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-viper'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'viper_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/viper_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/viper_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/viper/viper_1.10.0_src_all.tar.gz']), ('md5', '4d7ad13a2d8fe5542a235f0d42f0d6d3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-kernsmooth', 'r-mixtools']), ('run', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-kernsmooth', 'r-mixtools'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'viper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/viper.html'), ('license', 'file LICENSE'), ('summary', 'Inference of protein activity from gene expression data, including the VIPER and msVIPER algorithms')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-viper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-viper'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'viper_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/viper_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/viper_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/viper/viper_1.10.0_src_all.tar.gz']), ('md5', '4d7ad13a2d8fe5542a235f0d42f0d6d3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-kernsmooth', 'r-mixtools']), ('run', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-kernsmooth', 'r-mixtools'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'viper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/viper.html'), ('license', 'file LICENSE'), ('summary', 'Inference of protein activity from gene expression data, including the VIPER and msVIPER algorithms')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-zlibbioc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-zlibbioc'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'zlibbioc_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/zlibbioc_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/zlibbioc_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/zlibbioc/zlibbioc_1.20.0_src_all.tar.gz']), ('md5', 'f68b45fca70f39c7958fb43c0504c546')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'zlibbioc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/zlibbioc.html'), ('license', 'Artistic-2.0 + file LICENSE'), ('summary', 'This package uses the source code of zlib-1.2.5 to create libraries for systems that do not have these available via other means (most Linux and Mac users should have system-level access to zlib, and no direct need for this package). See the vignette for instructions on use.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-zlibbioc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-zlibbioc'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'zlibbioc_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/zlibbioc_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/zlibbioc_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/zlibbioc/zlibbioc_1.20.0_src_all.tar.gz']), ('md5', 'f68b45fca70f39c7958fb43c0504c546')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'zlibbioc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/zlibbioc.html'), ('license', 'Artistic-2.0 + file LICENSE'), ('summary', 'This package uses the source code of zlib-1.2.5 to create libraries for systems that do not have these available via other means (most Linux and Mac users should have system-level access to zlib, and no direct need for this package). See the vignette for instructions on use.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/locarna/1.9.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'locarna'), ('version', '1.9.1')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Multiple alignment of RNAs')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'locarna-1.9.1.tar.gz'), ('url', 'https://github.com/s-will/LocARNA/releases/download/v1.9.1/locarna-1.9.1.tar.gz'), ('sha256', 'd5c51282ff2f33e9fc90d6b3d85e807d1a2503272c4401d1bdce77f546178b45')])), ('requirements', CommentedMap([('build', ['viennarna ==2.3.3', 'llvm', 'gcc']), ('run', ['viennarna ==2.3.3', 'perl', 'libgcc'])])), ('test', CommentedMap([('commands', ['mlocarna --version', 'locarna --version', 'locarna_p --version', 'sparse --version', 'exparna_p --version', 'echo -e \">D10744\\\\nGGAAAAUUGAUCAUCGGCAAGAUAAGUUAUUUACUAAAUAAUAGGAUUUAAUAACCUGGUGAGUUCGAAUCUCACAUUUUCCG\" | locarna_rnafold_pp --in-loop --stacking --noLP -p0.5 |head -n3'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/locarna/1.9.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'locarna'), ('version', '1.9.1')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software/LocARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Multiple alignment of RNAs')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'locarna-1.9.1.tar.gz'), ('url', 'https://github.com/s-will/LocARNA/releases/download/v1.9.1/locarna-1.9.1.tar.gz'), ('sha256', 'd5c51282ff2f33e9fc90d6b3d85e807d1a2503272c4401d1bdce77f546178b45')])), ('requirements', CommentedMap([('build', ['viennarna ==2.3.3', 'llvm', 'gcc']), ('run', ['viennarna ==2.3.3', 'perl', 'libgcc'])])), ('test', CommentedMap([('commands', ['mlocarna --version', 'locarna --version', 'locarna_p --version', 'sparse --version', 'exparna_p --version', 'echo -e \">D10744\\\\nGGAAAAUUGAUCAUCGGCAAGAUAAGUUAUUUACUAAAUAAUAGGAUUUAAUAACCUGGUGAGUUCGAAUCUCACAUUUUCCG\" | locarna_rnafold_pp --in-loop --stacking --noLP -p0.5 |head -n3'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-splots/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-splots'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'splots_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/splots_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/splots_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/splots/splots_1.42.0_src_all.tar.gz']), ('md5', 'ce2288e62b5fcd7c9ae038f6338e8aad')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'splots\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/splots.html'), ('license', 'LGPL'), ('summary', 'The splots package provides the plotScreen function for visualising data in microtitre plate or slide format.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-splots/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-splots'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'splots_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/splots_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/splots_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/splots/splots_1.42.0_src_all.tar.gz']), ('md5', 'ce2288e62b5fcd7c9ae038f6338e8aad')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'splots\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/splots.html'), ('license', 'LGPL'), ('summary', 'The splots package provides the plotScreen function for visualising data in microtitre plate or slide format.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trimmomatic/0.35/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.usadellab.org/cms/?page=trimmomatic'), ('license', 'GPLv3'), ('summary', 'A flexible read trimming tool for Illumina NGS data')])), ('package', CommentedMap([('name', 'trimmomatic'), ('version', '0.35')])), ('build', CommentedMap([('number', 3), ('skip', False)])), ('source', CommentedMap([('fn', 'Trimmomatic-0.35.zip'), ('md5', '564e62f4f0f9f56d22762b237d4c69c4'), ('url', 'http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-0.35.zip')])), ('requirements', CommentedMap([('run', ['java-jdk >=8'])])), ('test', CommentedMap([('commands', ['unzip -t $PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/trimmomatic.jar &> /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trimmomatic/0.35/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.usadellab.org/cms/?page=trimmomatic'), ('license', 'GPLv3'), ('summary', 'A flexible read trimming tool for Illumina NGS data')])), ('package', CommentedMap([('name', 'trimmomatic'), ('version', '0.35')])), ('build', CommentedMap([('number', 3), ('skip', False)])), ('source', CommentedMap([('fn', 'Trimmomatic-0.35.zip'), ('md5', '564e62f4f0f9f56d22762b237d4c69c4'), ('url', 'http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-0.35.zip')])), ('requirements', CommentedMap([('run', ['java-jdk >=8'])])), ('test', CommentedMap([('commands', ['unzip -t $PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/trimmomatic.jar &> /dev/null'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-xde/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-xde'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'XDE_2.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/XDE_2.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/XDE_2.22.0.tar.gz', 'https://depot.galaxyproject.org/software/XDE/XDE_2.22.0_src_all.tar.gz']), ('md5', '690f26084d79dfccdc0da9903ff79d2c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'bioconductor-genefilter', 'bioconductor-mergemaid', 'r-base', 'r-gtools', 'r-mvtnorm']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'bioconductor-genefilter', 'bioconductor-mergemaid', 'r-base', 'r-gtools', 'r-mvtnorm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'XDE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/XDE.html'), ('license', 'LGPL-2'), ('summary', 'Multi-level model for cross-study detection of differential gene expression.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-xde/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-xde'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'XDE_2.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/XDE_2.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/XDE_2.22.0.tar.gz', 'https://depot.galaxyproject.org/software/XDE/XDE_2.22.0_src_all.tar.gz']), ('md5', '690f26084d79dfccdc0da9903ff79d2c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'bioconductor-genefilter', 'bioconductor-mergemaid', 'r-base', 'r-gtools', 'r-mvtnorm']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics', 'bioconductor-genefilter', 'bioconductor-mergemaid', 'r-base', 'r-gtools', 'r-mvtnorm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'XDE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/XDE.html'), ('license', 'LGPL-2'), ('summary', 'Multi-level model for cross-study detection of differential gene expression.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kmergenie/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://kmergenie.bx.psu.edu/<{ name }}-<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm', 'zlib', 'r-base']), ('run', ['python', 'libgcc', 'zlib', 'r-base'])])), ('test', CommentedMap([('commands', ['kmergenie --help'])])), ('about', CommentedMap([('home', 'http://kmergenie.bx.psu.edu/'), ('license', 'free software license'), ('summary', 'KmerGenie estimates the best k-mer length for genome de novo assembly')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kmergenie/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://kmergenie.bx.psu.edu/<{ name }}-<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm', 'zlib', 'r-base']), ('run', ['python', 'libgcc', 'zlib', 'r-base'])])), ('test', CommentedMap([('commands', ['kmergenie --help'])])), ('about', CommentedMap([('home', 'http://kmergenie.bx.psu.edu/'), ('license', 'free software license'), ('summary', 'KmerGenie estimates the best k-mer length for genome de novo assembly')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-aseb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-aseb'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'ASEB_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASEB_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/ASEB_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/ASEB/ASEB_1.20.0_src_all.tar.gz']), ('md5', '24c4f3e438eecc9d9b7904a529f7096f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASEB\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASEB.html'), ('license', 'GPL (>= 3)'), ('summary', 'ASEB is an R package to predict lysine sites that can be acetylated by a specific KAT-family.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-aseb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-aseb'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'ASEB_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASEB_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/ASEB_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/ASEB/ASEB_1.20.0_src_all.tar.gz']), ('md5', '24c4f3e438eecc9d9b7904a529f7096f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASEB\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASEB.html'), ('license', 'GPL (>= 3)'), ('summary', 'ASEB is an R package to predict lysine sites that can be acetylated by a specific KAT-family.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shorah/1.1.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'shorah'), ('version', '1.1.0')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'shorah-1.1.0.tar.bz2'), ('url', 'https://github.com/cbg-ethz/shorah/releases/download/v1.1.0/shorah-1.1.0.tar.bz2'), ('sha256', 'e4fe6cfb854163cb0963a977b9f08faee68027298c247b7e305337f3ebc3234c')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'gsl', 'zlib']), ('run', ['python', 'samtools', 'biopython', 'numpy', 'gsl', 'zlib'])])), ('about', CommentedMap([('home', 'https://github.com/cbg-ethz/shorah'), ('license', 'GPLv3'), ('summary', 'The Short Reads Assembly into Haplotypes (ShoRAH) program for inferring viral haplotypes from NGS data')])), ('test', CommentedMap([('commands', ['dec.py -h', 'shorah.py -h'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/shorah/1.1.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'shorah'), ('version', '1.1.0')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'shorah-1.1.0.tar.bz2'), ('url', 'https://github.com/cbg-ethz/shorah/releases/download/v1.1.0/shorah-1.1.0.tar.bz2'), ('sha256', 'e4fe6cfb854163cb0963a977b9f08faee68027298c247b7e305337f3ebc3234c')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'gsl', 'zlib']), ('run', ['python', 'samtools', 'biopython', 'numpy', 'gsl', 'zlib'])])), ('about', CommentedMap([('home', 'https://github.com/cbg-ethz/shorah'), ('license', 'GPLv3'), ('summary', 'The Short Reads Assembly into Haplotypes (ShoRAH) program for inferring viral haplotypes from NGS data')])), ('test', CommentedMap([('commands', ['dec.py -h', 'shorah.py -h'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pepstat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pepstat'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'pepStat_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pepStat_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/pepStat_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/pepStat/pepStat_1.10.0_src_all.tar.gz']), ('md5', '37bf864d9516d0a41f16c5c79d1d2591')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'r-base', 'r-data.table', 'r-fields', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biobase', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'r-base', 'r-data.table', 'r-fields', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pepStat\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pepStat.html'), ('license', 'Artistic-2.0'), ('summary', 'Statistical analysis of peptide microarrays')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pepstat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pepstat'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'pepStat_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pepStat_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/pepStat_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/pepStat/pepStat_1.10.0_src_all.tar.gz']), ('md5', '37bf864d9516d0a41f16c5c79d1d2591')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'r-base', 'r-data.table', 'r-fields', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biobase', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'r-base', 'r-data.table', 'r-fields', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pepStat\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pepStat.html'), ('license', 'Artistic-2.0'), ('summary', 'Statistical analysis of peptide microarrays')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationforge/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationforge'), ('version', '1.14.2')])), ('source', CommentedMap([('fn', 'AnnotationForge_1.14.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/AnnotationForge_1.14.2.tar.gz'), ('md5', '7a4ce57b476f58367b8807d2a7e96c04')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.33.14', 'bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite', 'r-xml']), ('run', ['bioconductor-annotationdbi >=1.33.14', 'bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationForge\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/AnnotationForge.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides code for generating Annotation packages and their databases.  Packages produced are intended to be used with AnnotationDbi.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationforge/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationforge'), ('version', '1.14.2')])), ('source', CommentedMap([('fn', 'AnnotationForge_1.14.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/AnnotationForge_1.14.2.tar.gz'), ('md5', '7a4ce57b476f58367b8807d2a7e96c04')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.33.14', 'bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite', 'r-xml']), ('run', ['bioconductor-annotationdbi >=1.33.14', 'bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationForge\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/AnnotationForge.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides code for generating Annotation packages and their databases.  Packages produced are intended to be used with AnnotationDbi.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mbased/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mbased'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MBASED_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MBASED_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MBASED_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MBASED/MBASED_1.10.0_src_all.tar.gz']), ('md5', 'f1f31202aea10cf5921accb1197a3315')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-genomicranges', 'bioconductor-summarizedexperiment', 'r-base', 'r-runit']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-genomicranges', 'bioconductor-summarizedexperiment', 'r-base', 'r-runit'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MBASED\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MBASED.html'), ('license', 'Artistic-2.0'), ('summary', 'The package implements MBASED algorithm for detecting allele-specific gene expression from RNA count data, where allele counts at individual loci (SNVs) are integrated into a gene-specific measure of ASE, and utilizes simulations to appropriately assess the statistical significance of observed ASE.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mbased/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mbased'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MBASED_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MBASED_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MBASED_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MBASED/MBASED_1.10.0_src_all.tar.gz']), ('md5', 'f1f31202aea10cf5921accb1197a3315')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-genomicranges', 'bioconductor-summarizedexperiment', 'r-base', 'r-runit']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-genomicranges', 'bioconductor-summarizedexperiment', 'r-base', 'r-runit'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MBASED\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MBASED.html'), ('license', 'Artistic-2.0'), ('summary', 'The package implements MBASED algorithm for detecting allele-specific gene expression from RNA count data, where allele counts at individual loci (SNVs) are integrated into a gene-specific measure of ASE, and utilizes simulations to appropriately assess the statistical significance of observed ASE.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pics'), ('version', '2.20.0')])), ('source', CommentedMap([('fn', 'PICS_2.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PICS_2.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/PICS_2.20.0.tar.gz', 'https://depot.galaxyproject.org/software/PICS/PICS_2.20.0_src_all.tar.gz']), ('md5', '9a4ac1081ae3a764caae3774aefd55b4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.3', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.1.3', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PICS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PICS.html'), ('license', 'Artistic-2.0'), ('summary', 'Probabilistic inference of ChIP-Seq using an empirical Bayes mixture model approach.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pics'), ('version', '2.20.0')])), ('source', CommentedMap([('fn', 'PICS_2.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PICS_2.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/PICS_2.20.0.tar.gz', 'https://depot.galaxyproject.org/software/PICS/PICS_2.20.0_src_all.tar.gz']), ('md5', '9a4ac1081ae3a764caae3774aefd55b4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.3', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.1.3', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PICS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PICS.html'), ('license', 'Artistic-2.0'), ('summary', 'Probabilistic inference of ChIP-Seq using an empirical Bayes mixture model approach.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/subread/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://subread.sourceforge.net/'), ('license', 'GPLv3'), ('summary', 'High-performance read alignment, quantification, and mutation discovery')])), ('package', CommentedMap([('name', 'subread'), ('version', '1.5.3')])), ('build', CommentedMap([('number', 0)])), ('test', CommentedMap([('commands', ['featureCounts -v 2>&1 > /dev/null'])])), ('source', CommentedMap([('fn', 'subread-1.5.3-MacOSX-x86_64.tar.gz'), ('url', 'https://sourceforge.net/projects/subread/files/subread-1.5.3/subread-1.5.3-MacOSX-x86_64.tar.gz'), ('md5', 'd1d9d05226484a1df184f7ef6d541ffa')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/subread/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://subread.sourceforge.net/'), ('license', 'GPLv3'), ('summary', 'High-performance read alignment, quantification, and mutation discovery')])), ('package', CommentedMap([('name', 'subread'), ('version', '1.5.3')])), ('build', CommentedMap([('number', 0)])), ('test', CommentedMap([('commands', ['featureCounts -v 2>&1 > /dev/null'])])), ('source', CommentedMap([('fn', 'subread-1.5.3-MacOSX-x86_64.tar.gz'), ('url', 'https://sourceforge.net/projects/subread/files/subread-1.5.3/subread-1.5.3-MacOSX-x86_64.tar.gz'), ('md5', 'd1d9d05226484a1df184f7ef6d541ffa')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biomformat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biomformat'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'biomformat_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/biomformat_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/biomformat_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/biomformat/biomformat_1.2.0_src_all.tar.gz']), ('md5', '0a418f48c731375953ff7389b9b6f7a0')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-rhdf5', 'r-base', 'r-jsonlite >=0.9.16', 'r-matrix >=1.2', 'r-plyr >=1.8', 'r-matrix']), ('run', ['bioconductor-rhdf5', 'r-base', 'r-jsonlite >=0.9.16', 'r-matrix >=1.2', 'r-plyr >=1.8', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biomformat\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/biomformat.html'), ('license', 'GPL-2'), ('summary', 'This is an R package for interfacing with the BIOM format. This package includes basic tools for reading biom-format files, accessing and subsetting data tables from a biom object (which is more complex than a single table), as well as limited support for writing a biom-object back to a biom-format file. The design of this API is intended to match the python API and other tools included with the biom-format project, but with a decidedly \"R flavor\" that should be familiar to R users. This includes S4 classes and methods, as well as extensions of common core functions/methods.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biomformat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biomformat'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'biomformat_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/biomformat_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/biomformat_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/biomformat/biomformat_1.2.0_src_all.tar.gz']), ('md5', '0a418f48c731375953ff7389b9b6f7a0')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-rhdf5', 'r-base', 'r-jsonlite >=0.9.16', 'r-matrix >=1.2', 'r-plyr >=1.8', 'r-matrix']), ('run', ['bioconductor-rhdf5', 'r-base', 'r-jsonlite >=0.9.16', 'r-matrix >=1.2', 'r-plyr >=1.8', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biomformat\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/biomformat.html'), ('license', 'GPL-2'), ('summary', 'This is an R package for interfacing with the BIOM format. This package includes basic tools for reading biom-format files, accessing and subsetting data tables from a biom object (which is more complex than a single table), as well as limited support for writing a biom-object back to a biom-format file. The design of this API is intended to match the python API and other tools included with the biom-format project, but with a decidedly \"R flavor\" that should be familiar to R users. This includes S4 classes and methods, as well as extensions of common core functions/methods.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ntw/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ntw'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'NTW_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/NTW_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/NTW_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/NTW/NTW_1.26.0_src_all.tar.gz']), ('md5', '5b746a15be789dcc4fe2b859f29bf0bc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mvtnorm']), ('run', ['r-base', 'r-mvtnorm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NTW\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/NTW.html'), ('license', 'GPL-2'), ('summary', 'This package predicts the gene-gene interaction network and identifies the direct transcriptional targets of the perturbation using an ODE (Ordinary Differential Equation) based method.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ntw/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ntw'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'NTW_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/NTW_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/NTW_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/NTW/NTW_1.26.0_src_all.tar.gz']), ('md5', '5b746a15be789dcc4fe2b859f29bf0bc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mvtnorm']), ('run', ['r-base', 'r-mvtnorm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NTW\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/NTW.html'), ('license', 'GPL-2'), ('summary', 'This package predicts the gene-gene interaction network and identifies the direct transcriptional targets of the perturbation using an ODE (Ordinary Differential Equation) based method.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/commet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'commet'), ('version', '24.7.14')])), ('source', CommentedMap([('fn', '6802910e8dbd50b40d569783a3b0c062e2466da6.zip'), ('url', 'https://github.com/pierrepeterlongo/commet/archive/6802910e8dbd50b40d569783a3b0c062e2466da6.zip'), ('md5', '20f906448270e2d329f32c1071344a9d'), ('patches', ['commet.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'r-base']), ('run', ['libgcc', 'r-base', 'r-gplots', 'libgfortran', 'python'])])), ('test', CommentedMap([('commands', ['Commet.py -h'])])), ('about', CommentedMap([('home', 'https://colibread.inria.fr/software/commet/'), ('license', 'GNU Affero General Public License'), ('summary', 'Comparing and combining multiple metagenomic datasets')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/commet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'commet'), ('version', '24.7.14')])), ('source', CommentedMap([('fn', '6802910e8dbd50b40d569783a3b0c062e2466da6.zip'), ('url', 'https://github.com/pierrepeterlongo/commet/archive/6802910e8dbd50b40d569783a3b0c062e2466da6.zip'), ('md5', '20f906448270e2d329f32c1071344a9d'), ('patches', ['commet.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'r-base']), ('run', ['libgcc', 'r-base', 'r-gplots', 'libgfortran', 'python'])])), ('test', CommentedMap([('commands', ['Commet.py -h'])])), ('about', CommentedMap([('home', 'https://colibread.inria.fr/software/commet/'), ('license', 'GNU Affero General Public License'), ('summary', 'Comparing and combining multiple metagenomic datasets')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedgraphtobigwig/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedgraphtobigwig'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedGraphToBigWig'), ('fn', 'bedGraphToBigWig'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedGraphToBigWig $PREFIX/bin; chmod +x $PREFIX/bin/bedGraphToBigWig')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedGraphToBigWig'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert a bedGraph file to bigWig format.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-bedgraphtobigwig/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-bedgraphtobigwig'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/bedGraphToBigWig'), ('fn', 'bedGraphToBigWig'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('script', 'mkdir -p $PREFIX/bin; cp bedGraphToBigWig $PREFIX/bin; chmod +x $PREFIX/bin/bedGraphToBigWig')])), ('requirements', CommentedMap([('build', ['libpng', 'mysql', 'zlib', 'openssl', 'gcc']), ('run', ['libpng', 'mysql', 'zlib', 'openssl', 'libgcc'])])), ('test', CommentedMap([('commands', ['which bedGraphToBigWig'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Convert a bedGraph file to bigWig format.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-diffbind/2.2.12/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-diffbind'), ('version', '2.2.12')])), ('source', CommentedMap([('fn', 'DiffBind_2.2.12.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/DiffBind_2.2.12.tar.gz', 'https://depot.galaxyproject.org/software/DiffBind/DiffBind_2.2.12_src_all.tar.gz']), ('sha256', '9ee1454c851559ac713b5ef83e22f577c801aae76f843ada3a4c033725e46d70')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-systempiper', 'bioconductor-zlibbioc', 'r-amap', 'r-base', 'r-dplyr', 'r-gplots', 'r-lattice', 'r-locfit', 'r-rcolorbrewer', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-biocparallel', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-systempiper', 'bioconductor-zlibbioc', 'r-amap', 'r-base', 'r-dplyr', 'r-gplots', 'r-lattice', 'r-locfit', 'r-rcolorbrewer', 'r-rcpp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DiffBind\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/DiffBind.html'), ('license', 'Artistic-2.0'), ('summary', 'Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-diffbind/2.2.12/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-diffbind'), ('version', '2.2.12')])), ('source', CommentedMap([('fn', 'DiffBind_2.2.12.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/DiffBind_2.2.12.tar.gz', 'https://depot.galaxyproject.org/software/DiffBind/DiffBind_2.2.12_src_all.tar.gz']), ('sha256', '9ee1454c851559ac713b5ef83e22f577c801aae76f843ada3a4c033725e46d70')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-systempiper', 'bioconductor-zlibbioc', 'r-amap', 'r-base', 'r-dplyr', 'r-gplots', 'r-lattice', 'r-locfit', 'r-rcolorbrewer', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-biocparallel', 'bioconductor-deseq2', 'bioconductor-edger', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-systempiper', 'bioconductor-zlibbioc', 'r-amap', 'r-base', 'r-dplyr', 'r-gplots', 'r-lattice', 'r-locfit', 'r-rcolorbrewer', 'r-rcpp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DiffBind\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/DiffBind.html'), ('license', 'Artistic-2.0'), ('summary', 'Compute differentially bound sites from multiple ChIP-seq experiments using affinity (quantitative) data. Also enables occupancy (overlap) analysis and plotting functions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-massspecwavelet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-massspecwavelet'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'MassSpecWavelet_1.40.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/MassSpecWavelet_1.40.0.tar.gz'), ('md5', '878b0dec9f50294ec4cb487b02de4c65')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-waveslim']), ('run', ['r-base', 'r-waveslim'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MassSpecWavelet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/MassSpecWavelet.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Processing Mass Spectrometry spectrum by using wavelet based algorithm')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-massspecwavelet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-massspecwavelet'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'MassSpecWavelet_1.40.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/MassSpecWavelet_1.40.0.tar.gz'), ('md5', '878b0dec9f50294ec4cb487b02de4c65')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-waveslim']), ('run', ['r-base', 'r-waveslim'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MassSpecWavelet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/MassSpecWavelet.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Processing Mass Spectrometry spectrum by using wavelet based algorithm')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-smap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-smap'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'SMAP_1.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SMAP_1.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/SMAP_1.40.0.tar.gz', 'https://depot.galaxyproject.org/software/SMAP/SMAP_1.40.0_src_all.tar.gz']), ('md5', '8e6d638d02011fe362a6bc7c012a77ad')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SMAP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SMAP.html'), ('license', 'GPL-2'), ('summary', 'Functions and classes for DNA copy number profiling of array-CGH data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-smap/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-smap'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'SMAP_1.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SMAP_1.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/SMAP_1.40.0.tar.gz', 'https://depot.galaxyproject.org/software/SMAP/SMAP_1.40.0_src_all.tar.gz']), ('md5', '8e6d638d02011fe362a6bc7c012a77ad')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SMAP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SMAP.html'), ('license', 'GPL-2'), ('summary', 'Functions and classes for DNA copy number profiling of array-CGH data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chipqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chipqc'), ('version', '1.10.3')])), ('source', CommentedMap([('fn', 'ChIPQC_1.10.3.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/ChIPQC_1.10.3.tar.gz', 'https://depot.galaxyproject.org/software/ChIPQC/ChIPQC_1.10.3_src_all.tar.gz']), ('sha256', 'd1dd1ce67a24baec546985941cf3f9dabfdbd933932c5d8d9f8bdc663d7c4bd9')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-chipseq >=1.12.0', 'bioconductor-diffbind', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges >=1.17.19', 'bioconductor-iranges >=1.99.17', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.1.0', 'bioconductor-txdb.celegans.ucsc.ce6.ensgene', 'bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene', 'bioconductor-txdb.hsapiens.ucsc.hg18.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg38.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm10.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm9.knowngene', 'bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene', 'r-base', 'r-ggplot2', 'r-gtools', 'r-nozzle.r1', 'r-reshape2']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-chipseq >=1.12.0', 'bioconductor-diffbind', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges >=1.17.19', 'bioconductor-iranges >=1.99.17', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.1.0', 'bioconductor-txdb.celegans.ucsc.ce6.ensgene', 'bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene', 'bioconductor-txdb.hsapiens.ucsc.hg18.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg38.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm10.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm9.knowngene', 'bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene', 'r-base', 'r-ggplot2', 'r-gtools', 'r-nozzle.r1', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ChIPQC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/ChIPQC.html'), ('license', 'GPL (>= 3)'), ('summary', 'Quality metrics for ChIPseq data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chipqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chipqc'), ('version', '1.10.3')])), ('source', CommentedMap([('fn', 'ChIPQC_1.10.3.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/ChIPQC_1.10.3.tar.gz', 'https://depot.galaxyproject.org/software/ChIPQC/ChIPQC_1.10.3_src_all.tar.gz']), ('sha256', 'd1dd1ce67a24baec546985941cf3f9dabfdbd933932c5d8d9f8bdc663d7c4bd9')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-chipseq >=1.12.0', 'bioconductor-diffbind', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges >=1.17.19', 'bioconductor-iranges >=1.99.17', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.1.0', 'bioconductor-txdb.celegans.ucsc.ce6.ensgene', 'bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene', 'bioconductor-txdb.hsapiens.ucsc.hg18.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg38.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm10.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm9.knowngene', 'bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene', 'r-base', 'r-ggplot2', 'r-gtools', 'r-nozzle.r1', 'r-reshape2']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-chipseq >=1.12.0', 'bioconductor-diffbind', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges >=1.17.19', 'bioconductor-iranges >=1.99.17', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-s4vectors >=0.1.0', 'bioconductor-txdb.celegans.ucsc.ce6.ensgene', 'bioconductor-txdb.dmelanogaster.ucsc.dm3.ensgene', 'bioconductor-txdb.hsapiens.ucsc.hg18.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg19.knowngene', 'bioconductor-txdb.hsapiens.ucsc.hg38.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm10.knowngene', 'bioconductor-txdb.mmusculus.ucsc.mm9.knowngene', 'bioconductor-txdb.rnorvegicus.ucsc.rn4.ensgene', 'r-base', 'r-ggplot2', 'r-gtools', 'r-nozzle.r1', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ChIPQC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/ChIPQC.html'), ('license', 'GPL (>= 3)'), ('summary', 'Quality metrics for ChIPseq data.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-camera/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-camera'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'CAMERA_1.28.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CAMERA_1.28.0.tar.gz'), ('md5', 'd8f8c9bcf40564f440c572ec169f4ec3')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'bioconductor-xcms >=1.13.5', 'r-base', 'r-hmisc', 'r-igraph']), ('run', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'bioconductor-xcms >=1.13.5', 'r-base', 'r-hmisc', 'r-igraph'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CAMERA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CAMERA.html'), ('license', 'GPL (>= 2)'), ('summary', 'Annotation of peaklists generated by xcms, rule based annotation of isotopes and adducts, EIC correlation based tagging of unknown adducts and fragments')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-camera/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-camera'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'CAMERA_1.28.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CAMERA_1.28.0.tar.gz'), ('md5', 'd8f8c9bcf40564f440c572ec169f4ec3')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'bioconductor-xcms >=1.13.5', 'r-base', 'r-hmisc', 'r-igraph']), ('run', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-rbgl', 'bioconductor-xcms >=1.13.5', 'r-base', 'r-hmisc', 'r-igraph'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CAMERA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CAMERA.html'), ('license', 'GPL (>= 2)'), ('summary', 'Annotation of peaklists generated by xcms, rule based annotation of isotopes and adducts, EIC correlation based tagging of unknown adducts and fragments')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ceas/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ceas'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'CEAS-Package-1.0.2.tar.gz'), ('md5', 'faf202ff613794ab9cd07f282c6e2663'), ('url', 'http://liulab.dfci.harvard.edu/CEAS/src/CEAS-Package-1.0.2.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', 'True [py3k or osx]')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'mysql-python']), ('run', ['python', 'mysql-python'])])), ('test', CommentedMap([('imports', ['CEAS']), ('commands', ['ceas --version > /dev/null', 'build_genomeBG  --version > /dev/null', 'sitepro  --version > /dev/null', 'gca --version > /dev/null'])])), ('about', CommentedMap([('home', 'http://liulab.dfci.harvard.edu/CEAS'), ('license', 'Artistic'), ('license_file', 'Artistic_License.pdf'), ('summary', 'CEAS: Cis-regulatory Element Annotation System')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ceas/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ceas'), ('version', '1.0.2')])), ('source', CommentedMap([('fn', 'CEAS-Package-1.0.2.tar.gz'), ('md5', 'faf202ff613794ab9cd07f282c6e2663'), ('url', 'http://liulab.dfci.harvard.edu/CEAS/src/CEAS-Package-1.0.2.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', 'True [py3k or osx]')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'mysql-python']), ('run', ['python', 'mysql-python'])])), ('test', CommentedMap([('imports', ['CEAS']), ('commands', ['ceas --version > /dev/null', 'build_genomeBG  --version > /dev/null', 'sitepro  --version > /dev/null', 'gca --version > /dev/null'])])), ('about', CommentedMap([('home', 'http://liulab.dfci.harvard.edu/CEAS'), ('license', 'Artistic'), ('license_file', 'Artistic_License.pdf'), ('summary', 'CEAS: Cis-regulatory Element Annotation System')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bgeedb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bgeedb'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'BgeeDB_2.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BgeeDB_2.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/BgeeDB_2.2.0.tar.gz', 'https://depot.galaxyproject.org/software/BgeeDB/BgeeDB_2.2.0_src_all.tar.gz']), ('md5', '40e0c3f4017d1ebe40bb165057d1c81d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-topgo', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-rcurl', 'r-tidyr']), ('run', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-topgo', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-rcurl', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BgeeDB\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BgeeDB.html'), ('license', 'GPL-2'), ('summary', 'A package for the annotation and gene expression data download from Bgee database, and TopAnat analysis: GO-like enrichment of anatomical terms, mapped to genes by expression patterns.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bgeedb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bgeedb'), ('version', '2.2.0')])), ('source', CommentedMap([('fn', 'BgeeDB_2.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BgeeDB_2.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/BgeeDB_2.2.0.tar.gz', 'https://depot.galaxyproject.org/software/BgeeDB/BgeeDB_2.2.0_src_all.tar.gz']), ('md5', '40e0c3f4017d1ebe40bb165057d1c81d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-topgo', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-rcurl', 'r-tidyr']), ('run', ['bioconductor-biobase', 'bioconductor-graph', 'bioconductor-topgo', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-rcurl', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BgeeDB\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BgeeDB.html'), ('license', 'GPL-2'), ('summary', 'A package for the annotation and gene expression data download from Bgee database, and TopAnat analysis: GO-like enrichment of anatomical terms, mapped to genes by expression patterns.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-paircompviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-paircompviz'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'paircompviz_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/paircompviz_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/paircompviz_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/paircompviz/paircompviz_1.14.0_src_all.tar.gz']), ('md5', 'b329f0bceb502f17cfd26caf99b9a285')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-rgraphviz', 'r-base']), ('run', ['bioconductor-rgraphviz', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'paircompviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/paircompviz.html'), ('license', 'GPL (>=3.0)'), ('summary', 'This package provides visualization of the results from the multiple (i.e. pairwise) comparison tests such as pairwise.t.test, pairwise.prop.test or pairwise.wilcox.test. The groups being compared are visualized as nodes in Hasse diagram. Such approach enables very clear and vivid depiction of which group is significantly greater than which others, especially if comparing a large number of groups.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-paircompviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-paircompviz'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'paircompviz_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/paircompviz_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/paircompviz_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/paircompviz/paircompviz_1.14.0_src_all.tar.gz']), ('md5', 'b329f0bceb502f17cfd26caf99b9a285')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-rgraphviz', 'r-base']), ('run', ['bioconductor-rgraphviz', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'paircompviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/paircompviz.html'), ('license', 'GPL (>=3.0)'), ('summary', 'This package provides visualization of the results from the multiple (i.e. pairwise) comparison tests such as pairwise.t.test, pairwise.prop.test or pairwise.wilcox.test. The groups being compared are visualized as nodes in Hasse diagram. Such approach enables very clear and vivid depiction of which group is significantly greater than which others, especially if comparing a large number of groups.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nlstradamus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nlstradamus'), ('version', 1.8)])), ('about', CommentedMap([('home', 'http://www.moseslab.csb.utoronto.ca/NLStradamus/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'NLStradamus: a simple Hidden Markov Model for nuclear localization signal prediction.')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'NLStradamus.1.8.tar.gz'), ('url', 'http://www.moseslab.csb.utoronto.ca/NLStradamus/NLStradamus/NLStradamus.1.8.tar.gz'), ('sha256', '73ef3a77c6c1132f213d44bc2a48a8182e652b01b8cd83d06b30832a682ec0d0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['NLStradamus -h'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/nlstradamus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'nlstradamus'), ('version', 1.8)])), ('about', CommentedMap([('home', 'http://www.moseslab.csb.utoronto.ca/NLStradamus/'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'NLStradamus: a simple Hidden Markov Model for nuclear localization signal prediction.')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'NLStradamus.1.8.tar.gz'), ('url', 'http://www.moseslab.csb.utoronto.ca/NLStradamus/NLStradamus/NLStradamus.1.8.tar.gz'), ('sha256', '73ef3a77c6c1132f213d44bc2a48a8182e652b01b8cd83d06b30832a682ec0d0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['NLStradamus -h'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xtandem/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.thegpm.org/TANDEM/index.html'), ('license', 'Artistic License'), ('summary', 'X! Tandem open source is software that can match tandem mass spectra with peptide sequences, in a process that has come to be known as protein identification')])), ('package', CommentedMap([('name', 'xtandem'), ('version', '15.12.15.2')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'tandem-linux-15-12-15-2.zip'), ('url', 'ftp://ftp.thegpm.org/projects/tandem/source/tandem-linux-15-12-15-2.zip'), ('md5', '40058cd84fc3f56d671d32ff393f7aa3'), ('patches', ['makefile.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'expat']), ('run', ['libgcc', 'zlib', 'expat'])])), ('test', CommentedMap([('commands', [\"xtandem help 2>&1 | grep 'TANDEM'\"])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xtandem/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.thegpm.org/TANDEM/index.html'), ('license', 'Artistic License'), ('summary', 'X! Tandem open source is software that can match tandem mass spectra with peptide sequences, in a process that has come to be known as protein identification')])), ('package', CommentedMap([('name', 'xtandem'), ('version', '15.12.15.2')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'tandem-linux-15-12-15-2.zip'), ('url', 'ftp://ftp.thegpm.org/projects/tandem/source/tandem-linux-15-12-15-2.zip'), ('md5', '40058cd84fc3f56d671d32ff393f7aa3'), ('patches', ['makefile.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'expat']), ('run', ['libgcc', 'zlib', 'expat'])])), ('test', CommentedMap([('commands', [\"xtandem help 2>&1 | grep 'TANDEM'\"])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/breakdancer/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/genome/breakdancer'), ('license', 'GPLv3'), ('summary', 'SV detection from paired end reads mapping')])), ('package', CommentedMap([('name', 'breakdancer'), ('version', '1.4.5')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'perl', 'cmake >=2.8', 'gcc', 'zlib', 'ncurses']), ('run', ['libgcc', 'perl-math-cdf', 'perl-statistics-descriptive', 'zlib'])])), ('test', CommentedMap([('commands', ['breakdancer-max 2>&1 | grep \"Usage: breakdancer-max\" > /dev/null'])])), ('source', CommentedMap([('fn', 'v1.4.5.tar.gz'), ('url', 'https://github.com/genome/breakdancer/archive/v1.4.5.tar.gz'), ('md5', '3a3fcb3731b0d9e0573c528bdcaeb18b'), ('patches', ['builddeps.patch'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/breakdancer/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/genome/breakdancer'), ('license', 'GPLv3'), ('summary', 'SV detection from paired end reads mapping')])), ('package', CommentedMap([('name', 'breakdancer'), ('version', '1.4.5')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'perl', 'cmake >=2.8', 'gcc', 'zlib', 'ncurses']), ('run', ['libgcc', 'perl-math-cdf', 'perl-statistics-descriptive', 'zlib'])])), ('test', CommentedMap([('commands', ['breakdancer-max 2>&1 | grep \"Usage: breakdancer-max\" > /dev/null'])])), ('source', CommentedMap([('fn', 'v1.4.5.tar.gz'), ('url', 'https://github.com/genome/breakdancer/archive/v1.4.5.tar.gz'), ('md5', '3a3fcb3731b0d9e0573c528bdcaeb18b'), ('patches', ['builddeps.patch'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-uniprot.ws/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-uniprot.ws'), ('version', '2.16.0')])), ('source', CommentedMap([('fn', 'UniProt.ws_2.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/UniProt.ws_2.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/UniProt.ws_2.16.0.tar.gz', 'https://depot.galaxyproject.org/software/UniProt.ws/UniProt.ws_2.16.0_src_all.tar.gz']), ('md5', 'e2e671c9899e415d9049df31523d084f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics >=0.13.8', 'r-base', 'r-rcurl', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics >=0.13.8', 'r-base', 'r-rcurl', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'UniProt.ws\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/UniProt.ws.html'), ('license', 'Artistic License 2.0'), ('summary', 'A collection of functions for retrieving, processing and repackaging the UniProt web services.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-uniprot.ws/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-uniprot.ws'), ('version', '2.16.0')])), ('source', CommentedMap([('fn', 'UniProt.ws_2.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/UniProt.ws_2.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/UniProt.ws_2.16.0.tar.gz', 'https://depot.galaxyproject.org/software/UniProt.ws/UniProt.ws_2.16.0_src_all.tar.gz']), ('md5', 'e2e671c9899e415d9049df31523d084f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics >=0.13.8', 'r-base', 'r-rcurl', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics >=0.13.8', 'r-base', 'r-rcurl', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'UniProt.ws\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/UniProt.ws.html'), ('license', 'Artistic License 2.0'), ('summary', 'A collection of functions for retrieving, processing and repackaging the UniProt web services.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affycomp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affycomp'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'affycomp_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/affycomp_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/affycomp_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/affycomp/affycomp_1.52.0_src_all.tar.gz']), ('md5', 'efa67e67701f2083fadbed99bf7d60b9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.3.3', 'r-base']), ('run', ['bioconductor-biobase >=2.3.3', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affycomp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/affycomp.html'), ('license', 'GPL (>= 2)'), ('summary', 'The package contains functions that can be used to compare expression measures for Affymetrix Oligonucleotide Arrays.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affycomp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affycomp'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'affycomp_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/affycomp_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/affycomp_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/affycomp/affycomp_1.52.0_src_all.tar.gz']), ('md5', 'efa67e67701f2083fadbed99bf7d60b9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.3.3', 'r-base']), ('run', ['bioconductor-biobase >=2.3.3', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affycomp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/affycomp.html'), ('license', 'GPL (>= 2)'), ('summary', 'The package contains functions that can be used to compare expression measures for Affymetrix Oligonucleotide Arrays.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-globaltest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-globaltest'), ('version', '5.30.0')])), ('source', CommentedMap([('fn', 'globaltest_5.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/globaltest_5.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/globaltest_5.30.0.tar.gz', 'https://depot.galaxyproject.org/software/globaltest/globaltest_5.30.0_src_all.tar.gz']), ('md5', 'baddf5286a53396d041bd459d54dc4af')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'r-base', 'r-survival']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'r-base', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'globaltest\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/globaltest.html'), ('license', 'GPL (>= 2)'), ('summary', 'The global test tests groups of covariates (or features) for association with a response variable. This package implements the test with diagnostic plots and multiple testing utilities, along with several functions to facilitate the use of this test for gene set testing of GO and KEGG terms.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-globaltest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-globaltest'), ('version', '5.30.0')])), ('source', CommentedMap([('fn', 'globaltest_5.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/globaltest_5.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/globaltest_5.30.0.tar.gz', 'https://depot.galaxyproject.org/software/globaltest/globaltest_5.30.0_src_all.tar.gz']), ('md5', 'baddf5286a53396d041bd459d54dc4af')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'r-base', 'r-survival']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'r-base', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'globaltest\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/globaltest.html'), ('license', 'GPL (>= 2)'), ('summary', 'The global test tests groups of covariates (or features) for association with a response variable. This package implements the test with diagnostic plots and multiple testing utilities, along with several functions to facilitate the use of this test for gene set testing of GO and KEGG terms.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ribotaper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ribotaper'), ('version', '1.3.1a')])), ('source', CommentedMap([('fn', 'RiboTaper-1.3.1a.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/RiboTaper/RiboTaper-1.3.1a.tar.gz'), ('md5', '5f753a25646e7f5bb3e3f1865129e1f5')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['bedtools ==2.17.0', 'samtools', 'r-iterators', 'r-foreach', 'r-domc', 'r-ade4', 'r-seqinr', 'r-multitaper', 'r-xnomial']), ('run', ['bedtools ==2.17.0', 'samtools', 'r-iterators', 'r-foreach', 'r-domc', 'r-ade4', 'r-seqinr', 'r-multitaper', 'r-xnomial'])])), ('test', CommentedMap([('commands', ['Ribotaper.sh 2>&1  | grep Usage > /dev/null', 'create_annotations_files.bash 2>&1  | grep Usage > /dev/null', 'create_metaplots.bash 2>&1  | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/RiboTaper_126/'), ('license', 'GPL'), ('summary', 'RiboTaper is a new analysis pipeline for Ribosome Profiling (Ribo-seq) experiments, which exploits the triplet periodicity of ribosomal footprints to call translated regions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ribotaper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ribotaper'), ('version', '1.3.1a')])), ('source', CommentedMap([('fn', 'RiboTaper-1.3.1a.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/RiboTaper/RiboTaper-1.3.1a.tar.gz'), ('md5', '5f753a25646e7f5bb3e3f1865129e1f5')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['bedtools ==2.17.0', 'samtools', 'r-iterators', 'r-foreach', 'r-domc', 'r-ade4', 'r-seqinr', 'r-multitaper', 'r-xnomial']), ('run', ['bedtools ==2.17.0', 'samtools', 'r-iterators', 'r-foreach', 'r-domc', 'r-ade4', 'r-seqinr', 'r-multitaper', 'r-xnomial'])])), ('test', CommentedMap([('commands', ['Ribotaper.sh 2>&1  | grep Usage > /dev/null', 'create_annotations_files.bash 2>&1  | grep Usage > /dev/null', 'create_metaplots.bash 2>&1  | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/RiboTaper_126/'), ('license', 'GPL'), ('summary', 'RiboTaper is a new analysis pipeline for Ribosome Profiling (Ribo-seq) experiments, which exploits the triplet periodicity of ribosomal footprints to call translated regions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metfrag/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://c-ruttkies.github.io/MetFrag/'), ('license', 'GNU Lesser General Public License version 2.1 or later.'), ('summary', 'MetFrag is a freely available software for the annotation of high precision tandem mass spectra of metabolites which is a first and critical step for the identification of a molecular structure. Candidate molecules of different databases are fragmented in silico and matched against mass to charge values. A score calculated using the fragment peak matches gives hints to the quality of the candidate spectrum assignment.')])), ('package', CommentedMap([('name', 'metfrag'), ('version', '2.3.1')])), ('source', CommentedMap([('fn', 'MetFrag2.3.1-CL.jar'), ('url', 'http://msbi.ipb-halle.de/~cruttkie/metfrag/MetFrag2.3.1-CL.jar'), ('sha256', '4261ea5abd023016bd99ef5fdd197eb74306586a4a1d637c87d45e36809d21f4')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk >=7'])])), ('test', CommentedMap([('commands', ['metfrag --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/metfrag/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://c-ruttkies.github.io/MetFrag/'), ('license', 'GNU Lesser General Public License version 2.1 or later.'), ('summary', 'MetFrag is a freely available software for the annotation of high precision tandem mass spectra of metabolites which is a first and critical step for the identification of a molecular structure. Candidate molecules of different databases are fragmented in silico and matched against mass to charge values. A score calculated using the fragment peak matches gives hints to the quality of the candidate spectrum assignment.')])), ('package', CommentedMap([('name', 'metfrag'), ('version', '2.3.1')])), ('source', CommentedMap([('fn', 'MetFrag2.3.1-CL.jar'), ('url', 'http://msbi.ipb-halle.de/~cruttkie/metfrag/MetFrag2.3.1-CL.jar'), ('sha256', '4261ea5abd023016bd99ef5fdd197eb74306586a4a1d637c87d45e36809d21f4')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk >=7'])])), ('test', CommentedMap([('commands', ['metfrag --help'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flexbar/2.5.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flexbar'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/seqan/flexbar/archive/v<{ version }}.tar.gz'), ('fn', 'flexbar_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'tbb', 'seqan-library ==1.4.2', 'bzip2', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'tbb', 'seqan-library ==1.4.2', 'bzip2', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['flexbar -h'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/flexbar'), ('license', 'BSD-3-Clause'), ('summary', 'Flexible barcode and adapter removal')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/flexbar/2.5.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'flexbar'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/seqan/flexbar/archive/v<{ version }}.tar.gz'), ('fn', 'flexbar_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'tbb', 'seqan-library ==1.4.2', 'bzip2', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'tbb', 'seqan-library ==1.4.2', 'bzip2', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['flexbar -h'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/flexbar'), ('license', 'BSD-3-Clause'), ('summary', 'Flexible barcode and adapter removal')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-tripal/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-tripal'), ('version', '2.0.4')])), ('source', CommentedMap([('fn', 'tripal-2.0.4.tar.gz'), ('url', 'https://pypi.python.org/packages/35/69/30bf090fb1eb0f0418506412f308c14a83490084b79f72d1c5a5990313c6/tripal-2.0.4.tar.gz'), ('md5', '8aa799a7be8ab2788b376a7bdfd7c728')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=2.4.3', 'wrapt', 'click', 'pyyaml']), ('run', ['python', 'requests >=2.4.3', 'wrapt', 'click', 'pyyaml', 'future'])])), ('test', CommentedMap([('imports', ['tripal', 'tripal.analysis', 'tripal.db', 'tripal.feature', 'tripal.job', 'tripal.organism']), ('commands', ['LANG=C.UTF-8 tripaille --help > /dev/null', 'LANG=C.UTF-8 tripaille analysis --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/galaxy-genome-annotation/python-tripal'), ('license', 'MIT License'), ('summary', 'Tripal API library'), ('license_family', 'MIT')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/python-tripal/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'python-tripal'), ('version', '2.0.4')])), ('source', CommentedMap([('fn', 'tripal-2.0.4.tar.gz'), ('url', 'https://pypi.python.org/packages/35/69/30bf090fb1eb0f0418506412f308c14a83490084b79f72d1c5a5990313c6/tripal-2.0.4.tar.gz'), ('md5', '8aa799a7be8ab2788b376a7bdfd7c728')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests >=2.4.3', 'wrapt', 'click', 'pyyaml']), ('run', ['python', 'requests >=2.4.3', 'wrapt', 'click', 'pyyaml', 'future'])])), ('test', CommentedMap([('imports', ['tripal', 'tripal.analysis', 'tripal.db', 'tripal.feature', 'tripal.job', 'tripal.organism']), ('commands', ['LANG=C.UTF-8 tripaille --help > /dev/null', 'LANG=C.UTF-8 tripaille analysis --help > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/galaxy-genome-annotation/python-tripal'), ('license', 'MIT License'), ('summary', 'Tripal API library'), ('license_family', 'MIT')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))]))\n",
      "EXTRA has been found: tripal\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sagenhaft/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sagenhaft'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'sagenhaft_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sagenhaft_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/sagenhaft/sagenhaft_1.46.0_src_all.tar.gz']), ('md5', 'f093d7720c376de3edf1795cc487f2dd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-sparsem >=0.73']), ('run', ['r-base', 'r-sparsem >=0.73'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sagenhaft\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sagenhaft.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package implements several functions useful for analysis of gene expression data by sequencing tags as done in SAGE (Serial Analysis of Gene Expressen) data, i.e. extraction of a SAGE library from sequence files, sequence error correction, library comparison. Sequencing error correction is implementing using an Expectation Maximization Algorithm based on a Mixture Model of tag counts.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sagenhaft/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sagenhaft'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'sagenhaft_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sagenhaft_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/sagenhaft/sagenhaft_1.46.0_src_all.tar.gz']), ('md5', 'f093d7720c376de3edf1795cc487f2dd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-sparsem >=0.73']), ('run', ['r-base', 'r-sparsem >=0.73'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sagenhaft\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sagenhaft.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package implements several functions useful for analysis of gene expression data by sequencing tags as done in SAGE (Serial Analysis of Gene Expressen) data, i.e. extraction of a SAGE library from sequence files, sequence error correction, library comparison. Sequencing error correction is implementing using an Expectation Maximization Algorithm based on a Mixture Model of tag counts.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igdiscover/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://igdiscover.se/'), ('license', 'MIT'), ('summary', 'Analyze antibody repertoires and discover new V genes')])), ('package', CommentedMap([('name', 'igdiscover'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'igdiscover-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/i/igdiscover/igdiscover-<{ version }}.tar.gz'), ('md5', '8322eb84ac78d91fdcd0f9fc4a6ba8e8')])), ('requirements', CommentedMap([('run', ['python', 'sqt >=0.8.0', 'pandas >=0.16.2', 'numpy', 'matplotlib', 'snakemake >=3.9.0', 'cutadapt', 'xopen >=0.1.1', 'seaborn >=0.6,<0.8', 'scipy >=0.16.1', 'ruamel.yaml', 'muscle', 'pear', 'flash', 'igblast ==1.5.0', 'vsearch ==1.9.7', 'fastqc']), ('build', ['python', 'sqt >=0.8.0', 'pandas >=0.16.2', 'numpy', 'matplotlib', 'snakemake >=3.9.0', 'cutadapt', 'xopen >=0.1.1', 'seaborn >=0.6,<0.8', 'scipy >=0.16.1', 'ruamel.yaml'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('script', 'python3 setup.py install')])), ('test', CommentedMap([('commands', ['igdiscover --help > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igdiscover/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://igdiscover.se/'), ('license', 'MIT'), ('summary', 'Analyze antibody repertoires and discover new V genes')])), ('package', CommentedMap([('name', 'igdiscover'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'igdiscover-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/i/igdiscover/igdiscover-<{ version }}.tar.gz'), ('md5', '8322eb84ac78d91fdcd0f9fc4a6ba8e8')])), ('requirements', CommentedMap([('run', ['python', 'sqt >=0.8.0', 'pandas >=0.16.2', 'numpy', 'matplotlib', 'snakemake >=3.9.0', 'cutadapt', 'xopen >=0.1.1', 'seaborn >=0.6,<0.8', 'scipy >=0.16.1', 'ruamel.yaml', 'muscle', 'pear', 'flash', 'igblast ==1.5.0', 'vsearch ==1.9.7', 'fastqc']), ('build', ['python', 'sqt >=0.8.0', 'pandas >=0.16.2', 'numpy', 'matplotlib', 'snakemake >=3.9.0', 'cutadapt', 'xopen >=0.1.1', 'seaborn >=0.6,<0.8', 'scipy >=0.16.1', 'ruamel.yaml'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('script', 'python3 setup.py install')])), ('test', CommentedMap([('commands', ['igdiscover --help > /dev/null'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-causalr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-causalr'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'CausalR_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CausalR_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/CausalR_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/CausalR/CausalR_1.8.0_src_all.tar.gz']), ('md5', '7ce1907a1bb84e05edff6e99694ac0e9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph']), ('run', ['r-base', 'r-igraph'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CausalR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CausalR.html'), ('license', 'GPL (>= 2)'), ('summary', 'Causal network analysis methods for regulator prediction and network reconstruction from genome scale data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-causalr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-causalr'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'CausalR_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CausalR_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/CausalR_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/CausalR/CausalR_1.8.0_src_all.tar.gz']), ('md5', '7ce1907a1bb84e05edff6e99694ac0e9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph']), ('run', ['r-base', 'r-igraph'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CausalR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CausalR.html'), ('license', 'GPL (>= 2)'), ('summary', 'Causal network analysis methods for regulator prediction and network reconstruction from genome scale data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prank/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'prank'), ('version', 'v.150803')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'prank.source.150803.tgz'), ('url', 'http://wasabiapp.org/download/prank/prank.source.150803.tgz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'http://wasabiapp.org/software/prank/'), ('license', 'GPL-3'), ('summary', 'PRANK is a probabilistic multiple alignment program for DNA, codon and amino-acid sequences.')])), ('test', CommentedMap([('commands', ['prank -version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prank/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'prank'), ('version', 'v.150803')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'prank.source.150803.tgz'), ('url', 'http://wasabiapp.org/download/prank/prank.source.150803.tgz')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'http://wasabiapp.org/software/prank/'), ('license', 'GPL-3'), ('summary', 'PRANK is a probabilistic multiple alignment program for DNA, codon and amino-acid sequences.')])), ('test', CommentedMap([('commands', ['prank -version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-subseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-subseq'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'subSeq_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/subSeq_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/subSeq_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/subSeq/subSeq_1.6.0_src_all.tar.gz']), ('md5', 'c45c6d6082f6f0c36553f4bc3e5aaf0d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-qvalue >=1.99', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-ggplot2', 'r-magrittr', 'r-tidyr']), ('run', ['bioconductor-biobase', 'bioconductor-qvalue >=1.99', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-ggplot2', 'r-magrittr', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'subSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/subSeq.html'), ('license', 'MIT + file LICENSE'), ('summary', 'Subsampling of high throughput sequencing count data for use in experiment design and analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-subseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-subseq'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'subSeq_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/subSeq_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/subSeq_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/subSeq/subSeq_1.6.0_src_all.tar.gz']), ('md5', 'c45c6d6082f6f0c36553f4bc3e5aaf0d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-qvalue >=1.99', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-ggplot2', 'r-magrittr', 'r-tidyr']), ('run', ['bioconductor-biobase', 'bioconductor-qvalue >=1.99', 'r-base', 'r-data.table', 'r-digest', 'r-dplyr', 'r-ggplot2', 'r-magrittr', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'subSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/subSeq.html'), ('license', 'MIT + file LICENSE'), ('summary', 'Subsampling of high throughput sequencing count data for use in experiment design and analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-splinter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-splinter'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'SPLINTER_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SPLINTER_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/SPLINTER_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/SPLINTER/SPLINTER_1.2.0_src_all.tar.gz']), ('md5', '3ecdf9ad450532bdb2b6cfbabb80b8e6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-seqlogo', 'r-base', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-seqlogo', 'r-base', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SPLINTER\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SPLINTER.html'), ('license', 'GPL-2'), ('summary', 'SPLINTER provides tools to analyze alternative splicing sites, interpret outcomes based on sequence information, select and design primers for site validiation and give visual representation of the event to guide downstream experiments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-splinter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-splinter'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'SPLINTER_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SPLINTER_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/SPLINTER_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/SPLINTER/SPLINTER_1.2.0_src_all.tar.gz']), ('md5', '3ecdf9ad450532bdb2b6cfbabb80b8e6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-seqlogo', 'r-base', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-bsgenome.mmusculus.ucsc.mm9', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-seqlogo', 'r-base', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SPLINTER\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SPLINTER.html'), ('license', 'GPL-2'), ('summary', 'SPLINTER provides tools to analyze alternative splicing sites, interpret outcomes based on sequence information, select and design primers for site validiation and give visual representation of the event to guide downstream experiments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stamp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'stamp'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'STAMP-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/s/stamp/STAMP-<{ version }}.tar.gz'), ('md5', '81a5e0df4094d65007c3c27bf1796ec8'), ('patches', ['stamp.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy >=1.9.1', 'scipy >=0.15.1', 'matplotlib >=1.4.2', 'six >=1.3', 'biom-format >=2.0.1', 'pyqi >=0.3.2']), ('run', ['python', 'numpy >=1.9.1', 'scipy >=0.15.1', 'matplotlib >=1.4.2', 'six >=1.3', 'biom-format >=2.0.1', 'pyqi >=0.3.2', 'pyqt 4.*'])])), ('test', CommentedMap([('imports', ['stamp', 'stamp.GUI', 'stamp.metagenomics', 'stamp.metagenomics.fileIO', 'stamp.metagenomics.simulations', 'stamp.metagenomics.stats', 'stamp.metagenomics.stats.CI', 'stamp.metagenomics.stats.distributions', 'stamp.metagenomics.stats.empiricalTests', 'stamp.metagenomics.stats.tests', 'stamp.plugins', 'stamp.plugins.common', 'stamp.plugins.common.multipleComparisonCorrections', 'stamp.plugins.groups', 'stamp.plugins.groups.effectSizeFilters', 'stamp.plugins.groups.plots', 'stamp.plugins.groups.plots.configGUI', 'stamp.plugins.groups.statisticalTests', 'stamp.plugins.multiGroups', 'stamp.plugins.multiGroups.effectSizeFilters', 'stamp.plugins.multiGroups.plots', 'stamp.plugins.multiGroups.plots.configGUI', 'stamp.plugins.multiGroups.postHoc', 'stamp.plugins.multiGroups.statisticalTests', 'stamp.plugins.samples', 'stamp.plugins.samples.confidenceIntervalMethods', 'stamp.plugins.samples.effectSizeFilters', 'stamp.plugins.samples.plots', 'stamp.plugins.samples.plots.configGUI', 'stamp.plugins.samples.statisticalTests']), ('commands', ['checkHierarchy.py -h'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/stamp/'), ('license', 'GPL-3.0'), ('license_file', 'LICENSE.txt'), ('summary', 'A graphical software package for analyzing taxonomic and functional profiles.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stamp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'stamp'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'STAMP-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/s/stamp/STAMP-<{ version }}.tar.gz'), ('md5', '81a5e0df4094d65007c3c27bf1796ec8'), ('patches', ['stamp.patch'])])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'numpy >=1.9.1', 'scipy >=0.15.1', 'matplotlib >=1.4.2', 'six >=1.3', 'biom-format >=2.0.1', 'pyqi >=0.3.2']), ('run', ['python', 'numpy >=1.9.1', 'scipy >=0.15.1', 'matplotlib >=1.4.2', 'six >=1.3', 'biom-format >=2.0.1', 'pyqi >=0.3.2', 'pyqt 4.*'])])), ('test', CommentedMap([('imports', ['stamp', 'stamp.GUI', 'stamp.metagenomics', 'stamp.metagenomics.fileIO', 'stamp.metagenomics.simulations', 'stamp.metagenomics.stats', 'stamp.metagenomics.stats.CI', 'stamp.metagenomics.stats.distributions', 'stamp.metagenomics.stats.empiricalTests', 'stamp.metagenomics.stats.tests', 'stamp.plugins', 'stamp.plugins.common', 'stamp.plugins.common.multipleComparisonCorrections', 'stamp.plugins.groups', 'stamp.plugins.groups.effectSizeFilters', 'stamp.plugins.groups.plots', 'stamp.plugins.groups.plots.configGUI', 'stamp.plugins.groups.statisticalTests', 'stamp.plugins.multiGroups', 'stamp.plugins.multiGroups.effectSizeFilters', 'stamp.plugins.multiGroups.plots', 'stamp.plugins.multiGroups.plots.configGUI', 'stamp.plugins.multiGroups.postHoc', 'stamp.plugins.multiGroups.statisticalTests', 'stamp.plugins.samples', 'stamp.plugins.samples.confidenceIntervalMethods', 'stamp.plugins.samples.effectSizeFilters', 'stamp.plugins.samples.plots', 'stamp.plugins.samples.plots.configGUI', 'stamp.plugins.samples.statisticalTests']), ('commands', ['checkHierarchy.py -h'])])), ('about', CommentedMap([('home', 'http://pypi.python.org/pypi/stamp/'), ('license', 'GPL-3.0'), ('license_file', 'LICENSE.txt'), ('summary', 'A graphical software package for analyzing taxonomic and functional profiles.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seq2hla/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'seq2hla'), ('version', '2.2')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '95e554b1d191.zip'), ('url', 'https://bitbucket.org/sebastian_boegel/seq2hla/get/95e554b1d191.zip'), ('patches', ['seq2HLA.patch'])])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'biopython >=1.58', 'bowtie ==1.1.2', 'r-base'])])), ('test', CommentedMap([('commands', ['seq2HLA -h'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/sebastian_boegel/seq2hla'), ('license', 'MIT'), ('summary', 'Precision HLA typing and expression from next-generation RNA sequencing data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seq2hla/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'seq2hla'), ('version', '2.2')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', '95e554b1d191.zip'), ('url', 'https://bitbucket.org/sebastian_boegel/seq2hla/get/95e554b1d191.zip'), ('patches', ['seq2HLA.patch'])])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'biopython >=1.58', 'bowtie ==1.1.2', 'r-base'])])), ('test', CommentedMap([('commands', ['seq2HLA -h'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/sebastian_boegel/seq2hla'), ('license', 'MIT'), ('summary', 'Precision HLA typing and expression from next-generation RNA sequencing data')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metaseq'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'metaSeq_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/metaSeq_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/metaSeq_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/metaSeq/metaSeq_1.16.0_src_all.tar.gz']), ('md5', '3212d61d0c3a8572b21d5cd4b28a34e4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-noiseq', 'r-base', 'r-rcpp', 'r-snow', 'gcc', 'llvm']), ('run', ['bioconductor-noiseq', 'r-base', 'r-rcpp', 'r-snow'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metaSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/metaSeq.html'), ('license', 'Artistic-2.0'), ('summary', \"The probabilities by one-sided NOISeq are combined by Fisher's method or Stouffer's method\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metaseq'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'metaSeq_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/metaSeq_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/metaSeq_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/metaSeq/metaSeq_1.16.0_src_all.tar.gz']), ('md5', '3212d61d0c3a8572b21d5cd4b28a34e4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-noiseq', 'r-base', 'r-rcpp', 'r-snow', 'gcc', 'llvm']), ('run', ['bioconductor-noiseq', 'r-base', 'r-rcpp', 'r-snow'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metaSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/metaSeq.html'), ('license', 'Artistic-2.0'), ('summary', \"The probabilities by one-sided NOISeq are combined by Fisher's method or Stouffer's method\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/piranha/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://smithlabresearch.org/software/piranha/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Piranha is a peak-caller for CLIP- and RIP-Seq data.')])), ('package', CommentedMap([('name', 'piranha'), ('version', '1.2.1')])), ('source', CommentedMap([('fn', 'piranha-1.2.1.tar.gz'), ('url', 'http://smithlabresearch.org/downloads/piranha-1.2.1.tar.gz'), ('md5', '4ca70055b54ae0212d2b3641aafcb976'), ('patches', ['0001-compile-with-conda-gsl-patch.patch'])])), ('build', CommentedMap([('number', 2), ('skip', True), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['autoconf', 'gcc', 'llvm', 'gawk', 'python', 'bamtools', 'gsl <{CONDA_GSL}}*']), ('run', ['libgcc', 'bamtools', 'gsl <{CONDA_GSL}}*'])])), ('test', CommentedMap([('commands', ['Piranha 2>&1 | grep \"^Usage\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/piranha/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://smithlabresearch.org/software/piranha/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Piranha is a peak-caller for CLIP- and RIP-Seq data.')])), ('package', CommentedMap([('name', 'piranha'), ('version', '1.2.1')])), ('source', CommentedMap([('fn', 'piranha-1.2.1.tar.gz'), ('url', 'http://smithlabresearch.org/downloads/piranha-1.2.1.tar.gz'), ('md5', '4ca70055b54ae0212d2b3641aafcb976'), ('patches', ['0001-compile-with-conda-gsl-patch.patch'])])), ('build', CommentedMap([('number', 2), ('skip', True), ('string', 'gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['autoconf', 'gcc', 'llvm', 'gawk', 'python', 'bamtools', 'gsl <{CONDA_GSL}}*']), ('run', ['libgcc', 'bamtools', 'gsl <{CONDA_GSL}}*'])])), ('test', CommentedMap([('commands', ['Piranha 2>&1 | grep \"^Usage\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/atac/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://kmer.sourceforge.net/wiki/index.php/Overview_of_the_ATAC_process'), ('license', 'GPL'), ('summary', 'ATAC is a computational process for comparative mapping between two genome assemblies, or between two different genomes')])), ('source', CommentedMap([('fn', 'ATAC-r2008.tar.bz2'), ('url', 'http://sourceforge.net/projects/kmer/files/ATAC-r2008.tar.bz2'), ('md5', 'e23ef1d41836b29332b70271b7c9eb74')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'perl', 'bzip2']), ('run', ['libgcc', 'perl', 'python'])])), ('test', CommentedMap([('commands', [\"atac.pl 2>&1 | grep 'atac.pl'\"])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/atac/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://kmer.sourceforge.net/wiki/index.php/Overview_of_the_ATAC_process'), ('license', 'GPL'), ('summary', 'ATAC is a computational process for comparative mapping between two genome assemblies, or between two different genomes')])), ('source', CommentedMap([('fn', 'ATAC-r2008.tar.bz2'), ('url', 'http://sourceforge.net/projects/kmer/files/ATAC-r2008.tar.bz2'), ('md5', 'e23ef1d41836b29332b70271b7c9eb74')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'perl', 'bzip2']), ('run', ['libgcc', 'perl', 'python'])])), ('test', CommentedMap([('commands', [\"atac.pl 2>&1 | grep 'atac.pl'\"])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rseqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rseqc'), ('version', '2.6.4')])), ('source', CommentedMap([('fn', 'RSeQC-2.6.4.tar.gz'), ('url', 'https://pypi.python.org/packages/61/58/1d9c280d8d852f7c5e873c0635952fc19305b0805e9f0ace50e8ef2f93c3/RSeQC-2.6.4.tar.gz'), ('md5', '935779c452ffc84f3b8b9fb3d485c782')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython >=0.17', 'pysam', 'bx-python', 'numpy', 'nose']), ('run', ['python', 'cython >=0.17', 'pysam', 'bx-python', 'numpy', 'r-base'])])), ('test', CommentedMap([('imports', ['qcmodule', 'qcmodule.SAM']), ('commands', ['read_distribution.py 2>&1 | grep Usage > /dev/null', 'geneBody_coverage.py 2>&1 | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'http://rseqc.sourceforge.net/'), ('license', 'GNU General Public License v2 (GPLv2)'), ('summary', 'RNA-seq QC Package'), ('license_family', 'GPL2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rseqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rseqc'), ('version', '2.6.4')])), ('source', CommentedMap([('fn', 'RSeQC-2.6.4.tar.gz'), ('url', 'https://pypi.python.org/packages/61/58/1d9c280d8d852f7c5e873c0635952fc19305b0805e9f0ace50e8ef2f93c3/RSeQC-2.6.4.tar.gz'), ('md5', '935779c452ffc84f3b8b9fb3d485c782')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython >=0.17', 'pysam', 'bx-python', 'numpy', 'nose']), ('run', ['python', 'cython >=0.17', 'pysam', 'bx-python', 'numpy', 'r-base'])])), ('test', CommentedMap([('imports', ['qcmodule', 'qcmodule.SAM']), ('commands', ['read_distribution.py 2>&1 | grep Usage > /dev/null', 'geneBody_coverage.py 2>&1 | grep Usage > /dev/null'])])), ('about', CommentedMap([('home', 'http://rseqc.sourceforge.net/'), ('license', 'GNU General Public License v2 (GPLv2)'), ('summary', 'RNA-seq QC Package'), ('license_family', 'GPL2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbm'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'RBM_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RBM_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/RBM_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/RBM/RBM_1.8.0_src_all.tar.gz']), ('md5', 'd773904a351f356c9a08a884f3af855e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-marray', 'r-base']), ('run', ['bioconductor-limma', 'bioconductor-marray', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RBM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RBM.html'), ('license', 'GPL (>= 2)'), ('summary', 'Use A Resampling-Based Empirical Bayes Approach to Assess Differential Expression in Two-Color Microarrays and RNA-Seq data sets.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbm'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'RBM_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RBM_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/RBM_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/RBM/RBM_1.8.0_src_all.tar.gz']), ('md5', 'd773904a351f356c9a08a884f3af855e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-marray', 'r-base']), ('run', ['bioconductor-limma', 'bioconductor-marray', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RBM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RBM.html'), ('license', 'GPL (>= 2)'), ('summary', 'Use A Resampling-Based Empirical Bayes Approach to Assess Differential Expression in Two-Color Microarrays and RNA-Seq data sets.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-phyloseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-phyloseq'), ('version', '1.19.1')])), ('source', CommentedMap([('fn', 'phyloseq_1.19.1.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/phyloseq_1.19.1.tar.gz'), ('md5', 'b8daecd87cf8528c8abed4ab0b8729d3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.14.0', 'bioconductor-biomformat >=0.4.0', 'bioconductor-biostrings >=2.28.0', 'bioconductor-multtest >=2.16.0', 'r-base', 'r-ade4 >=1.7.2', 'r-ape >=3.1.1', 'r-data.table >=1.9.6', 'r-foreach >=1.4.2', 'r-ggplot2 >=2.1.0', 'r-igraph >=0.7.0', 'r-plyr >=1.8', 'r-reshape2 >=1.2.2', 'r-scales >=0.3.0', 'r-vegan >=2.0.10', 'gmp']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.14.0', 'bioconductor-biomformat >=0.4.0', 'bioconductor-biostrings >=2.28.0', 'bioconductor-multtest >=2.16.0', 'r-base', 'r-ade4 >=1.7.2', 'r-ape >=3.1.1', 'r-data.table >=1.9.6', 'r-foreach >=1.4.2', 'r-ggplot2 >=2.1.0', 'r-igraph >=0.7.0', 'r-plyr >=1.8', 'r-reshape2 >=1.2.2', 'r-scales >=0.3.0', 'r-vegan >=2.0.10', 'gmp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'phyloseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/phyloseq.html'), ('license', 'AGPL-3'), ('summary', 'phyloseq provides a set of classes and tools to facilitate the import, storage, analysis, and graphical display of microbiome census data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-phyloseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-phyloseq'), ('version', '1.19.1')])), ('source', CommentedMap([('fn', 'phyloseq_1.19.1.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/phyloseq_1.19.1.tar.gz'), ('md5', 'b8daecd87cf8528c8abed4ab0b8729d3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.14.0', 'bioconductor-biomformat >=0.4.0', 'bioconductor-biostrings >=2.28.0', 'bioconductor-multtest >=2.16.0', 'r-base', 'r-ade4 >=1.7.2', 'r-ape >=3.1.1', 'r-data.table >=1.9.6', 'r-foreach >=1.4.2', 'r-ggplot2 >=2.1.0', 'r-igraph >=0.7.0', 'r-plyr >=1.8', 'r-reshape2 >=1.2.2', 'r-scales >=0.3.0', 'r-vegan >=2.0.10', 'gmp']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.14.0', 'bioconductor-biomformat >=0.4.0', 'bioconductor-biostrings >=2.28.0', 'bioconductor-multtest >=2.16.0', 'r-base', 'r-ade4 >=1.7.2', 'r-ape >=3.1.1', 'r-data.table >=1.9.6', 'r-foreach >=1.4.2', 'r-ggplot2 >=2.1.0', 'r-igraph >=0.7.0', 'r-plyr >=1.8', 'r-reshape2 >=1.2.2', 'r-scales >=0.3.0', 'r-vegan >=2.0.10', 'gmp'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'phyloseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/phyloseq.html'), ('license', 'AGPL-3'), ('summary', 'phyloseq provides a set of classes and tools to facilitate the import, storage, analysis, and graphical display of microbiome census data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-switchde/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-switchde'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'switchde_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/switchde_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/switchde/switchde_1.2.0_src_all.tar.gz']), ('md5', 'ce5a7cf7527304adcaeee2f1bcb46cca')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-dplyr', 'r-ggplot2']), ('run', ['bioconductor-biobase', 'r-base', 'r-dplyr', 'r-ggplot2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'switchde\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/switchde.html'), ('license', 'GPL (>= 2)'), ('summary', 'Inference and detection of switch-like differential expression across single-cell RNA-seq trajectories.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-switchde/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-switchde'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'switchde_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/switchde_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/switchde/switchde_1.2.0_src_all.tar.gz']), ('md5', 'ce5a7cf7527304adcaeee2f1bcb46cca')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-dplyr', 'r-ggplot2']), ('run', ['bioconductor-biobase', 'r-base', 'r-dplyr', 'r-ggplot2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'switchde\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/switchde.html'), ('license', 'GPL (>= 2)'), ('summary', 'Inference and detection of switch-like differential expression across single-cell RNA-seq trajectories.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghbase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghbase'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'CGHbase_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CGHbase_1.32.0.tar.gz'), ('md5', '905e8bac8f42dd1b0eaf22bdb148743a')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-marray', 'r-base']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-marray', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CGHbase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CGHbase.html'), ('license', 'GPL'), ('summary', 'Contains functions and classes that are needed by arrayCGH packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghbase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghbase'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'CGHbase_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/CGHbase_1.32.0.tar.gz'), ('md5', '905e8bac8f42dd1b0eaf22bdb148743a')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-marray', 'r-base']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-marray', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CGHbase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/CGHbase.html'), ('license', 'GPL'), ('summary', 'Contains functions and classes that are needed by arrayCGH packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-illuminaio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-illuminaio'), ('version', '0.14.0')])), ('source', CommentedMap([('fn', 'illuminaio_0.14.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/illuminaio_0.14.0.tar.gz'), ('md5', '53189e29b2e2a8d6ca3a2a86f9f0c53f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-base64']), ('run', ['r-base', 'r-base64'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'illuminaio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/illuminaio.html'), ('license', 'GPL-2'), ('summary', \"Tools for parsing Illumina's microarray output files, including IDAT.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-illuminaio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-illuminaio'), ('version', '0.14.0')])), ('source', CommentedMap([('fn', 'illuminaio_0.14.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/illuminaio_0.14.0.tar.gz'), ('md5', '53189e29b2e2a8d6ca3a2a86f9f0c53f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-base64']), ('run', ['r-base', 'r-base64'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'illuminaio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/illuminaio.html'), ('license', 'GPL-2'), ('summary', \"Tools for parsing Illumina's microarray output files, including IDAT.\")]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationfuncs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationfuncs'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'AnnotationFuncs_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AnnotationFuncs_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/AnnotationFuncs_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/AnnotationFuncs/AnnotationFuncs_1.26.0_src_all.tar.gz']), ('md5', '6ff063493468ddf5598f370a583f0799')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'r-base', 'r-dbi']), ('run', ['bioconductor-annotationdbi', 'r-base', 'r-dbi'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationFuncs\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AnnotationFuncs.html'), ('license', 'GPL-2'), ('summary', 'Functions for handling translating between different identifieres using the Biocore Data Team data-packages (e.g. org.Bt.eg.db).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationfuncs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationfuncs'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'AnnotationFuncs_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AnnotationFuncs_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/AnnotationFuncs_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/AnnotationFuncs/AnnotationFuncs_1.26.0_src_all.tar.gz']), ('md5', '6ff063493468ddf5598f370a583f0799')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'r-base', 'r-dbi']), ('run', ['bioconductor-annotationdbi', 'r-base', 'r-dbi'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationFuncs\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AnnotationFuncs.html'), ('license', 'GPL-2'), ('summary', 'Functions for handling translating between different identifieres using the Biocore Data Team data-packages (e.g. org.Bt.eg.db).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-proper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-proper'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'PROPER_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PROPER_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/PROPER_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/PROPER/PROPER_1.8.0_src_all.tar.gz']), ('md5', 'ada1ccfbcd1f50a3715bbca5ecddf46e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-edger', 'r-base']), ('run', ['bioconductor-edger', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PROPER\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PROPER.html'), ('license', 'GPL'), ('summary', 'This package provide simulation based methods for evaluating the statistical power in differential expression analysis from RNA-seq data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-proper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-proper'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'PROPER_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PROPER_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/PROPER_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/PROPER/PROPER_1.8.0_src_all.tar.gz']), ('md5', 'ada1ccfbcd1f50a3715bbca5ecddf46e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-edger', 'r-base']), ('run', ['bioconductor-edger', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PROPER\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PROPER.html'), ('license', 'GPL'), ('summary', 'This package provide simulation based methods for evaluating the statistical power in differential expression analysis from RNA-seq data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pkgdeptools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pkgdeptools'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'pkgDepTools_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pkgDepTools_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/pkgDepTools_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/pkgDepTools/pkgDepTools_1.42.0_src_all.tar.gz']), ('md5', '5ccffeff4f1e1d22577771a338b88585')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-rbgl', 'r-base']), ('run', ['bioconductor-graph', 'bioconductor-rbgl', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pkgDepTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pkgDepTools.html'), ('license', 'GPL-2'), ('summary', 'This package provides tools for computing and analyzing dependency relationships among R packages.  It provides tools for building a graph-based representation of the dependencies among all packages in a list of CRAN-style package repositories.  There are also utilities for computing installation order of a given package.  If the RCurl package is available, an estimate of the download size required to install a given package and its dependencies can be obtained.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pkgdeptools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pkgdeptools'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'pkgDepTools_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pkgDepTools_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/pkgDepTools_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/pkgDepTools/pkgDepTools_1.42.0_src_all.tar.gz']), ('md5', '5ccffeff4f1e1d22577771a338b88585')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-rbgl', 'r-base']), ('run', ['bioconductor-graph', 'bioconductor-rbgl', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pkgDepTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pkgDepTools.html'), ('license', 'GPL-2'), ('summary', 'This package provides tools for computing and analyzing dependency relationships among R packages.  It provides tools for building a graph-based representation of the dependencies among all packages in a list of CRAN-style package repositories.  There are also utilities for computing installation order of a given package.  If the RCurl package is available, an estimate of the download size required to install a given package and its dependencies can be obtained.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ssize/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ssize'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'ssize_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ssize_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/ssize_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/ssize/ssize_1.50.0_src_all.tar.gz']), ('md5', 'f7d12775503e22319187141e941f8060')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gdata', 'r-xtable']), ('run', ['r-base', 'r-gdata', 'r-xtable'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ssize\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ssize.html'), ('license', 'LGPL'), ('summary', 'Functions for computing and displaying sample size information for gene expression arrays.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ssize/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ssize'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'ssize_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ssize_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/ssize_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/ssize/ssize_1.50.0_src_all.tar.gz']), ('md5', 'f7d12775503e22319187141e941f8060')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gdata', 'r-xtable']), ('run', ['r-base', 'r-gdata', 'r-xtable'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ssize\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ssize.html'), ('license', 'LGPL'), ('summary', 'Functions for computing and displaying sample size information for gene expression arrays.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sangerseqr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sangerseqr'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'sangerseqR_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sangerseqR_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/sangerseqR_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/sangerseqR/sangerseqR_1.12.0_src_all.tar.gz']), ('md5', 'd526e7a5fbc4694c861b34b583efbd35')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'r-base', 'r-shiny']), ('run', ['bioconductor-biostrings', 'r-base', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sangerseqR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sangerseqR.html'), ('license', 'GPL-2'), ('summary', 'This package contains several tools for analyzing Sanger Sequencing data files in R, including reading .scf and .ab1 files, making basecalls and plotting chromatograms.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sangerseqr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sangerseqr'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'sangerseqR_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sangerseqR_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/sangerseqR_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/sangerseqR/sangerseqR_1.12.0_src_all.tar.gz']), ('md5', 'd526e7a5fbc4694c861b34b583efbd35')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'r-base', 'r-shiny']), ('run', ['bioconductor-biostrings', 'r-base', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sangerseqR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sangerseqR.html'), ('license', 'GPL-2'), ('summary', 'This package contains several tools for analyzing Sanger Sequencing data files in R, including reading .scf and .ab1 files, making basecalls and plotting chromatograms.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-triform/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-triform'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'triform_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/triform_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/triform_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/triform/triform_1.18.0_src_all.tar.gz']), ('md5', '81b0783c1cb3e8f7b2e2fe2f034b99fd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-iranges >=2.5.27', 'r-base', 'r-yaml']), ('run', ['bioconductor-biocgenerics', 'bioconductor-iranges >=2.5.27', 'r-base', 'r-yaml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'triform\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/triform.html'), ('license', 'GPL-2'), ('summary', 'The Triform algorithm uses model-free statistics to identify peak-like distributions of TF ChIP sequencing reads, taking advantage of an improved peak definition in combination with known profile characteristics.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-triform/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-triform'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'triform_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/triform_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/triform_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/triform/triform_1.18.0_src_all.tar.gz']), ('md5', '81b0783c1cb3e8f7b2e2fe2f034b99fd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-iranges >=2.5.27', 'r-base', 'r-yaml']), ('run', ['bioconductor-biocgenerics', 'bioconductor-iranges >=2.5.27', 'r-base', 'r-yaml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'triform\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/triform.html'), ('license', 'GPL-2'), ('summary', 'The Triform algorithm uses model-free statistics to identify peak-like distributions of TF ChIP sequencing reads, taking advantage of an improved peak definition in combination with known profile characteristics.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gffutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gffutils'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'gffutils-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/b4/ea/54ca403a8d471849606fb432b9afcf73c8c5105ea2dd87b8d38bd5217c5a/gffutils-<{ version }}.tar.gz'), ('sha256', 'e1b0bf2b422ec3491be13d77bf7baf71d274abd844e8f8b6c1217a95b236218b')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyfaidx', 'six', 'argh', 'argcomplete', 'simplejson']), ('run', ['python', 'pyfaidx', 'six', 'argh', 'argcomplete', 'simplejson'])])), ('test', CommentedMap([('imports', ['gffutils']), ('commands', ['gffutils-cli -h'])])), ('about', CommentedMap([('home', 'https://github.com/daler/gffutils'), ('license', 'MIT'), ('summary', 'Work with GFF and GTF files in a flexible database framework')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gffutils/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gffutils'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'gffutils-<{ version }}.tar.gz'), ('url', 'https://pypi.python.org/packages/b4/ea/54ca403a8d471849606fb432b9afcf73c8c5105ea2dd87b8d38bd5217c5a/gffutils-<{ version }}.tar.gz'), ('sha256', 'e1b0bf2b422ec3491be13d77bf7baf71d274abd844e8f8b6c1217a95b236218b')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pyfaidx', 'six', 'argh', 'argcomplete', 'simplejson']), ('run', ['python', 'pyfaidx', 'six', 'argh', 'argcomplete', 'simplejson'])])), ('test', CommentedMap([('imports', ['gffutils']), ('commands', ['gffutils-cli -h'])])), ('about', CommentedMap([('home', 'https://github.com/daler/gffutils'), ('license', 'MIT'), ('summary', 'Work with GFF and GTF files in a flexible database framework')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/amos/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'amos'), ('version', '3.1.0')])), ('about', CommentedMap([('home', 'http://amos.sourceforge.net/wiki/index.php/AMOS'), ('summary', 'A Modular, Open-Source whole genome assembler'), ('license', 'Artistic License')])), ('source', CommentedMap([('git_url', 'http://git.code.sf.net/p/amos/code'), ('git_rev', '9bd658')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('test', CommentedMap([('commands', ['Minimo -h > /dev/null'])])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'boost', 'autoconf', 'automake', 'blat', 'mummer', 'perl', 'perl-statistics-descriptive', 'perl-xml-parser', 'perl-dbi', 'jellyfish', 'python']), ('run', ['mummer', 'perl', 'perl-statistics-descriptive', 'perl-xml-parser', 'perl-dbi', 'jellyfish', 'python'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/amos/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'amos'), ('version', '3.1.0')])), ('about', CommentedMap([('home', 'http://amos.sourceforge.net/wiki/index.php/AMOS'), ('summary', 'A Modular, Open-Source whole genome assembler'), ('license', 'Artistic License')])), ('source', CommentedMap([('git_url', 'http://git.code.sf.net/p/amos/code'), ('git_rev', '9bd658')])), ('build', CommentedMap([('number', 3), ('skip', True)])), ('test', CommentedMap([('commands', ['Minimo -h > /dev/null'])])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'boost', 'autoconf', 'automake', 'blat', 'mummer', 'perl', 'perl-statistics-descriptive', 'perl-xml-parser', 'perl-dbi', 'jellyfish', 'python']), ('run', ['mummer', 'perl', 'perl-statistics-descriptive', 'perl-xml-parser', 'perl-dbi', 'jellyfish', 'python'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rama/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rama'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'rama_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rama_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/rama_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/rama/rama_1.50.0_src_all.tar.gz']), ('md5', 'ba1171551c6b72493af478168bd0cbaf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rama\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rama.html'), ('license', 'GPL (>= 2)'), ('summary', 'Robust estimation of cDNA microarray intensities with replicates. The package uses a Bayesian hierarchical model for the robust estimation. Outliers are modeled explicitly using a t-distribution, and the model also addresses classical issues such as design effects, normalization, transformation, and nonconstant variance.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rama/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rama'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'rama_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rama_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/rama_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/rama/rama_1.50.0_src_all.tar.gz']), ('md5', 'ba1171551c6b72493af478168bd0cbaf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rama\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rama.html'), ('license', 'GPL (>= 2)'), ('summary', 'Robust estimation of cDNA microarray intensities with replicates. The package uses a Bayesian hierarchical model for the robust estimation. Outliers are modeled explicitly using a t-distribution, and the model also addresses classical issues such as design effects, normalization, transformation, and nonconstant variance.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/star-fusion/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'star-fusion'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'STAR-Fusion_v<{ version }}.FULL.tar.gz'), ('url', 'https://github.com/STAR-Fusion/STAR-Fusion/releases/download/v<{ version }}/STAR-Fusion_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['blast', 'perl', 'perl-set-intervaltree', 'perl-db-file', 'perl-uri', 'star', 'gmap', 'bowtie'])])), ('test', CommentedMap([('commands', ['STAR-Fusion --help 2>&1 | grep Optional > /dev/null', 'prep_genome_lib.pl -h 2>&1 | grep \"STAR-Fusion\" > /dev/null', 'blast_and_promiscuity_filter.pl -h 2>&1 | grep Optional > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/STAR-Fusion/STAR-Fusion'), ('license', 'BSD-3-Clause'), ('summary', 'A fusion gene caller for STAR. The FusionFilter tools to create needed index data structures are not included.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/star-fusion/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'star-fusion'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'STAR-Fusion_v<{ version }}.FULL.tar.gz'), ('url', 'https://github.com/STAR-Fusion/STAR-Fusion/releases/download/v<{ version }}/STAR-Fusion_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['blast', 'perl', 'perl-set-intervaltree', 'perl-db-file', 'perl-uri', 'star', 'gmap', 'bowtie'])])), ('test', CommentedMap([('commands', ['STAR-Fusion --help 2>&1 | grep Optional > /dev/null', 'prep_genome_lib.pl -h 2>&1 | grep \"STAR-Fusion\" > /dev/null', 'blast_and_promiscuity_filter.pl -h 2>&1 | grep Optional > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/STAR-Fusion/STAR-Fusion'), ('license', 'BSD-3-Clause'), ('summary', 'A fusion gene caller for STAR. The FusionFilter tools to create needed index data structures are not included.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pathnet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pathnet'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'PathNet_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PathNet_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/PathNet_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/PathNet/PathNet_1.16.0_src_all.tar.gz']), ('md5', '35fd5d2a6f7365635db7f0f225dcd0a5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PathNet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PathNet.html'), ('license', 'GPL-3'), ('summary', 'PathNet uses topological information present in pathways and differential expression levels of genes (obtained from microarray experiment) to identify pathways that are 1) significantly enriched and 2) associated with each other in the context of differential expression. The algorithm is described in: PathNet: A tool for pathway analysis using topological information. Dutta B, Wallqvist A, and Reifman J. Source Code for Biology and Medicine 2012 Sep 24;7(1):10.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pathnet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pathnet'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'PathNet_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PathNet_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/PathNet_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/PathNet/PathNet_1.16.0_src_all.tar.gz']), ('md5', '35fd5d2a6f7365635db7f0f225dcd0a5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PathNet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PathNet.html'), ('license', 'GPL-3'), ('summary', 'PathNet uses topological information present in pathways and differential expression levels of genes (obtained from microarray experiment) to identify pathways that are 1) significantly enriched and 2) associated with each other in the context of differential expression. The algorithm is described in: PathNet: A tool for pathway analysis using topological information. Dutta B, Wallqvist A, and Reifman J. Source Code for Biology and Medicine 2012 Sep 24;7(1):10.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gage'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'gage_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/gage_<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-keggrest', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-keggrest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gage\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/gage.html'), ('license', 'GPL (>=2.0)'), ('summary', 'GAGE is a published method for gene set (enrichment or GSEA) or pathway analysis. GAGE is generally applicable independent of microarray or RNA-Seq data attributes including sample sizes, experimental designs, assay platforms, and other types of heterogeneity, and consistently achieves superior performance over other frequently used methods. In gage package, we provide functions for basic GAGE analysis, result processing and presentation. We have also built pipeline routines for of multiple GAGE analyses in a batch, comparison between parallel analyses, and combined analysis of heterogeneous data from different sources/studies. In addition, we provide demo microarray data and commonly used gene set data based on KEGG pathways and GO terms. These funtions and data are also useful for gene set analysis using other methods.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gage'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'gage_<{ version }}.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/gage_<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-keggrest', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-graph', 'bioconductor-keggrest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gage\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/gage.html'), ('license', 'GPL (>=2.0)'), ('summary', 'GAGE is a published method for gene set (enrichment or GSEA) or pathway analysis. GAGE is generally applicable independent of microarray or RNA-Seq data attributes including sample sizes, experimental designs, assay platforms, and other types of heterogeneity, and consistently achieves superior performance over other frequently used methods. In gage package, we provide functions for basic GAGE analysis, result processing and presentation. We have also built pipeline routines for of multiple GAGE analyses in a batch, comparison between parallel analyses, and combined analysis of heterogeneous data from different sources/studies. In addition, we provide demo microarray data and commonly used gene set data based on KEGG pathways and GO terms. These funtions and data are also useful for gene set analysis using other methods.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-vbmp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-vbmp'), ('version', '1.44.0')])), ('source', CommentedMap([('fn', 'vbmp_1.44.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/vbmp_1.44.0.tar.gz', 'https://bioarchive.galaxyproject.org/vbmp_1.44.0.tar.gz', 'https://depot.galaxyproject.org/software/vbmp/vbmp_1.44.0_src_all.tar.gz']), ('md5', 'fcb2f0ee9fa8829d5a29a038769f65e9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'vbmp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/vbmp.html'), ('license', 'GPL (>= 2)'), ('summary', 'Variational Bayesian Multinomial Probit Regression with Gaussian Process Priors. It estimates class membership posterior probability employing variational and sparse approximation to the full posterior. This software also incorporates feature weighting by means of Automatic Relevance Determination.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-vbmp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-vbmp'), ('version', '1.44.0')])), ('source', CommentedMap([('fn', 'vbmp_1.44.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/vbmp_1.44.0.tar.gz', 'https://bioarchive.galaxyproject.org/vbmp_1.44.0.tar.gz', 'https://depot.galaxyproject.org/software/vbmp/vbmp_1.44.0_src_all.tar.gz']), ('md5', 'fcb2f0ee9fa8829d5a29a038769f65e9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'vbmp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/vbmp.html'), ('license', 'GPL (>= 2)'), ('summary', 'Variational Bayesian Multinomial Probit Regression with Gaussian Process Priors. It estimates class membership posterior probability employing variational and sparse approximation to the full posterior. This software also incorporates feature weighting by means of Automatic Relevance Determination.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nudge/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nudge'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'nudge_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nudge_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/nudge_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/nudge/nudge_1.42.0_src_all.tar.gz']), ('md5', 'bb1641c652cdb1dd7193933a0073d9bd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nudge\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nudge.html'), ('license', 'GPL-2'), ('summary', 'Package for normalizing microarray data in single and multiple replicate experiments and fitting a normal-uniform mixture to detect differentially expressed genes in the cases where the two samples are being compared directly or indirectly (via a common reference sample)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nudge/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nudge'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'nudge_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nudge_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/nudge_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/nudge/nudge_1.42.0_src_all.tar.gz']), ('md5', 'bb1641c652cdb1dd7193933a0073d9bd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nudge\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nudge.html'), ('license', 'GPL-2'), ('summary', 'Package for normalizing microarray data in single and multiple replicate experiments and fitting a normal-uniform mixture to detect differentially expressed genes in the cases where the two samples are being compared directly or indirectly (via a common reference sample)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/openms/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower  }}'), ('version', '<{ version  }}')])), ('source', CommentedMap([('fn', 'OpenMS-<{ version }}_src_contrib_doc.tar.gz'), ('url', 'https://github.com/OpenMS/OpenMS/releases/download/Release2.2.0/OpenMS-<{ version }}-src.tar.gz'), ('sha256', '61e7d24890abb6f462e2f026efb0dc88b84639b228d96e919c8074d3b11d286e')])), ('build', CommentedMap([('number', 0), ('string', 'py<{CONDA_PY}}_boost<{ boost }}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'glib', 'autoconf', 'automake', 'python', 'zlib', 'libtool', 'xerces-c', 'gsl', 'boost  ==<{ boost }}', 'eigen', 'glpk  >=4.60', 'bzip2', 'qt  <5', 'libsvm >=3.21', 'seqan-library ==1.4.2', 'coinmp']), ('run', ['python', 'libgcc', 'glib', 'zlib', 'xerces-c', 'gsl', 'boost  ==<{ boost }}', 'eigen', 'glpk  >=4.60', 'bzip2', 'qt  <5', 'libsvm >=3.21', 'coinmp'])])), ('test', CommentedMap([('commands', ['FileMerger --help'])])), ('about', CommentedMap([('home', 'https://github.com/OpenMS/OpenMS'), ('license', 'BSD'), ('summary', 'OpenMS is an open-source software C++ library for LC-MS data management and analyses')])), ('extra', CommentedMap([('notes', 'This package can exceed the travis build time. If so, this package will be build by @bioconda/core locally via\\nsimulate-travis.py and pushed directly. If you need an update for this package please make your changes,\\ncreate a PR and contact @bioconda/core.\\n')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/openms/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower  }}'), ('version', '<{ version  }}')])), ('source', CommentedMap([('fn', 'OpenMS-<{ version }}_src_contrib_doc.tar.gz'), ('url', 'https://github.com/OpenMS/OpenMS/releases/download/Release2.2.0/OpenMS-<{ version }}-src.tar.gz'), ('sha256', '61e7d24890abb6f462e2f026efb0dc88b84639b228d96e919c8074d3b11d286e')])), ('build', CommentedMap([('number', 0), ('string', 'py<{CONDA_PY}}_boost<{ boost }}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'glib', 'autoconf', 'automake', 'python', 'zlib', 'libtool', 'xerces-c', 'gsl', 'boost  ==<{ boost }}', 'eigen', 'glpk  >=4.60', 'bzip2', 'qt  <5', 'libsvm >=3.21', 'seqan-library ==1.4.2', 'coinmp']), ('run', ['python', 'libgcc', 'glib', 'zlib', 'xerces-c', 'gsl', 'boost  ==<{ boost }}', 'eigen', 'glpk  >=4.60', 'bzip2', 'qt  <5', 'libsvm >=3.21', 'coinmp'])])), ('test', CommentedMap([('commands', ['FileMerger --help'])])), ('about', CommentedMap([('home', 'https://github.com/OpenMS/OpenMS'), ('license', 'BSD'), ('summary', 'OpenMS is an open-source software C++ library for LC-MS data management and analyses')])), ('extra', CommentedMap([('notes', 'This package can exceed the travis build time. If so, this package will be build by @bioconda/core locally via\\nsimulate-travis.py and pushed directly. If you need an update for this package please make your changes,\\ncreate a PR and contact @bioconda/core.\\n')]))]))\n",
      "EXTRA has been found: openms\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-olin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-olin'), ('version', '1.54.0')])), ('source', CommentedMap([('fn', 'OLIN_1.54.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OLIN_1.54.0.tar.gz', 'https://bioarchive.galaxyproject.org/OLIN_1.54.0.tar.gz', 'https://depot.galaxyproject.org/software/OLIN/OLIN_1.54.0_src_all.tar.gz']), ('md5', 'b659246e12660920db5522a38109161c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-locfit']), ('run', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-locfit'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OLIN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OLIN.html'), ('license', 'GPL-2'), ('summary', 'Functions for normalisation of two-color microarrays by optimised local regression and for detection of artefacts in microarray data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-olin/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-olin'), ('version', '1.54.0')])), ('source', CommentedMap([('fn', 'OLIN_1.54.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OLIN_1.54.0.tar.gz', 'https://bioarchive.galaxyproject.org/OLIN_1.54.0.tar.gz', 'https://depot.galaxyproject.org/software/OLIN/OLIN_1.54.0_src_all.tar.gz']), ('md5', 'b659246e12660920db5522a38109161c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-locfit']), ('run', ['bioconductor-limma', 'bioconductor-marray', 'r-base', 'r-locfit'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OLIN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OLIN.html'), ('license', 'GPL-2'), ('summary', 'Functions for normalisation of two-color microarrays by optimised local regression and for detection of artefacts in microarray data')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igvtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'igvtools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'igvtools_<{ version }}.zip'), ('url', 'http://data.broadinstitute.org/igv/projects/downloads/igvtools_<{ version }}.zip'), ('sha256', '1f15d226c0773d7a39bb839a9f32043214cd45399c08d34aef89cb69b1f777f0')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['igvtools help'])])), ('about', CommentedMap([('home', 'http://www.broadinstitute.org/igv/'), ('summary', 'command line tools for IGV'), ('license', 'MIT License'), ('license_family', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/igvtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'igvtools'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'igvtools_<{ version }}.zip'), ('url', 'http://data.broadinstitute.org/igv/projects/downloads/igvtools_<{ version }}.zip'), ('sha256', '1f15d226c0773d7a39bb839a9f32043214cd45399c08d34aef89cb69b1f777f0')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['igvtools help'])])), ('about', CommentedMap([('home', 'http://www.broadinstitute.org/igv/'), ('summary', 'command line tools for IGV'), ('license', 'MIT License'), ('license_family', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pash/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pash'), ('version', '3.0.6.2')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'pash-3.0.6.2.tar.gz'), ('url', 'https://genboree.org/pash-supp/pash-3.0.6.2.tar.gz'), ('md5', 'f475acdb1ad8dd712cdf16913849b860')])), ('requirements', CommentedMap([('build', ['gcc', 'glib', 'ruby']), ('run', ['libgcc', 'glib', 'ruby'])])), ('about', CommentedMap([('home', 'http://www.bioinformatics.bbsrc.ac.uk/projects/bismark/'), ('license', 'Unknown'), ('summary', 'A versatile software package for read mapping and integrative analysis of genomic and epigenomic variation using massively parallel DNA sequencing')])), ('test', CommentedMap([('commands', ['keyFreq --help 2> /dev/null', 'makeIgnoreList --help 2> /dev/null', 'pash2SAM --help 2> /dev/null', 'pash-3.0lx --help > /dev/null', 'buildFastaIndex.rb --help | grep \"PROGRAM DESCRIPTION\" > /dev/null', 'pashToLff.rb --help 2>&1 | grep \"pash output\" > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pash/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pash'), ('version', '3.0.6.2')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'pash-3.0.6.2.tar.gz'), ('url', 'https://genboree.org/pash-supp/pash-3.0.6.2.tar.gz'), ('md5', 'f475acdb1ad8dd712cdf16913849b860')])), ('requirements', CommentedMap([('build', ['gcc', 'glib', 'ruby']), ('run', ['libgcc', 'glib', 'ruby'])])), ('about', CommentedMap([('home', 'http://www.bioinformatics.bbsrc.ac.uk/projects/bismark/'), ('license', 'Unknown'), ('summary', 'A versatile software package for read mapping and integrative analysis of genomic and epigenomic variation using massively parallel DNA sequencing')])), ('test', CommentedMap([('commands', ['keyFreq --help 2> /dev/null', 'makeIgnoreList --help 2> /dev/null', 'pash2SAM --help 2> /dev/null', 'pash-3.0lx --help > /dev/null', 'buildFastaIndex.rb --help | grep \"PROGRAM DESCRIPTION\" > /dev/null', 'pashToLff.rb --help 2>&1 | grep \"pash output\" > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/antismash/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'https://bitbucket.org/antismash/antismash/downloads/<{ name|lower  }}-<{ version }}.tar.gz'), ('md5', '98e7a6402f0614acc80d8e87b17ed36b')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'argparse', 'backports.lzma', 'bcbiogff', 'biopython >=1.65', 'blast ==2.2.31', 'clustalw', 'cssselect', 'diamond ==0.8.36', 'ete2', 'fasttree ==2.1.9', 'glimmer ==3.02', 'glimmerhmm ==3.0.4', 'helperlibs', 'hmmer', 'hmmer2', 'openjdk', 'libxml2 ==2.9.4', 'mafft', 'matplotlib', 'meme', 'muscle', 'networkx', 'numpy', 'pandas', 'perl-bioperl', 'perl-pathtools', 'prodigal', 'pyexcelerator', 'pyquery ==1.2.9', 'pysvg', 'scipy', 'scikit-learn ==0.18.2', 'xz ==5.2.2'])])), ('test', CommentedMap([('commands', ['antismash -h'])])), ('about', CommentedMap([('home', 'http://antismash.secondarymetabolites.org/'), ('license', 'GPL3'), ('license_file', 'LICENSE.txt'), ('summary', 'antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/antismash/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'https://bitbucket.org/antismash/antismash/downloads/<{ name|lower  }}-<{ version }}.tar.gz'), ('md5', '98e7a6402f0614acc80d8e87b17ed36b')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python', 'argparse', 'backports.lzma', 'bcbiogff', 'biopython >=1.65', 'blast ==2.2.31', 'clustalw', 'cssselect', 'diamond ==0.8.36', 'ete2', 'fasttree ==2.1.9', 'glimmer ==3.02', 'glimmerhmm ==3.0.4', 'helperlibs', 'hmmer', 'hmmer2', 'openjdk', 'libxml2 ==2.9.4', 'mafft', 'matplotlib', 'meme', 'muscle', 'networkx', 'numpy', 'pandas', 'perl-bioperl', 'perl-pathtools', 'prodigal', 'pyexcelerator', 'pyquery ==1.2.9', 'pysvg', 'scipy', 'scikit-learn ==0.18.2', 'xz ==5.2.2'])])), ('test', CommentedMap([('commands', ['antismash -h'])])), ('about', CommentedMap([('home', 'http://antismash.secondarymetabolites.org/'), ('license', 'GPL3'), ('license_file', 'LICENSE.txt'), ('summary', 'antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bamtools/2.3.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bamtools'), ('version', '2.3.0')])), ('source', CommentedMap([('fn', 'bamtools-2.3.0.tar.gz'), ('url', 'https://github.com/pezmaster31/bamtools/archive/v2.3.0.tar.gz'), ('sha256', '288046e6d5d41afdc5fce8608c5641cf2b8e670644587c1315b90bbe92f039af'), ('patches', ['bamtools_lib.diff'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['bamtools --help > /dev/null 2>&1'])])), ('about', CommentedMap([('home', 'https://github.com/pezmaster31/bamtools'), ('license', 'MIT'), ('summary', 'C++ API & command-line toolkit for working with BAM data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bamtools/2.3.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bamtools'), ('version', '2.3.0')])), ('source', CommentedMap([('fn', 'bamtools-2.3.0.tar.gz'), ('url', 'https://github.com/pezmaster31/bamtools/archive/v2.3.0.tar.gz'), ('sha256', '288046e6d5d41afdc5fce8608c5641cf2b8e670644587c1315b90bbe92f039af'), ('patches', ['bamtools_lib.diff'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['bamtools --help > /dev/null 2>&1'])])), ('about', CommentedMap([('home', 'https://github.com/pezmaster31/bamtools'), ('license', 'MIT'), ('summary', 'C++ API & command-line toolkit for working with BAM data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-tap-harness-env/3.30/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/TAP::Harness::Env'), ('license', 'perl_5'), ('summary', 'Parsing harness related environmental variables where appropriate')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-tap-harness-env/3.30/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/TAP::Harness::Env'), ('license', 'perl_5'), ('summary', 'Parsing harness related environmental variables where appropriate')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/samtools/1.1/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/samtools'), ('license', 'MIT'), ('summary', 'Tools for dealing with SAM, BAM and CRAM files')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'samtools'), ('version', '1.1')])), ('requirements', CommentedMap([('build', ['ncurses ==5.9', 'zlib ==1.2.8']), ('run', ['ncurses ==5.9', 'zlib ==1.2.8'])])), ('source', CommentedMap([('fn', 'samtools-1.1.tar.bz2'), ('sha256', 'c24d26c303153d72b5bf3cc11f72c6c375a4ca1140cc485648c8c5473414b7f8'), ('url', 'http://downloads.sourceforge.net/project/samtools/samtools/1.1/samtools-1.1.tar.bz2')])), ('test', CommentedMap([('commands', ['samtools view --help 2>&1 | grep Notes > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/samtools/1.1/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/samtools/samtools'), ('license', 'MIT'), ('summary', 'Tools for dealing with SAM, BAM and CRAM files')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'samtools'), ('version', '1.1')])), ('requirements', CommentedMap([('build', ['ncurses ==5.9', 'zlib ==1.2.8']), ('run', ['ncurses ==5.9', 'zlib ==1.2.8'])])), ('source', CommentedMap([('fn', 'samtools-1.1.tar.bz2'), ('sha256', 'c24d26c303153d72b5bf3cc11f72c6c375a4ca1140cc485648c8c5473414b7f8'), ('url', 'http://downloads.sourceforge.net/project/samtools/samtools/1.1/samtools-1.1.tar.bz2')])), ('test', CommentedMap([('commands', ['samtools view --help 2>&1 | grep Notes > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cellmapper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cellmapper'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'CellMapper_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CellMapper_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/CellMapper_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/CellMapper/CellMapper_1.2.0_src_all.tar.gz']), ('md5', '318fb0d8cedd6560d880e4a834c56f40')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CellMapper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CellMapper.html'), ('license', 'Artistic-2.0'), ('summary', 'Infers cell type-specific expression based on co-expression similarity with known cell type marker genes. Can make accurate predictions using publicly available expression data, even when a cell type has not been isolated before.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cellmapper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cellmapper'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'CellMapper_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CellMapper_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/CellMapper_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/CellMapper/CellMapper_1.2.0_src_all.tar.gz']), ('md5', '318fb0d8cedd6560d880e4a834c56f40')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CellMapper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CellMapper.html'), ('license', 'Artistic-2.0'), ('summary', 'Infers cell type-specific expression based on co-expression similarity with known cell type marker genes. Can make accurate predictions using publicly available expression data, even when a cell type has not been isolated before.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snpsift/4.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://snpeff.sourceforge.net/SnpSift.html'), ('license', 'LGPLv3'), ('summary', 'Toolbox that allows you to filter and manipulate annotated files')])), ('package', CommentedMap([('name', 'snpsift'), ('version', '4.2')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'snpEff_v4_2_core.zip'), ('md5', '25ae5b062d57072de6cfb8677ca3625a'), ('url', 'http://downloads.sourceforge.net/project/snpeff/snpEff_v4_2_core.zip')])), ('requirements', CommentedMap([('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['SnpSift 2>&1 > /dev/null | grep \"SnpSift version 4.2\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snpsift/4.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://snpeff.sourceforge.net/SnpSift.html'), ('license', 'LGPLv3'), ('summary', 'Toolbox that allows you to filter and manipulate annotated files')])), ('package', CommentedMap([('name', 'snpsift'), ('version', '4.2')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'snpEff_v4_2_core.zip'), ('md5', '25ae5b062d57072de6cfb8677ca3625a'), ('url', 'http://downloads.sourceforge.net/project/snpeff/snpEff_v4_2_core.zip')])), ('requirements', CommentedMap([('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['SnpSift 2>&1 > /dev/null | grep \"SnpSift version 4.2\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cummerbund/2.8.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cummerbund'), ('version', '2.8.2')])), ('source', CommentedMap([('fn', 'cummeRbund_2.8.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.0/bioc/src/contrib/cummeRbund_2.8.2.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gviz', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-fastcluster', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-rsqlite']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gviz', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-fastcluster', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cummeRbund\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/cummeRbund.html'), ('license', 'Artistic-2.0'), ('summary', 'Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data.  In addition, provides numerous plotting functions for commonly used visualizations.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cummerbund/2.8.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cummerbund'), ('version', '2.8.2')])), ('source', CommentedMap([('fn', 'cummeRbund_2.8.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.0/bioc/src/contrib/cummeRbund_2.8.2.tar.gz')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gviz', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-fastcluster', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-rsqlite']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gviz', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-fastcluster', 'r-ggplot2', 'r-plyr', 'r-reshape2', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cummeRbund\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/cummeRbund.html'), ('license', 'Artistic-2.0'), ('summary', 'Allows for persistent storage, access, exploration, and manipulation of Cufflinks high-throughput sequencing data.  In addition, provides numerous plotting functions for commonly used visualizations.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyteomics/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://bitbucket.org/levitsky/pyteomics'), ('license', 'Apache License, Version 2.0'), ('summary', 'A framework for proteomics data analysis.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'pyteomics'), ('version', '3.4')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'numpy', 'lxml', 'matplotlib', 'pandas', 'sqlalchemy'])])), ('source', CommentedMap([('fn', 'pyteomics-3.4.tar.gz'), ('md5', '3c8d1bcc16c53100e531a10a2853f679'), ('url', 'https://pypi.python.org/packages/02/2d/0dfb9bd5b02db6228e3277020209ddb7217e12a76e39a1cb0d1c70756e30/pyteomics-3.4.tar.gz')])), ('test', CommentedMap([('imports', ['pyteomics', 'pyteomics.mass', 'pyteomics.openms'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyteomics/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://bitbucket.org/levitsky/pyteomics'), ('license', 'Apache License, Version 2.0'), ('summary', 'A framework for proteomics data analysis.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'pyteomics'), ('version', '3.4')])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'numpy', 'lxml', 'matplotlib', 'pandas', 'sqlalchemy'])])), ('source', CommentedMap([('fn', 'pyteomics-3.4.tar.gz'), ('md5', '3c8d1bcc16c53100e531a10a2853f679'), ('url', 'https://pypi.python.org/packages/02/2d/0dfb9bd5b02db6228e3277020209ddb7217e12a76e39a1cb0d1c70756e30/pyteomics-3.4.tar.gz')])), ('test', CommentedMap([('imports', ['pyteomics', 'pyteomics.mass', 'pyteomics.openms'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-soap-lite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-soap-lite'), ('version', '1.19')])), ('source', CommentedMap([('fn', 'SOAP-Lite-1.19.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PH/PHRED/SOAP-Lite-1.19.tar.gz'), ('md5', 'eceb84ca5aabebf855e37c53f53d39ce')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-class-inspector', 'perl-io-sessiondata', 'perl-io-socket-ssl', 'perl-libwww-perl', 'perl-lwp-protocol-https', 'perl-mime-lite', 'perl-mime-tools', 'perl-task-weaken', 'perl-uri', 'perl-xml-parser', 'perl-test-warn', 'perl-test-xml', 'perl-xml-parser-lite']), ('run', ['perl-threaded', 'perl-class-inspector', 'perl-io-sessiondata', 'perl-io-socket-ssl', 'perl-libwww-perl', 'perl-lwp-protocol-https', 'perl-task-weaken', 'perl-uri', 'perl-xml-parser', 'perl-mime-tools', 'perl-mime-lite'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/SOAP-Lite'), ('license', 'perl_5'), ('summary', \"Perl's Web Services Toolkit\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-soap-lite/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-soap-lite'), ('version', '1.19')])), ('source', CommentedMap([('fn', 'SOAP-Lite-1.19.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/P/PH/PHRED/SOAP-Lite-1.19.tar.gz'), ('md5', 'eceb84ca5aabebf855e37c53f53d39ce')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-class-inspector', 'perl-io-sessiondata', 'perl-io-socket-ssl', 'perl-libwww-perl', 'perl-lwp-protocol-https', 'perl-mime-lite', 'perl-mime-tools', 'perl-task-weaken', 'perl-uri', 'perl-xml-parser', 'perl-test-warn', 'perl-test-xml', 'perl-xml-parser-lite']), ('run', ['perl-threaded', 'perl-class-inspector', 'perl-io-sessiondata', 'perl-io-socket-ssl', 'perl-libwww-perl', 'perl-lwp-protocol-https', 'perl-task-weaken', 'perl-uri', 'perl-xml-parser', 'perl-mime-tools', 'perl-mime-lite'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/SOAP-Lite'), ('license', 'perl_5'), ('summary', \"Perl's Web Services Toolkit\")]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prinseq/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://prinseq.sourceforge.net/'), ('license', 'GPLv3'), ('summary', 'PRINSEQ can be used to filter, reformat, or trim your genomic and metagenomic sequence data')])), ('package', CommentedMap([('name', 'prinseq'), ('version', '0.20.4')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('source', CommentedMap([('fn', 'prinseq-lite-0.20.4.tar.gz'), ('md5', '3be1a572073ebbbecfeba42a42853ff5'), ('url', 'http://iweb.dl.sourceforge.net/project/prinseq/standalone/prinseq-lite-0.20.4.tar.gz')])), ('requirements', CommentedMap([('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', ['prinseq-lite.pl -version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/prinseq/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://prinseq.sourceforge.net/'), ('license', 'GPLv3'), ('summary', 'PRINSEQ can be used to filter, reformat, or trim your genomic and metagenomic sequence data')])), ('package', CommentedMap([('name', 'prinseq'), ('version', '0.20.4')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('source', CommentedMap([('fn', 'prinseq-lite-0.20.4.tar.gz'), ('md5', '3be1a572073ebbbecfeba42a42853ff5'), ('url', 'http://iweb.dl.sourceforge.net/project/prinseq/standalone/prinseq-lite-0.20.4.tar.gz')])), ('requirements', CommentedMap([('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', ['prinseq-lite.pl -version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gcrma/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gcrma'), ('version', '2.48.0')])), ('source', CommentedMap([('fn', 'gcrma_2.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/gcrma_2.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/gcrma_2.48.0.tar.gz', 'https://depot.galaxyproject.org/software/gcrma/gcrma_2.48.0_src_all.tar.gz']), ('md5', 'c3a2f21c6f93458d1a123594e437a210')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.23.2', 'bioconductor-affyio >=1.13.3', 'bioconductor-biobase', 'bioconductor-biocinstaller', 'bioconductor-biostrings >=2.11.32', 'bioconductor-xvector', 'r-base']), ('run', ['bioconductor-affy >=1.23.2', 'bioconductor-affyio >=1.13.3', 'bioconductor-biobase', 'bioconductor-biocinstaller', 'bioconductor-biostrings >=2.11.32', 'bioconductor-xvector', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gcrma\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/gcrma.html'), ('license', 'LGPL'), ('summary', 'Background adjustment using sequence information')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gcrma/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gcrma'), ('version', '2.48.0')])), ('source', CommentedMap([('fn', 'gcrma_2.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/gcrma_2.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/gcrma_2.48.0.tar.gz', 'https://depot.galaxyproject.org/software/gcrma/gcrma_2.48.0_src_all.tar.gz']), ('md5', 'c3a2f21c6f93458d1a123594e437a210')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.23.2', 'bioconductor-affyio >=1.13.3', 'bioconductor-biobase', 'bioconductor-biocinstaller', 'bioconductor-biostrings >=2.11.32', 'bioconductor-xvector', 'r-base']), ('run', ['bioconductor-affy >=1.23.2', 'bioconductor-affyio >=1.13.3', 'bioconductor-biobase', 'bioconductor-biocinstaller', 'bioconductor-biostrings >=2.11.32', 'bioconductor-xvector', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gcrma\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/gcrma.html'), ('license', 'LGPL'), ('summary', 'Background adjustment using sequence information')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gffcompare/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'gffcompare-<{ version }}.tar.gz'), ('url', 'https://github.com/gpertea/gffcompare/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256hash }}'), ('patches', ['Makefile.patch', 'Makefile.osx.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'gclib']), ('run', ['gclib', 'libgcc'])])), ('test', CommentedMap([('commands', ['gffcompare -v'])])), ('about', CommentedMap([('home', 'https://github.com/gpertea/gffcompare'), ('license', 'Artistic License 2.0'), ('summary', 'GffCompare by Geo Pertea')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gffcompare/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'gffcompare-<{ version }}.tar.gz'), ('url', 'https://github.com/gpertea/gffcompare/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256hash }}'), ('patches', ['Makefile.patch', 'Makefile.osx.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'gclib']), ('run', ['gclib', 'libgcc'])])), ('test', CommentedMap([('commands', ['gffcompare -v'])])), ('about', CommentedMap([('home', 'https://github.com/gpertea/gffcompare'), ('license', 'Artistic License 2.0'), ('summary', 'GffCompare by Geo Pertea')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-triplex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-triplex'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'triplex_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/triplex_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/triplex_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/triplex/triplex_1.16.0_src_all.tar.gz']), ('md5', 'b43ae8cf4b947dfb76e2d7bf91a336d8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.39.10', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.5.14', 'bioconductor-xvector >=0.11.6', 'r-base']), ('run', ['bioconductor-biostrings >=2.39.10', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.5.14', 'bioconductor-xvector >=0.11.6', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'triplex\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/triplex.html'), ('license', 'BSD_2_clause + file LICENSE'), ('summary', 'This package provides functions for identification and visualization of potential intramolecular triplex patterns in DNA sequence. The main functionality is to detect the positions of subsequences capable of folding into an intramolecular triplex (H-DNA) in a much larger sequence. The potential H-DNA (triplexes) should be made of as many cannonical nucleotide triplets as possible. The package includes visualization showing the exact base-pairing in 1D, 2D or 3D.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-triplex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-triplex'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'triplex_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/triplex_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/triplex_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/triplex/triplex_1.16.0_src_all.tar.gz']), ('md5', 'b43ae8cf4b947dfb76e2d7bf91a336d8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.39.10', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.5.14', 'bioconductor-xvector >=0.11.6', 'r-base']), ('run', ['bioconductor-biostrings >=2.39.10', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.5.27', 'bioconductor-s4vectors >=0.5.14', 'bioconductor-xvector >=0.11.6', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'triplex\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/triplex.html'), ('license', 'BSD_2_clause + file LICENSE'), ('summary', 'This package provides functions for identification and visualization of potential intramolecular triplex patterns in DNA sequence. The main functionality is to detect the positions of subsequences capable of folding into an intramolecular triplex (H-DNA) in a much larger sequence. The potential H-DNA (triplexes) should be made of as many cannonical nucleotide triplets as possible. The package includes visualization showing the exact base-pairing in 1D, 2D or 3D.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/angsd/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('about', CommentedMap([('home', 'http://www.popgen.dk/angsd/index.php/ANGSD'), ('license', 'GPLv3, MIT'), ('summary', 'ANGSD: Analysis of next generation Sequencing Data')])), ('package', CommentedMap([('name', 'angsd'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['htslib', 'gcc', 'llvm']), ('run', ['libgcc', 'htslib'])])), ('test', CommentedMap([('commands', ['angsd 2>&1 | grep \"angsd version\"', 'thetaStat 2>&1 | grep \"neutrality test\"'])])), ('source', CommentedMap([('fn', 'angsd-<{ version }}.tar.gz'), ('url', 'https://github.com/ANGSD/angsd/archive/<{ version }}.tar.gz'), ('md5', '89a5dcbff49de6b72e474b6f4bf8d112'), ('patches', ['Makefile.patch'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/angsd/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('about', CommentedMap([('home', 'http://www.popgen.dk/angsd/index.php/ANGSD'), ('license', 'GPLv3, MIT'), ('summary', 'ANGSD: Analysis of next generation Sequencing Data')])), ('package', CommentedMap([('name', 'angsd'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['htslib', 'gcc', 'llvm']), ('run', ['libgcc', 'htslib'])])), ('test', CommentedMap([('commands', ['angsd 2>&1 | grep \"angsd version\"', 'thetaStat 2>&1 | grep \"neutrality test\"'])])), ('source', CommentedMap([('fn', 'angsd-<{ version }}.tar.gz'), ('url', 'https://github.com/ANGSD/angsd/archive/<{ version }}.tar.gz'), ('md5', '89a5dcbff49de6b72e474b6f4bf8d112'), ('patches', ['Makefile.patch'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-made4/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-made4'), ('version', '1.44.0')])), ('source', CommentedMap([('fn', 'made4_1.44.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/made4_1.44.0.tar.gz'), ('md5', '68c0e762f8fca1b702a66b14df1e7f66')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ade4', 'r-rcolorbrewer', 'r-gplots', 'r-scatterplot3d']), ('run', ['r-base', 'r-ade4', 'r-rcolorbrewer', 'r-gplots', 'r-scatterplot3d'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'made4\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/made4.html'), ('license', 'Artistic-2.0'), ('summary', 'Multivariate data analysis and graphical display of microarray data. Functions include between group analysis and coinertia analysis. It contains functions that require ADE4.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-made4/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-made4'), ('version', '1.44.0')])), ('source', CommentedMap([('fn', 'made4_1.44.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/made4_1.44.0.tar.gz'), ('md5', '68c0e762f8fca1b702a66b14df1e7f66')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ade4', 'r-rcolorbrewer', 'r-gplots', 'r-scatterplot3d']), ('run', ['r-base', 'r-ade4', 'r-rcolorbrewer', 'r-gplots', 'r-scatterplot3d'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'made4\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/made4.html'), ('license', 'Artistic-2.0'), ('summary', 'Multivariate data analysis and graphical display of microarray data. Functions include between group analysis and coinertia analysis. It contains functions that require ADE4.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-preprocesscore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-preprocesscore'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'preprocessCore_1.34.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/preprocessCore_1.34.0.tar.gz'), ('md5', '845eddebdf2c68a93103622ac9cc31a1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'preprocessCore\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/preprocessCore.html'), ('license', 'LGPL (>= 2)'), ('summary', 'A library of core preprocessing routines')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-preprocesscore/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-preprocesscore'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'preprocessCore_1.34.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/preprocessCore_1.34.0.tar.gz'), ('md5', '845eddebdf2c68a93103622ac9cc31a1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'preprocessCore\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/preprocessCore.html'), ('license', 'LGPL (>= 2)'), ('summary', 'A library of core preprocessing routines')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sights/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sights'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'sights_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sights_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/sights_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/sights/sights_1.2.0_src_all.tar.gz']), ('md5', '26e07f1d549bd6b6eb5ad2c1dcb8e3fd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-qvalue >=2.2', 'r-base', 'r-ggplot2 >=2.0', 'r-lattice >=0.2', 'r-mass >=7.3', 'r-reshape2 >=1.4']), ('run', ['bioconductor-qvalue >=2.2', 'r-base', 'r-ggplot2 >=2.0', 'r-lattice >=0.2', 'r-mass >=7.3', 'r-reshape2 >=1.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sights\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sights.html'), ('license', 'GPL-3 | file LICENSE'), ('summary', 'SIGHTS is a suite of normalization methods, statistical tests, and diagnostic graphical tools for high throughput screening (HTS) assays. HTS assays use microtitre plates to screen large libraries of compounds for their biological, chemical, or biochemical activity.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sights/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sights'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'sights_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sights_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/sights_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/sights/sights_1.2.0_src_all.tar.gz']), ('md5', '26e07f1d549bd6b6eb5ad2c1dcb8e3fd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-qvalue >=2.2', 'r-base', 'r-ggplot2 >=2.0', 'r-lattice >=0.2', 'r-mass >=7.3', 'r-reshape2 >=1.4']), ('run', ['bioconductor-qvalue >=2.2', 'r-base', 'r-ggplot2 >=2.0', 'r-lattice >=0.2', 'r-mass >=7.3', 'r-reshape2 >=1.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sights\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sights.html'), ('license', 'GPL-3 | file LICENSE'), ('summary', 'SIGHTS is a suite of normalization methods, statistical tests, and diagnostic graphical tools for high throughput screening (HTS) assays. HTS assays use microtitre plates to screen large libraries of compounds for their biological, chemical, or biochemical activity.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mulcom/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mulcom'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'Mulcom_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Mulcom_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/Mulcom_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/Mulcom/Mulcom_1.26.0_src_all.tar.gz']), ('md5', 'c1d015fbeded777ad696baed0263fb78')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-fields']), ('run', ['bioconductor-biobase', 'r-base', 'r-fields'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Mulcom\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Mulcom.html'), ('license', 'GPL-2'), ('summary', 'Identification of differentially expressed genes and false discovery rate (FDR) calculation by Multiple Comparison test')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mulcom/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mulcom'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'Mulcom_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Mulcom_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/Mulcom_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/Mulcom/Mulcom_1.26.0_src_all.tar.gz']), ('md5', 'c1d015fbeded777ad696baed0263fb78')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-fields']), ('run', ['bioconductor-biobase', 'r-base', 'r-fields'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Mulcom\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Mulcom.html'), ('license', 'GPL-2'), ('summary', 'Identification of differentially expressed genes and false discovery rate (FDR) calculation by Multiple Comparison test')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ebarrays/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ebarrays'), ('version', '2.40.0')])), ('source', CommentedMap([('fn', 'EBarrays_2.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/EBarrays_2.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/EBarrays_2.40.0.tar.gz', 'https://depot.galaxyproject.org/software/EBarrays/EBarrays_2.40.0_src_all.tar.gz']), ('md5', '44c9760016b30cadc193fb9aa8c30089')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-cluster', 'r-lattice']), ('run', ['bioconductor-biobase', 'r-base', 'r-cluster', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'EBarrays\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/EBarrays.html'), ('license', 'GPL (>= 2)'), ('summary', 'EBarrays provides tools for the analysis of replicated/unreplicated microarray data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ebarrays/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ebarrays'), ('version', '2.40.0')])), ('source', CommentedMap([('fn', 'EBarrays_2.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/EBarrays_2.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/EBarrays_2.40.0.tar.gz', 'https://depot.galaxyproject.org/software/EBarrays/EBarrays_2.40.0_src_all.tar.gz']), ('md5', '44c9760016b30cadc193fb9aa8c30089')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-cluster', 'r-lattice']), ('run', ['bioconductor-biobase', 'r-base', 'r-cluster', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'EBarrays\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/EBarrays.html'), ('license', 'GPL (>= 2)'), ('summary', 'EBarrays provides tools for the analysis of replicated/unreplicated microarray data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-swathxtend/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-swathxtend'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'SwathXtend_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SwathXtend_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/SwathXtend_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/SwathXtend/SwathXtend_1.4.0_src_all.tar.gz']), ('md5', 'a3fb504b0b49d5b000d51516d2b36044')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-e1071', 'r-lattice', 'r-openxlsx', 'r-venndiagram']), ('run', ['r-base', 'r-e1071', 'r-lattice', 'r-openxlsx', 'r-venndiagram'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SwathXtend\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SwathXtend.html'), ('license', 'GPL-2'), ('summary', 'It contains utility functions for integrating spectral libraries for SWATH and statistical data analysis for SWATH generated data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-swathxtend/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-swathxtend'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'SwathXtend_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SwathXtend_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/SwathXtend_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/SwathXtend/SwathXtend_1.4.0_src_all.tar.gz']), ('md5', 'a3fb504b0b49d5b000d51516d2b36044')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-e1071', 'r-lattice', 'r-openxlsx', 'r-venndiagram']), ('run', ['r-base', 'r-e1071', 'r-lattice', 'r-openxlsx', 'r-venndiagram'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SwathXtend\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SwathXtend.html'), ('license', 'GPL-2'), ('summary', 'It contains utility functions for integrating spectral libraries for SWATH and statistical data analysis for SWATH generated data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motifbreakr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motifbreakr'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'motifbreakR_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/motifbreakR_1.4.0.tar.gz'), ('md5', '66d72ab927fb1da866a9376416fc2e79')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-motifdb', 'bioconductor-motifstack', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation', 'r-base', 'r-grimport', 'r-matrixstats', 'r-stringr', 'r-tfmpvalue']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-motifdb', 'bioconductor-motifstack', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation', 'r-base', 'r-grimport', 'r-matrixstats', 'r-stringr', 'r-tfmpvalue'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'motifbreakR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/motifbreakR.html'), ('license', 'GPL-2'), ('summary', 'We introduce motifbreakR, which allows the biologist to judge in the first place whether the sequence surrounding the polymorphism is a good match, and in the second place how much information is gained or lost in one allele of the polymorphism relative to another. MotifbreakR is both flexible and extensible over previous offerings; giving a choice of algorithms for interrogation of genomes with motifs from public sources that users can choose from; these are 1) a weighted-sum probability matrix, 2) log-probabilities, and 3) weighted by relative entropy. MotifbreakR can predict effects for novel or previously described variants in public databases, making it suitable for tasks beyond the scope of its original design. Lastly, it can be used to interrogate any genome curated within Bioconductor (currently there are 22).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motifbreakr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motifbreakr'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'motifbreakR_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/motifbreakR_1.4.0.tar.gz'), ('md5', '66d72ab927fb1da866a9376416fc2e79')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-motifdb', 'bioconductor-motifstack', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation', 'r-base', 'r-grimport', 'r-matrixstats', 'r-stringr', 'r-tfmpvalue']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-motifdb', 'bioconductor-motifstack', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation', 'r-base', 'r-grimport', 'r-matrixstats', 'r-stringr', 'r-tfmpvalue'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'motifbreakR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/motifbreakR.html'), ('license', 'GPL-2'), ('summary', 'We introduce motifbreakR, which allows the biologist to judge in the first place whether the sequence surrounding the polymorphism is a good match, and in the second place how much information is gained or lost in one allele of the polymorphism relative to another. MotifbreakR is both flexible and extensible over previous offerings; giving a choice of algorithms for interrogation of genomes with motifs from public sources that users can choose from; these are 1) a weighted-sum probability matrix, 2) log-probabilities, and 3) weighted by relative entropy. MotifbreakR can predict effects for novel or previously described variants in public databases, making it suitable for tasks beyond the scope of its original design. Lastly, it can be used to interrogate any genome curated within Bioconductor (currently there are 22).')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/illuminate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'illuminate'), ('version', '0.6.3')])), ('source', CommentedMap([('fn', 'illuminate-0.6.3.zip'), ('url', 'https://bitbucket.org/ilmn8/illuminate/get/e19b8afefead.zip'), ('md5', 'cfea74ea8b646884df2396eb7e6d6e9e')])), ('build', CommentedMap([('entry_points', ['illuminate = illuminate.__main__:collect_args'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'bitstring >=3.1.0', 'docopt', 'numpy >=1.6.2', 'pandas >=0.14', 'openpyxl ==1.8.6', 'xmltodict', 'gcc']), ('run', ['python', 'bitstring >=3.1.0', 'docopt', 'numpy >=1.6.2', 'pandas >=0.14', 'openpyxl ==1.8.6', 'xmltodict', 'libgcc'])])), ('test', CommentedMap([('imports', ['illuminate']), ('commands', ['illuminate --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/invitae/illuminate'), ('license', 'MIT'), ('summary', 'Analytics toolkit for Illumina sequencer metrics.'), ('license_family', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/illuminate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'illuminate'), ('version', '0.6.3')])), ('source', CommentedMap([('fn', 'illuminate-0.6.3.zip'), ('url', 'https://bitbucket.org/ilmn8/illuminate/get/e19b8afefead.zip'), ('md5', 'cfea74ea8b646884df2396eb7e6d6e9e')])), ('build', CommentedMap([('entry_points', ['illuminate = illuminate.__main__:collect_args'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'bitstring >=3.1.0', 'docopt', 'numpy >=1.6.2', 'pandas >=0.14', 'openpyxl ==1.8.6', 'xmltodict', 'gcc']), ('run', ['python', 'bitstring >=3.1.0', 'docopt', 'numpy >=1.6.2', 'pandas >=0.14', 'openpyxl ==1.8.6', 'xmltodict', 'libgcc'])])), ('test', CommentedMap([('imports', ['illuminate']), ('commands', ['illuminate --help'])])), ('about', CommentedMap([('home', 'https://bitbucket.org/invitae/illuminate'), ('license', 'MIT'), ('summary', 'Analytics toolkit for Illumina sequencer metrics.'), ('license_family', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xxmotif/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xxmotif'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'xxmotif-<{ version }}.tar.gz'), ('url', 'https://github.com/soedinglab/xxmotif/archive/<{ version }}.tar.gz'), ('sha256', '235eb8ad562c4757a549751bb290f443ff70c29a142edbff0f03440f097b61cb')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'https://github.com/soedinglab/xxmotif'), ('summary', 'eXhaustive, weight matriX-based motif discovery in nucleotide sequences'), ('license', 'GPLv3'), ('license_file', 'LICENSE')])), ('test', CommentedMap([('commands', ['test -f ${PREFIX}/bin/XXmotif'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/xxmotif/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'xxmotif'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'xxmotif-<{ version }}.tar.gz'), ('url', 'https://github.com/soedinglab/xxmotif/archive/<{ version }}.tar.gz'), ('sha256', '235eb8ad562c4757a549751bb290f443ff70c29a142edbff0f03440f097b61cb')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'https://github.com/soedinglab/xxmotif'), ('summary', 'eXhaustive, weight matriX-based motif discovery in nucleotide sequences'), ('license', 'GPLv3'), ('license_file', 'LICENSE')])), ('test', CommentedMap([('commands', ['test -f ${PREFIX}/bin/XXmotif'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-quantro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-quantro'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'quantro_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/quantro_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/quantro_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/quantro/quantro_1.10.0_src_all.tar.gz']), ('md5', '0928efa544591e3f4e3c50147cda79d1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-minfi', 'r-base', 'r-doparallel', 'r-foreach', 'r-ggplot2', 'r-iterators', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase', 'bioconductor-minfi', 'r-base', 'r-doparallel', 'r-foreach', 'r-ggplot2', 'r-iterators', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'quantro\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/quantro.html'), ('license', 'GPL (>=3)'), ('summary', 'A data-driven test for the assumptions of quantile normalization using raw data such as objects that inherit eSets (e.g. ExpressionSet, MethylSet). Group level information about each sample (such as Tumor / Normal status) must also be provided because the test assesses if there are global differences in the distributions between the user-defined groups.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-quantro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-quantro'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'quantro_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/quantro_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/quantro_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/quantro/quantro_1.10.0_src_all.tar.gz']), ('md5', '0928efa544591e3f4e3c50147cda79d1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-minfi', 'r-base', 'r-doparallel', 'r-foreach', 'r-ggplot2', 'r-iterators', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase', 'bioconductor-minfi', 'r-base', 'r-doparallel', 'r-foreach', 'r-ggplot2', 'r-iterators', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'quantro\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/quantro.html'), ('license', 'GPL (>=3)'), ('summary', 'A data-driven test for the assumptions of quantile normalization using raw data such as objects that inherit eSets (e.g. ExpressionSet, MethylSet). Group level information about each sample (such as Tumor / Normal status) must also be provided because the test assesses if there are global differences in the distributions between the user-defined groups.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-graph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-graph'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'graph_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/graph_1.50.0.tar.gz'), ('md5', 'd1fbfd7dfffce6f46a0e1d288fa5858a'), ('patches', ['Makevars.patch'])])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.13.11', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.13.11', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'graph\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/graph.html'), ('license', 'Artistic-2.0'), ('summary', 'A package that implements some simple graph handling capabilities.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-graph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-graph'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'graph_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/graph_1.50.0.tar.gz'), ('md5', 'd1fbfd7dfffce6f46a0e1d288fa5858a'), ('patches', ['Makevars.patch'])])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.13.11', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.13.11', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'graph\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/graph.html'), ('license', 'Artistic-2.0'), ('summary', 'A package that implements some simple graph handling capabilities.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biseq'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'BiSeq_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BiSeq_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/BiSeq_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/BiSeq/BiSeq_1.16.0_src_all.tar.gz']), ('md5', 'c0579533cd705c74f9e8dc513996c713')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-globaltest', 'bioconductor-iranges >=1.17.24', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-betareg', 'r-formula', 'r-lokern']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-globaltest', 'bioconductor-iranges >=1.17.24', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-betareg', 'r-formula', 'r-lokern'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BiSeq.html'), ('license', 'LGPL-3'), ('summary', 'The BiSeq package provides useful classes and functions to handle and analyze targeted bisulfite sequencing (BS) data such as reduced-representation bisulfite sequencing (RRBS) data. In particular, it implements an algorithm to detect differentially methylated regions (DMRs). The package takes already aligned BS data from one or multiple samples.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biseq'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'BiSeq_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BiSeq_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/BiSeq_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/BiSeq/BiSeq_1.16.0_src_all.tar.gz']), ('md5', 'c0579533cd705c74f9e8dc513996c713')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-globaltest', 'bioconductor-iranges >=1.17.24', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-betareg', 'r-formula', 'r-lokern']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-globaltest', 'bioconductor-iranges >=1.17.24', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-betareg', 'r-formula', 'r-lokern'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BiSeq.html'), ('license', 'LGPL-3'), ('summary', 'The BiSeq package provides useful classes and functions to handle and analyze targeted bisulfite sequencing (BS) data such as reduced-representation bisulfite sequencing (RRBS) data. In particular, it implements an algorithm to detect differentially methylated regions (DMRs). The package takes already aligned BS data from one or multiple samples.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastool/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastool'), ('version', '0.1.4')])), ('source', CommentedMap([('fn', '0.1.4.tar.gz'), ('url', 'https://github.com/fstrozzi/Fastool/archive/0.1.4.tar.gz'), ('sha256', 'f86f02fced9479b77d6d27a442b66e66f0c55d40d5ad2ddd91a752de57024540')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['fastool 2>&1 | grep append_to > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/fstrozzi/Fastool'), ('license', 'MIT'), ('summary', 'A simple and quick tool to read huge FastQ and FastA files (both normal and gzipped) and manipulate them.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastool/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastool'), ('version', '0.1.4')])), ('source', CommentedMap([('fn', '0.1.4.tar.gz'), ('url', 'https://github.com/fstrozzi/Fastool/archive/0.1.4.tar.gz'), ('sha256', 'f86f02fced9479b77d6d27a442b66e66f0c55d40d5ad2ddd91a752de57024540')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['fastool 2>&1 | grep append_to > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/fstrozzi/Fastool'), ('license', 'MIT'), ('summary', 'A simple and quick tool to read huge FastQ and FastA files (both normal and gzipped) and manipulate them.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-scope-guard/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-scope-guard'), ('version', '0.21')])), ('source', CommentedMap([('fn', 'Scope-Guard-0.21.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CH/CHOCOLATE/Scope-Guard-0.21.tar.gz'), ('md5', 'be57b915d23ddac7677ef2ad9e52b92a')])), ('build', None), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-simple', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Scope::Guard'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Scope::Guard'), ('license', 'perl_5'), ('summary', 'lexically-scoped resource management')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-scope-guard/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-scope-guard'), ('version', '0.21')])), ('source', CommentedMap([('fn', 'Scope-Guard-0.21.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CH/CHOCOLATE/Scope-Guard-0.21.tar.gz'), ('md5', 'be57b915d23ddac7677ef2ad9e52b92a')])), ('build', None), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-test-simple', 'perl-extutils-makemaker']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('imports', ['Scope::Guard'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Scope::Guard'), ('license', 'perl_5'), ('summary', 'lexically-scoped resource management')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-npgsea/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-npgsea'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'npGSEA_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/npGSEA_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/npGSEA_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/npGSEA/npGSEA_1.12.0_src_all.tar.gz']), ('md5', '15869313306ed711156c582821d4c4bb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.24.0', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.24.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'npGSEA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/npGSEA.html'), ('license', 'Artistic-2.0'), ('summary', 'Current gene set enrichment methods rely upon permutations for inference.  These approaches are computationally expensive and have minimum achievable p-values based on the number of permutations, not on the actual observed statistics.  We have derived three parametric approximations to the permutation distributions of two gene set enrichment test statistics.  We are able to reduce the computational burden and granularity issues of permutation testing with our method, which is implemented in this package. npGSEA calculates gene set enrichment statistics and p-values without the computational cost of permutations.  It is applicable in settings where one or many gene sets are of interest.  There are also built-in plotting functions to help users visualize results.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-npgsea/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-npgsea'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'npGSEA_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/npGSEA_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/npGSEA_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/npGSEA/npGSEA_1.12.0_src_all.tar.gz']), ('md5', '15869313306ed711156c582821d4c4bb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.24.0', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.24.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'npGSEA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/npGSEA.html'), ('license', 'Artistic-2.0'), ('summary', 'Current gene set enrichment methods rely upon permutations for inference.  These approaches are computationally expensive and have minimum achievable p-values based on the number of permutations, not on the actual observed statistics.  We have derived three parametric approximations to the permutation distributions of two gene set enrichment test statistics.  We are able to reduce the computational burden and granularity issues of permutation testing with our method, which is implemented in this package. npGSEA calculates gene set enrichment statistics and p-values without the computational cost of permutations.  It is applicable in settings where one or many gene sets are of interest.  There are also built-in plotting functions to help users visualize results.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-base/2.23/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'base-2.23.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/base-2.23.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use base\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/base'), ('license', 'unknown'), ('summary', 'compile-time class fields')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-base/2.23/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'base-2.23.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/R/RJ/RJBS/base-2.23.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl', 'perl-extutils-makemaker']), ('run', ['perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use base\"'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/base'), ('license', 'unknown'), ('summary', 'compile-time class fields')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mgfr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mgfr'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'MGFR_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MGFR_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/MGFR_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/MGFR/MGFR_1.2.0_src_all.tar.gz']), ('md5', '50d79a1e21138cf4f84326811d3da6ba')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-biomart', 'r-base']), ('run', ['bioconductor-annotate', 'bioconductor-biomart', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MGFR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MGFR.html'), ('license', 'GPL-3'), ('summary', 'The package is designed to detect marker genes from RNA-seq data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mgfr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mgfr'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'MGFR_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MGFR_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/MGFR_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/MGFR/MGFR_1.2.0_src_all.tar.gz']), ('md5', '50d79a1e21138cf4f84326811d3da6ba')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-biomart', 'r-base']), ('run', ['bioconductor-annotate', 'bioconductor-biomart', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MGFR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MGFR.html'), ('license', 'GPL-3'), ('summary', 'The package is designed to detect marker genes from RNA-seq data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hisat2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hisat2'), ('version', '2.1.0')])), ('source', CommentedMap([('fn', 'hisat2-2.1.0-OSX_x86_64.zip'), ('url', 'ftp://ftp.ccb.jhu.edu/pub/infphilo/hisat2/downloads/hisat2-2.1.0-OSX_x86_64.zip'), ('sha256', '7416d5381ffa64914e15f6a930cfb9eb3ab1f5b9894ef8189930d1670384cd81')])), ('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'https://ccb.jhu.edu/software/hisat2/index.shtml'), ('license', 'GPLv3'), ('summary', 'graph-based alignment of next generation sequencing reads to a population of genomes')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['libgcc', 'python', 'perl'])])), ('test', CommentedMap([('commands', ['hisat2 --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hisat2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hisat2'), ('version', '2.1.0')])), ('source', CommentedMap([('fn', 'hisat2-2.1.0-OSX_x86_64.zip'), ('url', 'ftp://ftp.ccb.jhu.edu/pub/infphilo/hisat2/downloads/hisat2-2.1.0-OSX_x86_64.zip'), ('sha256', '7416d5381ffa64914e15f6a930cfb9eb3ab1f5b9894ef8189930d1670384cd81')])), ('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'https://ccb.jhu.edu/software/hisat2/index.shtml'), ('license', 'GPLv3'), ('summary', 'graph-based alignment of next generation sequencing reads to a population of genomes')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['libgcc', 'python', 'perl'])])), ('test', CommentedMap([('commands', ['hisat2 --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gothic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gothic'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'GOTHiC_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/GOTHiC_1.10.0.tar.gz'), ('md5', 'd53ee31c7fe5d8924ce29eb3497da4ad')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.38', 'bioconductor-shortread', 'r-base', 'r-data.table', 'r-ggplot2']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.38', 'bioconductor-shortread', 'r-base', 'r-data.table', 'r-ggplot2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GOTHiC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GOTHiC.html'), ('license', 'GPL-3'), ('summary', 'This is a Hi-C analysis package using a cumulative binomial test to detect interactions between distal genomic loci that have significantly more reads than expected by chance in Hi-C experiments. It takes mapped paired NGS reads as input and gives back the list of significant interactions for a given bin size in the genome.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gothic/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gothic'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'GOTHiC_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/GOTHiC_1.10.0.tar.gz'), ('md5', 'd53ee31c7fe5d8924ce29eb3497da4ad')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.38', 'bioconductor-shortread', 'r-base', 'r-data.table', 'r-ggplot2']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.38', 'bioconductor-shortread', 'r-base', 'r-data.table', 'r-ggplot2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GOTHiC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GOTHiC.html'), ('license', 'GPL-3'), ('summary', 'This is a Hi-C analysis package using a cumulative binomial test to detect interactions between distal genomic loci that have significantly more reads than expected by chance in Hi-C experiments. It takes mapped paired NGS reads as input and gives back the list of significant interactions for a given bin size in the genome.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/scoary/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'scoary'), ('version', '1.6.9')])), ('source', CommentedMap([('fn', 'v1.6.9.tar.gz'), ('url', 'https://github.com/AdmiralenOla/Scoary/archive/v1.6.9.tar.gz')])), ('build', CommentedMap([('entry_points', ['scoary=scoary.methods:main', 'scoary_GUI=scoary.GUI:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scipy >=0.16', 'argparse', 'ete3', 'six']), ('run', ['python', 'scipy >=0.16', 'argparse', 'ete3', 'six'])])), ('test', CommentedMap([('imports', ['scoary']), ('commands', ['scoary --help'])])), ('about', CommentedMap([('home', 'https://github.com/AdmiralenOla/Scoary'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Microbial pan-GWAS using the output from Roary'), ('license_family', 'GPL3')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/scoary/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'scoary'), ('version', '1.6.9')])), ('source', CommentedMap([('fn', 'v1.6.9.tar.gz'), ('url', 'https://github.com/AdmiralenOla/Scoary/archive/v1.6.9.tar.gz')])), ('build', CommentedMap([('entry_points', ['scoary=scoary.methods:main', 'scoary_GUI=scoary.GUI:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'scipy >=0.16', 'argparse', 'ete3', 'six']), ('run', ['python', 'scipy >=0.16', 'argparse', 'ete3', 'six'])])), ('test', CommentedMap([('imports', ['scoary']), ('commands', ['scoary --help'])])), ('about', CommentedMap([('home', 'https://github.com/AdmiralenOla/Scoary'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Microbial pan-GWAS using the output from Roary'), ('license_family', 'GPL3')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/structure/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'structure_kernel_source.tar.gz'), ('md5', '4e0591678cdbfe79347d272b5dceeda1'), ('url', 'https://web.stanford.edu/group/pritchardlab/structure_software/release_versions/v<{ version }}/structure_kernel_source.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['structure | grep \"Version <{ version }}\"'])])), ('about', CommentedMap([('home', 'https://web.stanford.edu/group/pritchardlab/structure.html'), ('license', 'MIT'), ('summary', 'The program structure is a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed. It can be applied to most of the commonly-used genetic markers, including SNPS, microsatellites, RFLPs and AFLPs.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/structure/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'structure_kernel_source.tar.gz'), ('md5', '4e0591678cdbfe79347d272b5dceeda1'), ('url', 'https://web.stanford.edu/group/pritchardlab/structure_software/release_versions/v<{ version }}/structure_kernel_source.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['structure | grep \"Version <{ version }}\"'])])), ('about', CommentedMap([('home', 'https://web.stanford.edu/group/pritchardlab/structure.html'), ('license', 'MIT'), ('summary', 'The program structure is a free software package for using multi-locus genotype data to investigate population structure. Its uses include inferring the presence of distinct populations, assigning individuals to populations, studying hybrid zones, identifying migrants and admixed individuals, and estimating population allele frequencies in situations where many individuals are migrants or admixed. It can be applied to most of the commonly-used genetic markers, including SNPS, microsatellites, RFLPs and AFLPs.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/segway/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'segway'), ('version', '2.0')])), ('source', CommentedMap([('fn', 'segway-2.0.tar.gz'), ('url', 'https://pypi.python.org/packages/09/13/96398f873521032ee80b8665c59785445f4870159f8cb47840e22e8fd156/segway-2.0.tar.gz'), ('md5', 'b4bd3d4ebfcfe1e5540f8da523956764')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('entry_points', ['segway = segway.run:main', 'segway-task = segway.task:main', 'segway-layer = segway.layer:main', 'segway-winner = segway.winner:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'genomedata', 'gmtk', 'textinput', 'optbuild', 'optplus', 'pytables >=3.0', 'numpy', 'forked-path', 'hdf5', 'hgtools', 'colorbrewer', 'drmaa >=0.4a3']), ('run', ['python', 'genomedata', 'gmtk', 'textinput', 'optbuild', 'optplus', 'pytables >=3.0', 'numpy', 'forked-path', 'hdf5', 'colorbrewer', 'drmaa >=0.4a3'])])), ('test', CommentedMap([('imports', ['segway']), ('commands', ['segway --help'])])), ('about', CommentedMap([('home', 'http://segway.hoffmanlab.org/'), ('license', 'GPL2'), ('summary', 'a tool for easy pattern discovery and identification in functional genomics data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/segway/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'segway'), ('version', '2.0')])), ('source', CommentedMap([('fn', 'segway-2.0.tar.gz'), ('url', 'https://pypi.python.org/packages/09/13/96398f873521032ee80b8665c59785445f4870159f8cb47840e22e8fd156/segway-2.0.tar.gz'), ('md5', 'b4bd3d4ebfcfe1e5540f8da523956764')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('entry_points', ['segway = segway.run:main', 'segway-task = segway.task:main', 'segway-layer = segway.layer:main', 'segway-winner = segway.winner:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'genomedata', 'gmtk', 'textinput', 'optbuild', 'optplus', 'pytables >=3.0', 'numpy', 'forked-path', 'hdf5', 'hgtools', 'colorbrewer', 'drmaa >=0.4a3']), ('run', ['python', 'genomedata', 'gmtk', 'textinput', 'optbuild', 'optplus', 'pytables >=3.0', 'numpy', 'forked-path', 'hdf5', 'colorbrewer', 'drmaa >=0.4a3'])])), ('test', CommentedMap([('imports', ['segway']), ('commands', ['segway --help'])])), ('about', CommentedMap([('home', 'http://segway.hoffmanlab.org/'), ('license', 'GPL2'), ('summary', 'a tool for easy pattern discovery and identification in functional genomics data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-papi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-papi'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'PAPi_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PAPi_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/PAPi_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/PAPi/PAPi_1.16.0_src_all.tar.gz']), ('md5', '428a61cedc96ed775381b0c4798e8cb5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-keggrest', 'r-base', 'r-svdialogs']), ('run', ['bioconductor-keggrest', 'r-base', 'r-svdialogs'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PAPi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PAPi.html'), ('license', 'GPL(>= 2)'), ('summary', 'The Pathway Activity Profiling - PAPi - is an R package for predicting the activity of metabolic pathways based solely on a metabolomics data set containing a list of metabolites identified and their respective abundances in different biological samples. PAPi generates hypothesis that improves the final biological interpretation. See Aggio, R.B.M; Ruggiero, K. and Villas-Boas, S.G. (2010) - Pathway Activity Profiling (PAPi): from metabolite profile to metabolic pathway activity. Bioinformatics.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-papi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-papi'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'PAPi_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PAPi_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/PAPi_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/PAPi/PAPi_1.16.0_src_all.tar.gz']), ('md5', '428a61cedc96ed775381b0c4798e8cb5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-keggrest', 'r-base', 'r-svdialogs']), ('run', ['bioconductor-keggrest', 'r-base', 'r-svdialogs'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PAPi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PAPi.html'), ('license', 'GPL(>= 2)'), ('summary', 'The Pathway Activity Profiling - PAPi - is an R package for predicting the activity of metabolic pathways based solely on a metabolomics data set containing a list of metabolites identified and their respective abundances in different biological samples. PAPi generates hypothesis that improves the final biological interpretation. See Aggio, R.B.M; Ruggiero, K. and Villas-Boas, S.G. (2010) - Pathway Activity Profiling (PAPi): from metabolite profile to metabolic pathway activity. Bioinformatics.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mgsa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mgsa'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'mgsa_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mgsa_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/mgsa_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/mgsa/mgsa_1.24.0_src_all.tar.gz']), ('md5', '1a0757f3de04eff70284e297afbae1dc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gplots']), ('run', ['r-base', 'r-gplots'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mgsa\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mgsa.html'), ('license', 'Artistic-2.0'), ('summary', 'Model-based Gene Set Analysis (MGSA) is a Bayesian modeling approach for gene set enrichment. The package mgsa implements MGSA and tools to use MGSA together with the Gene Ontology.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mgsa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mgsa'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'mgsa_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mgsa_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/mgsa_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/mgsa/mgsa_1.24.0_src_all.tar.gz']), ('md5', '1a0757f3de04eff70284e297afbae1dc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gplots']), ('run', ['r-base', 'r-gplots'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mgsa\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mgsa.html'), ('license', 'Artistic-2.0'), ('summary', 'Model-based Gene Set Analysis (MGSA) is a Bayesian modeling approach for gene set enrichment. The package mgsa implements MGSA and tools to use MGSA together with the Gene Ontology.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ringo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ringo'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'Ringo_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/Ringo_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/Ringo_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/Ringo/Ringo_1.38.0_src_all.tar.gz']), ('md5', '17a1b47d16f09dd4525f26b272c48539')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.14.1', 'bioconductor-biocgenerics >=0.1.11', 'bioconductor-genefilter', 'bioconductor-limma', 'bioconductor-vsn', 'r-base', 'r-lattice', 'r-matrix', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase >=1.14.1', 'bioconductor-biocgenerics >=0.1.11', 'bioconductor-genefilter', 'bioconductor-limma', 'bioconductor-vsn', 'r-base', 'r-lattice', 'r-matrix', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Ringo\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Ringo.html'), ('license', 'Artistic-2.0'), ('summary', 'The package Ringo facilitates the primary analysis of ChIP-chip data. The main functionalities of the package are data read-in, quality assessment, data visualisation and identification of genomic regions showing enrichment in ChIP-chip. The package has functions to deal with two-color oligonucleotide microarrays from NimbleGen used in ChIP-chip projects, but also contains more general functions for ChIP-chip data analysis, given that the data is supplied as RGList (raw)  or ExpressionSet (pre- processed). The package employs functions from various other packages of the Bioconductor project and provides additional ChIP-chip-specific and NimbleGen-specific functionalities.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ringo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ringo'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'Ringo_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/Ringo_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/Ringo_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/Ringo/Ringo_1.38.0_src_all.tar.gz']), ('md5', '17a1b47d16f09dd4525f26b272c48539')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.14.1', 'bioconductor-biocgenerics >=0.1.11', 'bioconductor-genefilter', 'bioconductor-limma', 'bioconductor-vsn', 'r-base', 'r-lattice', 'r-matrix', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase >=1.14.1', 'bioconductor-biocgenerics >=0.1.11', 'bioconductor-genefilter', 'bioconductor-limma', 'bioconductor-vsn', 'r-base', 'r-lattice', 'r-matrix', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Ringo\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Ringo.html'), ('license', 'Artistic-2.0'), ('summary', 'The package Ringo facilitates the primary analysis of ChIP-chip data. The main functionalities of the package are data read-in, quality assessment, data visualisation and identification of genomic regions showing enrichment in ChIP-chip. The package has functions to deal with two-color oligonucleotide microarrays from NimbleGen used in ChIP-chip projects, but also contains more general functions for ChIP-chip data analysis, given that the data is supplied as RGList (raw)  or ExpressionSet (pre- processed). The package employs functions from various other packages of the Bioconductor project and provides additional ChIP-chip-specific and NimbleGen-specific functionalities.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/star/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'star'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/alexdobin/STAR/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'https://github.com/alexdobin/STAR'), ('license', 'GPLv3'), ('summary', 'An RNA-seq read aligner.')])), ('test', CommentedMap([('commands', ['STAR --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/star/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'star'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/alexdobin/STAR/archive/<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'https://github.com/alexdobin/STAR'), ('license', 'GPLv3'), ('summary', 'An RNA-seq read aligner.')])), ('test', CommentedMap([('commands', ['STAR --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ebseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ebseq'), ('version', '1.14.0')])), ('source', CommentedMap([('url', 'https://bioarchive.galaxyproject.org/EBSeq_1.14.0.tar.gz'), ('md5', 'ea44b6345b561cf238845d5690aa69e3')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-blockmodeling', 'r-gplots', 'r-testthat']), ('run', ['r-base', 'r-blockmodeling', 'r-gplots', 'r-testthat'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'EBSeq\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/EBSeq.html'), ('license', 'Artistic-2.0'), ('summary', 'Differential Expression analysis at both gene and isoform level using RNA-seq data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ebseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ebseq'), ('version', '1.14.0')])), ('source', CommentedMap([('url', 'https://bioarchive.galaxyproject.org/EBSeq_1.14.0.tar.gz'), ('md5', 'ea44b6345b561cf238845d5690aa69e3')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-blockmodeling', 'r-gplots', 'r-testthat']), ('run', ['r-base', 'r-blockmodeling', 'r-gplots', 'r-testthat'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'EBSeq\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/EBSeq.html'), ('license', 'Artistic-2.0'), ('summary', 'Differential Expression analysis at both gene and isoform level using RNA-seq data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bowtie/1.0.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://bowtie-bio.sourceforge.net/index.shtml'), ('license', 'GPLv3'), ('summary', 'An ultrafast memory-efficient short read aligner')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'bowtie'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'bowtie-1.0.0-src.zip'), ('sha256', '51e434a78e053301f82ae56f4e94f71f97b19f7175324777a7305c8f88c5bac5'), ('url', 'http://downloads.sourceforge.net/project/bowtie-bio/bowtie/1.0.0/bowtie-1.0.0-src.zip')])), ('test', CommentedMap([('commands', ['(bowtie --version 2>&1) > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bowtie/1.0.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://bowtie-bio.sourceforge.net/index.shtml'), ('license', 'GPLv3'), ('summary', 'An ultrafast memory-efficient short read aligner')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'bowtie'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'bowtie-1.0.0-src.zip'), ('sha256', '51e434a78e053301f82ae56f4e94f71f97b19f7175324777a7305c8f88c5bac5'), ('url', 'http://downloads.sourceforge.net/project/bowtie-bio/bowtie/1.0.0/bowtie-1.0.0-src.zip')])), ('test', CommentedMap([('commands', ['(bowtie --version 2>&1) > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sepa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sepa'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'SEPA_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SEPA_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/SEPA_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/SEPA/SEPA_1.6.0_src_all.tar.gz']), ('md5', '93c9ffd58ae48df9f51a396fc0dbe6d0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-org.hs.eg.db', 'bioconductor-org.mm.eg.db', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-reshape2', 'r-segmented', 'r-shiny']), ('run', ['bioconductor-org.hs.eg.db', 'bioconductor-org.mm.eg.db', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-reshape2', 'r-segmented', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SEPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SEPA.html'), ('license', 'GPL(>=2)'), ('summary', 'Given single-cell RNA-seq data and true experiment time of cells or pseudo-time cell ordering, SEPA provides convenient functions for users to assign genes into different gene expression patterns such as constant, monotone increasing and increasing then decreasing. SEPA then performs GO enrichment analysis to analysis the functional roles of genes with same or similar patterns.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sepa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sepa'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'SEPA_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SEPA_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/SEPA_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/SEPA/SEPA_1.6.0_src_all.tar.gz']), ('md5', '93c9ffd58ae48df9f51a396fc0dbe6d0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-org.hs.eg.db', 'bioconductor-org.mm.eg.db', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-reshape2', 'r-segmented', 'r-shiny']), ('run', ['bioconductor-org.hs.eg.db', 'bioconductor-org.mm.eg.db', 'bioconductor-topgo', 'r-base', 'r-ggplot2', 'r-reshape2', 'r-segmented', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SEPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SEPA.html'), ('license', 'GPL(>=2)'), ('summary', 'Given single-cell RNA-seq data and true experiment time of cells or pseudo-time cell ordering, SEPA provides convenient functions for users to assign genes into different gene expression patterns such as constant, monotone increasing and increasing then decreasing. SEPA then performs GO enrichment analysis to analysis the functional roles of genes with same or similar patterns.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqpattern/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqpattern'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'seqPattern_1.6.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-seqpattern/bioconductor-seqpattern_1.6.0_src_all.tar.gz'), ('md5', 'b0f51e76f8e30b7bcd2969f97622d83c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'r-base', 'r-plotrix', 'r-kernsmooth']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'r-base', 'r-plotrix', 'r-kernsmooth'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'seqPattern\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/seqPattern.html'), ('license', 'GPL-3'), ('summary', 'Visualising oligonucleotide patterns and sequence motifs occurrences across a large set of sequences centred at a common reference point and sorted by a user defined feature.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqpattern/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqpattern'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'seqPattern_1.6.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-seqpattern/bioconductor-seqpattern_1.6.0_src_all.tar.gz'), ('md5', 'b0f51e76f8e30b7bcd2969f97622d83c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'r-base', 'r-plotrix', 'r-kernsmooth']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'r-base', 'r-plotrix', 'r-kernsmooth'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'seqPattern\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/seqPattern.html'), ('license', 'GPL-3'), ('summary', 'Visualising oligonucleotide patterns and sequence motifs occurrences across a large set of sequences centred at a common reference point and sorted by a user defined feature.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/proteowizard/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'proteowizard'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'http://teamcity.labkey.org/guestAuth/repository/download/bt81/365914:id/pwiz-src-<{ version }}.tar.bz2'), ('sha1', '4bf02eb83755b3df822a2d35b39d577be84f8282')])), ('build', CommentedMap([('script', 'bash -c \"./quickbuild.sh -j<{ CPU_COUNT }} address-model=64 runtime-link=shared executables && cp build-linux-x86_64/gcc-release-x86_64/* $PREFIX/bin\"'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'https://proteowizard.sourceforge.net'), ('license', 'Apache 2.0'), ('summary', 'Tools for dealing with mass spectrometry files (e.g., mzML, mzXML, mzIdentML, MGF)')])), ('test', CommentedMap([('commands', ['msconvert --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/proteowizard/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'proteowizard'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'http://teamcity.labkey.org/guestAuth/repository/download/bt81/365914:id/pwiz-src-<{ version }}.tar.bz2'), ('sha1', '4bf02eb83755b3df822a2d35b39d577be84f8282')])), ('build', CommentedMap([('script', 'bash -c \"./quickbuild.sh -j<{ CPU_COUNT }} address-model=64 runtime-link=shared executables && cp build-linux-x86_64/gcc-release-x86_64/* $PREFIX/bin\"'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('about', CommentedMap([('home', 'https://proteowizard.sourceforge.net'), ('license', 'Apache 2.0'), ('summary', 'Tools for dealing with mass spectrometry files (e.g., mzML, mzXML, mzIdentML, MGF)')])), ('test', CommentedMap([('commands', ['msconvert --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/schema-salad/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'schema-salad'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'schema-salad-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/s/schema-salad/schema-salad-<{ version }}.tar.gz'), ('md5', 'f45cd87c34a90ccedae2d00278a9c0af')])), ('build', CommentedMap([('entry_points', ['schema-salad-tool=schema_salad.main:main']), ('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'rdflib >=4.2.0', 'rdflib-jsonld >=0.3.0', 'mistune', 'typing >=3.5.2,<3.6', 'ruamel.yaml >=0.12.4,<0.15', 'cachecontrol >=0.11.7,<0.12', 'lockfile', 'avro-python2 ==1.8.1', 'avro-python3']), ('run', ['python', 'requests', 'rdflib >=4.2.0', 'rdflib-jsonld >=0.3.0', 'mistune', 'typing >=3.5.2,<3.6', 'ruamel.yaml >=0.12.4,<0.15', 'cachecontrol >=0.11.7,<0.12', 'lockfile', 'avro-python2 ==1.8.1', 'avro-python3'])])), ('test', CommentedMap([('imports', ['schema_salad']), ('commands', ['schema-salad-tool --help'])])), ('about', CommentedMap([('home', 'https://github.com/common-workflow-language/schema_salad'), ('license', 'Apache 2.0'), ('summary', 'Schema Annotations for Linked Avro Data (SALAD)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/schema-salad/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'schema-salad'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'schema-salad-<{ version }}.tar.gz'), ('url', 'https://pypi.io/packages/source/s/schema-salad/schema-salad-<{ version }}.tar.gz'), ('md5', 'f45cd87c34a90ccedae2d00278a9c0af')])), ('build', CommentedMap([('entry_points', ['schema-salad-tool=schema_salad.main:main']), ('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'requests', 'rdflib >=4.2.0', 'rdflib-jsonld >=0.3.0', 'mistune', 'typing >=3.5.2,<3.6', 'ruamel.yaml >=0.12.4,<0.15', 'cachecontrol >=0.11.7,<0.12', 'lockfile', 'avro-python2 ==1.8.1', 'avro-python3']), ('run', ['python', 'requests', 'rdflib >=4.2.0', 'rdflib-jsonld >=0.3.0', 'mistune', 'typing >=3.5.2,<3.6', 'ruamel.yaml >=0.12.4,<0.15', 'cachecontrol >=0.11.7,<0.12', 'lockfile', 'avro-python2 ==1.8.1', 'avro-python3'])])), ('test', CommentedMap([('imports', ['schema_salad']), ('commands', ['schema-salad-tool --help'])])), ('about', CommentedMap([('home', 'https://github.com/common-workflow-language/schema_salad'), ('license', 'Apache 2.0'), ('summary', 'Schema Annotations for Linked Avro Data (SALAD)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-signer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-signer'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'signeR_1.0.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/signeR_1.0.1.tar.gz', 'https://depot.galaxyproject.org/software/signeR/signeR_1.0.1_src_all.tar.gz']), ('md5', 'edc96646e9c9e8f88a0a147f4662095b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome >=1.36.3', 'bioconductor-genomicranges', 'bioconductor-variantannotation', 'r-base', 'r-class', 'r-nloptr', 'r-nmf', 'r-pmcmr', 'r-rcpp', 'r-rcpparmadillo', 'gcc']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome >=1.36.3', 'bioconductor-genomicranges', 'bioconductor-variantannotation', 'r-base', 'r-class', 'r-nloptr', 'r-nmf', 'r-pmcmr', 'r-rcpp', 'r-rcpparmadillo', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'signeR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/signeR.html'), ('license', 'GPL-3'), ('summary', 'The signeR package provides an empirical Bayesian approach to mutational signature discovery. It is designed to analyze single nucleotide variaton (SNV) counts in cancer genomes, but can also be applied to other features as well. Functionalities to characterize signatures or genome samples according to exposure patterns are also provided.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-signer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-signer'), ('version', '1.0.1')])), ('source', CommentedMap([('fn', 'signeR_1.0.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/signeR_1.0.1.tar.gz', 'https://depot.galaxyproject.org/software/signeR/signeR_1.0.1_src_all.tar.gz']), ('md5', 'edc96646e9c9e8f88a0a147f4662095b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome >=1.36.3', 'bioconductor-genomicranges', 'bioconductor-variantannotation', 'r-base', 'r-class', 'r-nloptr', 'r-nmf', 'r-pmcmr', 'r-rcpp', 'r-rcpparmadillo', 'gcc']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-bsgenome >=1.36.3', 'bioconductor-genomicranges', 'bioconductor-variantannotation', 'r-base', 'r-class', 'r-nloptr', 'r-nmf', 'r-pmcmr', 'r-rcpp', 'r-rcpparmadillo', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'signeR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/signeR.html'), ('license', 'GPL-3'), ('summary', 'The signeR package provides an empirical Bayesian approach to mutational signature discovery. It is designed to analyze single nucleotide variaton (SNV) counts in cancer genomes, but can also be applied to other features as well. Functionalities to characterize signatures or genome samples according to exposure patterns are also provided.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/recon/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://eddylab.org/software/recon/'), ('license', 'GNU General Public License'), ('summary', 'The RECON package performs de novo identification and classification of repeat sequence families from genomic sequences.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'recon'), ('version', '1.08')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'RECON-1.08.tar.gz'), ('url', 'http://www.repeatmasker.org/RECON-1.08.tar.gz')])), ('test', CommentedMap([('commands', ['imagespread 2>&1 | grep usage'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/recon/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://eddylab.org/software/recon/'), ('license', 'GNU General Public License'), ('summary', 'The RECON package performs de novo identification and classification of repeat sequence families from genomic sequences.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'recon'), ('version', '1.08')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'RECON-1.08.tar.gz'), ('url', 'http://www.repeatmasker.org/RECON-1.08.tar.gz')])), ('test', CommentedMap([('commands', ['imagespread 2>&1 | grep usage'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rrdp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rrdp'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'rRDP_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rRDP_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/rRDP_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/rRDP/rRDP_1.10.0_src_all.tar.gz']), ('md5', '51ee5791ce729b0b57b62931b901736a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.26.2', 'r-base']), ('run', ['bioconductor-biostrings >=2.26.2', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rRDP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rRDP.html'), ('license', 'GPL-2 | file LICENSE'), ('summary', 'Seamlessly interfaces RDP classifier (version 2.9).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rrdp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rrdp'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'rRDP_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rRDP_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/rRDP_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/rRDP/rRDP_1.10.0_src_all.tar.gz']), ('md5', '51ee5791ce729b0b57b62931b901736a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings >=2.26.2', 'r-base']), ('run', ['bioconductor-biostrings >=2.26.2', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rRDP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rRDP.html'), ('license', 'GPL-2 | file LICENSE'), ('summary', 'Seamlessly interfaces RDP classifier (version 2.9).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocparallel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocparallel'), ('version', '1.6.6')])), ('source', CommentedMap([('fn', 'BiocParallel_1.6.6.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/BiocParallel_1.6.6.tar.gz', 'https://depot.galaxyproject.org/software/BiocParallel/BiocParallel_1.6.6_src_all.tar.gz']), ('md5', '445ec78a20e348d678d2e7b8dc5b4354')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-futile.logger', 'r-snow']), ('run', ['r-base', 'r-futile.logger', 'r-snow'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiocParallel\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BiocParallel.html'), ('license', 'GPL-2 | GPL-3'), ('summary', 'This package provides modified versions and novel implementation of functions for parallel evaluation, tailored to use with Bioconductor objects.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocparallel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocparallel'), ('version', '1.6.6')])), ('source', CommentedMap([('fn', 'BiocParallel_1.6.6.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/BiocParallel_1.6.6.tar.gz', 'https://depot.galaxyproject.org/software/BiocParallel/BiocParallel_1.6.6_src_all.tar.gz']), ('md5', '445ec78a20e348d678d2e7b8dc5b4354')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-futile.logger', 'r-snow']), ('run', ['r-base', 'r-futile.logger', 'r-snow'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiocParallel\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BiocParallel.html'), ('license', 'GPL-2 | GPL-3'), ('summary', 'This package provides modified versions and novel implementation of functions for parallel evaluation, tailored to use with Bioconductor objects.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqlogo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqlogo'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'seqLogo_1.38.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/seqLogo_1.38.0.tar.gz'), ('md5', '7fe87115d75b80a1b25def5fc254b260')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'seqLogo\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/seqLogo.html'), ('license', 'LGPL (>= 2)'), ('summary', 'seqLogo takes the position weight matrix of a DNA sequence motif and plots the corresponding sequence logo as introduced by Schneider and Stephens (1990).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqlogo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqlogo'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'seqLogo_1.38.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/seqLogo_1.38.0.tar.gz'), ('md5', '7fe87115d75b80a1b25def5fc254b260')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'seqLogo\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/seqLogo.html'), ('license', 'LGPL (>= 2)'), ('summary', 'seqLogo takes the position weight matrix of a DNA sequence motif and plots the corresponding sequence logo as introduced by Schneider and Stephens (1990).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomicranges/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomicranges'), ('version', '1.26.4')])), ('source', CommentedMap([('fn', 'GenomicRanges_1.26.4.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/GenomicRanges_1.26.4.tar.gz'), ('md5', '86c7d60ee21a86abc9f48b6527877bf4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.17.5', 'bioconductor-genomeinfodb >=1.1.20', 'bioconductor-iranges >=2.7.8', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.17.5', 'bioconductor-genomeinfodb >=1.1.20', 'bioconductor-iranges >=2.7.8', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenomicRanges\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/GenomicRanges.html'), ('license', 'Artistic-2.0'), ('summary', 'The ability to efficiently represent and manipulate genomic annotations and alignments is playing a central role when it comes to analyzing high-throughput sequencing data (a.k.a. NGS data). The GenomicRanges package defines general purpose containers for storing and manipulating genomic intervals and variables defined along a genome. More specialized containers for representing and manipulating short alignments against a reference genome, or a matrix-like summarization of an experiment, are defined in the GenomicAlignments and SummarizedExperiment packages respectively. Both packages build on top of the GenomicRanges infrastructure.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomicranges/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomicranges'), ('version', '1.26.4')])), ('source', CommentedMap([('fn', 'GenomicRanges_1.26.4.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/GenomicRanges_1.26.4.tar.gz'), ('md5', '86c7d60ee21a86abc9f48b6527877bf4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.17.5', 'bioconductor-genomeinfodb >=1.1.20', 'bioconductor-iranges >=2.7.8', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.17.5', 'bioconductor-genomeinfodb >=1.1.20', 'bioconductor-iranges >=2.7.8', 'bioconductor-s4vectors >=0.9.47', 'bioconductor-xvector', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenomicRanges\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/GenomicRanges.html'), ('license', 'Artistic-2.0'), ('summary', 'The ability to efficiently represent and manipulate genomic annotations and alignments is playing a central role when it comes to analyzing high-throughput sequencing data (a.k.a. NGS data). The GenomicRanges package defines general purpose containers for storing and manipulating genomic intervals and variables defined along a genome. More specialized containers for representing and manipulating short alignments against a reference genome, or a matrix-like summarization of an experiment, are defined in the GenomicAlignments and SummarizedExperiment packages respectively. Both packages build on top of the GenomicRanges infrastructure.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/freebayes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'freebayes'), ('version', '1.1.0.46')])), ('source', CommentedMap([('git_url', 'https://github.com/ekg/freebayes.git'), ('git_tag', '8d2b3a060da473e1f4f89be04edfce5cba63f1d3'), ('patches', ['qcfail.diff'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['cmake', 'zlib 1.2.8', 'bzip2 1.0.*', 'python', 'gcc', 'llvm']), ('run', ['zlib 1.2.8', 'bzip2 1.0.*', 'python', 'libgcc', 'htslib <{CONDA_HTSLIB}}*'])])), ('test', CommentedMap([('commands', ['freebayes --version', 'vcfstreamsort -h'])])), ('about', CommentedMap([('home', 'https://github.com/ekg/freebayes'), ('license', 'MIT'), ('summary', 'Bayesian haplotype-based polymorphism discovery and genotyping')])), ('extra', CommentedMap([('skip-lints', ['uses_git_url', 'missing_hash'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/freebayes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'freebayes'), ('version', '1.1.0.46')])), ('source', CommentedMap([('git_url', 'https://github.com/ekg/freebayes.git'), ('git_tag', '8d2b3a060da473e1f4f89be04edfce5cba63f1d3'), ('patches', ['qcfail.diff'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['cmake', 'zlib 1.2.8', 'bzip2 1.0.*', 'python', 'gcc', 'llvm']), ('run', ['zlib 1.2.8', 'bzip2 1.0.*', 'python', 'libgcc', 'htslib <{CONDA_HTSLIB}}*'])])), ('test', CommentedMap([('commands', ['freebayes --version', 'vcfstreamsort -h'])])), ('about', CommentedMap([('home', 'https://github.com/ekg/freebayes'), ('license', 'MIT'), ('summary', 'Bayesian haplotype-based polymorphism discovery and genotyping')])), ('extra', CommentedMap([('skip-lints', ['uses_git_url', 'missing_hash'])]))]))\n",
      "EXTRA has been found: freebayes\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/peakranger/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'peakranger'), ('version', 1.18)])), ('source', CommentedMap([('fn', 'PeakRanger-1.18-src.zip'), ('url', 'http://downloads.sourceforge.net/project/ranger/Source%20codes/PeakRanger-1.18-src.zip'), ('md5', '4b47352ee3200096f2ee8745a436b559'), ('patches', ['makefile.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['boost <{CONDA_BOOST}}*', 'zlib <{CONDA_ZLIB}}*']), ('run', ['boost <{CONDA_BOOST}}*', 'zlib <{CONDA_ZLIB}}*', 'r-base'])])), ('test', CommentedMap([('commands', ['peakranger nr --version'])])), ('about', CommentedMap([('home', 'http://ranger.sourceforge.net'), ('summary', 'PeakRanger is a multi-purporse software suite for analyzing next-generation sequencing (NGS) data.'), ('license', 'Artistic License 2.0'), ('license_file', 'COPYING')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/peakranger/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'peakranger'), ('version', 1.18)])), ('source', CommentedMap([('fn', 'PeakRanger-1.18-src.zip'), ('url', 'http://downloads.sourceforge.net/project/ranger/Source%20codes/PeakRanger-1.18-src.zip'), ('md5', '4b47352ee3200096f2ee8745a436b559'), ('patches', ['makefile.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['boost <{CONDA_BOOST}}*', 'zlib <{CONDA_ZLIB}}*']), ('run', ['boost <{CONDA_BOOST}}*', 'zlib <{CONDA_ZLIB}}*', 'r-base'])])), ('test', CommentedMap([('commands', ['peakranger nr --version'])])), ('about', CommentedMap([('home', 'http://ranger.sourceforge.net'), ('summary', 'PeakRanger is a multi-purporse software suite for analyzing next-generation sequencing (NGS) data.'), ('license', 'Artistic License 2.0'), ('license_file', 'COPYING')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cramtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cramtools'), ('version', '3.0.b127')])), ('source', CommentedMap([('fn', 'cramtools-3.0.jar'), ('url', 'https://github.com/enasequence/cramtools/raw/9a5977ff59bbc6319bd4b0fb06ccea414492d064/cramtools-3.0.jar'), ('md5', 'be01d4ad46549d48ec6cb77e48f868dd')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['cramtools -h'])])), ('about', CommentedMap([('home', 'http://www.ebi.ac.uk/ena/software/cram-toolkit'), ('license', 'Apache v2.0'), ('summary', 'A set of Java tools and APIs for efficient compression of sequence read data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cramtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cramtools'), ('version', '3.0.b127')])), ('source', CommentedMap([('fn', 'cramtools-3.0.jar'), ('url', 'https://github.com/enasequence/cramtools/raw/9a5977ff59bbc6319bd4b0fb06ccea414492d064/cramtools-3.0.jar'), ('md5', 'be01d4ad46549d48ec6cb77e48f868dd')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('requirements', CommentedMap([('run', ['openjdk'])])), ('test', CommentedMap([('commands', ['cramtools -h'])])), ('about', CommentedMap([('home', 'http://www.ebi.ac.uk/ena/software/cram-toolkit'), ('license', 'Apache v2.0'), ('summary', 'A set of Java tools and APIs for efficient compression of sequence read data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snpeff/4.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://snpeff.sourceforge.net/'), ('license', 'LGPLv3'), ('summary', 'Genetic variant annotation and effect prediction toolbox')])), ('package', CommentedMap([('name', 'snpsift'), ('version', '4.2')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'snpEff_v4_2_core.zip'), ('md5', '25ae5b062d57072de6cfb8677ca3625a'), ('url', 'http://downloads.sourceforge.net/project/snpeff/snpEff_v4_2_core.zip')])), ('requirements', CommentedMap([('run', ['java-jdk >=8'])])), ('test', CommentedMap([('commands', ['snpEff -version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snpeff/4.2/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://snpeff.sourceforge.net/'), ('license', 'LGPLv3'), ('summary', 'Genetic variant annotation and effect prediction toolbox')])), ('package', CommentedMap([('name', 'snpsift'), ('version', '4.2')])), ('build', CommentedMap([('number', 1), ('skip', False)])), ('source', CommentedMap([('fn', 'snpEff_v4_2_core.zip'), ('md5', '25ae5b062d57072de6cfb8677ca3625a'), ('url', 'http://downloads.sourceforge.net/project/snpeff/snpEff_v4_2_core.zip')])), ('requirements', CommentedMap([('run', ['java-jdk >=8'])])), ('test', CommentedMap([('commands', ['snpEff -version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mipp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mipp'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'MiPP_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MiPP_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/MiPP_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/MiPP/MiPP_1.48.0_src_all.tar.gz']), ('md5', '5d9d5dfbe0a0f6ccba21673466bb9f41')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-mass']), ('run', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MiPP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MiPP.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package finds optimal sets of genes that seperate samples into two or more classes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mipp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mipp'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'MiPP_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MiPP_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/MiPP_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/MiPP/MiPP_1.48.0_src_all.tar.gz']), ('md5', '5d9d5dfbe0a0f6ccba21673466bb9f41')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-mass']), ('run', ['bioconductor-biobase', 'r-base', 'r-e1071', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MiPP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MiPP.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package finds optimal sets of genes that seperate samples into two or more classes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bicare/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bicare'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'BicARE_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BicARE_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/BicARE_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/BicARE/BicARE_1.34.0_src_all.tar.gz']), ('md5', '4cad470020ab4fe967673def474010b8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-gseabase', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-gseabase', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BicARE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BicARE.html'), ('license', 'GPL-2'), ('summary', 'Biclustering Analysis and Results Exploration')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bicare/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bicare'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'BicARE_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BicARE_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/BicARE_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/BicARE/BicARE_1.34.0_src_all.tar.gz']), ('md5', '4cad470020ab4fe967673def474010b8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.5.5', 'bioconductor-gseabase', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase >=2.5.5', 'bioconductor-gseabase', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BicARE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BicARE.html'), ('license', 'GPL-2'), ('summary', 'Biclustering Analysis and Results Exploration')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/soapsplice/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'soapsplice'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'SOAPsplice-v<{ version }}.tar.gz'), ('url', 'http://soap.genomics.org.cn/down/SOAPsplice-v<{ version }}.tar.gz'), ('md5', 'df038fd6657885b77932111652e9203d')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['zlib']), ('run', ['zlib'])])), ('test', CommentedMap([('commands', ['{ soapsplice || true; } | grep SOAPsplice'])])), ('about', CommentedMap([('home', 'http://soap.genomics.org.cn/soapsplice.html'), ('summary', 'We have developed a tool SOAPsplice for genome-wide ab initio detection of splice junction sites from RNA-Seq, a method using new generation sequencing technologies to sequence the messenger RNA.'), ('license', 'freely available')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/soapsplice/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'soapsplice'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'SOAPsplice-v<{ version }}.tar.gz'), ('url', 'http://soap.genomics.org.cn/down/SOAPsplice-v<{ version }}.tar.gz'), ('md5', 'df038fd6657885b77932111652e9203d')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['zlib']), ('run', ['zlib'])])), ('test', CommentedMap([('commands', ['{ soapsplice || true; } | grep SOAPsplice'])])), ('about', CommentedMap([('home', 'http://soap.genomics.org.cn/soapsplice.html'), ('summary', 'We have developed a tool SOAPsplice for genome-wide ab initio detection of splice junction sites from RNA-Seq, a method using new generation sequencing technologies to sequence the messenger RNA.'), ('license', 'freely available')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgsea/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgsea'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'RGSEA_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RGSEA_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/RGSEA_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/RGSEA/RGSEA_1.10.0_src_all.tar.gz']), ('md5', '95fe347168bedf66f6005b4b39262d9f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'r-base']), ('run', ['bioconductor-biocgenerics', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RGSEA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RGSEA.html'), ('license', 'GPL(>=3)'), ('summary', 'Combining bootstrap aggregating and Gene set enrichment analysis (GSEA), RGSEA is a classfication algorithm with high robustness and no over-fitting problem. It performs well especially for the data generated from different exprements.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgsea/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgsea'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'RGSEA_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RGSEA_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/RGSEA_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/RGSEA/RGSEA_1.10.0_src_all.tar.gz']), ('md5', '95fe347168bedf66f6005b4b39262d9f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'r-base']), ('run', ['bioconductor-biocgenerics', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RGSEA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RGSEA.html'), ('license', 'GPL(>=3)'), ('summary', 'Combining bootstrap aggregating and Gene set enrichment analysis (GSEA), RGSEA is a classfication algorithm with high robustness and no over-fitting problem. It performs well especially for the data generated from different exprements.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-regioner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-regioner'), ('version', '1.6.2')])), ('source', CommentedMap([('fn', 'regioneR_1.6.2.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-regioner/bioconductor-regioner_1.6.2_src_all.tar.gz'), ('md5', '93ecdf4d04f690580c0f1681bddc1246')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'r-base', 'r-memoise']), ('run', ['bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'r-base', 'r-memoise'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'regioneR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/regioneR.html'), ('license', 'Artistic-2.0'), ('summary', 'regioneR offers a statistical framework based on customizable permutation tests to assess the association between genomic region sets and other genomic features.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-regioner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-regioner'), ('version', '1.6.2')])), ('source', CommentedMap([('fn', 'regioneR_1.6.2.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-regioner/bioconductor-regioner_1.6.2_src_all.tar.gz'), ('md5', '93ecdf4d04f690580c0f1681bddc1246')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'r-base', 'r-memoise']), ('run', ['bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'r-base', 'r-memoise'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'regioneR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/regioneR.html'), ('license', 'Artistic-2.0'), ('summary', 'regioneR offers a statistical framework based on customizable permutation tests to assess the association between genomic region sets and other genomic features.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kobas/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kobas'), ('version', '2.1.1')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('source', CommentedMap([('url', 'http://kobas.cbi.pku.edu.cn/kobas-2.1.1/kobas-2.1.1.tar.gz'), ('md5', '38cd6c000de06de24f71e6d29599928f'), ('patches', ['package.patch'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools', 'bioconductor-qvalue', 'blast', 'numpy', 'pandas', 'biopython', 'sqlite', 'r-base', 'rpy2 >=2.8.5'])])), ('test', CommentedMap([('imports', ['kobas']), ('commands', ['kobas-annotate -h', 'kobas-identify -h'])])), ('about', CommentedMap([('home', 'http://http://kobas.cbi.pku.edu.cn/index.php')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kobas/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kobas'), ('version', '2.1.1')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('source', CommentedMap([('url', 'http://kobas.cbi.pku.edu.cn/kobas-2.1.1/kobas-2.1.1.tar.gz'), ('md5', '38cd6c000de06de24f71e6d29599928f'), ('patches', ['package.patch'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'setuptools', 'bioconductor-qvalue', 'blast', 'numpy', 'pandas', 'biopython', 'sqlite', 'r-base', 'rpy2 >=2.8.5'])])), ('test', CommentedMap([('imports', ['kobas']), ('commands', ['kobas-annotate -h', 'kobas-identify -h'])])), ('about', CommentedMap([('home', 'http://http://kobas.cbi.pku.edu.cn/index.php')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-nastiseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-nastiseq'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'NASTIseq_1.0.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/duke/NASTIseq/NASTIseq_1.0.tar.gz'), ('md5', 'e5dd75117e910dec6f6c22b48f570730')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rocr', 'bioconductor-deseq']), ('run', ['r-base', 'r-rocr', 'bioconductor-deseq'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NASTIseq\\')\"'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/NASTIseq_104/'), ('license', 'GPL-2.0'), ('summary', 'Pairs of RNA molecules transcribed from partially or entirely complementary loci are called cis-natural antisense transcripts (cis-NATs), and they play key roles in the regulation of gene expression in many organisms. A promising experimental tool for profiling sense and antisense transcription is strand-specific RNA sequencing (ssRNA-seq). To identify cis-NATs using ssRNA-seq, we developed a new computational method based on model comparison that incorporates the inherent variable efficiency of generating perfectly strand-specific libraries. Applying the method to new ssRNA-seq data from whole root and cell-type specific Arabidopsis libraries confirmed most of the known cis-NAT pairs and identified hundreds of additional cis-NAT pairs.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-nastiseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-nastiseq'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'NASTIseq_1.0.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/duke/NASTIseq/NASTIseq_1.0.tar.gz'), ('md5', 'e5dd75117e910dec6f6c22b48f570730')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rocr', 'bioconductor-deseq']), ('run', ['r-base', 'r-rocr', 'bioconductor-deseq'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NASTIseq\\')\"'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/NASTIseq_104/'), ('license', 'GPL-2.0'), ('summary', 'Pairs of RNA molecules transcribed from partially or entirely complementary loci are called cis-natural antisense transcripts (cis-NATs), and they play key roles in the regulation of gene expression in many organisms. A promising experimental tool for profiling sense and antisense transcription is strand-specific RNA sequencing (ssRNA-seq). To identify cis-NATs using ssRNA-seq, we developed a new computational method based on model comparison that incorporates the inherent variable efficiency of generating perfectly strand-specific libraries. Applying the method to new ssRNA-seq data from whole root and cell-type specific Arabidopsis libraries confirmed most of the known cis-NAT pairs and identified hundreds of additional cis-NAT pairs.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/oases/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'oases'), ('version', '0.2.09')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('git_url', 'https://github.com/dzerbino/oases.git'), ('git_rev', '0.2.09')])), ('requirements', CommentedMap([('build', ['gcc', 'openmpi']), ('run', ['libgcc', 'velvet'])])), ('test', CommentedMap([('commands', ['oases | grep \"oases - De novo transcriptome assembler for the Velvet package\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.ebi.ac.uk/~zerbino/oases/'), ('license', 'GPL'), ('summary', 'De novo transcriptome assembler for short reads')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/oases/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'oases'), ('version', '0.2.09')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('git_url', 'https://github.com/dzerbino/oases.git'), ('git_rev', '0.2.09')])), ('requirements', CommentedMap([('build', ['gcc', 'openmpi']), ('run', ['libgcc', 'velvet'])])), ('test', CommentedMap([('commands', ['oases | grep \"oases - De novo transcriptome assembler for the Velvet package\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.ebi.ac.uk/~zerbino/oases/'), ('license', 'GPL'), ('summary', 'De novo transcriptome assembler for short reads')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-profia/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-profia'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'proFIA_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/proFIA_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/proFIA/proFIA_1.2.0_src_all.tar.gz']), ('md5', '1298970f476ef1086577e96a66062c66')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-ropls', 'bioconductor-xcms', 'r-base', 'r-fnn', 'r-maxlik', 'r-minpack.lm', 'r-pracma']), ('run', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-ropls', 'bioconductor-xcms', 'r-base', 'r-fnn', 'r-maxlik', 'r-minpack.lm', 'r-pracma'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'proFIA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/proFIA.html'), ('license', 'CeCILL'), ('summary', 'Flow Injection Analysis coupled to High-Resolution Mass Spectrometry is a promising approach for high-throughput metabolomics. FIA- HRMS data, however, cannot be pre-processed with current software tools which rely on liquid chromatography separation, or handle low resolution data only. Here we present the proFIA package, which implements a new methodology to pre-process FIA-HRMS raw data (netCDF, mzData, mzXML, and mzML) including noise modelling and injection peak reconstruction, and generate the peak table. The workflow includes noise modelling, band detection and filtering then signal matching and missing value imputation. The peak table can then be exported as a .tsv file for further analysis. Visualisations to assess the quality of the data and of the signal made are easely produced.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-profia/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-profia'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'proFIA_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/proFIA_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/proFIA/proFIA_1.2.0_src_all.tar.gz']), ('md5', '1298970f476ef1086577e96a66062c66')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-ropls', 'bioconductor-xcms', 'r-base', 'r-fnn', 'r-maxlik', 'r-minpack.lm', 'r-pracma']), ('run', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-ropls', 'bioconductor-xcms', 'r-base', 'r-fnn', 'r-maxlik', 'r-minpack.lm', 'r-pracma'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'proFIA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/proFIA.html'), ('license', 'CeCILL'), ('summary', 'Flow Injection Analysis coupled to High-Resolution Mass Spectrometry is a promising approach for high-throughput metabolomics. FIA- HRMS data, however, cannot be pre-processed with current software tools which rely on liquid chromatography separation, or handle low resolution data only. Here we present the proFIA package, which implements a new methodology to pre-process FIA-HRMS raw data (netCDF, mzData, mzXML, and mzML) including noise modelling and injection peak reconstruction, and generate the peak table. The workflow includes noise modelling, band detection and filtering then signal matching and missing value imputation. The peak table can then be exported as a .tsv file for further analysis. Visualisations to assess the quality of the data and of the signal made are easely produced.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-deepsnv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-deepsnv'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'deepSNV_1.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/deepSNV_1.20.0.tar.gz'), ('md5', 'a545ff264d5c188ddb770cfe00e9de7a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rhtslib', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.13.44', 'r-base', 'r-vgam']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rhtslib', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.13.44', 'r-base', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'deepSNV\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/deepSNV.html'), ('license', 'GPL-3'), ('summary', 'This package provides provides quantitative variant callers for detecting subclonal mutations in ultra-deep (>=100x coverage) sequencing experiments. The deepSNV algorithm is used for a comparative setup with a control experiment of the same loci and uses a beta-binomial model and a likelihood ratio test to discriminate sequencing errors and subclonal SNVs. The shearwater algorithm computes a Bayes classifier based on a beta-binomial model for variant calling with multiple samples for precisely estimating model parameters such as local error rates and dispersion and prior knowledge, e.g. from variation data bases such as COSMIC.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-deepsnv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-deepsnv'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'deepSNV_1.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/deepSNV_1.20.0.tar.gz'), ('md5', 'a545ff264d5c188ddb770cfe00e9de7a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rhtslib', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.13.44', 'r-base', 'r-vgam']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rhtslib', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.13.44', 'r-base', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'deepSNV\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/deepSNV.html'), ('license', 'GPL-3'), ('summary', 'This package provides provides quantitative variant callers for detecting subclonal mutations in ultra-deep (>=100x coverage) sequencing experiments. The deepSNV algorithm is used for a comparative setup with a control experiment of the same loci and uses a beta-binomial model and a likelihood ratio test to discriminate sequencing errors and subclonal SNVs. The shearwater algorithm computes a Bayes classifier based on a beta-binomial model for variant calling with multiple samples for precisely estimating model parameters such as local error rates and dispersion and prior knowledge, e.g. from variation data bases such as COSMIC.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gromacs/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.gromacs.org/'), ('license', 'GNU Lesser General Public License (LGPL)'), ('summary', 'GROMACS is a versatile package to perform molecular dynamics.')])), ('package', CommentedMap([('name', 'gromacs'), ('version', '4.6.5')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'gromacs-4.6.5.tar.gz'), ('url', 'ftp://ftp.gromacs.org/pub/gromacs/gromacs-4.6.5.tar.gz'), ('md5', '184a537ea52429dbc2a681f98b46ed84')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['mdrun -version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gromacs/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.gromacs.org/'), ('license', 'GNU Lesser General Public License (LGPL)'), ('summary', 'GROMACS is a versatile package to perform molecular dynamics.')])), ('package', CommentedMap([('name', 'gromacs'), ('version', '4.6.5')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'gromacs-4.6.5.tar.gz'), ('url', 'ftp://ftp.gromacs.org/pub/gromacs/gromacs-4.6.5.tar.gz'), ('md5', '184a537ea52429dbc2a681f98b46ed84')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['mdrun -version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genomedata/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genomedata'), ('version', '1.3.6')])), ('source', CommentedMap([('fn', 'genomedata-1.3.6.tar.gz'), ('url', 'https://pypi.python.org/packages/cd/3a/a466d62a79ccc0926f32c9998f712c14140a0593fc9a487504b202dd18bd/genomedata-1.3.6.tar.gz'), ('md5', '55bcd4d4ca8e31617abad7f11ddb70b6')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('entry_points', ['genomedata-info=genomedata._info:main', 'genomedata-query=genomedata._query:main', 'genomedata-histogram=genomedata._histogram:main', 'genomedata-load=genomedata.load_genomedata:main', 'genomedata-load-seq=genomedata._load_seq:main', 'genomedata-load-assembly=genomedata._load_seq:main', 'genomedata-open-data=genomedata._open_data:main', 'genomedata-close-data=genomedata._close_data:main', 'genomedata-report=genomedata._report:main', 'genomedata-erase-data=genomedata._erase_data:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'forked-path', 'pytables >=3.0', 'textinput', 'gcc', 'hdf5', 'hgtools']), ('run', ['python', 'numpy', 'forked-path', 'pytables >=3.0', 'textinput', 'libgcc', 'hdf5', 'hgtools'])])), ('test', CommentedMap([('imports', ['genomedata']), ('commands', ['genomedata-info --help', 'genomedata-query --help', 'genomedata-histogram --help', 'genomedata-load --help', 'genomedata-load-seq --help', 'genomedata-load-assembly --help', 'genomedata-open-data --help', 'genomedata-close-data --help', 'genomedata-report --help', 'genomedata-erase-data --help'])])), ('about', CommentedMap([('home', 'http://genomedata.hoffmanlab.org/'), ('license', 'GPL2'), ('summary', 'tools for accessing large amounts of genomic data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genomedata/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genomedata'), ('version', '1.3.6')])), ('source', CommentedMap([('fn', 'genomedata-1.3.6.tar.gz'), ('url', 'https://pypi.python.org/packages/cd/3a/a466d62a79ccc0926f32c9998f712c14140a0593fc9a487504b202dd18bd/genomedata-1.3.6.tar.gz'), ('md5', '55bcd4d4ca8e31617abad7f11ddb70b6')])), ('build', CommentedMap([('preserve_egg_dir', True), ('skip', True), ('entry_points', ['genomedata-info=genomedata._info:main', 'genomedata-query=genomedata._query:main', 'genomedata-histogram=genomedata._histogram:main', 'genomedata-load=genomedata.load_genomedata:main', 'genomedata-load-seq=genomedata._load_seq:main', 'genomedata-load-assembly=genomedata._load_seq:main', 'genomedata-open-data=genomedata._open_data:main', 'genomedata-close-data=genomedata._close_data:main', 'genomedata-report=genomedata._report:main', 'genomedata-erase-data=genomedata._erase_data:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy', 'forked-path', 'pytables >=3.0', 'textinput', 'gcc', 'hdf5', 'hgtools']), ('run', ['python', 'numpy', 'forked-path', 'pytables >=3.0', 'textinput', 'libgcc', 'hdf5', 'hgtools'])])), ('test', CommentedMap([('imports', ['genomedata']), ('commands', ['genomedata-info --help', 'genomedata-query --help', 'genomedata-histogram --help', 'genomedata-load --help', 'genomedata-load-seq --help', 'genomedata-load-assembly --help', 'genomedata-open-data --help', 'genomedata-close-data --help', 'genomedata-report --help', 'genomedata-erase-data --help'])])), ('about', CommentedMap([('home', 'http://genomedata.hoffmanlab.org/'), ('license', 'GPL2'), ('summary', 'tools for accessing large amounts of genomic data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hicup/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hicup'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['perl', 'bowtie2', 'bowtie', 'r-base', 'samtools'])])), ('source', CommentedMap([('fn', 'hicup_v<{ version }}.tar.gz'), ('url', 'http://www.bioinformatics.babraham.ac.uk/projects/hicup/hicup_v<{ version }}.tar.gz'), ('md5', '58702fb9c7f2f4887f4113bf284d1b98'), ('patches', ['get_captured_reads.patch'])])), ('test', CommentedMap([('commands', ['hicup_mapper', 'hicup_digester', 'hicup_truncater', 'hicup_deduplicator', 'hicup_truncater', 'hicup', 'hicup2hicpipe', 'hicup2gothic', 'hicup2fithic', 'hicup2homer', 'get_captured_reads || [[ $? == 255 ]]'])])), ('about', CommentedMap([('home', 'http://www.bioinformatics.babraham.ac.uk/projects/hicup/'), ('license', 'GPLv3'), ('summary', 'A tool for mapping and performing quality control on Hi-C data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/hicup/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'hicup'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('run', ['perl', 'bowtie2', 'bowtie', 'r-base', 'samtools'])])), ('source', CommentedMap([('fn', 'hicup_v<{ version }}.tar.gz'), ('url', 'http://www.bioinformatics.babraham.ac.uk/projects/hicup/hicup_v<{ version }}.tar.gz'), ('md5', '58702fb9c7f2f4887f4113bf284d1b98'), ('patches', ['get_captured_reads.patch'])])), ('test', CommentedMap([('commands', ['hicup_mapper', 'hicup_digester', 'hicup_truncater', 'hicup_deduplicator', 'hicup_truncater', 'hicup', 'hicup2hicpipe', 'hicup2gothic', 'hicup2fithic', 'hicup2homer', 'get_captured_reads || [[ $? == 255 ]]'])])), ('about', CommentedMap([('home', 'http://www.bioinformatics.babraham.ac.uk/projects/hicup/'), ('license', 'GPLv3'), ('summary', 'A tool for mapping and performing quality control on Hi-C data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/carna/1.3.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'carna'), ('version', '1.3.2')])), ('about', CommentedMap([('home', 'https://www.bioinf.uni-leipzig.de/~will/Software/CARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Constraint-based Alignment of RNA Ensembles')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'carna-1.3.2.tar.gz'), ('url', 'http://www.bioinf.uni-leipzig.de/~will/Software/CARNA/Releases/carna-1.3.2.tar.gz'), ('sha256', '279ce4417c8cd637511d4d35efb83cb63194cee7608d89a75542f14d4c5ee48c')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'gecode 4.4.0', 'viennarna 2.2.8', 'locarna 1.8.11']), ('run', ['libgcc', 'gecode 4.4.0', 'locarna 1.8.11'])])), ('test', CommentedMap([('commands', ['carna --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/carna/1.3.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'carna'), ('version', '1.3.2')])), ('about', CommentedMap([('home', 'https://www.bioinf.uni-leipzig.de/~will/Software/CARNA/'), ('license', 'GPL'), ('license_file', 'COPYING'), ('summary', 'Constraint-based Alignment of RNA Ensembles')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', 'carna-1.3.2.tar.gz'), ('url', 'http://www.bioinf.uni-leipzig.de/~will/Software/CARNA/Releases/carna-1.3.2.tar.gz'), ('sha256', '279ce4417c8cd637511d4d35efb83cb63194cee7608d89a75542f14d4c5ee48c')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'gecode 4.4.0', 'viennarna 2.2.8', 'locarna 1.8.11']), ('run', ['libgcc', 'gecode 4.4.0', 'locarna 1.8.11'])])), ('test', CommentedMap([('commands', ['carna --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastme/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastme'), ('version', '2.1.5')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'fastme-2.1.5.tar.gz'), ('url', 'http://www.atgc-montpellier.fr/download/sources/fastme/fastme-2.1.5.tar.gz'), ('sha256', 'af636864902f1c038203ad370da6ed2b97edaaa97bfde060ec7aab7b05672277')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['fastme --help > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.atgc-montpellier.fr/fastme/binaries.php'), ('summary', 'a comprehensive, accurate and fast distance-based phylogeny inference program.'), ('license', 'GPLv3')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastme/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'fastme'), ('version', '2.1.5')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'fastme-2.1.5.tar.gz'), ('url', 'http://www.atgc-montpellier.fr/download/sources/fastme/fastme-2.1.5.tar.gz'), ('sha256', 'af636864902f1c038203ad370da6ed2b97edaaa97bfde060ec7aab7b05672277')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['fastme --help > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.atgc-montpellier.fr/fastme/binaries.php'), ('summary', 'a comprehensive, accurate and fast distance-based phylogeny inference program.'), ('license', 'GPLv3')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowdensity/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowdensity'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'flowDensity_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/flowDensity_1.10.0.tar.gz'), ('md5', 'f1eb297bc6454a4bc4732459a5200a87')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore', 'r-base', 'r-car', 'r-geomap', 'r-gplots', 'r-rfoc', 'r-nloptr']), ('run', ['bioconductor-flowcore', 'r-base', 'r-car', 'r-geomap', 'r-gplots', 'r-rfoc', 'r-nloptr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowDensity\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowDensity.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides tools for automated sequential gating analogous to the manual gating strategy based on the density of the data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-flowdensity/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-flowdensity'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'flowDensity_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/flowDensity_1.10.0.tar.gz'), ('md5', 'f1eb297bc6454a4bc4732459a5200a87')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore', 'r-base', 'r-car', 'r-geomap', 'r-gplots', 'r-rfoc', 'r-nloptr']), ('run', ['bioconductor-flowcore', 'r-base', 'r-car', 'r-geomap', 'r-gplots', 'r-rfoc', 'r-nloptr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'flowDensity\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/flowDensity.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides tools for automated sequential gating analogous to the manual gating strategy based on the density of the data.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/diamond/0.9.10/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/diamond/0.9.10/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/bbuchfink/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['diamond --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://ab.inf.uni-tuebingen.de/software/diamond/'), ('license', 'BSD'), ('summary', 'Accelerated BLAST compatible local sequence aligner')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-dbchip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-dbchip'), ('version', '1.1.6')])), ('source', CommentedMap([('fn', 'DBChIP_1.1.6.tar.gz'), ('url', 'http://pages.cs.wisc.edu/~kliang/DBChIP/DBChIP_1.1.6.tar.gz'), ('md5', 'f4b22bb2051ad6b2d33d4687754e8cee')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-deseq', 'bioconductor-edger']), ('run', ['r-base', 'bioconductor-deseq', 'bioconductor-edger'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DBChIP\\')\"'])])), ('about', CommentedMap([('home', 'http://pages.cs.wisc.edu/~kliang/DBChIP'), ('license', 'GPL (>= 2)'), ('summary', 'ChIP-seq differential binding')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-dbchip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-dbchip'), ('version', '1.1.6')])), ('source', CommentedMap([('fn', 'DBChIP_1.1.6.tar.gz'), ('url', 'http://pages.cs.wisc.edu/~kliang/DBChIP/DBChIP_1.1.6.tar.gz'), ('md5', 'f4b22bb2051ad6b2d33d4687754e8cee')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-deseq', 'bioconductor-edger']), ('run', ['r-base', 'bioconductor-deseq', 'bioconductor-edger'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DBChIP\\')\"'])])), ('about', CommentedMap([('home', 'http://pages.cs.wisc.edu/~kliang/DBChIP'), ('license', 'GPL (>= 2)'), ('summary', 'ChIP-seq differential binding')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-absseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-absseq'), ('version', '1.22.8')])), ('source', CommentedMap([('fn', 'ABSSeq_1.22.8.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ABSSeq_1.22.8.tar.gz', 'https://depot.galaxyproject.org/software/ABSSeq/ABSSeq_1.22.8_src_all.tar.gz']), ('md5', 'bfdb1800f2e7c60dfa7f6b43026ec8f9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base', 'r-locfit']), ('run', ['bioconductor-limma', 'r-base', 'r-locfit'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ABSSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ABSSeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'Inferring differential expression genes by absolute counts difference between two groups, utilizing Negative binomial distribution and moderating fold-change according to heterogeneity of dispersion across expression level.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-absseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-absseq'), ('version', '1.22.8')])), ('source', CommentedMap([('fn', 'ABSSeq_1.22.8.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ABSSeq_1.22.8.tar.gz', 'https://depot.galaxyproject.org/software/ABSSeq/ABSSeq_1.22.8_src_all.tar.gz']), ('md5', 'bfdb1800f2e7c60dfa7f6b43026ec8f9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base', 'r-locfit']), ('run', ['bioconductor-limma', 'r-base', 'r-locfit'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ABSSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ABSSeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'Inferring differential expression genes by absolute counts difference between two groups, utilizing Negative binomial distribution and moderating fold-change according to heterogeneity of dispersion across expression level.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bwa/0.7.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bwa'), ('version', '0.7.8')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'bwa-0.7.8.tar.bz2'), ('url', 'http://downloads.sourceforge.net/project/bio-bwa/bwa-0.7.8.tar.bz2'), ('md5', '51ca7a29907022080398b2b88207e33c')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('about', CommentedMap([('home', 'http://bio-bwa.sourceforge.net'), ('license', 'MIT'), ('summary', 'The BWA read mapper.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bwa/0.7.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bwa'), ('version', '0.7.8')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'bwa-0.7.8.tar.bz2'), ('url', 'http://downloads.sourceforge.net/project/bio-bwa/bwa-0.7.8.tar.bz2'), ('md5', '51ca7a29907022080398b2b88207e33c')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('about', CommentedMap([('home', 'http://bio-bwa.sourceforge.net'), ('license', 'MIT'), ('summary', 'The BWA read mapper.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqgsea/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqgsea'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'SeqGSEA_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SeqGSEA_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/SeqGSEA_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/SeqGSEA/SeqGSEA_1.16.0_src_all.tar.gz']), ('md5', 'd2f4041cdee40dbd41dbe49d2111f0e9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-deseq', 'r-base', 'r-doparallel']), ('run', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-deseq', 'r-base', 'r-doparallel'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SeqGSEA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SeqGSEA.html'), ('license', 'GPL (>= 3)'), ('summary', \"The package generally provides methods for gene set enrichment analysis of high-throughput RNA-Seq data by integrating differential expression and splicing. It uses negative binomial distribution to model read count data, which accounts for sequencing biases and biological variation. Based on permutation tests, statistical significance can also be achieved regarding each gene's differential expression and splicing, respectively.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-seqgsea/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-seqgsea'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'SeqGSEA_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SeqGSEA_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/SeqGSEA_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/SeqGSEA/SeqGSEA_1.16.0_src_all.tar.gz']), ('md5', 'd2f4041cdee40dbd41dbe49d2111f0e9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-deseq', 'r-base', 'r-doparallel']), ('run', ['bioconductor-biobase', 'bioconductor-biomart', 'bioconductor-deseq', 'r-base', 'r-doparallel'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SeqGSEA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SeqGSEA.html'), ('license', 'GPL (>= 3)'), ('summary', \"The package generally provides methods for gene set enrichment analysis of high-throughput RNA-Seq data by integrating differential expression and splicing. It uses negative binomial distribution to model read count data, which accounts for sequencing biases and biological variation. Based on permutation tests, statistical significance can also be achieved regarding each gene's differential expression and splicing, respectively.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-r3cseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-r3cseq'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'r3Cseq_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/r3Cseq_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/r3Cseq_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/r3Cseq/r3Cseq_1.22.0_src_all.tar.gz']), ('md5', '46b95951434f664799a4cf783ace3bd8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-data.table', 'r-rcolorbrewer', 'r-sqldf', 'r-vgam']), ('run', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-data.table', 'r-rcolorbrewer', 'r-sqldf', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'r3Cseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/r3Cseq.html'), ('license', 'GPL-3'), ('summary', 'This package is an implementation of data analysis for the long-range interactions from 3C-seq assay.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-r3cseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-r3cseq'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'r3Cseq_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/r3Cseq_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/r3Cseq_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/r3Cseq/r3Cseq_1.22.0_src_all.tar.gz']), ('md5', '46b95951434f664799a4cf783ace3bd8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-data.table', 'r-rcolorbrewer', 'r-sqldf', 'r-vgam']), ('run', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-qvalue', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'r-base', 'r-data.table', 'r-rcolorbrewer', 'r-sqldf', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'r3Cseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/r3Cseq.html'), ('license', 'GPL-3'), ('summary', 'This package is an implementation of data analysis for the long-range interactions from 3C-seq assay.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gridss/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gridss'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'gridss-<{ version }}.jar'), ('url', 'https://github.com/PapenfussLab/gridss/releases/download/v<{ version }}/gridss-<{version}}-jar-with-dependencies.jar'), ('md5', '6ccca8c848540e40da45407550e5db5a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('run', ['openjdk >=8', 'python'])])), ('test', CommentedMap([('commands', [\"gridss gridss.CallVariants --help 2>&1 | grep 'USAGE: CallVariants'\"])])), ('about', CommentedMap([('home', 'https://github.com/PapenfussLab/gridss'), ('license', 'GPLv3'), ('summary', 'GRIDSS: a Genomic Rearrangement IDentification Software Suite')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gridss/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gridss'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'gridss-<{ version }}.jar'), ('url', 'https://github.com/PapenfussLab/gridss/releases/download/v<{ version }}/gridss-<{version}}-jar-with-dependencies.jar'), ('md5', '6ccca8c848540e40da45407550e5db5a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('run', ['openjdk >=8', 'python'])])), ('test', CommentedMap([('commands', [\"gridss gridss.CallVariants --help 2>&1 | grep 'USAGE: CallVariants'\"])])), ('about', CommentedMap([('home', 'https://github.com/PapenfussLab/gridss'), ('license', 'GPLv3'), ('summary', 'GRIDSS: a Genomic Rearrangement IDentification Software Suite')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/erne/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://downloads.sourceforge.net/project/<{ name }}/<{ version }}/<{ name }}-<{ version }}-source.tar.gz'), ('md5', '3c26b80654e1ef6ef61085f0ecf75e65'), ('patches', ['osxfixes.patch'])])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'llvmdev', 'libcxx', 'boost ==1.63.0', 'boost <{CONDA_BOOST}}*', 'zlib', 'bzip2']), ('run', ['libgcc', 'libcxx', 'boost <{CONDA_BOOST}}*', 'boost ==1.63.0', 'zlib', 'bzip2'])])), ('test', CommentedMap([('commands', ['erne-map --version | grep ERNE', 'erne-bs5 --version | grep ERNE', 'erne-create --version | grep ERNE', 'erne-filter --version | grep ERNE', 'erne-meth --version | grep ERNE'])])), ('about', CommentedMap([('home', 'http://erne.sourceforge.net'), ('license', 'GPLv3'), ('summary', 'ERNE - Extended Randomized Numerical alignEr')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/erne/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://downloads.sourceforge.net/project/<{ name }}/<{ version }}/<{ name }}-<{ version }}-source.tar.gz'), ('md5', '3c26b80654e1ef6ef61085f0ecf75e65'), ('patches', ['osxfixes.patch'])])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'llvmdev', 'libcxx', 'boost ==1.63.0', 'boost <{CONDA_BOOST}}*', 'zlib', 'bzip2']), ('run', ['libgcc', 'libcxx', 'boost <{CONDA_BOOST}}*', 'boost ==1.63.0', 'zlib', 'bzip2'])])), ('test', CommentedMap([('commands', ['erne-map --version | grep ERNE', 'erne-bs5 --version | grep ERNE', 'erne-create --version | grep ERNE', 'erne-filter --version | grep ERNE', 'erne-meth --version | grep ERNE'])])), ('about', CommentedMap([('home', 'http://erne.sourceforge.net'), ('license', 'GPLv3'), ('summary', 'ERNE - Extended Randomized Numerical alignEr')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pymzml/0.7.7/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pymzml'), ('version', '0.7.7')])), ('source', CommentedMap([('fn', 'pymzML-0.7.7.zip'), ('url', 'https://pypi.python.org/packages/da/35/cbcc508d55f68927c1fb6574b45f7103fa0817d269c770e4bd7a6c52509c/pymzML-0.7.7.zip'), ('md5', '3fb7e7799983e7616257a5e96cfb5911')])), ('build', CommentedMap([('number', 0), ('script', '$PYTHON setup.py install')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pymzml'])])), ('about', CommentedMap([('home', 'http://pymzml.github.com'), ('license', 'GNU General Public License (GPL)'), ('summary', 'high-throughput mzML parsing'), ('license_family', 'LGPL')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pymzml/0.7.7/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pymzml'), ('version', '0.7.7')])), ('source', CommentedMap([('fn', 'pymzML-0.7.7.zip'), ('url', 'https://pypi.python.org/packages/da/35/cbcc508d55f68927c1fb6574b45f7103fa0817d269c770e4bd7a6c52509c/pymzML-0.7.7.zip'), ('md5', '3fb7e7799983e7616257a5e96cfb5911')])), ('build', CommentedMap([('number', 0), ('script', '$PYTHON setup.py install')])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['pymzml'])])), ('about', CommentedMap([('home', 'http://pymzml.github.com'), ('license', 'GNU General Public License (GPL)'), ('summary', 'high-throughput mzML parsing'), ('license_family', 'LGPL')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plink/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'plink'), ('version', '1.90b4')])), ('source', CommentedMap([('fn', '8b44285c90a932c189c35c8171177d324c47ba91.tar.gz'), ('url', 'https://github.com/chrchang/plink-ng/archive/8b44285c90a932c189c35c8171177d324c47ba91.tar.gz'), ('md5', '88290f567034153f48e8ca6bb595b3a8'), ('sha1', '9f41c77daa84c365aa659adf0619197fde1337bf'), ('sha256', 'af7bfa27bb303f9bf19b15d4a99e5e8f85de76013b54c05f3479385830598b4d')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'openblas', 'zlib']), ('run', ['libgcc', 'openblas', 'zlib'])])), ('test', CommentedMap([('commands', ['plink --help | grep PLINK'])])), ('about', CommentedMap([('home', 'https://www.cog-genomics.org/plink'), ('license', 'GPLv3'), ('license_file', 'LICENSE'), ('summary', 'Whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/plink/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'plink'), ('version', '1.90b4')])), ('source', CommentedMap([('fn', '8b44285c90a932c189c35c8171177d324c47ba91.tar.gz'), ('url', 'https://github.com/chrchang/plink-ng/archive/8b44285c90a932c189c35c8171177d324c47ba91.tar.gz'), ('md5', '88290f567034153f48e8ca6bb595b3a8'), ('sha1', '9f41c77daa84c365aa659adf0619197fde1337bf'), ('sha256', 'af7bfa27bb303f9bf19b15d4a99e5e8f85de76013b54c05f3479385830598b4d')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'openblas', 'zlib']), ('run', ['libgcc', 'openblas', 'zlib'])])), ('test', CommentedMap([('commands', ['plink --help | grep PLINK'])])), ('about', CommentedMap([('home', 'https://www.cog-genomics.org/plink'), ('license', 'GPLv3'), ('license_file', 'LICENSE'), ('summary', 'Whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-parody/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-parody'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'parody_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/parody_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/parody_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/parody/parody_1.34.0_src_all.tar.gz']), ('md5', '9dca708335f65ef8799b7b19247f1700')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'parody\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/parody.html'), ('license', 'Artistic-2.0'), ('summary', 'routines for univariate and multivariate outlier detection with a focus on parametric methods, but support for some methods based on resistant statistics')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-parody/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-parody'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'parody_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/parody_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/parody_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/parody/parody_1.34.0_src_all.tar.gz']), ('md5', '9dca708335f65ef8799b7b19247f1700')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'parody\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/parody.html'), ('license', 'Artistic-2.0'), ('summary', 'routines for univariate and multivariate outlier detection with a focus on parametric methods, but support for some methods based on resistant statistics')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-plw/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-plw'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'plw_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/plw_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/plw/plw_1.36.0_src_all.tar.gz']), ('md5', 'e8a4447aeb071da118ad6c79b39c9998')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.23.4', 'r-base', 'r-mass']), ('run', ['bioconductor-affy >=1.23.4', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'plw\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/plw.html'), ('license', 'GPL-2'), ('summary', 'Probe level Locally moderated Weighted median-t (PLW) and Locally Moderated Weighted-t (LMW).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-plw/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-plw'), ('version', '1.36.0')])), ('source', CommentedMap([('fn', 'plw_1.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/plw_1.36.0.tar.gz', 'https://depot.galaxyproject.org/software/plw/plw_1.36.0_src_all.tar.gz']), ('md5', 'e8a4447aeb071da118ad6c79b39c9998')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.23.4', 'r-base', 'r-mass']), ('run', ['bioconductor-affy >=1.23.4', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'plw\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/plw.html'), ('license', 'GPL-2'), ('summary', 'Probe level Locally moderated Weighted median-t (PLW) and Locally Moderated Weighted-t (LMW).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-teqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-teqc'), ('version', '3.16.0')])), ('source', CommentedMap([('fn', 'TEQC_3.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TEQC_3.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/TEQC_3.16.0.tar.gz', 'https://depot.galaxyproject.org/software/TEQC/TEQC_3.16.0_src_all.tar.gz']), ('md5', '8ebc30b256d07069ab9a012f23469a54')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rsamtools', 'r-base', 'r-hwriter']), ('run', ['bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rsamtools', 'r-base', 'r-hwriter'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TEQC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TEQC.html'), ('license', 'GPL (>= 2)'), ('summary', 'Target capture experiments combine hybridization-based (in solution or on microarrays) capture and enrichment of genomic regions of interest (e.g. the exome) with high throughput sequencing of the captured DNA fragments. This package provides functionalities for assessing and visualizing the quality of the target enrichment process, like specificity and sensitivity of the capture, per-target read coverage and so on.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-teqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-teqc'), ('version', '3.16.0')])), ('source', CommentedMap([('fn', 'TEQC_3.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TEQC_3.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/TEQC_3.16.0.tar.gz', 'https://depot.galaxyproject.org/software/TEQC/TEQC_3.16.0_src_all.tar.gz']), ('md5', '8ebc30b256d07069ab9a012f23469a54')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rsamtools', 'r-base', 'r-hwriter']), ('run', ['bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rsamtools', 'r-base', 'r-hwriter'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TEQC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TEQC.html'), ('license', 'GPL (>= 2)'), ('summary', 'Target capture experiments combine hybridization-based (in solution or on microarrays) capture and enrichment of genomic regions of interest (e.g. the exome) with high throughput sequencing of the captured DNA fragments. This package provides functionalities for assessing and visualizing the quality of the target enrichment process, like specificity and sensitivity of the capture, per-target read coverage and so on.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-acgh/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-acgh'), ('version', '1.54.0')])), ('source', CommentedMap([('fn', 'aCGH_1.54.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/aCGH_1.54.0.tar.gz', 'https://bioarchive.galaxyproject.org/aCGH_1.54.0.tar.gz', 'https://depot.galaxyproject.org/software/aCGH/aCGH_1.54.0_src_all.tar.gz']), ('md5', '8e5bd1800b2760e46fcab4179c2e920c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base', 'r-cluster', 'r-survival']), ('run', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base', 'r-cluster', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'aCGH\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/aCGH.html'), ('license', 'GPL-2'), ('summary', 'Functions for reading aCGH data from image analysis output files and clone information files, creation of aCGH S3 objects for storing these data. Basic methods for accessing/replacing, subsetting, printing and plotting aCGH objects.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-acgh/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-acgh'), ('version', '1.54.0')])), ('source', CommentedMap([('fn', 'aCGH_1.54.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/aCGH_1.54.0.tar.gz', 'https://bioarchive.galaxyproject.org/aCGH_1.54.0.tar.gz', 'https://depot.galaxyproject.org/software/aCGH/aCGH_1.54.0_src_all.tar.gz']), ('md5', '8e5bd1800b2760e46fcab4179c2e920c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base', 'r-cluster', 'r-survival']), ('run', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base', 'r-cluster', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'aCGH\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/aCGH.html'), ('license', 'GPL-2'), ('summary', 'Functions for reading aCGH data from image analysis output files and clone information files, creation of aCGH S3 objects for storing these data. Basic methods for accessing/replacing, subsetting, printing and plotting aCGH objects.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-panp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-panp'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'panp_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/panp_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/panp_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/panp/panp_1.46.0_src_all.tar.gz']), ('md5', 'e001987527a2b8e67d1066e36d75b293')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.23.4', 'bioconductor-biobase >=2.5.5', 'r-base']), ('run', ['bioconductor-affy >=1.23.4', 'bioconductor-biobase >=2.5.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'panp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/panp.html'), ('license', 'GPL (>= 2)'), ('summary', 'A function to make gene presence/absence calls based on distance from negative strand matching probesets (NSMP) which are derived from Affymetrix annotation. PANP is applied after gene expression values are created, and therefore can be used after any preprocessing method such as MAS5 or GCRMA, or PM-only methods like RMA. NSMP sets have been established for the HGU133A and HGU133-Plus-2.0 chipsets to date.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-panp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-panp'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'panp_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/panp_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/panp_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/panp/panp_1.46.0_src_all.tar.gz']), ('md5', 'e001987527a2b8e67d1066e36d75b293')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.23.4', 'bioconductor-biobase >=2.5.5', 'r-base']), ('run', ['bioconductor-affy >=1.23.4', 'bioconductor-biobase >=2.5.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'panp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/panp.html'), ('license', 'GPL (>= 2)'), ('summary', 'A function to make gene presence/absence calls based on distance from negative strand matching probesets (NSMP) which are derived from Affymetrix annotation. PANP is applied after gene expression values are created, and therefore can be used after any preprocessing method such as MAS5 or GCRMA, or PM-only methods like RMA. NSMP sets have been established for the HGU133A and HGU133-Plus-2.0 chipsets to date.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/platypus-variant/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.well.ox.ac.uk/platypus'), ('license', 'GPLv3'), ('summary', 'A Haplotype-Based Variant Caller For Next Generation Sequence Data')])), ('package', CommentedMap([('name', 'platypus-variant'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'Platypus_<{ version }}.tgz'), ('url', 'https://github.com/chapmanb/Platypus/releases/download/<{ version }}/Platypus_<{ version }}.tgz'), ('md5', '84b1ef968bf638e357c447abab39c463')])), ('requirements', CommentedMap([('build', ['python', 'htslib <{CONDA_HTSLIB}}*']), ('run', ['python', 'htslib <{CONDA_HTSLIB}}*'])])), ('test', CommentedMap([('commands', ['platypus callVariants -h'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/platypus-variant/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.well.ox.ac.uk/platypus'), ('license', 'GPLv3'), ('summary', 'A Haplotype-Based Variant Caller For Next Generation Sequence Data')])), ('package', CommentedMap([('name', 'platypus-variant'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'Platypus_<{ version }}.tgz'), ('url', 'https://github.com/chapmanb/Platypus/releases/download/<{ version }}/Platypus_<{ version }}.tgz'), ('md5', '84b1ef968bf638e357c447abab39c463')])), ('requirements', CommentedMap([('build', ['python', 'htslib <{CONDA_HTSLIB}}*']), ('run', ['python', 'htslib <{CONDA_HTSLIB}}*'])])), ('test', CommentedMap([('commands', ['platypus callVariants -h'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motifdb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motifdb'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'MotifDb_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/MotifDb_1.16.0.tar.gz'), ('md5', 'b30e09247f857e7bca54681acdd4e849')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MotifDb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/MotifDb.html'), ('license', 'Artistic-2.0 | file LICENSE'), ('summary', 'More than 2000 annotated position frequency matrices from nine public sources, for multiple organisms.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motifdb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motifdb'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'MotifDb_1.16.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/MotifDb_1.16.0.tar.gz'), ('md5', 'b30e09247f857e7bca54681acdd4e849')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MotifDb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/MotifDb.html'), ('license', 'Artistic-2.0 | file LICENSE'), ('summary', 'More than 2000 annotated position frequency matrices from nine public sources, for multiple organisms.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eqtlbma/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eqtlbma'), ('version', '1.3.1')])), ('source', CommentedMap([('fn', 'v1.3.1.tar.gz'), ('url', 'https://github.com/timflutre/eqtlbma/archive/v1.3.1.tar.gz'), ('md5', '7c9469c61b5d50cd7113c620d30a0142'), ('patches', ['utils_script.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'autoconf 2.69 pl5.*', 'automake 1.15 pl5.*', 'libtool', 'gsl >=1.15,<2.0', 'zlib', 'texinfo']), ('run', ['libgcc', 'gsl >=1.15,<2.0', 'zlib', 'r-base', 'r-mass', 'bioconductor-genomicranges'])])), ('test', CommentedMap([('commands', ['eqtlbma_bf -h', 'eqtlbma_hm -h', 'eqtlbma_avg_bfs -h'])])), ('about', CommentedMap([('home', 'https://github.com/timflutre/eqtlbma'), ('license', 'GPLv3'), ('license_file', 'COPYING'), ('summary', 'Package to detect eQTLs jointly in multiple subgroups (e.g. tissues) via Bayesian Model Averaging.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eqtlbma/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eqtlbma'), ('version', '1.3.1')])), ('source', CommentedMap([('fn', 'v1.3.1.tar.gz'), ('url', 'https://github.com/timflutre/eqtlbma/archive/v1.3.1.tar.gz'), ('md5', '7c9469c61b5d50cd7113c620d30a0142'), ('patches', ['utils_script.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'autoconf 2.69 pl5.*', 'automake 1.15 pl5.*', 'libtool', 'gsl >=1.15,<2.0', 'zlib', 'texinfo']), ('run', ['libgcc', 'gsl >=1.15,<2.0', 'zlib', 'r-base', 'r-mass', 'bioconductor-genomicranges'])])), ('test', CommentedMap([('commands', ['eqtlbma_bf -h', 'eqtlbma_hm -h', 'eqtlbma_avg_bfs -h'])])), ('about', CommentedMap([('home', 'https://github.com/timflutre/eqtlbma'), ('license', 'GPLv3'), ('license_file', 'COPYING'), ('summary', 'Package to detect eQTLs jointly in multiple subgroups (e.g. tissues) via Bayesian Model Averaging.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnaseqpower/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnaseqpower'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'RNASeqPower_1.14.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/RNASeqPower_1.14.0.tar.gz'), ('md5', 'b803cf98a37c07405f6843dc75f3aef9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RNASeqPower\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RNASeqPower.html'), ('license', 'LGPL (>=2)'), ('summary', 'RNA-seq, sample size')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnaseqpower/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnaseqpower'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'RNASeqPower_1.14.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/RNASeqPower_1.14.0.tar.gz'), ('md5', 'b803cf98a37c07405f6843dc75f3aef9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RNASeqPower\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RNASeqPower.html'), ('license', 'LGPL (>=2)'), ('summary', 'RNA-seq, sample size')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-acde/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-acde'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'acde_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/acde_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/acde/acde_1.6.0_src_all.tar.gz']), ('md5', 'e92ce91f75bab3bb1d79995bec1b42cc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-boot >=1.3']), ('run', ['r-base', 'r-boot >=1.3'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'acde\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/acde.html'), ('license', 'GPL-3'), ('summary', \"This package provides a multivariate inferential analysis method for detecting differentially expressed genes in gene expression data. It uses artificial components, close to the data's principal components but with an exact interpretation in terms of differential genetic expression, to identify differentially expressed genes while controlling the false discovery rate (FDR). The methods on this package are described in the vignette or in the article 'Multivariate Method for Inferential Identification of Differentially Expressed Genes in Gene Expression Experiments' by J. P. Acosta, L. Lopez-Kleine and S. Restrepo (2015, pending publication).\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-acde/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-acde'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'acde_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/acde_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/acde/acde_1.6.0_src_all.tar.gz']), ('md5', 'e92ce91f75bab3bb1d79995bec1b42cc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-boot >=1.3']), ('run', ['r-base', 'r-boot >=1.3'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'acde\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/acde.html'), ('license', 'GPL-3'), ('summary', \"This package provides a multivariate inferential analysis method for detecting differentially expressed genes in gene expression data. It uses artificial components, close to the data's principal components but with an exact interpretation in terms of differential genetic expression, to identify differentially expressed genes while controlling the false discovery rate (FDR). The methods on this package are described in the vignette or in the article 'Multivariate Method for Inferential Identification of Differentially Expressed Genes in Gene Expression Experiments' by J. P. Acosta, L. Lopez-Kleine and S. Restrepo (2015, pending publication).\")]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/takeabreak/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'takeabreak'), ('version', '1.1.2')])), ('source', CommentedMap([('fn', 'TakeABreak-v1.1.2-Source.tar.gz'), ('url', 'https://github.com/GATB/TakeABreak/releases/download/v1.1.2/TakeABreak-v1.1.2-Source.tar.gz'), ('md5', '9170a6d61f51e53189822df830648fd0')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['TakeABreak -help'])])), ('about', CommentedMap([('home', 'https://colibread.inria.fr/software/takeabreak/'), ('license', 'GNU Affero General Public License'), ('summary', 'tool that can detect inversion breakpoints directly from raw NGS reads, without the need of any reference genome and without de novo assembling the genomes')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/takeabreak/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'takeabreak'), ('version', '1.1.2')])), ('source', CommentedMap([('fn', 'TakeABreak-v1.1.2-Source.tar.gz'), ('url', 'https://github.com/GATB/TakeABreak/releases/download/v1.1.2/TakeABreak-v1.1.2-Source.tar.gz'), ('md5', '9170a6d61f51e53189822df830648fd0')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'cmake', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['TakeABreak -help'])])), ('about', CommentedMap([('home', 'https://colibread.inria.fr/software/takeabreak/'), ('license', 'GNU Affero General Public License'), ('summary', 'tool that can detect inversion breakpoints directly from raw NGS reads, without the need of any reference genome and without de novo assembling the genomes')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ssviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ssviz'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'ssviz_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ssviz_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/ssviz_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/ssviz/ssviz_1.10.0_src_all.tar.gz']), ('md5', 'e9b7d0ef0379968ee84d3228753ebc58')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-rsamtools', 'r-base', 'r-ggplot2', 'r-rcolorbrewer', 'r-reshape']), ('run', ['bioconductor-biostrings', 'bioconductor-rsamtools', 'r-base', 'r-ggplot2', 'r-rcolorbrewer', 'r-reshape'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ssviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ssviz.html'), ('license', 'GPL-2'), ('summary', 'Small RNA sequencing viewer')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ssviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ssviz'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'ssviz_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ssviz_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/ssviz_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/ssviz/ssviz_1.10.0_src_all.tar.gz']), ('md5', 'e9b7d0ef0379968ee84d3228753ebc58')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-rsamtools', 'r-base', 'r-ggplot2', 'r-rcolorbrewer', 'r-reshape']), ('run', ['bioconductor-biostrings', 'bioconductor-rsamtools', 'r-base', 'r-ggplot2', 'r-rcolorbrewer', 'r-reshape'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ssviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ssviz.html'), ('license', 'GPL-2'), ('summary', 'Small RNA sequencing viewer')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-manta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-manta'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'manta_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/manta_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/manta_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/manta/manta_1.22.0_src_all.tar.gz']), ('md5', '3f77679b1a72c82bd988352ceb00a20d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-edger >=2.5.13', 'r-base', 'r-caroline >=0.6.6', 'r-hmisc']), ('run', ['bioconductor-edger >=2.5.13', 'r-base', 'r-caroline >=0.6.6', 'r-hmisc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'manta\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/manta.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for robust comparative metatranscriptomics.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-manta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-manta'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'manta_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/manta_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/manta_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/manta/manta_1.22.0_src_all.tar.gz']), ('md5', '3f77679b1a72c82bd988352ceb00a20d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-edger >=2.5.13', 'r-base', 'r-caroline >=0.6.6', 'r-hmisc']), ('run', ['bioconductor-edger >=2.5.13', 'r-base', 'r-caroline >=0.6.6', 'r-hmisc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'manta\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/manta.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for robust comparative metatranscriptomics.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ripseeker/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ripseeker'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'RIPSeeker_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RIPSeeker_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/RIPSeeker_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/RIPSeeker/RIPSeeker_1.16.0_src_all.tar.gz']), ('md5', '41eb01d93e0fda73703df182d861b5c6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RIPSeeker\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RIPSeeker.html'), ('license', 'GPL-2'), ('summary', 'Infer and discriminate RIP peaks from RIP-seq alignments using two-state HMM with negative binomial emission probability. While RIPSeeker is specifically tailored for RIP-seq data analysis, it also provides a suite of bioinformatics tools integrated within this self-contained software package comprehensively addressing issues ranging from post-alignments processing to visualization and annotation.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ripseeker/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ripseeker'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'RIPSeeker_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RIPSeeker_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/RIPSeeker_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/RIPSeeker/RIPSeeker_1.16.0_src_all.tar.gz']), ('md5', '41eb01d93e0fda73703df182d861b5c6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RIPSeeker\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RIPSeeker.html'), ('license', 'GPL-2'), ('summary', 'Infer and discriminate RIP peaks from RIP-seq alignments using two-state HMM with negative binomial emission probability. While RIPSeeker is specifically tailored for RIP-seq data analysis, it also provides a suite of bioinformatics tools integrated within this self-contained software package comprehensively addressing issues ranging from post-alignments processing to visualization and annotation.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-degreport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-degreport'), ('version', '1.8.2')])), ('source', CommentedMap([('fn', 'DEGreport_1.8.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/DEGreport_1.8.2.tar.gz', 'https://depot.galaxyproject.org/software/DEGreport/DEGreport_1.8.2_src_all.tar.gz']), ('md5', '4006a779e33f3bb68308ba9f4a7d9f84')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-edger', 'r-base', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biocgenerics', 'bioconductor-edger', 'r-base', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DEGreport\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/DEGreport.html'), ('license', 'GPL (>=2)'), ('summary', 'Creation of a HTML report of differential expression analyses of count data. It integrates some of the code mentioned in DESeq2 and edgeR vignettes, and report a ranked list of genes according to the fold changes mean and variability for each selected gene.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-degreport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-degreport'), ('version', '1.8.2')])), ('source', CommentedMap([('fn', 'DEGreport_1.8.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/DEGreport_1.8.2.tar.gz', 'https://depot.galaxyproject.org/software/DEGreport/DEGreport_1.8.2_src_all.tar.gz']), ('md5', '4006a779e33f3bb68308ba9f4a7d9f84')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-edger', 'r-base', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biocgenerics', 'bioconductor-edger', 'r-base', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DEGreport\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/DEGreport.html'), ('license', 'GPL (>=2)'), ('summary', 'Creation of a HTML report of differential expression analyses of count data. It integrates some of the code mentioned in DESeq2 and edgeR vignettes, and report a ranked list of genes according to the fold changes mean and variability for each selected gene.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pepxmltab/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pepxmltab'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'pepXMLTab_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pepXMLTab_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/pepXMLTab_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/pepXMLTab/pepXMLTab_1.10.0_src_all.tar.gz']), ('md5', '3d0ccaed81b9dd5dba8fb0fb7fab829e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-xml >=3.98-1.1']), ('run', ['r-base', 'r-xml >=3.98-1.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pepXMLTab\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pepXMLTab.html'), ('license', 'Artistic-2.0'), ('summary', 'Parsing pepXML files based one XML package. The package tries to handle pepXML files generated from different softwares. The output will be a peptide-spectrum-matching tabular file. The package also provide function to filter the PSMs based on FDR.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pepxmltab/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pepxmltab'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'pepXMLTab_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pepXMLTab_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/pepXMLTab_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/pepXMLTab/pepXMLTab_1.10.0_src_all.tar.gz']), ('md5', '3d0ccaed81b9dd5dba8fb0fb7fab829e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-xml >=3.98-1.1']), ('run', ['r-base', 'r-xml >=3.98-1.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pepXMLTab\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pepXMLTab.html'), ('license', 'Artistic-2.0'), ('summary', 'Parsing pepXML files based one XML package. The package tries to handle pepXML files generated from different softwares. The output will be a peptide-spectrum-matching tabular file. The package also provide function to filter the PSMs based on FDR.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tspair/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tspair'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'tspair_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/tspair_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/tspair_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/tspair/tspair_1.34.0_src_all.tar.gz']), ('md5', '53493dd6c05e65839d80be76df2c033f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.4.0', 'r-base']), ('run', ['bioconductor-biobase >=2.4.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'tspair\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/tspair.html'), ('license', 'GPL-2'), ('summary', 'These functions calculate the pair of genes that show the maximum difference in ranking between two user specified groups. This \"top scoring pair\" maximizes the average of sensitivity and specificity over all rank based classifiers using a pair of genes in the data set. The advantage of classifying samples based on only the relative rank of a pair of genes is (a) the classifiers are much simpler and often more interpretable than more complicated classification schemes and (b) if arrays can be classified using only a pair of genes, PCR based tests could be used for classification of samples. See the references for the tspcalc() function for references regarding TSP classifiers.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tspair/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tspair'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'tspair_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/tspair_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/tspair_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/tspair/tspair_1.34.0_src_all.tar.gz']), ('md5', '53493dd6c05e65839d80be76df2c033f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.4.0', 'r-base']), ('run', ['bioconductor-biobase >=2.4.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'tspair\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/tspair.html'), ('license', 'GPL-2'), ('summary', 'These functions calculate the pair of genes that show the maximum difference in ranking between two user specified groups. This \"top scoring pair\" maximizes the average of sensitivity and specificity over all rank based classifiers using a pair of genes in the data set. The advantage of classifying samples based on only the relative rank of a pair of genes is (a) the classifiers are much simpler and often more interpretable than more complicated classification schemes and (b) if arrays can be classified using only a pair of genes, PCR based tests could be used for classification of samples. See the references for the tspcalc() function for references regarding TSP classifiers.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-polyfit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-polyfit'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'Polyfit_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Polyfit_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/Polyfit_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/Polyfit/Polyfit_1.10.0_src_all.tar.gz']), ('md5', 'c07c8cbf0bf82f73e22a2b15892e8187')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-deseq', 'r-base']), ('run', ['bioconductor-deseq', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Polyfit\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Polyfit.html'), ('license', 'GPL (>= 3)'), ('summary', 'Polyfit is an add-on to the packages DESeq which ensures the p-value distribution is uniform over the interval [0, 1] for data satisfying the null hypothesis of no differential expression, and uses an adpated Storey-Tibshiran method to calculate q-values.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-polyfit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-polyfit'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'Polyfit_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Polyfit_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/Polyfit_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/Polyfit/Polyfit_1.10.0_src_all.tar.gz']), ('md5', 'c07c8cbf0bf82f73e22a2b15892e8187')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-deseq', 'r-base']), ('run', ['bioconductor-deseq', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Polyfit\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Polyfit.html'), ('license', 'GPL (>= 3)'), ('summary', 'Polyfit is an add-on to the packages DESeq which ensures the p-value distribution is uniform over the interval [0, 1] for data satisfying the null hypothesis of no differential expression, and uses an adpated Storey-Tibshiran method to calculate q-values.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mpfe/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mpfe'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'MPFE_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MPFE_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/MPFE_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/MPFE/MPFE_1.12.0_src_all.tar.gz']), ('md5', 'efc12549261886fd1c4ed958bc29d416')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MPFE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MPFE.html'), ('license', 'GPL (>= 3)'), ('summary', 'Estimate distribution of methylation patterns from a table of counts from a bisulphite sequencing experiment given a non-conversion rate and read error rate.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mpfe/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mpfe'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'MPFE_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MPFE_1.12.0.tar.gz', 'https://bioarchive.galaxyproject.org/MPFE_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/MPFE/MPFE_1.12.0_src_all.tar.gz']), ('md5', 'efc12549261886fd1c4ed958bc29d416')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MPFE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MPFE.html'), ('license', 'GPL (>= 3)'), ('summary', 'Estimate distribution of methylation patterns from a table of counts from a bisulphite sequencing experiment given a non-conversion rate and read error rate.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/blat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'blat'), ('version', 35)])), ('source', CommentedMap([('fn', 'blatSrc35.zip'), ('url', 'https://users.soe.ucsc.edu/~kent/src/blatSrc35.zip'), ('md5', '16e546b8843b85e0b0f2fa603cd78724')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['libpng', 'zlib']), ('run', ['libpng', 'zlib'])])), ('about', CommentedMap([('home', 'https://genome.ucsc.edu/FAQ/FAQblat.html'), ('license', 'Free for academic, nonprofit and personal use'), ('summary', 'BLAT is a bioinformatics software tool which performs rapid mRNA/DNA and cross-species protein alignments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/blat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'blat'), ('version', 35)])), ('source', CommentedMap([('fn', 'blatSrc35.zip'), ('url', 'https://users.soe.ucsc.edu/~kent/src/blatSrc35.zip'), ('md5', '16e546b8843b85e0b0f2fa603cd78724')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['libpng', 'zlib']), ('run', ['libpng', 'zlib'])])), ('about', CommentedMap([('home', 'https://genome.ucsc.edu/FAQ/FAQblat.html'), ('license', 'Free for academic, nonprofit and personal use'), ('summary', 'BLAT is a bioinformatics software tool which performs rapid mRNA/DNA and cross-species protein alignments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/motus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'motus'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'mOTUs.Linux64bits.tar.gz'), ('url', 'http://www.bork.embl.de/software/mOTU/share/mOTUs.Linux64bits.tar.gz'), ('md5', '4c1fc6dc2ea803c9ee58d96bc6301b8d')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', ['mOTUs.pl | head -2 | tail -1 | grep -q \"^For single-sample mode:\"; echo $?'])])), ('about', CommentedMap([('home', 'http://www.bork.embl.de/software/mOTU'), ('license', 'GNU GPL v3'), ('summary', 'mOTUs Metagenomic species profiling using universal phylogenetic marker genes')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/motus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'motus'), ('version', '1.0')])), ('source', CommentedMap([('fn', 'mOTUs.Linux64bits.tar.gz'), ('url', 'http://www.bork.embl.de/software/mOTU/share/mOTUs.Linux64bits.tar.gz'), ('md5', '4c1fc6dc2ea803c9ee58d96bc6301b8d')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('test', CommentedMap([('commands', ['mOTUs.pl | head -2 | tail -1 | grep -q \"^For single-sample mode:\"; echo $?'])])), ('about', CommentedMap([('home', 'http://www.bork.embl.de/software/mOTU'), ('license', 'GNU GPL v3'), ('summary', 'mOTUs Metagenomic species profiling using universal phylogenetic marker genes')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bumphunter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bumphunter'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'bumphunter_1.12.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/bumphunter_1.12.0.tar.gz'), ('md5', 'ad8d1badfa41a1bd08d2a736d6e63488')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.3.23', 'bioconductor-limma', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dorng', 'r-foreach', 'r-iterators', 'r-locfit', 'r-matrixstats']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.3.23', 'bioconductor-limma', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dorng', 'r-foreach', 'r-iterators', 'r-locfit', 'r-matrixstats'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'bumphunter\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/bumphunter.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for finding bumps in genomic data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bumphunter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bumphunter'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'bumphunter_1.12.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/bumphunter_1.12.0.tar.gz'), ('md5', 'ad8d1badfa41a1bd08d2a736d6e63488')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.3.23', 'bioconductor-limma', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dorng', 'r-foreach', 'r-iterators', 'r-locfit', 'r-matrixstats']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=2.3.23', 'bioconductor-limma', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dorng', 'r-foreach', 'r-iterators', 'r-locfit', 'r-matrixstats'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'bumphunter\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/bumphunter.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for finding bumps in genomic data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methtargetedngs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methtargetedngs'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'MethTargetedNGS_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MethTargetedNGS_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/MethTargetedNGS_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/MethTargetedNGS/MethTargetedNGS_1.8.0_src_all.tar.gz']), ('md5', '26100031f59e66d819fbd2780f4ef7c4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'r-base', 'r-gplots', 'r-seqinr', 'r-stringr']), ('run', ['bioconductor-biostrings', 'r-base', 'r-gplots', 'r-seqinr', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MethTargetedNGS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MethTargetedNGS.html'), ('license', 'Artistic-2.0'), ('summary', 'Perform step by step methylation analysis of Next Generation Sequencing data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methtargetedngs/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methtargetedngs'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'MethTargetedNGS_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MethTargetedNGS_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/MethTargetedNGS_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/MethTargetedNGS/MethTargetedNGS_1.8.0_src_all.tar.gz']), ('md5', '26100031f59e66d819fbd2780f4ef7c4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'r-base', 'r-gplots', 'r-seqinr', 'r-stringr']), ('run', ['bioconductor-biostrings', 'r-base', 'r-gplots', 'r-seqinr', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MethTargetedNGS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MethTargetedNGS.html'), ('license', 'Artistic-2.0'), ('summary', 'Perform step by step methylation analysis of Next Generation Sequencing data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-impute/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-impute'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'impute_1.46.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/impute_1.46.0.tar.gz'), ('md5', '8d0adfd6536a82a0c39cea3c5783ffe4')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'impute\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/impute.html'), ('license', 'GPL-2'), ('summary', 'Imputation for microarray data (currently KNN only)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-impute/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-impute'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'impute_1.46.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/impute_1.46.0.tar.gz'), ('md5', '8d0adfd6536a82a0c39cea3c5783ffe4')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'impute\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/impute.html'), ('license', 'GPL-2'), ('summary', 'Imputation for microarray data (currently KNN only)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/selscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'selscan'), ('version', '1.1.0b')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', '1.1.0b.zip'), ('url', 'https://github.com/szpiech/selscan/archive/1.1.0b.zip'), ('md5', '02a39fff61fadcfb5df248d37111141f'), ('patches', ['Makefile.linux.patch', 'Makefile.mac.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', [\"selscan 2>&1 | grep 'selscan' &> /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/szpiech/selscan'), ('license', 'GPLv3'), ('summary', 'a program to calculate EHH-based scans for positive selection in genomes')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/selscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'selscan'), ('version', '1.1.0b')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', '1.1.0b.zip'), ('url', 'https://github.com/szpiech/selscan/archive/1.1.0b.zip'), ('md5', '02a39fff61fadcfb5df248d37111141f'), ('patches', ['Makefile.linux.patch', 'Makefile.mac.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', [\"selscan 2>&1 | grep 'selscan' &> /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/szpiech/selscan'), ('license', 'GPLv3'), ('summary', 'a program to calculate EHH-based scans for positive selection in genomes')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/exparna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'exparna'), ('version', '1.0.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/BackofenLab/ExpaRNA/archive/1.0.1.tar.gz'), ('md5', '2120711de20d4f62ded1e0394cd0e601')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'automake', 'autoconf', 'perl-threaded', 'viennarna']), ('run', ['libgcc', 'viennarna'])])), ('test', CommentedMap([('commands', ['ExpaRNA -h'])])), ('about', CommentedMap([('home', 'https://github.com/BackofenLab/ExpaRNA'), ('license', 'GPL-2.0'), ('license_file', 'LICENSE'), ('summary', 'The program finds the longest common subsequence of exact pattern matches (LCS-EPM)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/exparna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'exparna'), ('version', '1.0.1')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('url', 'https://github.com/BackofenLab/ExpaRNA/archive/1.0.1.tar.gz'), ('md5', '2120711de20d4f62ded1e0394cd0e601')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'automake', 'autoconf', 'perl-threaded', 'viennarna']), ('run', ['libgcc', 'viennarna'])])), ('test', CommentedMap([('commands', ['ExpaRNA -h'])])), ('about', CommentedMap([('home', 'https://github.com/BackofenLab/ExpaRNA'), ('license', 'GPL-2.0'), ('license_file', 'LICENSE'), ('summary', 'The program finds the longest common subsequence of exact pattern matches (LCS-EPM)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-stringdb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-stringdb'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'STRINGdb_1.16.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/STRINGdb_1.16.0.tar.gz'), ('md5', '2892f288a504491e56686a22650aa4e5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gplots', 'r-hash', 'r-igraph', 'r-plotrix', 'r-plyr', 'r-png', 'r-rcolorbrewer', 'r-rcurl', 'r-sqldf']), ('run', ['r-base', 'r-gplots', 'r-hash', 'r-igraph', 'r-plotrix', 'r-plyr', 'r-png', 'r-rcolorbrewer', 'r-rcurl', 'r-sqldf'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'STRINGdb\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/STRINGdb.html'), ('license', 'GPL-2'), ('summary', 'The STRINGdb package provides a R interface to the STRING protein-protein interactions database (http://www.string-db.org)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-stringdb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-stringdb'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'STRINGdb_1.16.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/STRINGdb_1.16.0.tar.gz'), ('md5', '2892f288a504491e56686a22650aa4e5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-gplots', 'r-hash', 'r-igraph', 'r-plotrix', 'r-plyr', 'r-png', 'r-rcolorbrewer', 'r-rcurl', 'r-sqldf']), ('run', ['r-base', 'r-gplots', 'r-hash', 'r-igraph', 'r-plotrix', 'r-plyr', 'r-png', 'r-rcolorbrewer', 'r-rcurl', 'r-sqldf'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'STRINGdb\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/STRINGdb.html'), ('license', 'GPL-2'), ('summary', 'The STRINGdb package provides a R interface to the STRING protein-protein interactions database (http://www.string-db.org)')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-multtest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-multtest'), ('version', '2.28.0')])), ('source', CommentedMap([('fn', 'multtest_2.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/multtest_2.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/multtest_2.28.0.tar.gz', 'https://depot.galaxyproject.org/software/multtest/multtest_2.28.0_src_all.tar.gz']), ('md5', 'a82a1564e15fdc5afe6efc4c33a33bbc')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-mass', 'r-survival']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-mass', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'multtest\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/multtest.html'), ('license', 'LGPL'), ('summary', 'Non-parametric bootstrap and permutation resampling-based multiple testing procedures (including empirical Bayes methods) for controlling the family-wise error rate (FWER), generalized family-wise error rate (gFWER), tail probability of the proportion of false positives (TPPFP), and false discovery rate (FDR).  Several choices of bootstrap-based null distribution are implemented (centered, centered and scaled, quantile-transformed). Single-step and step-wise methods are available. Tests based on a variety of t- and F-statistics (including t-statistics based on regression parameters from linear and survival models as well as those based on correlation parameters) are included.  When probing hypotheses with t-statistics, users may also select a potentially faster null distribution which is multivariate normal with mean zero and variance covariance matrix derived from the vector influence function.  Results are reported in terms of adjusted p-values, confidence regions and test statistic cutoffs. The procedures are directly applicable to identifying differentially expressed genes in DNA microarray experiments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-multtest/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-multtest'), ('version', '2.28.0')])), ('source', CommentedMap([('fn', 'multtest_2.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.3/bioc/src/contrib/multtest_2.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/multtest_2.28.0.tar.gz', 'https://depot.galaxyproject.org/software/multtest/multtest_2.28.0_src_all.tar.gz']), ('md5', 'a82a1564e15fdc5afe6efc4c33a33bbc')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-mass', 'r-survival']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'r-base', 'r-mass', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'multtest\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/multtest.html'), ('license', 'LGPL'), ('summary', 'Non-parametric bootstrap and permutation resampling-based multiple testing procedures (including empirical Bayes methods) for controlling the family-wise error rate (FWER), generalized family-wise error rate (gFWER), tail probability of the proportion of false positives (TPPFP), and false discovery rate (FDR).  Several choices of bootstrap-based null distribution are implemented (centered, centered and scaled, quantile-transformed). Single-step and step-wise methods are available. Tests based on a variety of t- and F-statistics (including t-statistics based on regression parameters from linear and survival models as well as those based on correlation parameters) are included.  When probing hypotheses with t-statistics, users may also select a potentially faster null distribution which is multivariate normal with mean zero and variance covariance matrix derived from the vector influence function.  Results are reported in terms of adjusted p-values, confidence regions and test statistic cutoffs. The procedures are directly applicable to identifying differentially expressed genes in DNA microarray experiments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastqc/0.11.3/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.bioinformatics.babraham.ac.uk/projects/fastqc/'), ('license', 'GPL'), ('summary', 'A quality control tool for high throughput sequence data.')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk', 'perl-threaded'])])), ('package', CommentedMap([('name', 'fastqc'), ('version', '0.11.3')])), ('source', CommentedMap([('fn', 'fastqc_v0.11.3.zip'), ('url', 'http://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v0.11.3.zip'), ('patches', ['hashbang.patch'])])), ('test', CommentedMap([('commands', [\"fastqc --version | grep '^FastQC v0\\\\.11\\\\.3\\\\b'\"])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/fastqc/0.11.3/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.bioinformatics.babraham.ac.uk/projects/fastqc/'), ('license', 'GPL'), ('summary', 'A quality control tool for high throughput sequence data.')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk', 'perl-threaded'])])), ('package', CommentedMap([('name', 'fastqc'), ('version', '0.11.3')])), ('source', CommentedMap([('fn', 'fastqc_v0.11.3.zip'), ('url', 'http://www.bioinformatics.babraham.ac.uk/projects/fastqc/fastqc_v0.11.3.zip'), ('patches', ['hashbang.patch'])])), ('test', CommentedMap([('commands', [\"fastqc --version | grep '^FastQC v0\\\\.11\\\\.3\\\\b'\"])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mgfm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mgfm'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MGFM_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MGFM_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MGFM_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MGFM/MGFM_1.10.0_src_all.tar.gz']), ('md5', 'ad048930fe4233344085409b2a5b11cf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'r-base']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MGFM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MGFM.html'), ('license', 'GPL-3'), ('summary', 'The package is designed to detect marker genes from Microarray gene expression data sets')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mgfm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mgfm'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MGFM_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MGFM_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MGFM_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MGFM/MGFM_1.10.0_src_all.tar.gz']), ('md5', 'ad048930fe4233344085409b2a5b11cf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'r-base']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MGFM\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MGFM.html'), ('license', 'GPL-3'), ('summary', 'The package is designed to detect marker genes from Microarray gene expression data sets')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-simbindprofiles/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-simbindprofiles'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'SimBindProfiles_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SimBindProfiles_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/SimBindProfiles_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/SimBindProfiles/SimBindProfiles_1.14.0_src_all.tar.gz']), ('md5', '34be39f2a337a2975203476b8aab849f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-ringo', 'r-base', 'r-mclust']), ('run', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-ringo', 'r-base', 'r-mclust'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SimBindProfiles\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SimBindProfiles.html'), ('license', 'GPL-3'), ('summary', 'SimBindProfiles identifies common and unique binding regions in genome tiling array data. This package does not rely on peak calling, but directly compares binding profiles processed on the same array platform. It implements a simple threshold approach, thus allowing retrieval of commonly and differentially bound regions between datasets as well as events of compensation and increased binding.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-simbindprofiles/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-simbindprofiles'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'SimBindProfiles_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SimBindProfiles_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/SimBindProfiles_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/SimBindProfiles/SimBindProfiles_1.14.0_src_all.tar.gz']), ('md5', '34be39f2a337a2975203476b8aab849f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-ringo', 'r-base', 'r-mclust']), ('run', ['bioconductor-biobase', 'bioconductor-limma', 'bioconductor-ringo', 'r-base', 'r-mclust'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SimBindProfiles\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SimBindProfiles.html'), ('license', 'GPL-3'), ('summary', 'SimBindProfiles identifies common and unique binding regions in genome tiling array data. This package does not rely on peak calling, but directly compares binding profiles processed on the same array platform. It implements a simple threshold approach, thus allowing retrieval of commonly and differentially bound regions between datasets as well as events of compensation and increased binding.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biobase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biobase'), ('version', '2.34.0')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/bioconductor-biobase/bioconductor-biobase_2.34.0_src_all.tar.gz'), ('sha256', 'f4ab46c3a02e2c9e9f70b99d751c8749a5b528a53e5117c3290e698a7992494b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.3.2', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.3.2', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Biobase\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/Biobase.html'), ('license', 'Artistic-2.0'), ('summary', 'Functions that are needed by many other packages or which replace R functions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biobase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biobase'), ('version', '2.34.0')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/bioconductor-biobase/bioconductor-biobase_2.34.0_src_all.tar.gz'), ('sha256', 'f4ab46c3a02e2c9e9f70b99d751c8749a5b528a53e5117c3290e698a7992494b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.3.2', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.3.2', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Biobase\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/Biobase.html'), ('license', 'Artistic-2.0'), ('summary', 'Functions that are needed by many other packages or which replace R functions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sispa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sispa'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'SISPA_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SISPA_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/SISPA_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/SISPA/SISPA_1.6.0_src_all.tar.gz']), ('md5', '8af8654a5736fa2ad073a1ebab30d9ec')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genefilter', 'bioconductor-gsva', 'r-base', 'r-changepoint', 'r-data.table', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-genefilter', 'bioconductor-gsva', 'r-base', 'r-changepoint', 'r-data.table', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SISPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SISPA.html'), ('license', 'GPL-2'), ('summary', 'Sample Integrated Set Profile Analysis (SISPA) is a method designed to define sample groups with similar gene set enrichment profiles.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sispa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sispa'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'SISPA_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SISPA_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/SISPA_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/SISPA/SISPA_1.6.0_src_all.tar.gz']), ('md5', '8af8654a5736fa2ad073a1ebab30d9ec')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genefilter', 'bioconductor-gsva', 'r-base', 'r-changepoint', 'r-data.table', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-genefilter', 'bioconductor-gsva', 'r-base', 'r-changepoint', 'r-data.table', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SISPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SISPA.html'), ('license', 'GPL-2'), ('summary', 'Sample Integrated Set Profile Analysis (SISPA) is a method designed to define sample groups with similar gene set enrichment profiles.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-michip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-michip'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'MiChip_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MiChip_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/MiChip_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/MiChip/MiChip_1.30.0_src_all.tar.gz']), ('md5', '0242abbd32d6d5a13c2f384a79cf99b0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MiChip\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MiChip.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package takes the MiChip miRNA microarray .grp scanner output files and parses these out, providing summary and plotting functions\\tto analyse MiChip hybridizations. A set of hybridizations is packaged into an ExpressionSet allowing it to be used by other\\tBioConductor packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-michip/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-michip'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'MiChip_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MiChip_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/MiChip_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/MiChip/MiChip_1.30.0_src_all.tar.gz']), ('md5', '0242abbd32d6d5a13c2f384a79cf99b0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MiChip\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MiChip.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package takes the MiChip miRNA microarray .grp scanner output files and parses these out, providing summary and plotting functions\\tto analyse MiChip hybridizations. A set of hybridizations is packaged into an ExpressionSet allowing it to be used by other\\tBioConductor packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dsrc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dsrc'), ('version', '2014.12.17')])), ('source', CommentedMap([('fn', 'v2.0.2.tar.gz'), ('url', 'https://github.com/lrog/dsrc/archive/v2.0.2.tar.gz')])), ('build', CommentedMap([('skip', True), ('number', 1), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'zlib', 'boost <{CONDA_BOOST}}*'])])), ('tests', CommentedMap([('commands', ['dsrc -h'])])), ('about', CommentedMap([('home', 'https://github.com/lrog/dsrc'), ('summary', 'high-performance compression of sequencing reads stored in FASTQ format'), ('license', 'GNU GPL 2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/dsrc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'dsrc'), ('version', '2014.12.17')])), ('source', CommentedMap([('fn', 'v2.0.2.tar.gz'), ('url', 'https://github.com/lrog/dsrc/archive/v2.0.2.tar.gz')])), ('build', CommentedMap([('skip', True), ('number', 1), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'zlib', 'boost <{CONDA_BOOST}}*'])])), ('tests', CommentedMap([('commands', ['dsrc -h'])])), ('about', CommentedMap([('home', 'https://github.com/lrog/dsrc'), ('summary', 'high-performance compression of sequencing reads stored in FASTQ format'), ('license', 'GNU GPL 2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gat'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://pypi.io/packages/source/g/gat/gat-<{ version }}.tar.gz'), ('md5', 'c0271d7235eddfdc912afa1ee4b7d278')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'zlib', 'gcc', 'llvm', 'setuptools >=1.1', 'cython >=0.19', 'numpy >=1.7', 'scipy >=0.11', 'matplotlib >=1.3.0']), ('run', ['python', 'zlib', 'cython >=0.19', 'numpy >=1.7', 'scipy >=0.11', 'matplotlib >=1.3.0'])])), ('test', CommentedMap([('imports', ['gat']), ('commands', ['gat-run.py -h'])])), ('about', CommentedMap([('summary', 'Genomic Association Tester'), ('license', 'MIT'), ('home', 'https://github.com/AndreasHeger/gat')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'gat'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://pypi.io/packages/source/g/gat/gat-<{ version }}.tar.gz'), ('md5', 'c0271d7235eddfdc912afa1ee4b7d278')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'zlib', 'gcc', 'llvm', 'setuptools >=1.1', 'cython >=0.19', 'numpy >=1.7', 'scipy >=0.11', 'matplotlib >=1.3.0']), ('run', ['python', 'zlib', 'cython >=0.19', 'numpy >=1.7', 'scipy >=0.11', 'matplotlib >=1.3.0'])])), ('test', CommentedMap([('imports', ['gat']), ('commands', ['gat-run.py -h'])])), ('about', CommentedMap([('summary', 'Genomic Association Tester'), ('license', 'MIT'), ('home', 'https://github.com/AndreasHeger/gat')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nnnorm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nnnorm'), ('version', '2.40.0')])), ('source', CommentedMap([('fn', 'nnNorm_2.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nnNorm_2.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/nnNorm_2.40.0.tar.gz', 'https://depot.galaxyproject.org/software/nnNorm/nnNorm_2.40.0_src_all.tar.gz']), ('md5', 'e5d78c1120e7cd09a530637c89811eb8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-marray', 'r-base', 'r-nnet']), ('run', ['bioconductor-marray', 'r-base', 'r-nnet'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nnNorm\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nnNorm.html'), ('license', 'LGPL'), ('summary', 'This package allows to detect and correct for spatial and intensity biases with two-channel microarray data. The normalization method implemented in this package is based on robust neural networks fitting.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nnnorm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nnnorm'), ('version', '2.40.0')])), ('source', CommentedMap([('fn', 'nnNorm_2.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nnNorm_2.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/nnNorm_2.40.0.tar.gz', 'https://depot.galaxyproject.org/software/nnNorm/nnNorm_2.40.0_src_all.tar.gz']), ('md5', 'e5d78c1120e7cd09a530637c89811eb8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-marray', 'r-base', 'r-nnet']), ('run', ['bioconductor-marray', 'r-base', 'r-nnet'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nnNorm\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nnNorm.html'), ('license', 'LGPL'), ('summary', 'This package allows to detect and correct for spatial and intensity biases with two-channel microarray data. The normalization method implemented in this package is based on robust neural networks fitting.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/last/874/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://last.cbrc.jp/'), ('license', 'GPLv3'), ('summary', 'LAST finds similar regions between sequences, and aligns them. It is designed for comparing large datasets to each other (e.g. vertebrate genomes and/or large numbers of DNA reads).')])), ('package', CommentedMap([('name', 'last'), ('version', '874')])), ('source', CommentedMap([('fn', 'last-874.zip'), ('md5', '499c75e5f7dd66fd8692d87e06843a13'), ('url', 'http://last.cbrc.jp/last-874.zip'), ('patches', ['maf-convert.23patch'])])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm']), ('run', ['python', 'libgcc', 'future'])])), ('test', CommentedMap([('commands', ['lastal -V', 'lastdb -V', 'lastal8 -h', 'lastdb8 -h', 'last-split -V', 'last-pair-probs -V', 'last-merge-batches -V', 'maf-convert -h'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/last/874/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://last.cbrc.jp/'), ('license', 'GPLv3'), ('summary', 'LAST finds similar regions between sequences, and aligns them. It is designed for comparing large datasets to each other (e.g. vertebrate genomes and/or large numbers of DNA reads).')])), ('package', CommentedMap([('name', 'last'), ('version', '874')])), ('source', CommentedMap([('fn', 'last-874.zip'), ('md5', '499c75e5f7dd66fd8692d87e06843a13'), ('url', 'http://last.cbrc.jp/last-874.zip'), ('patches', ['maf-convert.23patch'])])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['python', 'gcc', 'llvm']), ('run', ['python', 'libgcc', 'future'])])), ('test', CommentedMap([('commands', ['lastal -V', 'lastdb -V', 'lastal8 -h', 'lastdb8 -h', 'last-split -V', 'last-pair-probs -V', 'last-merge-batches -V', 'maf-convert -h'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sushi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sushi'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'Sushi_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Sushi_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/Sushi_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/Sushi/Sushi_1.14.0_src_all.tar.gz']), ('md5', '045d9d479da85852673e02940c6ab843')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biomart', 'r-base', 'r-zoo']), ('run', ['bioconductor-biomart', 'r-base', 'r-zoo'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Sushi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Sushi.html'), ('license', 'GPL (>= 2)'), ('summary', 'Flexible, quantitative, and integrative genomic visualizations for publication-quality multi-panel figures')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sushi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sushi'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'Sushi_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Sushi_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/Sushi_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/Sushi/Sushi_1.14.0_src_all.tar.gz']), ('md5', '045d9d479da85852673e02940c6ab843')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biomart', 'r-base', 'r-zoo']), ('run', ['bioconductor-biomart', 'r-base', 'r-zoo'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Sushi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Sushi.html'), ('license', 'GPL (>= 2)'), ('summary', 'Flexible, quantitative, and integrative genomic visualizations for publication-quality multi-panel figures')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-basic4cseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-basic4cseq'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Basic4Cseq_1.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Basic4Cseq_1.6.0.tar.gz'), ('md5', '0a056c1a35f02918bfd165339fd6c00f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-bsgenome.ecoli.ncbi.20080805', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'r-base', 'r-catools', 'r-rcircos']), ('run', ['bioconductor-biostrings', 'bioconductor-bsgenome.ecoli.ncbi.20080805', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'r-base', 'r-catools', 'r-rcircos'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Basic4Cseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Basic4Cseq.html'), ('license', 'LGPL-3'), ('summary', \"Basic4Cseq is an R/Bioconductor package for basic filtering, analysis and subsequent visualization of 4C-seq data. Virtual fragment libraries can be created for any BSGenome package, and filter functions for both reads and fragments and basic quality controls are included. Fragment data in the vicinity of the experiment's viewpoint can be visualized as a coverage plot based on a running median approach and a multi-scale contact profile.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-basic4cseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-basic4cseq'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Basic4Cseq_1.6.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Basic4Cseq_1.6.0.tar.gz'), ('md5', '0a056c1a35f02918bfd165339fd6c00f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-bsgenome.ecoli.ncbi.20080805', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'r-base', 'r-catools', 'r-rcircos']), ('run', ['bioconductor-biostrings', 'bioconductor-bsgenome.ecoli.ncbi.20080805', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'r-base', 'r-catools', 'r-rcircos'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Basic4Cseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Basic4Cseq.html'), ('license', 'LGPL-3'), ('summary', \"Basic4Cseq is an R/Bioconductor package for basic filtering, analysis and subsequent visualization of 4C-seq data. Virtual fragment libraries can be created for any BSGenome package, and filter functions for both reads and fragments and basic quality controls are included. Fragment data in the vicinity of the experiment's viewpoint can be visualized as a coverage plot based on a running median approach and a multi-scale contact profile.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rmat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rmat'), ('version', '3.26.0')])), ('source', CommentedMap([('fn', 'rMAT_3.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rMAT_3.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/rMAT_3.26.0.tar.gz', 'https://depot.galaxyproject.org/software/rMAT/rMAT_3.26.0_src_all.tar.gz']), ('md5', '5a4637f45a1c0f73bf9532534d2b6bcd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affxparser', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.3', 'bioconductor-iranges >=1.13.10', 'r-base']), ('run', ['bioconductor-affxparser', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.3', 'bioconductor-iranges >=1.13.10', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rMAT\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rMAT.html'), ('license', 'Artistic-2.0'), ('summary', \"This package is an R version of the package MAT and contains functions to parse and merge Affymetrix BPMAP and CEL tiling array files (using C++ based Fusion SDK and Bioconductor package affxparser), normalize tiling arrays using sequence specific models, detect enriched regions from ChIP-chip experiments. Note: users should have GSL and GenomeGraphs installed. Windows users: 'consult the README file available in the inst directory of the source distribution for necessary configuration instructions'. Snow Leopard users can take advantage of increase speed with Grand Central Dispatch!\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rmat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rmat'), ('version', '3.26.0')])), ('source', CommentedMap([('fn', 'rMAT_3.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rMAT_3.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/rMAT_3.26.0.tar.gz', 'https://depot.galaxyproject.org/software/rMAT/rMAT_3.26.0_src_all.tar.gz']), ('md5', '5a4637f45a1c0f73bf9532534d2b6bcd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affxparser', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.3', 'bioconductor-iranges >=1.13.10', 'r-base']), ('run', ['bioconductor-affxparser', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics >=0.1.3', 'bioconductor-iranges >=1.13.10', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rMAT\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rMAT.html'), ('license', 'Artistic-2.0'), ('summary', \"This package is an R version of the package MAT and contains functions to parse and merge Affymetrix BPMAP and CEL tiling array files (using C++ based Fusion SDK and Bioconductor package affxparser), normalize tiling arrays using sequence specific models, detect enriched regions from ChIP-chip experiments. Note: users should have GSL and GenomeGraphs installed. Windows users: 'consult the README file available in the inst directory of the source distribution for necessary configuration instructions'. Snow Leopard users can take advantage of increase speed with Grand Central Dispatch!\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/beast/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://beast.bio.ed.ac.uk'), ('license', 'GPLv2+'), ('summary', 'BEAST is a cross-platform program for Bayesian analysis of molecular sequences using MCMC')])), ('package', CommentedMap([('name', 'beast'), ('version', '1.8.4')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'BEASTv1.8.4.tgz'), ('url', 'https://github.com/beast-dev/beast-mcmc/releases/download/v1.8.4/BEASTv1.8.4.tgz'), ('md5', 'cb8752340c1f77a22d39ca4fe09687b0')])), ('requirements', CommentedMap([('run', ['openjdk', 'beagle-lib'])])), ('test', CommentedMap([('commands', ['beast -version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/beast/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://beast.bio.ed.ac.uk'), ('license', 'GPLv2+'), ('summary', 'BEAST is a cross-platform program for Bayesian analysis of molecular sequences using MCMC')])), ('package', CommentedMap([('name', 'beast'), ('version', '1.8.4')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'BEASTv1.8.4.tgz'), ('url', 'https://github.com/beast-dev/beast-mcmc/releases/download/v1.8.4/BEASTv1.8.4.tgz'), ('md5', 'cb8752340c1f77a22d39ca4fe09687b0')])), ('requirements', CommentedMap([('run', ['openjdk', 'beagle-lib'])])), ('test', CommentedMap([('commands', ['beast -version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/clark/1.2.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'clark'), ('version', '1.2.3')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'CLARKV1.2.3.tar.gz'), ('url', 'http://clark.cs.ucr.edu/Download/CLARKV1.2.3.tar.gz'), ('md5', '32d5643476452c7e70539663fe253ff2')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['CLARK --help'])])), ('about', CommentedMap([('home', 'http://clark.cs.ucr.edu/'), ('license', 'GPL3'), ('license_file', 'LICENSE_GNU_GPL.txt'), ('summary', 'Fast, accurate and versatile k-mer based classification system')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/clark/1.2.3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'clark'), ('version', '1.2.3')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'CLARKV1.2.3.tar.gz'), ('url', 'http://clark.cs.ucr.edu/Download/CLARKV1.2.3.tar.gz'), ('md5', '32d5643476452c7e70539663fe253ff2')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['CLARK --help'])])), ('about', CommentedMap([('home', 'http://clark.cs.ucr.edu/'), ('license', 'GPL3'), ('license_file', 'LICENSE_GNU_GPL.txt'), ('summary', 'Fast, accurate and versatile k-mer based classification system')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-goseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-goseq'), ('version', '1.26.0')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/bioconductor-goseq/bioconductor-goseq_1.26.0_src_all.tar.gz'), ('md5', 'da46e940cc1cf1f624a06c908f07f910')])), ('build', CommentedMap([('number', 0), ('skip', False), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genelendatabase', 'bioconductor-go.db', 'r-base', 'r-biasedurn', 'r-mgcv']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genelendatabase', 'bioconductor-go.db', 'r-base', 'r-biasedurn', 'r-mgcv'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'goseq\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/goseq.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Detects Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-goseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-goseq'), ('version', '1.26.0')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/bioconductor-goseq/bioconductor-goseq_1.26.0_src_all.tar.gz'), ('md5', 'da46e940cc1cf1f624a06c908f07f910')])), ('build', CommentedMap([('number', 0), ('skip', False), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genelendatabase', 'bioconductor-go.db', 'r-base', 'r-biasedurn', 'r-mgcv']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biocgenerics', 'bioconductor-genelendatabase', 'bioconductor-go.db', 'r-base', 'r-biasedurn', 'r-mgcv'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'goseq\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/goseq.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Detects Gene Ontology and/or other user defined categories which are over/under represented in RNA-seq data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gsva/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gsva'), ('version', '1.24.1')])), ('source', CommentedMap([('fn', 'GSVA_1.24.1.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-gsva/bioconductor-gsva_1.24.1_src_all.tar.gz'), ('md5', '93579e46cd3bc4f3a9db0b9bfa7a4921')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.17.4', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.17.4', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GSVA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GSVA.html'), ('license', 'GPL (>= 2)'), ('summary', 'Gene Set Variation Analysis (GSVA) is a non-parametric, unsupervised method for estimating variation of gene set enrichment through the samples of a expression data set. GSVA performs a change in coordinate systems, transforming the data from a gene by sample matrix to a gene-set by sample matrix, thereby allowing the evaluation of pathway enrichment for each sample. This new matrix of GSVA enrichment scores facilitates applying standard analytical methods like functional enrichment, survival analysis, clustering, CNV-pathway analysis or cross-tissue pathway analysis, in a pathway-centric manner.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gsva/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gsva'), ('version', '1.24.1')])), ('source', CommentedMap([('fn', 'GSVA_1.24.1.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-gsva/bioconductor-gsva_1.24.1_src_all.tar.gz'), ('md5', '93579e46cd3bc4f3a9db0b9bfa7a4921')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.17.4', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-gseabase >=1.17.4', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GSVA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GSVA.html'), ('license', 'GPL (>= 2)'), ('summary', 'Gene Set Variation Analysis (GSVA) is a non-parametric, unsupervised method for estimating variation of gene set enrichment through the samples of a expression data set. GSVA performs a change in coordinate systems, transforming the data from a gene by sample matrix to a gene-set by sample matrix, thereby allowing the evaluation of pathway enrichment for each sample. This new matrix of GSVA enrichment scores facilitates applying standard analytical methods like functional enrichment, survival analysis, clustering, CNV-pathway analysis or cross-tissue pathway analysis, in a pathway-centric manner.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-oscope/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-oscope'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Oscope_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Oscope_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/Oscope_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/Oscope/Oscope_1.6.0_src_all.tar.gz']), ('md5', 'ce210cd09760b511bddb8c1f12b56eb0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'bioconductor-ebseq', 'r-base', 'r-cluster', 'r-testthat']), ('run', ['bioconductor-biocparallel', 'bioconductor-ebseq', 'r-base', 'r-cluster', 'r-testthat'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Oscope\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Oscope.html'), ('license', 'Artistic-2.0'), ('summary', 'Oscope is a statistical pipeline developed to identifying and recovering the base cycle profiles of oscillating genes in an unsynchronized single cell RNA-seq experiment. The Oscope pipeline includes three modules: a sine model module to search for candidate oscillator pairs; a K-medoids clustering module to cluster candidate oscillators into groups; and an extended nearest insertion module to recover the base cycle order for each oscillator group.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-oscope/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-oscope'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Oscope_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Oscope_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/Oscope_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/Oscope/Oscope_1.6.0_src_all.tar.gz']), ('md5', 'ce210cd09760b511bddb8c1f12b56eb0')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'bioconductor-ebseq', 'r-base', 'r-cluster', 'r-testthat']), ('run', ['bioconductor-biocparallel', 'bioconductor-ebseq', 'r-base', 'r-cluster', 'r-testthat'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Oscope\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Oscope.html'), ('license', 'Artistic-2.0'), ('summary', 'Oscope is a statistical pipeline developed to identifying and recovering the base cycle profiles of oscillating genes in an unsynchronized single cell RNA-seq experiment. The Oscope pipeline includes three modules: a sine model module to search for candidate oscillator pairs; a K-medoids clustering module to cluster candidate oscillators into groups; and an extended nearest insertion module to recover the base cycle order for each oscillator group.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-siggenes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-siggenes'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'siggenes_1.46.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/siggenes_1.46.0.tar.gz'), ('md5', '106de15f38812cccfd5b3e20c3fb13fb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'siggenes\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/siggenes.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Identification of differentially expressed genes and estimation of the False Discovery Rate (FDR) using both the Significance Analysis of Microarrays (SAM) and the Empirical Bayes Analyses of Microarrays (EBAM).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-siggenes/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-siggenes'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'siggenes_1.46.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/siggenes_1.46.0.tar.gz'), ('md5', '106de15f38812cccfd5b3e20c3fb13fb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'siggenes\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/siggenes.html'), ('license', 'LGPL (>= 2)'), ('summary', 'Identification of differentially expressed genes and estimation of the False Discovery Rate (FDR) using both the Significance Analysis of Microarrays (SAM) and the Empirical Bayes Analyses of Microarrays (EBAM).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rpa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rpa'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'RPA_1.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RPA_1.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/RPA_1.32.0.tar.gz', 'https://depot.galaxyproject.org/software/RPA/RPA_1.32.0_src_all.tar.gz']), ('md5', '06e6dde0f7e06243e740837923498434')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biocgenerics', 'bioconductor-phyloseq', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-biocgenerics', 'bioconductor-phyloseq', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RPA.html'), ('license', 'BSD_2_clause + file LICENSE'), ('summary', 'Probabilistic analysis of probe reliability and differential gene expression on short oligonucleotide arrays. Lahti et al. \"Probabilistic Analysis of Probe Reliability in Differential Gene Expression Studies with Short Oligonucleotide Arrays\", TCBB/IEEE, 2011. http://doi.ieeecomputersociety.org/10.1109/TCBB.2009.38')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rpa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rpa'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'RPA_1.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RPA_1.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/RPA_1.32.0.tar.gz', 'https://depot.galaxyproject.org/software/RPA/RPA_1.32.0_src_all.tar.gz']), ('md5', '06e6dde0f7e06243e740837923498434')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biocgenerics', 'bioconductor-phyloseq', 'r-base']), ('run', ['bioconductor-affy', 'bioconductor-biocgenerics', 'bioconductor-phyloseq', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RPA.html'), ('license', 'BSD_2_clause + file LICENSE'), ('summary', 'Probabilistic analysis of probe reliability and differential gene expression on short oligonucleotide arrays. Lahti et al. \"Probabilistic Analysis of Probe Reliability in Differential Gene Expression Studies with Short Oligonucleotide Arrays\", TCBB/IEEE, 2011. http://doi.ieeecomputersociety.org/10.1109/TCBB.2009.38')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnits/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnits'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'Rnits_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Rnits_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/Rnits_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/Rnits/Rnits_1.10.0_src_all.tar.gz']), ('md5', 'b0c1ae5bb71e7210ff21b886de98a4a3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-impute', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-boot', 'r-ggplot2', 'r-reshape2']), ('run', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-impute', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-boot', 'r-ggplot2', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rnits\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Rnits.html'), ('license', 'GPL-3'), ('summary', 'R/Bioconductor package for normalization, curve registration and inference in time course gene expression data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnits/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnits'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'Rnits_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Rnits_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/Rnits_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/Rnits/Rnits_1.10.0_src_all.tar.gz']), ('md5', 'b0c1ae5bb71e7210ff21b886de98a4a3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-impute', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-boot', 'r-ggplot2', 'r-reshape2']), ('run', ['bioconductor-affy', 'bioconductor-biobase', 'bioconductor-impute', 'bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-boot', 'r-ggplot2', 'r-reshape2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rnits\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Rnits.html'), ('license', 'GPL-3'), ('summary', 'R/Bioconductor package for normalization, curve registration and inference in time course gene expression data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-trio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-trio'), ('version', '3.14.0')])), ('source', CommentedMap([('fn', 'trio_3.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/trio_3.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/trio_3.14.0.tar.gz', 'https://depot.galaxyproject.org/software/trio/trio_3.14.0_src_all.tar.gz']), ('md5', 'c9c8042544db35d7a6a2c21f462a01df')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'trio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/trio.html'), ('license', 'LGPL-2'), ('summary', 'Testing SNPs and SNP interactions with a genotypic TDT. This package furthermore contains functions for computing pairwise values of LD measures and for identifying LD blocks, as well as functions for setting up matched case pseudo-control genotype data for case-parent trios in order to run trio logic regression, for imputing missing genotypes in trios, for simulating case-parent trios with disease risk dependent on SNP interaction, and for power and sample size calculation in trio data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-trio/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-trio'), ('version', '3.14.0')])), ('source', CommentedMap([('fn', 'trio_3.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/trio_3.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/trio_3.14.0.tar.gz', 'https://depot.galaxyproject.org/software/trio/trio_3.14.0_src_all.tar.gz']), ('md5', 'c9c8042544db35d7a6a2c21f462a01df')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'trio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/trio.html'), ('license', 'LGPL-2'), ('summary', 'Testing SNPs and SNP interactions with a genotypic TDT. This package furthermore contains functions for computing pairwise values of LD measures and for identifying LD blocks, as well as functions for setting up matched case pseudo-control genotype data for case-parent trios in order to run trio logic regression, for imputing missing genotypes in trios, for simulating case-parent trios with disease risk dependent on SNP interaction, and for power and sample size calculation in trio data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocviews/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocviews'), ('version', '1.44.0')])), ('source', CommentedMap([('fn', 'biocViews_1.44.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/biocViews_1.44.0.tar.gz', 'https://bioarchive.galaxyproject.org/biocViews_1.44.0.tar.gz', 'https://depot.galaxyproject.org/software/biocViews/biocViews_1.44.0_src_all.tar.gz']), ('md5', '6e34b966aee2fd4fc808c40a88fa588a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph >=1.9.26', 'bioconductor-rbgl >=1.13.5', 'r-base >=2.4.0', 'readline 6.2', 'r-rcurl', 'r-runit', 'r-xml']), ('run', ['bioconductor-biobase', 'bioconductor-graph >=1.9.26', 'bioconductor-rbgl >=1.13.5', 'r-base >=2.4.0', 'readline 6.2', 'r-rcurl', 'r-runit', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biocViews\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/biocViews.html'), ('license', 'Artistic-2.0'), ('summary', \"Infrastructure to support Bioconductor 'views' used to classify software packages. 'biocViews' are directed acyclic graphs of terms from a controlled vocabulary. There are three major classifications, corresponding to 'software', 'annotation', and 'experiment data' packages.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocviews/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocviews'), ('version', '1.44.0')])), ('source', CommentedMap([('fn', 'biocViews_1.44.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/biocViews_1.44.0.tar.gz', 'https://bioarchive.galaxyproject.org/biocViews_1.44.0.tar.gz', 'https://depot.galaxyproject.org/software/biocViews/biocViews_1.44.0_src_all.tar.gz']), ('md5', '6e34b966aee2fd4fc808c40a88fa588a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-graph >=1.9.26', 'bioconductor-rbgl >=1.13.5', 'r-base >=2.4.0', 'readline 6.2', 'r-rcurl', 'r-runit', 'r-xml']), ('run', ['bioconductor-biobase', 'bioconductor-graph >=1.9.26', 'bioconductor-rbgl >=1.13.5', 'r-base >=2.4.0', 'readline 6.2', 'r-rcurl', 'r-runit', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'biocViews\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/biocViews.html'), ('license', 'Artistic-2.0'), ('summary', \"Infrastructure to support Bioconductor 'views' used to classify software packages. 'biocViews' are directed acyclic graphs of terms from a controlled vocabulary. There are three major classifications, corresponding to 'software', 'annotation', and 'experiment data' packages.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cntools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cntools'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'CNTools_1.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CNTools_1.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/CNTools_1.32.0.tar.gz', 'https://depot.galaxyproject.org/software/CNTools/CNTools_1.32.0_src_all.tar.gz']), ('md5', '52790802246df29a86b624aed44eb7c6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genefilter', 'r-base']), ('run', ['bioconductor-genefilter', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CNTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CNTools.html'), ('license', 'LGPL'), ('summary', 'This package provides tools to convert the output of segmentation analysis using DNAcopy to a matrix structure with overlapping segments as rows and samples as columns so that other computational analyses can be applied to segmented data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cntools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cntools'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'CNTools_1.32.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CNTools_1.32.0.tar.gz', 'https://bioarchive.galaxyproject.org/CNTools_1.32.0.tar.gz', 'https://depot.galaxyproject.org/software/CNTools/CNTools_1.32.0_src_all.tar.gz']), ('md5', '52790802246df29a86b624aed44eb7c6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genefilter', 'r-base']), ('run', ['bioconductor-genefilter', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CNTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CNTools.html'), ('license', 'LGPL'), ('summary', 'This package provides tools to convert the output of segmentation analysis using DNAcopy to a matrix structure with overlapping segments as rows and samples as columns so that other computational analyses can be applied to segmented data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-path2ppi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-path2ppi'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Path2PPI_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Path2PPI_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/Path2PPI/Path2PPI_1.6.0_src_all.tar.gz']), ('md5', 'f1090e5beb965dda232dc020ed82278c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph >=1.0.1']), ('run', ['r-base', 'r-igraph >=1.0.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Path2PPI\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Path2PPI.html'), ('license', 'GPL (>= 2)'), ('summary', 'Package to predict protein-protein interaction (PPI) networks in target organisms for which only a view information about PPIs is available. Path2PPI predicts PPI networks based on sets of proteins which can belong to a certain pathway from well-established model organisms. It helps to combine and transfer information of a certain pathway or biological process from several reference organisms to one target organism. Path2PPI only depends on the sequence similarity of the involved proteins.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-path2ppi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-path2ppi'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'Path2PPI_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/Path2PPI_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/Path2PPI/Path2PPI_1.6.0_src_all.tar.gz']), ('md5', 'f1090e5beb965dda232dc020ed82278c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph >=1.0.1']), ('run', ['r-base', 'r-igraph >=1.0.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Path2PPI\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/Path2PPI.html'), ('license', 'GPL (>= 2)'), ('summary', 'Package to predict protein-protein interaction (PPI) networks in target organisms for which only a view information about PPIs is available. Path2PPI predicts PPI networks based on sets of proteins which can belong to a certain pathway from well-established model organisms. It helps to combine and transfer information of a certain pathway or biological process from several reference organisms to one target organism. Path2PPI only depends on the sequence similarity of the involved proteins.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-s4vectors/0.8.11/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-s4vectors'), ('version', '0.8.11')])), ('source', CommentedMap([('fn', 'S4Vectors_0.8.11.tar.gz'), ('url', 'http://bioconductor.org/packages/3.2/bioc/src/contrib/S4Vectors_0.8.11.tar.gz'), ('md5', '02b20a992bf46bb2cbdef8c90d76b421')])), ('build', CommentedMap([('skip', False), ('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.15.10', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.15.10', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'S4Vectors\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/S4Vectors.html'), ('license', 'Artistic-2.0'), ('summary', 'The S4Vectors package defines the Vector and List virtual classes and a set of generic functions that extend the semantic of ordinary vectors and lists in R. Package developers can easily implement vector-like or list-like objects as concrete subclasses of Vector or List. In addition, a few low-level concrete subclasses of general interest (e.g. DataFrame, Rle, and Hits) are implemented in the S4Vectors package itself (many more are implemented in the IRanges package and in other Bioconductor infrastructure packages).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-s4vectors/0.8.11/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-s4vectors'), ('version', '0.8.11')])), ('source', CommentedMap([('fn', 'S4Vectors_0.8.11.tar.gz'), ('url', 'http://bioconductor.org/packages/3.2/bioc/src/contrib/S4Vectors_0.8.11.tar.gz'), ('md5', '02b20a992bf46bb2cbdef8c90d76b421')])), ('build', CommentedMap([('skip', False), ('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.15.10', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.15.10', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'S4Vectors\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/S4Vectors.html'), ('license', 'Artistic-2.0'), ('summary', 'The S4Vectors package defines the Vector and List virtual classes and a set of generic functions that extend the semantic of ordinary vectors and lists in R. Package developers can easily implement vector-like or list-like objects as concrete subclasses of Vector or List. In addition, a few low-level concrete subclasses of general interest (e.g. DataFrame, Rle, and Hits) are implemented in the S4Vectors package itself (many more are implemented in the IRanges package and in other Bioconductor infrastructure packages).')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/quake/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://www.cbcb.umd.edu/software/quake/'), ('summary', 'Quake is a package to correct substitution sequencing errors in experiments with deep coverage (e.g. >15X), specifically intended for Illumina sequencing reads')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://www.cbcb.umd.edu/software/quake/downloads/<{ name }}-<{ version }}.tar.gz'), ('md5', 'f8e26c0c29d22fc0d49fcfb5dc8945d3')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*', 'jellyfish', 'r-base', 'r-vgam', 'python'])])), ('test', CommentedMap([('commands', ['quake.py --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/quake/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('about', CommentedMap([('home', 'http://www.cbcb.umd.edu/software/quake/'), ('summary', 'Quake is a package to correct substitution sequencing errors in experiments with deep coverage (e.g. >15X), specifically intended for Illumina sequencing reads')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://www.cbcb.umd.edu/software/quake/downloads/<{ name }}-<{ version }}.tar.gz'), ('md5', 'f8e26c0c29d22fc0d49fcfb5dc8945d3')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['gcc', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*', 'jellyfish', 'r-base', 'r-vgam', 'python'])])), ('test', CommentedMap([('commands', ['quake.py --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomicalignments/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomicalignments'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'GenomicAlignments_1.10.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-genomicalignments/bioconductor-genomicalignments_1.10.0_src_all.tar.gz'), ('md5', '6e6af96adcad077abdb5889e5620cd6a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenomicAlignments\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GenomicAlignments.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides efficient containers for storing and manipulating short genomic alignments (typically obtained by aligning short reads to a reference genome). This includes read counting, computing the coverage, junction detection, and working with the nucleotide content of the alignments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genomicalignments/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genomicalignments'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'GenomicAlignments_1.10.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-genomicalignments/bioconductor-genomicalignments_1.10.0_src_all.tar.gz'), ('md5', '6e6af96adcad077abdb5889e5620cd6a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GenomicAlignments\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GenomicAlignments.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides efficient containers for storing and manipulating short genomic alignments (typically obtained by aligning short reads to a reference genome). This includes read counting, computing the coverage, junction detection, and working with the nucleotide content of the alignments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mergemaid/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mergemaid'), ('version', '2.48.0')])), ('source', CommentedMap([('fn', 'MergeMaid_2.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MergeMaid_2.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/MergeMaid_2.48.0.tar.gz', 'https://depot.galaxyproject.org/software/MergeMaid/MergeMaid_2.48.0_src_all.tar.gz']), ('md5', 'afc57533d7efaa0c17490ff86f2d4b29')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-mass', 'r-survival']), ('run', ['bioconductor-biobase', 'r-base', 'r-mass', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MergeMaid\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MergeMaid.html'), ('license', 'GPL (>= 2)'), ('summary', \"The functions in this R extension are intended for cross-study comparison of gene expression array data. Required from the user is gene expression matrices, their corresponding gene-id vectors and other useful information, and they could be 'list','matrix', or 'ExpressionSet'. The main function is 'mergeExprs' which transforms the input objects into data in the merged format, such that common genes in different datasets can be easily found. And the function 'intcor' calculate the correlation coefficients. Other functions use the output from 'modelOutcome' to graphically display the results and cross-validate associations of gene expression data with survival.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mergemaid/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mergemaid'), ('version', '2.48.0')])), ('source', CommentedMap([('fn', 'MergeMaid_2.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MergeMaid_2.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/MergeMaid_2.48.0.tar.gz', 'https://depot.galaxyproject.org/software/MergeMaid/MergeMaid_2.48.0_src_all.tar.gz']), ('md5', 'afc57533d7efaa0c17490ff86f2d4b29')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-mass', 'r-survival']), ('run', ['bioconductor-biobase', 'r-base', 'r-mass', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MergeMaid\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MergeMaid.html'), ('license', 'GPL (>= 2)'), ('summary', \"The functions in this R extension are intended for cross-study comparison of gene expression array data. Required from the user is gene expression matrices, their corresponding gene-id vectors and other useful information, and they could be 'list','matrix', or 'ExpressionSet'. The main function is 'mergeExprs' which transforms the input objects into data in the merged format, such that common genes in different datasets can be easily found. And the function 'intcor' calculate the correlation coefficients. Other functions use the output from 'modelOutcome' to graphically display the results and cross-validate associations of gene expression data with survival.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/wham/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'wham'), ('version', '1.7.0.311')])), ('source', CommentedMap([('git_url', 'https://github.com/zeeev/wham.git'), ('git_rev', '4e8c9bee568211596cdc634d15600114e75d2619'), ('md5', 'unneeded')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'git']), ('run', ['libgcc'])])), ('test', None), ('about', CommentedMap([('home', 'https://github.com/jewmanchue/wham'), ('license', 'MIT'), ('summary', 'Structural variant detection and association testing')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/wham/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'wham'), ('version', '1.7.0.311')])), ('source', CommentedMap([('git_url', 'https://github.com/zeeev/wham.git'), ('git_rev', '4e8c9bee568211596cdc634d15600114e75d2619'), ('md5', 'unneeded')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['cmake', 'gcc', 'git']), ('run', ['libgcc'])])), ('test', None), ('about', CommentedMap([('home', 'https://github.com/jewmanchue/wham'), ('license', 'MIT'), ('summary', 'Structural variant detection and association testing')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/verifybamid/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}LibStatGen.<{ version }}.tgz'), ('url', 'https://github.com/statgen/<{ name }}/releases/download/v<{ version }}/<{ name }}LibStatGen.<{ version }}.tgz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['verifyBamID -h 2>&1 | grep -q verifyBamID'])])), ('about', CommentedMap([('home', 'http://genome.sph.umich.edu/wiki/VerifyBamID'), ('license', 'GPL3'), ('summary', 'verifyBamID verifies identity and purity of sequence data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/verifybamid/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name }}LibStatGen.<{ version }}.tgz'), ('url', 'https://github.com/statgen/<{ name }}/releases/download/v<{ version }}/<{ name }}LibStatGen.<{ version }}.tgz'), ('md5', '<{ md5 }}')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('test', CommentedMap([('commands', ['verifyBamID -h 2>&1 | grep -q verifyBamID'])])), ('about', CommentedMap([('home', 'http://genome.sph.umich.edu/wiki/VerifyBamID'), ('license', 'GPL3'), ('summary', 'verifyBamID verifies identity and purity of sequence data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/limix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'limix'), ('version', '0.7.12')])), ('source', CommentedMap([('fn', 'limix-0.7.12.tar.gz'), ('url', 'https://pypi.python.org/packages/source/l/limix/limix-0.7.12.tar.gz'), ('md5', '4437c627f35afade5b4c62fcfb65d995')])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'gcc', 'setuptools', 'cython', 'h5py', 'numpy', 'scipy']), ('run', ['python >=2.7,<3', 'gcc', 'cython', 'numpy', 'scipy', 'h5py'])])), ('build', CommentedMap([('skip', True)])), ('test', CommentedMap([('imports', ['limix', 'limix.core', 'limix.core.association', 'limix.core.covar', 'limix.core.covar.test', 'limix.core.covar.util', 'limix.core.gp', 'limix.core.likelihood', 'limix.core.linalg', 'limix.core.mean', 'limix.core.optimize', 'limix.core.test', 'limix.core.type', 'limix.core.utils', 'limix.deprecated', 'limix.deprecated.io', 'limix.deprecated.modules', 'limix.deprecated.stats', 'limix.deprecated.utils', 'limix.ensemble', 'limix.io', 'limix.io_', 'limix.modules', 'limix.mtSet', 'limix.mtSet.core', 'limix.stats', 'limix.utils'])])), ('about', CommentedMap([('home', 'http://pmbio.github.io/limix/'), ('license', 'BSD'), ('summary', 'A flexible and fast mixed model toolbox written in C++/python')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/limix/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'limix'), ('version', '0.7.12')])), ('source', CommentedMap([('fn', 'limix-0.7.12.tar.gz'), ('url', 'https://pypi.python.org/packages/source/l/limix/limix-0.7.12.tar.gz'), ('md5', '4437c627f35afade5b4c62fcfb65d995')])), ('requirements', CommentedMap([('build', ['python >=2.7,<3', 'gcc', 'setuptools', 'cython', 'h5py', 'numpy', 'scipy']), ('run', ['python >=2.7,<3', 'gcc', 'cython', 'numpy', 'scipy', 'h5py'])])), ('build', CommentedMap([('skip', True)])), ('test', CommentedMap([('imports', ['limix', 'limix.core', 'limix.core.association', 'limix.core.covar', 'limix.core.covar.test', 'limix.core.covar.util', 'limix.core.gp', 'limix.core.likelihood', 'limix.core.linalg', 'limix.core.mean', 'limix.core.optimize', 'limix.core.test', 'limix.core.type', 'limix.core.utils', 'limix.deprecated', 'limix.deprecated.io', 'limix.deprecated.modules', 'limix.deprecated.stats', 'limix.deprecated.utils', 'limix.ensemble', 'limix.io', 'limix.io_', 'limix.modules', 'limix.mtSet', 'limix.mtSet.core', 'limix.stats', 'limix.utils'])])), ('about', CommentedMap([('home', 'http://pmbio.github.io/limix/'), ('license', 'BSD'), ('summary', 'A flexible and fast mixed model toolbox written in C++/python')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/smalt/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'smalt'), ('version', '0.7.6')])), ('source', CommentedMap([('fn', 'smalt-0.7.6-static.tar.gz'), ('url', 'https://sourceforge.net/projects/smalt/files/smalt-0.7.6-static.tar.gz'), ('md5', '57ed011836fda8fb5e34604dd1858b5e')])), ('requirements', CommentedMap([('build', ['gcc', 'libgcc', 'libtool', 'pkg-config', 'bambamc', 'zlib', 'python >=2.6,<3', 'samtools']), ('run', ['bambamc', 'zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['smalt version 2>&1 | grep \"SMALT - Sequence Mapping and Alignment Tool\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.sanger.ac.uk/science/tools/smalt-0'), ('license', 'GPLv3'), ('summary', 'SMALT aligns DNA sequencing reads with a reference genome.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/smalt/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'smalt'), ('version', '0.7.6')])), ('source', CommentedMap([('fn', 'smalt-0.7.6-static.tar.gz'), ('url', 'https://sourceforge.net/projects/smalt/files/smalt-0.7.6-static.tar.gz'), ('md5', '57ed011836fda8fb5e34604dd1858b5e')])), ('requirements', CommentedMap([('build', ['gcc', 'libgcc', 'libtool', 'pkg-config', 'bambamc', 'zlib', 'python >=2.6,<3', 'samtools']), ('run', ['bambamc', 'zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['smalt version 2>&1 | grep \"SMALT - Sequence Mapping and Alignment Tool\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.sanger.ac.uk/science/tools/smalt-0'), ('license', 'GPLv3'), ('summary', 'SMALT aligns DNA sequencing reads with a reference genome.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-weaver/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-weaver'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'weaver_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/weaver_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/weaver_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/weaver/weaver_1.42.0_src_all.tar.gz']), ('md5', 'b7f8eeadaa00abc9e9b56b27af2bd1f6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-codetools', 'r-digest']), ('run', ['r-base', 'r-codetools', 'r-digest'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'weaver\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/weaver.html'), ('license', 'GPL-2'), ('summary', 'This package provides enhancements on the Sweave() function in the base package.  In particular a facility for caching code chunk results is included.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-weaver/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-weaver'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'weaver_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/weaver_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/weaver_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/weaver/weaver_1.42.0_src_all.tar.gz']), ('md5', 'b7f8eeadaa00abc9e9b56b27af2bd1f6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-codetools', 'r-digest']), ('run', ['r-base', 'r-codetools', 'r-digest'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'weaver\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/weaver.html'), ('license', 'GPL-2'), ('summary', 'This package provides enhancements on the Sweave() function in the base package.  In particular a facility for caching code chunk results is included.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-asgsca/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-asgsca'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'ASGSCA_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASGSCA_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/ASGSCA_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/ASGSCA/ASGSCA_1.10.0_src_all.tar.gz']), ('md5', '66e1202d89d90a8e4752c5ddb5fb87fb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass', 'r-matrix']), ('run', ['r-base', 'r-mass', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASGSCA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASGSCA.html'), ('license', 'GPL-3'), ('summary', 'The package provides tools to model and test the association between multiple genotypes and multiple traits, taking into account the prior biological knowledge. Genes, and clinical pathways are incorporated in the model as latent variables. The method is based on Generalized Structured Component Analysis (GSCA).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-asgsca/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-asgsca'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'ASGSCA_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/ASGSCA_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/ASGSCA_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/ASGSCA/ASGSCA_1.10.0_src_all.tar.gz']), ('md5', '66e1202d89d90a8e4752c5ddb5fb87fb')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass', 'r-matrix']), ('run', ['r-base', 'r-mass', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ASGSCA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/ASGSCA.html'), ('license', 'GPL-3'), ('summary', 'The package provides tools to model and test the association between multiple genotypes and multiple traits, taking into account the prior biological knowledge. Genes, and clinical pathways are incorporated in the model as latent variables. The method is based on Generalized Structured Component Analysis (GSCA).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dexseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dexseq'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'DEXSeq_<{ version }}.tar.gz'), ('url', 'https://bioconductor.org/packages/release/bioc/src/contrib/DEXSeq_<{ version }}.tar.gz'), ('md5', '37baa66bb10474b43d4ea96594330911')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biomart', 'bioconductor-deseq2 >=1.14.1', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'bioconductor-genomicranges >=1.23.7', 'bioconductor-iranges >=2.5.17', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-hwriter', 'r-rcolorbrewer', 'r-statmod', 'r-stringr']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biomart', 'bioconductor-deseq2 >=1.14.1', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'bioconductor-genomicranges >=1.23.7', 'bioconductor-iranges >=2.5.17', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-hwriter', 'r-rcolorbrewer', 'r-statmod', 'r-stringr', 'python'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DEXSeq\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/DEXSeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dexseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dexseq'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'DEXSeq_<{ version }}.tar.gz'), ('url', 'https://bioconductor.org/packages/release/bioc/src/contrib/DEXSeq_<{ version }}.tar.gz'), ('md5', '37baa66bb10474b43d4ea96594330911')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biomart', 'bioconductor-deseq2 >=1.14.1', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'bioconductor-genomicranges >=1.23.7', 'bioconductor-iranges >=2.5.17', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-hwriter', 'r-rcolorbrewer', 'r-statmod', 'r-stringr']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biomart', 'bioconductor-deseq2 >=1.14.1', 'bioconductor-genefilter', 'bioconductor-geneplotter', 'bioconductor-genomicranges >=1.23.7', 'bioconductor-iranges >=2.5.17', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-hwriter', 'r-rcolorbrewer', 'r-statmod', 'r-stringr', 'python'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DEXSeq\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/DEXSeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'The package is focused on finding differential exon usage using RNA-seq exon counts between samples with different experimental designs. It provides functions that allows the user to make the necessary statistical tests based on a model that uses the negative binomial distribution to estimate the variance between biological replicates and generalized linear models for testing. The package also provides functions for the visualization and exploration of the results.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/repeatmodeler/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.repeatmasker.org/RepeatModeler.html'), ('license', 'Open Software License v2.1'), ('summary', 'RepeatModeler is a de-novo repeat family identification and modeling package.')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'repeatmodeler'), ('version', '1.0.8')])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl', 'repeatmasker', 'rmblast', 'trf', 'repeatscout', 'recon'])])), ('source', CommentedMap([('fn', 'RepeatModeler-open-1-0-8.tar.gz'), ('url', 'http://www.repeatmasker.org/RepeatModeler-open-1-0-8.tar.gz'), ('patches', ['abs_path.patch'])])), ('test', CommentedMap([('commands', ['RepeatModeler > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/repeatmodeler/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://www.repeatmasker.org/RepeatModeler.html'), ('license', 'Open Software License v2.1'), ('summary', 'RepeatModeler is a de-novo repeat family identification and modeling package.')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'repeatmodeler'), ('version', '1.0.8')])), ('requirements', CommentedMap([('build', ['perl']), ('run', ['perl', 'repeatmasker', 'rmblast', 'trf', 'repeatscout', 'recon'])])), ('source', CommentedMap([('fn', 'RepeatModeler-open-1-0-8.tar.gz'), ('url', 'http://www.repeatmasker.org/RepeatModeler-open-1-0-8.tar.gz'), ('patches', ['abs_path.patch'])])), ('test', CommentedMap([('commands', ['RepeatModeler > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-scan.upc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-scan.upc'), ('version', '2.18.0')])), ('source', CommentedMap([('fn', 'SCAN.UPC_2.18.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/SCAN.UPC_2.18.0.tar.gz'), ('md5', '26993908ea71b6f444aac156ca972b1d')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-foreach', 'bioconductor-biobase >=2.6.0', 'bioconductor-oligo', 'bioconductor-biostrings', 'bioconductor-geoquery', 'bioconductor-sva', 'bioconductor-affy', 'bioconductor-affyio', 'r-dbi', 'r-rsqlite', 'r-mass']), ('run', ['r-base', 'r-foreach', 'bioconductor-biobase >=2.6.0', 'bioconductor-oligo', 'bioconductor-biostrings', 'bioconductor-geoquery', 'bioconductor-sva', 'bioconductor-affy', 'bioconductor-affyio', 'r-dbi', 'r-rsqlite', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SCAN.UPC\\')\"'])])), ('about', CommentedMap([('license', 'MIT'), ('home', 'http://bioconductor.org/packages/release/bioc/html/SCAN.UPC.html'), ('summary', 'SCAN is a microarray normalization method to facilitate personalized-medicine workflows.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-scan.upc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-scan.upc'), ('version', '2.18.0')])), ('source', CommentedMap([('fn', 'SCAN.UPC_2.18.0.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/SCAN.UPC_2.18.0.tar.gz'), ('md5', '26993908ea71b6f444aac156ca972b1d')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-foreach', 'bioconductor-biobase >=2.6.0', 'bioconductor-oligo', 'bioconductor-biostrings', 'bioconductor-geoquery', 'bioconductor-sva', 'bioconductor-affy', 'bioconductor-affyio', 'r-dbi', 'r-rsqlite', 'r-mass']), ('run', ['r-base', 'r-foreach', 'bioconductor-biobase >=2.6.0', 'bioconductor-oligo', 'bioconductor-biostrings', 'bioconductor-geoquery', 'bioconductor-sva', 'bioconductor-affy', 'bioconductor-affyio', 'r-dbi', 'r-rsqlite', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SCAN.UPC\\')\"'])])), ('about', CommentedMap([('license', 'MIT'), ('home', 'http://bioconductor.org/packages/release/bioc/html/SCAN.UPC.html'), ('summary', 'SCAN is a microarray normalization method to facilitate personalized-medicine workflows.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-wgcna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-wgcna'), ('version', '1.51')])), ('source', CommentedMap([('fn', 'WGCNA_1.51.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/WGCNA_1.51.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/WGCNA/WGCNA_1.51.tar.gz']), ('sha256', 'bd6be9ea2a71a41cd758b9ae21065afc7e5d1870c88c8ee8428f6f7338b4fb43')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'r-hmisc', 'r-doparallel', 'r-dynamictreecut >=1.62', 'r-fastcluster', 'r-foreach', 'bioconductor-impute', 'r-matrixstats >=0.8.1', 'bioconductor-preprocesscore', 'r-survival', 'posix', '<{native}}toolchain', 'gcc']), ('run', ['r-base', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'r-hmisc', 'r-doparallel', 'r-dynamictreecut >=1.62', 'r-fastcluster', 'r-foreach', 'bioconductor-impute', 'r-matrixstats >=0.8.1', 'bioconductor-preprocesscore', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'WGCNA\\')\"', '\"%R%\" -e \"library(\\'WGCNA\\')\"'])])), ('about', CommentedMap([('home', 'http://www.genetics.ucla.edu/labs/horvath/CoexpressionNetwork/Rpackages/WGCNA/'), ('license', 'GPL (>= 2)'), ('summary', 'Functions necessary to perform Weighted Correlation Network Analysis on high-dimensional data. Includes functions for rudimentary data cleaning, construction of correlation networks, module identification, summarization, and relating of variables and modules to sample traits. Also includes a number of utility functions for data manipulation and visualization.'), ('license_family', 'GPL3')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-wgcna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-wgcna'), ('version', '1.51')])), ('source', CommentedMap([('fn', 'WGCNA_1.51.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/WGCNA_1.51.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/WGCNA/WGCNA_1.51.tar.gz']), ('sha256', 'bd6be9ea2a71a41cd758b9ae21065afc7e5d1870c88c8ee8428f6f7338b4fb43')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'r-hmisc', 'r-doparallel', 'r-dynamictreecut >=1.62', 'r-fastcluster', 'r-foreach', 'bioconductor-impute', 'r-matrixstats >=0.8.1', 'bioconductor-preprocesscore', 'r-survival', 'posix', '<{native}}toolchain', 'gcc']), ('run', ['r-base', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'r-hmisc', 'r-doparallel', 'r-dynamictreecut >=1.62', 'r-fastcluster', 'r-foreach', 'bioconductor-impute', 'r-matrixstats >=0.8.1', 'bioconductor-preprocesscore', 'r-survival'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'WGCNA\\')\"', '\"%R%\" -e \"library(\\'WGCNA\\')\"'])])), ('about', CommentedMap([('home', 'http://www.genetics.ucla.edu/labs/horvath/CoexpressionNetwork/Rpackages/WGCNA/'), ('license', 'GPL (>= 2)'), ('summary', 'Functions necessary to perform Weighted Correlation Network Analysis on high-dimensional data. Includes functions for rudimentary data cleaning, construction of correlation networks, module identification, summarization, and relating of variables and modules to sample traits. Also includes a number of utility functions for data manipulation and visualization.'), ('license_family', 'GPL3')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mirage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mirage'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'MiRaGE_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MiRaGE_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/MiRaGE_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/MiRaGE/MiRaGE_1.18.0_src_all.tar.gz']), ('md5', 'ca996ba9edb5d1f21674b67bc63878d7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase >=2.23.3', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase >=2.23.3', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MiRaGE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MiRaGE.html'), ('license', 'GPL'), ('summary', 'The package contains functions for inferece of target gene regulation by miRNA, based on only target gene expression profile.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mirage/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mirage'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'MiRaGE_1.18.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MiRaGE_1.18.0.tar.gz', 'https://bioarchive.galaxyproject.org/MiRaGE_1.18.0.tar.gz', 'https://depot.galaxyproject.org/software/MiRaGE/MiRaGE_1.18.0_src_all.tar.gz']), ('md5', 'ca996ba9edb5d1f21674b67bc63878d7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase >=2.23.3', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase >=2.23.3', 'bioconductor-biocgenerics', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MiRaGE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MiRaGE.html'), ('license', 'GPL'), ('summary', 'The package contains functions for inferece of target gene regulation by miRNA, based on only target gene expression profile.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cner'), ('version', '1.10.2')])), ('source', CommentedMap([('fn', 'CNEr_1.10.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/CNEr_1.10.2.tar.gz', 'https://depot.galaxyproject.org/software/CNEr/CNEr_1.10.2_src_all.tar.gz']), ('sha256', '31160b6b3a4711199c315884335080c876cdea21e46c6a6ff689bf0ad48fe28b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate >=1.50.0', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.4', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.9', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-go.db >=3.3.0', 'bioconductor-iranges >=2.5.27', 'bioconductor-keggrest >=1.14.0', 'bioconductor-rtracklayer >=1.25.5', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.4', 'r-base', 'r-ggplot2 >=2.1.0', 'r-powerlaw >=0.60.3', 'r-r.utils >=2.3.0', 'r-readr >=0.2.2', 'r-reshape2 >=1.4.1', 'r-rsqlite >=0.11.4']), ('run', ['bioconductor-annotate >=1.50.0', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.4', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.9', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-go.db >=3.3.0', 'bioconductor-iranges >=2.5.27', 'bioconductor-keggrest >=1.14.0', 'bioconductor-rtracklayer >=1.25.5', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.4', 'r-base', 'r-ggplot2 >=2.1.0', 'r-powerlaw >=0.60.3', 'r-r.utils >=2.3.0', 'r-readr >=0.2.2', 'r-reshape2 >=1.4.1', 'r-rsqlite >=0.11.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CNEr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/CNEr.html'), ('license', 'GPL-2 | file LICENSE'), ('summary', 'Large-scale identification and advanced visualization of sets of conserved noncoding elements.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cner/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cner'), ('version', '1.10.2')])), ('source', CommentedMap([('fn', 'CNEr_1.10.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/CNEr_1.10.2.tar.gz', 'https://depot.galaxyproject.org/software/CNEr/CNEr_1.10.2_src_all.tar.gz']), ('sha256', '31160b6b3a4711199c315884335080c876cdea21e46c6a6ff689bf0ad48fe28b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate >=1.50.0', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.4', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.9', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-go.db >=3.3.0', 'bioconductor-iranges >=2.5.27', 'bioconductor-keggrest >=1.14.0', 'bioconductor-rtracklayer >=1.25.5', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.4', 'r-base', 'r-ggplot2 >=2.1.0', 'r-powerlaw >=0.60.3', 'r-r.utils >=2.3.0', 'r-readr >=0.2.2', 'r-reshape2 >=1.4.1', 'r-rsqlite >=0.11.4']), ('run', ['bioconductor-annotate >=1.50.0', 'bioconductor-biocgenerics', 'bioconductor-biostrings >=2.33.4', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.9', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-go.db >=3.3.0', 'bioconductor-iranges >=2.5.27', 'bioconductor-keggrest >=1.14.0', 'bioconductor-rtracklayer >=1.25.5', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.4', 'r-base', 'r-ggplot2 >=2.1.0', 'r-powerlaw >=0.60.3', 'r-r.utils >=2.3.0', 'r-readr >=0.2.2', 'r-reshape2 >=1.4.1', 'r-rsqlite >=0.11.4'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CNEr\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/CNEr.html'), ('license', 'GPL-2 | file LICENSE'), ('summary', 'Large-scale identification and advanced visualization of sets of conserved noncoding elements.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-casper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-casper'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'casper_2.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/casper_2.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/casper_2.10.0.tar.gz', 'https://depot.galaxyproject.org/software/casper/casper_2.10.0_src_all.tar.gz']), ('md5', 'fa3ed4940649987086225eafec4412d4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-ebarrays', 'bioconductor-gaga', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-coda', 'r-gtools', 'r-mgcv', 'r-sqldf', 'r-survival', 'r-vgam']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-ebarrays', 'bioconductor-gaga', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-coda', 'r-gtools', 'r-mgcv', 'r-sqldf', 'r-survival', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'casper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/casper.html'), ('license', 'GPL (>=2)'), ('summary', 'Infer alternative splicing from paired-end RNA-seq data. The model is based on counting paths across exons, rather than pairwise exon connections, and estimates the fragment size and start distributions non-parametrically, which improves estimation precision.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-casper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-casper'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'casper_2.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/casper_2.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/casper_2.10.0.tar.gz', 'https://depot.galaxyproject.org/software/casper/casper_2.10.0_src_all.tar.gz']), ('md5', 'fa3ed4940649987086225eafec4412d4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-ebarrays', 'bioconductor-gaga', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-coda', 'r-gtools', 'r-mgcv', 'r-sqldf', 'r-survival', 'r-vgam']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-ebarrays', 'bioconductor-gaga', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-coda', 'r-gtools', 'r-mgcv', 'r-sqldf', 'r-survival', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'casper\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/casper.html'), ('license', 'GPL (>=2)'), ('summary', 'Infer alternative splicing from paired-end RNA-seq data. The model is based on counting paths across exons, rather than pairwise exon connections, and estimates the fragment size and start distributions non-parametrically, which improves estimation precision.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/impute2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'impute2'), ('version', '2.3.2')])), ('source', CommentedMap([('fn', 'impute_v2.3.2.tgz'), ('url', 'https://mathgen.stats.ox.ac.uk/impute/impute_v2.3.2_MacOSX_Intel.tgz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['impute2 2>&1 | grep IMPUTE'])])), ('about', CommentedMap([('home', 'https://mathgen.stats.ox.ac.uk/impute/impute_v2.html'), ('license', 'Academic use, commerically restricted (http://www.stats.ox.ac.uk/~marchini/software/gwas/gwas.html#licence)'), ('summary', 'Genotype imputation and haplotype phasing')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/impute2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'impute2'), ('version', '2.3.2')])), ('source', CommentedMap([('fn', 'impute_v2.3.2.tgz'), ('url', 'https://mathgen.stats.ox.ac.uk/impute/impute_v2.3.2_MacOSX_Intel.tgz')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['impute2 2>&1 | grep IMPUTE'])])), ('about', CommentedMap([('home', 'https://mathgen.stats.ox.ac.uk/impute/impute_v2.html'), ('license', 'Academic use, commerically restricted (http://www.stats.ox.ac.uk/~marchini/software/gwas/gwas.html#licence)'), ('summary', 'Genotype imputation and haplotype phasing')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/adam/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/bigdatagenomics/adam'), ('license', 'Apache 2'), ('summary', 'Genomics analysis platform built on Apache Avro, Apache Spark and Parquet')])), ('package', CommentedMap([('name', 'adam'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'adam-distribution-spark2_2.11-<{ version }}-bin.tar.gz'), ('url', 'https://search.maven.org/remotecontent?filepath=org/bdgenomics/adam/adam-distribution-spark2_2.11/0.22.0/adam-distribution-spark2_2.11-0.22.0-bin.tar.gz'), ('sha256', '31624954eb473f2ec24354af7e6303a47887cf2ca4076bb71f54618aef59e15b'), ('patches', ['adam-shell.patch', 'adam-submit.patch'])])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=8', 'pyspark'])])), ('test', CommentedMap([('commands', ['adam-submit --help', 'adam-submit transform --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/adam/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/bigdatagenomics/adam'), ('license', 'Apache 2'), ('summary', 'Genomics analysis platform built on Apache Avro, Apache Spark and Parquet')])), ('package', CommentedMap([('name', 'adam'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'adam-distribution-spark2_2.11-<{ version }}-bin.tar.gz'), ('url', 'https://search.maven.org/remotecontent?filepath=org/bdgenomics/adam/adam-distribution-spark2_2.11/0.22.0/adam-distribution-spark2_2.11-0.22.0-bin.tar.gz'), ('sha256', '31624954eb473f2ec24354af7e6303a47887cf2ca4076bb71f54618aef59e15b'), ('patches', ['adam-shell.patch', 'adam-submit.patch'])])), ('requirements', CommentedMap([('build', None), ('run', ['openjdk >=8', 'pyspark'])])), ('test', CommentedMap([('commands', ['adam-submit --help', 'adam-submit transform --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-multidataset/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-multidataset'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'MultiDataSet_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MultiDataSet_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/MultiDataSet_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/MultiDataSet/MultiDataSet_1.4.0_src_all.tar.gz']), ('md5', '8c8c27b4d6dfd42a2371d31e52ef5cd1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MultiDataSet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MultiDataSet.html'), ('license', 'file LICENSE'), ('summary', \"Implementation of the BRGE's (Bioinformatic Research Group in Epidemiology from Center for Research in Environmental Epidemiology) MultiDataSet and MethylationSet. MultiDataSet is designed for integrating multi omics data sets and MethylationSet to contain normalized methylation data. These package contains base classes for MEAL and rexposome packages.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-multidataset/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-multidataset'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'MultiDataSet_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MultiDataSet_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/MultiDataSet_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/MultiDataSet/MultiDataSet_1.4.0_src_all.tar.gz']), ('md5', '8c8c27b4d6dfd42a2371d31e52ef5cd1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MultiDataSet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MultiDataSet.html'), ('license', 'file LICENSE'), ('summary', \"Implementation of the BRGE's (Bioinformatic Research Group in Epidemiology from Center for Research in Environmental Epidemiology) MultiDataSet and MethylationSet. MultiDataSet is designed for integrating multi omics data sets and MethylationSet to contain normalized methylation data. These package contains base classes for MEAL and rexposome packages.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-outlierd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-outlierd'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'OutlierD_1.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OutlierD_1.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/OutlierD_1.40.0.tar.gz', 'https://depot.galaxyproject.org/software/OutlierD/OutlierD_1.40.0_src_all.tar.gz']), ('md5', '529ece912ce51d273345dda44d968c59')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-quantreg']), ('run', ['bioconductor-biobase', 'r-base', 'r-quantreg'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OutlierD\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OutlierD.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package detects outliers using quantile regression on the M-A scatterplots of high-throughput data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-outlierd/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-outlierd'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'OutlierD_1.40.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OutlierD_1.40.0.tar.gz', 'https://bioarchive.galaxyproject.org/OutlierD_1.40.0.tar.gz', 'https://depot.galaxyproject.org/software/OutlierD/OutlierD_1.40.0_src_all.tar.gz']), ('md5', '529ece912ce51d273345dda44d968c59')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-quantreg']), ('run', ['bioconductor-biobase', 'r-base', 'r-quantreg'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OutlierD\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OutlierD.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package detects outliers using quantile regression on the M-A scatterplots of high-throughput data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bracken/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/jenniferlu717/Bracken/archive/<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', ['perl', 'perl-list-moreutils', 'perl-parallel-forkmanager', 'python', 'kraken'])])), ('test', CommentedMap([('commands', ['count-kmer-abundances.pl --help', 'est_abundance.py -h', 'generate_kmer_distribution.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/jenniferlu717/Bracken'), ('license', 'GPL-3.0'), ('license_file', 'LICENSE'), ('summary', 'Bracken (Bayesian Reestimation of Abundance with KrakEN) is a highly accurate statistical method that computes the abundance of species in DNA sequences from a metagenomics sample.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bracken/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('url', 'https://github.com/jenniferlu717/Bracken/archive/<{ version }}.tar.gz'), ('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', ['perl', 'perl-list-moreutils', 'perl-parallel-forkmanager', 'python', 'kraken'])])), ('test', CommentedMap([('commands', ['count-kmer-abundances.pl --help', 'est_abundance.py -h', 'generate_kmer_distribution.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/jenniferlu717/Bracken'), ('license', 'GPL-3.0'), ('license_file', 'LICENSE'), ('summary', 'Bracken (Bayesian Reestimation of Abundance with KrakEN) is a highly accurate statistical method that computes the abundance of species in DNA sequences from a metagenomics sample.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/circos/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'circos'), ('version', '<{ version }}.<{ rev }}')])), ('source', CommentedMap([('fn', 'circos-<{ version }}-<{ rev }}.tgz'), ('url', 'http://circos.ca/distribution/circos-<{ version }}-<{ rev }}.tgz'), ('sha256', '3eb868b8f5881f73d714115ce3d235f79f4353e1879e17a479a8d9d5a3627d5f')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['perl', 'perl-clone', 'perl-config-general', 'perl-digest-perl-md5', 'perl-font-ttf', 'perl-gd', 'perl-list-moreutils', 'perl-math-bezier', 'perl-math-round', 'perl-math-vecstat', 'perl-params-validate', 'perl-readonly', 'perl-regexp-common', 'perl-svg', 'perl-set-intspan', 'perl-statistics-basic', 'perl-text-format', 'perl-time-hires'])])), ('test', CommentedMap([('commands', ['circos -modules', 'circos -v'])])), ('about', CommentedMap([('home', 'http://circos.ca'), ('summary', 'Circos is a software package for visualizing data and information. It visualizes data in a circular layout'), ('license', 'GNU General Public License v2 (GPLv2)'), ('license_family', 'GPL2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/circos/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'circos'), ('version', '<{ version }}.<{ rev }}')])), ('source', CommentedMap([('fn', 'circos-<{ version }}-<{ rev }}.tgz'), ('url', 'http://circos.ca/distribution/circos-<{ version }}-<{ rev }}.tgz'), ('sha256', '3eb868b8f5881f73d714115ce3d235f79f4353e1879e17a479a8d9d5a3627d5f')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['perl', 'perl-clone', 'perl-config-general', 'perl-digest-perl-md5', 'perl-font-ttf', 'perl-gd', 'perl-list-moreutils', 'perl-math-bezier', 'perl-math-round', 'perl-math-vecstat', 'perl-params-validate', 'perl-readonly', 'perl-regexp-common', 'perl-svg', 'perl-set-intspan', 'perl-statistics-basic', 'perl-text-format', 'perl-time-hires'])])), ('test', CommentedMap([('commands', ['circos -modules', 'circos -v'])])), ('about', CommentedMap([('home', 'http://circos.ca'), ('summary', 'Circos is a software package for visualizing data and information. It visualizes data in a circular layout'), ('license', 'GNU General Public License v2 (GPLv2)'), ('license_family', 'GPL2')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-spp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-spp'), ('version', '1.14post')])), ('source', CommentedMap([('fn', '1.14post.zip'), ('url', 'https://github.com/hms-dbmi/spp/archive/c6526d2bc8eeaaa889970cd2f8d30c998f3ea320.zip'), ('sha256', '61f0d21f2c5127e570bf619445c89968009cfbd32b995984cf20788f986e666b'), ('patches', ['Makevars.patch'])])), ('build', CommentedMap([('number', 0), ('string', 'r_<{CONDA_R}}_boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'boost <{CONDA_BOOST}}*', 'r-catools', 'bioconductor-rsamtools', 'gcc', 'llvm', 'zlib', 'bzip2']), ('run', ['r-base', 'boost <{CONDA_BOOST}}*', 'r-catools', 'bioconductor-rsamtools', 'libgcc', 'zlib', 'bzip2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(spp)\"', '$R -e \"library(spp); read.bam.tags(system.file(\\\\\"extdata\\\\\", \\\\\"ex1.bam\\\\\", package=\\\\\"Rsamtools\\\\\"))\"'])])), ('about', CommentedMap([('license', 'GPL'), ('summary', 'ChIP-seq peak caller'), ('home', 'http://compbio.med.harvard.edu/Supplements/ChIP-seq')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-spp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-spp'), ('version', '1.14post')])), ('source', CommentedMap([('fn', '1.14post.zip'), ('url', 'https://github.com/hms-dbmi/spp/archive/c6526d2bc8eeaaa889970cd2f8d30c998f3ea320.zip'), ('sha256', '61f0d21f2c5127e570bf619445c89968009cfbd32b995984cf20788f986e666b'), ('patches', ['Makevars.patch'])])), ('build', CommentedMap([('number', 0), ('string', 'r_<{CONDA_R}}_boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'boost <{CONDA_BOOST}}*', 'r-catools', 'bioconductor-rsamtools', 'gcc', 'llvm', 'zlib', 'bzip2']), ('run', ['r-base', 'boost <{CONDA_BOOST}}*', 'r-catools', 'bioconductor-rsamtools', 'libgcc', 'zlib', 'bzip2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(spp)\"', '$R -e \"library(spp); read.bam.tags(system.file(\\\\\"extdata\\\\\", \\\\\"ex1.bam\\\\\", package=\\\\\"Rsamtools\\\\\"))\"'])])), ('about', CommentedMap([('license', 'GPL'), ('summary', 'ChIP-seq peak caller'), ('home', 'http://compbio.med.harvard.edu/Supplements/ChIP-seq')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationtools'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'annotationTools_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/annotationTools_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/annotationTools_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/annotationTools/annotationTools_1.50.0_src_all.tar.gz']), ('md5', '203f2a9f550931fd499aa0a360655402')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'annotationTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/annotationTools.html'), ('license', 'GPL'), ('summary', 'Functions to annotate microarrays, find orthologs, and integrate heterogeneous gene expression profiles using annotation and other molecular biology information available as flat file database (plain text files).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationtools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationtools'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'annotationTools_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/annotationTools_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/annotationTools_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/annotationTools/annotationTools_1.50.0_src_all.tar.gz']), ('md5', '203f2a9f550931fd499aa0a360655402')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'annotationTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/annotationTools.html'), ('license', 'GPL'), ('summary', 'Functions to annotate microarrays, find orthologs, and integrate heterogeneous gene expression profiles using annotation and other molecular biology information available as flat file database (plain text files).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/idba/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/'), ('license', ''), ('summary', 'IDBA is a practical iterative De Bruijn Graph De Novo Assembler for sequence assembly in bioinformatics.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['autoconf', 'automake', 'gcc', 'gawk', 'python', 'perl-threaded']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/loneknightpy/idba/archive/<{ version }}.tar.gz'), ('md5', '303d9b4af7a7498b56ac9698028b4e15')])), ('test', CommentedMap([('commands', ['idba --help 2>&1 | grep \"Iterative de Bruijn Graph Assembler\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/idba/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/'), ('license', ''), ('summary', 'IDBA is a practical iterative De Bruijn Graph De Novo Assembler for sequence assembly in bioinformatics.')])), ('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('requirements', CommentedMap([('build', ['autoconf', 'automake', 'gcc', 'gawk', 'python', 'perl-threaded']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', '<{ version }}.tar.gz'), ('url', 'https://github.com/loneknightpy/idba/archive/<{ version }}.tar.gz'), ('md5', '303d9b4af7a7498b56ac9698028b4e15')])), ('test', CommentedMap([('commands', ['idba --help 2>&1 | grep \"Iterative de Bruijn Graph Assembler\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ibdseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ibdseq'), ('version', 'r1206')])), ('source', CommentedMap([('fn', 'ibdseq.r1206.jar'), ('url', 'http://faculty.washington.edu/browning/ibdseq/ibdseq.r1206.jar'), ('md5', '3963b1c810b2fe71082b0a1d44805cd9')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk >=6.0.77'])])), ('test', CommentedMap([('commands', ['ibdseq | grep \"[parameters]\"'])])), ('about', CommentedMap([('home', 'http://faculty.washington.edu/browning/ibdseq.html'), ('license', 'Apache v2.0'), ('summary', 'IBDseq is a software program for detecting segments of identity-by-descent (IBD) and homozygosity-by-descent (HBD) in unphased genetic sequence data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ibdseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ibdseq'), ('version', 'r1206')])), ('source', CommentedMap([('fn', 'ibdseq.r1206.jar'), ('url', 'http://faculty.washington.edu/browning/ibdseq/ibdseq.r1206.jar'), ('md5', '3963b1c810b2fe71082b0a1d44805cd9')])), ('build', CommentedMap([('detect_binary_files_with_prefix', True), ('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['java-jdk >=6.0.77'])])), ('test', CommentedMap([('commands', ['ibdseq | grep \"[parameters]\"'])])), ('about', CommentedMap([('home', 'http://faculty.washington.edu/browning/ibdseq.html'), ('license', 'Apache v2.0'), ('summary', 'IBDseq is a software program for detecting segments of identity-by-descent (IBD) and homozygosity-by-descent (HBD) in unphased genetic sequence data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nucleosim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nucleosim'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'nucleoSim_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nucleoSim_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/nucleoSim_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/nucleoSim/nucleoSim_1.4.0_src_all.tar.gz']), ('md5', '1efb2ee4c162e11775179d859c1b0a8b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-iranges', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-iranges', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nucleoSim\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nucleoSim.html'), ('license', 'Artistic-2.0'), ('summary', 'This package can generate a synthetic map with reads covering the nucleosome regions as well as a synthetic map with forward and reverse reads emulating next-generation sequencing. The user has choice between three different distributions for the read positioning: Normal, Student and Uniform.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nucleosim/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nucleosim'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'nucleoSim_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nucleoSim_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/nucleoSim_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/nucleoSim/nucleoSim_1.4.0_src_all.tar.gz']), ('md5', '1efb2ee4c162e11775179d859c1b0a8b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-iranges', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-iranges', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nucleoSim\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nucleoSim.html'), ('license', 'Artistic-2.0'), ('summary', 'This package can generate a synthetic map with reads covering the nucleosome regions as well as a synthetic map with forward and reverse reads emulating next-generation sequencing. The user has choice between three different distributions for the read positioning: Normal, Student and Uniform.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rmap/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://smithlabresearch.org/software/rmap/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'rmap is a short reads mapper for next-generation sequencing data')])), ('package', CommentedMap([('name', 'rmap'), ('version', '2.1')])), ('source', CommentedMap([('fn', 'rmap-2.1.tar.bz2'), ('url', 'http://smithlabresearch.org/downloads/rmap-2.1.tar.bz2'), ('sha256', '0196139025e1dfaa203b28176607fadcad50b4b0926eb53ef9fc7e95ec5dece7')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['rmap 2>&1 | grep \"^Usage\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rmap/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://smithlabresearch.org/software/rmap/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'rmap is a short reads mapper for next-generation sequencing data')])), ('package', CommentedMap([('name', 'rmap'), ('version', '2.1')])), ('source', CommentedMap([('fn', 'rmap-2.1.tar.bz2'), ('url', 'http://smithlabresearch.org/downloads/rmap-2.1.tar.bz2'), ('sha256', '0196139025e1dfaa203b28176607fadcad50b4b0926eb53ef9fc7e95ec5dece7')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['rmap 2>&1 | grep \"^Usage\"'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tssi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tssi'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'TSSi_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TSSi_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/TSSi_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/TSSi/TSSi_1.22.0_src_all.tar.gz']), ('md5', '2de488ce7fa30a257e2c1b3b3f09a2a3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=0.3.2', 'bioconductor-biocgenerics >=0.3.2', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base', 'r-hmisc', 'r-minqa', 'r-plyr']), ('run', ['bioconductor-biobase >=0.3.2', 'bioconductor-biocgenerics >=0.3.2', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base', 'r-hmisc', 'r-minqa', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TSSi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TSSi.html'), ('license', 'GPL-3'), ('summary', 'Identify and normalize transcription start sites in high-throughput sequencing data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tssi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tssi'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'TSSi_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TSSi_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/TSSi_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/TSSi/TSSi_1.22.0_src_all.tar.gz']), ('md5', '2de488ce7fa30a257e2c1b3b3f09a2a3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=0.3.2', 'bioconductor-biocgenerics >=0.3.2', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base', 'r-hmisc', 'r-minqa', 'r-plyr']), ('run', ['bioconductor-biobase >=0.3.2', 'bioconductor-biocgenerics >=0.3.2', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base', 'r-hmisc', 'r-minqa', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TSSi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TSSi.html'), ('license', 'GPL-3'), ('summary', 'Identify and normalize transcription start sites in high-throughput sequencing data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trf/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}'), ('url', 'https://tandem.bu.edu/trf/downloads/trf409.macosx'), ('md5', '14784824b6c2fae52f4b5c289ef67255')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['trf -v'])])), ('about', CommentedMap([('home', 'https://tandem.bu.edu/trf/trf.html'), ('license', 'Custom'), ('summary', 'Tandem Repeats Finder is a program to locate and display tandem repeats in DNA sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/trf/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}'), ('url', 'https://tandem.bu.edu/trf/downloads/trf409.macosx'), ('md5', '14784824b6c2fae52f4b5c289ef67255')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['trf -v'])])), ('about', CommentedMap([('home', 'https://tandem.bu.edu/trf/trf.html'), ('license', 'Custom'), ('summary', 'Tandem Repeats Finder is a program to locate and display tandem repeats in DNA sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-specond/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-specond'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'SpeCond_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SpeCond_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/SpeCond_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/SpeCond/SpeCond_1.30.0_src_all.tar.gz']), ('md5', '05b8043512a1d5c01d51b16435ed49cd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.15.13', 'r-base', 'r-fields', 'r-hwriter >=1.1', 'r-mclust >=3.3.1', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase >=1.15.13', 'r-base', 'r-fields', 'r-hwriter >=1.1', 'r-mclust >=3.3.1', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SpeCond\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SpeCond.html'), ('license', 'LGPL (>=2)'), ('summary', 'This package performs a gene expression data analysis to detect condition-specific genes. Such genes are significantly up- or down-regulated in a small number of conditions. It does so by fitting a mixture of normal distributions to the expression values. Conditions can be environmental conditions, different tissues, organs or any other sources that you wish to compare in terms of gene expression.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-specond/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-specond'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'SpeCond_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SpeCond_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/SpeCond_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/SpeCond/SpeCond_1.30.0_src_all.tar.gz']), ('md5', '05b8043512a1d5c01d51b16435ed49cd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.15.13', 'r-base', 'r-fields', 'r-hwriter >=1.1', 'r-mclust >=3.3.1', 'r-rcolorbrewer']), ('run', ['bioconductor-biobase >=1.15.13', 'r-base', 'r-fields', 'r-hwriter >=1.1', 'r-mclust >=3.3.1', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SpeCond\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SpeCond.html'), ('license', 'LGPL (>=2)'), ('summary', 'This package performs a gene expression data analysis to detect condition-specific genes. Such genes are significantly up- or down-regulated in a small number of conditions. It does so by fitting a mixture of normal distributions to the expression values. Conditions can be environmental conditions, different tissues, organs or any other sources that you wish to compare in terms of gene expression.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cd-hit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cd-hit'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'cd-hit-<{ version }}.tar.gz'), ('url', 'https://github.com/weizhongli/cdhit/archive/V<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', [\"cd-hit --help | grep 'Usage' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/weizhongli/cdhit'), ('license', 'GPLv2'), ('summary', 'Clusters and compares protein or nucleotide sequences')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cd-hit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cd-hit'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'cd-hit-<{ version }}.tar.gz'), ('url', 'https://github.com/weizhongli/cdhit/archive/V<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', [\"cd-hit --help | grep 'Usage' > /dev/null\"])])), ('about', CommentedMap([('home', 'https://github.com/weizhongli/cdhit'), ('license', 'GPLv2'), ('summary', 'Clusters and compares protein or nucleotide sequences')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gviz'), ('version', '1.16.5')])), ('source', CommentedMap([('fn', 'Gviz_1.16.5.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/Gviz_1.16.5.tar.gz'), ('md5', '81f61d0d12c823f91eff0a69d14b3943')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.27.5', 'bioconductor-biobase >=2.15.3', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biomart >=2.11.0', 'bioconductor-biostrings >=2.33.11', 'bioconductor-biovizbase >=1.13.8', 'bioconductor-bsgenome >=1.33.1', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.22', 'bioconductor-genomicranges >=1.17.20', 'bioconductor-iranges >=1.99.18', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.13', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.7', 'r-base', 'r-digest >=0.6.8', 'r-latticeextra >=0.6_26', 'r-matrixstats >=0.8.14', 'r-rcolorbrewer']), ('run', ['bioconductor-annotationdbi >=1.27.5', 'bioconductor-biobase >=2.15.3', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biomart >=2.11.0', 'bioconductor-biostrings >=2.33.11', 'bioconductor-biovizbase >=1.13.8', 'bioconductor-bsgenome >=1.33.1', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.22', 'bioconductor-genomicranges >=1.17.20', 'bioconductor-iranges >=1.99.18', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.13', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.7', 'r-base', 'r-digest >=0.6.8', 'r-latticeextra >=0.6_26', 'r-matrixstats >=0.8.14', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Gviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Gviz.html'), ('license', 'Artistic-2.0'), ('summary', 'Genomic data analyses requires integrated visualization of known genomic information and new experimental data.  Gviz uses the biomaRt and the rtracklayer packages to perform live annotation queries to Ensembl and UCSC and translates this to e.g. gene/transcript structures in viewports of the grid graphics package. This results in genomic information plotted together with your data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gviz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gviz'), ('version', '1.16.5')])), ('source', CommentedMap([('fn', 'Gviz_1.16.5.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/Gviz_1.16.5.tar.gz'), ('md5', '81f61d0d12c823f91eff0a69d14b3943')])), ('build', CommentedMap([('number', 2), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.27.5', 'bioconductor-biobase >=2.15.3', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biomart >=2.11.0', 'bioconductor-biostrings >=2.33.11', 'bioconductor-biovizbase >=1.13.8', 'bioconductor-bsgenome >=1.33.1', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.22', 'bioconductor-genomicranges >=1.17.20', 'bioconductor-iranges >=1.99.18', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.13', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.7', 'r-base', 'r-digest >=0.6.8', 'r-latticeextra >=0.6_26', 'r-matrixstats >=0.8.14', 'r-rcolorbrewer']), ('run', ['bioconductor-annotationdbi >=1.27.5', 'bioconductor-biobase >=2.15.3', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biomart >=2.11.0', 'bioconductor-biostrings >=2.33.11', 'bioconductor-biovizbase >=1.13.8', 'bioconductor-bsgenome >=1.33.1', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.22', 'bioconductor-genomicranges >=1.17.20', 'bioconductor-iranges >=1.99.18', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.13', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-xvector >=0.5.7', 'r-base', 'r-digest >=0.6.8', 'r-latticeextra >=0.6_26', 'r-matrixstats >=0.8.14', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Gviz\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Gviz.html'), ('license', 'Artistic-2.0'), ('summary', 'Genomic data analyses requires integrated visualization of known genomic information and new experimental data.  Gviz uses the biomaRt and the rtracklayer packages to perform live annotation queries to Ensembl and UCSC and translates this to e.g. gene/transcript structures in viewports of the grid graphics package. This results in genomic information plotted together with your data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tscan'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'TSCAN_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TSCAN_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/TSCAN_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/TSCAN/TSCAN_1.14.0_src_all.tar.gz']), ('md5', '490a22ef7ecccd8e5c5ff45a6c12ee7a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-combinat', 'r-fastica', 'r-ggplot2', 'r-gplots', 'r-igraph', 'r-mclust', 'r-mgcv', 'r-plyr', 'r-shiny']), ('run', ['r-base', 'r-combinat', 'r-fastica', 'r-ggplot2', 'r-gplots', 'r-igraph', 'r-mclust', 'r-mgcv', 'r-plyr', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TSCAN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TSCAN.html'), ('license', 'GPL(>=2)'), ('summary', 'TSCAN enables users to easily construct and tune pseudotemporal cell ordering as well as analyzing differentially expressed genes. TSCAN comes with a user-friendly GUI written in shiny. More features will come in the future.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tscan'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'TSCAN_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TSCAN_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/TSCAN_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/TSCAN/TSCAN_1.14.0_src_all.tar.gz']), ('md5', '490a22ef7ecccd8e5c5ff45a6c12ee7a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-combinat', 'r-fastica', 'r-ggplot2', 'r-gplots', 'r-igraph', 'r-mclust', 'r-mgcv', 'r-plyr', 'r-shiny']), ('run', ['r-base', 'r-combinat', 'r-fastica', 'r-ggplot2', 'r-gplots', 'r-igraph', 'r-mclust', 'r-mgcv', 'r-plyr', 'r-shiny'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TSCAN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TSCAN.html'), ('license', 'GPL(>=2)'), ('summary', 'TSCAN enables users to easily construct and tune pseudotemporal cell ordering as well as analyzing differentially expressed genes. TSCAN comes with a user-friendly GUI written in shiny. More features will come in the future.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mlst/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mlst'), ('version', '2.9')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'mlst_2.9.tar.gz'), ('url', 'https://github.com/tseemann/mlst/archive/2.9.tar.gz'), ('md5', '7421ad5a423440570f6eede2cfc4a684')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'perl', 'perl-bioperl', 'perl-moo', 'perl-list-moreutils', 'blast'])])), ('test', CommentedMap([('commands', ['mlst --list 2>&1 | grep leptospira > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/mlst'), ('license', 'GPL2'), ('license_file', 'LICENSE'), ('summary', 'Scan contig files against PubMLST typing schemes')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mlst/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'mlst'), ('version', '2.9')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'mlst_2.9.tar.gz'), ('url', 'https://github.com/tseemann/mlst/archive/2.9.tar.gz'), ('md5', '7421ad5a423440570f6eede2cfc4a684')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'perl', 'perl-bioperl', 'perl-moo', 'perl-list-moreutils', 'blast'])])), ('test', CommentedMap([('commands', ['mlst --list 2>&1 | grep leptospira > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/mlst'), ('license', 'GPL2'), ('license_file', 'LICENSE'), ('summary', 'Scan contig files against PubMLST typing schemes')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/intervene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'intervene'), ('version', '0.5.8')])), ('source', CommentedMap([('fn', 'intervene-0.5.8.tar.gz'), ('url', 'https://pypi.python.org/packages/a8/68/3d52c21b3cce52d22207973471fed7c113cfc4fbcaa49718620de14957a3/intervene-0.5.8.tar.gz'), ('md5', '31c74e6da434d6df38743072e24c6cac')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pybedtools', 'matplotlib', 'pandas', 'numpy', 'scipy', 'seaborn']), ('run', ['python', 'pybedtools', 'matplotlib', 'pandas', 'numpy', 'scipy', 'seaborn'])])), ('test', CommentedMap([('imports', ['intervene', 'intervene.example_data', 'intervene.example_data.ENCODE_hESC', 'intervene.example_data.Gene_list', 'intervene.example_data.dbSUPER_mm9', 'intervene.modules', 'intervene.modules.pairwise', 'intervene.modules.upset', 'intervene.modules.venn'])])), ('about', CommentedMap([('home', 'https://github.com/asntech/intervene'), ('license', 'MIT License'), ('summary', 'A tool for intersection and visualization of multiple gene or genomic region sets'), ('license_family', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/intervene/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'intervene'), ('version', '0.5.8')])), ('source', CommentedMap([('fn', 'intervene-0.5.8.tar.gz'), ('url', 'https://pypi.python.org/packages/a8/68/3d52c21b3cce52d22207973471fed7c113cfc4fbcaa49718620de14957a3/intervene-0.5.8.tar.gz'), ('md5', '31c74e6da434d6df38743072e24c6cac')])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'pybedtools', 'matplotlib', 'pandas', 'numpy', 'scipy', 'seaborn']), ('run', ['python', 'pybedtools', 'matplotlib', 'pandas', 'numpy', 'scipy', 'seaborn'])])), ('test', CommentedMap([('imports', ['intervene', 'intervene.example_data', 'intervene.example_data.ENCODE_hESC', 'intervene.example_data.Gene_list', 'intervene.example_data.dbSUPER_mm9', 'intervene.modules', 'intervene.modules.pairwise', 'intervene.modules.upset', 'intervene.modules.venn'])])), ('about', CommentedMap([('home', 'https://github.com/asntech/intervene'), ('license', 'MIT License'), ('summary', 'A tool for intersection and visualization of multiple gene or genomic region sets'), ('license_family', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-promise/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-promise'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'PROMISE_1.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PROMISE_1.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/PROMISE_1.28.0.tar.gz', 'https://depot.galaxyproject.org/software/PROMISE/PROMISE_1.28.0_src_all.tar.gz']), ('md5', 'ebb9cf1b6d9d5b580753c9c24ab49730')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PROMISE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PROMISE.html'), ('license', 'GPL (>= 2)'), ('summary', 'A general tool to identify genomic features with a specific biologically interesting pattern of associations with multiple endpoint variables as described in Pounds et. al. (2009) Bioinformatics 25: 2013-2019')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-promise/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-promise'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'PROMISE_1.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PROMISE_1.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/PROMISE_1.28.0.tar.gz', 'https://depot.galaxyproject.org/software/PROMISE/PROMISE_1.28.0_src_all.tar.gz']), ('md5', 'ebb9cf1b6d9d5b580753c9c24ab49730')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PROMISE\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PROMISE.html'), ('license', 'GPL (>= 2)'), ('summary', 'A general tool to identify genomic features with a specific biologically interesting pattern of associations with multiple endpoint variables as described in Pounds et. al. (2009) Bioinformatics 25: 2013-2019')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bamutil/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bamutil'), ('version', '1.0.14')])), ('source', CommentedMap([('fn', 'v1.0.14.tar.gz'), ('url', 'https://github.com/statgen/bamUtil/archive/v1.0.14.tar.gz'), ('sha256', 'f5ec8d5e98a3797742106c3413a4ab1622d8787e38b29b3df4cddb59d77efda5')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['bam --help 2>&1 | grep Version', 'bam squeeze --help 2>&1 | grep squeeze'])])), ('about', CommentedMap([('home', 'http://genome.sph.umich.edu/wiki/BamUtil'), ('license', 'GPLv3'), ('summary', 'Programs that perform operations on SAM/BAM files, all built into a single executable, bam.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bamutil/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bamutil'), ('version', '1.0.14')])), ('source', CommentedMap([('fn', 'v1.0.14.tar.gz'), ('url', 'https://github.com/statgen/bamUtil/archive/v1.0.14.tar.gz'), ('sha256', 'f5ec8d5e98a3797742106c3413a4ab1622d8787e38b29b3df4cddb59d77efda5')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['bam --help 2>&1 | grep Version', 'bam squeeze --help 2>&1 | grep squeeze'])])), ('about', CommentedMap([('home', 'http://genome.sph.umich.edu/wiki/BamUtil'), ('license', 'GPLv3'), ('summary', 'Programs that perform operations on SAM/BAM files, all built into a single executable, bam.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/spades/3.9.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'spades'), ('version', '3.9.0')])), ('source', CommentedMap([('fn', 'SPAdes-3.9.0-Darwin.tar.gz'), ('url', 'http://spades.bioinf.spbau.ru/release3.9.0/SPAdes-3.9.0-Darwin.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', ['python'])])), ('test', CommentedMap([('commands', ['spades.py --test'])])), ('about', CommentedMap([('home', 'http://bioinf.spbau.ru/en/spades'), ('license', 'GPLv2'), ('summary', 'SPAdes (St. Petersburg genome assembler) is intended for both standard isolates and single-cell MDA bacteria assemblies.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/spades/3.9.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'spades'), ('version', '3.9.0')])), ('source', CommentedMap([('fn', 'SPAdes-3.9.0-Darwin.tar.gz'), ('url', 'http://spades.bioinf.spbau.ru/release3.9.0/SPAdes-3.9.0-Darwin.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', None), ('run', ['python'])])), ('test', CommentedMap([('commands', ['spades.py --test'])])), ('about', CommentedMap([('home', 'http://bioinf.spbau.ru/en/spades'), ('license', 'GPLv2'), ('summary', 'SPAdes (St. Petersburg genome assembler) is intended for both standard isolates and single-cell MDA bacteria assemblies.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pcaexplorer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pcaexplorer'), ('version', '2.0.0')])), ('source', CommentedMap([('fn', 'pcaExplorer_2.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/pcaExplorer_2.0.0.tar.gz'), ('md5', '456fefb3d662f833b26a86810f26d692')])), ('build', CommentedMap([('skip', True), ('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biomart', 'bioconductor-deseq2', 'bioconductor-genefilter', 'bioconductor-genomicranges', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-topgo', 'r-base', 'r-d3heatmap', 'r-dt', 'r-ggplot2 >=2.0.0', 'r-ggrepel', 'r-knitr', 'r-nmf', 'r-pheatmap', 'r-plyr', 'r-rmarkdown', 'r-scales', 'r-shiny >=0.12.0', 'r-shinyace', 'r-shinybs', 'r-shinydashboard', 'r-threejs', 'r-tidyr']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biomart', 'bioconductor-deseq2', 'bioconductor-genefilter', 'bioconductor-genomicranges', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-topgo', 'r-base', 'r-d3heatmap', 'r-dt', 'r-ggplot2 >=2.0.0', 'r-ggrepel', 'r-knitr', 'r-nmf', 'r-pheatmap', 'r-plyr', 'r-rmarkdown', 'r-scales', 'r-shiny >=0.12.0', 'r-shinyace', 'r-shinybs', 'r-shinydashboard', 'r-threejs', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pcaExplorer\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/pcaExplorer.html'), ('license', 'MIT + file LICENSE'), ('summary', 'This package provides functionality for interactive visualization of RNA-seq datasets based on Principal Components Analysis. The methods provided allow for quick information extraction and effective data exploration. A Shiny application encapsulates the whole analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pcaexplorer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pcaexplorer'), ('version', '2.0.0')])), ('source', CommentedMap([('fn', 'pcaExplorer_2.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/pcaExplorer_2.0.0.tar.gz'), ('md5', '456fefb3d662f833b26a86810f26d692')])), ('build', CommentedMap([('skip', True), ('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biomart', 'bioconductor-deseq2', 'bioconductor-genefilter', 'bioconductor-genomicranges', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-topgo', 'r-base', 'r-d3heatmap', 'r-dt', 'r-ggplot2 >=2.0.0', 'r-ggrepel', 'r-knitr', 'r-nmf', 'r-pheatmap', 'r-plyr', 'r-rmarkdown', 'r-scales', 'r-shiny >=0.12.0', 'r-shinyace', 'r-shinybs', 'r-shinydashboard', 'r-threejs', 'r-tidyr']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biomart', 'bioconductor-deseq2', 'bioconductor-genefilter', 'bioconductor-genomicranges', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-topgo', 'r-base', 'r-d3heatmap', 'r-dt', 'r-ggplot2 >=2.0.0', 'r-ggrepel', 'r-knitr', 'r-nmf', 'r-pheatmap', 'r-plyr', 'r-rmarkdown', 'r-scales', 'r-shiny >=0.12.0', 'r-shinyace', 'r-shinybs', 'r-shinydashboard', 'r-threejs', 'r-tidyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pcaExplorer\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/pcaExplorer.html'), ('license', 'MIT + file LICENSE'), ('summary', 'This package provides functionality for interactive visualization of RNA-seq datasets based on Principal Components Analysis. The methods provided allow for quick information extraction and effective data exploration. A Shiny application encapsulates the whole analysis.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/msms/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'msms'), ('version', '2.6.1')])), ('source', CommentedMap([('fn', 'msms_i86_64Linux2_2.6.1.tar.gz'), ('url', 'http://mgltools.scripps.edu/downloads/tars/releases/MSMSRELEASE/REL2.6.1/msms_i86_64Linux2_2.6.1.tar.gz'), ('md5', '650d8c6fd49d9ce291f30ed530d5f313'), ('patches', ['pdb_to_xyzr.patch', 'pdb_to_xyzrn.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('has_prefix_files', ['bin/pdb_to_xyzr', 'bin/pdb_to_xyzrn'])])), ('test', CommentedMap([('commands', ['msms -h'])])), ('about', CommentedMap([('home', 'http://mgltools.scripps.edu/packages/MSMS/'), ('license', 'Free for academic use.'), ('summary', 'MSMS is a program written in the C programming language to compute molecular surfaces.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/msms/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'msms'), ('version', '2.6.1')])), ('source', CommentedMap([('fn', 'msms_i86_64Linux2_2.6.1.tar.gz'), ('url', 'http://mgltools.scripps.edu/downloads/tars/releases/MSMSRELEASE/REL2.6.1/msms_i86_64Linux2_2.6.1.tar.gz'), ('md5', '650d8c6fd49d9ce291f30ed530d5f313'), ('patches', ['pdb_to_xyzr.patch', 'pdb_to_xyzrn.patch'])])), ('build', CommentedMap([('number', 1), ('skip', True), ('has_prefix_files', ['bin/pdb_to_xyzr', 'bin/pdb_to_xyzrn'])])), ('test', CommentedMap([('commands', ['msms -h'])])), ('about', CommentedMap([('home', 'http://mgltools.scripps.edu/packages/MSMS/'), ('license', 'Free for academic use.'), ('summary', 'MSMS is a program written in the C programming language to compute molecular surfaces.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-stepnorm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-stepnorm'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'stepNorm_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/stepNorm_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/stepNorm_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/stepNorm/stepNorm_1.48.0_src_all.tar.gz']), ('md5', 'e0553374d83c0b631912e27f0f829ff7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-marray', 'r-base', 'r-mass']), ('run', ['bioconductor-marray', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'stepNorm\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/stepNorm.html'), ('license', 'LGPL'), ('summary', 'Stepwise normalization functions for cDNA microarray data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-stepnorm/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-stepnorm'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'stepNorm_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/stepNorm_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/stepNorm_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/stepNorm/stepNorm_1.48.0_src_all.tar.gz']), ('md5', 'e0553374d83c0b631912e27f0f829ff7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-marray', 'r-base', 'r-mass']), ('run', ['bioconductor-marray', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'stepNorm\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/stepNorm.html'), ('license', 'LGPL'), ('summary', 'Stepwise normalization functions for cDNA microarray data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ipo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ipo'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'IPO_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/IPO_1.0.0.tar.gz'), ('md5', 'd0cf5ff014677ed4ef4fcdb5a8c64bca')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-camera', 'bioconductor-xcms', 'r-base', 'r-rsm']), ('run', ['bioconductor-camera', 'bioconductor-xcms', 'r-base', 'r-rsm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'IPO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/IPO.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'The outcome of XCMS data processing strongly depends on the parameter settings. IPO (`Isotopologue Parameter Optimization`) is a parameter optimization tool that is applicable for different kinds of samples and liquid chromatography coupled to high resolution mass spectrometry devices, fast and free of labeling steps. IPO uses natural, stable 13C isotopes to calculate a peak picking score. Retention time correction is optimized by minimizing the relative retention time differences within features and grouping parameters are optimized by maximizing the number of features showing exactly one peak from each injection of a pooled sample. The different parameter settings are achieved by design of experiment. The resulting scores are evaluated using response surface models.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ipo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ipo'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'IPO_1.0.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/IPO_1.0.0.tar.gz'), ('md5', 'd0cf5ff014677ed4ef4fcdb5a8c64bca')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-camera', 'bioconductor-xcms', 'r-base', 'r-rsm']), ('run', ['bioconductor-camera', 'bioconductor-xcms', 'r-base', 'r-rsm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'IPO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/IPO.html'), ('license', 'GPL (>= 2) + file LICENSE'), ('summary', 'The outcome of XCMS data processing strongly depends on the parameter settings. IPO (`Isotopologue Parameter Optimization`) is a parameter optimization tool that is applicable for different kinds of samples and liquid chromatography coupled to high resolution mass spectrometry devices, fast and free of labeling steps. IPO uses natural, stable 13C isotopes to calculate a peak picking score. Retention time correction is optimized by minimizing the relative retention time differences within features and grouping parameters are optimized by maximizing the number of features showing exactly one peak from each injection of a pooled sample. The different parameter settings are achieved by design of experiment. The resulting scores are evaluated using response surface models.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/emboss/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'emboss'), ('version', '6.5.7')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'EMBOSS-6.5.7.tar.gz'), ('url', 'ftp://emboss.open-bio.org/pub/EMBOSS/old/6.5.0/EMBOSS-6.5.7.tar.gz'), ('md5', '6a2cb3f93d5e9415c74ab0f6b1ede5f0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'zlib']), ('run', ['zlib'])])), ('about', CommentedMap([('home', 'http://emboss.open-bio.org/'), ('license', 'GPL'), ('summary', 'The European Molecular Biology Open Software Suite')])), ('test', CommentedMap([('commands', ['needle --help', 'transeq --help', 'sixpack --help', 'seqret --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/emboss/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'emboss'), ('version', '6.5.7')])), ('build', CommentedMap([('number', 2)])), ('source', CommentedMap([('fn', 'EMBOSS-6.5.7.tar.gz'), ('url', 'ftp://emboss.open-bio.org/pub/EMBOSS/old/6.5.0/EMBOSS-6.5.7.tar.gz'), ('md5', '6a2cb3f93d5e9415c74ab0f6b1ede5f0')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'python', 'zlib']), ('run', ['zlib'])])), ('about', CommentedMap([('home', 'http://emboss.open-bio.org/'), ('license', 'GPL'), ('summary', 'The European Molecular Biology Open Software Suite')])), ('test', CommentedMap([('commands', ['needle --help', 'transeq --help', 'sixpack --help', 'seqret --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ensembl-vep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ensembl-vep'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'vep-<{ version }}.tar.gz'), ('url', 'https://github.com/Ensembl/ensembl-vep/archive/release/<{ version }}.tar.gz'), ('md5', 'efd312434d7a8b5aa1d50b31dc59015d')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['curl', 'perl', 'perl-archive-zip', 'perl-lwp-simple', 'unzip', 'perl-dbi']), ('run', ['perl', 'htslib <{CONDA_HTSLIB}}*', 'perl-bioperl', 'perl-bio-db-hts >=2.7', 'perl-cgi', 'perl-dbi', 'perl-dbd-mysql', 'perl-io-compress', 'perl-json', 'perl-perlio-gzip', 'perl-sereal', 'perl-set-intervaltree', 'unzip'])])), ('test', CommentedMap([('commands', ['vep --help', 'vep_convert_cache --help', 'vep_install --help', 'haplo --help', 'variant_recoder --help'])])), ('about', CommentedMap([('home', 'http://www.ensembl.org/info/docs/tools/vep/index.html'), ('license', 'Apache 2.0'), ('summary', 'The VEP determines the effect of your variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions.')])), ('extra', CommentedMap([('notes', \"This package installs only the variant effect predictor (VEP) library\\ncode. To install data libraries, you can use the 'vep_install' command\\ninstalled along with it. For example, to install the VEP library for human\\nGRCh38 to a directory\\n\\nvep_install -a cf -s homo_sapiens -y GRCh38 -c /output/path/to/GRCh38/vep --CONVERT\\n\\n(note that vep_install is renamed from INSTALL.pl\\n to avoid having generic script names in the PATH)\\n\\nThe --CONVERT flag is not required but improves lookup speeds during\\nruns. See the VEP documentation for more details\\n\\nhttp://www.ensembl.org/info/docs/tools/vep/script/vep_cache.html\\n\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ensembl-vep/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ensembl-vep'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'vep-<{ version }}.tar.gz'), ('url', 'https://github.com/Ensembl/ensembl-vep/archive/release/<{ version }}.tar.gz'), ('md5', 'efd312434d7a8b5aa1d50b31dc59015d')])), ('build', CommentedMap([('number', 0), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['curl', 'perl', 'perl-archive-zip', 'perl-lwp-simple', 'unzip', 'perl-dbi']), ('run', ['perl', 'htslib <{CONDA_HTSLIB}}*', 'perl-bioperl', 'perl-bio-db-hts >=2.7', 'perl-cgi', 'perl-dbi', 'perl-dbd-mysql', 'perl-io-compress', 'perl-json', 'perl-perlio-gzip', 'perl-sereal', 'perl-set-intervaltree', 'unzip'])])), ('test', CommentedMap([('commands', ['vep --help', 'vep_convert_cache --help', 'vep_install --help', 'haplo --help', 'variant_recoder --help'])])), ('about', CommentedMap([('home', 'http://www.ensembl.org/info/docs/tools/vep/index.html'), ('license', 'Apache 2.0'), ('summary', 'The VEP determines the effect of your variants (SNPs, insertions, deletions, CNVs or structural variants) on genes, transcripts, and protein sequence, as well as regulatory regions.')])), ('extra', CommentedMap([('notes', \"This package installs only the variant effect predictor (VEP) library\\ncode. To install data libraries, you can use the 'vep_install' command\\ninstalled along with it. For example, to install the VEP library for human\\nGRCh38 to a directory\\n\\nvep_install -a cf -s homo_sapiens -y GRCh38 -c /output/path/to/GRCh38/vep --CONVERT\\n\\n(note that vep_install is renamed from INSTALL.pl\\n to avoid having generic script names in the PATH)\\n\\nThe --CONVERT flag is not required but improves lookup speeds during\\nruns. See the VEP documentation for more details\\n\\nhttp://www.ensembl.org/info/docs/tools/vep/script/vep_cache.html\\n\")]))]))\n",
      "EXTRA has been found: vep\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/htseq/0.7.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'htseq'), ('version', '0.7.2')])), ('source', CommentedMap([('fn', 'HTSeq-0.7.2.tar.gz'), ('url', 'https://pypi.python.org/packages/46/f7/6105848893b1d280692eac4f4f3c08ed7f424cec636aeda66b50bbcf217e/HTSeq-0.7.2.tar.gz'), ('sha256', 'b9da6d6a9004e8e05a40ecf08f74f765ec0e0be0e835e82205a117aba2ce375b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'numpy', 'pysam >=0.9.0']), ('run', ['python', 'numpy', 'matplotlib', 'pysam >=0.9.0'])])), ('test', CommentedMap([('imports', ['HTSeq._HTSeq_internal', 'HTSeq.StepVector', 'HTSeq._version', 'HTSeq.scripts.count', 'HTSeq.scripts.qa']), ('commands', ['htseq-count -h', 'htseq-qa -h'])])), ('about', CommentedMap([('home', 'http://www-huber.embl.de/users/anders/HTSeq/'), ('license', 'GNU General Public License (GPL)'), ('summary', 'A framework to process and analyze data from high-throughput sequencing (HTS) assays'), ('license_family', 'LGPL')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/htseq/0.7.2/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'htseq'), ('version', '0.7.2')])), ('source', CommentedMap([('fn', 'HTSeq-0.7.2.tar.gz'), ('url', 'https://pypi.python.org/packages/46/f7/6105848893b1d280692eac4f4f3c08ed7f424cec636aeda66b50bbcf217e/HTSeq-0.7.2.tar.gz'), ('sha256', 'b9da6d6a9004e8e05a40ecf08f74f765ec0e0be0e835e82205a117aba2ce375b')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'cython', 'numpy', 'pysam >=0.9.0']), ('run', ['python', 'numpy', 'matplotlib', 'pysam >=0.9.0'])])), ('test', CommentedMap([('imports', ['HTSeq._HTSeq_internal', 'HTSeq.StepVector', 'HTSeq._version', 'HTSeq.scripts.count', 'HTSeq.scripts.qa']), ('commands', ['htseq-count -h', 'htseq-qa -h'])])), ('about', CommentedMap([('home', 'http://www-huber.embl.de/users/anders/HTSeq/'), ('license', 'GNU General Public License (GPL)'), ('summary', 'A framework to process and analyze data from high-throughput sequencing (HTS) assays'), ('license_family', 'LGPL')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-shortread/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-shortread'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'ShortRead_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ShortRead_1.32.0.tar.gz'), ('md5', '71d56164fd7995c0d257027bbd9e24d1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-biostrings >=2.37.1', 'bioconductor-genomeinfodb >=1.1.19', 'bioconductor-genomicalignments >=1.5.4', 'bioconductor-genomicranges >=1.21.6', 'bioconductor-iranges >=2.3.7', 'bioconductor-rsamtools >=1.21.4', 'bioconductor-s4vectors >=0.7.1', 'bioconductor-zlibbioc', 'r-base', 'r-hwriter', 'r-latticeextra']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-biostrings >=2.37.1', 'bioconductor-genomeinfodb >=1.1.19', 'bioconductor-genomicalignments >=1.5.4', 'bioconductor-genomicranges >=1.21.6', 'bioconductor-iranges >=2.3.7', 'bioconductor-rsamtools >=1.21.4', 'bioconductor-s4vectors >=0.7.1', 'bioconductor-zlibbioc', 'r-base', 'r-hwriter', 'r-latticeextra'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ShortRead\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ShortRead.html'), ('license', 'Artistic-2.0'), ('summary', 'This package implements sampling, iteration, and input of FASTQ files. The package includes functions for filtering and trimming reads, and for generating a quality assessment report. Data are represented as DNAStringSet-derived objects, and easily manipulated for a diversity of purposes.  The package also contains legacy support for early single-end, ungapped alignment formats.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-shortread/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-shortread'), ('version', '1.32.0')])), ('source', CommentedMap([('fn', 'ShortRead_1.32.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ShortRead_1.32.0.tar.gz'), ('md5', '71d56164fd7995c0d257027bbd9e24d1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-biostrings >=2.37.1', 'bioconductor-genomeinfodb >=1.1.19', 'bioconductor-genomicalignments >=1.5.4', 'bioconductor-genomicranges >=1.21.6', 'bioconductor-iranges >=2.3.7', 'bioconductor-rsamtools >=1.21.4', 'bioconductor-s4vectors >=0.7.1', 'bioconductor-zlibbioc', 'r-base', 'r-hwriter', 'r-latticeextra']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.11.3', 'bioconductor-biocparallel', 'bioconductor-biostrings >=2.37.1', 'bioconductor-genomeinfodb >=1.1.19', 'bioconductor-genomicalignments >=1.5.4', 'bioconductor-genomicranges >=1.21.6', 'bioconductor-iranges >=2.3.7', 'bioconductor-rsamtools >=1.21.4', 'bioconductor-s4vectors >=0.7.1', 'bioconductor-zlibbioc', 'r-base', 'r-hwriter', 'r-latticeextra'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ShortRead\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ShortRead.html'), ('license', 'Artistic-2.0'), ('summary', 'This package implements sampling, iteration, and input of FASTQ files. The package includes functions for filtering and trimming reads, and for generating a quality assessment report. Data are represented as DNAStringSet-derived objects, and easily manipulated for a diversity of purposes.  The package also contains legacy support for early single-end, ungapped alignment formats.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mlp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mlp'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'MLP_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MLP_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/MLP_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/MLP/MLP_1.24.0_src_all.tar.gz']), ('md5', 'c23abefc67186aa4f9f349ebdce590d8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-annotationdbi', 'r-base', 'r-gdata', 'r-gmodels', 'r-gplots', 'r-gtools', 'r-plotrix']), ('run', ['bioconductor-affy', 'bioconductor-annotationdbi', 'r-base', 'r-gdata', 'r-gmodels', 'r-gplots', 'r-gtools', 'r-plotrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MLP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MLP.html'), ('license', 'GPL-3'), ('summary', 'Mean Log P Analysis')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mlp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mlp'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'MLP_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MLP_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/MLP_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/MLP/MLP_1.24.0_src_all.tar.gz']), ('md5', 'c23abefc67186aa4f9f349ebdce590d8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy', 'bioconductor-annotationdbi', 'r-base', 'r-gdata', 'r-gmodels', 'r-gplots', 'r-gtools', 'r-plotrix']), ('run', ['bioconductor-affy', 'bioconductor-annotationdbi', 'r-base', 'r-gdata', 'r-gmodels', 'r-gplots', 'r-gtools', 'r-plotrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MLP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MLP.html'), ('license', 'GPL-3'), ('summary', 'Mean Log P Analysis')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-dbd-mysql/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-dbd-mysql'), ('version', '4.033')])), ('source', CommentedMap([('fn', 'DBD-mysql-4.033.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.033.tar.gz')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-dbi', 'mysqlclient']), ('run', ['libgcc', 'perl-threaded', 'perl-dbi', 'mysqlclient'])])), ('test', CommentedMap([('imports', ['DBD::mysql'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/DBD::mysql'), ('license', 'Perl'), ('summary', 'MySQL driver for the Perl5 Database Interface (DBI)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-dbd-mysql/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-dbd-mysql'), ('version', '4.033')])), ('source', CommentedMap([('fn', 'DBD-mysql-4.033.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/C/CA/CAPTTOFU/DBD-mysql-4.033.tar.gz')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['gcc', 'perl-threaded', 'perl-app-cpanminus', 'perl-module-build', 'perl-dbi', 'mysqlclient']), ('run', ['libgcc', 'perl-threaded', 'perl-dbi', 'mysqlclient'])])), ('test', CommentedMap([('imports', ['DBD::mysql'])])), ('about', CommentedMap([('home', 'https://metacpan.org/pod/DBD::mysql'), ('license', 'Perl'), ('summary', 'MySQL driver for the Perl5 Database Interface (DBI)')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affypdnn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affypdnn'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'affypdnn_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/affypdnn_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/affypdnn_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/affypdnn/affypdnn_1.50.0_src_all.tar.gz']), ('md5', 'b041e6646942df5d9769a8c051fe075d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.5', 'r-base']), ('run', ['bioconductor-affy >=1.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affypdnn\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/affypdnn.html'), ('license', 'LGPL'), ('summary', 'The package contains functions to perform the PDNN method described by Li Zhang et al.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affypdnn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affypdnn'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'affypdnn_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/affypdnn_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/affypdnn_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/affypdnn/affypdnn_1.50.0_src_all.tar.gz']), ('md5', 'b041e6646942df5d9769a8c051fe075d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.5', 'r-base']), ('run', ['bioconductor-affy >=1.5', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affypdnn\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/affypdnn.html'), ('license', 'LGPL'), ('summary', 'The package contains functions to perform the PDNN method described by Li Zhang et al.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mvgst/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mvgst'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'mvGST_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mvGST_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/mvGST_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/mvGST/mvGST_1.10.0_src_all.tar.gz']), ('md5', '029ab61ae4876a87d952915b8528071a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-graph', 'bioconductor-rgraphviz', 'bioconductor-topgo', 'r-base', 'r-gprofiler', 'r-stringr']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-graph', 'bioconductor-rgraphviz', 'bioconductor-topgo', 'r-base', 'r-gprofiler', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mvGST\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mvGST.html'), ('license', 'GPL-3'), ('summary', 'mvGST provides platform-independent tools to identify GO terms (gene sets) that are differentially active (up or down) in multiple contrasts of interest.  Given a matrix of one-sided p-values (rows for genes, columns for contrasts), mvGST uses meta-analytic methods to combine p-values for all genes annotated to each gene set, and then classify each gene set as being significantly more active (1), less active (-1), or not significantly differentially active (0) in each contrast of interest.  With multiple contrasts of interest, each gene set is assigned to a profile (across contrasts) of differential activity.  Tools are also provided for visualizing (in a GO graph) the gene sets classified to a given profile.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mvgst/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mvgst'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'mvGST_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mvGST_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/mvGST_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/mvGST/mvGST_1.10.0_src_all.tar.gz']), ('md5', '029ab61ae4876a87d952915b8528071a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-graph', 'bioconductor-rgraphviz', 'bioconductor-topgo', 'r-base', 'r-gprofiler', 'r-stringr']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-go.db', 'bioconductor-gostats', 'bioconductor-graph', 'bioconductor-rgraphviz', 'bioconductor-topgo', 'r-base', 'r-gprofiler', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mvGST\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mvGST.html'), ('license', 'GPL-3'), ('summary', 'mvGST provides platform-independent tools to identify GO terms (gene sets) that are differentially active (up or down) in multiple contrasts of interest.  Given a matrix of one-sided p-values (rows for genes, columns for contrasts), mvGST uses meta-analytic methods to combine p-values for all genes annotated to each gene set, and then classify each gene set as being significantly more active (1), less active (-1), or not significantly differentially active (0) in each contrast of interest.  With multiple contrasts of interest, each gene set is assigned to a profile (across contrasts) of differential activity.  Tools are also provided for visualizing (in a GO graph) the gene sets classified to a given profile.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-otubase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-otubase'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'OTUbase_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OTUbase_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/OTUbase_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/OTUbase/OTUbase_1.26.0_src_all.tar.gz']), ('md5', 'c5890afc116654b4d243ca6a60e49c35')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-shortread >=1.23.15', 'r-base', 'r-vegan']), ('run', ['bioconductor-biobase', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-shortread >=1.23.15', 'r-base', 'r-vegan'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OTUbase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OTUbase.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides a platform for Operational Taxonomic Unit based analysis')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-otubase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-otubase'), ('version', '1.26.0')])), ('source', CommentedMap([('fn', 'OTUbase_1.26.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/OTUbase_1.26.0.tar.gz', 'https://bioarchive.galaxyproject.org/OTUbase_1.26.0.tar.gz', 'https://depot.galaxyproject.org/software/OTUbase/OTUbase_1.26.0_src_all.tar.gz']), ('md5', 'c5890afc116654b4d243ca6a60e49c35')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-shortread >=1.23.15', 'r-base', 'r-vegan']), ('run', ['bioconductor-biobase', 'bioconductor-biostrings', 'bioconductor-iranges', 'bioconductor-s4vectors', 'bioconductor-shortread >=1.23.15', 'r-base', 'r-vegan'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'OTUbase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/OTUbase.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides a platform for Operational Taxonomic Unit based analysis')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgraph2js/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgraph2js'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'RGraph2js_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RGraph2js_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/RGraph2js_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/RGraph2js/RGraph2js_1.4.0_src_all.tar.gz']), ('md5', '8c976e7f4782606ff7c209333c47ecf1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base', 'r-digest', 'r-rjson', 'r-whisker']), ('run', ['bioconductor-graph', 'r-base', 'r-digest', 'r-rjson', 'r-whisker'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RGraph2js\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RGraph2js.html'), ('license', 'GPL-2'), ('summary', 'Generator of web pages which display interactive network/graph visualizations with D3js, jQuery and Raphael.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rgraph2js/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rgraph2js'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'RGraph2js_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RGraph2js_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/RGraph2js_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/RGraph2js/RGraph2js_1.4.0_src_all.tar.gz']), ('md5', '8c976e7f4782606ff7c209333c47ecf1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base', 'r-digest', 'r-rjson', 'r-whisker']), ('run', ['bioconductor-graph', 'r-base', 'r-digest', 'r-rjson', 'r-whisker'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RGraph2js\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RGraph2js.html'), ('license', 'GPL-2'), ('summary', 'Generator of web pages which display interactive network/graph visualizations with D3js, jQuery and Raphael.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sseq'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'sSeq_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sSeq_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/sSeq_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/sSeq/sSeq_1.14.0_src_all.tar.gz']), ('md5', 'acd8bce328fafcfdc09ae23091ce85a1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-catools', 'r-rcolorbrewer']), ('run', ['r-base', 'r-catools', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sSeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'The purpose of this package is to discover the genes that are differentially expressed between two conditions in RNA-seq experiments. Gene expression is measured in counts of transcripts and modeled with the Negative Binomial (NB) distribution using a shrinkage approach for dispersion estimation. The method of moment (MM) estimates for dispersion are shrunk towards an estimated target, which minimizes the average squared difference between the shrinkage estimates and the initial estimates. The exact per-gene probability under the NB model is calculated, and used to test the hypothesis that the expected expression of a gene in two conditions identically follow a NB distribution.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sseq'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'sSeq_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sSeq_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/sSeq_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/sSeq/sSeq_1.14.0_src_all.tar.gz']), ('md5', 'acd8bce328fafcfdc09ae23091ce85a1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-catools', 'r-rcolorbrewer']), ('run', ['r-base', 'r-catools', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sSeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sSeq.html'), ('license', 'GPL (>= 3)'), ('summary', 'The purpose of this package is to discover the genes that are differentially expressed between two conditions in RNA-seq experiments. Gene expression is measured in counts of transcripts and modeled with the Negative Binomial (NB) distribution using a shrinkage approach for dispersion estimation. The method of moment (MM) estimates for dispersion are shrunk towards an estimated target, which minimizes the average squared difference between the shrinkage estimates and the initial estimates. The exact per-gene probability under the NB model is calculated, and used to test the hypothesis that the expected expression of a gene in two conditions identically follow a NB distribution.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metacca/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metacca'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'metaCCA_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/metaCCA_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/metaCCA_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/metaCCA/metaCCA_1.4.0_src_all.tar.gz']), ('md5', '9623dba2a19fa1d8b9998506c1fd07dd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metaCCA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/metaCCA.html'), ('license', 'MIT + file LICENSE'), ('summary', 'metaCCA performs multivariate analysis of a single or multiple GWAS based on univariate regression coefficients. It allows multivariate representation of both phenotype and genotype. metaCCA extends the statistical technique of canonical correlation analysis to the setting where original individual-level records are not available, and employs a covariance shrinkage algorithm to achieve robustness.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-metacca/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-metacca'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'metaCCA_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/metaCCA_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/metaCCA_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/metaCCA/metaCCA_1.4.0_src_all.tar.gz']), ('md5', '9623dba2a19fa1d8b9998506c1fd07dd')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'metaCCA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/metaCCA.html'), ('license', 'MIT + file LICENSE'), ('summary', 'metaCCA performs multivariate analysis of a single or multiple GWAS based on univariate regression coefficients. It allows multivariate representation of both phenotype and genotype. metaCCA extends the statistical technique of canonical correlation analysis to the setting where original individual-level records are not available, and employs a covariance shrinkage algorithm to achieve robustness.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-vcftools-vcf/0.840/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-vcftools-vcf'), ('version', '0.840')])), ('source', CommentedMap([('fn', 'vcftools-0.1.11.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/vcftools/vcftools_0.1.11.tar.gz'), ('sha256', '81b98c8bc93e366e2f0b14eb78e2736ae1c790176939eef88dbf0546ab952aab')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Vcf;\"'])])), ('about', CommentedMap([('home', 'https://github.com/vcftools/vcftools'), ('license', 'LGPLv3'), ('summary', 'cpanm ready distribution of VCFtools Perl libraries')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-vcftools-vcf/0.840/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-vcftools-vcf'), ('version', '0.840')])), ('source', CommentedMap([('fn', 'vcftools-0.1.11.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/vcftools/vcftools_0.1.11.tar.gz'), ('sha256', '81b98c8bc93e366e2f0b14eb78e2736ae1c790176939eef88dbf0546ab952aab')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'perl'])])), ('test', CommentedMap([('commands', ['perl -e \"use Vcf;\"'])])), ('about', CommentedMap([('home', 'https://github.com/vcftools/vcftools'), ('license', 'LGPLv3'), ('summary', 'cpanm ready distribution of VCFtools Perl libraries')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-vsn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-vsn'), ('version', '3.38.0')])), ('source', CommentedMap([('fn', 'vsn_3.38.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/vsn_3.38.0.tar.gz'), ('md5', '1acea08a752560f8c187a2eb010567f7')])), ('build', CommentedMap([('skip', False), ('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biobase', 'bioconductor-limma', 'bioconductor-affy', 'r-ggplot2', 'r-hexbin']), ('run', ['r-base', 'bioconductor-biobase', 'bioconductor-limma', 'bioconductor-affy', 'r-ggplot2', 'r-hexbin'])])), ('test', CommentedMap([('commands', ['$R -e \"require(\\'vsn\\')\"'])])), ('about', CommentedMap([('home', 'http://www.bioconductor.org/packages/release/bioc/html/vsn.html'), ('license', 'GPL (>=2)'), ('summary', 'Data analysis, linear models and differential expression for microarray data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-vsn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-vsn'), ('version', '3.38.0')])), ('source', CommentedMap([('fn', 'vsn_3.38.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/vsn_3.38.0.tar.gz'), ('md5', '1acea08a752560f8c187a2eb010567f7')])), ('build', CommentedMap([('skip', False), ('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biobase', 'bioconductor-limma', 'bioconductor-affy', 'r-ggplot2', 'r-hexbin']), ('run', ['r-base', 'bioconductor-biobase', 'bioconductor-limma', 'bioconductor-affy', 'r-ggplot2', 'r-hexbin'])])), ('test', CommentedMap([('commands', ['$R -e \"require(\\'vsn\\')\"'])])), ('about', CommentedMap([('home', 'http://www.bioconductor.org/packages/release/bioc/html/vsn.html'), ('license', 'GPL (>=2)'), ('summary', 'Data analysis, linear models and differential expression for microarray data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ms/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ms'), ('version', '2014_03_04')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc'])])), ('source', CommentedMap([('fn', 'ms.tar.gz'), ('url', 'https://webshare.uchicago.edu/users/rhudson1/Public/ms.folder/ms.tar.gz'), ('md5', 'ef693dcc424fd5293b0bea4c68953700')])), ('test', CommentedMap([('commands', ['ms 2 1 -T > /dev/null'])])), ('about', CommentedMap([('home', 'http://home.uchicago.edu/rhudson1/source/mksamples.html'), ('license', 'Unknown'), ('summary', 'Generates random independent samples according to a simple Wright-Fisher neutral model.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ms/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ms'), ('version', '2014_03_04')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc'])])), ('source', CommentedMap([('fn', 'ms.tar.gz'), ('url', 'https://webshare.uchicago.edu/users/rhudson1/Public/ms.folder/ms.tar.gz'), ('md5', 'ef693dcc424fd5293b0bea4c68953700')])), ('test', CommentedMap([('commands', ['ms 2 1 -T > /dev/null'])])), ('about', CommentedMap([('home', 'http://home.uchicago.edu/rhudson1/source/mksamples.html'), ('license', 'Unknown'), ('summary', 'Generates random independent samples according to a simple Wright-Fisher neutral model.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqtk/r82/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'seqtk'), ('version', 'r82')])), ('source', CommentedMap([('fn', 'seqtk-r82.tar.gz'), ('url', 'https://github.com/lh3/seqtk/archive/5e1e8dbd506ea1ff8c77d468a1f27b8e8f73eac0.tar.gz'), ('sha256', 'dd4b54a3ba57279659153204aa59aa1d5efe4d307549ac7b8e93b67eb9d3248f')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/seqtk'), ('license', 'MIT License'), ('summary', 'Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format')])), ('test', CommentedMap([('commands', ['seqtk seq'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqtk/r82/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'seqtk'), ('version', 'r82')])), ('source', CommentedMap([('fn', 'seqtk-r82.tar.gz'), ('url', 'https://github.com/lh3/seqtk/archive/5e1e8dbd506ea1ff8c77d468a1f27b8e8f73eac0.tar.gz'), ('sha256', 'dd4b54a3ba57279659153204aa59aa1d5efe4d307549ac7b8e93b67eb9d3248f')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['libgcc', 'zlib'])])), ('about', CommentedMap([('home', 'https://github.com/lh3/seqtk'), ('license', 'MIT License'), ('summary', 'Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format')])), ('test', CommentedMap([('commands', ['seqtk seq'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sff2fastq/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/indraniel/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['sff2fastq -v'])])), ('about', CommentedMap([('home', 'https://github.com/indraniel/sff2fastq'), ('license', 'GPL-3'), ('license_family', 'GPL'), ('summary', 'Extract 454 Genome Sequencer reads from a SFF file and convert them into a FASTQ formatted output'), ('description', 'The program sff2fastq extracts read information from a SFF file, produced by\\nthe 454 genome sequencer, and outputs the sequences and quality scores in a\\nFASTQ format.\\n')])), ('extra', CommentedMap([('recipe-maintainers', ['keuv-grvl'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sff2fastq/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'https://github.com/indraniel/<{ name }}/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('md5', '<{ md5 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['sff2fastq -v'])])), ('about', CommentedMap([('home', 'https://github.com/indraniel/sff2fastq'), ('license', 'GPL-3'), ('license_family', 'GPL'), ('summary', 'Extract 454 Genome Sequencer reads from a SFF file and convert them into a FASTQ formatted output'), ('description', 'The program sff2fastq extracts read information from a SFF file, produced by\\nthe 454 genome sequencer, and outputs the sequences and quality scores in a\\nFASTQ format.\\n')])), ('extra', CommentedMap([('recipe-maintainers', ['keuv-grvl'])]))]))\n",
      "EXTRA has been found: sff2fastq\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sizepower/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sizepower'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'sizepower_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sizepower_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/sizepower_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/sizepower/sizepower_1.46.0_src_all.tar.gz']), ('md5', '98c92ff897d7b422932cd648018532c7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sizepower\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sizepower.html'), ('license', 'LGPL'), ('summary', 'This package has been prepared to assist users in computing either a sample size or power value for a microarray experimental study. The user is referred to the cited references for technical background on the methodology underpinning these calculations. This package provides support for five types of sample size and power calculations. These five types can be adapted in various ways to encompass many of the standard designs encountered in practice.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sizepower/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sizepower'), ('version', '1.46.0')])), ('source', CommentedMap([('fn', 'sizepower_1.46.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sizepower_1.46.0.tar.gz', 'https://bioarchive.galaxyproject.org/sizepower_1.46.0.tar.gz', 'https://depot.galaxyproject.org/software/sizepower/sizepower_1.46.0_src_all.tar.gz']), ('md5', '98c92ff897d7b422932cd648018532c7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sizepower\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sizepower.html'), ('license', 'LGPL'), ('summary', 'This package has been prepared to assist users in computing either a sample size or power value for a microarray experimental study. The user is referred to the cited references for technical background on the methodology underpinning these calculations. This package provides support for five types of sample size and power calculations. These five types can be adapted in various ways to encompass many of the standard designs encountered in practice.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rimmport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rimmport'), ('version', '1.4.2')])), ('source', CommentedMap([('fn', 'RImmPort_1.4.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RImmPort_1.4.2.tar.gz', 'https://depot.galaxyproject.org/software/RImmPort/RImmPort_1.4.2_src_all.tar.gz']), ('md5', '26cb626afa74b29ca5219e078b1ce8f2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-data.table', 'r-dbi', 'r-dplyr', 'r-plyr', 'r-reshape2', 'r-rsqlite', 'r-sqldf']), ('run', ['r-base', 'r-data.table', 'r-dbi', 'r-dplyr', 'r-plyr', 'r-reshape2', 'r-rsqlite', 'r-sqldf'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RImmPort\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RImmPort.html'), ('license', 'GPL-3'), ('summary', 'The RImmPort package simplifies access to ImmPort data for analysis in the R environment. It provides a standards-based interface to the ImmPort study data that is in a proprietary format.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rimmport/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rimmport'), ('version', '1.4.2')])), ('source', CommentedMap([('fn', 'RImmPort_1.4.2.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RImmPort_1.4.2.tar.gz', 'https://depot.galaxyproject.org/software/RImmPort/RImmPort_1.4.2_src_all.tar.gz']), ('md5', '26cb626afa74b29ca5219e078b1ce8f2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-data.table', 'r-dbi', 'r-dplyr', 'r-plyr', 'r-reshape2', 'r-rsqlite', 'r-sqldf']), ('run', ['r-base', 'r-data.table', 'r-dbi', 'r-dplyr', 'r-plyr', 'r-reshape2', 'r-rsqlite', 'r-sqldf'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RImmPort\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RImmPort.html'), ('license', 'GPL-3'), ('summary', 'The RImmPort package simplifies access to ImmPort data for analysis in the R environment. It provides a standards-based interface to the ImmPort study data that is in a proprietary format.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gseabase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gseabase'), ('version', '1.34.1')])), ('source', CommentedMap([('fn', 'GSEABase_1.34.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/GSEABase_1.34.1.tar.gz'), ('md5', '26f0a512fddadcb36162dd80fcf8a2d4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate >=1.45.3', 'bioconductor-annotationdbi', 'bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.13.8', 'bioconductor-graph >=1.37.2', 'r-base', 'r-xml']), ('run', ['bioconductor-annotate >=1.45.3', 'bioconductor-annotationdbi', 'bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.13.8', 'bioconductor-graph >=1.37.2', 'r-base', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GSEABase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GSEABase.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides classes and methods to support Gene Set Enrichment Analysis (GSEA).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gseabase/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gseabase'), ('version', '1.34.1')])), ('source', CommentedMap([('fn', 'GSEABase_1.34.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/GSEABase_1.34.1.tar.gz'), ('md5', '26f0a512fddadcb36162dd80fcf8a2d4')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate >=1.45.3', 'bioconductor-annotationdbi', 'bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.13.8', 'bioconductor-graph >=1.37.2', 'r-base', 'r-xml']), ('run', ['bioconductor-annotate >=1.45.3', 'bioconductor-annotationdbi', 'bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.13.8', 'bioconductor-graph >=1.37.2', 'r-base', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GSEABase\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GSEABase.html'), ('license', 'Artistic-2.0'), ('summary', 'This package provides classes and methods to support Gene Set Enrichment Analysis (GSEA).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ensembldb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ensembldb'), ('version', '1.6.2')])), ('source', CommentedMap([('fn', 'ensembldb_1.6.0.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/ensembldb_1.6.2.tar.gz'), ('sha256', '79b8f50cf2ddb7af0de7479a045f6cc57d763286dcc495bf932bce37e995bbc3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-annotationhub', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.23.18', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-annotationhub', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.23.18', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ensembldb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ensembldb.html'), ('license', 'LGPL'), ('summary', 'The package provides functions to create and use transcript centric annotation databases/packages. The annotation for the databases are directly fetched from Ensembl using their Perl API. The functionality and data is similar to that of the TxDb packages from the GenomicFeatures package, but, in addition to retrieve all gene/transcript models and annotations from the database, the ensembldb package provides also a filter framework allowing to retrieve annotations for specific entries like genes encoded on a chromosome region or transcript models of lincRNA genes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ensembldb/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ensembldb'), ('version', '1.6.2')])), ('source', CommentedMap([('fn', 'ensembldb_1.6.0.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/ensembldb_1.6.2.tar.gz'), ('sha256', '79b8f50cf2ddb7af0de7479a045f6cc57d763286dcc495bf932bce37e995bbc3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-annotationhub', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.23.18', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite']), ('run', ['bioconductor-annotationdbi >=1.31.19', 'bioconductor-annotationhub', 'bioconductor-biobase', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.23.18', 'bioconductor-genomicranges >=1.23.21', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'r-base', 'r-dbi', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ensembldb\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ensembldb.html'), ('license', 'LGPL'), ('summary', 'The package provides functions to create and use transcript centric annotation databases/packages. The annotation for the databases are directly fetched from Ensembl using their Perl API. The functionality and data is similar to that of the TxDb packages from the GenomicFeatures package, but, in addition to retrieve all gene/transcript models and annotations from the database, the ensembldb package provides also a filter framework allowing to retrieve annotations for specific entries like genes encoded on a chromosome region or transcript models of lincRNA genes.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r2r/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r2r'), ('version', '1.0.4')])), ('about', CommentedMap([('home', 'http://breaker.research.yale.edu/R2R/'), ('license', 'GPL-2'), ('license_file', 'LICENSE'), ('summary', 'software to speed depiction of aesthetic consensus RNA secondary structures')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'R2R-1.0.4.tgz'), ('url', 'http://breaker.research.yale.edu/R2R/R2R-1.0.4.tgz'), ('sha256', '3578c8ad5dfc2a4e6c4f0613ca3e98a1f352af661f60886a1616c5e6d8e7440d')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc', 'perl-threaded', 'perl-clone']), ('run', ['libgcc', 'perl-threaded', 'perl-clone'])])), ('test', CommentedMap([('commands', ['r2r --GSC-weighted-consensus $PREFIX/share/demo1.sto $PREFIX/share/demo1.cons.sto 3 0.97 0.9 0.75 4 0.97 0.9 0.75 0.5 0.1'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r2r/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r2r'), ('version', '1.0.4')])), ('about', CommentedMap([('home', 'http://breaker.research.yale.edu/R2R/'), ('license', 'GPL-2'), ('license_file', 'LICENSE'), ('summary', 'software to speed depiction of aesthetic consensus RNA secondary structures')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'R2R-1.0.4.tgz'), ('url', 'http://breaker.research.yale.edu/R2R/R2R-1.0.4.tgz'), ('sha256', '3578c8ad5dfc2a4e6c4f0613ca3e98a1f352af661f60886a1616c5e6d8e7440d')])), ('requirements', CommentedMap([('build', ['llvm', 'gcc', 'perl-threaded', 'perl-clone']), ('run', ['libgcc', 'perl-threaded', 'perl-clone'])])), ('test', CommentedMap([('commands', ['r2r --GSC-weighted-consensus $PREFIX/share/demo1.sto $PREFIX/share/demo1.cons.sto 3 0.97 0.9 0.75 4 0.97 0.9 0.75 0.5 0.1'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghmcr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghmcr'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'cghMCR_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/cghMCR_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/cghMCR_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/cghMCR/cghMCR_1.34.0_src_all.tar.gz']), ('md5', '6c46dd1d8db2d9bbd8a7481da8d85523')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.6', 'bioconductor-cntools', 'bioconductor-dnacopy', 'bioconductor-limma', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.1.6', 'bioconductor-cntools', 'bioconductor-dnacopy', 'bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cghMCR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/cghMCR.html'), ('license', 'LGPL'), ('summary', 'This package provides functions to identify genomic regions of interests based on segmented copy number data from multiple samples.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cghmcr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cghmcr'), ('version', '1.34.0')])), ('source', CommentedMap([('fn', 'cghMCR_1.34.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/cghMCR_1.34.0.tar.gz', 'https://bioarchive.galaxyproject.org/cghMCR_1.34.0.tar.gz', 'https://depot.galaxyproject.org/software/cghMCR/cghMCR_1.34.0_src_all.tar.gz']), ('md5', '6c46dd1d8db2d9bbd8a7481da8d85523')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.6', 'bioconductor-cntools', 'bioconductor-dnacopy', 'bioconductor-limma', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.1.6', 'bioconductor-cntools', 'bioconductor-dnacopy', 'bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cghMCR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/cghMCR.html'), ('license', 'LGPL'), ('summary', 'This package provides functions to identify genomic regions of interests based on segmented copy number data from multiple samples.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/paralyzer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'paralyzer'), ('version', 1.5)])), ('source', CommentedMap([('fn', 'PARalyzer_v1_5.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/duke/PARalyzer/PARalyzer_v1_5.tar.gz'), ('md5', '09d82ffbda7787a6f16553576b40e6ca')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['java-jdk']), ('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['PARalyzer 2G'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/PARalyzer_85/'), ('license', 'PARalyzer is provided for academic use only, if you wish to use it in another setting please contact Uwe Ohler.'), ('summary', 'We developed the PARalyzer algorithm to generate a high resolution map of interaction sites between RNA-binding proteins and their targets. The algorithm utilizes the deep sequencing reads generated by PAR-CLIP (Photoactivatable-Ribonucleoside-Enhanced Crosslinking and Immunoprecipitation) protocol.The use of photoactivatable nucleotides in the PAR-CLIP protocol results in more efficient crosslinking between the RNA-binding protein and its target relative to other CLIP methods; in addition a nucleotide substitution occurs at the site of crosslinking, providing for single-nucleotide resolution binding information. PARalyzer utilizes this nucleotide substition in a kernel density estimate classifier to generate the high resolution set of Protein-RNA interaction sites.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/paralyzer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'paralyzer'), ('version', 1.5)])), ('source', CommentedMap([('fn', 'PARalyzer_v1_5.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/duke/PARalyzer/PARalyzer_v1_5.tar.gz'), ('md5', '09d82ffbda7787a6f16553576b40e6ca')])), ('build', CommentedMap([('skip', True), ('number', 0)])), ('requirements', CommentedMap([('build', ['java-jdk']), ('run', ['java-jdk'])])), ('test', CommentedMap([('commands', ['PARalyzer 2G'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/PARalyzer_85/'), ('license', 'PARalyzer is provided for academic use only, if you wish to use it in another setting please contact Uwe Ohler.'), ('summary', 'We developed the PARalyzer algorithm to generate a high resolution map of interaction sites between RNA-binding proteins and their targets. The algorithm utilizes the deep sequencing reads generated by PAR-CLIP (Photoactivatable-Ribonucleoside-Enhanced Crosslinking and Immunoprecipitation) protocol.The use of photoactivatable nucleotides in the PAR-CLIP protocol results in more efficient crosslinking between the RNA-binding protein and its target relative to other CLIP methods; in addition a nucleotide substitution occurs at the site of crosslinking, providing for single-nucleotide resolution binding information. PARalyzer utilizes this nucleotide substition in a kernel density estimate classifier to generate the high resolution set of Protein-RNA interaction sites.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-transview/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-transview'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'TransView_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TransView_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/TransView_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/TransView/TransView_1.20.0_src_all.tar.gz']), ('md5', '634dc44651702b77eb7c5e7dc9718656')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools >=1.19.38', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-zlibbioc', 'r-base', 'r-gplots']), ('run', ['bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools >=1.19.38', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-zlibbioc', 'r-base', 'r-gplots'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TransView\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TransView.html'), ('license', 'GPL-3'), ('summary', 'This package provides efficient tools to generate, access and display read densities of sequencing based data sets such as from RNA-Seq and ChIP-Seq.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-transview/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-transview'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'TransView_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/TransView_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/TransView_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/TransView/TransView_1.20.0_src_all.tar.gz']), ('md5', '634dc44651702b77eb7c5e7dc9718656')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools >=1.19.38', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-zlibbioc', 'r-base', 'r-gplots']), ('run', ['bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools >=1.19.38', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-zlibbioc', 'r-base', 'r-gplots'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'TransView\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/TransView.html'), ('license', 'GPL-3'), ('summary', 'This package provides efficient tools to generate, access and display read densities of sequencing based data sets such as from RNA-Seq and ChIP-Seq.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-alpine/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-alpine'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'alpine_1.0.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-alpine/bioconductor-alpine_1.0.0_src_all.tar.gz'), ('md5', 'f70ca32c5336aca7745ee3875bbbf28f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-graph', 'bioconductor-iranges', 'bioconductor-rbgl', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-speedglm', 'r-stringr']), ('run', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-graph', 'bioconductor-iranges', 'bioconductor-rbgl', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-speedglm', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'alpine\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/alpine.html'), ('license', 'GPL (>=2)'), ('summary', 'Fragment sequence bias modeling and correction for RNA-seq transcript abundance estimation.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-alpine/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-alpine'), ('version', '1.0.0')])), ('source', CommentedMap([('fn', 'alpine_1.0.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-alpine/bioconductor-alpine_1.0.0_src_all.tar.gz'), ('md5', 'f70ca32c5336aca7745ee3875bbbf28f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-graph', 'bioconductor-iranges', 'bioconductor-rbgl', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-speedglm', 'r-stringr']), ('run', ['bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-graph', 'bioconductor-iranges', 'bioconductor-rbgl', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base', 'r-speedglm', 'r-stringr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'alpine\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/alpine.html'), ('license', 'GPL (>=2)'), ('summary', 'Fragment sequence bias modeling and correction for RNA-seq transcript abundance estimation.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/exonerate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'exonerate'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'exonerate-<{ version }}.tar.gz'), ('url', 'http://ftp.ebi.ac.uk/pub/software/vertebrategenomics/exonerate/exonerate-<{ version }}.tar.gz'), ('md5', '126fbade003b80b663a1d530c56f1904')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'glib', 'pcre', 'pkg-config', 'automake', 'autoconf']), ('run', ['libgcc', 'glib', 'pcre'])])), ('test', CommentedMap([('commands', ['exonerate 2>&1 | grep \"exonerate: A generic sequence comparison tool\"'])])), ('about', CommentedMap([('home', 'https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate'), ('license', 'GPL-3.0'), ('summary', 'Exonerate - A generic tool for pairwise sequence comparison / alignment')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/exonerate/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'exonerate'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'exonerate-<{ version }}.tar.gz'), ('url', 'http://ftp.ebi.ac.uk/pub/software/vertebrategenomics/exonerate/exonerate-<{ version }}.tar.gz'), ('md5', '126fbade003b80b663a1d530c56f1904')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'glib', 'pcre', 'pkg-config', 'automake', 'autoconf']), ('run', ['libgcc', 'glib', 'pcre'])])), ('test', CommentedMap([('commands', ['exonerate 2>&1 | grep \"exonerate: A generic sequence comparison tool\"'])])), ('about', CommentedMap([('home', 'https://www.ebi.ac.uk/about/vertebrate-genomics/software/exonerate'), ('license', 'GPL-3.0'), ('summary', 'Exonerate - A generic tool for pairwise sequence comparison / alignment')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cnvkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cnvkit'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'cnvkit-4c94834.tar.gz'), ('url', 'https://github.com/etal/cnvkit/archive/4c94834.tar.gz'), ('md5', 'ee3e12b06673a3f64a019c05e35c4993')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'atlas', 'numpy >=1.9', 'scipy >=0.9', 'pandas >=0.18.1', 'matplotlib >=1.1', 'biopython >=1.62', 'reportlab >=3.0', 'pysam >=0.9.1.4', 'r-pscbs', 'r-cghflasso', 'pyfaidx >=0.4.7', 'future >=0.15.2', 'futures >=3.0'])])), ('test', CommentedMap([('imports', ['cnvlib', 'skgenome']), ('commands', ['cnvkit.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/etal/cnvkit'), ('license', 'Apache License 2.0'), ('summary', 'Copy number variant detection from targeted DNA sequencing')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/cnvkit/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'cnvkit'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'cnvkit-4c94834.tar.gz'), ('url', 'https://github.com/etal/cnvkit/archive/4c94834.tar.gz'), ('md5', 'ee3e12b06673a3f64a019c05e35c4993')])), ('build', CommentedMap([('number', 2)])), ('requirements', CommentedMap([('build', ['python', 'setuptools']), ('run', ['python', 'atlas', 'numpy >=1.9', 'scipy >=0.9', 'pandas >=0.18.1', 'matplotlib >=1.1', 'biopython >=1.62', 'reportlab >=3.0', 'pysam >=0.9.1.4', 'r-pscbs', 'r-cghflasso', 'pyfaidx >=0.4.7', 'future >=0.15.2', 'futures >=3.0'])])), ('test', CommentedMap([('imports', ['cnvlib', 'skgenome']), ('commands', ['cnvkit.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/etal/cnvkit'), ('license', 'Apache License 2.0'), ('summary', 'Copy number variant detection from targeted DNA sequencing')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-beadarraysnp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-beadarraysnp'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'beadarraySNP_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/beadarraySNP_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/beadarraySNP_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/beadarraySNP/beadarraySNP_1.42.0_src_all.tar.gz']), ('md5', '54396dead6d9f58a76adabd9d938aaf3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.14', 'bioconductor-quantsmooth', 'r-base']), ('run', ['bioconductor-biobase >=2.14', 'bioconductor-quantsmooth', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'beadarraySNP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/beadarraySNP.html'), ('license', 'GPL-2'), ('summary', 'Importing data from Illumina SNP experiments and performing copy number calculations and reports.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-beadarraysnp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-beadarraysnp'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'beadarraySNP_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/beadarraySNP_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/beadarraySNP_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/beadarraySNP/beadarraySNP_1.42.0_src_all.tar.gz']), ('md5', '54396dead6d9f58a76adabd9d938aaf3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.14', 'bioconductor-quantsmooth', 'r-base']), ('run', ['bioconductor-biobase >=2.14', 'bioconductor-quantsmooth', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'beadarraySNP\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/beadarraySNP.html'), ('license', 'GPL-2'), ('summary', 'Importing data from Illumina SNP experiments and performing copy number calculations and reports.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rpx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rpx'), ('version', '1.12.1')])), ('source', CommentedMap([('fn', 'rpx_1.12.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rpx_1.12.1.tar.gz', 'https://depot.galaxyproject.org/software/rpx/rpx_1.12.1_src_all.tar.gz']), ('md5', '4174cd01da022a04523e8745204f28a3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcurl', 'r-xml2']), ('run', ['r-base', 'r-rcurl', 'r-xml2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rpx\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rpx.html'), ('license', 'GPL-2'), ('summary', 'This package implements an interface to proteomics data submitted to the ProteomeXchange consortium.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rpx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rpx'), ('version', '1.12.1')])), ('source', CommentedMap([('fn', 'rpx_1.12.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rpx_1.12.1.tar.gz', 'https://depot.galaxyproject.org/software/rpx/rpx_1.12.1_src_all.tar.gz']), ('md5', '4174cd01da022a04523e8745204f28a3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcurl', 'r-xml2']), ('run', ['r-base', 'r-rcurl', 'r-xml2'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rpx\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rpx.html'), ('license', 'GPL-2'), ('summary', 'This package implements an interface to proteomics data submitted to the ProteomeXchange consortium.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocgenerics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocgenerics'), ('version', '0.22.0')])), ('source', CommentedMap([('fn', 'BiocGenerics_0.22.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BiocGenerics_0.22.0.tar.gz'), ('md5', '8de9aa737dfca9b53404a1b32169b007')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiocGenerics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BiocGenerics.html'), ('license', 'Artistic-2.0'), ('summary', 'S4 generic functions needed by many Bioconductor packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-biocgenerics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-biocgenerics'), ('version', '0.22.0')])), ('source', CommentedMap([('fn', 'BiocGenerics_0.22.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/BiocGenerics_0.22.0.tar.gz'), ('md5', '8de9aa737dfca9b53404a1b32169b007')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BiocGenerics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/BiocGenerics.html'), ('license', 'Artistic-2.0'), ('summary', 'S4 generic functions needed by many Bioconductor packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-geoquery/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-geoquery'), ('version', '2.38.4')])), ('source', CommentedMap([('fn', 'GEOquery_2.38.4.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/GEOquery_2.38.4.tar.gz'), ('md5', '54c96cb6d2b4eab9421b7d47180f26d6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-httr', 'r-rcurl', 'r-xml']), ('run', ['bioconductor-biobase', 'r-base', 'r-httr', 'r-rcurl', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GEOquery\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GEOquery.html'), ('license', 'GPL-2'), ('summary', 'The NCBI Gene Expression Omnibus (GEO) is a public repository of microarray data.  Given the rich and varied nature of this resource, it is only natural to want to apply BioConductor tools to these data.  GEOquery is the bridge between GEO and BioConductor.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-geoquery/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-geoquery'), ('version', '2.38.4')])), ('source', CommentedMap([('fn', 'GEOquery_2.38.4.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/GEOquery_2.38.4.tar.gz'), ('md5', '54c96cb6d2b4eab9421b7d47180f26d6')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-httr', 'r-rcurl', 'r-xml']), ('run', ['bioconductor-biobase', 'r-base', 'r-httr', 'r-rcurl', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GEOquery\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/GEOquery.html'), ('license', 'GPL-2'), ('summary', 'The NCBI Gene Expression Omnibus (GEO) is a public repository of microarray data.  Given the rich and varied nature of this resource, it is only natural to want to apply BioConductor tools to these data.  GEOquery is the bridge between GEO and BioConductor.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-riboseqr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-riboseqr'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'riboSeqR_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/riboSeqR_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/riboSeqR_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/riboSeqR/riboSeqR_1.10.0_src_all.tar.gz']), ('md5', '422b50a08dd8ba4b1e7bde5935806998')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bayseq', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-seqlogo', 'r-abind', 'r-base']), ('run', ['bioconductor-bayseq', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-seqlogo', 'r-abind', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'riboSeqR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/riboSeqR.html'), ('license', 'GPL-3'), ('summary', 'Plotting functions, frameshift detection and parsing of sequencing data from ribosome profiling experiments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-riboseqr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-riboseqr'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'riboSeqR_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/riboSeqR_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/riboSeqR_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/riboSeqR/riboSeqR_1.10.0_src_all.tar.gz']), ('md5', '422b50a08dd8ba4b1e7bde5935806998')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-bayseq', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-seqlogo', 'r-abind', 'r-base']), ('run', ['bioconductor-bayseq', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-seqlogo', 'r-abind', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'riboSeqR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/riboSeqR.html'), ('license', 'GPL-3'), ('summary', 'Plotting functions, frameshift detection and parsing of sequencing data from ribosome profiling experiments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-consensusclusterplus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-consensusclusterplus'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'ConsensusClusterPlus_1.40.0.tar.gz'), ('url', 'https://bioconductor.org/packages/release/bioc/src/contrib/ConsensusClusterPlus_1.40.0.tar.gz'), ('md5', 'ae9231f43df080d1f37d5c23719051fe')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-all', 'r-base', 'r-cluster']), ('run', ['bioconductor-biobase', 'bioconductor-all', 'r-base', 'r-cluster'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ConsensusClusterPlus\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html'), ('license', 'GPL-2'), ('summary', 'Algorithm for determining cluster count and membership by stability evidence in unsupervised analysis.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-consensusclusterplus/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-consensusclusterplus'), ('version', '1.40.0')])), ('source', CommentedMap([('fn', 'ConsensusClusterPlus_1.40.0.tar.gz'), ('url', 'https://bioconductor.org/packages/release/bioc/src/contrib/ConsensusClusterPlus_1.40.0.tar.gz'), ('md5', 'ae9231f43df080d1f37d5c23719051fe')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-all', 'r-base', 'r-cluster']), ('run', ['bioconductor-biobase', 'bioconductor-all', 'r-base', 'r-cluster'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ConsensusClusterPlus\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html'), ('license', 'GPL-2'), ('summary', 'Algorithm for determining cluster count and membership by stability evidence in unsupervised analysis.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bedtools/2.26.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bedtools'), ('version', '2.26.0gx')])), ('source', CommentedMap([('fn', 'bedtools-2.26.0.tar.gz'), ('url', 'https://github.com/arq5x/bedtools2/releases/download/v2.26.0/bedtools-2.26.0.tar.gz'), ('sha256', '65f32f32cbf1b91ba42854b40c604aa6a16c7d3b3ec110d6acf438eb22df0a4a'), ('patches', ['2.26.0_postrelease.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['bedtools'])])), ('about', CommentedMap([('home', 'http://bedtools.readthedocs.org/'), ('license', 'GPL v2'), ('summary', 'A powerful toolset for genome arithmetic')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bedtools/2.26.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bedtools'), ('version', '2.26.0gx')])), ('source', CommentedMap([('fn', 'bedtools-2.26.0.tar.gz'), ('url', 'https://github.com/arq5x/bedtools2/releases/download/v2.26.0/bedtools-2.26.0.tar.gz'), ('sha256', '65f32f32cbf1b91ba42854b40c604aa6a16c7d3b3ec110d6acf438eb22df0a4a'), ('patches', ['2.26.0_postrelease.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['bedtools'])])), ('about', CommentedMap([('home', 'http://bedtools.readthedocs.org/'), ('license', 'GPL v2'), ('summary', 'A powerful toolset for genome arithmetic')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-snpediar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-snpediar'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'SNPediaR_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SNPediaR_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/SNPediaR_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/SNPediaR/SNPediaR_1.2.0_src_all.tar.gz']), ('md5', 'e27dc812f1fe1de1ab81d0ea9b64780e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-jsonlite', 'r-rcurl']), ('run', ['r-base', 'r-jsonlite', 'r-rcurl'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SNPediaR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SNPediaR.html'), ('license', 'GPL-2'), ('summary', 'SNPediaR provides some tools for downloading and parsing data from the SNPedia web site <http://www.snpedia.com>. The implemented functions allow users to import the wiki text available in SNPedia pages and to extract the most relevant information out of them. If some information in the downloaded pages is not automatically processed by the library functions, users can easily implement their own parsers to access it in an efficient way.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-snpediar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-snpediar'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'SNPediaR_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SNPediaR_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/SNPediaR_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/SNPediaR/SNPediaR_1.2.0_src_all.tar.gz']), ('md5', 'e27dc812f1fe1de1ab81d0ea9b64780e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-jsonlite', 'r-rcurl']), ('run', ['r-base', 'r-jsonlite', 'r-rcurl'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SNPediaR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SNPediaR.html'), ('license', 'GPL-2'), ('summary', 'SNPediaR provides some tools for downloading and parsing data from the SNPedia web site <http://www.snpedia.com>. The implemented functions allow users to import the wiki text available in SNPedia pages and to extract the most relevant information out of them. If some information in the downloaded pages is not automatically processed by the library functions, users can easily implement their own parsers to access it in an efficient way.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-spikeli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-spikeli'), ('version', '2.36.0')])), ('source', CommentedMap([('fn', 'spikeLI_2.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/spikeLI_2.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/spikeLI_2.36.0.tar.gz', 'https://depot.galaxyproject.org/software/spikeLI/spikeLI_2.36.0_src_all.tar.gz']), ('md5', 'fcfab36e7d98f614b8850e8fa5470890')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'spikeLI\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/spikeLI.html'), ('license', 'GPL-2'), ('summary', 'SpikeLI  is a package that performs the analysis of the Affymetrix spike-in data using the Langmuir Isotherm. The aim of this package is to show the advantages of a physical-chemistry based analysis of the Affymetrix microarray data compared to the traditional methods. The spike-in (or Latin square) data for the HGU95 and HGU133 chipsets have been downloaded from the Affymetrix web site. The model used in the spikeLI package is described in details in E. Carlon and T. Heim, Physica A 362, 433 (2006).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-spikeli/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-spikeli'), ('version', '2.36.0')])), ('source', CommentedMap([('fn', 'spikeLI_2.36.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/spikeLI_2.36.0.tar.gz', 'https://bioarchive.galaxyproject.org/spikeLI_2.36.0.tar.gz', 'https://depot.galaxyproject.org/software/spikeLI/spikeLI_2.36.0_src_all.tar.gz']), ('md5', 'fcfab36e7d98f614b8850e8fa5470890')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'spikeLI\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/spikeLI.html'), ('license', 'GPL-2'), ('summary', 'SpikeLI  is a package that performs the analysis of the Affymetrix spike-in data using the Langmuir Isotherm. The aim of this package is to show the advantages of a physical-chemistry based analysis of the Affymetrix microarray data compared to the traditional methods. The spike-in (or Latin square) data for the HGU95 and HGU133 chipsets have been downloaded from the Affymetrix web site. The model used in the spikeLI package is described in details in E. Carlon and T. Heim, Physica A 362, 433 (2006).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-gpseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-gpseq'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'GPseq_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/GPseq_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/GPseq/GPseq_<{ version }}.tar.gz']), ('sha256', '5121b6a4723efbe91a6a86074a100c383257a1ee2628e9d9be574c4c888bbd4c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GPseq\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=GPseq'), ('license', 'GPL-2'), ('summary', 'Some functions for modeling sequence read counts as a generalized poisson model and to use this model for detecting differentially expressed genes in different conditions and differentially spliced exons.'), ('license_family', 'GPL2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-gpseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-gpseq'), ('version', '<{ version|replace(\"-\", \"_\") }}')])), ('source', CommentedMap([('fn', 'GPseq_<{ version }}.tar.gz'), ('url', ['https://cran.r-project.org/src/contrib/GPseq_<{ version }}.tar.gz', 'https://cran.r-project.org/src/contrib/Archive/GPseq/GPseq_<{ version }}.tar.gz']), ('sha256', '5121b6a4723efbe91a6a86074a100c383257a1ee2628e9d9be574c4c888bbd4c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'GPseq\\')\"'])])), ('about', CommentedMap([('home', 'https://CRAN.R-project.org/package=GPseq'), ('license', 'GPL-2'), ('summary', 'Some functions for modeling sequence read counts as a generalized poisson model and to use this model for detecting differentially expressed genes in different conditions and differentially spliced exons.'), ('license_family', 'GPL2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-antiprofiles/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-antiprofiles'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'antiProfiles_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/antiProfiles_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/antiProfiles_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/antiProfiles/antiProfiles_1.16.0_src_all.tar.gz']), ('md5', '13a0e36ace84d5e293d39766e61429ae')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-locfit >=1.5', 'r-matrixstats >=0.50.0']), ('run', ['r-base', 'r-locfit >=1.5', 'r-matrixstats >=0.50.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'antiProfiles\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/antiProfiles.html'), ('license', 'Artistic-2.0'), ('summary', 'Implements gene expression anti-profiles as described in Corrada Bravo et al., BMC Bioinformatics 2012, 13:272 doi:10.1186/1471-2105-13-272.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-antiprofiles/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-antiprofiles'), ('version', '1.16.0')])), ('source', CommentedMap([('fn', 'antiProfiles_1.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/antiProfiles_1.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/antiProfiles_1.16.0.tar.gz', 'https://depot.galaxyproject.org/software/antiProfiles/antiProfiles_1.16.0_src_all.tar.gz']), ('md5', '13a0e36ace84d5e293d39766e61429ae')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-locfit >=1.5', 'r-matrixstats >=0.50.0']), ('run', ['r-base', 'r-locfit >=1.5', 'r-matrixstats >=0.50.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'antiProfiles\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/antiProfiles.html'), ('license', 'Artistic-2.0'), ('summary', 'Implements gene expression anti-profiles as described in Corrada Bravo et al., BMC Bioinformatics 2012, 13:272 doi:10.1186/1471-2105-13-272.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-iranges/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-iranges'), ('version', '2.8.2')])), ('source', CommentedMap([('fn', 'IRanges_2.8.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/IRanges_2.8.2.tar.gz'), ('md5', 'ae1e6b78c6b84e4df6bec44e98e2136c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.19.1', 'bioconductor-s4vectors >=0.11.19', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.19.1', 'bioconductor-s4vectors >=0.11.19', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'IRanges\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/IRanges.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides efficient low-level and highly reusable S4 classes for storing, manipulating and aggregating over annotated ranges of integers. Implements an algebra of range operations, including efficient algorithms for finding overlaps and nearest neighbors. Defines efficient list-like classes for storing, transforming and aggregating large grouped data, i.e., collections of atomic vectors and DataFrames.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-iranges/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-iranges'), ('version', '2.8.2')])), ('source', CommentedMap([('fn', 'IRanges_2.8.2.tar.gz'), ('url', 'http://bioconductor.org/packages/3.4/bioc/src/contrib/IRanges_2.8.2.tar.gz'), ('md5', 'ae1e6b78c6b84e4df6bec44e98e2136c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.19.1', 'bioconductor-s4vectors >=0.11.19', 'r-base']), ('run', ['bioconductor-biocgenerics >=0.19.1', 'bioconductor-s4vectors >=0.11.19', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'IRanges\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/IRanges.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides efficient low-level and highly reusable S4 classes for storing, manipulating and aggregating over annotated ranges of integers. Implements an algebra of range operations, including efficient algorithms for finding overlaps and nearest neighbors. Defines efficient list-like classes for storing, transforming and aggregating large grouped data, i.e., collections of atomic vectors and DataFrames.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bioperl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bioperl'), ('version', '1.6.924')])), ('build', CommentedMap([('number', 7)])), ('requirements', CommentedMap([('build', ['perl', 'perl-bioperl-core ==1.6.924', 'perl-bioperl-run']), ('run', ['perl', 'perl-bioperl-core ==1.6.924', 'perl-bioperl-run'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/BioPerl'), ('license', 'perl_5'), ('summary', 'Bioinformatics Toolkit')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-bioperl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-bioperl'), ('version', '1.6.924')])), ('build', CommentedMap([('number', 7)])), ('requirements', CommentedMap([('build', ['perl', 'perl-bioperl-core ==1.6.924', 'perl-bioperl-run']), ('run', ['perl', 'perl-bioperl-core ==1.6.924', 'perl-bioperl-run'])])), ('about', CommentedMap([('home', 'http://metacpan.org/pod/BioPerl'), ('license', 'perl_5'), ('summary', 'Bioinformatics Toolkit')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-edaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-edaseq'), ('version', '2.8.0')])), ('source', CommentedMap([('fn', 'EDASeq_2.8.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/EDASeq_2.8.0.tar.gz'), ('md5', '07196d42966d048d0873f0659d63f637')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-aroma.light', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-deseq', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.13.9', 'bioconductor-rsamtools >=1.5.75', 'bioconductor-shortread >=1.11.42', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-aroma.light', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-deseq', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.13.9', 'bioconductor-rsamtools >=1.5.75', 'bioconductor-shortread >=1.11.42', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'EDASeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/EDASeq.html'), ('license', 'Artistic-2.0'), ('summary', 'Numerical and graphical summaries of RNA-Seq read data. Within-lane normalization procedures to adjust for GC-content effect (or other gene-level effects) on read counts: loess robust local regression, global-scaling, and full-quantile normalization (Risso et al., 2011). Between-lane normalization procedures to adjust for distributional differences between lanes (e.g., sequencing depth): global-scaling and full-quantile normalization (Bullard et al., 2010).')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-edaseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-edaseq'), ('version', '2.8.0')])), ('source', CommentedMap([('fn', 'EDASeq_2.8.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/EDASeq_2.8.0.tar.gz'), ('md5', '07196d42966d048d0873f0659d63f637')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-aroma.light', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-deseq', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.13.9', 'bioconductor-rsamtools >=1.5.75', 'bioconductor-shortread >=1.11.42', 'r-base']), ('run', ['bioconductor-annotationdbi', 'bioconductor-aroma.light', 'bioconductor-biobase >=2.15.1', 'bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-biostrings', 'bioconductor-deseq', 'bioconductor-genomicfeatures', 'bioconductor-genomicranges', 'bioconductor-iranges >=1.13.9', 'bioconductor-rsamtools >=1.5.75', 'bioconductor-shortread >=1.11.42', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'EDASeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/EDASeq.html'), ('license', 'Artistic-2.0'), ('summary', 'Numerical and graphical summaries of RNA-Seq read data. Within-lane normalization procedures to adjust for GC-content effect (or other gene-level effects) on read counts: loess robust local regression, global-scaling, and full-quantile normalization (Risso et al., 2011). Between-lane normalization procedures to adjust for distributional differences between lanes (e.g., sequencing depth): global-scaling and full-quantile normalization (Bullard et al., 2010).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnasnp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rnasnp'), ('version', 1.2)])), ('source', CommentedMap([('fn', 'RNAsnp-1.2.tar.gz'), ('url', 'http://rth.dk/resources/rnasnp/software_src/RNAsnp-1.2.tar.gz'), ('sha256', '4fb63b7ff149eca463e84e2f9e3c430e34d5e381dbda827b89e452408cead094')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['set +e && printf \"A3G\" > snp.txt && printf \">gi|56682960|ref|NM_000146.3| Homo sapiens ferritin, light polypeptide (FTL), mRNA\\\\nGCAGTTCGGCGGTCCCGCGGGTCTGTCTCTTGCTTCAACAGTGTTTGGACGGAACAGATCCGGGGACTCTCTTCCAGCCTCCGACCGCCCTCCGATTTCCTCTCCGCTTGCAACCTCCGGGACCATCTTCTCGGCCATCTCCTGCTTCTGGGACCTGCCAGCACCGTTTTTGTGGTTAGCTCCTTCTTGCCAACCAACCA\" > seq.fa && RNAsnp -f seq.fa -s snp.txt; exit_code=$? && if [ $exit_code != 160 ]; then exit $exit_code ;fi'])])), ('about', CommentedMap([('home', 'http://rth.dk/resources/rnasnp/software'), ('license', 'RTH'), ('license_file', 'COPYING'), ('summary', 'Efficient detection of local RNA secondary structure changes induced by SNPs.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnasnp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rnasnp'), ('version', 1.2)])), ('source', CommentedMap([('fn', 'RNAsnp-1.2.tar.gz'), ('url', 'http://rth.dk/resources/rnasnp/software_src/RNAsnp-1.2.tar.gz'), ('sha256', '4fb63b7ff149eca463e84e2f9e3c430e34d5e381dbda827b89e452408cead094')])), ('build', CommentedMap([('number', 3)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['set +e && printf \"A3G\" > snp.txt && printf \">gi|56682960|ref|NM_000146.3| Homo sapiens ferritin, light polypeptide (FTL), mRNA\\\\nGCAGTTCGGCGGTCCCGCGGGTCTGTCTCTTGCTTCAACAGTGTTTGGACGGAACAGATCCGGGGACTCTCTTCCAGCCTCCGACCGCCCTCCGATTTCCTCTCCGCTTGCAACCTCCGGGACCATCTTCTCGGCCATCTCCTGCTTCTGGGACCTGCCAGCACCGTTTTTGTGGTTAGCTCCTTCTTGCCAACCAACCA\" > seq.fa && RNAsnp -f seq.fa -s snp.txt; exit_code=$? && if [ $exit_code != 160 ]; then exit $exit_code ;fi'])])), ('about', CommentedMap([('home', 'http://rth.dk/resources/rnasnp/software'), ('license', 'RTH'), ('license_file', 'COPYING'), ('summary', 'Efficient detection of local RNA secondary structure changes induced by SNPs.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-liftover/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-liftover'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/liftOver'), ('fn', 'liftOver'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'libpng', 'mysql', 'openssl', 'zlib']), ('run', ['libgcc', 'libpng', 'mysql', 'openssl', 'zlib'])])), ('test', CommentedMap([('commands', ['which liftOver'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Move annotations from one assembly to another')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/ucsc-liftover/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'ucsc-liftover'), ('version', '332')])), ('source', CommentedMap([('url', 'http://hgdownload.cse.ucsc.edu/admin/exe/macOSX.x86_64/liftOver'), ('fn', 'liftOver'), ('md5', '8c2663c7bd302a77cdf52b2e9e85e2cd'), ('patches', ['include.patch'])])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'libpng', 'mysql', 'openssl', 'zlib']), ('run', ['libgcc', 'libpng', 'mysql', 'openssl', 'zlib'])])), ('test', CommentedMap([('commands', ['which liftOver'])])), ('about', CommentedMap([('home', 'http://hgdownload.cse.ucsc.edu/admin/exe/'), ('license', 'varies; see http://genome.ucsc.edu/license'), ('summary', 'Move annotations from one assembly to another')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dnacopy/1.48.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dnacopy'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'DNAcopy_1.48.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/DNAcopy_1.48.0.tar.gz'), ('sha256', '4094d45c761d2494912f07dc6f8f70f6c37c1831220bebc28404a9d95203fbbf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DNAcopy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/DNAcopy.html'), ('license', 'GPL (>= 2)'), ('summary', 'Implements the circular binary segmentation (CBS) algorithm to segment DNA copy number data and identify genomic regions with abnormal copy number.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dnacopy/1.48.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dnacopy'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'DNAcopy_1.48.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/DNAcopy_1.48.0.tar.gz'), ('sha256', '4094d45c761d2494912f07dc6f8f70f6c37c1831220bebc28404a9d95203fbbf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'gcc']), ('run', ['r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DNAcopy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/DNAcopy.html'), ('license', 'GPL (>= 2)'), ('summary', 'Implements the circular binary segmentation (CBS) algorithm to segment DNA copy number data and identify genomic regions with abnormal copy number.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chipseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chipseq'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'chipseq_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/chipseq_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/chipseq_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/chipseq/chipseq_1.24.0_src_all.tar.gz']), ('sha256', 'c09252008bb8dc25e3cf6896fff9632a9618f8d485f4ccb58a29e92cd0f5aa84')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.1', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-shortread', 'r-base', 'r-lattice']), ('run', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.1', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-shortread', 'r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'chipseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/chipseq.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for helping process short read data for chipseq experiments')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chipseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chipseq'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'chipseq_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/chipseq_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/chipseq_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/chipseq/chipseq_1.24.0_src_all.tar.gz']), ('sha256', 'c09252008bb8dc25e3cf6896fff9632a9618f8d485f4ccb58a29e92cd0f5aa84')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.1', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-shortread', 'r-base', 'r-lattice']), ('run', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.1', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-shortread', 'r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'chipseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.4/bioc/html/chipseq.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools for helping process short read data for chipseq experiments')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yara/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'yara'), ('version', '0.9.6')])), ('source', CommentedMap([('fn', 'yara-0.9.6-Linux-x86_64.zip'), ('url', 'http://packages.seqan.de/yara/yara-0.9.6-Linux-x86_64.zip'), ('md5', 'fca52e83f786b71ce622f108a9e18a24')])), ('test', CommentedMap([('commands', ['yara_indexer --help 2>&1 > /dev/null', 'yara_mapper --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/seqan/blob/develop/apps/yara/README.rst'), ('license', 'https://raw.githubusercontent.com/seqan/seqan/develop/apps/yara/LICENSE'), ('summary', 'Yara is an exact tool for aligning DNA sequencing reads to reference genomes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yara/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('package', CommentedMap([('name', 'yara'), ('version', '0.9.6')])), ('source', CommentedMap([('fn', 'yara-0.9.6-Linux-x86_64.zip'), ('url', 'http://packages.seqan.de/yara/yara-0.9.6-Linux-x86_64.zip'), ('md5', 'fca52e83f786b71ce622f108a9e18a24')])), ('test', CommentedMap([('commands', ['yara_indexer --help 2>&1 > /dev/null', 'yara_mapper --help 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'https://github.com/seqan/seqan/blob/develop/apps/yara/README.rst'), ('license', 'https://raw.githubusercontent.com/seqan/seqan/develop/apps/yara/LICENSE'), ('summary', 'Yara is an exact tool for aligning DNA sequencing reads to reference genomes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-lpsymphony/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-lpsymphony'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'lpsymphony_1.2.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/lpsymphony_1.2.0.tar.gz'), ('md5', '043beed308f85f461dfd2760ade9362c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['gcc', 'r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'lpsymphony\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/lpsymphony.html'), ('license', 'EPL'), ('summary', \"This package was derived from Rsymphony_0.1-17 from CRAN. These packages provide an R interface to SYMPHONY, an open-source linear programming solver written in C++. The main difference between this package and Rsymphony is that it includes the solver source code (SYMPHONY version 5.6), while Rsymphony expects to find header and library files on the users' system. Thus the intention of lpsymphony is to provide an easy to install interface to SYMPHONY. For Windows, precompiled DLLs are included in this package.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-lpsymphony/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-lpsymphony'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'lpsymphony_1.2.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/lpsymphony_1.2.0.tar.gz'), ('md5', '043beed308f85f461dfd2760ade9362c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['gcc', 'r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'lpsymphony\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/lpsymphony.html'), ('license', 'EPL'), ('summary', \"This package was derived from Rsymphony_0.1-17 from CRAN. These packages provide an R interface to SYMPHONY, an open-source linear programming solver written in C++. The main difference between this package and Rsymphony is that it includes the solver source code (SYMPHONY version 5.6), while Rsymphony expects to find header and library files on the users' system. Thus the intention of lpsymphony is to provide an easy to install interface to SYMPHONY. For Windows, precompiled DLLs are included in this package.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-kegggraph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-kegggraph'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'KEGGgraph_1.30.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/KEGGgraph_1.30.0.tar.gz'), ('md5', '885c16a5dd746ce7a276bd538fc7602f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base', 'r-xml >=2.3-0']), ('run', ['bioconductor-graph', 'r-base', 'r-xml >=2.3-0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'KEGGgraph\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/KEGGgraph.html'), ('license', 'GPL (>= 2)'), ('summary', 'KEGGGraph is an interface between KEGG pathway and graph object as well as a collection of tools to analyze, dissect and visualize these graphs. It parses the regularly updated KGML (KEGG XML) files into graph models maintaining all essential pathway attributes. The package offers functionalities including parsing, graph operation, visualization and etc.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-kegggraph/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-kegggraph'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'KEGGgraph_1.30.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/KEGGgraph_1.30.0.tar.gz'), ('md5', '885c16a5dd746ce7a276bd538fc7602f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base', 'r-xml >=2.3-0']), ('run', ['bioconductor-graph', 'r-base', 'r-xml >=2.3-0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'KEGGgraph\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/KEGGgraph.html'), ('license', 'GPL (>= 2)'), ('summary', 'KEGGGraph is an interface between KEGG pathway and graph object as well as a collection of tools to analyze, dissect and visualize these graphs. It parses the regularly updated KGML (KEGG XML) files into graph models maintaining all essential pathway attributes. The package offers functionalities including parsing, graph operation, visualization and etc.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cand/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cand'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'CAnD_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CAnD_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/CAnD_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/CAnD/CAnD_1.8.0_src_all.tar.gz']), ('md5', '7128428f743b57ee1e02500334a0242b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ggplot2', 'r-reshape']), ('run', ['r-base', 'r-ggplot2', 'r-reshape'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CAnD\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CAnD.html'), ('license', 'Artistic-2.0'), ('summary', 'Functions to perform the CAnD test on a set of ancestry proportions. For a particular ancestral subpopulation, a user will supply the estimated ancestry proportion for each sample, and each chromosome or chromosomal segment of interest. A p-value for each chromosome as well as an overall CAnD p-value will be returned for each test. Plotting functions are also available.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cand/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cand'), ('version', '1.8.0')])), ('source', CommentedMap([('fn', 'CAnD_1.8.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CAnD_1.8.0.tar.gz', 'https://bioarchive.galaxyproject.org/CAnD_1.8.0.tar.gz', 'https://depot.galaxyproject.org/software/CAnD/CAnD_1.8.0_src_all.tar.gz']), ('md5', '7128428f743b57ee1e02500334a0242b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ggplot2', 'r-reshape']), ('run', ['r-base', 'r-ggplot2', 'r-reshape'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CAnD\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CAnD.html'), ('license', 'Artistic-2.0'), ('summary', 'Functions to perform the CAnD test on a set of ancestry proportions. For a particular ancestral subpopulation, a user will supply the estimated ancestry proportion for each sample, and each chromosome or chromosomal segment of interest. A p-value for each chromosome as well as an overall CAnD p-value will be returned for each test. Plotting functions are also available.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vcflib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vcflib'), ('version', '1.0.0_rc1')])), ('source', CommentedMap([('git_url', 'https://github.com/ekg/vcflib.git'), ('git_rev', '3e03c7a10098b3888aaa258d4e339ddcdbf3b951')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['vcfallelicprimitives -h'])])), ('about', CommentedMap([('home', 'https://github.com/ekg/vcflib'), ('license', 'MIT'), ('summary', 'Command-line tools for manipulating VCF files')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vcflib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'vcflib'), ('version', '1.0.0_rc1')])), ('source', CommentedMap([('git_url', 'https://github.com/ekg/vcflib.git'), ('git_rev', '3e03c7a10098b3888aaa258d4e339ddcdbf3b951')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['vcfallelicprimitives -h'])])), ('about', CommentedMap([('home', 'https://github.com/ekg/vcflib'), ('license', 'MIT'), ('summary', 'Command-line tools for manipulating VCF files')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genesplicer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genesplicer'), ('version', '0_2003.04.03')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'GeneSplicer.tar.gz'), ('url', 'http://www.cs.jhu.edu/~genomics/GeneSplicer/GeneSplicer.tar.gz'), ('md5', 'e490f8632f03b5fe3d42dd5b73f1e3c5')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['genesplicer'])])), ('about', CommentedMap([('home', 'http://www.cs.jhu.edu/~genomics/GeneSplicer'), ('license', 'OSI'), ('summary', 'GeneSplicer : A computational method for splice site prediction')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/genesplicer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'genesplicer'), ('version', '0_2003.04.03')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('source', CommentedMap([('fn', 'GeneSplicer.tar.gz'), ('url', 'http://www.cs.jhu.edu/~genomics/GeneSplicer/GeneSplicer.tar.gz'), ('md5', 'e490f8632f03b5fe3d42dd5b73f1e3c5')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['genesplicer'])])), ('about', CommentedMap([('home', 'http://www.cs.jhu.edu/~genomics/GeneSplicer'), ('license', 'OSI'), ('summary', 'GeneSplicer : A computational method for splice site prediction')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snoscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://lowelab.ucsc.edu/software/snoscan-0.9.tar.gz'), ('md5', '30719dc179f00c892d50ce070336d0a1'), ('patches', ['patches/sort-snos.patch', 'patches/sqio.c.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'zlib', 'perl'])])), ('test', CommentedMap([('commands', ['snoscan -h > /dev/null', 'sort-snos 2>&1 | grep \"Usage\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://lowelab.ucsc.edu/snoscan'), ('summary', 'Search for C/D box methylation guide snoRNA genes in a genomic sequence'), ('licence', 'GPL')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/snoscan/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://lowelab.ucsc.edu/software/snoscan-0.9.tar.gz'), ('md5', '30719dc179f00c892d50ce070336d0a1'), ('patches', ['patches/sort-snos.patch', 'patches/sqio.c.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'perl', 'perl-app-cpanminus', 'perl-module-build']), ('run', ['libgcc', 'zlib', 'perl'])])), ('test', CommentedMap([('commands', ['snoscan -h > /dev/null', 'sort-snos 2>&1 | grep \"Usage\" > /dev/null'])])), ('about', CommentedMap([('home', 'http://lowelab.ucsc.edu/snoscan'), ('summary', 'Search for C/D box methylation guide snoRNA genes in a genomic sequence'), ('licence', 'GPL')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-macat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-macat'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'macat_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/macat_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/macat_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/macat/macat_1.50.0_src_all.tar.gz']), ('md5', '1fef5c90534f564c40cc4708be621d11')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-biobase', 'r-base']), ('run', ['bioconductor-annotate', 'bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'macat\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/macat.html'), ('license', 'Artistic-2.0'), ('summary', \"This library contains functions to investigate links between differential gene expression and the chromosomal localization of the genes. MACAT is motivated by the common observation of phenomena involving large chromosomal regions in tumor cells. MACAT is the implementation of a statistical approach for identifying significantly differentially expressed chromosome regions. The functions have been tested on a publicly available data set about acute lymphoblastic leukemia (Yeoh et al.Cancer Cell 2002), which is provided in the library 'stjudem'.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-macat/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-macat'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'macat_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/macat_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/macat_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/macat/macat_1.50.0_src_all.tar.gz']), ('md5', '1fef5c90534f564c40cc4708be621d11')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-biobase', 'r-base']), ('run', ['bioconductor-annotate', 'bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'macat\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/macat.html'), ('license', 'Artistic-2.0'), ('summary', \"This library contains functions to investigate links between differential gene expression and the chromosomal localization of the genes. MACAT is motivated by the common observation of phenomena involving large chromosomal regions in tumor cells. MACAT is the implementation of a statistical approach for identifying significantly differentially expressed chromosome regions. The functions have been tested on a publicly available data set about acute lymphoblastic leukemia (Yeoh et al.Cancer Cell 2002), which is provided in the library 'stjudem'.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-topgo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-topgo'), ('version', '2.24.0')])), ('source', CommentedMap([('fn', 'topGO_2.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/topGO_2.24.0.tar.gz'), ('md5', '633251cfa90f4e9f7346ca0b48dcc413')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.7.19', 'bioconductor-biobase >=2.0.0', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-go.db >=2.3.0', 'bioconductor-graph >=1.14.0', 'r-base', 'r-dbi', 'r-matrixstats', 'r-sparsem >=0.73', 'r-lattice']), ('run', ['bioconductor-annotationdbi >=1.7.19', 'bioconductor-biobase >=2.0.0', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-go.db >=2.3.0', 'bioconductor-graph >=1.14.0', 'r-base', 'r-dbi', 'r-matrixstats', 'r-sparsem >=0.73', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'topGO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/topGO.html'), ('license', 'LGPL'), ('summary', 'topGO package provides tools for testing GO terms while accounting for the topology of the GO graph. Different test statistics and different methods for eliminating local similarities and dependencies between GO terms can be implemented and applied.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-topgo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-topgo'), ('version', '2.24.0')])), ('source', CommentedMap([('fn', 'topGO_2.24.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/topGO_2.24.0.tar.gz'), ('md5', '633251cfa90f4e9f7346ca0b48dcc413')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi >=1.7.19', 'bioconductor-biobase >=2.0.0', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-go.db >=2.3.0', 'bioconductor-graph >=1.14.0', 'r-base', 'r-dbi', 'r-matrixstats', 'r-sparsem >=0.73', 'r-lattice']), ('run', ['bioconductor-annotationdbi >=1.7.19', 'bioconductor-biobase >=2.0.0', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-go.db >=2.3.0', 'bioconductor-graph >=1.14.0', 'r-base', 'r-dbi', 'r-matrixstats', 'r-sparsem >=0.73', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'topGO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/topGO.html'), ('license', 'LGPL'), ('summary', 'topGO package provides tools for testing GO terms while accounting for the topology of the GO graph. Different test statistics and different methods for eliminating local similarities and dependencies between GO terms can be implemented and applied.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/translatorx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'translatorx'), ('version', '1.1')])), ('source', CommentedMap([('fn', 'translatorx_vLocal.pl'), ('url', 'http://pc16141.mncn.csic.es/cgi-bin/translatorx_vLocal.pl'), ('sha256', 'd0b88c3cedd253ea4972567191ba1dddd4c79841bfb411d6e7bdaae0fbb266c3'), ('patches', ['translatorx.patch'])])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['perl', 'muscle'])])), ('test', CommentedMap([('commands', ['translatorx'])])), ('about', CommentedMap([('home', 'http://pc16141.mncn.csic.es/'), ('license', None), ('summary', 'Multiple alignment of nucleotide sequences guided by amino acid information')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/translatorx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'translatorx'), ('version', '1.1')])), ('source', CommentedMap([('fn', 'translatorx_vLocal.pl'), ('url', 'http://pc16141.mncn.csic.es/cgi-bin/translatorx_vLocal.pl'), ('sha256', 'd0b88c3cedd253ea4972567191ba1dddd4c79841bfb411d6e7bdaae0fbb266c3'), ('patches', ['translatorx.patch'])])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('run', ['perl', 'muscle'])])), ('test', CommentedMap([('commands', ['translatorx'])])), ('about', CommentedMap([('home', 'http://pc16141.mncn.csic.es/'), ('license', None), ('summary', 'Multiple alignment of nucleotide sequences guided by amino acid information')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bedops/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'http://bedops.readthedocs.io'), ('license', 'GPLv2'), ('summary', 'High-performance genomic feature operations.')])), ('package', CommentedMap([('name', 'bedops'), ('version', '2.4.27')])), ('source', CommentedMap([('url', 'https://github.com/bedops/bedops/archive/v2.4.27.tar.gz'), ('fn', 'bedops-2.4.27.tar.gz'), ('sha256', '23a4e1ccae608a80f1cda99e24b7c1a314914adb2ff098753f5aac143746569d')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['bedops --help', 'gtf2bed --help'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bedops/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0)])), ('about', CommentedMap([('home', 'http://bedops.readthedocs.io'), ('license', 'GPLv2'), ('summary', 'High-performance genomic feature operations.')])), ('package', CommentedMap([('name', 'bedops'), ('version', '2.4.27')])), ('source', CommentedMap([('url', 'https://github.com/bedops/bedops/archive/v2.4.27.tar.gz'), ('fn', 'bedops-2.4.27.tar.gz'), ('sha256', '23a4e1ccae608a80f1cda99e24b7c1a314914adb2ff098753f5aac143746569d')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['bedops --help', 'gtf2bed --help'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-twilight/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-twilight'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'twilight_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/twilight_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/twilight_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/twilight/twilight_1.52.0_src_all.tar.gz']), ('md5', '5de401cba564f99a6cb3987c03c6d37a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.12.0', 'r-base']), ('run', ['bioconductor-biobase >=1.12.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'twilight\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/twilight.html'), ('license', 'GPL (>= 2)'), ('summary', \"In a typical microarray setting with gene expression data observed under two conditions, the local false discovery rate describes the probability that a gene is not differentially expressed between the two conditions given its corrresponding observed score or p-value level. The resulting curve of p-values versus local false discovery rate offers an insight into the twilight zone between clear differential and clear non-differential gene expression. Package 'twilight' contains two main functions: Function twilight.pval performs a two-condition test on differences in means for a given input matrix or expression set and computes permutation based p-values. Function twilight performs a stochastic downhill search to estimate local false discovery rates and effect size distributions. The package further provides means to filter for permutations that describe the null distribution correctly. Using filtered permutations, the influence of hidden confounders could be diminished.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-twilight/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-twilight'), ('version', '1.52.0')])), ('source', CommentedMap([('fn', 'twilight_1.52.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/twilight_1.52.0.tar.gz', 'https://bioarchive.galaxyproject.org/twilight_1.52.0.tar.gz', 'https://depot.galaxyproject.org/software/twilight/twilight_1.52.0_src_all.tar.gz']), ('md5', '5de401cba564f99a6cb3987c03c6d37a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.12.0', 'r-base']), ('run', ['bioconductor-biobase >=1.12.0', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'twilight\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/twilight.html'), ('license', 'GPL (>= 2)'), ('summary', \"In a typical microarray setting with gene expression data observed under two conditions, the local false discovery rate describes the probability that a gene is not differentially expressed between the two conditions given its corrresponding observed score or p-value level. The resulting curve of p-values versus local false discovery rate offers an insight into the twilight zone between clear differential and clear non-differential gene expression. Package 'twilight' contains two main functions: Function twilight.pval performs a two-condition test on differences in means for a given input matrix or expression set and computes permutation based p-values. Function twilight performs a stochastic downhill search to estimate local false discovery rates and effect size distributions. The package further provides means to filter for permutations that describe the null distribution correctly. Using filtered permutations, the influence of hidden confounders could be diminished.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/randfold/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'randfold'), ('version', '2.0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('url', 'http://bioinformatics.psb.ugent.be/supplementary_data/erbon/nov2003/downloads/randfold-2.0.1.tar.gz'), ('md5', 'c9ebf7dc9d62fa4554a738a15fe1ded8'), ('patches', ['randfold.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['afetch -h', 'alistat -h', 'compalign -h', 'compstruct -h', 'revcomp -h', 'seqsplit -h', 'seqstat -h', 'sfetch -h', 'shuffle -h', 'sindex -h', 'sreformat -h', 'translate -h', 'weight -h', 'randfold -h'])])), ('about', CommentedMap([('home', 'http://bioinformatics.psb.ugent.be/software/details/Randfold'), ('license', 'GNU GPLv2'), ('license_file', 'LICENSE'), ('summary', 'Minimum free energy of folding randomization test software')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/randfold/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'randfold'), ('version', '2.0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('url', 'http://bioinformatics.psb.ugent.be/supplementary_data/erbon/nov2003/downloads/randfold-2.0.1.tar.gz'), ('md5', 'c9ebf7dc9d62fa4554a738a15fe1ded8'), ('patches', ['randfold.patch'])])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['afetch -h', 'alistat -h', 'compalign -h', 'compstruct -h', 'revcomp -h', 'seqsplit -h', 'seqstat -h', 'sfetch -h', 'shuffle -h', 'sindex -h', 'sreformat -h', 'translate -h', 'weight -h', 'randfold -h'])])), ('about', CommentedMap([('home', 'http://bioinformatics.psb.ugent.be/software/details/Randfold'), ('license', 'GNU GPLv2'), ('license_file', 'LICENSE'), ('summary', 'Minimum free energy of folding randomization test software')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pindel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pindel'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 2), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'pindel-<{ version }}.tar.gz'), ('url', 'https://github.com/genome/pindel/archive/v<{ version }}.tar.gz'), ('sha256', '7f21fda0b751d420831724d96e60873ce332139cfd24396e81c7f1ae2f707a19')])), ('requirements', CommentedMap([('build', ['htslib <{CONDA_HTSLIB}}*', 'gcc']), ('run', ['htslib <{CONDA_HTSLIB}}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['pindel -h 2>&1 | grep \"Pindel version\"', 'pindel2vcf  --help | grep Example:'])])), ('about', CommentedMap([('home', 'http://gmt.genome.wustl.edu/packages/pindel/index.html'), ('license', 'GPLv3'), ('summary', 'Pindel can detect breakpoints of large deletions, medium sized insertions, inversions, tandem duplications and other structural variants at single-based resolution from next-gen sequence data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pindel/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pindel'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 2), ('skip', True), ('string', 'htslib<{CONDA_HTSLIB}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'pindel-<{ version }}.tar.gz'), ('url', 'https://github.com/genome/pindel/archive/v<{ version }}.tar.gz'), ('sha256', '7f21fda0b751d420831724d96e60873ce332139cfd24396e81c7f1ae2f707a19')])), ('requirements', CommentedMap([('build', ['htslib <{CONDA_HTSLIB}}*', 'gcc']), ('run', ['htslib <{CONDA_HTSLIB}}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['pindel -h 2>&1 | grep \"Pindel version\"', 'pindel2vcf  --help | grep Example:'])])), ('about', CommentedMap([('home', 'http://gmt.genome.wustl.edu/packages/pindel/index.html'), ('license', 'GPLv3'), ('summary', 'Pindel can detect breakpoints of large deletions, medium sized insertions, inversions, tandem duplications and other structural variants at single-based resolution from next-gen sequence data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-timecourse/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-timecourse'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'timecourse_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/timecourse_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/timecourse_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/timecourse/timecourse_1.48.0_src_all.tar.gz']), ('md5', '8c0c49039012d1c74aa81c877a19cd33')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma >=1.8.6', 'bioconductor-marray', 'r-base', 'r-mass']), ('run', ['bioconductor-biobase', 'bioconductor-limma >=1.8.6', 'bioconductor-marray', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'timecourse\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/timecourse.html'), ('license', 'LGPL'), ('summary', 'Functions for data analysis and graphical displays for developmental microarray time course data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-timecourse/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-timecourse'), ('version', '1.48.0')])), ('source', CommentedMap([('fn', 'timecourse_1.48.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/timecourse_1.48.0.tar.gz', 'https://bioarchive.galaxyproject.org/timecourse_1.48.0.tar.gz', 'https://depot.galaxyproject.org/software/timecourse/timecourse_1.48.0_src_all.tar.gz']), ('md5', '8c0c49039012d1c74aa81c877a19cd33')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-limma >=1.8.6', 'bioconductor-marray', 'r-base', 'r-mass']), ('run', ['bioconductor-biobase', 'bioconductor-limma >=1.8.6', 'bioconductor-marray', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'timecourse\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/timecourse.html'), ('license', 'LGPL'), ('summary', 'Functions for data analysis and graphical displays for developmental microarray time course data.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vsearch/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/torognes/vsearch/archive/v<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['automake', 'autoconf', 'm4', 'gcc', 'llvm', 'perl', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['vsearch --version'])])), ('about', CommentedMap([('home', 'https://github.com/torognes/vsearch'), ('license', 'dual-licensed under GPLv3+ or BSD 2-clause'), ('summary', 'a versatile open source tool for metagenomics (USEARCH alternative)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/vsearch/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/torognes/vsearch/archive/v<{ version }}.tar.gz'), ('sha256', '<{ hash }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['automake', 'autoconf', 'm4', 'gcc', 'llvm', 'perl', 'zlib <{CONDA_ZLIB}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('commands', ['vsearch --version'])])), ('about', CommentedMap([('home', 'https://github.com/torognes/vsearch'), ('license', 'dual-licensed under GPLv3+ or BSD 2-clause'), ('summary', 'a versatile open source tool for metagenomics (USEARCH alternative)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mrbayes/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name | lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name | lower }}-<{ version }}.tar.gz'), ('url', 'https://sourceforge.net/projects/<{ name | lower }}/files/<{ name | lower }}/<{ version }}/<{ name | lower }}-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'autoconf', 'automake', 'readline 6.*', 'openmpi', 'beagle-lib']), ('run', ['libgcc', 'readline 6.*', 'openmpi', 'beagle-lib'])])), ('test', CommentedMap([('commands', ['echo Version | mb', 'echo Version | mpirun -mca plm_rsh_agent true --allow-run-as-root mb-mpi'])])), ('about', CommentedMap([('home', 'http://mrbayes.sourceforge.net'), ('license', 'GPLv3'), ('license_family', 'GPL'), ('license_file', 'src/gpl.txt'), ('summary', 'Bayesian Inference of Phylogeny'), ('description', 'MrBayes is a program for Bayesian inference and model choice across a wide\\nrange of phylogenetic and evolutionary models. MrBayes uses Markov chain\\nMonte Carlo (MCMC) methods to estimate the posterior distribution of model\\nparameters.\\n')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mrbayes/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name | lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name | lower }}-<{ version }}.tar.gz'), ('url', 'https://sourceforge.net/projects/<{ name | lower }}/files/<{ name | lower }}/<{ version }}/<{ name | lower }}-<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'autoconf', 'automake', 'readline 6.*', 'openmpi', 'beagle-lib']), ('run', ['libgcc', 'readline 6.*', 'openmpi', 'beagle-lib'])])), ('test', CommentedMap([('commands', ['echo Version | mb', 'echo Version | mpirun -mca plm_rsh_agent true --allow-run-as-root mb-mpi'])])), ('about', CommentedMap([('home', 'http://mrbayes.sourceforge.net'), ('license', 'GPLv3'), ('license_family', 'GPL'), ('license_file', 'src/gpl.txt'), ('summary', 'Bayesian Inference of Phylogeny'), ('description', 'MrBayes is a program for Bayesian inference and model choice across a wide\\nrange of phylogenetic and evolutionary models. MrBayes uses Markov chain\\nMonte Carlo (MCMC) methods to estimate the posterior distribution of model\\nparameters.\\n')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-exomecopy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-exomecopy'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'exomeCopy_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/exomeCopy_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/exomeCopy_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/exomeCopy/exomeCopy_1.22.0_src_all.tar.gz']), ('md5', 'd944c56edbb5dec52fa7410e2c3c01b2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'r-base']), ('run', ['bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'exomeCopy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/exomeCopy.html'), ('license', 'GPL (>= 2)'), ('summary', 'Detection of copy number variants (CNV) from exome sequencing samples, including unpaired samples.  The package implements a hidden Markov model which uses positional covariates, such as background read depth and GC-content, to simultaneously normalize and segment the samples into regions of constant copy count.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-exomecopy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-exomecopy'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'exomeCopy_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/exomeCopy_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/exomeCopy_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/exomeCopy/exomeCopy_1.22.0_src_all.tar.gz']), ('md5', 'd944c56edbb5dec52fa7410e2c3c01b2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'r-base']), ('run', ['bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.23.16', 'bioconductor-iranges >=2.5.27', 'bioconductor-rsamtools', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'exomeCopy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/exomeCopy.html'), ('license', 'GPL (>= 2)'), ('summary', 'Detection of copy number variants (CNV) from exome sequencing samples, including unpaired samples.  The package implements a hidden Markov model which uses positional covariates, such as background read depth and GC-content, to simultaneously normalize and segment the samples into regions of constant copy count.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-doppelgangr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-doppelgangr'), ('version', '1.4.1')])), ('source', CommentedMap([('fn', 'doppelgangR_1.4.1.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/doppelgangR_1.4.1.tar.gz'), ('md5', 'f677adff8eee1342a0e2eea5edbef632')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-impute', 'bioconductor-sva', 'r-base', 'r-digest', 'r-mnormt']), ('run', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-impute', 'bioconductor-sva', 'r-base', 'r-digest', 'r-mnormt'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'doppelgangR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/doppelgangR.html'), ('license', 'GPL (>=2.0)'), ('summary', 'The main function is doppelgangR(), which takes as minimal input a list of ExpressionSet object, and searches all list pairs for duplicated samples.  The search is based on the genomic data (exprs(eset)), phenotype/clinical data (pData(eset)), and \"smoking guns\" - supposedly unique identifiers found in pData(eset).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-doppelgangr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-doppelgangr'), ('version', '1.4.1')])), ('source', CommentedMap([('fn', 'doppelgangR_1.4.1.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/doppelgangR_1.4.1.tar.gz'), ('md5', 'f677adff8eee1342a0e2eea5edbef632')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-impute', 'bioconductor-sva', 'r-base', 'r-digest', 'r-mnormt']), ('run', ['bioconductor-biobase', 'bioconductor-biocparallel', 'bioconductor-impute', 'bioconductor-sva', 'r-base', 'r-digest', 'r-mnormt'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'doppelgangR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/doppelgangR.html'), ('license', 'GPL (>=2.0)'), ('summary', 'The main function is doppelgangR(), which takes as minimal input a list of ExpressionSet object, and searches all list pairs for duplicated samples.  The search is based on the genomic data (exprs(eset)), phenotype/clinical data (pData(eset)), and \"smoking guns\" - supposedly unique identifiers found in pData(eset).')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-m3drop/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-m3drop'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'M3Drop_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/M3Drop_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/M3Drop_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/M3Drop/M3Drop_1.2.0_src_all.tar.gz']), ('md5', 'e2bcdd73fbc61c739d13db83b5ace1db')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-bbmle', 'r-gplots', 'r-numderiv', 'r-rcolorbrewer', 'r-statmod']), ('run', ['r-base', 'r-bbmle', 'r-gplots', 'r-numderiv', 'r-rcolorbrewer', 'r-statmod'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'M3Drop\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/M3Drop.html'), ('license', 'GPL (>=2)'), ('summary', 'This package fits a Michaelis-Menten model to the pattern of dropouts in single-cell RNASeq data. This model is used as a null to identify significantly variable (i.e. differentially expressed) genes for use in downstream analysis, such as clustering cells.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-m3drop/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-m3drop'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'M3Drop_1.2.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/M3Drop_1.2.0.tar.gz', 'https://bioarchive.galaxyproject.org/M3Drop_1.2.0.tar.gz', 'https://depot.galaxyproject.org/software/M3Drop/M3Drop_1.2.0_src_all.tar.gz']), ('md5', 'e2bcdd73fbc61c739d13db83b5ace1db')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-bbmle', 'r-gplots', 'r-numderiv', 'r-rcolorbrewer', 'r-statmod']), ('run', ['r-base', 'r-bbmle', 'r-gplots', 'r-numderiv', 'r-rcolorbrewer', 'r-statmod'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'M3Drop\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/M3Drop.html'), ('license', 'GPL (>=2)'), ('summary', 'This package fits a Michaelis-Menten model to the pattern of dropouts in single-cell RNASeq data. This model is used as a null to identify significantly variable (i.e. differentially expressed) genes for use in downstream analysis, such as clustering cells.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mops/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mops'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MoPS_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MoPS_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MoPS_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MoPS/MoPS_1.10.0_src_all.tar.gz']), ('md5', '96dd954498d1a6acd69dfdb9310a6f44')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MoPS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MoPS.html'), ('license', 'GPL-3'), ('summary', 'Identification and characterization of periodic fluctuations in time-series data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mops/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mops'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'MoPS_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MoPS_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/MoPS_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/MoPS/MoPS_1.10.0_src_all.tar.gz']), ('md5', '96dd954498d1a6acd69dfdb9310a6f44')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base']), ('run', ['bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MoPS\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MoPS.html'), ('license', 'GPL-3'), ('summary', 'Identification and characterization of periodic fluctuations in time-series data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-massarray/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-massarray'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'MassArray_1.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MassArray_1.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/MassArray_1.28.0.tar.gz', 'https://depot.galaxyproject.org/software/MassArray/MassArray_1.28.0_src_all.tar.gz']), ('md5', '71153966697991f134350a2c23c14082')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MassArray\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MassArray.html'), ('license', 'GPL (>=2)'), ('summary', \"This package is designed for the import, quality control, analysis, and visualization of methylation data generated using Sequenom's MassArray platform.  The tools herein contain a highly detailed amplicon prediction for optimal assay design.  Also included are quality control measures of data, such as primer dimer and bisulfite conversion efficiency estimation.  Methylation data are calculated using the same algorithms contained in the EpiTyper software package.  Additionally, automatic SNP-detection can be used to flag potentially confounded data from specific CG sites.  Visualization includes barplots of methylation data as well as UCSC Genome Browser-compatible BED tracks.  Multiple assays can be positionally combined for integrated analysis.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-massarray/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-massarray'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'MassArray_1.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MassArray_1.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/MassArray_1.28.0.tar.gz', 'https://depot.galaxyproject.org/software/MassArray/MassArray_1.28.0_src_all.tar.gz']), ('md5', '71153966697991f134350a2c23c14082')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MassArray\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MassArray.html'), ('license', 'GPL (>=2)'), ('summary', \"This package is designed for the import, quality control, analysis, and visualization of methylation data generated using Sequenom's MassArray platform.  The tools herein contain a highly detailed amplicon prediction for optimal assay design.  Also included are quality control measures of data, such as primer dimer and bisulfite conversion efficiency estimation.  Methylation data are calculated using the same algorithms contained in the EpiTyper software package.  Additionally, automatic SNP-detection can be used to flag potentially confounded data from specific CG sites.  Visualization includes barplots of methylation data as well as UCSC Genome Browser-compatible BED tracks.  Multiple assays can be positionally combined for integrated analysis.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pcan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pcan'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'PCAN_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PCAN_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/PCAN_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/PCAN/PCAN_1.4.0_src_all.tar.gz']), ('md5', 'ac243936d5a2e1c52abeb00ce1ea8372')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'r-base']), ('run', ['bioconductor-biocparallel', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PCAN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PCAN.html'), ('license', 'CC BY-NC-ND 4.0'), ('summary', 'Phenotypes comparison based on a pathway consensus approach. Assess the relationship between candidate genes and a set of phenotypes based on additional genes related to the candidate (e.g. Pathways or network neighbors).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pcan/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pcan'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'PCAN_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PCAN_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/PCAN_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/PCAN/PCAN_1.4.0_src_all.tar.gz']), ('md5', 'ac243936d5a2e1c52abeb00ce1ea8372')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocparallel', 'r-base']), ('run', ['bioconductor-biocparallel', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PCAN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PCAN.html'), ('license', 'CC BY-NC-ND 4.0'), ('summary', 'Phenotypes comparison based on a pathway consensus approach. Assess the relationship between candidate genes and a set of phenotypes based on additional genes related to the candidate (e.g. Pathways or network neighbors).')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/primer3/1.1.4/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/primer3/primer3/1.1.4/primer3-1.1.4.tar.gz'), ('md5', 'd9bbdb749a7eea25dd8c653e272d5f39')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc', 'llvm']), ('run', ['perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', [\"primer3_core --help 2>&1 | grep 'This is primer3'\"])])), ('about', CommentedMap([('home', 'http://primer3.sourceforge.net/'), ('license', 'GPLv2'), ('license_file', 'COPYING.txt'), ('summary', 'Design PCR primers from DNA sequence. From mispriming libraries to sequence quality data to the generation of internal oligos, primer3 does it.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/primer3/1.1.4/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/primer3/primer3/1.1.4/primer3-1.1.4.tar.gz'), ('md5', 'd9bbdb749a7eea25dd8c653e272d5f39')])), ('requirements', CommentedMap([('build', ['perl-threaded', 'gcc', 'llvm']), ('run', ['perl-threaded', 'libgcc'])])), ('test', CommentedMap([('commands', [\"primer3_core --help 2>&1 | grep 'This is primer3'\"])])), ('about', CommentedMap([('home', 'http://primer3.sourceforge.net/'), ('license', 'GPLv2'), ('license_file', 'COPYING.txt'), ('summary', 'Design PCR primers from DNA sequence. From mispriming libraries to sequence quality data to the generation of internal oligos, primer3 does it.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/abyss/2.0.1-k128/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}-k128'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://www.bcgsc.ca/platform/bioinfo/software/<{ name|lower }}/releases/<{ version }}/<{ name|lower }}-<{ version }}.tar.gz'), ('md5', '596c56a476b26a294b46e6e36890c136')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'boost <{CONDA_BOOST}}*', 'google-sparsehash', 'sqlite', 'openmpi']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*', 'bwa', 'pandoc', 'sqlite', 'openmpi'])])), ('test', CommentedMap([('commands', ['ABYSS --version'])])), ('about', CommentedMap([('home', 'http://www.bcgsc.ca/platform/bioinfo/software/abyss'), ('license', 'GPL3'), ('license_file', 'LICENSE'), ('summary', 'Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/abyss/2.0.1-k128/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}-k128'), ('version', '<{ version }}')])), ('build', CommentedMap([('skip', True), ('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', '<{ name|lower }}-<{ version }}.tar.gz'), ('url', 'http://www.bcgsc.ca/platform/bioinfo/software/<{ name|lower }}/releases/<{ version }}/<{ name|lower }}-<{ version }}.tar.gz'), ('md5', '596c56a476b26a294b46e6e36890c136')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'boost <{CONDA_BOOST}}*', 'google-sparsehash', 'sqlite', 'openmpi']), ('run', ['libgcc', 'boost <{CONDA_BOOST}}*', 'bwa', 'pandoc', 'sqlite', 'openmpi'])])), ('test', CommentedMap([('commands', ['ABYSS --version'])])), ('about', CommentedMap([('home', 'http://www.bcgsc.ca/platform/bioinfo/software/abyss'), ('license', 'GPL3'), ('license_file', 'LICENSE'), ('summary', 'Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyfaidx/0.4.8.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyfaidx'), ('version', '0.4.8.1')])), ('source', CommentedMap([('fn', 'pyfaidx-0.4.8.1.tar.gz'), ('url', 'https://pypi.python.org/packages/5f/0e/b22911d548b5fe043629675fccd7c0b8f96b76b0ec0c72a484f4f0615c42/pyfaidx-0.4.8.1.tar.gz'), ('md5', 'f33beb242463eed92381571da4a3aa56')])), ('build', CommentedMap([('skip', False), ('entry_points', ['faidx = pyfaidx.cli:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('test', CommentedMap([('imports', ['pyfaidx']), ('commands', ['faidx --help'])])), ('about', CommentedMap([('home', 'http://mattshirley.com'), ('license', 'MIT License'), ('summary', 'pyfaidx: efficient pythonic random access to fasta subsequences')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pyfaidx/0.4.8.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pyfaidx'), ('version', '0.4.8.1')])), ('source', CommentedMap([('fn', 'pyfaidx-0.4.8.1.tar.gz'), ('url', 'https://pypi.python.org/packages/5f/0e/b22911d548b5fe043629675fccd7c0b8f96b76b0ec0c72a484f4f0615c42/pyfaidx-0.4.8.1.tar.gz'), ('md5', 'f33beb242463eed92381571da4a3aa56')])), ('build', CommentedMap([('skip', False), ('entry_points', ['faidx = pyfaidx.cli:main'])])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'six']), ('run', ['python', 'six'])])), ('test', CommentedMap([('imports', ['pyfaidx']), ('commands', ['faidx --help'])])), ('about', CommentedMap([('home', 'http://mattshirley.com'), ('license', 'MIT License'), ('summary', 'pyfaidx: efficient pythonic random access to fasta subsequences')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/manta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'manta'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'manta-<{ version }}.tar.bz2'), ('url', 'https://github.com/Illumina/manta/releases/download/v<{ version }}/manta-<{ version }}.centos5_x86_64.tar.bz2'), ('md5', '30d0a28d83ed513e5e785f41291e64d8')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['configManta.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/Illumina/manta'), ('license', 'GPLv3'), ('summary', 'Structural variant and indel caller for mapped sequencing data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/manta/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'manta'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'manta-<{ version }}.tar.bz2'), ('url', 'https://github.com/Illumina/manta/releases/download/v<{ version }}/manta-<{ version }}.centos5_x86_64.tar.bz2'), ('md5', '30d0a28d83ed513e5e785f41291e64d8')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['configManta.py -h'])])), ('about', CommentedMap([('home', 'https://github.com/Illumina/manta'), ('license', 'GPLv3'), ('summary', 'Structural variant and indel caller for mapped sequencing data')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mmnet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mmnet'), ('version', '1.13.0')])), ('source', CommentedMap([('fn', 'mmnet_1.13.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mmnet_1.13.0.tar.gz', 'https://bioarchive.galaxyproject.org/mmnet_1.13.0.tar.gz', 'https://depot.galaxyproject.org/software/mmnet/mmnet_1.13.0_src_all.tar.gz']), ('md5', '90508186eaaa876bff44b42fb6e4a46d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-keggrest', 'r-base', 'r-biom', 'r-flexmix', 'r-ggplot2', 'r-igraph', 'r-matrix', 'r-plyr', 'r-rcurl', 'r-reshape2', 'r-rjsonio', 'r-stringr', 'r-xml']), ('run', ['bioconductor-biobase', 'bioconductor-keggrest', 'r-base', 'r-biom', 'r-flexmix', 'r-ggplot2', 'r-igraph', 'r-matrix', 'r-plyr', 'r-rcurl', 'r-reshape2', 'r-rjsonio', 'r-stringr', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mmnet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mmnet.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package gives the implementations microbiome metabolic network constructing and analyzing. It introduces a unique metagenomic systems biology approach, mapping metagenomic data to the KEGG global metabolic pathway and constructing a systems-level network. The system-level network and the next topological analysis will be of great help to analysis the various functional properties, including regulation and metabolic functionality of the metagenome.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mmnet/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mmnet'), ('version', '1.13.0')])), ('source', CommentedMap([('fn', 'mmnet_1.13.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/mmnet_1.13.0.tar.gz', 'https://bioarchive.galaxyproject.org/mmnet_1.13.0.tar.gz', 'https://depot.galaxyproject.org/software/mmnet/mmnet_1.13.0_src_all.tar.gz']), ('md5', '90508186eaaa876bff44b42fb6e4a46d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-keggrest', 'r-base', 'r-biom', 'r-flexmix', 'r-ggplot2', 'r-igraph', 'r-matrix', 'r-plyr', 'r-rcurl', 'r-reshape2', 'r-rjsonio', 'r-stringr', 'r-xml']), ('run', ['bioconductor-biobase', 'bioconductor-keggrest', 'r-base', 'r-biom', 'r-flexmix', 'r-ggplot2', 'r-igraph', 'r-matrix', 'r-plyr', 'r-rcurl', 'r-reshape2', 'r-rjsonio', 'r-stringr', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mmnet\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/mmnet.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package gives the implementations microbiome metabolic network constructing and analyzing. It introduces a unique metagenomic systems biology approach, mapping metagenomic data to the KEGG global metabolic pathway and constructing a systems-level network. The system-level network and the next topological analysis will be of great help to analysis the various functional properties, including regulation and metabolic functionality of the metagenome.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bhc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bhc'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'BHC_1.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BHC_1.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/BHC_1.28.0.tar.gz', 'https://depot.galaxyproject.org/software/BHC/BHC_1.28.0_src_all.tar.gz']), ('md5', '8b0542c53177dcd0d4eb0adfce13078f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BHC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BHC.html'), ('license', 'GPL-3'), ('summary', 'The method performs bottom-up hierarchical clustering, using a Dirichlet Process (infinite mixture) to model uncertainty in the data and Bayesian model selection to decide at each step which clusters to merge.  This avoids several limitations of traditional methods, for example how many clusters there should be and how to choose a principled distance metric.  This implementation accepts multinomial (i.e. discrete, with 2+ categories) or time-series data. This version also includes a randomised algorithm which is more efficient for larger data sets.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bhc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bhc'), ('version', '1.28.0')])), ('source', CommentedMap([('fn', 'BHC_1.28.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BHC_1.28.0.tar.gz', 'https://bioarchive.galaxyproject.org/BHC_1.28.0.tar.gz', 'https://depot.galaxyproject.org/software/BHC/BHC_1.28.0_src_all.tar.gz']), ('md5', '8b0542c53177dcd0d4eb0adfce13078f')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BHC\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BHC.html'), ('license', 'GPL-3'), ('summary', 'The method performs bottom-up hierarchical clustering, using a Dirichlet Process (infinite mixture) to model uncertainty in the data and Bayesian model selection to decide at each step which clusters to merge.  This avoids several limitations of traditional methods, for example how many clusters there should be and how to choose a principled distance metric.  This implementation accepts multinomial (i.e. discrete, with 2+ categories) or time-series data. This version also includes a randomised algorithm which is more efficient for larger data sets.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bismark/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bismark'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'bismark-v<{ version }}.tar.gz'), ('url', 'https://www.bioinformatics.babraham.ac.uk/projects/bismark/bismark_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', None), ('run', ['perl', 'samtools', 'bowtie2'])])), ('about', CommentedMap([('home', 'https://www.bioinformatics.babraham.ac.uk/projects/bismark/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('license_family', 'GPL3'), ('summary', 'Bismark is a program to map bisulfite treated sequencing reads to a genome of interest and perform methylation calls in a single step. The output can be easily imported into a genome viewer, such as SeqMonk, and enables a researcher to analyse the methylation levels of their samples straight away.')])), ('test', CommentedMap([('commands', ['bam2nuc --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark --help > /dev/null', 'bismark2bedGraph --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark2report --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark2summary --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark_genome_preparation --help > /dev/null', 'bismark_methylation_extractor --help > /dev/null', 'coverage2cytosine --help 2>&1 | grep SYNOPSIS > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bismark/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bismark'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('fn', 'bismark-v<{ version }}.tar.gz'), ('url', 'https://www.bioinformatics.babraham.ac.uk/projects/bismark/bismark_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('requirements', CommentedMap([('build', None), ('run', ['perl', 'samtools', 'bowtie2'])])), ('about', CommentedMap([('home', 'https://www.bioinformatics.babraham.ac.uk/projects/bismark/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('license_family', 'GPL3'), ('summary', 'Bismark is a program to map bisulfite treated sequencing reads to a genome of interest and perform methylation calls in a single step. The output can be easily imported into a genome viewer, such as SeqMonk, and enables a researcher to analyse the methylation levels of their samples straight away.')])), ('test', CommentedMap([('commands', ['bam2nuc --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark --help > /dev/null', 'bismark2bedGraph --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark2report --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark2summary --help 2>&1 | grep SYNOPSIS > /dev/null', 'bismark_genome_preparation --help > /dev/null', 'bismark_methylation_extractor --help > /dev/null', 'coverage2cytosine --help 2>&1 | grep SYNOPSIS > /dev/null'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sspa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sspa'), ('version', '2.16.0')])), ('source', CommentedMap([('fn', 'SSPA_2.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SSPA_2.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/SSPA_2.16.0.tar.gz', 'https://depot.galaxyproject.org/software/SSPA/SSPA_2.16.0_src_all.tar.gz']), ('md5', 'c0c2b503e9eb785c66b222dd6dc8d45c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-lattice']), ('run', ['bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SSPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SSPA.html'), ('license', 'GPL (>= 2)'), ('summary', 'General Sample size and power analysis for microarray and next-generation sequencing data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sspa/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sspa'), ('version', '2.16.0')])), ('source', CommentedMap([('fn', 'SSPA_2.16.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SSPA_2.16.0.tar.gz', 'https://bioarchive.galaxyproject.org/SSPA_2.16.0.tar.gz', 'https://depot.galaxyproject.org/software/SSPA/SSPA_2.16.0_src_all.tar.gz']), ('md5', 'c0c2b503e9eb785c66b222dd6dc8d45c')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-lattice']), ('run', ['bioconductor-limma', 'bioconductor-qvalue', 'r-base', 'r-lattice'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SSPA\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SSPA.html'), ('license', 'GPL (>= 2)'), ('summary', 'General Sample size and power analysis for microarray and next-generation sequencing data.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ggcyto/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ggcyto'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'ggcyto_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ggcyto_1.4.0.tar.gz'), ('md5', 'c9dac98b39ce05da851dba598a345b7a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore >=1.41.5', 'bioconductor-flowworkspace >=3.17.24', 'bioconductor-ncdfflow >=2.17.1', 'r-base', 'r-data.table', 'r-ggplot2 >=2.0.0', 'r-gridextra', 'r-plyr', 'r-rcolorbrewer', 'r-scales']), ('run', ['bioconductor-flowcore >=1.41.5', 'bioconductor-flowworkspace >=3.17.24', 'bioconductor-ncdfflow >=2.17.1', 'r-base', 'r-data.table', 'r-ggplot2 >=2.0.0', 'r-gridextra', 'r-plyr', 'r-rcolorbrewer', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ggcyto\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ggcyto.html'), ('license', 'Artistic-2.0'), ('summary', 'With the dedicated fority method implemented for flowSet, ncdfFlowSet and GatingSet classes, both raw and gated flow cytometry data can be plotted directly with ggplot. ggcyto wrapper and some customed layers also make it easy to add gates and population statistics to the plot.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ggcyto/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ggcyto'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'ggcyto_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ggcyto_1.4.0.tar.gz'), ('md5', 'c9dac98b39ce05da851dba598a345b7a')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore >=1.41.5', 'bioconductor-flowworkspace >=3.17.24', 'bioconductor-ncdfflow >=2.17.1', 'r-base', 'r-data.table', 'r-ggplot2 >=2.0.0', 'r-gridextra', 'r-plyr', 'r-rcolorbrewer', 'r-scales']), ('run', ['bioconductor-flowcore >=1.41.5', 'bioconductor-flowworkspace >=3.17.24', 'bioconductor-ncdfflow >=2.17.1', 'r-base', 'r-data.table', 'r-ggplot2 >=2.0.0', 'r-gridextra', 'r-plyr', 'r-rcolorbrewer', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ggcyto\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ggcyto.html'), ('license', 'Artistic-2.0'), ('summary', 'With the dedicated fority method implemented for flowSet, ncdfFlowSet and GatingSet classes, both raw and gated flow cytometry data can be plotted directly with ggplot. ggcyto wrapper and some customed layers also make it easy to add gates and population statistics to the plot.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ramigo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ramigo'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'RamiGO_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RamiGO_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/RamiGO_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/RamiGO/RamiGO_1.22.0_src_all.tar.gz']), ('md5', '746153d0c5edfd483a34ae3bd046d22b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-rcytoscape', 'r-base', 'r-gsubfn', 'r-igraph', 'r-png', 'r-rcurl']), ('run', ['bioconductor-graph', 'bioconductor-rcytoscape', 'r-base', 'r-gsubfn', 'r-igraph', 'r-png', 'r-rcurl'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RamiGO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RamiGO.html'), ('license', 'Artistic-2.0'), ('summary', 'R interface sending requests to AmiGO visualize, retrieving DAG GO trees, parsing GraphViz DOT format files and exporting GML files for Cytoscape. Also uses RCytoscape to interactively display AmiGO trees in Cytoscape.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ramigo/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ramigo'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'RamiGO_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/RamiGO_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/RamiGO_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/RamiGO/RamiGO_1.22.0_src_all.tar.gz']), ('md5', '746153d0c5edfd483a34ae3bd046d22b')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-rcytoscape', 'r-base', 'r-gsubfn', 'r-igraph', 'r-png', 'r-rcurl']), ('run', ['bioconductor-graph', 'bioconductor-rcytoscape', 'r-base', 'r-gsubfn', 'r-igraph', 'r-png', 'r-rcurl'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RamiGO\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/RamiGO.html'), ('license', 'Artistic-2.0'), ('summary', 'R interface sending requests to AmiGO visualize, retrieving DAG GO trees, parsing GraphViz DOT format files and exporting GML files for Cytoscape. Also uses RCytoscape to interactively display AmiGO trees in Cytoscape.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kallisto/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kallisto'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'kallisto-<{ version }}.tar.gz'), ('url', 'https://github.com/pachterlab/kallisto/archive/v<{ version }}.tar.gz'), ('sha256', '2164938c2c61c04e338c4c132cf749f56d39e6f0b4c517121bca1fbc218e430e')])), ('build', CommentedMap([('number', 0), ('string', 'hdf5<{CONDA_HDF5}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['hdf5 <{CONDA_HDF5}}', 'cmake', 'zlib', 'gcc']), ('run', ['hdf5 <{CONDA_HDF5}}', 'libgcc'])])), ('test', CommentedMap([('files', ['t.fa', 't.fq']), ('commands', ['kallisto version'])])), ('about', CommentedMap([('home', 'http://pachterlab.github.io/kallisto'), ('license', 'BSD_2_Clause'), ('summary', 'Quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kallisto/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'kallisto'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'kallisto-<{ version }}.tar.gz'), ('url', 'https://github.com/pachterlab/kallisto/archive/v<{ version }}.tar.gz'), ('sha256', '2164938c2c61c04e338c4c132cf749f56d39e6f0b4c517121bca1fbc218e430e')])), ('build', CommentedMap([('number', 0), ('string', 'hdf5<{CONDA_HDF5}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['hdf5 <{CONDA_HDF5}}', 'cmake', 'zlib', 'gcc']), ('run', ['hdf5 <{CONDA_HDF5}}', 'libgcc'])])), ('test', CommentedMap([('files', ['t.fa', 't.fq']), ('commands', ['kallisto version'])])), ('about', CommentedMap([('home', 'http://pachterlab.github.io/kallisto'), ('license', 'BSD_2_Clause'), ('summary', 'Quantifying abundances of transcripts from RNA-Seq data, or more generally of target sequences using high-throughput sequencing reads.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chicago/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chicago'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'Chicago_1.2.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Chicago_1.2.0.tar.gz'), ('md5', 'cce9ec0fbb7b589c3862146c77afc260')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-data.table', 'r-delaporte', 'r-hmisc', 'r-matrixstats']), ('run', ['r-base', 'r-data.table', 'r-delaporte', 'r-hmisc', 'r-matrixstats'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Chicago\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Chicago.html'), ('license', 'Artistic-2.0'), ('summary', 'A pipeline for analysing Capture Hi-C data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-chicago/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-chicago'), ('version', '1.2.0')])), ('source', CommentedMap([('fn', 'Chicago_1.2.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Chicago_1.2.0.tar.gz'), ('md5', 'cce9ec0fbb7b589c3862146c77afc260')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-data.table', 'r-delaporte', 'r-hmisc', 'r-matrixstats']), ('run', ['r-base', 'r-data.table', 'r-delaporte', 'r-hmisc', 'r-matrixstats'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Chicago\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Chicago.html'), ('license', 'Artistic-2.0'), ('summary', 'A pipeline for analysing Capture Hi-C data.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationdbi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationdbi'), ('version', '1.38.0')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/bioconductor-annotationdbi/bioconductor-annotationdbi_1.38.0_src_all.tar.gz'), ('sha256', '47bc40347c3383ffb992bfd31768a6f69860fc02d9b2231073f504a451aacef5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-iranges', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dbi', 'r-rsqlite']), ('run', ['bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-iranges', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dbi', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationDbi\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/AnnotationDbi.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides user interface and database connection code for annotation data packages using SQLite data storage.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-annotationdbi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-annotationdbi'), ('version', '1.38.0')])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/bioconductor-annotationdbi/bioconductor-annotationdbi_1.38.0_src_all.tar.gz'), ('sha256', '47bc40347c3383ffb992bfd31768a6f69860fc02d9b2231073f504a451aacef5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-iranges', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dbi', 'r-rsqlite']), ('run', ['bioconductor-biobase >=1.17.0', 'bioconductor-biocgenerics >=0.15.10', 'bioconductor-iranges', 'bioconductor-s4vectors >=0.9.25', 'r-base', 'r-dbi', 'r-rsqlite'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AnnotationDbi\\')\"'])])), ('about', CommentedMap([('home', 'https://bioconductor.org/packages/release/bioc/html/AnnotationDbi.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides user interface and database connection code for annotation data packages using SQLite data storage.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/paml/4.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'paml'), ('version', '4.9')])), ('source', CommentedMap([('fn', 'paml_4.9_src_all.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/paml/paml_4.9_src_all.tar.gz'), ('md5', '7c3638fdf19ec44b7b2636477d893de6')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['chi2 1 .1', 'evolver 1 | grep EVOLVER', 'codeml /dev/null 2>&1 | grep paml'])])), ('about', CommentedMap([('home', 'http://abacus.gene.ucl.ac.uk/software/paml.html'), ('summary', 'A package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood.'), ('license', 'GNU General Public License v3 (GPLv3)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/paml/4.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'paml'), ('version', '4.9')])), ('source', CommentedMap([('fn', 'paml_4.9_src_all.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/paml/paml_4.9_src_all.tar.gz'), ('md5', '7c3638fdf19ec44b7b2636477d893de6')])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['chi2 1 .1', 'evolver 1 | grep EVOLVER', 'codeml /dev/null 2>&1 | grep paml'])])), ('about', CommentedMap([('home', 'http://abacus.gene.ucl.ac.uk/software/paml.html'), ('summary', 'A package of programs for phylogenetic analyses of DNA or protein sequences using maximum likelihood.'), ('license', 'GNU General Public License v3 (GPLv3)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pasta/1.7.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pasta'), ('version', '1.7.8')])), ('source', CommentedMap([('fn', '43076b8.tar.gz'), ('url', 'https://github.com/smirarab/pasta/archive/43076b8.tar.gz'), ('md5', 'a085bb4f6b4116566a61b785be580ae1')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['certifi 2016.2.28 py27_0', 'dendropy 4.1.0 py27_0', 'gcc', 'gnu-wget', 'java-jdk >=6', 'llvm', 'pcre', 'python >=2.7,<3', 'pymongo 3.3.0 py27_0']), ('run', ['dendropy 4.1.0 py27_0', 'java-jdk >=6', 'libgcc', 'pcre', 'python >=2.7,<3', 'pymongo 3.3.0 py27_0'])])), ('test', CommentedMap([('imports', ['pasta']), ('commands', ['clustalw2 -help | grep OPTIONS', 'hmmalign -h | grep HMMER', 'hmmbuild -h | grep HMMER', 'muscle -version | grep MUSCLE', 'prank -help | grep prank', 'raxml -h | grep raxmlHPC', 'raxmlp -h | grep raxmlHPC'])])), ('about', CommentedMap([('home', 'https://github.com/smirarab/pasta'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'An implementation of the PASTA (Practical Alignment using Sate and TrAnsitivity) algorithm')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pasta/1.7.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pasta'), ('version', '1.7.8')])), ('source', CommentedMap([('fn', '43076b8.tar.gz'), ('url', 'https://github.com/smirarab/pasta/archive/43076b8.tar.gz'), ('md5', 'a085bb4f6b4116566a61b785be580ae1')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['certifi 2016.2.28 py27_0', 'dendropy 4.1.0 py27_0', 'gcc', 'gnu-wget', 'java-jdk >=6', 'llvm', 'pcre', 'python >=2.7,<3', 'pymongo 3.3.0 py27_0']), ('run', ['dendropy 4.1.0 py27_0', 'java-jdk >=6', 'libgcc', 'pcre', 'python >=2.7,<3', 'pymongo 3.3.0 py27_0'])])), ('test', CommentedMap([('imports', ['pasta']), ('commands', ['clustalw2 -help | grep OPTIONS', 'hmmalign -h | grep HMMER', 'hmmbuild -h | grep HMMER', 'muscle -version | grep MUSCLE', 'prank -help | grep prank', 'raxml -h | grep raxmlHPC', 'raxmlp -h | grep raxmlHPC'])])), ('about', CommentedMap([('home', 'https://github.com/smirarab/pasta'), ('license', 'GNU General Public License v3 or later (GPLv3+)'), ('summary', 'An implementation of the PASTA (Practical Alignment using Sate and TrAnsitivity) algorithm')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/krona/2.6/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/marbl/Krona'), ('license', 'BSD'), ('summary', 'Krona Tools is a set of scripts to create Krona charts from several Bioinformatics tools as well as from text and XML files.')])), ('build', CommentedMap([('number', 4)])), ('package', CommentedMap([('name', 'krona'), ('version', 2.6)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('source', CommentedMap([('fn', 'KronaTools-2.6.tar'), ('sha256', '4f04ece119f0d32fb4e61e36ffd1a7809ad053d4db5dcb5f2e7dd4fd3fc8b1de'), ('url', 'https://github.com/marbl/Krona/releases/download/v2.6/KronaTools-2.6.tar')])), ('test', CommentedMap([('commands', ['ktImportTaxonomy', 'ktImportXML'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/krona/2.6/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/marbl/Krona'), ('license', 'BSD'), ('summary', 'Krona Tools is a set of scripts to create Krona charts from several Bioinformatics tools as well as from text and XML files.')])), ('build', CommentedMap([('number', 4)])), ('package', CommentedMap([('name', 'krona'), ('version', 2.6)])), ('requirements', CommentedMap([('build', ['perl-threaded']), ('run', ['perl-threaded'])])), ('source', CommentedMap([('fn', 'KronaTools-2.6.tar'), ('sha256', '4f04ece119f0d32fb4e61e36ffd1a7809ad053d4db5dcb5f2e7dd4fd3fc8b1de'), ('url', 'https://github.com/marbl/Krona/releases/download/v2.6/KronaTools-2.6.tar')])), ('test', CommentedMap([('commands', ['ktImportTaxonomy', 'ktImportXML'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gaga/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gaga'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'gaga_2.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/gaga_2.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/gaga_2.22.0.tar.gz', 'https://depot.galaxyproject.org/software/gaga/gaga_2.22.0_src_all.tar.gz']), ('md5', 'e9f2419240b574fdb7467578ec7250a2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-ebarrays', 'r-base', 'r-coda', 'r-mgcv']), ('run', ['bioconductor-biobase', 'bioconductor-ebarrays', 'r-base', 'r-coda', 'r-mgcv'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gaga\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/gaga.html'), ('license', 'GPL (>= 2)'), ('summary', 'Implements the GaGa model for high-throughput data analysis, including differential expression analysis, supervised gene clustering and classification. Additionally, it performs sequential sample size calculations using the GaGa and LNNGV models (the latter from EBarrays package).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-gaga/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-gaga'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'gaga_2.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/gaga_2.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/gaga_2.22.0.tar.gz', 'https://depot.galaxyproject.org/software/gaga/gaga_2.22.0_src_all.tar.gz']), ('md5', 'e9f2419240b574fdb7467578ec7250a2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-ebarrays', 'r-base', 'r-coda', 'r-mgcv']), ('run', ['bioconductor-biobase', 'bioconductor-ebarrays', 'r-base', 'r-coda', 'r-mgcv'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'gaga\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/gaga.html'), ('license', 'GPL (>= 2)'), ('summary', 'Implements the GaGa model for high-throughput data analysis, including differential expression analysis, supervised gene clustering and classification. Additionally, it performs sequential sample size calculations using the GaGa and LNNGV models (the latter from EBarrays package).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-protgenerics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-protgenerics'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'ProtGenerics_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ProtGenerics_1.4.0.tar.gz'), ('md5', '129914ba3eba2a8efcc7995d0656a8cf')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ProtGenerics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ProtGenerics.html'), ('license', 'Artistic-2.0'), ('summary', 'S4 generic functions needed by Bioconductor proteomics packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-protgenerics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-protgenerics'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'ProtGenerics_1.4.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ProtGenerics_1.4.0.tar.gz'), ('md5', '129914ba3eba2a8efcc7995d0656a8cf')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ProtGenerics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ProtGenerics.html'), ('license', 'Artistic-2.0'), ('summary', 'S4 generic functions needed by Bioconductor proteomics packages.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-agdex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-agdex'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'AGDEX_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AGDEX_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/AGDEX_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/AGDEX/AGDEX_1.24.0_src_all.tar.gz']), ('md5', '678b3c153050238576764230c00b1bca')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AGDEX\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AGDEX.html'), ('license', 'GPL Version 2 or later'), ('summary', 'A tool to evaluate agreement of differential expression for cross-species genomics')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-agdex/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-agdex'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'AGDEX_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AGDEX_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/AGDEX_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/AGDEX/AGDEX_1.24.0_src_all.tar.gz']), ('md5', '678b3c153050238576764230c00b1bca')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-gseabase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AGDEX\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AGDEX.html'), ('license', 'GPL Version 2 or later'), ('summary', 'A tool to evaluate agreement of differential expression for cross-species genomics')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-masigpro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-masigpro'), ('version', '1.49.3')])), ('source', CommentedMap([('fn', 'maSigPro_1.49.0.tar.gz'), ('url', 'http://bioconductor.org/packages/devel/bioc/src/contrib/maSigPro_1.49.3.tar.gz'), ('md5', 'fc8332ffd3752ac8139fbf5037938d2e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-mclust', 'r-venn', 'r-mass']), ('run', ['bioconductor-biobase', 'r-base', 'r-mclust', 'r-venn', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'maSigPro\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/maSigPro.html'), ('license', 'GPL (>= 2)'), ('summary', 'maSigPro is a regression based approach to find genes for which there are significant gene expression profile differences between experimental groups in time course microarray and RNA-Seq experiments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-masigpro/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-masigpro'), ('version', '1.49.3')])), ('source', CommentedMap([('fn', 'maSigPro_1.49.0.tar.gz'), ('url', 'http://bioconductor.org/packages/devel/bioc/src/contrib/maSigPro_1.49.3.tar.gz'), ('md5', 'fc8332ffd3752ac8139fbf5037938d2e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-mclust', 'r-venn', 'r-mass']), ('run', ['bioconductor-biobase', 'r-base', 'r-mclust', 'r-venn', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'maSigPro\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/maSigPro.html'), ('license', 'GPL (>= 2)'), ('summary', 'maSigPro is a regression based approach to find genes for which there are significant gene expression profile differences between experimental groups in time course microarray and RNA-Seq experiments.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qiime/1.9.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qiime'), ('version', '1.9.1')])), ('source', CommentedMap([('fn', 'qiime-1.9.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/q/qiime/qiime-1.9.1.tar.gz'), ('md5', '32d4db2b2c888dbbfc6aa33f02725526'), ('sha1', '2ab4ac9beb745ed3eca3243312cc9ce14e8a20d8'), ('sha256', 'c12dcdae1fe44762a49668276d7e93f487d87a0dc2ff8ca696ac0eff9d92a8a8')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy x.x', 'scipy >=0.14.0', 'cogent ==1.5.3', 'natsort <4.0.0', 'pynast ==1.2.2', 'qcli >=0.1.1,<0.2.0', 'gdata', 'biom-format >=2.1.4,<2.2.0', 'emperor >=0.9.51,<1.0.0', 'scikit-bio >=0.2.3,<0.3.0', 'matplotlib <=1.9', 'burrito-fillings >=0.1.1,<0.2.0', 'pandas >=0.13.1', 'burrito >=0.9.1,<1.0.0', 'qiime-default-reference >=0.1.2,<0.2.0', 'mock', 'nose', 'xz']), ('run', ['python', 'numpy x.x', 'scipy >=0.14.0', 'cogent ==1.5.3', 'natsort <4.0.0', 'pynast ==1.2.2', 'qcli >=0.1.1,<0.2.0', 'gdata', 'biom-format >=2.1.4,<2.2.0', 'emperor >=0.9.51,<1.0.0', 'scikit-bio >=0.2.3,<0.3.0', 'matplotlib <=1.9', 'burrito-fillings >=0.1.1,<0.2.0', 'pandas >=0.13.1', 'burrito >=0.9.1,<1.0.0', 'qiime-default-reference >=0.1.2,<0.2.0', 'mock', 'nose', 'libgcc', 'glib', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libsm', 'xorg-libxrender', 'xorg-libxau', 'xz'])])), ('test', CommentedMap([('imports', ['qiime', 'qiime.util', 'qiime.check_id_map', 'qiime.sort']), ('commands', ['add_alpha_to_mapping_file.py -h', 'add_qiime_labels.py -h', 'adjust_seq_orientation.py -h', 'align_seqs.py -h', 'alpha_diversity.py -h', 'alpha_rarefaction.py -h', 'amplicon_contingency_table.py', 'ampliconnoise.py -h', 'assign_taxonomy.py -h', 'beta_diversity.py -h', 'beta_diversity_through_plots.py -h', 'beta_significance.py -h', 'blast_wrapper.py -h', 'categorized_dist_scatterplot.py -h', 'check_id_map.py -h', 'clean_raxml_parsimony_tree.py -h', 'cluster_quality.py -h', 'collapse_samples.py -h', 'collate_alpha.py -h', 'compare_alpha_diversity.py -h', 'compare_categories.py -h', 'compare_distance_matrices.py -h', 'compare_taxa_summaries.py -h', 'compare_trajectories.py -h', 'compute_core_microbiome.py -h', 'compute_taxonomy_ratios.py -h', 'conditional_uncovered_probability.py -h', 'consensus_tree.py -h', 'convert_fastaqual_fastq.py -h', 'convert_unifrac_sample_mapping_to_otu_table.py -h', 'core_diversity_analyses.py -h', 'count_seqs.py -h', 'demultiplex_fasta.py -h', 'denoise_wrapper.py -h', 'denoiser.py -h', 'denoiser_preprocess.py -h', 'denoiser_worker.py -h', 'detrend.py -h', 'differential_abundance.py -h', 'dissimilarity_mtx_stats.py -h', 'distance_matrix_from_mapping.py -h', 'estimate_observation_richness.py -h', 'exclude_seqs_by_blast.py -h', 'extract_barcodes.py -h', 'extract_reads_from_interleaved_file.py -h', 'extract_seqs_by_sample_id.py -h', 'filter_alignment.py -h', 'filter_distance_matrix.py -h', 'filter_fasta.py -h', 'filter_otus_by_sample.py -h', 'filter_otus_from_otu_table.py -h', 'filter_samples_from_otu_table.py -h', 'filter_taxa_from_otu_table.py -h', 'filter_tree.py -h', 'fix_arb_fasta.py -h', 'group_significance.py -h', 'identify_chimeric_seqs.py -h', 'identify_missing_files.py -h', 'identify_paired_differences.py -h', 'inflate_denoiser_output.py -h', 'jackknifed_beta_diversity.py -h', 'join_paired_ends.py -h', 'load_remote_mapping_file.py -h', 'make_2d_plots.py -h', 'make_bipartite_network.py -h', 'make_bootstrapped_tree.py -h', 'make_distance_boxplots.py -h', 'make_distance_comparison_plots.py -h', 'make_fastq.py -h', 'make_library_id_lists.py -h', 'make_otu_heatmap.py -h', 'make_otu_heatmap_html.py -h', 'make_otu_network.py -h', 'make_otu_table.py -h', 'make_per_library_sff.py -h', 'make_phylogeny.py -h', 'make_prefs_file.py -h', 'make_qiime_py_file.py -h', 'make_rarefaction_plots.py -h', 'make_tep.py -h', 'map_reads_to_reference.py -h', 'merge_mapping_files.py -h', 'merge_otu_maps.py -h', 'merge_otu_tables.py -h', 'multiple_extract_barcodes.py -h', 'multiple_join_paired_ends.py -h', 'multiple_rarefactions.py -h', 'multiple_rarefactions_even_depth.py -h', 'multiple_split_libraries_fastq.py -h', 'neighbor_joining.py -h', 'nmds.py -h', 'normalize_table.py -h', 'observation_metadata_correlation.py -h', 'otu_category_significance.py -h', 'parallel_align_seqs_pynast.py -h', 'parallel_alpha_diversity.py -h', 'parallel_assign_taxonomy_blast.py -h', 'parallel_assign_taxonomy_rdp.py -h', 'parallel_assign_taxonomy_uclust.py -h', 'parallel_beta_diversity.py -h', 'parallel_blast.py -h', 'parallel_identify_chimeric_seqs.py -h', 'parallel_map_reads_to_reference.py -h', 'parallel_merge_otu_tables.py -h', 'parallel_multiple_rarefactions.py -h', 'parallel_pick_otus_blast.py -h', 'parallel_pick_otus_sortmerna.py -h', 'parallel_pick_otus_trie.py -h', 'parallel_pick_otus_uclust_ref.py -h', 'parallel_pick_otus_usearch61_ref.py -h', 'pick_closed_reference_otus.py -h', 'pick_de_novo_otus.py -h', 'pick_open_reference_otus.py -h', 'pick_otus.py -h', 'pick_rep_set.py -h', 'plot_rank_abundance_graph.py -h', 'plot_semivariogram.py -h', 'plot_taxa_summary.py -h', 'poller.py -h', 'principal_coordinates.py -h', 'print_metadata_stats.py -h', 'print_qiime_config.py -h', 'process_iseq.py -h', 'process_qseq.py -h', 'process_sff.py -h', 'quality_scores_plot.py -h', 'relatedness.py -h', 'shared_phylotypes.py -h', 'simsam.py -h', 'single_rarefaction.py -h', 'sort_otu_table.py -h', 'split_libraries.py -h', 'split_libraries_fastq.py -h', 'split_libraries_lea_seq.py -h', 'split_otu_table.py -h', 'split_otu_table_by_taxonomy.py -h', 'split_sequence_file_on_sample_ids.py -h', 'start_parallel_jobs.py -h', 'start_parallel_jobs_sc.py -h', 'start_parallel_jobs_slurm.py -h', 'start_parallel_jobs_torque.py -h', 'subsample_fasta.py -h', 'summarize_otu_by_cat.py -h', 'summarize_taxa.py -h', 'summarize_taxa_through_plots.py -h', 'supervised_learning.py -h', 'swarm_breaker.py -h', 'transform_coordinate_matrices.py -h', 'tree_compare.py -h', 'trflp_file_to_otu_table.py -h', 'trim_sff_primers.py -h', 'truncate_fasta_qual_files.py -h', 'truncate_reverse_primer.py -h', 'unweight_fasta.py -h', 'upgma_cluster.py -h', 'validate_demultiplexed_fasta.py -h', 'validate_mapping_file.py -h'])])), ('about', CommentedMap([('home', 'http://www.qiime.org'), ('license', 'GNU General Public License v2 (GPLv2)'), ('summary', 'Quantitative Insights Into Microbial Ecology')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/qiime/1.9.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'qiime'), ('version', '1.9.1')])), ('source', CommentedMap([('fn', 'qiime-1.9.1.tar.gz'), ('url', 'https://pypi.python.org/packages/source/q/qiime/qiime-1.9.1.tar.gz'), ('md5', '32d4db2b2c888dbbfc6aa33f02725526'), ('sha1', '2ab4ac9beb745ed3eca3243312cc9ce14e8a20d8'), ('sha256', 'c12dcdae1fe44762a49668276d7e93f487d87a0dc2ff8ca696ac0eff9d92a8a8')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'setuptools', 'numpy x.x', 'scipy >=0.14.0', 'cogent ==1.5.3', 'natsort <4.0.0', 'pynast ==1.2.2', 'qcli >=0.1.1,<0.2.0', 'gdata', 'biom-format >=2.1.4,<2.2.0', 'emperor >=0.9.51,<1.0.0', 'scikit-bio >=0.2.3,<0.3.0', 'matplotlib <=1.9', 'burrito-fillings >=0.1.1,<0.2.0', 'pandas >=0.13.1', 'burrito >=0.9.1,<1.0.0', 'qiime-default-reference >=0.1.2,<0.2.0', 'mock', 'nose', 'xz']), ('run', ['python', 'numpy x.x', 'scipy >=0.14.0', 'cogent ==1.5.3', 'natsort <4.0.0', 'pynast ==1.2.2', 'qcli >=0.1.1,<0.2.0', 'gdata', 'biom-format >=2.1.4,<2.2.0', 'emperor >=0.9.51,<1.0.0', 'scikit-bio >=0.2.3,<0.3.0', 'matplotlib <=1.9', 'burrito-fillings >=0.1.1,<0.2.0', 'pandas >=0.13.1', 'burrito >=0.9.1,<1.0.0', 'qiime-default-reference >=0.1.2,<0.2.0', 'mock', 'nose', 'libgcc', 'glib', 'xorg-libxext', 'xorg-libxdmcp', 'xorg-libsm', 'xorg-libxrender', 'xorg-libxau', 'xz'])])), ('test', CommentedMap([('imports', ['qiime', 'qiime.util', 'qiime.check_id_map', 'qiime.sort']), ('commands', ['add_alpha_to_mapping_file.py -h', 'add_qiime_labels.py -h', 'adjust_seq_orientation.py -h', 'align_seqs.py -h', 'alpha_diversity.py -h', 'alpha_rarefaction.py -h', 'amplicon_contingency_table.py', 'ampliconnoise.py -h', 'assign_taxonomy.py -h', 'beta_diversity.py -h', 'beta_diversity_through_plots.py -h', 'beta_significance.py -h', 'blast_wrapper.py -h', 'categorized_dist_scatterplot.py -h', 'check_id_map.py -h', 'clean_raxml_parsimony_tree.py -h', 'cluster_quality.py -h', 'collapse_samples.py -h', 'collate_alpha.py -h', 'compare_alpha_diversity.py -h', 'compare_categories.py -h', 'compare_distance_matrices.py -h', 'compare_taxa_summaries.py -h', 'compare_trajectories.py -h', 'compute_core_microbiome.py -h', 'compute_taxonomy_ratios.py -h', 'conditional_uncovered_probability.py -h', 'consensus_tree.py -h', 'convert_fastaqual_fastq.py -h', 'convert_unifrac_sample_mapping_to_otu_table.py -h', 'core_diversity_analyses.py -h', 'count_seqs.py -h', 'demultiplex_fasta.py -h', 'denoise_wrapper.py -h', 'denoiser.py -h', 'denoiser_preprocess.py -h', 'denoiser_worker.py -h', 'detrend.py -h', 'differential_abundance.py -h', 'dissimilarity_mtx_stats.py -h', 'distance_matrix_from_mapping.py -h', 'estimate_observation_richness.py -h', 'exclude_seqs_by_blast.py -h', 'extract_barcodes.py -h', 'extract_reads_from_interleaved_file.py -h', 'extract_seqs_by_sample_id.py -h', 'filter_alignment.py -h', 'filter_distance_matrix.py -h', 'filter_fasta.py -h', 'filter_otus_by_sample.py -h', 'filter_otus_from_otu_table.py -h', 'filter_samples_from_otu_table.py -h', 'filter_taxa_from_otu_table.py -h', 'filter_tree.py -h', 'fix_arb_fasta.py -h', 'group_significance.py -h', 'identify_chimeric_seqs.py -h', 'identify_missing_files.py -h', 'identify_paired_differences.py -h', 'inflate_denoiser_output.py -h', 'jackknifed_beta_diversity.py -h', 'join_paired_ends.py -h', 'load_remote_mapping_file.py -h', 'make_2d_plots.py -h', 'make_bipartite_network.py -h', 'make_bootstrapped_tree.py -h', 'make_distance_boxplots.py -h', 'make_distance_comparison_plots.py -h', 'make_fastq.py -h', 'make_library_id_lists.py -h', 'make_otu_heatmap.py -h', 'make_otu_heatmap_html.py -h', 'make_otu_network.py -h', 'make_otu_table.py -h', 'make_per_library_sff.py -h', 'make_phylogeny.py -h', 'make_prefs_file.py -h', 'make_qiime_py_file.py -h', 'make_rarefaction_plots.py -h', 'make_tep.py -h', 'map_reads_to_reference.py -h', 'merge_mapping_files.py -h', 'merge_otu_maps.py -h', 'merge_otu_tables.py -h', 'multiple_extract_barcodes.py -h', 'multiple_join_paired_ends.py -h', 'multiple_rarefactions.py -h', 'multiple_rarefactions_even_depth.py -h', 'multiple_split_libraries_fastq.py -h', 'neighbor_joining.py -h', 'nmds.py -h', 'normalize_table.py -h', 'observation_metadata_correlation.py -h', 'otu_category_significance.py -h', 'parallel_align_seqs_pynast.py -h', 'parallel_alpha_diversity.py -h', 'parallel_assign_taxonomy_blast.py -h', 'parallel_assign_taxonomy_rdp.py -h', 'parallel_assign_taxonomy_uclust.py -h', 'parallel_beta_diversity.py -h', 'parallel_blast.py -h', 'parallel_identify_chimeric_seqs.py -h', 'parallel_map_reads_to_reference.py -h', 'parallel_merge_otu_tables.py -h', 'parallel_multiple_rarefactions.py -h', 'parallel_pick_otus_blast.py -h', 'parallel_pick_otus_sortmerna.py -h', 'parallel_pick_otus_trie.py -h', 'parallel_pick_otus_uclust_ref.py -h', 'parallel_pick_otus_usearch61_ref.py -h', 'pick_closed_reference_otus.py -h', 'pick_de_novo_otus.py -h', 'pick_open_reference_otus.py -h', 'pick_otus.py -h', 'pick_rep_set.py -h', 'plot_rank_abundance_graph.py -h', 'plot_semivariogram.py -h', 'plot_taxa_summary.py -h', 'poller.py -h', 'principal_coordinates.py -h', 'print_metadata_stats.py -h', 'print_qiime_config.py -h', 'process_iseq.py -h', 'process_qseq.py -h', 'process_sff.py -h', 'quality_scores_plot.py -h', 'relatedness.py -h', 'shared_phylotypes.py -h', 'simsam.py -h', 'single_rarefaction.py -h', 'sort_otu_table.py -h', 'split_libraries.py -h', 'split_libraries_fastq.py -h', 'split_libraries_lea_seq.py -h', 'split_otu_table.py -h', 'split_otu_table_by_taxonomy.py -h', 'split_sequence_file_on_sample_ids.py -h', 'start_parallel_jobs.py -h', 'start_parallel_jobs_sc.py -h', 'start_parallel_jobs_slurm.py -h', 'start_parallel_jobs_torque.py -h', 'subsample_fasta.py -h', 'summarize_otu_by_cat.py -h', 'summarize_taxa.py -h', 'summarize_taxa_through_plots.py -h', 'supervised_learning.py -h', 'swarm_breaker.py -h', 'transform_coordinate_matrices.py -h', 'tree_compare.py -h', 'trflp_file_to_otu_table.py -h', 'trim_sff_primers.py -h', 'truncate_fasta_qual_files.py -h', 'truncate_reverse_primer.py -h', 'unweight_fasta.py -h', 'upgma_cluster.py -h', 'validate_demultiplexed_fasta.py -h', 'validate_mapping_file.py -h'])])), ('about', CommentedMap([('home', 'http://www.qiime.org'), ('license', 'GNU General Public License v2 (GPLv2)'), ('summary', 'Quantitative Insights Into Microbial Ecology')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))]))\n",
      "EXTRA has been found: qiime\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqprep/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'seqprep'), ('version', 1.1)])), ('source', CommentedMap([('fn', 'v1.1.tar.gz'), ('url', 'https://github.com/jstjohn/SeqPrep/archive/v1.1.tar.gz'), ('md5', '42bd8e62e8bf99befb37ed29ce047115'), ('patches', ['cflags.patch'])])), ('requirements', CommentedMap([('build', ['zlib']), ('run', ['zlib'])])), ('about', CommentedMap([('home', 'https://github.com/jstjohn/SeqPrep'), ('license', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqprep/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'seqprep'), ('version', 1.1)])), ('source', CommentedMap([('fn', 'v1.1.tar.gz'), ('url', 'https://github.com/jstjohn/SeqPrep/archive/v1.1.tar.gz'), ('md5', '42bd8e62e8bf99befb37ed29ce047115'), ('patches', ['cflags.patch'])])), ('requirements', CommentedMap([('build', ['zlib']), ('run', ['zlib'])])), ('about', CommentedMap([('home', 'https://github.com/jstjohn/SeqPrep'), ('license', 'MIT')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/khmer/2.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://khmer.readthedocs.org/'), ('license', 'BSD-3-Clause'), ('summary', 'khmer is a library and suite of command line tools for working with DNA sequence. It is primarily aimed at short-read sequencing data such as that produced by the Illumina platform. khmer takes a k-mer-centric approach to sequence analysis, hence the name.')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'khmer'), ('version', '2.0')])), ('requirements', CommentedMap([('build', ['python', 'bz2file ==0.98', 'screed ==0.9', 'gcc', 'llvm']), ('run', ['python', 'bz2file ==0.98', 'screed ==0.9', 'libgcc'])])), ('source', CommentedMap([('fn', 'khmer-2.0.tar.gz'), ('sha256', '6a8c6809b50adaf2d7d7307eefcca173f4b536329562530400fb1c88a12e6071'), ('url', 'https://pypi.python.org/packages/source/k/khmer/khmer-2.0.tar.gz')])), ('test', CommentedMap([('imports', ['khmer', 'khmer.kfile', 'khmer.khmer_args', 'khmer.utils'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/khmer/2.0/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://khmer.readthedocs.org/'), ('license', 'BSD-3-Clause'), ('summary', 'khmer is a library and suite of command line tools for working with DNA sequence. It is primarily aimed at short-read sequencing data such as that produced by the Illumina platform. khmer takes a k-mer-centric approach to sequence analysis, hence the name.')])), ('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'khmer'), ('version', '2.0')])), ('requirements', CommentedMap([('build', ['python', 'bz2file ==0.98', 'screed ==0.9', 'gcc', 'llvm']), ('run', ['python', 'bz2file ==0.98', 'screed ==0.9', 'libgcc'])])), ('source', CommentedMap([('fn', 'khmer-2.0.tar.gz'), ('sha256', '6a8c6809b50adaf2d7d7307eefcca173f4b536329562530400fb1c88a12e6071'), ('url', 'https://pypi.python.org/packages/source/k/khmer/khmer-2.0.tar.gz')])), ('test', CommentedMap([('imports', ['khmer', 'khmer.kfile', 'khmer.khmer_args', 'khmer.utils'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genefilter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genefilter'), ('version', '1.58.1')])), ('source', CommentedMap([('fn', 'genefilter_1.58.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/genefilter_1.58.1.tar.gz', 'https://depot.galaxyproject.org/software/genefilter/genefilter_1.58.1_src_all.tar.gz']), ('md5', 'bc1a90bdf93d8db994220545cd80f438')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-s4vectors >=0.9.42', 'r-base', 'r-survival', 'gcc']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-s4vectors >=0.9.42', 'r-base', 'r-survival', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'genefilter\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/genefilter.html'), ('license', 'Artistic-2.0'), ('summary', 'Some basic functions for filtering genes')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-genefilter/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-genefilter'), ('version', '1.58.1')])), ('source', CommentedMap([('fn', 'genefilter_1.58.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/genefilter_1.58.1.tar.gz', 'https://depot.galaxyproject.org/software/genefilter/genefilter_1.58.1_src_all.tar.gz']), ('md5', 'bc1a90bdf93d8db994220545cd80f438')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-s4vectors >=0.9.42', 'r-base', 'r-survival', 'gcc']), ('run', ['bioconductor-annotate', 'bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-s4vectors >=0.9.42', 'r-base', 'r-survival', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'genefilter\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/genefilter.html'), ('license', 'Artistic-2.0'), ('summary', 'Some basic functions for filtering genes')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnaseqcomp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnaseqcomp'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'rnaseqcomp_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rnaseqcomp_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/rnaseqcomp_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/rnaseqcomp/rnaseqcomp_1.6.0_src_all.tar.gz']), ('md5', '3aabf98f1c100ac627d37fc797e48bb9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rnaseqcomp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rnaseqcomp.html'), ('license', 'GPL-3'), ('summary', 'Several quantitative and visualized benchmarks for RNA-seq quantification pipelines. Two-condition quantifications for genes, transcripts, junctions or exons by each pipeline with necessary meta information should be organized into numeric matrices in order to proceed the evaluation.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rnaseqcomp/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rnaseqcomp'), ('version', '1.6.0')])), ('source', CommentedMap([('fn', 'rnaseqcomp_1.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/rnaseqcomp_1.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/rnaseqcomp_1.6.0.tar.gz', 'https://depot.galaxyproject.org/software/rnaseqcomp/rnaseqcomp_1.6.0_src_all.tar.gz']), ('md5', '3aabf98f1c100ac627d37fc797e48bb9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-rcolorbrewer']), ('run', ['r-base', 'r-rcolorbrewer'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'rnaseqcomp\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/rnaseqcomp.html'), ('license', 'GPL-3'), ('summary', 'Several quantitative and visualized benchmarks for RNA-seq quantification pipelines. Two-condition quantifications for genes, transcripts, junctions or exons by each pipeline with necessary meta information should be organized into numeric matrices in order to proceed the evaluation.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/discovar/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://www.broadinstitute.org/software/discovar'), ('license', 'MIT'), ('summary', 'Suitable for variant calling with reference and de novo assembly of small genomes.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'discovar'), ('version', '52488')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'discovar-52488.tar.gz'), ('url', 'ftp://ftp.broadinstitute.org/pub/crd/Discovar/latest_source_code/discovar-52488.tar.gz')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/discovar/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://www.broadinstitute.org/software/discovar'), ('license', 'MIT'), ('summary', 'Suitable for variant calling with reference and de novo assembly of small genomes.')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'discovar'), ('version', '52488')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'discovar-52488.tar.gz'), ('url', 'ftp://ftp.broadinstitute.org/pub/crd/Discovar/latest_source_code/discovar-52488.tar.gz')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mzr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mzr'), ('version', '2.6.3')])), ('source', CommentedMap([('fn', 'mzR_2.6.3.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-mzr/bioconductor-mzr_2.6.3_src_all.tar.gz'), ('md5', 'c36c4a9ec2ec82642f5ffae56d9a08ea')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-protgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-rcpp >=0.10.1', 'libnetcdf ==4.3.3.1', 'gcc']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-protgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-rcpp >=0.10.1', 'libnetcdf ==4.3.3.1', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mzR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/mzR.html'), ('license', 'Artistic-2.0'), ('summary', 'mzR provides a unified API to the common file formats and parsers available for mass spectrometry data. It comes with a wrapper for the ISB random access parser for mass spectrometry mzXML, mzData and mzML files. The package contains the original code written by the ISB, and a subset of the proteowizard library for mzML and mzIdentML. The netCDF reading code has previously been used in XCMS.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mzr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mzr'), ('version', '2.6.3')])), ('source', CommentedMap([('fn', 'mzR_2.6.3.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-mzr/bioconductor-mzr_2.6.3_src_all.tar.gz'), ('md5', 'c36c4a9ec2ec82642f5ffae56d9a08ea')])), ('build', CommentedMap([('number', 1), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-protgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-rcpp >=0.10.1', 'libnetcdf ==4.3.3.1', 'gcc']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics >=0.13.6', 'bioconductor-protgenerics', 'bioconductor-zlibbioc', 'r-base', 'r-rcpp >=0.10.1', 'libnetcdf ==4.3.3.1', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mzR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/mzR.html'), ('license', 'Artistic-2.0'), ('summary', 'mzR provides a unified API to the common file formats and parsers available for mass spectrometry data. It comes with a wrapper for the ISB random access parser for mass spectrometry mzXML, mzData and mzML files. The package contains the original code written by the ISB, and a subset of the proteowizard library for mzML and mzIdentML. The netCDF reading code has previously been used in XCMS.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dupradar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dupradar'), ('version', '1.2.2')])), ('source', CommentedMap([('fn', 'dupRadar_1.2.2.tar.gz'), ('url', 'http://www.imsbio.co.jp/RGM-files/R_BC/download/dupRadar_1.2.2.tar.gz'), ('sha256', '0c6a907efe9035dc6ba386160f9f7fb47b18b39e8a2dcfeecb2743f14ad607d8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-rsubread >=1.14.1', 'r-kernsmooth', 'r-base']), ('run', ['bioconductor-rsubread >=1.14.1', 'r-kernsmooth', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'dupRadar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/dupRadar.html'), ('license', 'GPL-3'), ('summary', 'Duplication rate quality control for RNA-Seq datasets.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dupradar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dupradar'), ('version', '1.2.2')])), ('source', CommentedMap([('fn', 'dupRadar_1.2.2.tar.gz'), ('url', 'http://www.imsbio.co.jp/RGM-files/R_BC/download/dupRadar_1.2.2.tar.gz'), ('sha256', '0c6a907efe9035dc6ba386160f9f7fb47b18b39e8a2dcfeecb2743f14ad607d8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-rsubread >=1.14.1', 'r-kernsmooth', 'r-base']), ('run', ['bioconductor-rsubread >=1.14.1', 'r-kernsmooth', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'dupRadar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/dupRadar.html'), ('license', 'GPL-3'), ('summary', 'Duplication rate quality control for RNA-Seq datasets.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qrqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qrqc'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'qrqc_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/qrqc_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/qrqc_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/qrqc/qrqc_1.30.0_src_all.tar.gz']), ('md5', 'cd845c1919613e24d83aa91b4641afc9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-rsamtools >=1.19.38', 'r-base', 'r-brew', 'r-ggplot2', 'r-plyr', 'r-reshape', 'r-testthat', 'r-xtable']), ('run', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-rsamtools >=1.19.38', 'r-base', 'r-brew', 'r-ggplot2', 'r-plyr', 'r-reshape', 'r-testthat', 'r-xtable'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'qrqc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/qrqc.html'), ('license', 'GPL (>=2)'), ('summary', 'Quickly scans reads and gathers statistics on base and quality frequencies, read length, k-mers by position, and frequent sequences. Produces graphical output of statistics for use in quality control pipelines, and an optional HTML quality report. S4 SequenceSummary objects allow specific tests and functionality to be written around the data collected.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-qrqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-qrqc'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'qrqc_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/qrqc_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/qrqc_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/qrqc/qrqc_1.30.0_src_all.tar.gz']), ('md5', 'cd845c1919613e24d83aa91b4641afc9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-rsamtools >=1.19.38', 'r-base', 'r-brew', 'r-ggplot2', 'r-plyr', 'r-reshape', 'r-testthat', 'r-xtable']), ('run', ['bioconductor-biostrings', 'bioconductor-biovizbase', 'bioconductor-rsamtools >=1.19.38', 'r-base', 'r-brew', 'r-ggplot2', 'r-plyr', 'r-reshape', 'r-testthat', 'r-xtable'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'qrqc\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/qrqc.html'), ('license', 'GPL (>=2)'), ('summary', 'Quickly scans reads and gathers statistics on base and quality frequencies, read length, k-mers by position, and frequent sequences. Produces graphical output of statistics for use in quality control pipelines, and an optional HTML quality report. S4 SequenceSummary objects allow specific tests and functionality to be written around the data collected.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbgl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbgl'), ('version', '1.48.1')])), ('source', CommentedMap([('fn', 'RBGL_1.48.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/RBGL_1.48.1.tar.gz'), ('md5', 'e012e1022b4392c9dd90677ac69798c6')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base']), ('run', ['bioconductor-graph', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RBGL\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RBGL.html'), ('license', 'Artistic-2.0'), ('summary', 'A fairly extensive and comprehensive interface to the graph algorithms contained in the BOOST library.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rbgl/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rbgl'), ('version', '1.48.1')])), ('source', CommentedMap([('fn', 'RBGL_1.48.1.tar.gz'), ('url', 'http://bioconductor.org/packages/3.3/bioc/src/contrib/RBGL_1.48.1.tar.gz'), ('md5', 'e012e1022b4392c9dd90677ac69798c6')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'r-base']), ('run', ['bioconductor-graph', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'RBGL\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/RBGL.html'), ('license', 'Artistic-2.0'), ('summary', 'A fairly extensive and comprehensive interface to the graph algorithms contained in the BOOST library.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/peakzilla/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'peakzilla'), ('version', 1.0)])), ('source', CommentedMap([('fn', '0fc94780ee1a8c2b6395b5c7489e21a70eeb5217.tar.gz'), ('url', 'https://github.com/steinmann/peakzilla/archive/0fc94780ee1a8c2b6395b5c7489e21a70eeb5217.tar.gz'), ('sha256', '982d2bbcde736e4ce7837a7758ce37e8040cd1cc148c64a3856b9ea837f2740c'), ('patches', ['indent.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['peakzilla.py -h'])])), ('about', CommentedMap([('home', 'http://stark.imp.ac.at/data/peakzilla'), ('summary', 'Peakzilla identifies sites of enrichment and transcription factor binding sites from transcription factor ChIP-seq and ChIP-exo experiments at hight accuracy and resolution.'), ('license', 'GPLv2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/peakzilla/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'peakzilla'), ('version', 1.0)])), ('source', CommentedMap([('fn', '0fc94780ee1a8c2b6395b5c7489e21a70eeb5217.tar.gz'), ('url', 'https://github.com/steinmann/peakzilla/archive/0fc94780ee1a8c2b6395b5c7489e21a70eeb5217.tar.gz'), ('sha256', '982d2bbcde736e4ce7837a7758ce37e8040cd1cc148c64a3856b9ea837f2740c'), ('patches', ['indent.patch'])])), ('build', CommentedMap([('number', 1)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('commands', ['peakzilla.py -h'])])), ('about', CommentedMap([('home', 'http://stark.imp.ac.at/data/peakzilla'), ('summary', 'Peakzilla identifies sites of enrichment and transcription factor binding sites from transcription factor ChIP-seq and ChIP-exo experiments at hight accuracy and resolution.'), ('license', 'GPLv2')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-atlasrdf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-atlasrdf'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'AtlasRDF_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AtlasRDF_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/AtlasRDF/AtlasRDF_1.12.0_src_all.tar.gz']), ('md5', '3bff6e8da872b7ddf1de9962e81ef7b5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-hash', 'r-sparql']), ('run', ['r-base', 'r-hash', 'r-sparql'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AtlasRDF\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AtlasRDF.html'), ('license', 'Apache License 2.0'), ('summary', 'Query the Gene Expression Atlas RDF data at the European Bioinformatics Institute using genes, experimental factors (such as disease, cell type, compound treatments), pathways and proteins. Also contains a function to perform an enrichment of your gene list across Experimental Factor Ontology (EFO) using the Atlas background set.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-atlasrdf/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-atlasrdf'), ('version', '1.12.0')])), ('source', CommentedMap([('fn', 'AtlasRDF_1.12.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/AtlasRDF_1.12.0.tar.gz', 'https://depot.galaxyproject.org/software/AtlasRDF/AtlasRDF_1.12.0_src_all.tar.gz']), ('md5', '3bff6e8da872b7ddf1de9962e81ef7b5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-hash', 'r-sparql']), ('run', ['r-base', 'r-hash', 'r-sparql'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'AtlasRDF\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/AtlasRDF.html'), ('license', 'Apache License 2.0'), ('summary', 'Query the Gene Expression Atlas RDF data at the European Bioinformatics Institute using genes, experimental factors (such as disease, cell type, compound treatments), pathways and proteins. Also contains a function to perform an enrichment of your gene list across Experimental Factor Ontology (EFO) using the Atlas background set.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rapsearch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rapsearch'), ('version', 2.24)])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('source', CommentedMap([('fn', 'RAPSearch2.24_64bits.tar.gz.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/rapsearch2/RAPSearch2.24_64bits.tar.gz'), ('md5', '573ff48c2815c7e07f0268780c2590a8')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['rapsearch 2>&1 | grep rapsearch', 'prerapsearch 2>&1 | grep \"Usage\\\\:\"'])])), ('about', CommentedMap([('home', 'http://omics.informatics.indiana.edu/mg/RAPSearch2/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'RAPSearch2 is a tool for fast protein similarity searches.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rapsearch/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rapsearch'), ('version', 2.24)])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('source', CommentedMap([('fn', 'RAPSearch2.24_64bits.tar.gz.tar.gz'), ('url', 'http://downloads.sourceforge.net/project/rapsearch2/RAPSearch2.24_64bits.tar.gz'), ('md5', '573ff48c2815c7e07f0268780c2590a8')])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['rapsearch 2>&1 | grep rapsearch', 'prerapsearch 2>&1 | grep \"Usage\\\\:\"'])])), ('about', CommentedMap([('home', 'http://omics.informatics.indiana.edu/mg/RAPSearch2/'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'RAPSearch2 is a tool for fast protein similarity searches.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eigensoft/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eigensoft'), ('version', '6.0.1')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'EIG6.0.1.tar.gz'), ('url', 'https://data.broadinstitute.org/alkesgroup/EIGENSOFT/OLD/EIG6.0.1.tar.gz'), ('md5', 'a6ed8f2cb48fe76886dcb97af289596c')])), ('requirements', CommentedMap([('build', ['gcc', 'openblas', 'gsl ==1.16']), ('run', ['libgcc', 'openblas', 'gsl ==1.16', 'perl'])])), ('test', CommentedMap([('commands', ['smartpca 2>&1 | grep \"no parameters\"'])])), ('about', CommentedMap([('home', 'http://www.hsph.harvard.edu/alkes-price/software/'), ('license', 'Custom OSS'), ('summary', 'Population Genetics - Principal Components Analysis (PCA)')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/eigensoft/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'eigensoft'), ('version', '6.0.1')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'EIG6.0.1.tar.gz'), ('url', 'https://data.broadinstitute.org/alkesgroup/EIGENSOFT/OLD/EIG6.0.1.tar.gz'), ('md5', 'a6ed8f2cb48fe76886dcb97af289596c')])), ('requirements', CommentedMap([('build', ['gcc', 'openblas', 'gsl ==1.16']), ('run', ['libgcc', 'openblas', 'gsl ==1.16', 'perl'])])), ('test', CommentedMap([('commands', ['smartpca 2>&1 | grep \"no parameters\"'])])), ('about', CommentedMap([('home', 'http://www.hsph.harvard.edu/alkes-price/software/'), ('license', 'Custom OSS'), ('summary', 'Population Genetics - Principal Components Analysis (PCA)')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dnashaper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dnashaper'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'DNAshapeR_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/DNAshapeR_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/DNAshapeR_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/DNAshapeR/DNAshapeR_1.4.0_src_all.tar.gz']), ('sha256', '946ee0f468f94b4776c74b013b88c3f75091415378f84aebc4864cd569e46e9e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'r-base', 'r-fields', 'r-rcpp >=0.12.1', 'gcc', 'llvm']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'r-base', 'r-fields', 'r-rcpp >=0.12.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DNAshapeR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/DNAshapeR.html'), ('license', 'GPL-2'), ('summary', 'DNAhapeR is an R/BioConductor package for ultra-fast, high-throughput predictions of DNA shape features. The package allows to predict, visualize and encode DNA shape features for statistical learning.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-dnashaper/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-dnashaper'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'DNAshapeR_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/DNAshapeR_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/DNAshapeR_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/DNAshapeR/DNAshapeR_1.4.0_src_all.tar.gz']), ('sha256', '946ee0f468f94b4776c74b013b88c3f75091415378f84aebc4864cd569e46e9e')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'r-base', 'r-fields', 'r-rcpp >=0.12.1', 'gcc', 'llvm']), ('run', ['bioconductor-biostrings', 'bioconductor-genomicranges', 'r-base', 'r-fields', 'r-rcpp >=0.12.1'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'DNAshapeR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/DNAshapeR.html'), ('license', 'GPL-2'), ('summary', 'DNAhapeR is an R/BioConductor package for ultra-fast, high-throughput predictions of DNA shape features. The package allows to predict, visualize and encode DNA shape features for statistical learning.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sga/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'sga'), ('version', '0.10.13')])), ('source', CommentedMap([('fn', 'v0.10.13.tar.gz'), ('url', 'https://github.com/jts/sga/archive/v0.10.13.tar.gz'), ('md5', 'd4f6aefc48c940dba96cc6513649ecdd')])), ('requirements', CommentedMap([('build', ['bamtools']), ('run', ['bamtools'])])), ('about', CommentedMap([('home', 'https://github.com/jts/sga'), ('license', 'GPLv3'), ('summary', 'SGA - String Graph Assembler. SGA is a de novo assembler for DNA sequence reads. It is based on Gene Myers string graph formulation of assembly and uses the FM-index/Burrows-Wheeler transform to efficiently find overlaps between sequence reads.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/sga/meta.yaml', CommentedMap([('build', CommentedMap([('number', 0), ('skip', True)])), ('package', CommentedMap([('name', 'sga'), ('version', '0.10.13')])), ('source', CommentedMap([('fn', 'v0.10.13.tar.gz'), ('url', 'https://github.com/jts/sga/archive/v0.10.13.tar.gz'), ('md5', 'd4f6aefc48c940dba96cc6513649ecdd')])), ('requirements', CommentedMap([('build', ['bamtools']), ('run', ['bamtools'])])), ('about', CommentedMap([('home', 'https://github.com/jts/sga'), ('license', 'GPLv3'), ('summary', 'SGA - String Graph Assembler. SGA is a de novo assembler for DNA sequence reads. It is based on Gene Myers string graph formulation of assembly and uses the FM-index/Burrows-Wheeler transform to efficiently find overlaps between sequence reads.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sscu/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sscu'), ('version', '2.6.0')])), ('source', CommentedMap([('fn', 'sscu_2.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sscu_2.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/sscu_2.6.0.tar.gz', 'https://depot.galaxyproject.org/software/sscu/sscu_2.6.0_src_all.tar.gz']), ('md5', '7bdf279b3cfb60538d13df4e4fa25eb2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.16.1', 'bioconductor-biostrings >=2.36.4', 'r-base', 'r-seqinr >=3.1-3']), ('run', ['bioconductor-biocgenerics >=0.16.1', 'bioconductor-biostrings >=2.36.4', 'r-base', 'r-seqinr >=3.1-3'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sscu\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sscu.html'), ('license', 'GPL (>= 2)'), ('summary', \"The package calculates the indexes for selective stength in codon usage in bacteria species. (1) The package can calculate the strength of selected codon usage bias (sscu, also named as s_index) based on Paul Sharp's method. The method take into account of background mutation rate, and focus only on four pairs of codons with universal translational advantages in all bacterial species. Thus the sscu index is comparable among different species. (2) The package can detect the strength of translational accuracy selection by Akashi's test. The test tabulating all codons into four categories with the feature as conserved/variable amino acids and optimal/non-optimal codons. (3) Optimal codon lists (selected codons) can be calculated by either op_highly function (by using the highly expressed genes compared with all genes to identify optimal codons), or op_corre_CodonW/op_corre_NCprime function (by correlative method developed by Hershberg & Petrov). Users will have a list of optimal codons for further analysis, such as input to the Akashi's test. (4) The detailed codon usage information, such as RSCU value, number of optimal codons in the highly/all gene set, as well as the genomic gc3 value, can be calculate by the optimal_codon_statistics and genomic_gc3 function. (5) Furthermore, we added one test function low_frequency_op in the package. The function try to find the low frequency optimal codons, among all the optimal codons identified by the op_highly function.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sscu/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sscu'), ('version', '2.6.0')])), ('source', CommentedMap([('fn', 'sscu_2.6.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/sscu_2.6.0.tar.gz', 'https://bioarchive.galaxyproject.org/sscu_2.6.0.tar.gz', 'https://depot.galaxyproject.org/software/sscu/sscu_2.6.0_src_all.tar.gz']), ('md5', '7bdf279b3cfb60538d13df4e4fa25eb2')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.16.1', 'bioconductor-biostrings >=2.36.4', 'r-base', 'r-seqinr >=3.1-3']), ('run', ['bioconductor-biocgenerics >=0.16.1', 'bioconductor-biostrings >=2.36.4', 'r-base', 'r-seqinr >=3.1-3'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'sscu\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/sscu.html'), ('license', 'GPL (>= 2)'), ('summary', \"The package calculates the indexes for selective stength in codon usage in bacteria species. (1) The package can calculate the strength of selected codon usage bias (sscu, also named as s_index) based on Paul Sharp's method. The method take into account of background mutation rate, and focus only on four pairs of codons with universal translational advantages in all bacterial species. Thus the sscu index is comparable among different species. (2) The package can detect the strength of translational accuracy selection by Akashi's test. The test tabulating all codons into four categories with the feature as conserved/variable amino acids and optimal/non-optimal codons. (3) Optimal codon lists (selected codons) can be calculated by either op_highly function (by using the highly expressed genes compared with all genes to identify optimal codons), or op_corre_CodonW/op_corre_NCprime function (by correlative method developed by Hershberg & Petrov). Users will have a list of optimal codons for further analysis, such as input to the Akashi's test. (4) The detailed codon usage information, such as RSCU value, number of optimal codons in the highly/all gene set, as well as the genomic gc3 value, can be calculate by the optimal_codon_statistics and genomic_gc3 function. (5) Furthermore, we added one test function low_frequency_op in the package. The function try to find the low frequency optimal codons, among all the optimal codons identified by the op_highly function.\")]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pvac/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pvac'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'pvac_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pvac_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/pvac_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/pvac/pvac_1.24.0_src_all.tar.gz']), ('md5', '2a9ca3fc296ccadb3769422a20869e67')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.20.0', 'bioconductor-biobase', 'r-base']), ('run', ['bioconductor-affy >=1.20.0', 'bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pvac\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pvac.html'), ('license', 'LGPL (>= 2.0)'), ('summary', 'The package contains the function for filtering genes by the proportion of variation accounted for by the first principal component (PVAC).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-pvac/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-pvac'), ('version', '1.24.0')])), ('source', CommentedMap([('fn', 'pvac_1.24.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/pvac_1.24.0.tar.gz', 'https://bioarchive.galaxyproject.org/pvac_1.24.0.tar.gz', 'https://depot.galaxyproject.org/software/pvac/pvac_1.24.0_src_all.tar.gz']), ('md5', '2a9ca3fc296ccadb3769422a20869e67')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affy >=1.20.0', 'bioconductor-biobase', 'r-base']), ('run', ['bioconductor-affy >=1.20.0', 'bioconductor-biobase', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'pvac\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/pvac.html'), ('license', 'LGPL (>= 2.0)'), ('summary', 'The package contains the function for filtering genes by the proportion of variation accounted for by the first principal component (PVAC).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-fastseg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-fastseg'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'fastseg_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/fastseg_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/fastseg_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/fastseg/fastseg_1.20.0_src_all.tar.gz']), ('md5', '1840f449546b1d89983e95a6280006c1')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'fastseg\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/fastseg.html'), ('license', 'LGPL (>= 2.0)'), ('summary', \"fastseg implements a very fast and efficient segmentation algorithm. It has similar functionality as DNACopy (Olshen and Venkatraman 2004), but is considerably faster and more flexible. fastseg can segment data from DNA microarrays and data from next generation sequencing for example to detect copy number segments. Further it can segment data from RNA microarrays like tiling arrays to identify transcripts. Most generally, it can segment data given as a matrix or as a vector. Various data formats can be used as input to fastseg like expression set objects for microarrays or GRanges for sequencing data. The segmentation criterion of fastseg is based on a statistical test in a Bayesian framework, namely the cyber t-test (Baldi 2001). The speed-up arises from the facts, that sampling is not necessary in for fastseg and that a dynamic programming approach is used for calculation of the segments' first and higher order moments.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-fastseg/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-fastseg'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'fastseg_1.20.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/fastseg_1.20.0.tar.gz', 'https://bioarchive.galaxyproject.org/fastseg_1.20.0.tar.gz', 'https://depot.galaxyproject.org/software/fastseg/fastseg_1.20.0_src_all.tar.gz']), ('md5', '1840f449546b1d89983e95a6280006c1')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-s4vectors', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'fastseg\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/fastseg.html'), ('license', 'LGPL (>= 2.0)'), ('summary', \"fastseg implements a very fast and efficient segmentation algorithm. It has similar functionality as DNACopy (Olshen and Venkatraman 2004), but is considerably faster and more flexible. fastseg can segment data from DNA microarrays and data from next generation sequencing for example to detect copy number segments. Further it can segment data from RNA microarrays like tiling arrays to identify transcripts. Most generally, it can segment data given as a matrix or as a vector. Various data formats can be used as input to fastseg like expression set objects for microarrays or GRanges for sequencing data. The segmentation criterion of fastseg is based on a statistical test in a Bayesian framework, namely the cyber t-test (Baldi 2001). The speed-up arises from the facts, that sampling is not necessary in for fastseg and that a dynamic programming approach is used for calculation of the segments' first and higher order moments.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/effectivet3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'effectivet3'), ('version', '1.0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/TTSS_GUI/TTSS_GUI_1.0.1_src_all.jar'), ('fn', 'TTSS_GUI-1.0.1.jar'), ('sha256', '5a811a86a14dffbc5ae91dc45f40791642b8866266ffd2337b97d4c298129cab')])), ('requirements', CommentedMap([('build', ['openjdk >=6', 'curl', 'coreutils']), ('run', ['openjdk >=6', 'python'])])), ('about', CommentedMap([('home', 'http://www.compbio.dundee.ac.uk/nod'), ('license', 'Apache License, Version 2.0 + others used internally'), ('summary', 'Command line NoD (clinod), for  predicting nucleolar localization sequences.')])), ('test', CommentedMap([('files', ['empty.fasta']), ('commands', ['effectivet3 -f empty.fasta -m TTSS_ANIMAL-1.0.1.jar -t selective -q', 'effectivet3 -f empty.fasta -m TTSS_PLANT-1.0.1.jar -t selective -q', 'effectivet3 -f empty.fasta -m TTSS_STD-1.0.1.jar -t selective -q', 'effectivet3 -f empty.fasta -m TTSS_STD-2.0.2.jar -t selective -q'])])), ('extra', CommentedMap([('notes', 'EffectiveT3 is Java program which is packaged with a custom wrapper shell\\nscript. This shell wrapper is called \"effectivet3\" and is on $PATH by\\ndefault. By default \"-Xms512m -Xmx1g\" is set in the wrapper. If you want\\nto overwrite it you can specify these values directly after your binaries.\\nIf you have _JAVA_OPTIONS set globally this will take precedence.\\nFor example run it with \"effectivet3 -Xms512m -Xmx1g ...\"\\n')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/effectivet3/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'effectivet3'), ('version', '1.0.1')])), ('build', CommentedMap([('number', 0)])), ('source', CommentedMap([('url', 'https://depot.galaxyproject.org/software/TTSS_GUI/TTSS_GUI_1.0.1_src_all.jar'), ('fn', 'TTSS_GUI-1.0.1.jar'), ('sha256', '5a811a86a14dffbc5ae91dc45f40791642b8866266ffd2337b97d4c298129cab')])), ('requirements', CommentedMap([('build', ['openjdk >=6', 'curl', 'coreutils']), ('run', ['openjdk >=6', 'python'])])), ('about', CommentedMap([('home', 'http://www.compbio.dundee.ac.uk/nod'), ('license', 'Apache License, Version 2.0 + others used internally'), ('summary', 'Command line NoD (clinod), for  predicting nucleolar localization sequences.')])), ('test', CommentedMap([('files', ['empty.fasta']), ('commands', ['effectivet3 -f empty.fasta -m TTSS_ANIMAL-1.0.1.jar -t selective -q', 'effectivet3 -f empty.fasta -m TTSS_PLANT-1.0.1.jar -t selective -q', 'effectivet3 -f empty.fasta -m TTSS_STD-1.0.1.jar -t selective -q', 'effectivet3 -f empty.fasta -m TTSS_STD-2.0.2.jar -t selective -q'])])), ('extra', CommentedMap([('notes', 'EffectiveT3 is Java program which is packaged with a custom wrapper shell\\nscript. This shell wrapper is called \"effectivet3\" and is on $PATH by\\ndefault. By default \"-Xms512m -Xmx1g\" is set in the wrapper. If you want\\nto overwrite it you can specify these values directly after your binaries.\\nIf you have _JAVA_OPTIONS set globally this will take precedence.\\nFor example run it with \"effectivet3 -Xms512m -Xmx1g ...\"\\n')]))]))\n",
      "EXTRA has been found: effectivet3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-purecn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-purecn'), ('version', '1.6.3')])), ('source', CommentedMap([('fn', 'PureCN_1.6.3.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PureCN_1.6.3.tar.gz', 'https://depot.galaxyproject.org/software/PureCN/PureCN_1.6.3_src_all.tar.gz']), ('md5', '9dce6c929693f3d4967e019b83e79fef')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.20.3', 'bioconductor-iranges >=2.2.1', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.14.1', 'r-base', 'r-data.table', 'r-futile.logger', 'r-ggplot2', 'r-rcolorbrewer', 'r-vgam']), ('run', ['bioconductor-biostrings', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.20.3', 'bioconductor-iranges >=2.2.1', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.14.1', 'r-base', 'r-data.table', 'r-futile.logger', 'r-ggplot2', 'r-rcolorbrewer', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PureCN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PureCN.html'), ('license', 'Artistic-2.0'), ('summary', 'This package estimates tumor purity, copy number, and loss of heterozygosity (LOH), and classifies single nucleotide variants (SNVs) by somatic status and clonality. PureCN is designed for targeted short read sequencing data, integrates well with standard somatic variant detection and copy number pipelines, and has support for tumor samples without matching normal samples.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-purecn/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-purecn'), ('version', '1.6.3')])), ('source', CommentedMap([('fn', 'PureCN_1.6.3.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/PureCN_1.6.3.tar.gz', 'https://depot.galaxyproject.org/software/PureCN/PureCN_1.6.3_src_all.tar.gz']), ('md5', '9dce6c929693f3d4967e019b83e79fef')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.20.3', 'bioconductor-iranges >=2.2.1', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.14.1', 'r-base', 'r-data.table', 'r-futile.logger', 'r-ggplot2', 'r-rcolorbrewer', 'r-vgam']), ('run', ['bioconductor-biostrings', 'bioconductor-dnacopy', 'bioconductor-edger', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges >=1.20.3', 'bioconductor-iranges >=2.2.1', 'bioconductor-limma', 'bioconductor-rsamtools', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.14.1', 'r-base', 'r-data.table', 'r-futile.logger', 'r-ggplot2', 'r-rcolorbrewer', 'r-vgam'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'PureCN\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/PureCN.html'), ('license', 'Artistic-2.0'), ('summary', 'This package estimates tumor purity, copy number, and loss of heterozygosity (LOH), and classifies single nucleotide variants (SNVs) by somatic status and clonality. PureCN is designed for targeted short read sequencing data, integrates well with standard somatic variant detection and copy number pipelines, and has support for tumor samples without matching normal samples.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnaz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rnaz'), ('version', '2.1')])), ('source', CommentedMap([('fn', 'RNAz-2.1.tar.gz'), ('url', 'https://www.tbi.univie.ac.at/~wash/RNAz/RNAz-2.1.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('commands', ['RNAz -h'])])), ('about', CommentedMap([('home', 'https://www.tbi.univie.ac.at/~wash/RNAz/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'predicting structural noncoding RNAs')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnaz/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'rnaz'), ('version', '2.1')])), ('source', CommentedMap([('fn', 'RNAz-2.1.tar.gz'), ('url', 'https://www.tbi.univie.ac.at/~wash/RNAz/RNAz-2.1.tar.gz')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc', 'perl-threaded'])])), ('test', CommentedMap([('commands', ['RNAz -h'])])), ('about', CommentedMap([('home', 'https://www.tbi.univie.ac.at/~wash/RNAz/'), ('license', 'MIT-like'), ('license_file', 'COPYING'), ('summary', 'predicting structural noncoding RNAs')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/swalign/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'swalign'), ('version', '0.3.3')])), ('source', CommentedMap([('fn', 'swalign-0.3.3.tar.gz'), ('url', 'https://pypi.python.org/packages/72/97/cb26a99ee2e1796b0ab28a955e3568607826a44999654f6880d4e00174ac/swalign-0.3.3.tar.gz'), ('md5', '64c55d09775d4875bc419a08a43dc6fa')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['swalign'])])), ('about', CommentedMap([('home', 'http://github.com/mbreese/swalign/'), ('license', 'BSD'), ('license_file', '<{ RECIPE_DIR }}/LICENSE'), ('summary', 'Smith-Waterman local aligner')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/swalign/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'swalign'), ('version', '0.3.3')])), ('source', CommentedMap([('fn', 'swalign-0.3.3.tar.gz'), ('url', 'https://pypi.python.org/packages/72/97/cb26a99ee2e1796b0ab28a955e3568607826a44999654f6880d4e00174ac/swalign-0.3.3.tar.gz'), ('md5', '64c55d09775d4875bc419a08a43dc6fa')])), ('build', CommentedMap([('skip', True)])), ('requirements', CommentedMap([('build', ['python']), ('run', ['python'])])), ('test', CommentedMap([('imports', ['swalign'])])), ('about', CommentedMap([('home', 'http://github.com/mbreese/swalign/'), ('license', 'BSD'), ('license_file', '<{ RECIPE_DIR }}/LICENSE'), ('summary', 'Smith-Waterman local aligner')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rsubread/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rsubread'), ('version', '1.25.2')])), ('source', CommentedMap([('fn', 'Rsubread_1.25.2.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Rsubread_1.25.2.tar.gz'), ('md5', 'b883dd93fcf007438c1780a0b2a46fb9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rsubread\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Rsubread.html'), ('license', 'GPL-3'), ('summary', 'Provides powerful and easy-to-use tools for analyzing next-gen sequencing read data. Includes quality assessment of sequence reads, read alignment, read summarization, exon-exon junction detection, fusion detection, detection of short and long indels, absolute expression calling and SNP calling. Can be used with reads generated from any of the major sequencing platforms including Illumina GA/HiSeq/MiSeq, Roche GS-FLX, ABI SOLiD and LifeTech Ion PGM/Proton sequencers.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-rsubread/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-rsubread'), ('version', '1.25.2')])), ('source', CommentedMap([('fn', 'Rsubread_1.25.2.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/Rsubread_1.25.2.tar.gz'), ('md5', 'b883dd93fcf007438c1780a0b2a46fb9')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base']), ('run', ['r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'Rsubread\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/Rsubread.html'), ('license', 'GPL-3'), ('summary', 'Provides powerful and easy-to-use tools for analyzing next-gen sequencing read data. Includes quality assessment of sequence reads, read alignment, read summarization, exon-exon junction detection, fusion detection, detection of short and long indels, absolute expression calling and SNP calling. Can be used with reads generated from any of the major sequencing platforms including Illumina GA/HiSeq/MiSeq, Roche GS-FLX, ABI SOLiD and LifeTech Ion PGM/Proton sequencers.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mafft/7.310/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://mafft.cbrc.jp/alignment/software/'), ('license', 'BSD'), ('summary', 'Multiple alignment program for amino acid or nucleotide sequences based on fast Fourier transform')])), ('package', CommentedMap([('name', 'mafft'), ('version', '7.310')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'mafft-7.310-without-extensions-src.tgz'), ('md5', 'fcb68a47d4506b93bb8463ff29b3cbc4'), ('url', 'http://mafft.cbrc.jp/alignment/software/mafft-7.310-without-extensions-src.tgz'), ('patches', ['osx-makefile.patch', 'linux-makefile.patch'])])), ('test', CommentedMap([('commands', ['mafft --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mafft/7.310/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://mafft.cbrc.jp/alignment/software/'), ('license', 'BSD'), ('summary', 'Multiple alignment program for amino acid or nucleotide sequences based on fast Fourier transform')])), ('package', CommentedMap([('name', 'mafft'), ('version', '7.310')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('source', CommentedMap([('fn', 'mafft-7.310-without-extensions-src.tgz'), ('md5', 'fcb68a47d4506b93bb8463ff29b3cbc4'), ('url', 'http://mafft.cbrc.jp/alignment/software/mafft-7.310-without-extensions-src.tgz'), ('patches', ['osx-makefile.patch', 'linux-makefile.patch'])])), ('test', CommentedMap([('commands', ['mafft --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kat/2.3.4/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Release-<{version}}.tar.gz'), ('url', 'https://github.com/TGAC/KAT/archive/Release-<{version}}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('about', CommentedMap([('home', 'https://github.com/TGAC/KAT'), ('license', 'GPL3'), ('license_file', 'COPYING'), ('summary', 'KAT is a suite of tools that analyse jellyfish hashes or sequence files (fasta or fastq) using kmer counts')])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['llvm', 'libgcc', 'python', 'zlib', 'autoconf', 'automake', 'libtool', 'boost <{CONDA_BOOST}}*', 'gnuplot', 'jellyfish', 'perl', 'matplotlib', 'scipy', 'numpy']), ('run', ['python', 'libgcc', 'zlib', 'boost <{CONDA_BOOST}}*', 'gnuplot', 'jellyfish', 'matplotlib', 'scipy', 'numpy'])])), ('test', CommentedMap([('commands', ['kat --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/kat/2.3.4/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'Release-<{version}}.tar.gz'), ('url', 'https://github.com/TGAC/KAT/archive/Release-<{version}}.tar.gz'), ('sha256', '<{ sha256 }}')])), ('about', CommentedMap([('home', 'https://github.com/TGAC/KAT'), ('license', 'GPL3'), ('license_file', 'COPYING'), ('summary', 'KAT is a suite of tools that analyse jellyfish hashes or sequence files (fasta or fastq) using kmer counts')])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['llvm', 'libgcc', 'python', 'zlib', 'autoconf', 'automake', 'libtool', 'boost <{CONDA_BOOST}}*', 'gnuplot', 'jellyfish', 'perl', 'matplotlib', 'scipy', 'numpy']), ('run', ['python', 'libgcc', 'zlib', 'boost <{CONDA_BOOST}}*', 'gnuplot', 'jellyfish', 'matplotlib', 'scipy', 'numpy'])])), ('test', CommentedMap([('commands', ['kat --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-exomedepth/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-exomedepth'), ('version', '1.1.10')])), ('source', CommentedMap([('fn', 'ExomeDepth_1.1.10.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/ExomeDepth_1.1.10.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/ExomeDepth/ExomeDepth_1.1.10.tar.gz']), ('md5', '2354132d3a08d227474cd54efc5f7ae4')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biostrings', 'bioconductor-genomicalignments', 'bioconductor-genomicranges >=1.23.0', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-vgam >=0.8.4', 'r-aod', 'gcc', 'llvm']), ('run', ['r-base', 'bioconductor-biostrings', 'bioconductor-genomicalignments', 'bioconductor-genomicranges >=1.23.0', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-vgam >=0.8.4', 'r-aod', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ExomeDepth\\')\"'])])), ('about', CommentedMap([('home', 'https://cran.r-project.org/web/packages/ExomeDepth/index.html'), ('license', 'GPL-3'), ('summary', 'Calls copy number variants (CNVs) from targeted sequence data, typically exome sequencing experiments designed to identify the genetic basis of Mendelian disorders.'), ('license_family', 'GPL3')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-exomedepth/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-exomedepth'), ('version', '1.1.10')])), ('source', CommentedMap([('fn', 'ExomeDepth_1.1.10.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/ExomeDepth_1.1.10.tar.gz', 'http://cran.r-project.org/src/contrib/Archive/ExomeDepth/ExomeDepth_1.1.10.tar.gz']), ('md5', '2354132d3a08d227474cd54efc5f7ae4')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'bioconductor-biostrings', 'bioconductor-genomicalignments', 'bioconductor-genomicranges >=1.23.0', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-vgam >=0.8.4', 'r-aod', 'gcc', 'llvm']), ('run', ['r-base', 'bioconductor-biostrings', 'bioconductor-genomicalignments', 'bioconductor-genomicranges >=1.23.0', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-vgam >=0.8.4', 'r-aod', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ExomeDepth\\')\"'])])), ('about', CommentedMap([('home', 'https://cran.r-project.org/web/packages/ExomeDepth/index.html'), ('license', 'GPL-3'), ('summary', 'Calls copy number variants (CNVs) from targeted sequence data, typically exome sequencing experiments designed to identify the genetic basis of Mendelian disorders.'), ('license_family', 'GPL3')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pipmir/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pipmir'), ('version', 1.1)])), ('source', CommentedMap([('fn', 'PIPmiR_v1_1b_executable.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/duke/PIPmiR/PIPmiR_v1_1b_executable.tar.gz'), ('md5', '4103f0a148e4cc0217c616c1606f11c5')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['java-jdk', 'samtools', 'viennarna 1.8.5']), ('run', ['java-jdk', 'samtools', 'viennarna 1.8.5'])])), ('test', CommentedMap([('commands', ['PIPmiR'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/Pipeline_for_the_Identification_of_Plant_miRNAs_84/'), ('license', 'PIPmiR is provided for academic use only, if you wish to use it in another setting please contact Uwe Ohler.'), ('summary', 'We developed the PIPmiR algorithm to identify novel plant miRNA genes from a combination of deep sequencing data and genomic features.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pipmir/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pipmir'), ('version', 1.1)])), ('source', CommentedMap([('fn', 'PIPmiR_v1_1b_executable.tar.gz'), ('url', 'https://ohlerlab.mdc-berlin.de/files/duke/PIPmiR/PIPmiR_v1_1b_executable.tar.gz'), ('md5', '4103f0a148e4cc0217c616c1606f11c5')])), ('build', CommentedMap([('skip', True), ('number', 1)])), ('requirements', CommentedMap([('build', ['java-jdk', 'samtools', 'viennarna 1.8.5']), ('run', ['java-jdk', 'samtools', 'viennarna 1.8.5'])])), ('test', CommentedMap([('commands', ['PIPmiR'])])), ('about', CommentedMap([('home', 'https://ohlerlab.mdc-berlin.de/software/Pipeline_for_the_Identification_of_Plant_miRNAs_84/'), ('license', 'PIPmiR is provided for academic use only, if you wish to use it in another setting please contact Uwe Ohler.'), ('summary', 'We developed the PIPmiR algorithm to identify novel plant miRNA genes from a combination of deep sequencing data and genomic features.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/glimmer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'glimmer'), ('version', 3.02)])), ('about', CommentedMap([('home', 'https://ccb.jhu.edu/software/glimmer/index.shtml'), ('license', 'Custom'), ('license_file', 'LICENSE'), ('summary', 'Glimmer is a system for finding genes in microbial DNA')])), ('source', CommentedMap([('fn', 'glimmer302b.tar.gz'), ('url', 'https://ccb.jhu.edu/software/glimmer/glimmer302b.tar.gz'), ('md5', '344d012ae12596de905866fe9eb7f16c')])), ('build', CommentedMap([('number', 2)])), ('test', CommentedMap([('commands', ['glimmer3 | cat'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/glimmer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'glimmer'), ('version', 3.02)])), ('about', CommentedMap([('home', 'https://ccb.jhu.edu/software/glimmer/index.shtml'), ('license', 'Custom'), ('license_file', 'LICENSE'), ('summary', 'Glimmer is a system for finding genes in microbial DNA')])), ('source', CommentedMap([('fn', 'glimmer302b.tar.gz'), ('url', 'https://ccb.jhu.edu/software/glimmer/glimmer302b.tar.gz'), ('md5', '344d012ae12596de905866fe9eb7f16c')])), ('build', CommentedMap([('number', 2)])), ('test', CommentedMap([('commands', ['glimmer3 | cat'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-netpathminer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-netpathminer'), ('version', '1.12.1')])), ('source', CommentedMap([('fn', 'NetPathMiner_1.12.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/NetPathMiner_1.12.1.tar.gz', 'https://depot.galaxyproject.org/software/NetPathMiner/NetPathMiner_1.12.1_src_all.tar.gz']), ('md5', 'b4b7d04540ab6f03cba5dc3fa141e9cf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph >=1.0']), ('run', ['r-base', 'r-igraph >=1.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NetPathMiner\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/NetPathMiner.html'), ('license', 'GPL (>= 2)'), ('summary', 'NetPathMiner is a general framework for network path mining using genome-scale networks. It constructs networks from KGML, SBML and BioPAX files, providing three network representations, metabolic, reaction and gene representations. NetPathMiner finds active paths and applies machine learning methods to summarize found paths for easy interpretation. It also provides static and interactive visualizations of networks and paths to aid manual investigation.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-netpathminer/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-netpathminer'), ('version', '1.12.1')])), ('source', CommentedMap([('fn', 'NetPathMiner_1.12.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/NetPathMiner_1.12.1.tar.gz', 'https://depot.galaxyproject.org/software/NetPathMiner/NetPathMiner_1.12.1_src_all.tar.gz']), ('md5', 'b4b7d04540ab6f03cba5dc3fa141e9cf')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-igraph >=1.0']), ('run', ['r-base', 'r-igraph >=1.0'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NetPathMiner\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/NetPathMiner.html'), ('license', 'GPL (>= 2)'), ('summary', 'NetPathMiner is a general framework for network path mining using genome-scale networks. It constructs networks from KGML, SBML and BioPAX files, providing three network representations, metabolic, reaction and gene representations. NetPathMiner finds active paths and applies machine learning methods to summarize found paths for easy interpretation. It also provides static and interactive visualizations of networks and paths to aid manual investigation.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/jellyfish/2.2.3/meta.yaml', CommentedMap([('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'jellyfish'), ('version', '2.2.3')])), ('source', CommentedMap([('fn', 'v2.2.3.tar.gz'), ('url', 'https://github.com/gmarcais/Jellyfish/archive/v2.2.3.tar.gz'), ('md5', 'a63398ce1cbde0eddcb3493f5b84a9b7')])), ('requirements', CommentedMap([('build', ['gcc', 'autoconf', 'automake', 'libtool', 'pkg-config', 'yaggo >=1.5.8']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['jellyfish --version 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.genome.umd.edu/jellyfish.html'), ('license', 'GPLv3'), ('summary', 'Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA. A k-mer is a substring of length k, and counting the occurrences of all such substrings is a central step in many analyses of DNA sequence')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/jellyfish/2.2.3/meta.yaml', CommentedMap([('build', CommentedMap([('number', 1)])), ('package', CommentedMap([('name', 'jellyfish'), ('version', '2.2.3')])), ('source', CommentedMap([('fn', 'v2.2.3.tar.gz'), ('url', 'https://github.com/gmarcais/Jellyfish/archive/v2.2.3.tar.gz'), ('md5', 'a63398ce1cbde0eddcb3493f5b84a9b7')])), ('requirements', CommentedMap([('build', ['gcc', 'autoconf', 'automake', 'libtool', 'pkg-config', 'yaggo >=1.5.8']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['jellyfish --version 2>&1 > /dev/null'])])), ('about', CommentedMap([('home', 'http://www.genome.umd.edu/jellyfish.html'), ('license', 'GPLv3'), ('summary', 'Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA. A k-mer is a substring of length k, and counting the occurrences of all such substrings is a central step in many analyses of DNA sequence')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqbuster/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/lpantano/seqbuster'), ('license', 'MIT'), ('summary', 'miRNA and isomiR annotation')])), ('package', CommentedMap([('name', 'seqbuster'), ('version', '3.1')])), ('source', CommentedMap([('fn', 'seqbuster-3.1.tar.gz'), ('url', 'https://github.com/lpantano/seqbuster/archive/miraligner-3.1.tar.gz'), ('md5', '3b4d23fa9839910ac980c3fb2822fbfa')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('run', ['openjdk >=8'])])), ('test', CommentedMap([('commands', ['miraligner -h'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/seqbuster/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/lpantano/seqbuster'), ('license', 'MIT'), ('summary', 'miRNA and isomiR annotation')])), ('package', CommentedMap([('name', 'seqbuster'), ('version', '3.1')])), ('source', CommentedMap([('fn', 'seqbuster-3.1.tar.gz'), ('url', 'https://github.com/lpantano/seqbuster/archive/miraligner-3.1.tar.gz'), ('md5', '3b4d23fa9839910ac980c3fb2822fbfa')])), ('build', CommentedMap([('number', 2), ('skip', True)])), ('requirements', CommentedMap([('run', ['openjdk >=8'])])), ('test', CommentedMap([('commands', ['miraligner -h'])]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-savr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-savr'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'savR_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/savR_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/savR_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/savR/savR_1.14.0_src_all.tar.gz']), ('md5', '8b2eb7908fd014ae3ac9f1d4ac74fbf7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-reshape2', 'r-scales', 'r-xml']), ('run', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-reshape2', 'r-scales', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'savR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/savR.html'), ('license', 'AGPL-3'), ('summary', 'Parse Illumina Sequence Analysis Viewer (SAV) files, access data, and generate QC plots.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-savr/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-savr'), ('version', '1.14.0')])), ('source', CommentedMap([('fn', 'savR_1.14.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/savR_1.14.0.tar.gz', 'https://bioarchive.galaxyproject.org/savR_1.14.0.tar.gz', 'https://depot.galaxyproject.org/software/savR/savR_1.14.0_src_all.tar.gz']), ('md5', '8b2eb7908fd014ae3ac9f1d4ac74fbf7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-reshape2', 'r-scales', 'r-xml']), ('run', ['r-base', 'r-ggplot2', 'r-gridextra', 'r-reshape2', 'r-scales', 'r-xml'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'savR\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/savR.html'), ('license', 'AGPL-3'), ('summary', 'Parse Illumina Sequence Analysis Viewer (SAV) files, access data, and generate QC plots.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nem'), ('version', '2.50.0')])), ('source', CommentedMap([('fn', 'nem_2.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nem_2.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/nem_2.50.0.tar.gz', 'https://depot.galaxyproject.org/software/nem/nem_2.50.0_src_all.tar.gz']), ('md5', '67c37dcdaa983aedffac020cfa646632')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-limma', 'bioconductor-rbgl >=1.8.1', 'bioconductor-rgraphviz', 'r-base', 'r-boot', 'r-e1071', 'r-plotrix', 'r-rcolorbrewer', 'r-statmod']), ('run', ['bioconductor-graph', 'bioconductor-limma', 'bioconductor-rbgl >=1.8.1', 'bioconductor-rgraphviz', 'r-base', 'r-boot', 'r-e1071', 'r-plotrix', 'r-rcolorbrewer', 'r-statmod'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nem\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nem.html'), ('license', 'GPL (>= 2)'), ('summary', \"The package 'nem' allows to reconstruct features of pathways from the nested structure of perturbation effects. It takes as input (1.) a set of pathway components, which were perturbed, and (2.) phenotypic readout of these perturbations (e.g. gene expression, protein expression). The output is a directed graph representing the phenotypic hierarchy.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-nem/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-nem'), ('version', '2.50.0')])), ('source', CommentedMap([('fn', 'nem_2.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/nem_2.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/nem_2.50.0.tar.gz', 'https://depot.galaxyproject.org/software/nem/nem_2.50.0_src_all.tar.gz']), ('md5', '67c37dcdaa983aedffac020cfa646632')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-graph', 'bioconductor-limma', 'bioconductor-rbgl >=1.8.1', 'bioconductor-rgraphviz', 'r-base', 'r-boot', 'r-e1071', 'r-plotrix', 'r-rcolorbrewer', 'r-statmod']), ('run', ['bioconductor-graph', 'bioconductor-limma', 'bioconductor-rbgl >=1.8.1', 'bioconductor-rgraphviz', 'r-base', 'r-boot', 'r-e1071', 'r-plotrix', 'r-rcolorbrewer', 'r-statmod'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'nem\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/nem.html'), ('license', 'GPL (>= 2)'), ('summary', \"The package 'nem' allows to reconstruct features of pathways from the nested structure of perturbation effects. It takes as input (1.) a set of pathway components, which were perturbed, and (2.) phenotypic readout of these perturbations (e.g. gene expression, protein expression). The output is a directed graph representing the phenotypic hierarchy.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylpipe/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylpipe'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'methylPipe_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/methylPipe_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/methylPipe_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/methylPipe/methylPipe_1.10.0_src_all.tar.gz']), ('md5', 'eb95d03c3d651b420442809d475031e8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-marray', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-data.table', 'r-gplots']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-marray', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-data.table', 'r-gplots'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'methylPipe\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/methylPipe.html'), ('license', 'GPL(>=2)'), ('summary', 'Memory efficient analysis of base resolution DNA methylation data in both the CpG and non-CpG sequence context. Integration of DNA methylation data derived from any methodology providing base- or low-resolution data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methylpipe/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methylpipe'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'methylPipe_1.10.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/methylPipe_1.10.0.tar.gz', 'https://bioarchive.galaxyproject.org/methylPipe_1.10.0.tar.gz', 'https://depot.galaxyproject.org/software/methylPipe/methylPipe_1.10.0_src_all.tar.gz']), ('md5', 'eb95d03c3d651b420442809d475031e8')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-marray', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-data.table', 'r-gplots']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-gviz', 'bioconductor-iranges', 'bioconductor-marray', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment >=0.2.0', 'r-base', 'r-data.table', 'r-gplots'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'methylPipe\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/methylPipe.html'), ('license', 'GPL(>=2)'), ('summary', 'Memory efficient analysis of base resolution DNA methylation data in both the CpG and non-CpG sequence context. Integration of DNA methylation data derived from any methodology providing base- or low-resolution data.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stacks/1.44/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'stacks'), ('version', '1.44')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'stacks-1.44.tar.gz'), ('md5', '10e1c2cb3ed10bebcb035619d94ae7d3'), ('url', 'http://catchenlab.life.illinois.edu/stacks/source/stacks-1.44.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'sparsehash']), ('run', ['perl', 'velvet', 'libgcc', 'zlib', 'sparsehash'])])), ('test', CommentedMap([('commands', ['command -v cstacks', 'command -v pstacks', 'command -v process_radtags', 'command -v clone_filter'])])), ('about', CommentedMap([('home', 'http://catchenlab.life.illinois.edu/stacks/'), ('license', 'GPL'), ('summary', 'Stacks is a software pipeline for building loci from RAD-seq')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/stacks/1.44/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'stacks'), ('version', '1.44')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('source', CommentedMap([('fn', 'stacks-1.44.tar.gz'), ('md5', '10e1c2cb3ed10bebcb035619d94ae7d3'), ('url', 'http://catchenlab.life.illinois.edu/stacks/source/stacks-1.44.tar.gz')])), ('requirements', CommentedMap([('build', ['gcc', 'zlib', 'sparsehash']), ('run', ['perl', 'velvet', 'libgcc', 'zlib', 'sparsehash'])])), ('test', CommentedMap([('commands', ['command -v cstacks', 'command -v pstacks', 'command -v process_radtags', 'command -v clone_filter'])])), ('about', CommentedMap([('home', 'http://catchenlab.life.illinois.edu/stacks/'), ('license', 'GPL'), ('summary', 'Stacks is a software pipeline for building loci from RAD-seq')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affy'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'affy_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/affy_1.50.0.tar.gz'), ('md5', 'aed20ee9653de3ef15868b7e6fd901de')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affyio >=1.13.3', 'bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics >=0.1.12', 'bioconductor-biocinstaller', 'bioconductor-preprocesscore', 'bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-affyio >=1.13.3', 'bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics >=0.1.12', 'bioconductor-biocinstaller', 'bioconductor-preprocesscore', 'bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/affy.html'), ('license', 'LGPL (>= 2.0)'), ('summary', \"The package contains functions for exploratory oligonucleotide array analysis. The dependence on tkWidgets only concerns few convenience functions. 'affy' is fully functional without it.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-affy/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-affy'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'affy_1.50.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/affy_1.50.0.tar.gz'), ('md5', 'aed20ee9653de3ef15868b7e6fd901de')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-affyio >=1.13.3', 'bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics >=0.1.12', 'bioconductor-biocinstaller', 'bioconductor-preprocesscore', 'bioconductor-zlibbioc', 'r-base']), ('run', ['bioconductor-affyio >=1.13.3', 'bioconductor-biobase >=2.5.5', 'bioconductor-biocgenerics >=0.1.12', 'bioconductor-biocinstaller', 'bioconductor-preprocesscore', 'bioconductor-zlibbioc', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'affy\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/affy.html'), ('license', 'LGPL (>= 2.0)'), ('summary', \"The package contains functions for exploratory oligonucleotide array analysis. The dependence on tkWidgets only concerns few convenience functions. 'affy' is fully functional without it.\")]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/barrnap/0.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('url', 'https://github.com/tseemann/<{ name }}/archive/<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', []), ('run', ['perl >=5.22.0', 'hmmer >=3.1b'])])), ('test', CommentedMap([('source_files', ['examples/small.fna']), ('commands', ['(barrnap --help 2>&1; true) | grep -q Options', '(barrnap $PREFIX/lib/barrnap/examples/small.fna 2>&1; true) | grep -q \"Found 3 ribosomal RNA features\"'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/barrnap'), ('license', 'GPLv3'), ('summary', 'Find 5S, 16S and 23S ribosomal RNA genes in bacterial contigs')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/barrnap/0.8/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}_v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}'), ('url', 'https://github.com/tseemann/<{ name }}/archive/<{ version }}.tar.gz')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', []), ('run', ['perl >=5.22.0', 'hmmer >=3.1b'])])), ('test', CommentedMap([('source_files', ['examples/small.fna']), ('commands', ['(barrnap --help 2>&1; true) | grep -q Options', '(barrnap $PREFIX/lib/barrnap/examples/small.fna 2>&1; true) | grep -q \"Found 3 ribosomal RNA features\"'])])), ('about', CommentedMap([('home', 'https://github.com/tseemann/barrnap'), ('license', 'GPLv3'), ('summary', 'Find 5S, 16S and 23S ribosomal RNA genes in bacterial contigs')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/augustus/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://bioinf.uni-greifswald.de/augustus/binaries/old/augustus-3.2.3.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['patches/bam2hints.Makefile.patch', 'patches/filterBam.src.Makefile.patch', 'patches/filterBam.Makefile.patch', 'patches/compileSpliceCands.Makefile.patch'])])), ('build', CommentedMap([('number', 2), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib <{CONDA_ZLIB}}*', 'libgcc', 'bamtools', 'gsl', 'lp_solve', 'perl', 'perl-module-build', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*', 'bamtools', 'boost <{CONDA_BOOST}}*', 'gsl', 'lp_solve', 'perl', 'perl-app-cpanminus', 'perl-module-build', 'perl-yaml', 'perl-dbi', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('commands', ['augustus --version', 'autoAug.pl --help &>/dev/null', 'bedgraph2wig.pl --help &>/dev/null', 'blat2gbrowse.pl --help 2>&1 | grep \"convert blat file to gbrowse file\" &>/dev/null', 'blat2hints.pl --help', 'cegma2gff.pl --help 2>&1 | grep \"Could not open CEGMA file --help\"', 'checkParamArchive.pl --help 2>&1 | grep \"checkParamArchive.pl param-archive.tar.gz projectDir > utrPossible 2> errors\"', 'cleanDOSfasta.pl --help 2>&1 | grep \"Could not open file --help\"', 'clusterAndSplitGenes.pl --help', 'createAugustusJoblist.pl --help', 'del_from_prfl.pl --help 2>&1 | grep \"Need two arguments\"', 'evalCGP.pl --help', 'exonerate2hints.pl --help', 'echo \"hello\" | exoniphyDb2hints.pl', 'echo \"hello\" | extractTranscriptEnds.pl | grep \">seq1\"', 'filterGenesIn_mRNAname.pl --help', 'filterGenesOut_mRNAname.pl --help', 'filterGenes.pl --help 2>&1 | grep \"usage:filterGenes.pl namefile dbfile\"', 'filterInFrameStopCodons.pl --help 2>&1 | grep \"Could not open protein file\"', 'filterMaf.pl --help 2>&1 | grep \"filter blocks from a MAF alignment that intersect with a given genomic interval\"', 'echo \"hello\" | filter-ppx.pl | grep \"hello\"', 'filterPSL.pl --help', 'filterShrimp.pl --help', 'filterSpliceHints.pl --help', 'gbrowseold2gff3.pl --help 2>&1 | grep \"gbrowseold2gff3.pl format convert a GFF file\"', 'gbSmallDNA2gff.pl --help 2>&1 | grep \"Could not open genbank input file --help\"', 'getAnnoFasta.pl --help 2>&1 | grep \"Makes a fasta file with protein sequences (augustus.aa)\"', 'gff2gbSmallDNA.pl --help 2>&1 | grep \"convert GFF file and sequence fasta file to minimal genbank format\"', 'gffGetmRNA.pl --help', 'gp2othergp.pl --help 2>&1 | grep \"t open --help\"', 'gtf2gff.pl --help 2>&1 | grep \"gtf2gff.pl format convert a GTF file\"', 'hal2maf_split.pl --help 2>&1 | grep \"this script works on top of the \"', 'intron2exex.pl --help | grep \"intron2exex.pl make a fasta file with exon-exon sequences\"', 'join_mult_hints.pl --help', 'joinPeptides.pl --help 2>&1 | grep \"joinPeptides.pl in1.fa in2.fa > out.fa\"', 'maf2conswig.pl --help | grep \"maf2conswig.pl convert maf alignment format\"', 'makeMatchLists.pl --help  2>&1 | grep \"t open --help\"', 'makeUtrTrainingSet.pl --help 2>&1 | grep \"./makeUtrTrainingSet.pl -- make a genbank or gff training file with \"', 'maskNregions.pl --help 2>&1 | grep \"could not open file --help\"', 'moveParameters.pl --help', 'msa2prfl.pl --help 2>&1 | grep \"Converts a Multiple Sequence Alignment\"', 'new_species.pl --help', 'optimize_augustus.pl --help', 'partition_gtf2gb.pl --help 2>&1 | grep \"Partition a gtf file into gene clusters\"', 'pasapolyA2hints.pl --help 2>&1 | grep \"PASA-Output file in fasta-format\"', 'peptides2alternatives.pl --help 2>&1 | grep \"peptides2alternatives.pl peptides.aa augustus-genes.aa\"', 'peptides2hints.pl --help 2>&1 | grep \"peptides2hints.pl psl-file gff-file fa-file src pri\"', 'polyA2hints.pl --help 2>&1 | grep \"transcript end at position p gives rise to a tts hint\"', 'randomSplit.pl --help', 'ratewig.pl --help 2>&1 | grep \"ratewig /path/to/refseqfile/ /path/to/wigfile\"', 'rmRedundantHints.pl --help 2>&1 | grep \"rmRedundantHints.pl list1.hints list2.hints > nonredundant.hints\"', 'runAllSim4.pl --help', 'samMap.pl --help', 'scipiogff2gff.pl --help 2>&1 | grep \"convert a gff from scipio to a gff file\"', 'simpleFastaHeaders.pl --help 2>&1 | grep \"in.fa prefix out.fa mapping.txt\"', 'simplifyFastaHeaders.pl --help 2>&1 | grep \"in.fa nameStem out.fa header.map\"', 'splitMfasta.pl --help 2>&1 | grep \"Unknown option\"', 'split_wiggle.pl --help', 'summarizeACGTcontent.pl --help 2>&1 | grep \"Couldn\"', 'transMap2hints.pl --help 2>&1 | grep \"convert transmap alignments to hints file\"', 'uniquePeptides.pl --help 2>&1 | grep \"uniquePeptides.pl in.fa prefix > out.fa\"', 'utrgff2gbrowse.pl --help', 'wigchoose.pl --help', 'writeResultsPage.pl --help 2>&1 | grep \"writeResultsPage ID species-name db-file grails-out www-out AugustusConfigPath AugustusScriptsPath final-flag\"', 'yaml2gff.1.4.pl --help 2>&1 | grep \"< scipio.yaml > scipio.gff\"'])])), ('about', CommentedMap([('home', 'http://bioinf.uni-greifswald.de/augustus/'), ('license', 'MIT'), ('license_family', 'MIT'), ('summary', 'AUGUSTUS is a gene prediction program for eukaryotes written by Mario Stanke and Oliver Keller. It can be used as an ab initio program, which means it bases its prediction purely on the sequence. AUGUSTUS may also incorporate hints on the gene structure coming from extrinsic sources such as EST, MS/MS, protein alignments and synthenic genomic alignments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/augustus/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', '<{ name }}-<{ version }}.tar.gz'), ('url', 'http://bioinf.uni-greifswald.de/augustus/binaries/old/augustus-3.2.3.tar.gz'), ('sha256', '<{ sha256 }}'), ('patches', ['patches/bam2hints.Makefile.patch', 'patches/filterBam.src.Makefile.patch', 'patches/filterBam.Makefile.patch', 'patches/compileSpliceCands.Makefile.patch'])])), ('build', CommentedMap([('number', 2), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}'), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib <{CONDA_ZLIB}}*', 'libgcc', 'bamtools', 'gsl', 'lp_solve', 'perl', 'perl-module-build', 'boost <{CONDA_BOOST}}*']), ('run', ['libgcc', 'zlib <{CONDA_ZLIB}}*', 'bamtools', 'boost <{CONDA_BOOST}}*', 'gsl', 'lp_solve', 'perl', 'perl-app-cpanminus', 'perl-module-build', 'perl-yaml', 'perl-dbi', 'perl-scalar-list-utils'])])), ('test', CommentedMap([('commands', ['augustus --version', 'autoAug.pl --help &>/dev/null', 'bedgraph2wig.pl --help &>/dev/null', 'blat2gbrowse.pl --help 2>&1 | grep \"convert blat file to gbrowse file\" &>/dev/null', 'blat2hints.pl --help', 'cegma2gff.pl --help 2>&1 | grep \"Could not open CEGMA file --help\"', 'checkParamArchive.pl --help 2>&1 | grep \"checkParamArchive.pl param-archive.tar.gz projectDir > utrPossible 2> errors\"', 'cleanDOSfasta.pl --help 2>&1 | grep \"Could not open file --help\"', 'clusterAndSplitGenes.pl --help', 'createAugustusJoblist.pl --help', 'del_from_prfl.pl --help 2>&1 | grep \"Need two arguments\"', 'evalCGP.pl --help', 'exonerate2hints.pl --help', 'echo \"hello\" | exoniphyDb2hints.pl', 'echo \"hello\" | extractTranscriptEnds.pl | grep \">seq1\"', 'filterGenesIn_mRNAname.pl --help', 'filterGenesOut_mRNAname.pl --help', 'filterGenes.pl --help 2>&1 | grep \"usage:filterGenes.pl namefile dbfile\"', 'filterInFrameStopCodons.pl --help 2>&1 | grep \"Could not open protein file\"', 'filterMaf.pl --help 2>&1 | grep \"filter blocks from a MAF alignment that intersect with a given genomic interval\"', 'echo \"hello\" | filter-ppx.pl | grep \"hello\"', 'filterPSL.pl --help', 'filterShrimp.pl --help', 'filterSpliceHints.pl --help', 'gbrowseold2gff3.pl --help 2>&1 | grep \"gbrowseold2gff3.pl format convert a GFF file\"', 'gbSmallDNA2gff.pl --help 2>&1 | grep \"Could not open genbank input file --help\"', 'getAnnoFasta.pl --help 2>&1 | grep \"Makes a fasta file with protein sequences (augustus.aa)\"', 'gff2gbSmallDNA.pl --help 2>&1 | grep \"convert GFF file and sequence fasta file to minimal genbank format\"', 'gffGetmRNA.pl --help', 'gp2othergp.pl --help 2>&1 | grep \"t open --help\"', 'gtf2gff.pl --help 2>&1 | grep \"gtf2gff.pl format convert a GTF file\"', 'hal2maf_split.pl --help 2>&1 | grep \"this script works on top of the \"', 'intron2exex.pl --help | grep \"intron2exex.pl make a fasta file with exon-exon sequences\"', 'join_mult_hints.pl --help', 'joinPeptides.pl --help 2>&1 | grep \"joinPeptides.pl in1.fa in2.fa > out.fa\"', 'maf2conswig.pl --help | grep \"maf2conswig.pl convert maf alignment format\"', 'makeMatchLists.pl --help  2>&1 | grep \"t open --help\"', 'makeUtrTrainingSet.pl --help 2>&1 | grep \"./makeUtrTrainingSet.pl -- make a genbank or gff training file with \"', 'maskNregions.pl --help 2>&1 | grep \"could not open file --help\"', 'moveParameters.pl --help', 'msa2prfl.pl --help 2>&1 | grep \"Converts a Multiple Sequence Alignment\"', 'new_species.pl --help', 'optimize_augustus.pl --help', 'partition_gtf2gb.pl --help 2>&1 | grep \"Partition a gtf file into gene clusters\"', 'pasapolyA2hints.pl --help 2>&1 | grep \"PASA-Output file in fasta-format\"', 'peptides2alternatives.pl --help 2>&1 | grep \"peptides2alternatives.pl peptides.aa augustus-genes.aa\"', 'peptides2hints.pl --help 2>&1 | grep \"peptides2hints.pl psl-file gff-file fa-file src pri\"', 'polyA2hints.pl --help 2>&1 | grep \"transcript end at position p gives rise to a tts hint\"', 'randomSplit.pl --help', 'ratewig.pl --help 2>&1 | grep \"ratewig /path/to/refseqfile/ /path/to/wigfile\"', 'rmRedundantHints.pl --help 2>&1 | grep \"rmRedundantHints.pl list1.hints list2.hints > nonredundant.hints\"', 'runAllSim4.pl --help', 'samMap.pl --help', 'scipiogff2gff.pl --help 2>&1 | grep \"convert a gff from scipio to a gff file\"', 'simpleFastaHeaders.pl --help 2>&1 | grep \"in.fa prefix out.fa mapping.txt\"', 'simplifyFastaHeaders.pl --help 2>&1 | grep \"in.fa nameStem out.fa header.map\"', 'splitMfasta.pl --help 2>&1 | grep \"Unknown option\"', 'split_wiggle.pl --help', 'summarizeACGTcontent.pl --help 2>&1 | grep \"Couldn\"', 'transMap2hints.pl --help 2>&1 | grep \"convert transmap alignments to hints file\"', 'uniquePeptides.pl --help 2>&1 | grep \"uniquePeptides.pl in.fa prefix > out.fa\"', 'utrgff2gbrowse.pl --help', 'wigchoose.pl --help', 'writeResultsPage.pl --help 2>&1 | grep \"writeResultsPage ID species-name db-file grails-out www-out AugustusConfigPath AugustusScriptsPath final-flag\"', 'yaml2gff.1.4.pl --help 2>&1 | grep \"< scipio.yaml > scipio.gff\"'])])), ('about', CommentedMap([('home', 'http://bioinf.uni-greifswald.de/augustus/'), ('license', 'MIT'), ('license_family', 'MIT'), ('summary', 'AUGUSTUS is a gene prediction program for eukaryotes written by Mario Stanke and Oliver Keller. It can be used as an ab initio program, which means it bases its prediction purely on the sequence. AUGUSTUS may also incorporate hints on the gene structure coming from extrinsic sources such as EST, MS/MS, protein alignments and synthenic genomic alignments.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mosaik/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/wanpinglee/MOSAIK'), ('license', 'GPL 2.0+'), ('summary', 'MOSAIK is a stable, sensitive and open-source program for mapping second and third-generation sequencing reads to a reference genome.')])), ('package', CommentedMap([('name', 'mosaik'), ('version', '2.2.26')])), ('source', CommentedMap([('fn', 'e04c806bb1410cf1dbd1534991c46d696aec6723.zip'), ('md5', '2fd32477444978ec7662df77226cc896'), ('url', 'https://github.com/wanpinglee/MOSAIK/archive/e04c806bb1410cf1dbd1534991c46d696aec6723.zip'), ('patches', ['osx-makefile.patch', 'linux.inc.patch'])])), ('requirements', CommentedMap([('build', ['zlib', 'gcc']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['MosaikAligner --help | grep Usage > /dev/null'])])), ('build', CommentedMap([('number', 2), ('skip', False)]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/mosaik/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://github.com/wanpinglee/MOSAIK'), ('license', 'GPL 2.0+'), ('summary', 'MOSAIK is a stable, sensitive and open-source program for mapping second and third-generation sequencing reads to a reference genome.')])), ('package', CommentedMap([('name', 'mosaik'), ('version', '2.2.26')])), ('source', CommentedMap([('fn', 'e04c806bb1410cf1dbd1534991c46d696aec6723.zip'), ('md5', '2fd32477444978ec7662df77226cc896'), ('url', 'https://github.com/wanpinglee/MOSAIK/archive/e04c806bb1410cf1dbd1534991c46d696aec6723.zip'), ('patches', ['osx-makefile.patch', 'linux.inc.patch'])])), ('requirements', CommentedMap([('build', ['zlib', 'gcc']), ('run', ['zlib', 'libgcc'])])), ('test', CommentedMap([('commands', ['MosaikAligner --help | grep Usage > /dev/null'])])), ('build', CommentedMap([('number', 2), ('skip', False)]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-roar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-roar'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'roar_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/roar_1.10.0.tar.gz'), ('md5', 'b1d65f416358c3398f4334d6671d12c1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments >=0.99.4', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments >=0.99.4', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'roar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/roar.html'), ('license', 'GPL-3'), ('summary', 'Identify preferential usage of APA sites, comparing two biological conditions, starting from known alternative sites and alignments obtained from standard RNA-seq experiments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-roar/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-roar'), ('version', '1.10.0')])), ('source', CommentedMap([('fn', 'roar_1.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/roar_1.10.0.tar.gz'), ('md5', 'b1d65f416358c3398f4334d6671d12c1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments >=0.99.4', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base']), ('run', ['bioconductor-biocgenerics', 'bioconductor-genomeinfodb', 'bioconductor-genomicalignments >=0.99.4', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rtracklayer', 'bioconductor-s4vectors', 'bioconductor-summarizedexperiment', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'roar\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/roar.html'), ('license', 'GPL-3'), ('summary', 'Identify preferential usage of APA sites, comparing two biological conditions, starting from known alternative sites and alignments obtained from standard RNA-seq experiments.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cn.mops/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cn.mops'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'cn.mops_1.18.0.tar.gz'), ('url', 'https://bioconductor.org/packages/3.3/bioc/src/contrib/cn.mops_1.18.0.tar.gz'), ('md5', '3b763aedfb1ea1004141194f917432ef')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'gcc', 'llvm']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cn.mops\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/cn.mops.html'), ('license', 'LGPL (>= 2.0)'), ('summary', 'cn.mops (Copy Number estimation by a Mixture Of PoissonS) is a data processing pipeline for copy number variations and aberrations (CNVs and CNAs) from next generation sequencing (NGS) data. The package supplies functions to convert BAM files into read count matrices or genomic ranges objects, which are the input objects for cn.mops. cn.mops models the depths of coverage across samples at each genomic position. Therefore, it does not suffer from read count biases along chromosomes. Using a Bayesian approach, cn.mops decomposes read variations across samples into integer copy numbers and noise by its mixture components and Poisson distributions, respectively. cn.mops guarantees a low FDR because wrong detections are indicated by high noise and filtered out. cn.mops is very fast and written in C++.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-cn.mops/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-cn.mops'), ('version', '1.18.0')])), ('source', CommentedMap([('fn', 'cn.mops_1.18.0.tar.gz'), ('url', 'https://bioconductor.org/packages/3.3/bioc/src/contrib/cn.mops_1.18.0.tar.gz'), ('md5', '3b763aedfb1ea1004141194f917432ef')])), ('build', CommentedMap([('number', 1), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'gcc', 'llvm']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'r-base', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'cn.mops\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/cn.mops.html'), ('license', 'LGPL (>= 2.0)'), ('summary', 'cn.mops (Copy Number estimation by a Mixture Of PoissonS) is a data processing pipeline for copy number variations and aberrations (CNVs and CNAs) from next generation sequencing (NGS) data. The package supplies functions to convert BAM files into read count matrices or genomic ranges objects, which are the input objects for cn.mops. cn.mops models the depths of coverage across samples at each genomic position. Therefore, it does not suffer from read count biases along chromosomes. Using a Bayesian approach, cn.mops decomposes read variations across samples into integer copy numbers and noise by its mixture components and Poisson distributions, respectively. cn.mops guarantees a low FDR because wrong detections are indicated by high noise and filtered out. cn.mops is very fast and written in C++.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rsem/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://deweylab.github.io/RSEM/'), ('license', 'GPLv3'), ('summary', 'RSEM (RNA-Seq by Expectation-Maximization)')])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/']), ('number', 1), ('skip', False), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'boost <{CONDA_BOOST}}*', 'gcc', 'perl', 'perl-module-build', 'r-base', 'zlib']), ('run', ['bioconductor-biobase', 'bioconductor-ebseq', 'boost <{CONDA_BOOST}}*', 'libgcc', 'perl', 'perl-module-build', 'r-base', 'samtools 1.3', 'zlib'])])), ('test', CommentedMap([('commands', ['rsem-prepare-reference 2>&1 | grep reference_name > /dev/null', 'rsem-for-ebseq-find-DE 2>&1 | grep Usage > /dev/null', \"rsem-bam2wig foo bar foobar 2>&1 | grep 'fail to open file' > /dev/null\", 'which rsem-for-ebseq-calculate-clustering-info'])])), ('source', CommentedMap([('url', 'https://github.com/deweylab/RSEM/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rsem/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'https://deweylab.github.io/RSEM/'), ('license', 'GPLv3'), ('summary', 'RSEM (RNA-Seq by Expectation-Maximization)')])), ('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/']), ('number', 1), ('skip', False), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'boost <{CONDA_BOOST}}*', 'gcc', 'perl', 'perl-module-build', 'r-base', 'zlib']), ('run', ['bioconductor-biobase', 'bioconductor-ebseq', 'boost <{CONDA_BOOST}}*', 'libgcc', 'perl', 'perl-module-build', 'r-base', 'samtools 1.3', 'zlib'])])), ('test', CommentedMap([('commands', ['rsem-prepare-reference 2>&1 | grep reference_name > /dev/null', 'rsem-for-ebseq-find-DE 2>&1 | grep Usage > /dev/null', \"rsem-bam2wig foo bar foobar 2>&1 | grep 'fail to open file' > /dev/null\", 'which rsem-for-ebseq-calculate-clustering-info'])])), ('source', CommentedMap([('url', 'https://github.com/deweylab/RSEM/archive/v<{ version }}.tar.gz'), ('sha256', '<{ sha256 }}')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-minfi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-minfi'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'minfi_1.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/minfi_1.20.0.tar.gz'), ('md5', '1b5c8e55fd8fca426d6bb2c5072d36b3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings', 'bioconductor-bumphunter >=1.1.9', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-geoquery', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-preprocesscore', 'bioconductor-s4vectors', 'bioconductor-siggenes', 'bioconductor-summarizedexperiment >=1.1.6', 'r-base', 'r-beanplot', 'r-data.table', 'r-matrixstats >=0.50.0', 'r-mclust', 'r-nor1mix', 'r-quadprog', 'r-rcolorbrewer', 'r-reshape', 'r-nlme']), ('run', ['bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings', 'bioconductor-bumphunter >=1.1.9', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-geoquery', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-preprocesscore', 'bioconductor-s4vectors', 'bioconductor-siggenes', 'bioconductor-summarizedexperiment >=1.1.6', 'r-base', 'r-beanplot', 'r-data.table', 'r-matrixstats >=0.50.0', 'r-mclust', 'r-nor1mix', 'r-quadprog', 'r-rcolorbrewer', 'r-reshape', 'r-nlme'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'minfi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/minfi.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools to analyze & visualize Illumina Infinium methylation arrays.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-minfi/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-minfi'), ('version', '1.20.0')])), ('source', CommentedMap([('fn', 'minfi_1.20.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/minfi_1.20.0.tar.gz'), ('md5', '1b5c8e55fd8fca426d6bb2c5072d36b3')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings', 'bioconductor-bumphunter >=1.1.9', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-geoquery', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-preprocesscore', 'bioconductor-s4vectors', 'bioconductor-siggenes', 'bioconductor-summarizedexperiment >=1.1.6', 'r-base', 'r-beanplot', 'r-data.table', 'r-matrixstats >=0.50.0', 'r-mclust', 'r-nor1mix', 'r-quadprog', 'r-rcolorbrewer', 'r-reshape', 'r-nlme']), ('run', ['bioconductor-biobase >=2.17.8', 'bioconductor-biocgenerics >=0.15.3', 'bioconductor-biostrings', 'bioconductor-bumphunter >=1.1.9', 'bioconductor-genefilter', 'bioconductor-genomeinfodb', 'bioconductor-genomicranges', 'bioconductor-geoquery', 'bioconductor-illuminaio', 'bioconductor-iranges', 'bioconductor-limma', 'bioconductor-preprocesscore', 'bioconductor-s4vectors', 'bioconductor-siggenes', 'bioconductor-summarizedexperiment >=1.1.6', 'r-base', 'r-beanplot', 'r-data.table', 'r-matrixstats >=0.50.0', 'r-mclust', 'r-nor1mix', 'r-quadprog', 'r-rcolorbrewer', 'r-reshape', 'r-nlme'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'minfi\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/minfi.html'), ('license', 'Artistic-2.0'), ('summary', 'Tools to analyze & visualize Illumina Infinium methylation arrays.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ncdfflow/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ncdfflow'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'ncdfFlow_2.22.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ncdfFlow_2.22.0.tar.gz'), ('md5', 'b5a69c2d1776bb34deaf6e7a0bff3384')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore >=1.37.15', 'r-base', 'r-bh', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-flowviz', 'bioconductor-zlibbioc', 'r-rcpparmadillo', 'hdf5', 'gcc', 'llvm']), ('run', ['libgcc', 'bioconductor-flowcore >=1.37.15', 'r-base', 'r-bh', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-flowviz', 'bioconductor-zlibbioc', 'r-rcpparmadillo', 'hdf5'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ncdfFlow\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ncdfFlow.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides HDF5 storage based methods and functions for manipulation of flow cytometry data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ncdfflow/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ncdfflow'), ('version', '2.22.0')])), ('source', CommentedMap([('fn', 'ncdfFlow_2.22.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/ncdfFlow_2.22.0.tar.gz'), ('md5', 'b5a69c2d1776bb34deaf6e7a0bff3384')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-flowcore >=1.37.15', 'r-base', 'r-bh', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-flowviz', 'bioconductor-zlibbioc', 'r-rcpparmadillo', 'hdf5', 'gcc', 'llvm']), ('run', ['libgcc', 'bioconductor-flowcore >=1.37.15', 'r-base', 'r-bh', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-flowviz', 'bioconductor-zlibbioc', 'r-rcpparmadillo', 'hdf5'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ncdfFlow\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ncdfFlow.html'), ('license', 'Artistic-2.0'), ('summary', 'Provides HDF5 storage based methods and functions for manipulation of flow cytometry data.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/centrifuge/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/infphilo/centrifuge/archive/f39767eb57d8e175029c.tar.gz'), ('md5', '17750d3be1039d2b428fe237a9cd7e45')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'perl', 'python']), ('run', ['libgcc', 'zlib', 'perl', 'python'])])), ('test', CommentedMap([('commands', ['centrifuge --help'])])), ('about', CommentedMap([('home', 'https://github.com/infphilo/centrifuge'), ('license', 'GPL3'), ('summary', 'Classifier for metagenomic sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/centrifuge/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('build', CommentedMap([('number', 1)])), ('source', CommentedMap([('fn', '<{ name|lower }}_<{ version }}.tar.gz'), ('url', 'https://github.com/infphilo/centrifuge/archive/f39767eb57d8e175029c.tar.gz'), ('md5', '17750d3be1039d2b428fe237a9cd7e45')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'zlib', 'perl', 'python']), ('run', ['libgcc', 'zlib', 'perl', 'python'])])), ('test', CommentedMap([('commands', ['centrifuge --help'])])), ('about', CommentedMap([('home', 'https://github.com/infphilo/centrifuge'), ('license', 'GPL3'), ('summary', 'Classifier for metagenomic sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sagx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sagx'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'SAGx_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SAGx_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/SAGx_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/SAGx/SAGx_1.50.0_src_all.tar.gz']), ('md5', 'abcc404aac2dafb403caefd16764ac41')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SAGx\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SAGx.html'), ('license', 'GPL-3'), ('summary', 'A package for retrieval, preparation and analysis of data from the Affymetrix GeneChip. In particular the issue of identifying differentially expressed genes is addressed.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-sagx/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-sagx'), ('version', '1.50.0')])), ('source', CommentedMap([('fn', 'SAGx_1.50.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/SAGx_1.50.0.tar.gz', 'https://bioarchive.galaxyproject.org/SAGx_1.50.0.tar.gz', 'https://depot.galaxyproject.org/software/SAGx/SAGx_1.50.0_src_all.tar.gz']), ('md5', 'abcc404aac2dafb403caefd16764ac41')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base']), ('run', ['bioconductor-biobase', 'bioconductor-multtest', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SAGx\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/SAGx.html'), ('license', 'GPL-3'), ('summary', 'A package for retrieval, preparation and analysis of data from the Affymetrix GeneChip. In particular the issue of identifying differentially expressed genes is addressed.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-noiseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-noiseq'), ('version', '2.18.0')])), ('source', CommentedMap([('fn', 'NOISeq_2.18.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-noiseq/bioconductor-noiseq_2.18.0_src_all.tar.gz'), ('sha256', '050e6ab51f91e9a5574eefb708596f8128f8aa1c7c120c8c3593021580ff8447')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.13.11', 'r-base', 'r-matrix']), ('run', ['bioconductor-biobase >=2.13.11', 'r-base', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NOISeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/NOISeq.html'), ('license', 'Artistic-2.0'), ('summary', 'Analysis of RNA-seq expression data or other similar kind of data. Exploratory plots to evualuate saturation, count distribution, expression per chromosome, type of detected features, features length, etc. Differential expression between two experimental conditions with no parametric assumptions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-noiseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-noiseq'), ('version', '2.18.0')])), ('source', CommentedMap([('fn', 'NOISeq_2.18.0.tar.gz'), ('url', 'https://depot.galaxyproject.org/software/bioconductor-noiseq/bioconductor-noiseq_2.18.0_src_all.tar.gz'), ('sha256', '050e6ab51f91e9a5574eefb708596f8128f8aa1c7c120c8c3593021580ff8447')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase >=2.13.11', 'r-base', 'r-matrix']), ('run', ['bioconductor-biobase >=2.13.11', 'r-base', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'NOISeq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/NOISeq.html'), ('license', 'Artistic-2.0'), ('summary', 'Analysis of RNA-seq expression data or other similar kind of data. Exploratory plots to evualuate saturation, count distribution, expression per chromosome, type of detected features, features length, etc. Differential expression between two experimental conditions with no parametric assumptions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methped/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methped'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'MethPed_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MethPed_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/MethPed_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/MethPed/MethPed_1.4.0_src_all.tar.gz']), ('md5', '56c270065cbe5f49423a0a598b5715ef')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-randomforest']), ('run', ['bioconductor-biobase', 'r-base', 'r-randomforest'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MethPed\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MethPed.html'), ('license', 'GPL-2'), ('summary', 'Classification of pediatric tumors into biologically defined subtypes is challenging and multifaceted approaches are needed. For this aim, we developed a diagnostic classifier based on DNA methylation profiles. We offer MethPed as an easy-to-use toolbox that allows researchers and clinical diagnosticians to test single samples as well as large cohorts for subclass prediction of pediatric brain tumors.  The current version of MethPed can classify the following tumor diagnoses/subgroups: Diffuse Intrinsic Pontine Glioma (DIPG), Ependymoma, Embryonal tumors with multilayered rosettes (ETMR), Glioblastoma (GBM), Medulloblastoma (MB) - Group 3 (MB_Gr3), Group 4 (MB_Gr3), Group WNT (MB_WNT), Group SHH (MB_SHH) and Pilocytic Astrocytoma (PiloAstro).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-methped/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-methped'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'MethPed_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/MethPed_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/MethPed_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/MethPed/MethPed_1.4.0_src_all.tar.gz']), ('md5', '56c270065cbe5f49423a0a598b5715ef')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'r-base', 'r-randomforest']), ('run', ['bioconductor-biobase', 'r-base', 'r-randomforest'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MethPed\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MethPed.html'), ('license', 'GPL-2'), ('summary', 'Classification of pediatric tumors into biologically defined subtypes is challenging and multifaceted approaches are needed. For this aim, we developed a diagnostic classifier based on DNA methylation profiles. We offer MethPed as an easy-to-use toolbox that allows researchers and clinical diagnosticians to test single samples as well as large cohorts for subclass prediction of pediatric brain tumors.  The current version of MethPed can classify the following tumor diagnoses/subgroups: Diffuse Intrinsic Pontine Glioma (DIPG), Ependymoma, Embryonal tumors with multilayered rosettes (ETMR), Glioblastoma (GBM), Medulloblastoma (MB) - Group 3 (MB_Gr3), Group 4 (MB_Gr3), Group WNT (MB_WNT), Group SHH (MB_SHH) and Pilocytic Astrocytoma (PiloAstro).')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-varianttools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-varianttools'), ('version', '1.18.1')])), ('source', CommentedMap([('fn', 'VariantTools_1.18.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/VariantTools_1.18.1.tar.gz', 'https://depot.galaxyproject.org/software/VariantTools/VariantTools_1.18.1_src_all.tar.gz']), ('md5', 'ef1a9ba2218f49e5dbaf3aba1d0b26b1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.2', 'bioconductor-rsamtools >=1.17.6', 'bioconductor-rtracklayer >=1.25.3', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation >=1.11.16', 'r-base', 'r-matrix']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.2', 'bioconductor-rsamtools >=1.17.6', 'bioconductor-rtracklayer >=1.25.3', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation >=1.11.16', 'r-base', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'VariantTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/VariantTools.html'), ('license', 'Artistic-2.0'), ('summary', 'Explore, diagnose, and compare variant calls using filters.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-varianttools/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-varianttools'), ('version', '1.18.1')])), ('source', CommentedMap([('fn', 'VariantTools_1.18.1.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/VariantTools_1.18.1.tar.gz', 'https://depot.galaxyproject.org/software/VariantTools/VariantTools_1.18.1_src_all.tar.gz']), ('md5', 'ef1a9ba2218f49e5dbaf3aba1d0b26b1')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.2', 'bioconductor-rsamtools >=1.17.6', 'bioconductor-rtracklayer >=1.25.3', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation >=1.11.16', 'r-base', 'r-matrix']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biocparallel', 'bioconductor-biostrings', 'bioconductor-bsgenome', 'bioconductor-genomeinfodb', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.17.7', 'bioconductor-iranges >=1.99.2', 'bioconductor-rsamtools >=1.17.6', 'bioconductor-rtracklayer >=1.25.3', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-variantannotation >=1.11.16', 'r-base', 'r-matrix'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'VariantTools\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/VariantTools.html'), ('license', 'Artistic-2.0'), ('summary', 'Explore, diagnose, and compare variant calls using filters.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yass/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'yass'), ('version', '1.14')])), ('source', CommentedMap([('fn', 'yass-1.14.tar.gz'), ('url', 'http://bioinfo.lifl.fr/yass/files/yass-current/yass-1.14.tar.gz'), ('sha256', '2ea4d2a32bb17fb6de590b0e8bce5231e2506b490b3456700b4bc029544a1982')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['yass --help'])])), ('about', CommentedMap([('home', 'http://bioinfo.lifl.fr/yass/'), ('license', 'CeCILL Free Software License'), ('summary', 'YASS is a genomic similarity search tool, for nucleic (DNA/RNA) sequences in fasta or plain text format.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/yass/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'yass'), ('version', '1.14')])), ('source', CommentedMap([('fn', 'yass-1.14.tar.gz'), ('url', 'http://bioinfo.lifl.fr/yass/files/yass-current/yass-1.14.tar.gz'), ('sha256', '2ea4d2a32bb17fb6de590b0e8bce5231e2506b490b3456700b4bc029544a1982')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('test', CommentedMap([('commands', ['yass --help'])])), ('about', CommentedMap([('home', 'http://bioinfo.lifl.fr/yass/'), ('license', 'CeCILL Free Software License'), ('summary', 'YASS is a genomic similarity search tool, for nucleic (DNA/RNA) sequences in fasta or plain text format.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pysam/0.9.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pysam'), ('version', '0.9.1')])), ('source', CommentedMap([('fn', 'pysam-0.9.1.tar.gz'), ('url', 'https://pypi.python.org/packages/40/15/20b22dc3d017ec123e533d062b982b111b0214168905de3221b5caf5f766/pysam-0.9.1.tar.gz'), ('md5', 'd87765ad1e0afe534255c20cce5478e4')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'htslib ==1.3.1', 'samtools ==1.3.1', 'bcftools ==1.3.1', 'cython', 'python', 'setuptools', 'zlib', 'curl']), ('run', ['libgcc', 'htslib ==1.3.1', 'samtools ==1.3.1', 'bcftools ==1.3.1', 'python', 'zlib', 'curl'])])), ('test', CommentedMap([('imports', ['pysam'])])), ('about', CommentedMap([('home', 'https://github.com/pysam-developers/pysam'), ('license', 'MIT'), ('summary', 'Pysam is a python module for reading and manipulating Samfiles. It is a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pysam/0.9.1/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pysam'), ('version', '0.9.1')])), ('source', CommentedMap([('fn', 'pysam-0.9.1.tar.gz'), ('url', 'https://pypi.python.org/packages/40/15/20b22dc3d017ec123e533d062b982b111b0214168905de3221b5caf5f766/pysam-0.9.1.tar.gz'), ('md5', 'd87765ad1e0afe534255c20cce5478e4')])), ('build', CommentedMap([('number', 2), ('skip', False)])), ('requirements', CommentedMap([('build', ['gcc', 'llvm', 'htslib ==1.3.1', 'samtools ==1.3.1', 'bcftools ==1.3.1', 'cython', 'python', 'setuptools', 'zlib', 'curl']), ('run', ['libgcc', 'htslib ==1.3.1', 'samtools ==1.3.1', 'bcftools ==1.3.1', 'python', 'zlib', 'curl'])])), ('test', CommentedMap([('imports', ['pysam'])])), ('about', CommentedMap([('home', 'https://github.com/pysam-developers/pysam'), ('license', 'MIT'), ('summary', 'Pysam is a python module for reading and manipulating Samfiles. It is a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-mixomics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-mixomics'), ('version', '6.1.1')])), ('source', CommentedMap([('fn', 'mixOmics_6.1.1.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/mixOmics/mixOmics_6.1.1.tar.gz']), ('sha256', '0a0b5cc2cb7ffcadcf2a5eab35c0f2d8cc9d1a60e7c83086a3909742c02bb471')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass', 'r-rcolorbrewer', 'r-corpcor', 'r-dplyr', 'r-ellipse', 'r-ggplot2', 'r-igraph', 'r-lattice', 'r-plyr', 'r-reshape2', 'r-rgl', 'r-tidyr', 'libglu']), ('run', ['r-base', 'r-mass', 'r-rcolorbrewer', 'r-corpcor', 'r-dplyr', 'r-ellipse', 'r-ggplot2', 'r-igraph', 'r-lattice', 'r-plyr', 'r-reshape2', 'r-rgl', 'r-tidyr', 'libglu'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mixOmics\\')\"', '\"%R%\" -e \"library(\\'mixOmics\\')\"'])])), ('about', CommentedMap([('home', 'http://www.mixOmics.org'), ('license', 'GPL (>= 2)'), ('summary', 'Multivariate methods are well suited to large omics data sets where the number of\\nvariables (e.g. genes, proteins, metabolites) is much larger than the number of\\nsamples (patients, cells, mice). They have the appealing properties of reducing\\nthe dimension of the data by using instrumental variables (components), which are\\ndefined as combinations of all variables. Those components are then used to produce\\nuseful graphical outputs that enable better understanding of the relationships and\\ncorrelation structures between the different data sets that are integrated. mixOmics\\noffers a wide range of multivariate methods for the exploration and integration\\nof biological datasets with a particular focus on variable selection. The package\\nproposes several sparse multivariate models we have developed to identify the key\\nvariables that are highly correlated, and/or explain the biological outcome of interest.\\nThe data that can be analysed with mixOmics may come from high throughput sequencing\\ntechnologies, such as omics data (transcriptomics, metabolomics, proteomics, metagenomics\\netc) but also beyond the realm of omics (e.g. spectral imaging). The methods implemented\\nin mixOmics can also handle missing values without having to delete entire rows\\nwith missing data. A non exhaustive list of methods include generalised Canonical\\nCorrelation Analysis, sparse Partial Least Squares and sparse Discriminant Analysis.\\nRecently we implemented integrative methods to combine multiple data sets: horizontal\\nintegration with regularised Generalised Canonical Correlation Analysis and vertical\\nintegration with multi-group Partial Least Squares.\\n'), ('license_family', 'GPL3')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-mixomics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'r-mixomics'), ('version', '6.1.1')])), ('source', CommentedMap([('fn', 'mixOmics_6.1.1.tar.gz'), ('url', ['http://cran.r-project.org/src/contrib/Archive/mixOmics/mixOmics_6.1.1.tar.gz']), ('sha256', '0a0b5cc2cb7ffcadcf2a5eab35c0f2d8cc9d1a60e7c83086a3909742c02bb471')])), ('build', CommentedMap([('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-mass', 'r-rcolorbrewer', 'r-corpcor', 'r-dplyr', 'r-ellipse', 'r-ggplot2', 'r-igraph', 'r-lattice', 'r-plyr', 'r-reshape2', 'r-rgl', 'r-tidyr', 'libglu']), ('run', ['r-base', 'r-mass', 'r-rcolorbrewer', 'r-corpcor', 'r-dplyr', 'r-ellipse', 'r-ggplot2', 'r-igraph', 'r-lattice', 'r-plyr', 'r-reshape2', 'r-rgl', 'r-tidyr', 'libglu'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mixOmics\\')\"', '\"%R%\" -e \"library(\\'mixOmics\\')\"'])])), ('about', CommentedMap([('home', 'http://www.mixOmics.org'), ('license', 'GPL (>= 2)'), ('summary', 'Multivariate methods are well suited to large omics data sets where the number of\\nvariables (e.g. genes, proteins, metabolites) is much larger than the number of\\nsamples (patients, cells, mice). They have the appealing properties of reducing\\nthe dimension of the data by using instrumental variables (components), which are\\ndefined as combinations of all variables. Those components are then used to produce\\nuseful graphical outputs that enable better understanding of the relationships and\\ncorrelation structures between the different data sets that are integrated. mixOmics\\noffers a wide range of multivariate methods for the exploration and integration\\nof biological datasets with a particular focus on variable selection. The package\\nproposes several sparse multivariate models we have developed to identify the key\\nvariables that are highly correlated, and/or explain the biological outcome of interest.\\nThe data that can be analysed with mixOmics may come from high throughput sequencing\\ntechnologies, such as omics data (transcriptomics, metabolomics, proteomics, metagenomics\\netc) but also beyond the realm of omics (e.g. spectral imaging). The methods implemented\\nin mixOmics can also handle missing values without having to delete entire rows\\nwith missing data. A non exhaustive list of methods include generalised Canonical\\nCorrelation Analysis, sparse Partial Least Squares and sparse Discriminant Analysis.\\nRecently we implemented integrative methods to combine multiple data sets: horizontal\\nintegration with regularised Generalised Canonical Correlation Analysis and vertical\\nintegration with multi-group Partial Least Squares.\\n'), ('license_family', 'GPL3')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))]))\n",
      "EXTRA has been found: mixomics\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-piano/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-piano'), ('version', '1.12.1')])), ('source', CommentedMap([('fn', 'piano_1.12.1.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/piano_1.12.1.tar.gz'), ('md5', '38972b1001c9c336cb0e2f6b5368c832')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-marray', 'r >=2.14.0', 'r-gplots', 'r-igraph', 'r-relations']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-marray', 'r >=2.14.0', 'r-gplots', 'r-igraph', 'r-relations', 'libgcc', 'llvm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'piano\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/piano.html'), ('license', 'GPL (>=2)'), ('summary', 'Piano performs gene set analysis using various statistical methods, from different gene level statistics and a wide range of gene-set collections. Furthermore, the Piano package contains functions for combining the results of multiple runs of gene set analyses.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-piano/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-piano'), ('version', '1.12.1')])), ('source', CommentedMap([('fn', 'piano_1.12.1.tar.gz'), ('url', 'http://bioconductor.org/packages/release/bioc/src/contrib/piano_1.12.1.tar.gz'), ('md5', '38972b1001c9c336cb0e2f6b5368c832')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-marray', 'r >=2.14.0', 'r-gplots', 'r-igraph', 'r-relations']), ('run', ['bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-marray', 'r >=2.14.0', 'r-gplots', 'r-igraph', 'r-relations', 'libgcc', 'llvm'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'piano\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/piano.html'), ('license', 'GPL (>=2)'), ('summary', 'Piano performs gene set analysis using various statistical methods, from different gene level statistics and a wide range of gene-set collections. Furthermore, the Piano package contains functions for combining the results of multiple runs of gene set analyses.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/novoalign/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'novoalign'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'novocraftV<{ version }}.tar.gz'), ('url', 'http://www.novocraft.com/homebrew/novocraftV<{ version }}.MacOSX.tar.gz'), ('md5', '1a0f98015fe0bf7c8e135132063b312f')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['novoalign 2>&1 | grep ^Novoalign'])])), ('about', CommentedMap([('home', 'http://www.novocraft.com/products/novoalign/'), ('license', 'Commercial (requires license for multithreaded use)'), ('summary', 'Powerful tool designed for mapping of short reads onto a reference genome from Illumina, Ion Torrent, and 454 NGS platforms')])), ('extra', CommentedMap([('notes', 'Novoalign runs in single-threaded mode by default unless a \"novoalign.lic\" license file is provided in the same directory as its binaries. The license file can be copied in to the conda environment via the \"novoalign-license-register\" command.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/novoalign/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'novoalign'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'novocraftV<{ version }}.tar.gz'), ('url', 'http://www.novocraft.com/homebrew/novocraftV<{ version }}.MacOSX.tar.gz'), ('md5', '1a0f98015fe0bf7c8e135132063b312f')])), ('build', CommentedMap([('number', 0), ('skip', False)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['novoalign 2>&1 | grep ^Novoalign'])])), ('about', CommentedMap([('home', 'http://www.novocraft.com/products/novoalign/'), ('license', 'Commercial (requires license for multithreaded use)'), ('summary', 'Powerful tool designed for mapping of short reads onto a reference genome from Illumina, Ion Torrent, and 454 NGS platforms')])), ('extra', CommentedMap([('notes', 'Novoalign runs in single-threaded mode by default unless a \"novoalign.lic\" license file is provided in the same directory as its binaries. The license file can be copied in to the conda environment via the \"novoalign-license-register\" command.')]))]))\n",
      "EXTRA has been found: Novoalign\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/multiqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'multiqc'), ('version', '1.2')])), ('source', CommentedMap([('fn', 'v1.2.tar.gz'), ('url', 'https://github.com/ewels/MultiQC/archive/v1.2.tar.gz'), ('md5', '55dac8b0dc055810ae09ec9b72e0603e')])), ('build', CommentedMap([('number', 0), ('preserve_egg_dir', True)])), ('requirements', CommentedMap([('build', ['python', 'click', 'future >0.14.0', 'lzstring', 'jinja2 >=2.9', 'matplotlib', 'markdown', 'numpy', 'pyyaml', 'requests', 'simplejson', 'spectra', 'networkx <2']), ('run', ['python', 'click', 'future >0.14.0', 'lzstring', 'jinja2 >=2.9', 'matplotlib', 'markdown', 'numpy', 'pyyaml', 'requests', 'simplejson', 'spectra', 'networkx <2', 'libgcc'])])), ('test', CommentedMap([('imports', ['multiqc', 'multiqc.modules', 'multiqc.modules.adapterRemoval', 'multiqc.modules.afterqc', 'multiqc.modules.bamtools', 'multiqc.modules.bcftools', 'multiqc.modules.bcl2fastq', 'multiqc.modules.biobloomtools', 'multiqc.modules.bismark', 'multiqc.modules.bowtie1', 'multiqc.modules.bowtie2', 'multiqc.modules.busco', 'multiqc.modules.clusterflow', 'multiqc.modules.conpair', 'multiqc.modules.custom_content', 'multiqc.modules.cutadapt', 'multiqc.modules.disambiguate', 'multiqc.modules.fastq_screen', 'multiqc.modules.fastqc', 'multiqc.modules.featureCounts', 'multiqc.modules.flexbar', 'multiqc.modules.gatk', 'multiqc.modules.goleft_indexcov', 'multiqc.modules.hicup', 'multiqc.modules.hisat2', 'multiqc.modules.homer', 'multiqc.modules.htseq', 'multiqc.modules.jellyfish', 'multiqc.modules.kallisto', 'multiqc.modules.leehom', 'multiqc.modules.macs2', 'multiqc.modules.methylQA', 'multiqc.modules.peddy', 'multiqc.modules.picard', 'multiqc.modules.preseq', 'multiqc.modules.prokka', 'multiqc.modules.qorts', 'multiqc.modules.qualimap', 'multiqc.modules.quast', 'multiqc.modules.rna_seqc', 'multiqc.modules.rseqc', 'multiqc.modules.salmon', 'multiqc.modules.samblaster', 'multiqc.modules.samtools', 'multiqc.modules.skewer', 'multiqc.modules.slamdunk', 'multiqc.modules.snpeff', 'multiqc.modules.sortmerna', 'multiqc.modules.star', 'multiqc.modules.theta2', 'multiqc.modules.tophat', 'multiqc.modules.trimmomatic', 'multiqc.modules.vcftools', 'multiqc.plots', 'multiqc.templates', 'multiqc.templates.default', 'multiqc.templates.geo', 'multiqc.templates.sections', 'multiqc.templates.simple', 'multiqc.utils']), ('commands', ['LC_ALL=C.UTF-8 LANG=C.UTF-8 multiqc --version', 'LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 multiqc --version'])])), ('about', CommentedMap([('home', 'http://multiqc.info'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Create aggregate bioinformatics analysis reports across many samples and tools'), ('license_family', 'GPL3')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/multiqc/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'multiqc'), ('version', '1.2')])), ('source', CommentedMap([('fn', 'v1.2.tar.gz'), ('url', 'https://github.com/ewels/MultiQC/archive/v1.2.tar.gz'), ('md5', '55dac8b0dc055810ae09ec9b72e0603e')])), ('build', CommentedMap([('number', 0), ('preserve_egg_dir', True)])), ('requirements', CommentedMap([('build', ['python', 'click', 'future >0.14.0', 'lzstring', 'jinja2 >=2.9', 'matplotlib', 'markdown', 'numpy', 'pyyaml', 'requests', 'simplejson', 'spectra', 'networkx <2']), ('run', ['python', 'click', 'future >0.14.0', 'lzstring', 'jinja2 >=2.9', 'matplotlib', 'markdown', 'numpy', 'pyyaml', 'requests', 'simplejson', 'spectra', 'networkx <2', 'libgcc'])])), ('test', CommentedMap([('imports', ['multiqc', 'multiqc.modules', 'multiqc.modules.adapterRemoval', 'multiqc.modules.afterqc', 'multiqc.modules.bamtools', 'multiqc.modules.bcftools', 'multiqc.modules.bcl2fastq', 'multiqc.modules.biobloomtools', 'multiqc.modules.bismark', 'multiqc.modules.bowtie1', 'multiqc.modules.bowtie2', 'multiqc.modules.busco', 'multiqc.modules.clusterflow', 'multiqc.modules.conpair', 'multiqc.modules.custom_content', 'multiqc.modules.cutadapt', 'multiqc.modules.disambiguate', 'multiqc.modules.fastq_screen', 'multiqc.modules.fastqc', 'multiqc.modules.featureCounts', 'multiqc.modules.flexbar', 'multiqc.modules.gatk', 'multiqc.modules.goleft_indexcov', 'multiqc.modules.hicup', 'multiqc.modules.hisat2', 'multiqc.modules.homer', 'multiqc.modules.htseq', 'multiqc.modules.jellyfish', 'multiqc.modules.kallisto', 'multiqc.modules.leehom', 'multiqc.modules.macs2', 'multiqc.modules.methylQA', 'multiqc.modules.peddy', 'multiqc.modules.picard', 'multiqc.modules.preseq', 'multiqc.modules.prokka', 'multiqc.modules.qorts', 'multiqc.modules.qualimap', 'multiqc.modules.quast', 'multiqc.modules.rna_seqc', 'multiqc.modules.rseqc', 'multiqc.modules.salmon', 'multiqc.modules.samblaster', 'multiqc.modules.samtools', 'multiqc.modules.skewer', 'multiqc.modules.slamdunk', 'multiqc.modules.snpeff', 'multiqc.modules.sortmerna', 'multiqc.modules.star', 'multiqc.modules.theta2', 'multiqc.modules.tophat', 'multiqc.modules.trimmomatic', 'multiqc.modules.vcftools', 'multiqc.plots', 'multiqc.templates', 'multiqc.templates.default', 'multiqc.templates.geo', 'multiqc.templates.sections', 'multiqc.templates.simple', 'multiqc.utils']), ('commands', ['LC_ALL=C.UTF-8 LANG=C.UTF-8 multiqc --version', 'LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8 multiqc --version'])])), ('about', CommentedMap([('home', 'http://multiqc.info'), ('license', 'GNU General Public License v3 (GPLv3)'), ('summary', 'Create aggregate bioinformatics analysis reports across many samples and tools'), ('license_family', 'GPL3')])), ('extra', CommentedMap([('container', CommentedMap([('extended-base', True)]))]))]))\n",
      "EXTRA has been found: multiqc\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ggbio/1.22.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ggbio'), ('version', '1.22.0')])), ('source', CommentedMap([('url', 'https://bioarchive.galaxyproject.org/ggbio_1.22.0.tar.gz'), ('sha256', '5ba61f3a2065de08a4ca888d6da3fee6aec5ca42bc5a47df64bd4574f4d81033')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-biovizbase >=1.19.1', 'bioconductor-bsgenome', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.21.10', 'bioconductor-iranges', 'bioconductor-organismdbi', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-ggally', 'r-ggplot2 >=1.0.0', 'r-gridextra', 'r-gtable', 'r-hmisc', 'r-reshape2', 'r-scales']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-biovizbase >=1.19.1', 'bioconductor-bsgenome', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.21.10', 'bioconductor-iranges', 'bioconductor-organismdbi', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-ggally', 'r-ggplot2 >=1.0.0', 'r-gridextra', 'r-gtable', 'r-hmisc', 'r-reshape2', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ggbio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ggbio.html'), ('license', 'Artistic-2.0'), ('summary', 'The ggbio package extends and specializes the grammar of graphics for biological data. The graphics are designed to answer common scientific questions, in particular those often asked of high throughput genomics data. All core Bioconductor data structures are supported, where appropriate. The package supports detailed views of particular genomic regions, as well as genome-wide overviews. Supported overviews include ideograms and grand linear views. High-level plots include sequence fragment length, edge-linked interval to data view, mismatch pileup, and several splicing summaries.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-ggbio/1.22.0/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-ggbio'), ('version', '1.22.0')])), ('source', CommentedMap([('url', 'https://bioarchive.galaxyproject.org/ggbio_1.22.0.tar.gz'), ('sha256', '5ba61f3a2065de08a4ca888d6da3fee6aec5ca42bc5a47df64bd4574f4d81033')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-biovizbase >=1.19.1', 'bioconductor-bsgenome', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.21.10', 'bioconductor-iranges', 'bioconductor-organismdbi', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-ggally', 'r-ggplot2 >=1.0.0', 'r-gridextra', 'r-gtable', 'r-hmisc', 'r-reshape2', 'r-scales']), ('run', ['bioconductor-annotationdbi', 'bioconductor-biobase', 'bioconductor-biocgenerics', 'bioconductor-biostrings', 'bioconductor-biovizbase >=1.19.1', 'bioconductor-bsgenome', 'bioconductor-ensembldb >=1.3.8', 'bioconductor-genomeinfodb >=1.1.3', 'bioconductor-genomicalignments >=1.1.16', 'bioconductor-genomicfeatures >=1.17.13', 'bioconductor-genomicranges >=1.21.10', 'bioconductor-iranges', 'bioconductor-organismdbi', 'bioconductor-rsamtools >=1.17.28', 'bioconductor-rtracklayer >=1.25.16', 'bioconductor-s4vectors >=0.9.25', 'bioconductor-summarizedexperiment', 'bioconductor-variantannotation >=1.11.4', 'r-base', 'r-ggally', 'r-ggplot2 >=1.0.0', 'r-gridextra', 'r-gtable', 'r-hmisc', 'r-reshape2', 'r-scales'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'ggbio\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/ggbio.html'), ('license', 'Artistic-2.0'), ('summary', 'The ggbio package extends and specializes the grammar of graphics for biological data. The graphics are designed to answer common scientific questions, in particular those often asked of high throughput genomics data. All core Bioconductor data structures are supported, where appropriate. The package supports detailed views of particular genomic regions, as well as genome-wide overviews. Supported overviews include ideograms and grand linear views. High-level plots include sequence fragment length, edge-linked interval to data view, mismatch pileup, and several splicing summaries.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mosaics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mosaics'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'mosaics_2.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/mosaics_2.10.0.tar.gz'), ('md5', '06465236065b08ebbab99d610f597e7d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-mass', 'r-base', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-base', 'r-mass', 'r-rcpp', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mosaics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/mosaics.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package provides functions for fitting MOSAiCS and MOSAiCS-HMM, a statistical framework to analyze one-sample or two-sample ChIP-seq data of transcription factor binding and histone modification.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-mosaics/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-mosaics'), ('version', '2.10.0')])), ('source', CommentedMap([('fn', 'mosaics_2.10.0.tar.gz'), ('url', 'https://bioarchive.galaxyproject.org/mosaics_2.10.0.tar.gz'), ('md5', '06465236065b08ebbab99d610f597e7d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-mass', 'r-base', 'r-rcpp', 'gcc', 'llvm']), ('run', ['bioconductor-genomeinfodb', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-iranges', 'bioconductor-rsamtools', 'bioconductor-s4vectors', 'r-base', 'r-mass', 'r-rcpp', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'mosaics\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/release/bioc/html/mosaics.html'), ('license', 'GPL (>= 2)'), ('summary', 'This package provides functions for fitting MOSAiCS and MOSAiCS-HMM, a statistical framework to analyze one-sample or two-sample ChIP-seq data of transcription factor binding and histone modification.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motiv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motiv'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'MotIV_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/MotIV_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/MotIV_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/MotIV/MotIV_1.30.0_src_all.tar.gz']), ('md5', '19da23b2900e7e9b334084a7bd78c17c')])), ('build', CommentedMap([('string', 'bioconductor_<{CONDA_R}}_gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}'), ('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-biostrings >=1.24.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rgadem', 'bioconductor-s4vectors', 'r-base', 'gsl <{CONDA_GSL}}*', 'r-lattice', 'gcc', 'llvm']), ('run', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-biostrings >=1.24.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rgadem', 'bioconductor-s4vectors', 'r-base', 'r-lattice', 'gsl <{CONDA_GSL}}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MotIV\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MotIV.html'), ('license', 'GPL-2'), ('summary', 'This package makes use of STAMP for comparing a set of motifs to a given database (e.g. JASPAR). It can also be used to visualize motifs, motif distributions, modules and filter motifs.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-motiv/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-motiv'), ('version', '1.30.0')])), ('source', CommentedMap([('fn', 'MotIV_1.30.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.4/bioc/src/contrib/MotIV_1.30.0.tar.gz', 'https://bioarchive.galaxyproject.org/MotIV_1.30.0.tar.gz', 'https://depot.galaxyproject.org/software/MotIV/MotIV_1.30.0_src_all.tar.gz']), ('md5', '19da23b2900e7e9b334084a7bd78c17c')])), ('build', CommentedMap([('string', 'bioconductor_<{CONDA_R}}_gsl<{CONDA_GSL}}_<{PKG_BUILDNUM}}'), ('number', 0), ('skip', True), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-biostrings >=1.24.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rgadem', 'bioconductor-s4vectors', 'r-base', 'gsl <{CONDA_GSL}}*', 'r-lattice', 'gcc', 'llvm']), ('run', ['bioconductor-biocgenerics >=0.1.0', 'bioconductor-biostrings >=1.24.0', 'bioconductor-iranges >=1.13.5', 'bioconductor-rgadem', 'bioconductor-s4vectors', 'r-base', 'r-lattice', 'gsl <{CONDA_GSL}}*', 'libgcc'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'MotIV\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/MotIV.html'), ('license', 'GPL-2'), ('summary', 'This package makes use of STAMP for comparing a set of motifs to a given database (e.g. JASPAR). It can also be used to visualize motifs, motif distributions, modules and filter motifs.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/repeatmasker/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'repeatmasker'), ('version', '4.0.6')])), ('source', CommentedMap([('fn', 'RepeatMasker-open-4-0-6.tar.gz'), ('url', 'http://www.repeatmasker.org/RepeatMasker-open-4-0-6.tar.gz'), ('md5', '324bcb707b23b06adfde34704a8ec0c4'), ('patches', ['patch1'])])), ('build', CommentedMap([('number', 9), ('detect_binary_files_with_prefix', False), ('has_prefix_files', ['bin/RepeatMasker']), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'rmblast']), ('run', ['rmblast', 'hmmer', 'trf', 'perl', 'perl-text-soundex'])])), ('test', CommentedMap([('commands', [\"RepeatMasker | grep 'RepeatMasker - Mask repetitive DNA' > /dev/null\"])])), ('about', CommentedMap([('home', 'http://www.repeatmasker.org'), ('license', 'Open Software License v2.1'), ('summary', 'RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/repeatmasker/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'repeatmasker'), ('version', '4.0.6')])), ('source', CommentedMap([('fn', 'RepeatMasker-open-4-0-6.tar.gz'), ('url', 'http://www.repeatmasker.org/RepeatMasker-open-4-0-6.tar.gz'), ('md5', '324bcb707b23b06adfde34704a8ec0c4'), ('patches', ['patch1'])])), ('build', CommentedMap([('number', 9), ('detect_binary_files_with_prefix', False), ('has_prefix_files', ['bin/RepeatMasker']), ('skip', True)])), ('requirements', CommentedMap([('build', ['perl', 'rmblast']), ('run', ['rmblast', 'hmmer', 'trf', 'perl', 'perl-text-soundex'])])), ('test', CommentedMap([('commands', [\"RepeatMasker | grep 'RepeatMasker - Mask repetitive DNA' > /dev/null\"])])), ('about', CommentedMap([('home', 'http://www.repeatmasker.org'), ('license', 'Open Software License v2.1'), ('summary', 'RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-sartools/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('url', 'https://github.com/PF2-pasteur-fr/SARTools/archive/v<{ version }}.tar.gz'), ('sha256', 'e35df9c200e3fbdde43c340a5c5426715e891cd6424e4d4ebe8201d4afb23035')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-knitr', 'bioconductor-deseq2', 'bioconductor-edger', 'r-rmarkdown']), ('run', ['r-base', 'r-knitr', 'bioconductor-deseq2', 'bioconductor-edger', 'r-rmarkdown'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SARTools\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/PF2-pasteur-fr/SARTools'), ('license', 'GPL-2'), ('summary', 'SARTools provides R tools and an environment for the statistical analysis of RNA-Seq projects load and clean data, produce figures, perform statistical analysis/testing with DESeq2 or edgeR, export results and create final report.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/r-sartools/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name|lower }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'v<{ version }}.tar.gz'), ('url', 'https://github.com/PF2-pasteur-fr/SARTools/archive/v<{ version }}.tar.gz'), ('sha256', 'e35df9c200e3fbdde43c340a5c5426715e891cd6424e4d4ebe8201d4afb23035')])), ('build', CommentedMap([('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['r-base', 'r-knitr', 'bioconductor-deseq2', 'bioconductor-edger', 'r-rmarkdown']), ('run', ['r-base', 'r-knitr', 'bioconductor-deseq2', 'bioconductor-edger', 'r-rmarkdown'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'SARTools\\')\"'])])), ('about', CommentedMap([('home', 'https://github.com/PF2-pasteur-fr/SARTools'), ('license', 'GPL-2'), ('summary', 'SARTools provides R tools and an environment for the statistical analysis of RNA-Seq projects load and clean data, produce figures, perform statistical analysis/testing with DESeq2 or edgeR, export results and create final report.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pybedtools/0.6.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pybedtools'), ('version', '0.6.9')])), ('source', CommentedMap([('url', 'https://github.com/daler/pybedtools/archive/v0.6.9.tar.gz'), ('fn', 'v0.6.9.tar.gz'), ('md5', 'a24e4bcd0c89beb9535295db964f6a4a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'six', 'samtools', 'setuptools', 'pandas', 'cython', 'gcc', 'llvm', 'zlib <{CONDA_ZLIB}}*']), ('run', ['python', 'bedtools', 'six', 'samtools', 'pandas', 'libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('imports', ['pybedtools']), ('commands', ['python -c \"import pybedtools; help(pybedtools.BedTool.intersect)\" | grep \"Original BEDTools help\"'])])), ('about', CommentedMap([('home', 'https://github.com/daler/pybedtools'), ('license', 'MIT'), ('summary', 'Wraps BEDTools for use in Python and adds many additional features.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/pybedtools/0.6.9/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'pybedtools'), ('version', '0.6.9')])), ('source', CommentedMap([('url', 'https://github.com/daler/pybedtools/archive/v0.6.9.tar.gz'), ('fn', 'v0.6.9.tar.gz'), ('md5', 'a24e4bcd0c89beb9535295db964f6a4a')])), ('build', CommentedMap([('number', 0), ('skip', True)])), ('requirements', CommentedMap([('build', ['python', 'six', 'samtools', 'setuptools', 'pandas', 'cython', 'gcc', 'llvm', 'zlib <{CONDA_ZLIB}}*']), ('run', ['python', 'bedtools', 'six', 'samtools', 'pandas', 'libgcc', 'zlib <{CONDA_ZLIB}}*'])])), ('test', CommentedMap([('imports', ['pybedtools']), ('commands', ['python -c \"import pybedtools; help(pybedtools.BedTool.intersect)\" | grep \"Original BEDTools help\"'])])), ('about', CommentedMap([('home', 'https://github.com/daler/pybedtools'), ('license', 'MIT'), ('summary', 'Wraps BEDTools for use in Python and adds many additional features.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tweedeseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tweedeseq'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'tweeDEseq_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/tweeDEseq_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/tweeDEseq_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/tweeDEseq/tweeDEseq_1.22.0_src_all.tar.gz']), ('md5', '0e960d8738383e8da9e8127a8d1701e7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-cqn', 'bioconductor-edger', 'bioconductor-limma', 'r-base', 'r-mass']), ('run', ['bioconductor-cqn', 'bioconductor-edger', 'bioconductor-limma', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'tweeDEseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/tweeDEseq.html'), ('license', 'GPL (>= 2)'), ('summary', 'Differential expression analysis of RNA-seq using the Poisson-Tweedie family of distributions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-tweedeseq/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-tweedeseq'), ('version', '1.22.0')])), ('source', CommentedMap([('fn', 'tweeDEseq_1.22.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/tweeDEseq_1.22.0.tar.gz', 'https://bioarchive.galaxyproject.org/tweeDEseq_1.22.0.tar.gz', 'https://depot.galaxyproject.org/software/tweeDEseq/tweeDEseq_1.22.0_src_all.tar.gz']), ('md5', '0e960d8738383e8da9e8127a8d1701e7')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-cqn', 'bioconductor-edger', 'bioconductor-limma', 'r-base', 'r-mass']), ('run', ['bioconductor-cqn', 'bioconductor-edger', 'bioconductor-limma', 'r-base', 'r-mass'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'tweeDEseq\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/tweeDEseq.html'), ('license', 'GPL (>= 2)'), ('summary', 'Differential expression analysis of RNA-seq using the Poisson-Tweedie family of distributions.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gffread/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'gffread-<{ version }}.OSX_x86_64.tar.gz'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/gffread-<{ version }}.OSX_x86_64.tar.gz'), ('sha256', '65c2bda3107dd3fce524bf61636e06e0db29113611850e0e1f23c19254691c50')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['gffread --version'])])), ('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/gff.shtml'), ('license', 'Artistic License 2.0'), ('summary', 'GFF/GTF utility providing format conversions, region filtering, FASTA sequence extraction and more.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/gffread/meta.yaml', CommentedMap([('package', CommentedMap([('name', '<{ name }}'), ('version', '<{ version }}')])), ('source', CommentedMap([('fn', 'gffread-<{ version }}.OSX_x86_64.tar.gz'), ('url', 'http://ccb.jhu.edu/software/stringtie/dl/gffread-<{ version }}.OSX_x86_64.tar.gz'), ('sha256', '65c2bda3107dd3fce524bf61636e06e0db29113611850e0e1f23c19254691c50')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', None), ('run', None)])), ('test', CommentedMap([('commands', ['gffread --version'])])), ('about', CommentedMap([('home', 'http://ccb.jhu.edu/software/stringtie/gff.shtml'), ('license', 'Artistic License 2.0'), ('summary', 'GFF/GTF utility providing format conversions, region filtering, FASTA sequence extraction and more.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bcrank/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bcrank'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'BCRANK_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BCRANK_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/BCRANK_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/BCRANK/BCRANK_1.38.0_src_all.tar.gz']), ('md5', '82e9e56be5f987e71722178ef22c0fcc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'r-base']), ('run', ['bioconductor-biostrings', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BCRANK\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BCRANK.html'), ('license', 'GPL-2'), ('summary', 'Functions and classes for de novo prediction of transcription factor binding consensus by heuristic search')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-bcrank/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-bcrank'), ('version', '1.38.0')])), ('source', CommentedMap([('fn', 'BCRANK_1.38.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/BCRANK_1.38.0.tar.gz', 'https://bioarchive.galaxyproject.org/BCRANK_1.38.0.tar.gz', 'https://depot.galaxyproject.org/software/BCRANK/BCRANK_1.38.0_src_all.tar.gz']), ('md5', '82e9e56be5f987e71722178ef22c0fcc')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biostrings', 'r-base']), ('run', ['bioconductor-biostrings', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'BCRANK\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/BCRANK.html'), ('license', 'GPL-2'), ('summary', 'Functions and classes for de novo prediction of transcription factor binding consensus by heuristic search')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-calib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-calib'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'CALIB_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CALIB_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/CALIB_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/CALIB/CALIB_1.42.0_src_all.tar.gz']), ('md5', '505bfa5712ec91529ed491a1ae14fc0d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base']), ('run', ['bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CALIB\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CALIB.html'), ('license', 'LGPL'), ('summary', 'This package contains functions for normalizing spotted microarray data, based on a physically motivated calibration model. The model parameters and error distributions are estimated from external control spikes.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-calib/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-calib'), ('version', '1.42.0')])), ('source', CommentedMap([('fn', 'CALIB_1.42.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/CALIB_1.42.0.tar.gz', 'https://bioarchive.galaxyproject.org/CALIB_1.42.0.tar.gz', 'https://depot.galaxyproject.org/software/CALIB/CALIB_1.42.0_src_all.tar.gz']), ('md5', '505bfa5712ec91529ed491a1ae14fc0d')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-limma', 'r-base']), ('run', ['bioconductor-limma', 'r-base'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'CALIB\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/CALIB.html'), ('license', 'LGPL'), ('summary', 'This package contains functions for normalizing spotted microarray data, based on a physically motivated calibration model. The model parameters and error distributions are estimated from external control spikes.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/maker/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'maker'), ('version', '2.31.9')])), ('source', CommentedMap([('fn', 'maker-2.31.9.tgz.tar.gz'), ('url', 'http://yandell.topaz.genetics.utah.edu/maker_downloads/static/maker-2.31.9.tgz'), ('md5', '63d79e37d25c60d236658f847b74ea18')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['augustus >=3.2.3', 'blast', 'exonerate', 'infernal', 'mir-prefer', 'repeatmasker', 'snap', 'snoscan', 'perl-bioperl', 'perl-bit-vector', 'perl-dbd-pg', 'perl-dbd-sqlite', 'perl-dbi', 'perl-file-which', 'perl-io-all', 'perl-io-prompt', 'perl-inline-c', 'perl-perlio-gzip', 'perl-forks', 'perl-perl-unsafe-signals', 'openmpi', 'trnascan-se', 'postgresql']), ('run', ['augustus >=3.2.3', 'blast', 'exonerate', 'infernal', 'mir-prefer', 'repeatmasker', 'snap', 'snoscan', 'perl-bioperl', 'perl-bit-vector', 'perl-dbd-pg', 'perl-dbd-sqlite', 'perl-dbi', 'perl-file-which', 'perl-io-all', 'perl-io-prompt', 'perl-inline-c', 'perl-perlio-gzip', 'perl-forks', 'perl-perl-unsafe-signals', 'openmpi', 'trnascan-se', 'postgresql'])])), ('about', CommentedMap([('home', 'http://www.yandell-lab.org/software/maker.html'), ('license', 'GPL3'), ('summary', 'MAKER is a portable and easily configurable genome annotation pipeline.')])), ('test', CommentedMap([('commands', ['maker --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/maker/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'maker'), ('version', '2.31.9')])), ('source', CommentedMap([('fn', 'maker-2.31.9.tgz.tar.gz'), ('url', 'http://yandell.topaz.genetics.utah.edu/maker_downloads/static/maker-2.31.9.tgz'), ('md5', '63d79e37d25c60d236658f847b74ea18')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['augustus >=3.2.3', 'blast', 'exonerate', 'infernal', 'mir-prefer', 'repeatmasker', 'snap', 'snoscan', 'perl-bioperl', 'perl-bit-vector', 'perl-dbd-pg', 'perl-dbd-sqlite', 'perl-dbi', 'perl-file-which', 'perl-io-all', 'perl-io-prompt', 'perl-inline-c', 'perl-perlio-gzip', 'perl-forks', 'perl-perl-unsafe-signals', 'openmpi', 'trnascan-se', 'postgresql']), ('run', ['augustus >=3.2.3', 'blast', 'exonerate', 'infernal', 'mir-prefer', 'repeatmasker', 'snap', 'snoscan', 'perl-bioperl', 'perl-bit-vector', 'perl-dbd-pg', 'perl-dbd-sqlite', 'perl-dbi', 'perl-file-which', 'perl-io-all', 'perl-io-prompt', 'perl-inline-c', 'perl-perlio-gzip', 'perl-forks', 'perl-perl-unsafe-signals', 'openmpi', 'trnascan-se', 'postgresql'])])), ('about', CommentedMap([('home', 'http://www.yandell-lab.org/software/maker.html'), ('license', 'GPL3'), ('summary', 'MAKER is a portable and easily configurable genome annotation pipeline.')])), ('test', CommentedMap([('commands', ['maker --version'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-image-info/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-image-info'), ('version', '1.38')])), ('source', CommentedMap([('fn', 'Image-Info-1.38.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SR/SREZIC/Image-Info-1.38.tar.gz'), ('md5', '0fa4db4bdf14b6f453655bafa62d766a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-io-stringy', 'perl-xml-libxml']), ('run', ['perl-threaded', 'perl-io-stringy', 'perl-xml-libxml'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Image-Info'), ('license', 'perl_5'), ('summary', 'Extract meta information from image files')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/perl-image-info/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'perl-image-info'), ('version', '1.38')])), ('source', CommentedMap([('fn', 'Image-Info-1.38.tar.gz'), ('url', 'https://cpan.metacpan.org/authors/id/S/SR/SREZIC/Image-Info-1.38.tar.gz'), ('md5', '0fa4db4bdf14b6f453655bafa62d766a')])), ('build', CommentedMap([('number', 0)])), ('requirements', CommentedMap([('build', ['perl-threaded', 'perl-io-stringy', 'perl-xml-libxml']), ('run', ['perl-threaded', 'perl-io-stringy', 'perl-xml-libxml'])])), ('test', None), ('about', CommentedMap([('home', 'http://metacpan.org/pod/Image-Info'), ('license', 'perl_5'), ('summary', 'Extract meta information from image files')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/swarm/1.2.19/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'swarm'), ('version', '1.2.19')])), ('source', CommentedMap([('fn', '1.2.19.tar.gz'), ('url', 'https://github.com/torognes/swarm/archive/1.2.19.tar.gz'), ('md5', '4d147b9c94b8966760d0b737499a53d2')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('commands', ['swarm_breaker.py --version', 'swarm -v 2>&1  | grep Swarm'])])), ('about', CommentedMap([('home', 'https://github.com/torognes/swarm'), ('license', 'Affero GPL'), ('summary', 'A robust and fast clustering method for amplicon-based studies.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/swarm/1.2.19/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'swarm'), ('version', '1.2.19')])), ('source', CommentedMap([('fn', '1.2.19.tar.gz'), ('url', 'https://github.com/torognes/swarm/archive/1.2.19.tar.gz'), ('md5', '4d147b9c94b8966760d0b737499a53d2')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('requirements', CommentedMap([('build', ['gcc']), ('run', ['python', 'libgcc'])])), ('test', CommentedMap([('commands', ['swarm_breaker.py --version', 'swarm -v 2>&1  | grep Swarm'])])), ('about', CommentedMap([('home', 'https://github.com/torognes/swarm'), ('license', 'Affero GPL'), ('summary', 'A robust and fast clustering method for amplicon-based studies.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-recoup/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-recoup'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'recoup_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/recoup_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/recoup_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/recoup/recoup_1.4.0_src_all.tar.gz']), ('md5', 'b05790f192cee6ed1bec0031f6bae3c5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-complexheatmap', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-rtracklayer', 'r-base', 'r-circlize', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-complexheatmap', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-rtracklayer', 'r-base', 'r-circlize', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'recoup\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/recoup.html'), ('license', 'GPL (>= 3)'), ('summary', 'recoup calculates and plots signal profiles created from short sequence reads derived from Next Generation Sequencing technologies. The profiles provided are either sumarized curve profiles or heatmap profiles. Currently, recoup supports genomic profile plots for reads derived from ChIP-Seq and RNA-Seq experiments. The package uses ggplot2 and ComplexHeatmap graphics facilities for curve and heatmap coverage profiles respectively.')]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/bioconductor-recoup/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'bioconductor-recoup'), ('version', '1.4.0')])), ('source', CommentedMap([('fn', 'recoup_1.4.0.tar.gz'), ('url', ['http://bioconductor.org/packages/3.5/bioc/src/contrib/recoup_1.4.0.tar.gz', 'https://bioarchive.galaxyproject.org/recoup_1.4.0.tar.gz', 'https://depot.galaxyproject.org/software/recoup/recoup_1.4.0_src_all.tar.gz']), ('md5', 'b05790f192cee6ed1bec0031f6bae3c5')])), ('build', CommentedMap([('number', 0), ('rpaths', ['lib/R/lib/', 'lib/'])])), ('requirements', CommentedMap([('build', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-complexheatmap', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-rtracklayer', 'r-base', 'r-circlize', 'r-ggplot2', 'r-plyr']), ('run', ['bioconductor-biocgenerics', 'bioconductor-biomart', 'bioconductor-complexheatmap', 'bioconductor-genomicalignments', 'bioconductor-genomicranges', 'bioconductor-rtracklayer', 'r-base', 'r-circlize', 'r-ggplot2', 'r-plyr'])])), ('test', CommentedMap([('commands', ['$R -e \"library(\\'recoup\\')\"'])])), ('about', CommentedMap([('home', 'http://bioconductor.org/packages/3.5/bioc/html/recoup.html'), ('license', 'GPL (>= 3)'), ('summary', 'recoup calculates and plots signal profiles created from short sequence reads derived from Next Generation Sequencing technologies. The profiles provided are either sumarized curve profiles or heatmap profiles. Currently, recoup supports genomic profile plots for reads derived from ChIP-Seq and RNA-Seq experiments. The package uses ggplot2 and ComplexHeatmap graphics facilities for curve and heatmap coverage profiles respectively.')]))]))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/intarna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'intarna'), ('version', '2.0.5')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software'), ('license', 'MIT'), ('summary', 'Efficient target prediction incorporating seeding and accessibility of interaction sites')])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'intarna-2.0.5.tar.gz'), ('url', 'https://github.com/BackofenLab/IntaRNA/releases/download/v2.0.5/intaRNA-2.0.5.tar.gz'), ('sha256', '64e107b94e0cf732556e96a1ae13b380f1009d36cdae1f31e001cf06c9007295')])), ('requirements', CommentedMap([('build', ['gcc', 'boost <{CONDA_BOOST}}*', 'llvmdev ==3.9.1', 'libcxx', 'boost ==1.63.0', 'viennarna >=2.3.0,<3', 'doxygen']), ('run', ['libgcc', 'libcxx', 'boost <{CONDA_BOOST}}*', 'boost ==1.63.0', 'perl'])])), ('test', CommentedMap([('commands', ['IntaRNA -h', 'IntaRNA -t CCCCCCCCGGGGGGGGGGGGGG -q CCCCCCC'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/intarna/meta.yaml', CommentedMap([('package', CommentedMap([('name', 'intarna'), ('version', '2.0.5')])), ('about', CommentedMap([('home', 'http://www.bioinf.uni-freiburg.de/Software'), ('license', 'MIT'), ('summary', 'Efficient target prediction incorporating seeding and accessibility of interaction sites')])), ('build', CommentedMap([('number', 0), ('string', 'boost<{CONDA_BOOST}}_<{PKG_BUILDNUM}}')])), ('source', CommentedMap([('fn', 'intarna-2.0.5.tar.gz'), ('url', 'https://github.com/BackofenLab/IntaRNA/releases/download/v2.0.5/intaRNA-2.0.5.tar.gz'), ('sha256', '64e107b94e0cf732556e96a1ae13b380f1009d36cdae1f31e001cf06c9007295')])), ('requirements', CommentedMap([('build', ['gcc', 'boost <{CONDA_BOOST}}*', 'llvmdev ==3.9.1', 'libcxx', 'boost ==1.63.0', 'viennarna >=2.3.0,<3', 'doxygen']), ('run', ['libgcc', 'libcxx', 'boost <{CONDA_BOOST}}*', 'boost ==1.63.0', 'perl'])])), ('test', CommentedMap([('commands', ['IntaRNA -h', 'IntaRNA -t CCCCCCCCGGGGGGGGGGGGGG -q CCCCCCC'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnastructure/5.7/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://rna.urmc.rochester.edu/RNAstructure.html'), ('license', ''), ('summary', 'RNAstructure is a complete package for RNA and DNA secondary structure  prediction and analysis. It includes algorithms for   secondary structure  prediction, including facility to predict base pairing probabilities. It  also can be used to predict bimolecular structures and can predict the  equilibrium binding affinity of an oligonucleotide to a structured RNA  target. This is useful for siRNA design. It can also predict secondary  structures common to two, unaligned sequences, which is much more accurate  than single sequence secondary structure prediction. Finally, RNAstructure  can take a number of different types of experiment mapping data to constrain  or restrain structure prediction. These include chemical mapping, enzymatic  mapping, NMR, and SHAPE data.')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'rnastructure'), ('version', '5.7')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'RNAstructure-5.7.tgz'), ('sha256', '7d7060c2180b95e0d0e0a840499df48de3571e37ca6bef51325377fe7ca28ae6'), ('url', 'http://depot.galaxyproject.org/package/source/RNAstructure-5.7.tgz')])), ('test', CommentedMap([('commands', ['refold -h', 'ProbKnot -h', 'dot2ct -h'])]))]))\n",
      "('/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes/rnastructure/5.7/meta.yaml', CommentedMap([('about', CommentedMap([('home', 'http://rna.urmc.rochester.edu/RNAstructure.html'), ('license', ''), ('summary', 'RNAstructure is a complete package for RNA and DNA secondary structure  prediction and analysis. It includes algorithms for   secondary structure  prediction, including facility to predict base pairing probabilities. It  also can be used to predict bimolecular structures and can predict the  equilibrium binding affinity of an oligonucleotide to a structured RNA  target. This is useful for siRNA design. It can also predict secondary  structures common to two, unaligned sequences, which is much more accurate  than single sequence secondary structure prediction. Finally, RNAstructure  can take a number of different types of experiment mapping data to constrain  or restrain structure prediction. These include chemical mapping, enzymatic  mapping, NMR, and SHAPE data.')])), ('build', CommentedMap([('number', 1), ('skip', True)])), ('package', CommentedMap([('name', 'rnastructure'), ('version', '5.7')])), ('requirements', CommentedMap([('build', ['gcc', 'llvm']), ('run', ['libgcc'])])), ('source', CommentedMap([('fn', 'RNAstructure-5.7.tgz'), ('sha256', '7d7060c2180b95e0d0e0a840499df48de3571e37ca6bef51325377fe7ca28ae6'), ('url', 'http://depot.galaxyproject.org/package/source/RNAstructure-5.7.tgz')])), ('test', CommentedMap([('commands', ['refold -h', 'ProbKnot -h', 'dot2ct -h'])]))]))\n"
     ]
    }
   ],
   "source": [
    "bioconda_recipes_path = \"/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-recipes/recipes\"\n",
    "mapping_file = \"/Users/yperez/IdeaProjects/github-repo/biodocker/bioconda-biotools/mapping_matchonly.csv\"\n",
    "yaml_parser = init_yaml_parser()\n",
    "mapping = get_annotation_mapping(mapping_file)\n",
    "yaml_files = get_bioconda_yalm(bioconda_recipes_path, yaml_parser)\n",
    "annotate_yamls(yaml_files, mapping, yaml_parser)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
